<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Cancers (Basel)</span><span class="iso-abbrev" title="iso-abbrev">Cancers (Basel)</span><span class="publisher-id" title="publisher-id">cancers</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Cancers</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2072-6694</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7072722">7072722</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/cancers12020422">10.3390/cancers12020422</a></span><span class="publisher-id" title="publisher-id">cancers-12-00422</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
MAL2-Induced Actin-Based Protrusion Formation is Anti-Oncogenic in Hepatocellular Carcinoma</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-8647-9590</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-Coral</span><span class="given-names" tagx="given-names" title="given-names">Alfonso</span></span><a href="#af1-cancers-12-00422">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-0791-402X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">del Vecchio</span><span class="given-names" tagx="given-names" title="given-names">Gianna-Jade</span></span><a href="#af1-cancers-12-00422">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chahine</span><span class="given-names" tagx="given-names" title="given-names">Joeffrey J.</span></span><a href="#af1-cancers-12-00422">1</a><a href="#af2-cancers-12-00422">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kallakury</span><span class="given-names" tagx="given-names" title="given-names">Bhaskar V.</span></span><a href="#af2-cancers-12-00422">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">Pamela L.</span></span><a href="#af1-cancers-12-00422">1</a><a href="#c1-cancers-12-00422">*</a></span></div><span class="citation_author_institution" id="af1-cancers-12-00422">[1], <span class="email" tagx="email" title="email">alfonso.lopezcoral@nih.gov</span><span class="email" tagx="email" title="email">giannajade.delvecchio@gmail.com</span><span class="email" tagx="email" title="email">Joeffrey.J.Chahine@gunet.georgetown.edu</span></span><span class="citation_author_institution" id="af2-cancers-12-00422">[2], <span class="email" tagx="email" title="email">KALLAKUB@gunet.georgetown.edu</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">tuma@cua.edu</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-2</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">422</span><span class="history" title="history"><span class="received" title="received">received: 2019-11-26</span><span class="accepted" title="accepted">accepted: 2020-2-05</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Recent studies report that the polarity gene myelin and lymphocyte protein 2 (MAL2), is overexpressed in multiple human carcinomas largely at the transcript level. Because chromosome 8q24 amplification (where MAL2 resides) is associated with hepatocellular- and cholangio-carcinomas, we examined MAL2 protein expression in these human carcinoma lesions and adjacent benign tissue using immunohistochemistry. For comparison, we analyzed renal cell carcinomas that are not associated with chromosome 8q24 amplification. Surprisingly, we found that MAL2 protein levels were decreased in the malignant tissues compared to benign in all three carcinomas, suggesting MAL2 expression may be anti-oncogenic. Consistent with this conclusion, we determined that endogenously overexpressed MAL2 in HCC-derived Hep3B cells or exogenously expressed MAL2 in hepatoma-derived Clone 9 cells (that lack endogenous MAL2) promoted actin-based protrusion formation with a reciprocal decrease in invadopodia. MAL2 overexpression also led to decreased cell migration, invasion and proliferation (to a more modest extent) while loss of MAL2 expression reversed the phenotypes. Mutational analysis revealed that a putative Ena/VASP homology 1 recognition site confers the MAL2-phenotype suggesting its role in tumor suppression involves actin remodeling. To reconcile decreased MAL2 protein expression in human carcinomas and its anti-oncogenic phenotypes with increased transcript levels, we propose a transcriptional regulatory model for MAL2 transient overexpression.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
hepatocellular carcinoma</div> <div class="kwd" title="kwd">
cholangiocarcinoma</div> <div class="kwd" title="kwd">
tumor suppressor</div> <div class="kwd" title="kwd">
MAL2</div> <div class="kwd" title="kwd">
actin</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Epithelial cells are vital for the success of humans. They line all bodily organs providing a selective barrier between the external and internal worlds. The plasma membrane of a polarized cell is physically continuous, but functionally and compositionally divided into two domains: the apical and basolateral [<a href="#B1-cancers-12-00422">1</a>]. This surface polarity is required for proper epithelial cell function. How is surface polarity established and maintained? This question is especially important given that more than 80% of human cancers are derived from polarized epithelial cells and because epithelial-derived cancer cells either lose or fail to achieve polarity. Thus, another important question is how does dysregulation of cell polarity contribute to malignant transformation? This question is of particular importance to us given that the dysregulated expression and/or function of a host of proteins that regulate cell polarity are known to contribute to cancer progression [<a href="#B2-cancers-12-00422">2</a>]. Prominently on that list is MAL2 (myelin and lymphocyte protein 2), a protein we have been studying for over 15 years.</p> <p>In that time, we have determined that the 19 kDa tetraspanning integral membrane protein, MAL2, is an important regulator of hepatic polarized protein sorting and have shown that it functions in basolateral secretion, basolateral membrane protein delivery and basolateral-to-apical transcytosis [<a href="#B3-cancers-12-00422">3</a>,<a href="#B4-cancers-12-00422">4</a>], but what is important to these studies is the finding that MAL2 expression is highly up-regulated up to 100-fold in a host of human cancers, including renal cell carcinoma, mesothelioma, neuroblastoma, cholangiocarcinoma, and cancers of the colon, liver, stomach, breast, ovary and pancreas [<a href="#B5-cancers-12-00422">5</a>,<a href="#B6-cancers-12-00422">6</a>,<a href="#B7-cancers-12-00422">7</a>,<a href="#B8-cancers-12-00422">8</a>,<a href="#B9-cancers-12-00422">9</a>,<a href="#B10-cancers-12-00422">10</a>,<a href="#B11-cancers-12-00422">11</a>,<a href="#B12-cancers-12-00422">12</a>,<a href="#B13-cancers-12-00422">13</a>,<a href="#B14-cancers-12-00422">14</a>]. High MAL2 expression has also been linked to bad prognosis in patients with colorectal or pancreatic cancer [<a href="#B5-cancers-12-00422">5</a>,<a href="#B13-cancers-12-00422">13</a>]. Genomic instability gains in chromosome 8q24 (where MAL2 resides) are associated with several epithelial cell-derived human cancers [<a href="#B15-cancers-12-00422">15</a>] and likely explains enhanced MAL2 transcript expression in at least a subset of these cancers. Although MAL2 overexpression is likely not an oncogenic driver of tumorigenesis [<a href="#B2-cancers-12-00422">2</a>], it may contribute to neoplastic transformation by altering protein targeting that leads to dysregulated protein distributions which ultimately results in changes in cell morphology, signaling and migration.</p> <p>Because we have been investigating MAL2’s biochemical and functional properties in polarized hepatic cells, we chose to assess its potential role in mediating carcinogenesis in hepatocellular carcinoma (HCC). Importantly, this cancer type is associated with chromosome 8q24 amplification [<a href="#B16-cancers-12-00422">16</a>,<a href="#B17-cancers-12-00422">17</a>,<a href="#B18-cancers-12-00422">18</a>]. We started our investigation with a blinded immunohistochemical analysis of MAL2 protein expression levels (rather than just transcript levels as for the majority of the other studies) in human HCC and adjacent benign tissue. For comparison, we examined MAL2 protein levels in two other cancer types: cholangiocarcinoma, another cancer type associated with chromosome 8q24 amplification [<a href="#B19-cancers-12-00422">19</a>,<a href="#B20-cancers-12-00422">20</a>], and in renal cell carcinoma, a cancer type not associated with amplified chromosome 8q24. Surprisingly we found that MAL2 protein expression was significantly decreased in all three carcinomas, more consistent with MAL2 function as a tumor suppressor. To better understand these unexpected results, we performed migration, invasion and proliferation assays in three hepatic-derived cell lines: polarized, differentiated WIF-B cells (with endogenous MAL2 expression), hepatocellular carcinoma-(HCC)-derived Hep3B cells (with endogenous MAL2 levels) and hepatoma-derived Clone9 cells (with no MAL2 expression). The approach was straightforward and in three parts: (1) overexpress MAL2 in WIF-B cells; (2) knock down MAL2 in Hep3B HCC cells; and (3) express MAL2 in Clone 9 hepatoma cells. From these studies we determined that MAL2-induced filopodia-like protrusion formation (likely at the expense of invadopodia and or lamellipodia formation) is anti-oncogenic. Mutational analysis further revealed that MAL2 anti-oncogenic properties are selectively mediated via a cytosolic, N-terminal putative recognition site for EVH1 (enabled, VASP, homology1) motifs present in the actin-associated Ena/VASP proteins. These results further indicate that MAL2’s possible role in tumor suppression involves actin remodeling. We propose a model to reconcile our findings of decreased MAL2 protein expression and its anti-oncogenic phenotypes with the findings from others that MAL2 transcripts are overexpressed in epithelial-derived cancers.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results</div> <div class="2.1.mal2expressionisdecreasedintumorsfromhumanhcc,ccandrcc" title="sec"> <div class="title" tagx="title" title="title">
2.1. MAL2 Expression is Decreased in Tumors from Human HCC, CC and RCC</div> <p>Because of our long-standing interest in defining the mechanisms by which MAL2 regulates polarized hepatic protein sorting, we chose to examine MAL2 expression levels in resected human tissues from HCC and for comparison, from CC (another liver-derived cancer) and RCC using an analytically validated immunohistochemistry (IHC) test for clinical and diagnostic applications. Importantly, both HCC and CC are associated with chromosome 8q24 amplification whereas RCC is not [<a href="#B16-cancers-12-00422">16</a>,<a href="#B17-cancers-12-00422">17</a>,<a href="#B18-cancers-12-00422">18</a>,<a href="#B19-cancers-12-00422">19</a>,<a href="#B20-cancers-12-00422">20</a>]. Also importantly, this approach analyzes MAL2 protein levels rather than just transcript levels as was done for the majority of the other studies. In all, 18 different HCC cases, 23 CC cases and 20 RCC cases containing both (adjacent) benign and tumor components were labeled with antibodies against MAL2 and Ki-67 (to assess the proliferative index). To our surprise, MAL2 protein expression was decreased in the malignant tumors even when viewed without magnification. As shown in <a href="#cancers-12-00422-f001">Figure 1</a>A, the decreased MAL2 labeling was apparent in the tumor component (T) of a patient liver biopsy when examined by eye. Using a blinded semi-quantitative light microscopic scoring method, we further determined that MAL2 expression was significantly down regulated in the majority of samples examined (in 61–65% of the total cases examined) from all three carcinoma types displaying 2- to 3-fold decreases in immunolabeling (<a href="#cancers-12-00422-f001">Figure 1</a>B).</p> <p>Examples of the MAL2 staining patterns for each tissue type is shown in <a href="#cancers-12-00422-f001">Figure 1</a>C. In general and as expected, MAL2 was robustly detected in the terminally differentiated, benign component of all three carcinoma types. When examined at higher magnification (insets), very dense regions of MAL2 labeling were observed. Also as expected, Ki-67 labeling (to mark proliferating cells) was low in the benign component with little to no nuclear staining observed. Also as expected, Ki-67 expression was enhanced in the corresponding tumor lesions with numerous positive nuclei observed. However, MAL2 labeling in the tumor lesions was decreased and no dense immunoreactive clusters were observed. When quantitated across all samples, we determined that MAL2 expression was significantly down-regulated (<i>p</i> ≤ 0.001) by approximately two-fold in the tumors (<a href="#cancers-12-00422-f001">Figure 1</a>D) with a corresponding two- to five-fold increase in Ki-67 labeling (<a href="#cancers-12-00422-f001">Figure 1</a>E). These results are independently and surprisingly more consistent with MAL2 functioning as a tumor suppressor.</p> </div> <div class="2.2.ourmodelsystems" title="sec"> <div class="title" tagx="title" title="title">
2.2. Our Model Systems</div> <p>To further examine whether MAL2 expression is potentially tumor suppressive, we assayed common oncogenic properties of three hepatic-derived cells: polarized, hepatic WIF-B cells, nonpolarized, HCC-derived Hep3B cells and nonpolarized, hepatoma-derived Clone 9 cells. We first labeled each cell type for filamentous actin with phalloidin to highlight its specific surface features (<a href="#cancers-12-00422-f002">Figure 2</a>A). WIF-B cells exhibit a typical polarized hepatic morphology with bile canalicular-like structures fully sequestered from the external milieu (marked with an asterisk) with a thick cortical actin web on the cytoplasmic surface of the apical and basolateral plasma membranes (<a href="#cancers-12-00422-f002">Figure 2</a>A(a)). In contrast, nonpolarized Hep3B cells are characterized by multiple, long, filopodia-like cell-surface protrusions (<a href="#cancers-12-00422-f002">Figure 2</a>A(b)). Clone 9 cells are also non-polarized, but display a cuboidal morphology with no actin-based protrusions (<a href="#cancers-12-00422-f002">Figure 2</a>A(c)). Semi-quantitative reverse transcriptase PCR (RT-PCR) confirmed MAL2 mRNA expression in WIF-B and Hep3B cells and the lack of endogenous MAL2 expression in Clone 9 cells (<a href="#cancers-12-00422-f002">Figure 2</a>B). Immunoblots from whole cell lysates indicated that protein levels mirrored the transcript levels with no endogenous MAL2 expression observed in Clone 9 cells. Because WIF-B cells express rat MAL2 and Hep3B cells express human MAL2, different antibodies were used to probe the lysates such that immunoreactivity cannot be directly compared between immunoblots or with the RT-PCR gels. Nonetheless, MAL2 was detected in both cell lysates (<a href="#cancers-12-00422-f002">Figure 2</a>C). As previously reported by us and others [<a href="#B4-cancers-12-00422">4</a>,<a href="#B21-cancers-12-00422">21</a>], MAL2 immunoreactive species in lysates from WIF-B and Hep3B cells were detected at 19 kDa (the predicted MW), 25 kDa (arrow) and a diffuse set of bands ranging from 30–50 kDa.</p> </div> <div class="2.3.mal2overexpressioninpolarizedwif-bcellsisnotoncogenic" title="sec"> <div class="title" tagx="title" title="title">
2.3. MAL2 Overexpression in Polarized WIF-B Cells Is Not Oncogenic</div> <p>To rule out that MAL2 overexpression alone can promote a loss of cell polarity and a more malignant phenotype, we overexpressed C-terminally FLAG-tagged wild type MAL2 in polarized WIF-B cells using recombinant adenovirus. Because hepatocytes favor the indirect route for the delivery of newly-synthesized apical plasma membrane-spanning proteins to the apical surface [<a href="#B1-cancers-12-00422">1</a>], it was not surprising to detect the constitutively expressed MAL2 throughout the biosynthetic pipeline and in the intermediates of transcytosis after 24 h of infection. As shown in <a href="#cancers-12-00422-f002">Figure 2</a>D(a), FLAG-tagged MAL2 was present in the ER (diffuse labeling), the Golgi (discrete, peri-nuclear staining marked with arrows), the basolateral membrane (en route to the apical surface) and at the apical domain itself (<a href="#cancers-12-00422-f002">Figure 2</a>D(a)). To determine MAL2’s steady state distributions, we treated cells with 50 µg/mL of cycloheximide (CHX) for up to 4 h and monitored the newly-synthesized cohort as it was chased through the biosynthetic pathway. In cells treated for 1 h, the diffuse MAL2 ER labeling and discrete Golgi labeling were lost with a reciprocal increase in MAL2 detection at the basolateral surface (<a href="#cancers-12-00422-f002">Figure 2</a>D(b)). After 3 h, MAL2 basolateral labeling was lost with increased staining observed just adjacent to the apical surface in the cap-like sub-apical compartment (SAC) (marked with arrows) and at the apical surface itself (<a href="#cancers-12-00422-f002">Figure 2</a>D(c)). After 4 h, only apical labeling was observed (<a href="#cancers-12-00422-f002">Figure 2</a>D(d)) confirming that overexpressed MAL2 is apically-located at steady state as for endogenously-expressed MAL2 [<a href="#B4-cancers-12-00422">4</a>]. This MAL2 "chase" from the ER-to-Golgi biosynthetic pipeline to the basolateral surface, to the SAC and finally to the apical surface further indicate that newly-synthesized MAL2 is delivered to the apical domain via the indirect pathway [<a href="#B22-cancers-12-00422">22</a>,<a href="#B23-cancers-12-00422">23</a>,<a href="#B24-cancers-12-00422">24</a>].</p> <p>If MAL2 overexpression is oncogenic, a simple prediction is that the polarized WIF-B phenotype should be lost. However, we observed no changes in the distributions (<a href="#cancers-12-00422-f002">Figure 2</a>E) or protein expression levels (<a href="#cancers-12-00422-f002">Figure 2</a>F) of the tight junction component, zonula occludens-1 (ZO-1). As shown in <a href="#cancers-12-00422-f002">Figure 2</a>E, ZO-1 labeling in the belt-like structures surrounding the canalicular cysts at the contact sites of adjacent cells was maintained. When quantitated, no differences in the numbers of bile canaliculi were observed between uninfected (33 bile canaliculi/100 cells) and MAL2 overexpressing cells (39 bile canaliculi/100 cells) further indicating that polarity was not altered by MAL2-overexpression. Another simple prediction is that if MAL2 overexpression is not oncogenic, it should not promote cell migration in scratch assays. Live cell imaging of wound healing confirmed this prediction and revealed no differences in cell migration between control and infected cells. In both cases, no significant wound closure was observed even after 18 h of recovery (<a href="#cancers-12-00422-f002">Figure 2</a>G). Quantitation of the gap areas confirmed the morphological observations. In both control and wild type MAL2-expressing cells, ~90% of the initial gap area remained (94.0 ± 0.3 and 89.3 ± 2.1%, respectively, <i>n</i> = 3) after 18 h of recovery. Together these results indicate that MAL2 overexpression alone is not oncogenic.</p> </div> <div class="2.4.mal2knockdowninhcc-derivedhep3bcellsdecreasedactin-basedprotrusions,yetenhancedcellmigration" title="sec"> <div class="title" tagx="title" title="title">
2.4. MAL2 Knockdown in HCC-Derived Hep3B Cells Decreased Actin-Based Protrusions, yet Enhanced Cell Migration</div> <p>To determine whether endogenous MAL2 expression correlates with tumor suppression, we continued our analysis in human HCC-derived Hep3B cells. For these studies we chose an anti-sense approach using recombinant adenovirus as we have described [<a href="#B4-cancers-12-00422">4</a>,<a href="#B25-cancers-12-00422">25</a>]. As observed for WIF-B cells [<a href="#B4-cancers-12-00422">4</a>,<a href="#B25-cancers-12-00422">25</a>], MAL2 in Hep3B cells is relatively long-lived, and even after 4 days post-infection with anti-sense MAL2 adenoviruses, only partial knockdown (~30% reduction) was achieved (<a href="#cancers-12-00422-f003">Figure 3</a>A). </p> <p>As a read-out for an oncogenic phenotype, we assayed wound closure in control and MAL2 knockdown cells. Control Hep3B cells displayed only modest wound closure after 18 h (<a href="#cancers-12-00422-f003">Figure 3</a>B, bottom left panel), and when quantitated, only ~20% of the gap was closed (<a href="#cancers-12-00422-f003">Figure 3</a>C). Hep3B cells with reduced MAL2 expression showed enhanced migration (<a href="#cancers-12-00422-f003">Figure 3</a>B, bottom right panel) with almost ~30% of the gap closed after 18 h of recovery (<a href="#cancers-12-00422-f003">Figure 3</a>C). Although modest, this increase in gap closure was significant (<i>p</i> ≤ 0.01 for 6 and 12 h and <i>p</i> ≤ 0.002 for 18 h) and consistent with the ~30% decrease in MAL2 expression observed.</p> <p>When micrographs of migrating cells at the leading edge were enlarged (from boxed areas indicated in <a href="#cancers-12-00422-f003">Figure 3</a>B), we noticed that the numerous cell surface protrusions in control Hep3B cells (marked by arrowheads) were absent in cells knocked down for MAL2 expression (<a href="#cancers-12-00422-f004">Figure 4</a>A). Similar results were observed when cells were monitored using indirect immunofluorescence at higher magnification. In control Hep3B cells, many filopodia-like cell surface protrusions were present in cells at the leading edge with endogenous MAL2 detected in the tips (<a href="#cancers-12-00422-f004">Figure 4</a>B(a,b)). In contrast, cells knocked down for MAL2 expression had many fewer protrusions and displayed a more cuboidal phenotype (<a href="#cancers-12-00422-f004">Figure 4</a>B(c,d)). When quantitated, we found that MAL2 knockdown cells had 70% fewer protrusions than control Hep3B cells (<a href="#cancers-12-00422-f004">Figure 4</a>C). We also noticed circular, dorsal ruffle-like structures in MAL2 knockdown cells (an example is shown in <a href="#cancers-12-00422-f004">Figure 4</a>D) further implying enhanced invasiveness. To further determine whether MAL2 expression correlates with filopodia-like protrusion formation, we went back to the WIF-B cells. As shown in <a href="#cancers-12-00422-f004">Figure 4</a>E, when imaged at the leading edge of the scratch, both control cells (with endogenous MAL2 expression) and cells overexpressing wild type MAL2 exhibited protrusions. For comparison, we determined that Hep3B cells overexpressing wild type MAL2 had no further protrusion formation than in uninfected cells (<a href="#cancers-12-00422-f004">Figure 4</a>F). Together these results indicate that MAL2 protein expression promotes protrusion formation, yet impairs cell migration.</p> </div> <div class="2.5.mal2expressioninclone9cellsinducedprotrusionformation,yetimpairedcellmigrationandinvasion" title="sec"> <div class="title" tagx="title" title="title">
2.5. MAL2 Expression in Clone 9 Cells Induced Protrusion Formation, yet Impaired Cell Migration and Invasion</div> <p>To further confirm that MAL2 expression promotes actin-based protrusion formation, yet impairs cell migration, we expressed wild type MAL2 in the hepatoma-derived Clone 9 cells. Importantly, these cells lack endogenous MAL2 expression. As described above, Clone 9 cells are characterized by a cuboidal morphology with few cell surface protrusions (<a href="#cancers-12-00422-f005">Figure 5</a>A(a–c)). In contrast, in cells overexpressing MAL2, the cuboidal morphology was lost and more elongated cells were observed with long, cell surface filopodia-like, actin-based protrusions (<a href="#cancers-12-00422-f005">Figure 5</a>A(d–f)). As for both Hep3B and WIF-B cells, MAL2 expression was observed at the protrusion tips (<a href="#cancers-12-00422-f005">Figure 5</a>A(d–f)). To determine MAL2’s steady state distributions in Clone 9 cells, we treated cells with 50 µg/mL of cycloheximide (CHX) for up to 4 h. As observed for WIF-B cells, MAL2 distributions before cycloheximide treatment were observed throughout the biosynthetic pipeline (arrowheads mark the Golgi) and at the cell surface (<a href="#cancers-12-00422-f005">Figure 5</a>B(a)). In cells treated for 1 h, the diffuse MAL2 ER labeling and discrete Golgi labeling were diminished with a reciprocal increase in MAL2 detection at the cell surface (<a href="#cancers-12-00422-f005">Figure 5</a>B(b)). After 3 h, MAL2 surface labeling was lost with increased staining observed in the apical compartment, a structure that labels exclusively for apical membrane proteins in nonpolarized cells as we and other have previously extensively described (<a href="#cancers-12-00422-f005">Figure 5</a>B(c)) [<a href="#B26-cancers-12-00422">26</a>]. After 4 h, labeling at the "apical compartment" and cell surface were observed consistent with the recycling reported for apical residents between these two compartments in nonpolarized cells also as we have previously described (<a href="#cancers-12-00422-f005">Figure 5</a>B(d)) [<a href="#B26-cancers-12-00422">26</a>,<a href="#B27-cancers-12-00422">27</a>,<a href="#B28-cancers-12-00422">28</a>].</p> <p>To confirm that the MAL2-induced protrusions are indeed actin-based structures, we treated cells with the actin depolymerizing agent, latrunculin B. In cells treated for only 15 min with 5 μM latrunculin B (lat B), all cell surface protrusions were lost (<a href="#cancers-12-00422-f005">Figure 5</a>C(b)). </p> <p>After 60 min of washout, the protrusions were reformed and cell morphology was restored (<a href="#cancers-12-00422-f005">Figure 5</a>C(c)) confirming the protrusions are actin-based. This further implies that MAL2 may interact with actin and/or actin-associated proteins to promote protrusion formation (see below).</p> <p>In scratch assays, Clone 9 cells (with no MAL2 expression) exhibited robust migration with near complete wound closure observed 18 h after the scratch (<a href="#cancers-12-00422-f006">Figure 6</a>A, bottom left panel). In contrast, wild type MAL2 expression led to visibly impaired migration (<a href="#cancers-12-00422-f006">Figure 6</a>A, bottom right panel). When quantitated, ~80% of the wound was closed after 18 h by uninfected Clone 9 cells whereas in cells expressing MAL2, only ~20% of the wound was closed (<a href="#cancers-12-00422-f006">Figure 6</a>B). This latter value is remarkably consistent with the rate of wound closure observed for Hep3B cells (that endogenously express MAL2). Also as for Hep3B cells, Clone 9 cells expressing wild type MAL2 at the leading edge of the scratch displayed abundant cell surface protrusions (marked with arrowheads) (<a href="#cancers-12-00422-f006">Figure 6</a>C(b)). We also examined whether MAL2 expression impaired cell migration and invasion using Boyden chamber assays. Consistent with results from the scratch assay, cells expressing wild type MAL2 displayed decreased migration through filters in Boyden chambers (<a href="#cancers-12-00422-f006">Figure 6</a>D). When quantitated, 30% fewer MAL2-expressing cells had traversed the filter (<a href="#cancers-12-00422-f006">Figure 6</a>E). Invasion across matrigel-coated filters was impaired to an even greater extent by MAL2 expression (<a href="#cancers-12-00422-f006">Figure 6</a>F) with up to 60% fewer cells recovered from the underside of the filter (<a href="#cancers-12-00422-f006">Figure 6</a>G).</p> <p>To confirm that MAL2 expression impaired the invasive phenotype, we analyzed Clone 9 cells for expression and distribution of selected invadopodia markers. Although no changes in tyrosine kinase substrate with five SH3 domains (Tks5), mammalian-enabled protein (Mena), vasodilator-stimulated phosphoprotein (VASP) or focal adhesion kinase (FAK) expression levels were observed on immunoblots of lysates from uninfected or MAL2-expressing Clone 9 cells (<a href="#cancers-12-00422-f007">Figure 7</a>A), decreased invadopodia were observed morphologically. In uninfected cells, both Tks5 and Mena localized to juxta-nuclear puncta characteristic of invadopodia labeling (<a href="#cancers-12-00422-f007">Figure 7</a>B(a,b)). In contrast, these puncta were lost in MAL2-expressing cells (<a href="#cancers-12-00422-f007">Figure 7</a>B(c,d)). Interestingly, Mena localized to the extreme distal tips of the MAL2-induced protrusions just past regions of the tips positive for MAL2 labeling (<a href="#cancers-12-00422-f007">Figure 7</a>D(c–e)) consistent with them being filopodia-like structures. We also measured relative cell numbers in uninfected and MAL2-overexpressing cells up to three days after plating. As shown in <a href="#cancers-12-00422-f007">Figure 7</a>C, cells expressing MAL2 displayed a modest, but significant decrease in absorbance levels after 1 (<i>p</i> ≤ 0.006), 2 (<i>p</i> ≤ 0.006) or 3 days suggesting decreased proliferation. Together these data are consistent with MAL2 function as a tumor suppressor.</p> </div> <div class="2.6.mal2tumorsuppressoractivityismediatedbyaputativeevh1recognitionmotif" title="sec"> <div class="title" tagx="title" title="title">
2.6. MAL2 Tumor Suppressor Activity is Mediated by a Putative EVH1 Recognition Motif</div> <p>The MAL2 cytoplasmically-oriented N-terminal domain is the most divergent among MAL family members. Only MAL2 encodes VPPPP and FPAP sequences (underlined) that resemble the F/L/W/YPPPP recognition sites for EVH1 (enabled, VASP, homology1) motifs present in Ena/VASP proteins [<a href="#B29-cancers-12-00422">29</a>] (<a href="#cancers-12-00422-f008">Figure 8</a>A). Because MAL2 expression promoted actin-based protrusion formation at the likely expense of invadopodia formation, we generated recombinant adenoviruses expressing FLAG-tagged MAL2 mutants with each of these sites mutated to alanines alone (VPPPP and FPAP) or together (double) (indicated in red) (<a href="#cancers-12-00422-f008">Figure 8</a>A). A FLAG epitope was added to the C-terminus for each mutant.</p> <p>To first confirm that the MAL2 mutants fold properly, we examined their distributions in polarized WIF-B cells. As shown in <a href="#cancers-12-00422-f008">Figure 8</a>B, all three mutants showed identical distributions to that of overexpressed wild type MAL2 (see <a href="#cancers-12-00422-f002">Figure 2</a>D for comparison). As for wild type, the mutants were detected throughout the biosynthetic pathway and at both surface domains. We next examined the morphology of Clone 9 cells expressing the three MAL2 mutants. Unlike Clone 9 cells expressing wild type MAL2, no cell surface protrusions were observed in cells expressing MAL2 with both putative EVH1 motifs mutated (double) (compare <a href="#cancers-12-00422-f008">Figure 8</a>C(a–c) with <a href="#cancers-12-00422-f008">Figure 8</a>C(j–l)). The wild type phenotype was also lost in cells expressing the FPAP mutant MAL2 (<a href="#cancers-12-00422-f008">Figure 8</a>C(g–i)), but not the VPPPP mutant (<a href="#cancers-12-00422-f008">Figure 8</a>C(d–f)) indicating that the FPAP motif selectively mediates MAL2-induced protrusion formation. As predicted, cells with enhanced surface protrusion formation (expressing wild type or the VPPPP mutant) displayed decreased motility in scratch assays than those without protrusions (expressing the FPAP or double mutant MAL2) (<a href="#cancers-12-00422-f009">Figure 9</a>A). When quantitated, cells expressing the FPAP or double mutant MAL2 had overlapping rates of gap closure as uninfected cells (~80–90% closure after 18h) (<a href="#cancers-12-00422-f009">Figure 9</a>D) compared to only 20% observed in cells expressing wild type or VPPPP mutant MAL2 (<a href="#cancers-12-00422-f009">Figure 9</a>D). </p> <p>When Clone 9 cells at the leading edge (<a href="#cancers-12-00422-f009">Figure 9</a>B) were imaged by indirect immunofluorescence similar changes in cell morphology were apparent with protrusion formation observed in cells expressing wild type or VPPPP mutant MAL2 that was absent in cells expressing the FPAP or double MAL2 mutants (<a href="#cancers-12-00422-f009">Figure 9</a>C). When cells were scored for cell surface protrusions, more than half of cells expressing wild type (69%) or the VPPPP mutant MAL2 (56%) were positive for actin-based protrusions while significantly fewer uninfected cells (1%) or cells expressing the FPAP (7%) or double MAL2 mutants (16%) displayed protrusions. We also we monitored relative cell numbers in control and infected Clone 9 cells. Modest, but significant decreases in absorbance reading after 1 (<i>p</i> ≤ 0.006 for WT and ≤ 0.01 for VPPPP), 2 (<i>p</i> ≤ 0.008 for WT and ≤ 0.024 for VPPPP) or 3 days were observed suggesting that decreased proliferation correlated with protrusion formation.</p> <p>Boyden chamber migration and invasion assays were fully consistent with these results. Clone 9 cells expressing wild type or the VPPPP mutant MAL2 displayed significantly reduced migration (<a href="#cancers-12-00422-f010">Figure 10</a>A) and invasion (<a href="#cancers-12-00422-f010">Figure 10</a>C) than observed for uninfected cells or those expressing the mutant FPAP MAL2. When quantitated, cells expressing wild type or the VPPPP mutant MAL2 displayed a significant decrease in migration (by ~40%) and invasion (by ~60%) with no changes in migration observed in uninfected cells or cells expressing the FPAP mutant MAL2 (<a href="#cancers-12-00422-f010">Figure 10</a>B,D). Together these data indicate that the FPAP (but not VPPPP) motif mediates MAL2 tumor suppressor activity.</p> <p>Based on these results, one possible explanation to reconcile MAL2’s function as a tumor suppressor with its upregulated expression and link to bad prognosis is that overexpressed MAL2 harbors mutations in the FPAP motif thereby abolishing its tumor suppressor activities. However, results from the NCI Genomic Data Commons (GDC) [<a href="#B30-cancers-12-00422">30</a>] indicate that no such MAL2 mutations have been observed in almost 8000 patients representing 18 different cancer types (<a href="#cancers-12-00422-f011">Figure 11</a>). In fact, MAL2 exhibits a strikingly low overall somatic mutation rate with mutations observed in an average of 0.75% of the 7689 cases analyzed (<a href="#cancers-12-00422-f011">Figure 11</a>). Of those few mutations, none fall within the divergent MAL2 N-terminal domain, but instead map mainly to the transmembrane domains. Thus, MAL2 mutation patterns and/or rates cannot resolve this paradox. However, based on our reading of the primary literature, we have formulated a possible explanation to resolve this paradox (see Discussion).</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
3. Discussion</div> <p>Blinded and independent analysis of human HCC, CC and RCC tissue revealed that MAL2 protein expression levels were surprisingly, significantly decreased in malignant lesions more consistent with MAL2 function as a tumor suppressor. Using three hepatic cell model systems, we determined that MAL2 protein overexpression was anti-oncogenic and led to decreased cell migration, invasion and viability (albeit to a lesser extent). MAL2 expression also led to the formation of filopodia-like actin-based protrusions with a reciprocal decrease in invadopodia and/or lamellipodia formation. From mutational analysis, we determined that the putative EVH1 recognition motif (FPAP) in the cytoplasmically-oriented MAL2 N-terminal domain mediates protrusion formation and decreased migration and invasion.</p> <div class="3.1.mal2-mediatedactinremodelingandtumorigenesis" title="sec"> <div class="title" tagx="title" title="title">
3.1. MAL2-Mediated Actin Remodeling and Tumorigenesis</div> <p>The mutational analysis presented here suggests that MAL2 coordinates with actin binding proteins to regulate protein sorting in both polarized cells and that in the non-polarized cell context, this leads to protrusion formation. Many lines of evidence support a link between MAL2 function in vesicle docking, budding and fusion and the actin network. We determined that MAL2 redistributes from the apical membrane to sub-apical puncta upon depolymerization of the cortical actin web [<a href="#B21-cancers-12-00422">21</a>]. Inverted formin 2 (INF2) was identified as a MAL2 binding partner in yeast two hybrid analysis [<a href="#B31-cancers-12-00422">31</a>]. Studies in polarized HepG2 cells suggested that INF2-mediated actin remodeling is likely required for MAL2’s role in transcytosis. More recently, we determined that MAL2 may coordinate with its known binding partner, serine threonine kinase 16, and the actin binding protein, WD repeat-containing protein 1, to regulate secretory vesicle docking and fusion at the basolateral surface [<a href="#B25-cancers-12-00422">25</a>].</p> <p>Based on our studies here, we further propose that MAL2-mediated actin remodeling may also be mediated by Ena/VASP proteins that drive barbed-end actin filament assembly, and that in the nonpolarized cell context, this drives filopodia-like protrusion formation. We further propose that this protrusion formation may antagonize the oncogenic phenotype. In the absence of MAL2, actin, actin-binding proteins and other proteins are directed to sites of invadopodia formation thereby promoting the oncogenic phenotype. MAL2 expression redirects these components away from the ventral invadopodia sites to the lateral edges. This is consistent with Mena redistribution from ventral puncta to the tips of the MAL2-induced protrusions. However, an open question is why these protrusions are associated with decreased migration and proliferation. One possibility is that MAL2 is inducing more adhesive filopodia formation at the expense of more Arp2/3-rich lamellipodia associated with more rapid migration [<a href="#B32-cancers-12-00422">32</a>]. Nonetheless, as long as MAL2 expression is maintained, full oncogenic transformation may be prevented. A possibility we are actively investigating.</p> </div> <div class="3.2.amal2paradox" title="sec"> <div class="title" tagx="title" title="title">
3.2. A MAL2 Paradox</div> <p>This work was motivated by the finding that MAL2 is upregulated in a host of human epithelial-derived cancers at least at the transcript level. However, we determined that MAL2 protein expression levels are decreased in HCC, CC and RCC, displays an extremely low somatic mutation rate in human cancers and its function is more consistent with it being a tumor suppressor leaving us with a paradox. How can MAL2’s function as a tumor suppressor be reconciled with its upregulated expression and link to bad prognosis in human cancers? The amplification of chromosomal 8q24 has been documented in many epithelial cell-derived cancers and is particularly associated with progression of HCC and CC [<a href="#B33-cancers-12-00422">33</a>,<a href="#B34-cancers-12-00422">34</a>]. Although <i>MAL2</i> resides in this amplified region (at 8q24.12), another gene on chromosome 8q24 has garnered much more attention, <i>c-MYC</i> (at 8q24.21) [<a href="#B33-cancers-12-00422">33</a>,<a href="#B34-cancers-12-00422">34</a>] (<a href="#cancers-12-00422-f012">Figure 12</a>). Myc dysregulation and overexpression is found in virtually all human epithelial-derived cancers and is often associated with poor prognosis [<a href="#B35-cancers-12-00422">35</a>,<a href="#B36-cancers-12-00422">36</a>]. It is widely appreciated that Myc overexpression leads to the transcription of countless genes that promote transformation and metastasis [<a href="#B35-cancers-12-00422">35</a>,<a href="#B36-cancers-12-00422">36</a>]. Further reinforcing this progression, Myc also represses other transcription factors that activate genes involved in tumor suppression, including Myc-interacting zinc finger protein-1, (Miz1, also known as Pias2 and ZBTB17) [<a href="#B35-cancers-12-00422">35</a>,<a href="#B37-cancers-12-00422">37</a>]. In normal cells, Myc dimerizes with Max while Miz1, at core promoter sequences, promotes expression of genes involved in cell-cell and cell-matrix adhesion, autophagy, apoptosis and polarity (including MAL2). In cancer cells with high c-Myc expression, Myc-Max complexes are additionally recruited to Miz1 at its promoter, displacing other factors thereby repressing Miz1 transcriptional activity leading to decreased apoptosis, cell adhesion, autophagy, and polarity [<a href="#B35-cancers-12-00422">35</a>,<a href="#B36-cancers-12-00422">36</a>,<a href="#B37-cancers-12-00422">37</a>].</p> <p>To resolve the "MAL2 Paradox", we propose the following scenario. In human HCC and CC, chromosome 8q24 is amplified leading to enhanced expression of both MAL2 and c-Myc. As the cancer progresses, Myc expression is further enhanced leading to Miz1 transcriptional repression and loss of Miz1-specific target expression—including MAL2. Thus, the simple prediction is that enhanced MAL2 expression is associated with earlier stages of cancer progression and its expression is repressed during later stages and in metastases as Myc protein expression increases. We are systematically determining the expression patterns of MAL2, c-Myc and Miz1 (and other selected targets) in HCC and CC with increasing grades of tumors to establish this temporal relationship (manuscript in preparation).</p> </div> </div> <div class="4.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
4. Materials and Methods</div> <div class="4.1.reagentsandantibodies" title="sec"> <div class="title" tagx="title" title="title">
4.1. Reagents and Antibodies</div> <p>F12 and F12 (Coon’s modification) medium, latrunculin B, cycloheximide (Cat. No. C7698-1G) was from Sigma-Aldrich (St. Louis, MO, USA). Fetal bovine serum (FBS) and newborn calf serum were from Gemini Bio-Products (Woodland, CA, USA). Anti-α-tubulin mouse monoclonal antibodies (DM1A, Cat. No. T9026, lot no. 052M4837), anti-FLAG mouse monoclonal antibodies, were from Sigma-Aldrich. Rabbit polyclonal antibodies against full length human MAL2 (Cat. No. ab75347), and VASP (Cat. No. ab47461) were from AbCam (San Francisco, CA, USA). Rabbit polyclonal antibodies against Tks5 (Fish (M-300)) were from Santa Cruz (Dallas, TX, USA) (Cat. sc-30122). Rabbit polyclonal antibodies against Mena (Cat. No. NBP1-87914) were from NOVUS (Littleton, CO, USA). Rabbit polyclonal antibodies against rat MAL2 were generated and affinity purified as described previously [<a href="#B4-cancers-12-00422">4</a>]. Monoclonal antibodies against rat zonula occludens-1 (ZO-1) were generously provided by Dr. Ann Hubbard (Johns Hopkins University School of Medicine, Baltimore, MD, USA) and are now part of our lab stocks. Alexa488 and -568-conjugated secondary antibodies were purchased from Invitrogen Life Technologies (Carlsbad, CA, USA) and HRP-conjugated secondary antibodies were from Sigma-Aldrich.</p> </div> <div class="4.2.humantissueimmunohistochemistry(ihc)staining" title="sec"> <div class="title" tagx="title" title="title">
4.2. Human Tissue Immunohistochemistry (IHC) Staining</div> <p>It should be noted that this evaluation was conducted and interpreted in a blinded fashion, with the clinical team having no knowledge of the results of other experimental findings. Paraffin-embedded tissues were cut into ~4 µm sections and pre-treated with heat-induced epitope retrieval (HIER) and diluted Envision FLEX Target Retrieval Solution at pH 6.0 (50×) (K8004) for Ki-67 (IR626, Clonal MIB-1, Agilent, Santa Clara, CA, USA) and MAL2 (Cat. No. ab75347, Abcam). Deparaffinization, rehydration and epitope retrieval were performed in DAKO PT Link (PT100/PT101) preheated to 85 °C followed by epitope retrieval at 97 °C for 20 min. Samples were cooled at 65 °C. Sections were soaked in diluted Envision FLEX Wash Buffer (20×) (K8007) for 5 min. Staining was performed using an Autostainer Link 48 DAKO. Sections were blocked with FLEX Peroxidase Blocking Solution (SM801) for 5 min. Anti-Ki-67 (ready-to-use) and anti-MAL2 (1:100) antibodies were separately added to different slides for 20 min followed by FLEX Mouse Linker (SM804) for 15 min, FLEX HRP (SM802) for 20 min, FLEX DAB with Substrate-Chromogen (SM803) for 10 min and FLEX hematoxylin (SM806) for 5 min. Between the different applications, the sections were washed for 5 min with diluted Envision FLEX Wash Buffer (20×) (K8007).</p> </div> <div class="4.3.densitometryandstatisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.3. Densitometry and Statistical Analysis</div> <p>Densitometric comparison of immunoreactive bands was performed using ImageJ 1.47v software (National Institutes of Health, Bethesda, MD, USA). In general, images were inverted and averaged pixel intensity x area for each band was determined, blank-corrected and normalized to an appropriate loading control (see legends for details). Results were expressed as the mean ± SEM from at least three independent experiments. Comparisons between experimental groups were made using the Student’s two-tailed <i>t</i> test for paired data. <i>P</i>-values ≤ 0.05 were considered statistically significant.</p> </div> <div class="4.4.studyapproval" title="sec"> <div class="title" tagx="title" title="title">
4.4. Study Approval</div> <p>Immunohistochemical staining of the HCC, RCC, and CC tissue sections was strictly performed for research and investigational purposes. The studies presented here are considered secondary research for which consent is not required. The samples were retrieved from a large bank of patient biospecimens that were previously collected. The information provided about the biospecimens was recorded by the original investigator in such a manner that the identity of the human subjects cannot readily be ascertained directly or through identifiers linked to the subjects. Also, the current investigators will not contact the subjects and will not re-identify subjects. The findings will not modify the patients’ initial diagnoses or alter their current course of treatment. For these reasons Institutional Review Board exemption was granted from MedStar Georgetown University Hospital (IRB exemption number: 2018-0022) and the Catholic University of America (IRB exemption number 19-0031). The authors would like to thank the patients and their families for providing the tissue samples that were provided with informed consent at the time of collection.</p> </div> <div class="4.5.cellculture" title="sec"> <div class="title" tagx="title" title="title">
4.5. Cell Culture</div> <p>WIF-B cells were grown in a humidified 7% CO<sub>2</sub> incubator at 37 °C as described [<a href="#B38-cancers-12-00422">38</a>]. Briefly, cells were cultured in F12 (Coon’s modification) medium supplemented with 5% fetal bovine serum (FBS), 10 μM hypoxanthine, 40 nM aminoterpin and 1.6 μM thymidine, pH 7.0. Clone 9 cells were cultured in F-12 (Coon’s modification) medium supplemented with 10% new born calf serum. Hep3B cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM) supplemented with 10% FBS. Both Hep3B and Clone 9 cell lines were grown in a humidified 5% CO<sub>2</sub> incubator at 37 °C. WIF-B cells were seeded onto glass coverslips at 1.3 × 10<sup>4</sup> cells/cm<sup>2</sup> and cultured for 8–12 days until they reached maximal density and polarity. Clone 9 and Hep3B cells were seeded onto glass coverslips in 6-well dishes at 0.5–1.0 × 10<sup>6</sup> cells/well and cultured for 1–2 days.</p> </div> <div class="4.6.rnaextractionandsemi-quantitativert-pcr" title="sec"> <div class="title" tagx="title" title="title">
4.6. RNA Extraction and Semi-Quantitative RT-PCR</div> <p>Total RNA was extracted and purified with the RNeasy Mini Kit (Qiagen, Germantown, MD, USA) according to the manufacturer’s instructions. Reverse-transcription (RT)-PCR was performed with 1 µg RNA template using OneStep RT-PCR kit (Qiagen) also according to the manufacturer’s instructions. PCR was performed using the following parameters: 30 min at 50 °C, 15 min at 95 °C, 30 cycles of 30 s at 94 °C, 30 s at 55 °C, 1 min at 72 °C, and 10 min at 72 °C. Primer sequences used for amplification were as follows: rat MAL2, forward 5′-ATGTCGGCCGGCGGAG-3′, reverse 5′-TTACGGTCGCCATCTTC GCA-3′; human MAL2, forward 5′-CATCCTGCGGACCTACTCG-3′, reverse 5′-CACGGACACAAA CATGACC-3′; rat α-tubulin, forward 5′-CCAGGGCTTCTTGGTTTTCC-3′, reverse 5′-CGCTCAATG TCGAGGTTTCT-3′; human α-tubulin, forward 5′-ACACTGTGGTTGAGCCCTACA-3′, reverse 5′-GCTTGAGGGTGCGGAAA-3′. PCR products were analyzed on 1% agarose gel containing ethidium bromide.</p> </div> <div class="4.7.virusproductionandinfection" title="sec"> <div class="title" tagx="title" title="title">
4.7. Virus Production and Infection</div> <p>Full-length rat MAL2 in a pCMV-SPORT6 vector was purchased from Life Technologies (Carlsbad, CA, USA). Recombinant adenoviruses encoding C-terminally FLAG epitope-tagged full-length wild-type, mutated VPPPP-to-AAAAA, mutated FPAP-to-AAAA, double mutated VPPPP/FPAP-to-AAAAA/AAAA MAL2 mutants and antisense MAL2 were generated using the ViraPower Adenoviral Expression System (Life Technologies), according to the manufacturer’s instructions and as described. All mutations were made using the Q5 Site-Directed Mutagenesis kit (New England Biolabs, Ipswich, MA, USA) according to the manufacturer’s instructions and verified by plasmid sequencing (Retrogen, San Diego, CA, USA). WIF-B, Clone 9 and or Hep3B cells were infected with recombinant adenovirus particles for 60 min at 37 °C as described [<a href="#B39-cancers-12-00422">39</a>]. Complete medium was added to the cells and they were incubated an additional 16–24 h to allow for protein expression.</p> </div> <div class="4.8.immunofluorescencemicroscopy" title="sec"> <div class="title" tagx="title" title="title">
4.8. Immunofluorescence Microscopy</div> <p>Control and infected cells were grown on glass coverslips were either fixed on ice with chilled PBS containing 4% paraformaldehyde (PFA) for 1 min and permeabilized with ice-cold methanol for 10 min or fixed at room temperature with 3.7% PFA for 10 min and permeabilized for 5 min with 0.5% Triton X-100 in PBS. For phalloidin (Life Technologies, product A12379) staining, cells were permeabilized with 3.7% PFA for 15 min and permeabilized for 5 min with 0.1% Triton X-100/PBS. Cells were labeled for 1 h at RT with phalloidin diluted in 1% BSA/PBS to 1:20. Coverslips were mounted in DAPI-conjugated mounting media onto slides. Cells were processed for indirect immunofluorescence as previously described [<a href="#B26-cancers-12-00422">26</a>] Alexa 488- or 568-conjugated secondary antibodies were used at 100 nM. Labeled cells were visualized at RT by epifluorescence with an Olympus BX60 Fluorescence Microscope (OPELCO, Dulles, DC, USA) using an UPlanFl 60×/NA 1.3, phase 1, oil immersion objective. Images were taken with an HQ2 CoolSnap digital camera (Roper Scientific, Berlin, Germany) and Metamorph Imaging software (Molecular Devices, Sunnyvale, CA, USA). Adobe Photoshop (Adobe Systems Inc., Mountain View, CA, USA) was used to process images and to compile figures.</p> </div> <div class="4.9.2dscratchassays" title="sec"> <div class="title" tagx="title" title="title">
4.9. 2D Scratch Assays</div> <p>Cells were seeded onto glass coverslips in tissue culture plates to confluency. Cell monolayers were wounded with a plastic pipette tip and phase-contrast images were captured using the CytoSMART live cell imaging system from Lonza (Basel, Switzerland) every 5 min for up to 24 h. Cell coverage and motility was measured using Lonza CytoSMART system software.</p> </div> <div class="4.10.boydenchambermigrationandinvasionassays" title="sec"> <div class="title" tagx="title" title="title">
4.10. Boyden Chamber Migration and Invasion Assays</div> <p>Migration and invasion assays were conducted in 24-well Transwell inserts (8 µm pore; Costar, Corning, NY, USA). For the migration assays, cells were trypsinized 24 h post-infection and resuspended in serum-free media. 4 × 10<sup>4</sup> cells were seeded in 100 µL of serum-free media in the upper chamber of the Transwell insert. Five hundred µL of complete media was added to the lower chamber as a chemoattractant. For the invasion assays, cells were trypsinized 24 h post-infection and resuspended in serum-free media. 1 × 10<sup>5</sup> cells/well were seeded in 100 µL of serum-free media in the upper chamber of the Transwell insert precoated with 50 µL matrigel (1:3 dilution, BD Biosciences, San Jose, CA, USA). 500 µL of complete media was added to the lower chamber as a chemoattractant. Cells were allowed to migrate and/or invade during an overnight incubation at 37 °C. Cells that did not migrate or invade were removed from the inside of the upper chamber with a cotton swab. Cells on the filter undersides were fixed in 4% paraformaldehyde for 10 min at room temperature, rinsed with 1× PBS and nuclei stained with DAPI-conjugated mounting media and mounted on glass slides. Stained cells were visualized by epifluorescence microscopy. Five randomly chosen fields were visualized using a 40× objective and cells counted. Data is presented as the percent of control migrating or invading cells.</p> </div> <div class="4.11.cellproliferationassay" title="sec"> <div class="title" tagx="title" title="title">
4.11. Cell Proliferation Assay</div> <p>Four thousand cells were seeded onto 96-well plates at the same 24 h post-infection in F12 (Coon’s modification) medium supplemented with 5% fetal bovine serum (FBS) for 3 days. Cell proliferation was determined using the MTT assay (Sigma-Aldrich) according to the manufacturer’s instructions.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
5. Conclusions</div> <p>Blinded and independent analysis of human HCC, CC and RCC tissue revealed that MAL2 protein expression levels were surprisingly, significantly decreased in malignant lesions more consistent with MAL2 function as a tumor suppressor. Using three hepatic cell model systems, we determined that MAL2 protein overexpression was anti-oncogenic and led to decreased cell migration, invasion and viability (to a lesser extent). MAL2 expression also led to the formation of filopodia-like actin-based protrusions with a reciprocal decrease in invadopodia and/or lamellipodia formation. From mutational analysis, we determined that the putative EVH1 recognition motif (FPAP) in the cytoplasmically-oriented MAL2 N-terminal domain mediates protrusion formation and decreased migration and invasion. To reconcile our findings of decreased MAL2 protein expression in human HCC, CC and RCC and its anti-oncogenic phenotypes with studies from others where increased MAL2 increased transcript levels were measured in a variety of human carcinoma types, we propose a transcriptional regulatory model for MAL2 transient overexpression.</p> </div> </div> <div class="back" title="back"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>A.L.-C., J.J.C., B.V.K. and P.L.T. are responsible for the experimental design. A.L.-C., J.J.C. and G.-J.d.V. performed all experiments. All authors analyzed and interpreted data and compiled figures. P.L.T. and A.L.-C. Wrote and edited the manuscript with assistance from G.-J.d.V., J.J.C. and B.V.K. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by the National Institutes of Health grants DK082890 and AA017626 awarded to P.L.T.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-cancers-12-00422" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcytosis: Crossing cellular barriers</span><span class="source" tagx="source" title="source">Physiol. Rev.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">871</span><span class="lpage" tagx="lpage" title="lpage">932</span><span class="pub-id"><a href="https://dx.doi.org/10.1152/physrev.00001.2003">10.1152/physrev.00001.2003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12843411">12843411</a></span></span></li> <li tag="ref"><a name="B2-cancers-12-00422" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldenring</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A central role for vesicle trafficking in epithelial neoplasia: Intracellular highways to carcinogenesis</span><span class="source" tagx="source" title="source">Nat. Rev. Cancer</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">813</span><span class="lpage" tagx="lpage" title="lpage">820</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrc3601">10.1038/nrc3601</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24108097">24108097</a></span></span></li> <li tag="ref"><a name="B3-cancers-12-00422" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">In</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Striz</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernad</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Serine/threonine kinase 16 and MAL2 regulate constitutive secretion of soluble cargo in hepatic cells</span><span class="source" tagx="source" title="source">Biochem. J.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">463</span><span class="fpage" tagx="fpage" title="fpage">201</span><span class="lpage" tagx="lpage" title="lpage">213</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/BJ20140468">10.1042/BJ20140468</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25084525">25084525</a></span></span></li> <li tag="ref"><a name="B4-cancers-12-00422" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">In</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MAL2 selectively regulates polymeric IgA receptor delivery from the Golgi to the plasma membrane in WIF-B cells</span><span class="source" tagx="source" title="source">Traffic</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">1056</span><span class="lpage" tagx="lpage" title="lpage">1066</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1600-0854.2010.01074.x">10.1111/j.1600-0854.2010.01074.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20444237">20444237</a></span></span></li> <li tag="ref"><a name="B5-cancers-12-00422" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eguchi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohuchida</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kozono</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikenaga</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shindo</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujiwara</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akagawa</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohtsuka</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahata</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">MAL2 expression predicts distant metastasis and short survival in pancreatic cancer</span><span class="source" tagx="source" title="source">Surgery</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">154</span><span class="fpage" tagx="fpage" title="fpage">573</span><span class="lpage" tagx="lpage" title="lpage">582</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.surg.2013.03.010">10.1016/j.surg.2013.03.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23876361">23876361</a></span></span></li> <li tag="ref"><a name="B6-cancers-12-00422" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoang</span><span class="given-names" tagx="given-names" title="given-names">C.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D’Cunha</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kratzke</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casmey</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frizelle</span><span class="given-names" tagx="given-names" title="given-names">S.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maddaus</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kratzke</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gene expression profiling identifies matriptase overexpression in malignant mesothelioma</span><span class="source" tagx="source" title="source">Chest</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">125</span><span class="fpage" tagx="fpage" title="fpage">1843</span><span class="lpage" tagx="lpage" title="lpage">1852</span><span class="pub-id"><a href="https://dx.doi.org/10.1378/chest.125.5.1843">10.1378/chest.125.5.1843</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15136399">15136399</a></span></span></li> <li tag="ref"><a name="B7-cancers-12-00422" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">F.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandian</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramraj</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Natarajan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aravindan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herman</span><span class="given-names" tagx="given-names" title="given-names">T.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aravindan</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Acquired genetic alterations in tumor cells dictate the development of high-risk neuroblastoma and clinical outcomes</span><span class="source" tagx="source" title="source">BMC Cancer</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">514</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12885-015-1463-y">10.1186/s12885-015-1463-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26159519">26159519</a></span></span></li> <li tag="ref"><a name="B8-cancers-12-00422" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marazuela</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Expression of MAL and MAL2, two elements of the protein machinery for raft-mediated transport, in normal and neoplastic human tissue</span><span class="source" tagx="source" title="source">Histol. Histopathol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">925</span><span class="lpage" tagx="lpage" title="lpage">933</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15168355">15168355</a></span></span></li> <li tag="ref"><a name="B9-cancers-12-00422" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weis</span><span class="given-names" tagx="given-names" title="given-names">V.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersen</span><span class="given-names" tagx="given-names" title="given-names">C.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mills</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitehead</span><span class="given-names" tagx="given-names" title="given-names">R.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldenring</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Establishment of novel in vitro mouse chief cell and SPEM cultures identifies MAL2 as a marker of metaplasia in the stomach</span><span class="source" tagx="source" title="source">Am. J. Physiol. Gastrointest. Liver Physiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">307</span><span class="fpage" tagx="fpage" title="fpage">G777</span><span class="lpage" tagx="lpage" title="lpage">G792</span><span class="pub-id"><a href="https://dx.doi.org/10.1152/ajpgi.00169.2014">10.1152/ajpgi.00169.2014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25190476">25190476</a></span></span></li> <li tag="ref"><a name="B10-cancers-12-00422" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoo</span><span class="given-names" tagx="given-names" title="given-names">H.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yun</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwon</span><span class="given-names" tagx="given-names" title="given-names">J.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahn</span><span class="given-names" tagx="given-names" title="given-names">H.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seol</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic and expression alterations in association with the sarcomatous change of cholangiocarcinoma cells</span><span class="source" tagx="source" title="source">Exp. Mol. Med.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="https://dx.doi.org/10.3858/emm.2009.41.2.013">10.3858/emm.2009.41.2.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19287191">19287191</a></span></span></li> <li tag="ref"><a name="B11-cancers-12-00422" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byrne</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maleki</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardy</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gloss</span><span class="given-names" tagx="given-names" title="given-names">B.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murali</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scurry</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fanayan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emmanuel</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hacker</span><span class="given-names" tagx="given-names" title="given-names">N.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutherland</span><span class="given-names" tagx="given-names" title="given-names">R.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MAL2 and tumor protein D52 (TPD52) are frequently overexpressed in ovarian carcinoma, but differentially associated with histological subtype and patient outcome</span><span class="source" tagx="source" title="source">BMC Cancer</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">497</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2407-10-497">10.1186/1471-2407-10-497</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20846453">20846453</a></span></span></li> <li tag="ref"><a name="B12-cancers-12-00422" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dasgupta</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tripathi</span><span class="given-names" tagx="given-names" title="given-names">P.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharya-Chatterjee</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valentino</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chatterjee</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of molecular targets for immunotherapy of patients with head and neck squamous cell carcinoma</span><span class="source" tagx="source" title="source">Oral Oncol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">306</span><span class="lpage" tagx="lpage" title="lpage">316</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.oraloncology.2005.08.007">10.1016/j.oraloncology.2005.08.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16321566">16321566</a></span></span></li> <li tag="ref"><a name="B13-cancers-12-00422" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The four-transmembrane protein MAL2 and tumor protein D52 (TPD52) are highly expressed in colorectal cancer and correlated with poor prognosis</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0178515</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0178515">10.1371/journal.pone.0178515</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28562687">28562687</a></span></span></li> <li tag="ref"><a name="B14-cancers-12-00422" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marazuela</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin-Belmonte</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Lopez</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aranda</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Marco</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Expression and distribution of MAL2, an essential element of the machinery for basolateral-to-apical transcytosis, in human thyroid epithelial cells</span><span class="source" tagx="source" title="source">Endocrinology</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">145</span><span class="fpage" tagx="fpage" title="fpage">1011</span><span class="lpage" tagx="lpage" title="lpage">1016</span><span class="pub-id"><a href="https://dx.doi.org/10.1210/en.2003-0652">10.1210/en.2003-0652</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14576188">14576188</a></span></span></li> <li tag="ref"><a name="B15-cancers-12-00422" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farazi</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePinho</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The genetic and environmental basis of hepatocellular carcinoma</span><span class="source" tagx="source" title="source">Discov. Med.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">182</span><span class="lpage" tagx="lpage" title="lpage">186</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17234139">17234139</a></span></span></li> <li tag="ref"><a name="B16-cancers-12-00422" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farazi</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePinho</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatocellular carcinoma pathogenesis: From genes to environment</span><span class="source" tagx="source" title="source">Nat. Rev. Cancer</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">674</span><span class="lpage" tagx="lpage" title="lpage">687</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrc1934">10.1038/nrc1934</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16929323">16929323</a></span></span></li> <li tag="ref"><a name="B17-cancers-12-00422" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">X.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The genetic and epigenetic alterations in human hepatocellular carcinoma: A recent update</span><span class="source" tagx="source" title="source">Protein Cell</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">673</span><span class="lpage" tagx="lpage" title="lpage">691</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13238-014-0065-9">10.1007/s13238-014-0065-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24916440">24916440</a></span></span></li> <li tag="ref"><a name="B18-cancers-12-00422" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlaeger</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longerich</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schiller</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bewerunge</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehrabi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toedt</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleeff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehemann</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eils</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lichter</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Etiology-dependent molecular mechanisms in human hepatocarcinogenesis</span><span class="source" tagx="source" title="source">Hepatology</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">511</span><span class="lpage" tagx="lpage" title="lpage">520</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/hep.22033">10.1002/hep.22033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18161050">18161050</a></span></span></li> <li tag="ref"><a name="B19-cancers-12-00422" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalmasso</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carpentier</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guettier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Camilleri-Broet</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borelli</span><span class="given-names" tagx="given-names" title="given-names">W.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campos Dos Santos</span><span class="given-names" tagx="given-names" title="given-names">C.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castaing</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duclos-Vallee</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broet</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Patterns of chromosomal copy-number alterations in intrahepatic cholangiocarcinoma</span><span class="source" tagx="source" title="source">BMC Cancer</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">126</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12885-015-1111-6">10.1186/s12885-015-1111-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25879652">25879652</a></span></span></li> <li tag="ref"><a name="B20-cancers-12-00422" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koo</span><span class="given-names" tagx="given-names" title="given-names">S.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ihm</span><span class="given-names" tagx="given-names" title="given-names">C.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwon</span><span class="given-names" tagx="given-names" title="given-names">K.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic alterations in hepatocellular carcinoma and intrahepatic cholangiocarcinoma</span><span class="source" tagx="source" title="source">Cancer Genet. Cytogenet.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">130</span><span class="fpage" tagx="fpage" title="fpage">22</span><span class="lpage" tagx="lpage" title="lpage">28</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0165-4608(01)00460-5">10.1016/S0165-4608(01)00460-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11672769">11672769</a></span></span></li> <li tag="ref"><a name="B21-cancers-12-00422" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramnarayanan</span><span class="given-names" tagx="given-names" title="given-names">S.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MAL, but not MAL2, expression promotes the formation of cholesterol-dependent membrane domains that recruit apical proteins</span><span class="source" tagx="source" title="source">Biochem. J.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">439</span><span class="fpage" tagx="fpage" title="fpage">497</span><span class="lpage" tagx="lpage" title="lpage">504</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/BJ20110803">10.1042/BJ20110803</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21732912">21732912</a></span></span></li> <li tag="ref"><a name="B22-cancers-12-00422" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartles</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feracci</span><span class="given-names" tagx="given-names" title="given-names">H.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stieger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biogenesis of the rat hepatocyte plasma membrane in vivo: Comparison of the pathways taken by apical and basolateral proteins using subcellular fractionation</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">1241</span><span class="lpage" tagx="lpage" title="lpage">1251</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.105.3.1241">10.1083/jcb.105.3.1241</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3654750">3654750</a></span></span></li> <li tag="ref"><a name="B23-cancers-12-00422" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartles</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plasma membrane protein sorting in epithelial cells: Do secretory pathways hold the key?</span><span class="source" tagx="source" title="source">Trends Biochem. Sci.</span><span class="year" tagx="year" title="year">1988</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">184</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0968-0004(88)90147-8">10.1016/0968-0004(88)90147-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3255200">3255200</a></span></span></li> <li tag="ref"><a name="B24-cancers-12-00422" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schell</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maurice</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stieger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">5’nucleotidase is sorted to the apical domain of hepatocytes via an indirect route [published erratum appears in J Cell Biol 1993 Nov;123(3):following 767]</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">1173</span><span class="lpage" tagx="lpage" title="lpage">1182</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.119.5.1173">10.1083/jcb.119.5.1173</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1447295">1447295</a></span></span></li> <li tag="ref"><a name="B25-cancers-12-00422" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez-Coral</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Striz</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A Serine/Threonine Kinase 16-Based Phospho-Proteomics Screen Identifies WD Repeat Protein-1 As A Regulator Of Constitutive Secretion</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">13049</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-31426-1">10.1038/s41598-018-31426-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30158666">30158666</a></span></span></li> <li tag="ref"><a name="B26-cancers-12-00422" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyasae</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nonpolarized cells selectively sort apical proteins from cell surface to a novel compartment, but lack apical retention mechanisms</span><span class="source" tagx="source" title="source">Mol. Biol. Cell</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">3400</span><span class="lpage" tagx="lpage" title="lpage">3415</span><span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.02-04-0054">10.1091/mbc.02-04-0054</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12388745">12388745</a></span></span></li> <li tag="ref"><a name="B27-cancers-12-00422" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schell</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharjee</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lutsenko</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin Vb mediates Cu+ export in polarized hepatocytes</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">1179</span><span class="lpage" tagx="lpage" title="lpage">1189</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.175307">10.1242/jcs.175307</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26823605">26823605</a></span></span></li> <li tag="ref"><a name="B28-cancers-12-00422" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyasae</span><span class="given-names" tagx="given-names" title="given-names">L.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuma</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcytotic efflux from early endosomes is dependent on cholesterol and glycosphingolipids in polarized hepatic cells</span><span class="source" tagx="source" title="source">Mol. Biol. Cell</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">2689</span><span class="lpage" tagx="lpage" title="lpage">2705</span><span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.e02-12-0816">10.1091/mbc.e02-12-0816</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12857857">12857857</a></span></span></li> <li tag="ref"><a name="B29-cancers-12-00422" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renfranz</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beckerle</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Doing (F/L)PPPPs: EVH1 domains and their proline-rich partners in cell polarity and migration</span><span class="source" tagx="source" title="source">Curr. Opin. Cell Biol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">88</span><span class="lpage" tagx="lpage" title="lpage">103</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0955-0674(01)00299-X">10.1016/S0955-0674(01)00299-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11792550">11792550</a></span></span></li> <li tag="ref"><a name="B30-cancers-12-00422" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grossman</span><span class="given-names" tagx="given-names" title="given-names">R.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heath</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferretti</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varmus</span><span class="given-names" tagx="given-names" title="given-names">H.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowy</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kibbe</span><span class="given-names" tagx="given-names" title="given-names">W.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Staudt</span><span class="given-names" tagx="given-names" title="given-names">L.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Toward a Shared Vision for Cancer Genomic Data</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">375</span><span class="fpage" tagx="fpage" title="fpage">1109</span><span class="lpage" tagx="lpage" title="lpage">1112</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMp1607591">10.1056/NEJMp1607591</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27653561">27653561</a></span></span></li> <li tag="ref"><a name="B31-cancers-12-00422" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madrid</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aranda</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Fraticelli</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ventimiglia</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andres-Delgado</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shehata</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fanayan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shahheydari</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jimenez</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The formin INF2 regulates basolateral-to-apical transcytosis and lumen formation in association with Cdc42 and MAL2</span><span class="source" tagx="source" title="source">Dev. Cell</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">814</span><span class="lpage" tagx="lpage" title="lpage">827</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2010.04.001">10.1016/j.devcel.2010.04.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20493814">20493814</a></span></span></li> <li tag="ref"><a name="B32-cancers-12-00422" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rotty</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haynes</span><span class="given-names" tagx="given-names" title="given-names">E.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suarez</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winkelman</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">H.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haugh</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovar</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Profilin-1 serves as a gatekeeper for actin assembly by Arp2/3-dependent and -independent pathways</span><span class="source" tagx="source" title="source">Dev. Cell</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">54</span><span class="lpage" tagx="lpage" title="lpage">67</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2014.10.026">10.1016/j.devcel.2014.10.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25543281">25543281</a></span></span></li> <li tag="ref"><a name="B33-cancers-12-00422" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brisbin</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asmann</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">Y.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aakre</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jenkins</span><span class="given-names" tagx="given-names" title="given-names">R.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pharoah</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schumacher</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conti</span><span class="given-names" tagx="given-names" title="given-names">D.V.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Meta-analysis of 8q24 for seven cancers reveals a locus between NOV and ENPP2 associated with cancer development</span><span class="source" tagx="source" title="source">BMC Med. Genet.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">156</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2350-12-156">10.1186/1471-2350-12-156</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22142333">22142333</a></span></span></li> <li tag="ref"><a name="B34-cancers-12-00422" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huppi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pitt</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wahlberg</span><span class="given-names" tagx="given-names" title="given-names">B.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caplen</span><span class="given-names" tagx="given-names" title="given-names">N.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The 8q24 gene desert: An oasis of non-coding transcriptional activity</span><span class="source" tagx="source" title="source">Front. Genet.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fgene.2012.00069">10.3389/fgene.2012.00069</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22558003">22558003</a></span></span></li> <li tag="ref"><a name="B35-cancers-12-00422" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adhikary</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eilers</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptional regulation and transformation by Myc proteins</span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">635</span><span class="lpage" tagx="lpage" title="lpage">645</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm1703">10.1038/nrm1703</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16064138">16064138</a></span></span></li> <li tag="ref"><a name="B36-cancers-12-00422" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalkat</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Melo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hickman</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lourenco</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Redel</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Resetca</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamachi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tu</span><span class="given-names" tagx="given-names" title="given-names">W.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penn</span><span class="given-names" tagx="given-names" title="given-names">L.Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MYC Deregulation in Primary Human Cancers</span><span class="source" tagx="source" title="source">Genes</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">151</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/genes8060151">10.3390/genes8060151</a></span></span></li> <li tag="ref"><a name="B37-cancers-12-00422" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolf</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">C.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eilers</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levens</span><span class="given-names" tagx="given-names" title="given-names">D.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Taming of the beast: Shaping Myc-dependent amplification</span><span class="source" tagx="source" title="source">Trends Cell Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">241</span><span class="lpage" tagx="lpage" title="lpage">248</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tcb.2014.10.006">10.1016/j.tcb.2014.10.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25475704">25475704</a></span></span></li> <li tag="ref"><a name="B38-cancers-12-00422" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shanks</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassio</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lecoq</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">An improved rat hepatoma hybrid cell line. Generation and comparison with its hepatoma relatives and hepatocytes in vivo</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">107</span><span class="fpage" tagx="fpage" title="fpage">813</span><span class="lpage" tagx="lpage" title="lpage">825</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8056838">8056838</a></span></span></li> <li tag="ref"><a name="B39-cancers-12-00422" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bastaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braiterman</span><span class="given-names" tagx="given-names" title="given-names">L.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johns</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubbard</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Absence of direct delivery for single transmembrane apical proteins or their "Secretory" forms in polarized hepatic cells</span><span class="source" tagx="source" title="source">Mol. Biol. Cell.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">225</span><span class="lpage" tagx="lpage" title="lpage">237</span><span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.01-07-0376">10.1091/mbc.01-07-0376</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11809835">11809835</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>MAL2 expression patterns in human cancer tissue are consistent with its role as a tumor suppressor. (<b>A</b>) A liver biopsy was immunostained with diagnostically validated MAL2 antibodies. An unmagnified image of the biopsy mounted on the slide is shown. The areas containing the benign (B) and tumor (T) components are indicated. (<b>B</b>) Tumors from hepatocellular carcinoma (HCC), cholangiocarcinoma (CC) and renal cell carcinoma (RCC) biopsies were scored for MAL2 expression. Numbers of patient tumor samples that displayed 2- or 3-fold decreases in MAL2 protein expression are indicated. The total number of cases examined is shown (n) and the percent of cases examined that displayed decreased MAL2 protein expression is indicated. (<b>C</b>) Benign and cancerous lesions from human tissues from HCC, CC and RCC as indicated were immunostained with diagnostically validated MAL2 and Ki-67 antibodies. The black boxes indicate the regions of the micrograph enlarged insets. The fold-decrease in MAL2 expression (<b>D</b>) and fold-increase in Ki-67 expression (<b>E</b>) in cancerous lesions relative to benign across all patient biopsies are plotted. Values represent the mean ± SEM from 18 HCC patients, 23 CC patients and 20 RCC patients. ** <i>p</i> ≤ 0.001.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>MAL2 is expressed in normal and malignant liver-derived cell lines, but simply overexpressing MAL2 in polarized WIF-B cells is not oncogenic. (<b>A</b>) WIF-B (a), Hep3B (b) and Clone 9 cells (c) were labeled for actin with phalloidin. (<b>B</b>) Agarose gels are shown of MAL2 (upper panels) and α-tubulin (lower panels) cDNA amplified by RT-PCR from 1 µg total RNA isolated from WIF-B, Clone 9 or Hep3B cells as indicated. Human-specific MAL2 and α-tubulin primers were used for Hep3B cell amplification while rat-specific primers were used for WIF-B and Clone 9 cells. Numbers below the lanes represent the ratio of MAL2 mRNA expression levels normalized to α-tubulin expression levels. (<b>C</b>) Lysates from WIF-B and Clone 9 cells were immunoblotted with antibodies specific for rat MAL2 and lysates from Hep3B cells were immunoblotted for human MAL2. Molecular weight standards are indicated on the left in kDa. The bottom arrow marks the predicted 19 kDa MAL2 immunoreactive species. The bracket highlights a diffuse set of bands that has been described by us and others and the upper arrow indicates a 25 kDa species also detected by others. (<b>D</b>) WIF-B cells expressing FLAG-tagged wild type (WT) MAL2 were treated with 50 µg/mL of cycloheximide (CHX) for up to 4 h as indicated and immunolabeled for MAL2 with anti-FLAG antibodies. Arrowheads indicate MAL2 localization at the Golgi (a), basolateral membrane (b) SAC (c) or apical surface (d). Asterisks mark bile canaliculi. (<b>E</b>) Uninfected WIF-B cells and cells overexpressing wild type MAL2 were immunolabeled for ZO-1. Asterisks indicate bile canaliculi. (<b>F</b>) Lysates from uninfected or MAL2-overexpressing cells were immunolabeled for ZO-1 and tubulin as indicated. Molecular weight standards are indicated on the left in kDa. (<b>G</b>) Uninfected WIF-B cells or cells overexpressing MAL2 were subjected to a scratch assay. Wound healing was analyzed 0 h (upper panels) and 18 h (lower panels) after the scratch. White lines indicate the edges of the gap. Bar = 10 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>MAL2 knockdown in Hep3B cells enhances cell migration. (<b>A</b>) Lysates from uninfected (un) Hep3B cells or cells knocked down (KD) for MAL2 expression with anti-sense MAL2 recombinant adenoviruses for 4 days were immunoblotted for endogenous MAL2 or tubulin. Molecular weight markers are indicated on the left in kDa. A ~30% reduction in MAL2 levels was observed. (<b>B</b>) Wound healing was imaged at 0 and 18 h post-scratch. White lines indicate the border of the monolayer along the scratch. Boxed areas were enlarged and shown in <a href="#cancers-12-00422-f004">Figure 4</a>A. (<b>C</b>) The gap area remaining relative to the initial scratch area was determined 0, 6 (<i>p</i> ≤ 0.013), 12 (<i>p</i> ≤ 0.014) and 18 h (<i>p</i> ≤ 0.002) after recovery. The values represent the average ± SEM determined from at least three independent experiments.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>MAL2 expression in Hep3B cells correlates with actin-based protrusion formation. (<b>A</b>) The boxed areas from <a href="#cancers-12-00422-f003">Figure 3</a>B were enlarged. Arrowheads indicate long protrusions present in uninfected migrating cells that were lost in cells knocked down for MAL2 expression (MAL2 KD). (<b>B</b>) Uninfected cells and cells knocked down for MAL2 at the migrating edge were double-labeled for MAL2 (with anti-FLAG antibodies) and actin (with phalloidin). Arrowheads in panels a and b indicate long, MAL2- and actin-positive protrusions present in uninfected migrating cells whereas arrowheads in c and d point out the smooth surfaces in migrating cells knocked down for MAL2 expression. (<b>C</b>) Cells from Part B were scored for the presence of protrusions. Values represent the number of MAL2 KD cells with protrusions relative to uninfected cells. (<b>D</b>) Cells knocked down for MAL2 expression were immunolabeled for actin with phalloidin. Arrowheads mark dorsal ruffle-like structures. (<b>E</b>) Uninfected and MAL2 overexpressing WIF-B cells present at the edge of a scratch were labeled for actin with phalloidin. Arrowheads mark long protrusions present at the cell surface. (<b>F</b>) Hep3B cells overexpressing MAL2 were treated for 4 h with 50 µg/mL cycloheximide and immunolabeled for MAL2 with anti-FLAG antibodies. Arrowheads point to MAL2 labeling present at the ends of the long protrusions. Bar = 10 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>MAL2 expression in Clone 9 cells is associated with actin-based protrusion formation. (<b>A</b>) Uninfected Clone 9 cells and Clone 9 cells expressing wild type (WT) MAL2 were immunolabled for MAL2 (with anti-FLAG antibodies) and actin (with phalloidin). Nuclei were also stained with DAPI and merged images are shown in panels c and f. Arrowheads are marking actin-based protrusions in MAL2-expressing cells. (<b>B</b>) Clone 9 cells expressing FLAG-tagged wild type MAL2 were treated 50 µg/mL of cycloheximide (CHX) for up to 4 h as indicated and immunolabeled for MAL2 with anti-FLAG antibodies. Arrowheads indicate MAL2 localization at the Golgi (a), at the cell surface (b) and in the apical compartment (c,d). (<b>C</b>) Clone 9 cells expressing FLAG-tagged wild type MAL2 were immunolabeled for MAL2 (with anti-FLAG antibodies), actin (with phalloidin) and nuclei (with DAPI) after treatment with 5 μM latrunculin B (lat B) for 0 or 15 min or after washout for 60 min. Arrowheads in panels a and c mark long, actin-based protrusions in untreated cells and cells after latrunculin B washout. In panel b, arrowheads point to the smooth cell surfaces with no actin-based protrusion in cells treated for 15 min with latrunculin B. Bar = 10 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>MAL2 expression in Clone 9 cells is associated with actin-based protrusion formation, yet impaired migration. (<b>A</b>) Monolayers of uninfected Clone 9 cells or cells expressing wild type (WT) MAL2 were scratched and wound healing was imaged at 0 h (upper panels) and 18 h (lower panels). White lines indicate the edges of the gap. (<b>B</b>) The gap area remaining after 18 h of recovery was determined relative to the area of the initial gap after 0, 4 (<i>p</i> ≤ 0.033), 8 (<i>p</i> ≤ 0.008), 12 (<i>p</i> ≤ 0.008), 16 (<i>p</i> ≤ 0.006) and 18 h (<i>p</i> ≤ 0.008) after recovery. Values represent the mean ± SEM determined from at least three independent experiments. (<b>C</b>) The boxed areas indicated in Part A were enlarged. Arrowheads point to long surface protrusions in MAL2-expressing cells present at the edge of the scratch. (<b>D</b>–<b>G</b>) Uninfected cells or cells expressing wild type MAL2 were seeded in serum-free medium onto filters in Boyden chambers in the absence (D,E) or presence of matrigel (F,G). After 24 h, the nuclei of cells that had penetrated to the underside of the filter were labeled with DAPI and visualized (D,F). In (E,G), the numbers of MAL2-expressing cells that had penetrated the filter were determined relative to the numbers of uninfected cells and plotted as the percent of control (uninfected). Values represent the mean ± SEM from at least three independent experiments. The difference in invasion was significant (<i>p</i> ≤ 0.05).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>MAL2 overexpression disrupts invadopodia formation. (<b>A</b>) Lysates from uninfected Clone 9 cells and Clone 9 cells expressing wild type (WT) MAL2 were immunoblotted for Tks5, Mena, VASP, FAK and tubulin (as a loading control) as indicated. Molecular weight standards are indicated on the left in kDa. (<b>B</b>) Uninfected Clone 9 cells and Clone 9 cells expressing wild type MAL2 were immunolabeled for Tks5 or Mena as indicated. Arrowheads in panels a and b mark Tks5 and Mena present in juxta-nuclear invadopodia that are absent in cells expressing MAL2. (<b>C</b>) Uninfected and MAL2-expressing cells were seeded onto 96-well plates at 4000 cells/well and relative cell numbers monitored after 1 (<i>p</i> ≤ 0.006), 2 (<i>p</i> ≤ 0.006) or 3 days. Values represent the mean ± SEM from at least three independent experiments each performed in triplicate. (<b>D</b>) Uninfected Clone 9 cells were labeled for Mena. Arrowheads mark Mena present at the surface in patches. (<b>E</b>) Clone 9 cells expressing wild type MAL2 were immunolabeled for Mena and MAL2 as indicated. Arrowheads mark Mena present at the far distal tips of protrusions induced by MAL2 expression. Bar = 10 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>Mutations in a putative EVH1 recognition site decrease protrusion formation. (<b>A</b>) A diagram of the cytosolic, N-terminal amino acid sequence of MAL2 containing is shown. The two putative proline-rich EVH1 recognition sites are underlined and the single and double mutations are indicated in red. All constructs were FLAG-tagged at the C-terminus. (<b>B</b>) WIF-B cells expressing the wild type (WT) and mutant MAL2 proteins were immunolabeled for MAL2 with anti-FLAG antibodies. Asterisks mark bile canaliculi. (<b>C</b>) Clone 9 cells expressing wild type (WT) and the mutant MAL2 proteins were double labeled for MAL2 (with anti-FLAG antibodies) and actin (with phalloidin). Merged images are shown in c, f, i and l with DAPI labeling of nuclei. Arrowheads in a–f indicate surface protrusions in cells expressing wild type or VPPPP mutant MAL2. Bar = 10 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>The FPAP motif mediates MAL2-induced protrusion formation and impaired cell migration. (<b>A</b>) Monolayers of infected cells or cells expressing wild type (WT) or mutant MAL2 as indicated were scratched and wound healing was imaged 0 h (upper panels), 6 h (middle panels) and 12 h (lower panels) post-scratch. White lines indicate the edges of the gap. (<b>B</b>) The boxed areas in Part A were enlarged. Arrowheads in panels c and d indicate protrusions in cells expressing wild type or the VPPPP mutant MAL2 at the migrating edge. (<b>C</b>) Cells at the migrating edge were immunolabeled for MAL2 (with anti-FLAG antibodies). The Arrowheads in panels a and b indicate protrusions in cells expressing wild type or the VPPPP mutant MAL2 at the migrating edge. Bar = 10 µm. (<b>D</b>) The gap area remaining after 0, 6, 12 (<i>p</i> ≤ 0.008 for WT and ≤ 0.005 for VPPPP) and 18 h (<i>p</i> ≤ 0.008 for WT and ≤ 0.005 for VPPPP) of recovery was determined relative to the area of the initial gap in cells expressing wild type and mutant MAL2 as indicated. Values represent the mean ± SEM from at least five independent experiments. (<b>E</b>) Uninfected or MAL2-expressing cells were seeded onto 96-well plates at 4000 cells/well and relative cell numbers monitored after 1 (<i>p</i> ≤ 0.006 for WT and ≤0.01 for VPPPP), 2 (<i>p</i> ≤ 0.008 for WT and ≤0.024 for VPPPP) or 3 days. Values represent the mean ± SEM from at least three independent experiments each performed in triplicate.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 10</span></a></div>   <p>The FPAP motif mediates MAL2-induced protrusion formation and impaired cell migration. (<b>A</b>–<b>D</b>) Uninfected cells or cells expressing wild type of mutant MAL2 were seeded in serum-free medium onto filters in Boyden chambers in the absence (A,B) or presence of matrigel (C,D) cells. After 24 h, the nuclei of cells that had penetrated to the underside of the filter were labeled with DAPI and visualized (A,C). In (B,D), the numbers of MAL2-expressing cells that had penetrated the filter were determined relative to the numbers of uninfected cells and plotted as the percent of control (uninfected). In (B), * <i>p</i> ≤ 0.049 and in (D), <i>p</i> ≤ 0.007 for WT and ≤0.029 for VPPPP. Bar = 10 µm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 11</span></a></div>   <p>MAL2 has a low somatic mutation frequency in human cancers. MAL2 mutation rates for 18 different human cancer types were determined from data compiled by the NCI Genomic Data Commons. Below, the specific patient mutations (and consequences) in the MAL2 amino acid sequence are indicated in blue and red. MAL2 transmembrane domains are underlined.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 12</span></a></div>   <p>Explaining the MAL2 paradox. (<b>A</b>) <i>MAL2</i> and <i>c-Myc</i> are on chromosome 8q24, a region frequently amplified in epithelial-derived cancers. (<b>B</b>), In normal cells with low c-Myc (left), Miz1 binds a core promoter with p300 and nucleophosmin (npm) and activates transcription of genes involved in tumor suppression (including MAL2). Upon 8q24 amplification in early HCC, both MAL2 and myc levels are increased. As transformation progresses, Myc levels are increased (right), Myc-Max dimers bind Miz1, displace p300 and npm, recruit a methyl transferase and other factors thereby repressing Miz1 transcriptional activity and lead to decreased MAL2 expression and enhanced transformation.</p>   </div> </div>  </body></html>