<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Cell Death Dis</span><span class="iso-abbrev" title="iso-abbrev">Cell Death Dis</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Cell Death &amp;amp; Disease</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2041-4889</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6155292">6155292</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30250066">30250066</a></span><span class="publisher-id" title="publisher-id">1031</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41419-018-1031-x">10.1038/s41419-018-1031-x</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Enah overexpression is correlated with poor survival and aggressive phenotype in gastric cancer</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Di</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Li</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Xiaowei</span></span><a href="#Aff1">1</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">Yi</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Mingzuo</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Bing</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Min</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Weijie</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Hua</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">Huijie</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Kai</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Kaichun</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">+13709218743</span><span class="email" tagx="email" title="email">kaicwu@fmmu.edu.cn</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">+1348826046</span><span class="email" tagx="email" title="email">liangjie@fmmu.edu.cn</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">Gui</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">+13474364768</span><span class="email" tagx="email" title="email">renguigz@hotmail.com</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], </span><span class="citation_author_institution" id="Aff2">[2], </span><span class="citation_author_institution" id="Aff3">[3], </span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.415870.f</span><span class="institution" title="institution">Department of Gastroenterology, </span><span class="institution" title="institution">Navy General Hospital, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018-10</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">998</span><span class="history" title="history"><span class="received" title="received">received: 2018-4-4</span><span class="rev-recd" title="rev-recd">rev-recd: 2018-8-25</span><span class="accepted" title="accepted">accepted: 2018-8-27</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Enabled homolog (Enah), which is a member of the Ena/VASP family that also includes VASP (vasodilator-stimulated phosphoprotein) and Ena/VASP like, is a mammalian ortholog of Drosophila Enabled (Ena). An increasing number of studies demonstrated Enah overexpression is involved in human colorectal carcinomas, breast cancers and hepatocellular carcinoma. However, the significance of Enah expression in gastric cancer (GC) is poorly elucidated. Here, we demonstrate that Enah is upregulated in GC and associated with AJCC stage, depth of invasion and poor overall survival (OS). Knockdown of Enah inhibited GC cell proliferation and metastasis and vice versa. Further experiments suggested that p-Erk1/2, p-AKT, p-p65, Vimentin and Fibronectin were downregulated and E-cadherin was upregulated after Enah silencing, implicating altered functions in GC proliferation and metastasis. Thus, our study suggests that Enah is a harmful factor for GC and a novel target for GC treatment.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100001809</span><span class="institution" title="institution">National Natural Science Foundation of China (National Science Foundation of China)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">81301804</span><span class="award-id" tagx="award-id" title="award-id">81572302</span><span class="award-id" tagx="award-id" title="award-id">91542000</span><span class="award-id" tagx="award-id" title="award-id">81772650</span><span class="award-id" tagx="award-id" title="award-id">91542000</span><span class="award-id" tagx="award-id" title="award-id">81772650</span><span class="award-id" tagx="award-id" title="award-id">81772650</span><span class="award-id" tagx="award-id" title="award-id">81772650</span><span class="award-id" tagx="award-id" title="award-id">81772650</span><span class="award-id" tagx="award-id" title="award-id">81322037</span><span class="award-id" tagx="award-id" title="award-id">91542000</span><span class="award-id" tagx="award-id" title="award-id">81322037, 81772650, 81572302, 81421003</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Di</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Li</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Xiaowei</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">Yi</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Min</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Weijie</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">Huijie</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">Gui</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Science-technology Support Plan[2015BAI13B07]</span></div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100007262</span><span class="institution" title="institution">École des Mines de Nantes (École Nationale Supérieure des Mines de Nantes)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">81772650</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Mingzuo</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Key Research and Development Plan [2017YFC0908300]and National Science-technology Support Plan[2015BAI13B07].</span></div> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Independent Funds of the Key Laboratory [CBSKL2015Z12]</span></div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Gastric cancer (GC) is the fourth most common cancer and the second most common cause of cancer-related death<sup><a href="#CR1">1</a></sup>. Each year, approximately 990,000 people are diagnosed with GC in the world, of whom about 738,000 die from this disease<sup><a href="#CR2">2</a>,<a href="#CR3">3</a></sup>. GC incidence rates differ in sexes and nations. Rates are two to three folds higher in men than women<sup><a href="#CR2">2</a></sup>, and the highest incidence rates are observed in East Asia, East Europe, and South America, while the lowest rates in North America and most parts of Africa<sup><a href="#CR4">4</a></sup>. In recent years, with the combination of chemotherapy, radiotherapy and surgery treatment, the quality of life of GC patients has improved, but the prognosis of GC patients still makes people feel dissatisfactory. GC was always diagnosed at advanced stage with lymph node metastasis and distant metastasis, one of the most important reasons is lack of early molecular marker. Therefore, it is very important for us to look for better biomarkers to diagnose, guide clinical treatment and predict prognosis for GC.</p> <p id="Par3">Enabled homolog (Enah), which is a member of the Ena/VASP family that also includes VASP (vasodilator-stimulated phosphoprotein) and Ena/VASP like, is a mammalian ortholog of Drosophila Enabled (Ena)<sup><a href="#CR5">5</a></sup>. The Ena/VASP family plays an important role in regulating cell movement, morpholology and adhesion, processes required during invasion and metastasis<sup><a href="#CR6">6</a>,<a href="#CR7">7</a></sup>. Many studies have demonstrated that Enah is dysregulated in many human solid tumors including colorectal carcinomas<sup><a href="#CR8">8</a>,<a href="#CR9">9</a></sup>, hepatocellular carcinoma<sup><a href="#CR10">10</a></sup>, cervical carcinoma<sup><a href="#CR11">11</a></sup>, breast carcinoma<sup><a href="#CR12">12</a></sup>, salivary gland carcinoma<sup><a href="#CR13">13</a></sup> and pancreatic carcinoma<sup><a href="#CR14">14</a></sup>. However, the potential role of Enah in the development of GC is poorly elucidated.</p> <p id="Par4">In this study, we evaluated the expression of Enah through a public database Oncomine, a tissue microarray (TMA), 39 paired GC samples and GC cell lines and analyzed the correlation between Enah expression and clinicopathological parameters and survival of GC patients. Furthermore, we investigated the role of Enah in cell proliferation and metastasis in vitro and in vivo. We also explored three signaling pathways and EMT process that were significantly changed after knockdown and overexpression of Enah.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="humantissuesamplesandcelllinesandcultures" title="sec"> <div class="title" tagx="title" title="title">
Human tissue samples and cell lines and cultures</div> <p id="Par5">Thirty-nine pairs of human GC tissues and corresponding normal tissues were obtained from the Department of Digestive Diseases, Xijing Hospital. The ethics committee at Xijing Hospital of the Fourth Military Medical University approved this study, and all the patients gave written informed consent on the use of clinical specimens for medical research. The human GC cell lines (MKN28, MKN45, SGC7901, BGC823 and AGS) and the immortalized gastric epithelial cell line GES were preserved at our institute. All the cell lines were cultured in RPMI-1640 medium (GIBCO) supplemented with 10% foetal bovine serum (GIBCO) and 100 U/ml penicillin/streptomycin and cultured at 37 °C in a humidified incubator containing 5% CO<sub>2</sub>.</p> </div> <div class="quantitativereal-timepolymerasechainreaction(qrt-pcr)" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time polymerase chain reaction (qRT-PCR)</div> <p id="Par6">Total RNA from human GC cell lines, GC tissues and their adjacent normal tissues were extracted using TaKaRa MiniBEST Universal Extraction Kit (TaKaRa) according to the manufacturer’s instructions and cDNA was synthesized using PrimeScript<sup>TM</sup> RT Master Mix (Perfect Real Time) (TaKaRa). The reaction system (10 ul) contains 5x PrimeScript<sup>TM</sup> RT Master Mix, Total RNA and RNase Free dH2O. For PCR, the reaction system (20 ul) consists of SYBR Premix Ex Taq<sup>II</sup> 10 ul, PCR Forward Primer (10 uM) 1 ul, PCR Reverse Primer (10 uM) 1 ul, cDNA 2 ul and sterile distilled water (dH<sub>2</sub>O) 6 ul. The primer sequences were as follows: Enah (sense: 5′-GTGGCTCAACTGGATTCAGCA-3′, antisense 5′-AGGAATGGCACAGTTTATCACGA-3′);β-actin (sense 5′-TGGCACCCAGCACAATGAA-3′, antisense 5′- CTAAGTCATAGTCCGCCTAGAAGCA-3′). The relative quantitation of gene expression levels were determined by the 2<sup>−△△</sup>Ct method and β-actin was used as a reference.</p> </div> <div class="westernblot" title="sec"> <div class="title" tagx="title" title="title">
Western blot</div> <p id="Par7">Total proteins were extracted with lysis buffer (RIPA, protease inhibitor and phosphatase inhibitor) from the cultured cells. The proteins were separated by SDS-PAGE and transferred to nitrocellulose membrane. The membranes were blocked with 5% milk and then incubated with primary antibodies at 4 °C overnight. The primary antibodies used in the experiment were as following: Enah (BD Bioscience; US); PCNA (Cell Signaling Technology, USA), p-Erk1/2/Erk1/2 (Cell Signaling Technology, USA), p-STAT/STAT (Cell Signaling Technology, USA), p-AKT/AKT (Cell Signaling Technology, USA), p-p65/p65 NF-κB (Cell Signaling Technology, USA), E-cadherin (Cell Signaling Technology, USA),Vimentin (Abcam, USA), Fibronectin (Santa Cruz Biotechnology, USA) and β-actin (Sigma, USA). After washing three times with 1 × TBST, the membranes were incubated with horseradish peroxidase (HRP)-conjugated goat anti-mouse or anti-rabbit IgG (Zhong Shan Goldenbridge Biotech, China). At last, the membranes were visualized using Immobilon<sup>TM</sup> Western Chemiluminescent HRP Substrate (Millipore).</p> </div> <div class="lentiviraltransfection" title="sec"> <div class="title" tagx="title" title="title">
Lentiviral transfection</div> <p id="Par8">All the recombinant lentiviruses were purchased from Genechem Corporation (Shanghai, China). Lentiviral vectors encoding shRNAs were designated LV-shEnah and LV-shcontrol. Lentiviral vector encoding the human Enah gene was designated LV-Enah and an empty vector which was used as the negative control was designated NC. Lentiviral transfection was performed according to the protocol of the Genechem Recombinant Lentivirus Operation Manual provided by Genechem Corporation. The transfection efficiencies were determined by examining GFP expression under a microscope and the expression of Enah was measured using qRT-PCR and Western blot analysis.</p> </div> <div class="5-bromo-2-deoxyuridine(brdu)incorporatingassayandcolonyformingassay" title="sec"> <div class="title" tagx="title" title="title">
5-Bromo-2-deoxyUridine (BrdU) incorporating assay and colony forming assay</div> <p id="Par9">The BrdU incorporating assay was performed using BrdU Cell Proliferation Assay Kit (Millipore, USA) according to the manufacturer’s instructions. For colony forming assay, 1000 cells were seeded in six-well plates and cultured at 37 °C in a humidified incubator. After two weeks’ incubation, the cells were fixed with 4% paraformaldehyde, then stained with 0.5% gentian violet and observed under a digital camera.</p> </div> <div class="cellcycleandapoptosisanalysis" title="sec"> <div class="title" tagx="title" title="title">
Cell cycle and apoptosis analysis</div> <p id="Par10">Cell cycle and apoptosis were measured using a FACScan flow cytometer (BD Biosciences, Franklin Lakes, NJ). For cell cycle analysis, the cells were starved in serum-free medium for 24 h, then cultured normally for less than 24 h and harvested with tyrisin (approximately 1 × 10<sup>6</sup> cells), finally fixed with 75% ethanol and stained with propidium iodide (Sigma, MO). The Annexin V PE/7AAD Kit (Multi sciences) was used for apoptosis assay. The apoptosis of cells was induced with 5-FU (1 mg/ml) and stained according to the manufacturer’s protocol. The results were analyzed using Multicycle-DNA Cell Cycle Analyzed Software.</p> </div> <div class="high-contentcellmobilityassay" title="sec"> <div class="title" tagx="title" title="title">
High-content cell mobility assay</div> <p id="Par11">We performed high-content assays to assess cell motility and the protocol referred to the "Material and methods" section of a previous report<sup><a href="#CR15">15</a></sup>.</p> </div> <div class="transwellassay" title="sec"> <div class="title" tagx="title" title="title">
Transwell assay</div> <p id="Par12">The transwell assay was performed using 6.5-mm transwell chambers with a pore size of 8 μm (Corning Costar Corp., USA) as described previously<sup><a href="#CR15">15</a></sup>. All the assays were performed in triplicate.</p> </div> <div class="immunohistochemistry" title="sec"> <div class="title" tagx="title" title="title">
Immunohistochemistry</div> <p id="Par13">Paraffin-embedded slides and TMA were dewaxed with dimethylbenzene (I and II) and gradient alcohol (100, 95, 85, 75%), and antigen retrieval was carried out by high-pressure boiler in citrate buffer for 2 min. Slides were incubated with 3% hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) for 20 min, followed by incubation with goat serum for 30 min at room temperature. Never wash the slides with phosphate-buffered saline, just pour the goat serum on the slides. Then slides were incubated with primary antibodies at 4 °C overnight. Biotinylated secondary anti-rabbit or anti-rat antibodies were added and incubated for 1 h at room temperature. After incubating with streptavidin-HRP for 30 min, slides were stained with DAB (DAB chromogen: DAB substrate = 1:20) and counterstained with haematoxylin. After washing in running water, slides were dehydrated with gradient alcohol and dimethylbenzene. Finally, coverslips were used to mount onto the slides with neutral balsam.</p> <p id="Par14">The degree of immunostaining was observed and scored separately by two independent pathologists. Immunoreactivities were scored by the percentage of stained cells (0, no staining; 1, 0~25%; 2, 25~75%; 3, 75~100%) and the intensity of staining (0, no staining; 1, weak = light yellow; 2, moderate = yellow brown; 3,strong = brown).Then a final score ranging from 0 and 9 could be calculated by multiplication of two values and for the purposes of statistical analysis, all cases were grouped into four categories as negative (0~1;−), weak (2; +), moderate (3~5; ++), and strong (6–9; +++). At last, we defined ‘negative and weak expression’ and "moderate and strong expression" as "Enah low expression’ and ‘Enah high expression", respectively.</p> </div> <div class="immunofluorescencestaining" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence staining</div> <p id="Par15">We performed immunofluorescence staining to measure the expression of EMT-related markers and the protocol referred to the "Material and methods" section of a previous report<sup><a href="#CR16">16</a></sup>.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par16">Paired <i>t</i> test was used to analyze the differences of Enah expression in GC cell lines, GC tissues and corresponding adjacent normal tissues. Enah positive rates in GC tissues and matched adjacent normal tissues were compared using chi-square test. The relationship between Enah expression and clinicopathologic parameters was also assessed using chi-square test. Kaplan–Meier method was used to analyze OS of patients and comparisons were analyzed by log-rank test. Cox regression model was used for univariate and multivariate analysis, hazard ratios and their 95% confidence intervals were calculated. All data were analyzed using the Statistical Software Package for the Social Sciences (SPSS; Chicago, IL, USA). All statistical tests were two sided, and <i>p</i> &amp;lt; 0.05 was considered statistically significant.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="enahisupregulatedingctissuesandcelllinesandiscorrelatedwithclinicopathologicalparametersandsurvivalofgcpatients" title="sec"> <div class="title" tagx="title" title="title">
Enah is upregulated in GC tissues and cell lines and is correlated with clinicopathological parameters and survival of GC patients</div> <p id="Par17">To determine the role of Enah in GC, firstly, the Oncomine database was queried and the results showed that Enah mRNA expression was higher in GC tissues compared with corresponding normal tissues (Fig. <a href="#Fig1">1a</a>) and Enah copy number was also increased in GC tissues (Fig. <a href="#Fig1">1b</a>).Then, we obtained thirty-nine paired GC tissues and corresponding normal tissues for qRT-PCR and as shown in Fig. <a href="#Fig1">1c</a>, in comparison with normal tissues, the expression level of Enah mRNA is significantly higher in GC tissues. Furthermore, we measured the Enah expression through immunohistochemistry with a TMA which contained ninety paired GC tissues and adjacent normal tissues. We found that Enah positive rate was 84.44% (76/90) in GC tissues and 51.11% (46/90) in non-cancerous adjacent tissues with a significantly statistical difference (<i>p</i> &amp;lt; 0.001, Table <a href="#Tab1">1</a>) and Enah was mainly present in the cytoplasm of tumor cells (Fig. <a href="#Fig1">1d</a>). Finally, we measured the expression of Enah in five GC cell lines, and the results showed that Enah was up-regulated in two GC cell lines MKN45 and AGS compared with GES at not only RNA level (Fig. <a href="#Fig1">1e</a>) but also protein level (Fig. <a href="#Fig1">1f</a>). Collectively, these data demonstrated that Enah was increased in GC. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <div class="title" tagx="title" title="title">
Expression of Enah in GC tissues and cell lines.</div> <p><b>a</b> Enah expression at mRNA level in the Chen Gastric dataset grouped by no value (0), diffuse gastric adenocarcinoma (1), gastric adenocarcinoma (2), gastric intestinal type adenocarcinoma (3) and gastric mixed adenocarcinoma (4). <b>b</b> Enah copy number in TCGA Gastric dataset grouped by no value (0), diffuse gastric adenocarcinoma (1), gastric adenocarcinoma (2), gastric intestinal type adenocarcinoma (3), gastric papillary adenocarcinoma (4), gastric tubular adenocarcinoma (5), mucinous gastric adenocarcinoma (6) and signet ring cell gastric adenocarcinoma (7). <b>c</b> Expression of Enah in thirty-nine paired GC tisssues and adjacent normal tissues measured by qRT-PCR (left panel: combined comparison; right panel: intra-patient comparison; ****<i>p</i> &amp;lt; 0.0001). <b>d</b> Representative immunohistochemical staining of Enah in GC tissues and corresponding normal tissues (Scale: 50 um). <b>e</b> Expression of Enah in five GC cell line at mRNA level measured by qRT-PCR (****<i>p</i> &amp;lt; 0.0001). <b>f</b> Expression of Enah in five GC cell line at protein level measured by western blot (left panel) and semiquantitative analysis (right panel), with β-actin used as the loading control. <b>g</b> Kaplan–Meier Survival Analysis shows High expression level of Enah was associated with worse OS of GC patients (<i>n</i> = 80; Low expression: −~+; High expression: ++~+++; <i>p</i> &amp;lt; 0.05, log rank test)</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> <p>The expression level of Enah in GC tissues and adjacent normal tissues (<i>n</i> = 90)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Tissue types</th> <th>Total</th> <th colspan="3">Enah expression</th> <th><i>p</i></th> </tr> <tr> <th /> <th /> <th>Negative (−~+)</th> <th>Positive (++)</th> <th>Strong positive (+++)</th> <th /> </tr> </thead> <tbody> <tr> <td>Gastric cancer</td> <td>90</td> <td>14</td> <td>23</td> <td>53</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Normal tissue</td> <td>90</td> <td>44</td> <td>38</td> <td>8</td> <td /> </tr> </tbody> </table> </div> <p /> <p id="Par18">In order to understand the potential roles of Enah in the development of GC, we analyzed the relationship between Enah expression and clinicopathological characteristics. The results showed Enah expression was significantly associated with AJCC stage (<i>p</i> = 0.040) and depth of invasion (<i>p</i> &amp;lt; 0.001). However, no significant association was found between Enah expression and other clinicopathological features, such as gender, age, nodal status, distant metastasis and pathological classification (Table <a href="#Tab2">2</a>). <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>The relationship between Enah expression and clinicopathological characteristics of GC</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Variable</th> <th /> <th>Total</th> <th colspan="3">Enah expression</th> <th><i>p</i></th> </tr> <tr> <th /> <th /> <th /> <th>Negative (−~+)</th> <th>Positive (++)</th> <th>Strong positive (+++)</th> <th /> </tr> <tr> <th /> <th /> <th>90</th> <th>14</th> <th>23</th> <th>53</th> <th /> </tr> </thead> <tbody> <tr> <td rowspan="2">Gender</td> <td>Male</td> <td>70</td> <td>10</td> <td>21</td> <td>39</td> <td rowspan="2">0.192</td> </tr> <tr> <td>Female</td> <td>20</td> <td>4</td> <td>2</td> <td>14</td> </tr> <tr> <td rowspan="2">Age (y)</td> <td>≥60</td> <td>61</td> <td>8</td> <td>13</td> <td>40</td> <td rowspan="2">0.174</td> </tr> <tr> <td>&amp;lt;60</td> <td>29</td> <td>6</td> <td>10</td> <td>13</td> </tr> <tr> <td rowspan="2">AJCC stage</td> <td>Stage 1 + 2</td> <td>37</td> <td>10</td> <td>8</td> <td>19</td> <td rowspan="2">0.040</td> </tr> <tr> <td>Stage 3 + 4</td> <td>53</td> <td>4</td> <td>15</td> <td>34</td> </tr> <tr> <td rowspan="2">Depth of invasion</td> <td>T1 + T2</td> <td>11</td> <td>6</td> <td>1</td> <td>4</td> <td rowspan="2">&amp;lt;0.001</td> </tr> <tr> <td>T3 + T4</td> <td>79</td> <td>8</td> <td>22</td> <td>49</td> </tr> <tr> <td rowspan="2">Nodal status</td> <td>N0</td> <td>23</td> <td>4</td> <td>8</td> <td>11</td> <td rowspan="2">0.419</td> </tr> <tr> <td>N1 + N2 + N3</td> <td>67</td> <td>10</td> <td>15</td> <td>42</td> </tr> <tr> <td rowspan="2">Distant metastasis</td> <td>M0</td> <td>86</td> <td>14</td> <td>21</td> <td>51</td> <td rowspan="2">0.430</td> </tr> <tr> <td>M1</td> <td>4</td> <td>0</td> <td>2</td> <td>2</td> </tr> <tr> <td rowspan="2">Pathological classification</td> <td>I + II</td> <td>24</td> <td>5</td> <td>5</td> <td>14</td> <td rowspan="2">0.646</td> </tr> <tr> <td>III + IV</td> <td>66</td> <td>9</td> <td>18</td> <td>39</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>AJCC</i> American Joint Committee on Cancer</p> </div>  <p /> <p id="Par19">To identify whether Enah expression had an effect on the OS of GC patients, we performed Kaplan-Meier survival analysis on those patients with complete survival information (80 cases). The patients with high Enah expression had lower OS rate than those with low expression (<i>p</i> &amp;lt; 0.05, Log-rank test, Fig. <a href="#Fig1">1g</a>), with a 5-year OS rate of 22.1%, whereas that in patients with low expression level was 50.0%. Furthermore, we performed univariate and multivariate Cox regression analysis to examine whether Enah expression was an independent prognostic factor for GC patients. In univariate Cox regression analysis, the variables gender, age, depth of invasion, nodal status, distant metastasis, pathological classification and Enah expression were included. Among these parameters, different depth of invasion, nodal status, AJCC stage and Enah expression had significantly statistical differences in survival with <i>p</i>-values less than 0.05 (Table <a href="#Tab3">3</a>). In multivariate Cox regression analysis, only one factor, depth of invasion was still significantly different between groups with <i>p</i>-values less than 0.05 (Table <a href="#Tab4">4</a>). These results demonstrated that Enah expression may not be identified as an independent predictor for GC patients. <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Univariate analysis of prognostic markers in GC patients (<i>N</i> = 80)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Variable</th> <th>HR (95% CI)</th> <th><i>p</i></th> </tr> </thead> <tbody> <tr> <td>Gender</td> <td>0.815(0.448–1.481)</td> <td>0.501</td> </tr> <tr> <td colspan="3"> Male</td> </tr> <tr> <td colspan="3"> Female</td> </tr> <tr> <td>Age (y)</td> <td>0.863 (0.502–1.481)</td> <td>0.592</td> </tr> <tr> <td colspan="3">≥60</td> </tr> <tr> <td colspan="3">&amp;lt;60</td> </tr> <tr> <td>AJCC stage</td> <td>0.371 (0.213–0.647)</td> <td>0.001</td> </tr> <tr> <td colspan="3"> Stage 1 + 2</td> </tr> <tr> <td colspan="3"> Stage 3 + 4</td> </tr> <tr> <td>Depth of invasion</td> <td>0.123(0.030–0.506)</td> <td>0.004</td> </tr> <tr> <td colspan="3"> T1 + T2</td> </tr> <tr> <td colspan="3"> T3 + T4</td> </tr> <tr> <td>Nodal status</td> <td>0.362 (0.183–0.717)</td> <td>0.004</td> </tr> <tr> <td colspan="3"> N0</td> </tr> <tr> <td colspan="3"> N1 + N2 + N3</td> </tr> <tr> <td>Distant metastasis</td> <td>0.533 (0.192–1.482)</td> <td>0.228</td> </tr> <tr> <td colspan="3"> M0</td> </tr> <tr> <td colspan="3"> M1</td> </tr> <tr> <td>Pathological classification</td> <td>0.846(0.466–1.535)</td> <td>0.582</td> </tr> <tr> <td colspan="3"> I + II</td> </tr> <tr> <td colspan="3"> III + IV</td> </tr> <tr> <td>Enah</td> <td>0.596 (0.360–0.988)</td> <td>0.045</td> </tr> <tr> <td colspan="3"> Low expression (−~+)</td> </tr> <tr> <td colspan="3"> High expression(++~+++)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>HR</i> hazard ratio, <i>CI</i> confidence interval</p> </div>  <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> <p>Multivariate analysis of prognostic markers in GC patients (<i>N</i> = 80)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Variable</th> <th>HR (95% CI)</th> <th><i>p</i></th> </tr> </thead> <tbody> <tr> <td>Gender</td> <td>0.619 (0.316–1.209)</td> <td>0.160</td> </tr> <tr> <td colspan="3"> Male</td> </tr> <tr> <td colspan="3"> Female</td> </tr> <tr> <td>Age (y)</td> <td>1.159 (0.657–2.047)</td> <td>0.610</td> </tr> <tr> <td colspan="3">≥60</td> </tr> <tr> <td colspan="3">&amp;lt;60</td> </tr> <tr> <td>AJCC stage</td> <td>0.659 (0.357–1.218)</td> <td>0.183</td> </tr> <tr> <td colspan="3"> Stage 1 + 2</td> </tr> <tr> <td colspan="3"> Stage 3 + 4</td> </tr> <tr> <td>Depth of invasion</td> <td>0.179(0.042–0.769)</td> <td>0.021</td> </tr> <tr> <td colspan="3"> T1 + T2</td> </tr> <tr> <td colspan="3"> T3 + T4</td> </tr> <tr> <td>Nodal status</td> <td>0.687 (0.262–1.802)</td> <td>0.445</td> </tr> <tr> <td colspan="3"> N0</td> </tr> <tr> <td colspan="3"> N1 + N2 + N3</td> </tr> <tr> <td>Distant metastasis</td> <td>0.789(0.273–2.279)</td> <td>0.662</td> </tr> <tr> <td colspan="3"> M0</td> </tr> <tr> <td colspan="3"> M1</td> </tr> <tr> <td>Pathological classification</td> <td>1.031 (0.556–1.913)</td> <td>0.923</td> </tr> <tr> <td> I + II</td> <td /> <td /> </tr> <tr> <td> III + IV</td> <td /> <td /> </tr> <tr> <td>Enah</td> <td>0.684 (0.407–1.151)</td> <td>0.153</td> </tr> <tr> <td colspan="3"> Low expression (<b>−</b>~+)</td> </tr> <tr> <td colspan="3"> High expression(++~+++)</td> </tr> </tbody> </table> </div> <p /> </div> <div class="reductionofenahexpressioninhibitsgccellproliferationandmetastasisinvitro" title="sec"> <div class="title" tagx="title" title="title">
Reduction of Enah expression inhibits GC cell proliferation and metastasis in vitro</div> <p id="Par20">In order to determine the role of Enah in the GC cell proliferation and migration, we firstly established four stable cell lines (denoted MKN45-shcontrol, MKN45-shEnah, AGS-shcontrol and AGS-shEnah) after transfecting with the LV-shcontrol or LV-shEnah lentivirus, respectively, and the expression level of Enah was detected by PCR and WB. The expression of Enah was decreased compared with the control cells (Figs. <a href="#Fig2">2a, b</a>). PCNA (proliferating cell nuclear antigen), the cell-proliferation marker, was downregulated after Enah knockdown (Fig. <a href="#Fig2">2c</a>). BrdU assay showed that downregulation of Enah expression decreased the BrdU incorporation (Fig. <a href="#Fig2">2d</a>).Colony forming assay showed that reduced Enah level inhibited colony formation and colony cell number (Figs. <a href="#Fig2">2e, f</a>). Cell cycle assays showed that silencing Enah increased the G0/G1 population compared with control cells (Figs. <a href="#Fig2">2g</a>, <a href="#Fig2">h</a>). Apoptosis assays revealed that Enah inhibition led to an increased percentage of apoptotic GC cells (Fig. <a href="#Fig2">2i, j</a>). Compared with control cells, MKN45 and AGS cells showed dramatically impaired mobility after Enah knockdown, which was confirmed by high-content cell mobility assay (Figs. <a href="#Fig3">3a, b</a>). Transwell assay showed that downregulation of Enah expression significantly inhibited the migration and invasion capacities of the MKN45 and AGS cells (Figs. <a href="#Fig3">3c~f</a>). Collectively, these data demonstrated that Enah silencing played an inhibitory role in the GC cell proliferation and metastasis. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <div class="title" tagx="title" title="title">
Reduction of Enah expression inhibits GC cell proliferation and promotes cell apoptosis in vitro.</div> <p><b>a</b> Decreased mRNA level of Enah in LV-shEnah cells compared with LV-shcontrol cells. <b>b</b> Decreased protein level of Enah in LV-shEnah cells compared with LV-shcontrol cells (left panel) and semiquantitative analysis (right panel). <b>c</b> Decreased protein level of PCNA in LV-shEnah cells compared with LV-shcontrol cells (left panel) and semiquantitative analysis (right panel). <b>d</b> BrdU assay showed that less BrdU were incorporated in LV-shEnah cells compared with LV-shcontrol cells. <b>e</b>, <b>f</b> Colony forming assay showed decreased colony formation and colony cell number in LV-shEnah cells compared with LV-shcontrol cells. Representative images, magnification × 200 (<b>e</b>) and statistical analysis (<b>f</b>). <b>g</b>, <b>h</b> Cell cycle assays showed more cells arrested in the G0/G1 stages in LV-shEnah cells compared with LV-shcontrol cells. Representative images (<b>g</b>) and statistical analysis (<b>h</b>). <b>i</b>, <b>j</b> Apoptosis assays revealed an increased percentage of apoptotic GC cells in LV-shEnah cells compared with LV-shcontrol cells. Representative images (<b>i</b>) and statistical analysis (<b>j</b>). All the data are expressed as the means ± SD of three independent experiments. *<i>p</i> &amp;lt; 0.05; **<i>p</i> &amp;lt; 0.01; ***<i>p</i> &amp;lt; 0.001; ****<i>p</i> &amp;lt; 0.0001; LV-shEnah versus LV-shcontrol of MKN45 and AGS cells, respectively</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <div class="title" tagx="title" title="title">
Reduction of Enah expression inhibits GC cell metastasis in vitro.</div> <p><b>a</b>, <b>b</b> High-content cell mobility assay indicated impaired mobility in LV-shEnah cells compared with LV-shcontrol cells. MKN45 cell (<b>a</b>) and AGS cell (<b>b</b>). (C~F) Transwell assay showed the reduced ability of cell migration and invasion in LV-shEnah cells compared with LV-shcontrol cells. Representative photographs of migrating MKN45 and AGS cells, magnification × 200 (<b>c</b>) and statistical analysis of the relative numbers of migrating MKN45 and AGS cells (<b>d</b>); representative photographs of invading MKN45 and AGS cells, magnification × 200 (<b>e</b>) and statistical analysis of the relative numbers of invading MKN45 and AGS cells (<b>f</b>). All the data are expressed as the means ± SD of three independent experiments. *<i>p</i> &amp;lt; 0.05; **<i>p</i> &amp;lt; 0.01; ***<i>p</i> &amp;lt; 0.001; ****<i>p</i> &amp;lt; 0.0001; LV-shEnah versus LV-shcontrol of MKN45 and AGS cells, respectively</p>   </div> <p /> </div> <div class="enahknockdowninhibitsgccellproliferationandmetastasisinvivo" title="sec"> <div class="title" tagx="title" title="title">
Enah knockdown inhibits GC cell proliferation and metastasis in vivo</div> <p id="Par21">To further investigate the role of Enah in GC cell proliferation and metastasis in vivo, we performed subcutaneously implanted tumor model and tail-vein metastasis model with MKN45-shcontrol and MKN45-shEnah cells. Compared with control group, the xenograft showed lower weight and volume of MKN45 cell that had decreased expression of Enah (Figs. <a href="#Fig4">4a~c</a>). Moreover, the mice formed less tumor nodules in the liver and lung of Enah knockdown group in comparation with control group (Figs. <a href="#Fig4">4d~e</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <div class="title" tagx="title" title="title">
Enah downregulation inhibits GC cell proliferation and metastasis in vivo.</div> <p><b>a</b>~<b>c</b> The results of subcutaneously implanted tumor model showed that the xenograft showed lower weight and volume of MKN45-shEnah cells compared with MKN45-shcontrol cells. Representative images of tumors formed in nude mice (<i>n</i> = 5) (<b>a</b>); quantification of tumor weights of xenograft in mice (<b>b</b>); quantification of tumor growth curves of xenograft in mice (<b>c</b>). <b>d</b>~<b>e</b> The results of tail-vein metastasis model showed that the mice formed less tumor nodules in the liver and lung of MKN45-shEnah cells compared with MKN45-shcontrol cells. Representative images showing HE staining of liver and lung tissues isolated from mice injected with MKN45-shEnah cells or MKN45-shcontrol cells (scale: 50 um) (<b>d</b>); quantification of the average number of metastatic tumor nodules in the liver and lung of mice (<b>e</b>). **<i>p</i> &amp;lt; 0.01; ***<i>p</i> &amp;lt; 0.001; MKN45-shEnah versus MKN45-shcontrol cells</p>   </div> <p /> </div> <div class="overexpressionofenahpromotesgccellproliferationandmetastasisinvitroandinvivo" title="sec"> <div class="title" tagx="title" title="title">
Overexpression of Enah promotes GC cell proliferation and metastasis in vitro and in vivo</div> <p id="Par22">To further confirm the stimulative function of Enah in GC development, we upregulated Enah expression with lentivirus using the GC cell line SGC7901. After Enah stable transfected cell line (LV-Enah, SGC7901) was established, PCR and WB were used to detect the expression level of Enah. The expression of Enah in the stable cell line was increased compared with the control cell (NC) (Figs. <a href="#Fig5">5a~c</a>). PCNA was upregulated after overexpression of Enah (Figs. <a href="#Fig5">5b, d</a>). BrdU assay showed that upregulation of Enah expression increased the BrdU incorporation (Fig. <a href="#Fig5">5e</a>).Colony forming assay showed that increased Enah level promoted colony formation and colony cell number (Fig. <a href="#Fig5">5f</a>). Cell cycle assays showed that Enah overexpression decreased the G0/G1 population compared with control cells (Fig. <a href="#Fig5">5g</a>). Apoptosis assays revealed that Enah overexpression led to a decreased percentage of apoptotic GC cells (Fig. <a href="#Fig5">5h</a>). Compared with control cells, SGC7901 cell showed dramatically enhanced mobility after Enah upregulation, which was confirmed by high-content cell mobility assay (Fig. <a href="#Fig5">5i</a>). Transwell assay showed that upregulation of Enah expression significantly promoted the migration and invasion capacities of SGC7901 cell (Figs. <a href="#Fig5">5j, k</a>). Collectively, these data demonstrated that Enah overexpresion played a stimulative role in the GC cell proliferation and metastasis. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <div class="title" tagx="title" title="title">
Overexpression of Enah promotes GC cell proliferation, migration and inhibits cell apoptosis in vitro.</div> <p><b>a</b> Increased mRNA level of Enah in LV-Enah cells compared with NC cells. <b>b</b> Increased protein level of Enah and PCNA in LV-Enah cells compared with NC cells. <b>c</b> The semiquantitative analysis of increased protein level of Enah. <b>d</b> The semiquantitative analysis of increased protein level of PCNA. <b>e</b> BrdU assay showed that more BrdU were incorporated in LV-Enah cells compared with NC cells. <b>f</b> Colony forming assay showed increased colony formation and colony cell number in LV-Enah cells compared with NC cells. Representative images, magnification × 200 (left panel) and statistical analysis (right panel). <b>g</b> Cell cycle assays showed less cells arrested in the G0/G1 stages in LV-Enah cells compared with NC cells. Representative images (left panel) and statistical analysis (right panel). <b>h</b> Apoptosis assays revealed a decreased percentage of apoptotic GC cells in LV-Enah cells compared with NC cells. Representative images (left panel) and statistical analysis (right panel). <b>i</b> High-content cell mobility assay indicated enhanced mobility in LV-Enah cells compared with NC cells. <b>j</b> Transwell assay showed the enhanced ability of cell migration in LV-Enah cells compared with NC cells. Representative photographs of migrating SGC7901 cells, magnification × 200 (left panel) and statistical analysis of the relative numbers of migrating SGC7901 cells (right panel). <b>k</b> Transwell assay showed the enhanced ability of cell invasion in LV-Enah cells compared with NC cells. Representative photographs of invading SGC7901 cells, magnification × 200 (left panel) and statistical analysis of the relative numbers of invading SGC7901 cells (right panel). All the data are expressed as the means ± SD of three independent experiments. *<i>p</i> &amp;lt; 0.05; **<i>p</i> &amp;lt; 0.01; ***<i>p</i> &amp;lt; 0.001; ****<i>p</i> &amp;lt; 0.0001</p>   </div> <p /> <p id="Par23">In addition, functional assay in vivo also demonstrated that compared with control group, the xenograft showed heavier weight and bigger volume of SGC7901 cell that had increased expression of Enah (Figs. <a href="#Fig6">6a~c</a>). Moreover, the mice formed more tumor nodules in the liver and lung of SGC7901 cell that had increased expression of Enah compared with control group (Figs. <a href="#Fig6">6d~e</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <div class="title" tagx="title" title="title">
Overexpression of Enah promotes GC cell proliferation and metastasis in vivo.</div> <p><b>a</b>~<b>c</b> The results of subcutaneously implanted tumor model showed that the xenograft showed heavier weight and bigger volume of SGC7901 LV-Enah cells compared with NC cells. Representative images of tumors formed in nude mice (<i>n</i> = 3) (<b>a</b>); quantification of tumor weights of xenograft in mice (<b>b</b>); quantification of tumor growth curves of xenograft in mice (<b>c</b>). <b>d</b>~<b>e</b> The results of tail-vein metastasis model showed that the mice formed more tumor nodules in the liver and lung of SGC7901 LV-Enah cells compared with NC cells. Representative images showing HE staining of liver and lung tissues isolated from mice injected with SGC7901 LV-Enah cells or NC cells (scale:50um) (<b>d</b>); quantification of the average number of metastatic tumor nodules in the liver and lung of mice (<b>e</b>). *<i>p</i> &amp;lt; 0.05; **<i>p</i> &amp;lt; 0.01; ***<i>p</i> &amp;lt; 0.001</p>   </div> <p /> </div> <div class="reductionofenahdownregulatesp-erk1/2,p-aktandp-p65andinhibitsemt(epithelial-mesenchymaltransition)progressingccells" title="sec"> <div class="title" tagx="title" title="title">
Reduction of Enah downregulates p-Erk1/2, p-AKT and p-p65 and inhibits EMT (epithelial-mesenchymal transition) progress in GC cells</div> <p id="Par24">Although the role of Enah has been investigated in some tumors, the specific pathway by which Enah functions remains unclear. In order to elucidate the pathway through which Enah regulates cell proliferation and metastasis in GC, we detected protein expression of several canonical signaling pathway correlated with cell proliferation and metastasis, including MAPK pathway (p-Erk1/2/Erk1/2), JAK-STAT pathway (p-STAT3/STAT3), NF-κB pathway (p-p65/p65), PI3K/AKT pathway (p-AKT/AKT) and EMT markers such as E-cadherin, Vimentin and Fibronectin in MKN45 and AGS cells. P-Erk1/2, p-AKT, p-p65, Vimentin and Fibronectin were downregulated and E-cadherin was upregulated, while p-STAT3 exhibited no significant difference after reduction of Enah in MKN45 and AGS cells (Fig. <a href="#Fig7">7a</a>). To further investigate the role of Enah in EMT in GC, we examined the cell morphology as supplementary. MKN45 and AGS cells after Enah knockdown displayed a round or flat morphology with a short cytoplasmic process compared with control cells, whose morphology was elongated, spindle shaped and scattered (Fig. <a href="#Fig7">7b</a>). We also performed immunofluorescence staining to detect the expression of E-cadherin and Vimentin (Fig. <a href="#Fig7">7c</a>) and the results were consistent with Western blot. Moreover, we measured the expression of related molecules mentioned above, examined the cell morphology and performed immunofluorescence staining after Enah overexpression and got the opposite results (Supplementary Figure <a href="#MOESM1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <div class="title" tagx="title" title="title">
Reduction of Enah downregulates p-Erk1/2, p-AKT, and p-p65 and inhibits EMT (epithelial-mesenchymal transition) progress in GC cells.</div> <p><b>a</b> Decreased protein levels of p-Erk1/2, p-AKT, p-p65, Vimentin, Fibronectin, and increased protein level of E-cadherin in LV-shEnah cells compared with LV-shcontrol cells. No significant difference was detected in the protein expression of p-STAT3. <b>b</b> Morphology of MKN45-shcontrol, MKN45-shEnah, AGS-shcontrol and AGS-shEnah cells as visualized using phase-contrast microscopy (magnification × 200). <b>c</b> Immunofluorescence analysis of E-cadherin and Vimentin expression in MKN45-shcontrol, MKN45-shEnah, AGS-shcontrol and AGS-shEnah cells. *<i>p</i> &amp;lt; 0.05; **<i>p</i> &amp;lt; 0.01; ***<i>p</i> &amp;lt; 0.001; LV-shEnah versus LV-shcontrol of MKN45 and AGS cells, respectively</p>   </div> <p /> </div> <div class="mapk(erk1/2),aktandnf-κbpathwaysareinvolvedinenhancedgccellproliferationandmigrationinducedbyenahoverexpression" title="sec"> <div class="title" tagx="title" title="title">
MAPK (Erk1/2), AKT and NF-κB pathways are involved in enhanced GC cell proliferation and migration induced by Enah overexpression</div> <p id="Par25">To verify whether MAPK (Erk1/2), AKT and NF-κB pathways are involved in enhanced GC cell proliferation and migration induced by Enah overexpression, Erk1/2, AKT and NF-κB specific inhibitors GDC-0994(10 mM), PF-04691502(10 mM) and PDTC (100 mM) were used to treat SGC7901-LV-Enah cells at optimal concentration without significant cell death. After treatment with different inhibitors, the corresponding functional assays were performed and the results demonstrated that the abilities of cell proliferation and migration were significantly downregulated in SGC7901-LV-Enah cells after treating with Erk1/2, AKT and NF-κB inhibitors compared with control cells that were treated with DMSO (Supplementary Figure <a href="#MOESM1">2</a>).</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par26">Even though diagnostic technologies for GC have improved significantly, there is still lack of useful markers for predicting the recurrence and survival of GC patients.</p> <p id="Par27">Enah plays an important role in cell adhesion and motility by controlling the geometry of the actin filament network<sup><a href="#CR17">17</a>,<a href="#CR18">18</a></sup>. Many reports have demonstrated that Enah was upregulated during the process of malignant transformation of colorectal, cervical, pancreatic and breast cancers<sup><a href="#CR8">8</a>,<a href="#CR12">12</a>,<a href="#CR14">14</a>,<a href="#CR19">19</a></sup>. What’s more, Di Modugno et al<sup><a href="#CR12">12</a>,<a href="#CR19">19</a></sup>. found that: in breast cancer, Enah overexpression was associated with tumor size, highly invasive properties and advanced clinical stages. In addition, Hu et al<sup><a href="#CR10">10</a></sup>. also confirmed that Enah was upregulated in HCC, correlated with tumor differentiation and stage, and most importantly, its overexpression was related to worse prognosis and Enah may be a potential prognostic biomarker for HCC.</p> <p id="Par28">In the present study, we have demonstrated that Enah was overexpression in GC tissues and mainly present in the cytoplasm of tumor cells, which were consistent with some previous researches<sup><a href="#CR20">20</a>–<a href="#CR22">22</a></sup>. Meanwhile, we measured Enah expression in five GC cell line at mRNA and protein levels. Interestingly, Enah was upregulated in MKN45 and AGS, but not in MKN28, SGC7901, and BGC823, compared with GES. We also found that Enah expression was significantly associated with AJCC stage and depth of invasion. Furthermore, we demonstrated the prognostic significance of Enah in GC, indicating that Enah expression was inversely correlated with OS. Patients with high Enah expression had shorter survival than those with low expression. Multivariate analysis using the Cox proportional hazards model indicated that only depth of invasion, not Enah expression, was an independent prognostic factor for survival in GC patients. Therefore, in some extent, Enah may not be considered as a potential prognostic biomarker for GC patients. As we know, Enah could undergo alternative splicing to produce various protein isoforms with distinct function, which consists of a variant present in primary tumor cells but lost in invasive cells (Mena<sup><a href="#CR11">11</a></sup>), two invasive forms (Mena<sup>++</sup> and Mena<sup>+++</sup> or Mena<sup>INV</sup>), and the subtype MenaΔv6<sup><a href="#CR23">23</a></sup>. And when compared with the 570-amino acids pan-hMena, the isoforms Mena<sup>INV</sup> (+++) and Mena<sup>++</sup> contain a supplementary exon, next to the Ena/VASP homology 1 (EVH1) domain (a 4-amino acid region for ++ and a 19-amino acid region for +++); the variant hMena11a contains the exon 11a included within the EVH2 domain, adjacent to the F-actin motif. In contrast, hMenaΔv6 lacks the internal exon 6<sup><a href="#CR23">23</a></sup>. Intriguingly, it has been demonstrated that different Mena isoforms could be predictive of patient outcome in mutlivariant analysis<sup><a href="#CR24">24</a>–<a href="#CR28">28</a></sup>. Therefore, isoform-specific Enah detection may offer a useful signature for the diagnosis and prognosis of GC.</p> <p id="Par29">Next, functional assays in vitro demonstrated that Enah knockdown inhibited cell proliferation, migration, invasion and promoted cell apoptosis in MKN45 and AGS cells, whereas, the overexpression experiment with SGC7901 cell got the opposite results. These results correspond to those of previous studies in other cancers<sup><a href="#CR14">14</a>,<a href="#CR19">19</a>,<a href="#CR29">29</a>,<a href="#CR30">30</a></sup>.At the same time, in vivo assay using MKN45 and SGC7901 cells also verified the inhibitory and stimulative role of Enah on cell proliferation and metastasis. Here, we did not perform this assay using the AGS cell because in 2016, R Xing<sup><a href="#CR31">31</a></sup> and colleagues confirmed that the GC cell line AGS, which expresses low level of HLA-I upon activation of the NKp30/MAPK/interleukin (IL)-12 or IL-2 pathway, was susceptible to NK cell-mediated lysis, leading to the failure of AGS cells to form tumors in nude mice. Altogether, our data demonstrate that Enah plays an important role in GC pathogenesis by the promotion of cell proliferation and metastesis and the inhibition of apoptosis.</p> <p id="Par30">Furthermore, we confirmed that the activity of MAPK (Erk1/2), AKT pathway and NF-κB pathway were inhibited after Enah silencing while no significant difference was detected in JAK-STAT pathway. Among these three signaling pathways, the former two have been demonstrated that they could be activated by Enah in various tumor cells<sup><a href="#CR14">14</a>,<a href="#CR19">19</a>,<a href="#CR30">30</a></sup>, while the latter two were important for cell proliferation and metastasis<sup><a href="#CR32">32</a>,<a href="#CR33">33</a></sup>. Besides, we found that E-cadherin, the loss of which contributed to an EMT process and promoted tumor metastasis<sup><a href="#CR34">34</a>–<a href="#CR36">36</a></sup>, was upregulated and Vimentin and Fibronectin were downregulated after Enah knockdown, which was also reflected by the change of cell morphology and immunofluorescence staining. At the same time, the opposite results were obtained in GSC7901 cells with Enah overexpression, not only in terms of the change of these three pathways, but also in terms of EMT-related changes. Finally, in order to verify whether MAPK (Erk1/2), AKT, and NF-κB pathways are involved in enhanced GC cell proliferation and migration induced by Enah overexpression, Erk1/2, AKT and NF-κB specific inhibitors were used to treat SGC7901-LV-Enah cells and the results showed that the abilities of cell proliferation and migration were significantly downregulated in SGC7901-LV-Enah cells after treating with Erk1/2, AKT and NF-κB inhibitors. Altogether, these data suggest that MAPK (Erk1/2), AKT, NF-κB pathway and the EMT process may be involved in the role of Enah on GC cell proliferation and metastasis.</p> <p id="Par31">Although we investigated the role of Enah on cell proliferation and metastasis with loss-of-function and gain-of-function assays in vitro and in vivo and preliminarily explored the change of signaling pathways and EMT process, the potential molecular mechanism by which Enah exerts on GC needs to be further elucidated in the future. Notably, to our knowledge, EMT is regulated by a complex molecular network including some canonical signaling pathways, such as TGF-β, Wnt and Notch signaling pathways<sup><a href="#CR35">35</a></sup>. The Wnt and Notch signalling pathways play important roles in development and tumorigenesis and have reported crosstalks in these processes, which are not well understood<sup><a href="#CR37">37</a></sup>. Interestingly, Enah has been demonstrated as a transcriptional target of the Wnt/β-catenin pathway and a novel nexus for the Wnt/β-catenin and the Notch signalling cascades<sup><a href="#CR37">37</a></sup>.Moreover, the possible link of Mena with Wnt/β-catenin pathway was also proved experimentally for colorectal, breast and hepatocellular carcinoma cell lines<sup><a href="#CR38">38</a></sup>. Besides the well-known role of EMT on tumor cell invasiveness, the link between EMT and angiogenesis was also extensively studied and demonstrated in recent years<sup><a href="#CR39">39</a></sup>.Combined with our results, functional Wnt/β-catenin/ Notch-Enah-EMT axis and Enah-EMT-angiogenesis pathway may be good choices for further research.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2018_1031_MOESM1_ESM.docx">LINK</a> <p>Supplementary Information</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>These authors contributed equally: Di Chen, Li Xu, Xiaowei Li, Yi Chu.</p> </div> <div class="fn-type-" title=""> <p><b>Change history</b></p> <p>5/7/2019</p> <p>In the version of this article originally submitted, Fig 5j included a mismatched image which had been accidentally inserted by the authors during collation of the data. The final data was derived from three independent experiments, and three repetitions for each experiment to ensure the repeatability, reliability and scientificity of the experiment. During this process, the fields overlapped to some extent, giving rise to the mistake. It does not alter any inferences drawn from the data, and the statistical graph is correct. This error has been corrected in both the PDF and HTML versions of the Article.</p> </div> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p><b>Supplementary Information</b> accompanies this paper at (10.1038/s41419-018-1031-x).</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors thank State Key Laboratory of Cancer Biology and Xijing Hospital of Digestive Diseases (Xi’an, China) for providing specimens. We also thank for professors Zengshan Li and Yong Guo from pathology department, Xijing Hospital of Digestive Diseases for helping to analyze the data of tissue array.This work was supported by National Natural Science Foundation of China [81301804, 81322037, 81772650, 81572302, 81421003, 81627807, and 91542000], Independent Funds of the Key Laboratory [CBSKL2015Z12], National Key Research and Development Plan [2017YFC0908300] and National Science-technology Support Plan [2015BAI13B07].</p> <div class="authors’contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p id="Par32">D.C. gathered the data and wrote, reviewed, and edited the manuscript, L.X., X.L., and Y.C. helped to obtain the relevant data, M.J., B.X., M.Z., and W.W. analyzed and interpreted the data, H.W., H.K., and K.W. provided technical assistance for the study, K.W. provided critical comments on the manuscript, and J.L. and G.R. directed the project and edited the manuscript.</p> </div> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p>The authors declare that they have no conflict of interest.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jemal</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Center</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeSantis</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Global patterns of cancer incidence and mortality rates and trends</span><span class="source" tagx="source" title="source">Cancer Epidemiol. Biomark. Prev.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">1893</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/1055-9965.EPI-10-0437">10.1158/1055-9965.EPI-10-0437</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferlay</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Estimates of worldwide burden of cancer in 2008: GLOBOCAN 2008</span><span class="source" tagx="source" title="source">Int. J. Cancer</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">127</span><span class="fpage" tagx="fpage" title="fpage">2893</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ijc.25516">10.1002/ijc.25516</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21351269">21351269</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karimi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islami</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anandasabapathy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freedman</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamangar</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gastric cancer: descriptive epidemiology, risk factors, screening, and prevention</span><span class="source" tagx="source" title="source">Cancer Epidemiol. Biomark. Prev.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">700</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/1055-9965.EPI-13-1057">10.1158/1055-9965.EPI-13-1057</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forman</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burley</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gastric cancer: global pattern of the disease and an overview of environmental risk factors</span><span class="source" tagx="source" title="source">Best. Pract. Res. Clin. Gastroenterol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">633</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bpg.2006.04.008">10.1016/j.bpg.2006.04.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16997150">16997150</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urbanelli</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of human Enah gene</span><span class="source" tagx="source" title="source">Biochim. Biophys. Acta</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">1759</span><span class="fpage" tagx="fpage" title="fpage">99</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbaexp.2006.01.001">10.1016/j.bbaexp.2006.01.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16494957">16494957</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">FB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niebuhr</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reinhard</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehland</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soriano</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mena, a relative of VASP and Drosophila Enabled, is implicated in the control of microfilament dynamics</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">227</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(00)81341-0">10.1016/S0092-8674(00)81341-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8861907">8861907</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dent</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loureiro</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">FB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ena/VASP proteins: regulators of the actin cytoskeleton and cell migration</span><span class="source" tagx="source" title="source">Annu. Rev. Cell. Dev. Biol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">541</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.cellbio.19.050103.103356">10.1146/annurev.cellbio.19.050103.103356</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14570581">14570581</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The expression of cytoskeleton regulatory protein Mena in colorectal lesions</span><span class="source" tagx="source" title="source">Rom. J. Morphol. Embryol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">345</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18758639">18758639</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyoda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Aberrant expression of human ortholog of mammalian enabled (hMena) in human colorectal carcinomas: implications for its role in tumor progression</span><span class="source" tagx="source" title="source">Int. J. Oncol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">53</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19082477">19082477</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expression of cytoskeleton regulatory protein Mena in human hepatocellular carcinoma and its prognostic significance</span><span class="source" tagx="source" title="source">Med. Oncol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">939</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12032-014-0939-y">10.1007/s12032-014-0939-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24683008">24683008</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prantner</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chira</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ember</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">The immunohistochemical aspects of protein Mena in cervical lesions</span><span class="source" tagx="source" title="source">Rom. J. Morphol. Embryol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">213</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19434313">19434313</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Modugno</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The cytoskeleton regulatory protein hMena (ENAH) is overexpressed in human benign breast lesions with high risk of transformation and human epidermal growth factor receptor-2-positive/hormonal receptor-negative tumors</span><span class="source" tagx="source" title="source">Clin. Cancer Res.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1470</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/1078-0432.CCR-05-2027">10.1158/1078-0432.CCR-05-2027</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16533770">16533770</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mena, a new available marker in tumors of salivary glands?</span><span class="source" tagx="source" title="source">Eur. J. Histochem.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">e8</span><span class="pub-id"><a href="https://dx.doi.org/10.4081/ejh.2012.e8">10.4081/ejh.2012.e8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22472896">22472896</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pino</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Human Mena + 11a isoform serves as a marker of epithelial phenotype and sensitivity to epidermal growth factor receptor inhibition in human pancreatic cancer cell lines</span><span class="source" tagx="source" title="source">Clin. Cancer Res.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">4943</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/1078-0432.CCR-08-0436">10.1158/1078-0432.CCR-08-0436</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18676769">18676769</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The role of MALAT-1 in the invasion and metastasis of gastric cancer</span><span class="source" tagx="source" title="source">Scand. J. Gastroenterol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">52</span><span class="fpage" tagx="fpage" title="fpage">790</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00365521.2017.1280531">10.1080/00365521.2017.1280531</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28276823">28276823</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hypoxia-induced Bmi1 promotes renal tubular epithelial cell-mesenchymal transition and renal fibrosis via PI3K/Akt signal</span><span class="source" tagx="source" title="source">Mol. Biol. Cell.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2650</span><span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.e14-01-0044">10.1091/mbc.e14-01-0044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25009285">25009285</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Antagonism between Ena/VASP proteins and actin filament capping regulates fibroblast motility</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">509</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(02)00731-6">10.1016/S0092-8674(02)00731-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12086607">12086607</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barzik</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ena/VASP proteins enhance actin polymerization in the presence of barbed end capping proteins</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">280</span><span class="fpage" tagx="fpage" title="fpage">28653</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M503957200">10.1074/jbc.M503957200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15939738">15939738</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Modugno</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular cloning of hMena (ENAH) and its splice variant hMena + 11a: epidermal growth factor increases their expression and stimulates hMena + 11a phosphorylation in breast cancer cell lines</span><span class="source" tagx="source" title="source">Cancer Res.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">2657</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-06-1997">10.1158/0008-5472.CAN-06-1997</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17363586">17363586</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">maspin-related orchestration of aggressiveness of gastric cancer</span><span class="source" tagx="source" title="source">Appl. Immunohistochem. Mol. Morphol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">326</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/PAI.0000000000000189">10.1097/PAI.0000000000000189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26067133">26067133</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Junnila</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kokkola</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karjalainen-Lindsberg</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puolakkainen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monni</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-wide gene copy number and expression analysis of primary gastric tumors and gastric cancer cell lines</span><span class="source" tagx="source" title="source">BMC. Cancer</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">73</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2407-10-73">10.1186/1471-2407-10-73</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20187983">20187983</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Gastric cancer in young vs old Romanian patients: immunoprofile with emphasis on maspin and mena protein reactivity</span><span class="source" tagx="source" title="source">APMIS</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">223</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/apm.12347">10.1111/apm.12347</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25556597">25556597</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciortea</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ember</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">The possible role of Mena protein and its splicing-derived variants in embryogenesis, carcinogenesis, and tumor invasion: a systematic review of the literature</span><span class="source" tagx="source" title="source">Biomed. Res. Int.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">2013</span><span class="fpage" tagx="fpage" title="fpage">365192</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2013/365192">10.1155/2013/365192</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23956979">23956979</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bria</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Prognostic impact of alternative splicing-derived hMENA isoforms in resected, node-negative, non-small-cell lung cancer</span><span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">11054</span><span class="pub-id"><a href="https://dx.doi.org/10.18632/oncotarget.2609">10.18632/oncotarget.2609</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25373410">25373410</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forse</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Menacalc, a quantitative method of metastasis assessment, as a prognostic marker for axillary node-negative breast cancer</span><span class="source" tagx="source" title="source">BMC. Cancer</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">483</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12885-015-1468-6">10.1186/s12885-015-1468-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26112005">26112005</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agarwal</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Quantitative assessment of invasive mena isoforms (Menacalc) as an independent prognostic marker in breast cancer</span><span class="source" tagx="source" title="source">Breast Cancer Res.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">R124</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/bcr3318">10.1186/bcr3318</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22971274">22971274</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balsamo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The alternatively-included 11a sequence modifies the effects of Mena on actin cytoskeletal organization and cell behavior</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">35298</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep35298">10.1038/srep35298</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27748415">27748415</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oudin</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Tumor cell-driven extracellular matrix remodeling drives haptotaxis during metastatic progression</span><span class="source" tagx="source" title="source">Cancer Discov</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">516</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/2159-8290.CD-15-1183">10.1158/2159-8290.CD-15-1183</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26811325">26811325</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roussos</span><span class="given-names" tagx="given-names" title="given-names">ET</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mena deficiency delays tumor progression and decreases metastasis in polyoma middle-T transgenic mouse mammary tumors</span><span class="source" tagx="source" title="source">Breast Cancer Res.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">R101</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/bcr2784">10.1186/bcr2784</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21108830">21108830</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trono</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">hMENA(11a) contributes to HER3-mediated resistance to PI3K inhibitors in HER2-overexpressing breast cancer cells</span><span class="source" tagx="source" title="source">Oncogene</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">887</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/onc.2015.143">10.1038/onc.2015.143</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25961924">25961924</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Copy number variations of HLA-I and activation of NKp30 pathway determine the sensitivity of gastric cancer cells to the cytotoxicity of natural killer cells</span><span class="source" tagx="source" title="source">Oncogene</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">2584</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/onc.2015.324">10.1038/onc.2015.324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26364607">26364607</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayer</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arteaga</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="mixed-article-title" title="mixed-article-title">The PI3K/AKT pathway as a target for cancer treatment</span><span class="source" tagx="source" title="source">Annu. Rev. Med.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-med-062913-051343">10.1146/annurev-med-062913-051343</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26473415">26473415</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prabhu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mundade</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korc</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loehrer</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Critical role of NF-kappaB in pancreatic cancer</span><span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">10969</span><span class="pub-id"><a href="https://dx.doi.org/10.18632/oncotarget.2624">10.18632/oncotarget.2624</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25473891">25473891</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghahhari</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babashah</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Interplay between microRNAs and WNT/beta-catenin signalling pathway regulates epithelial-mesenchymal transition in cancer</span><span class="source" tagx="source" title="source">Eur. J. Cancer</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">1638</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejca.2015.04.021">10.1016/j.ejca.2015.04.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26025765">26025765</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinberg</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epithelial-mesenchymal transition: at the crossroads of development and tumor metastasis</span><span class="source" tagx="source" title="source">Dev. Cell.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">818</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2008.05.009">10.1016/j.devcel.2008.05.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18539112">18539112</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajwar</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expression and significance of cadherins and its subtypes in development and progression of oral cancers: a review</span><span class="source" tagx="source" title="source">J. Clin. Diagn. Res.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">E5</span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Najafov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">MENA is a transcriptional target of the Wnt/beta-catenin pathway</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e37013</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0037013">10.1371/journal.pone.0037013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22615875">22615875</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciortea</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ember</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">The possible role of Mena protein and its splicing-derived variants in embryogenesis, carcinogenesis, and tumor invasion: a systematic review of the literature</span><span class="source" tagx="source" title="source">Biomed. Res. Int.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">2013</span><span class="fpage" tagx="fpage" title="fpage">365192</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2013/365192">10.1155/2013/365192</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23956979">23956979</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurzu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Systematic review of the old and new concepts in the epithelial-mesenchymal transition of colorectal cancer</span><span class="source" tagx="source" title="source">World J. Gastroenterol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">6764</span><span class="pub-id"><a href="https://dx.doi.org/10.3748/wjg.v22.i30.6764">10.3748/wjg.v22.i30.6764</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27570416">27570416</a></span></span></li> </ul> </div> </div>  </body></html>