<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Nat Commun</span><span class="iso-abbrev" title="iso-abbrev">Nat Commun</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6414540">6414540</a></span><span class="publisher-id" title="publisher-id">8547</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41467-019-08547-w">10.1038/s41467-019-08547-w</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Homozygous frameshift mutations in <i>FAT1</i> cause a syndrome characterized by colobomatous-microphthalmia, ptosis, nephropathy and syndactyly</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lahrouchi</span><span class="given-names" tagx="given-names" title="given-names">Najim</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">George</span><span class="given-names" tagx="given-names" title="given-names">Aman</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratbi</span><span class="given-names" tagx="given-names" title="given-names">Ilham</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">Ronen</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elalaoui</span><span class="given-names" tagx="given-names" title="given-names">Siham C.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moosa</span><span class="given-names" tagx="given-names" title="given-names">Shahida</span></span><a href="#Aff5">5</a><a href="#Aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bharti</span><span class="given-names" tagx="given-names" title="given-names">Sanita</span></span><a href="#Aff2">2</a><a href="#Aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">Ruchi</span></span><a href="#Aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abu-Asab</span><span class="given-names" tagx="given-names" title="given-names">Mones</span></span><a href="#Aff8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onojafe</span><span class="given-names" tagx="given-names" title="given-names">Felix</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adadi</span><span class="given-names" tagx="given-names" title="given-names">Najlae</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6017-3796</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lodder</span><span class="given-names" tagx="given-names" title="given-names">Elisabeth M.</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laarabi</span><span class="given-names" tagx="given-names" title="given-names">Fatima-Zahra</span></span><a href="#Aff9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamsyah</span><span class="given-names" tagx="given-names" title="given-names">Yassine</span></span><a href="#Aff10">10</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elorch</span><span class="given-names" tagx="given-names" title="given-names">Hamza</span></span><a href="#Aff10">10</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chebbar</span><span class="given-names" tagx="given-names" title="given-names">Imane</span></span><a href="#Aff10">10</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-4874-2487</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Postma</span><span class="given-names" tagx="given-names" title="given-names">Alex V.</span></span><a href="#Aff11">11</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lougaris</span><span class="given-names" tagx="given-names" title="given-names">Vassilios</span></span><a href="#Aff12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plebani</span><span class="given-names" tagx="given-names" title="given-names">Alessandro</span></span><a href="#Aff12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altmueller</span><span class="given-names" tagx="given-names" title="given-names">Janine</span></span><a href="#Aff13">13</a><a href="#Aff14">14</a><a href="#Aff15">15</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyrieleis</span><span class="given-names" tagx="given-names" title="given-names">Henriette</span></span><a href="#Aff16">16</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meiner</span><span class="given-names" tagx="given-names" title="given-names">Vardiella</span></span><a href="#Aff17">17</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNeill</span><span class="given-names" tagx="given-names" title="given-names">Helen</span></span><a href="#Aff18">18</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bharti</span><span class="given-names" tagx="given-names" title="given-names">Kapil</span></span><a href="#Aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyonnet</span><span class="given-names" tagx="given-names" title="given-names">Stanislas</span></span><a href="#Aff19">19</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wollnik</span><span class="given-names" tagx="given-names" title="given-names">Bernd</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henrion-Caude</span><span class="given-names" tagx="given-names" title="given-names">Alexandra</span></span><a href="#Aff20">20</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berraho</span><span class="given-names" tagx="given-names" title="given-names">Amina</span></span><a href="#Aff10">10</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7130-0030</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hildebrandt</span><span class="given-names" tagx="given-names" title="given-names">Friedhelm</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bezzina</span><span class="given-names" tagx="given-names" title="given-names">Connie R.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">c.r.bezzina@amc.uva.nl</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">Brian P.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">brooksb@nei.nih.gov</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sefiani</span><span class="given-names" tagx="given-names" title="given-names">Abdelaziz</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">sefianigen@hotmail.com</span></span><a href="#Aff3">3</a><a href="#Aff9">9</a></span><span class="citation_author_institution" id="Aff1">[1], </span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2297 5165</span><span class="institution-id" title="institution-id">grid.94365.3d</span><span class="institution" title="institution">Ophthalmic Genetics and Visual Function Branch, National Eye Institute, </span><span class="institution" title="institution">NIH, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2168 4024</span><span class="institution-id" title="institution-id">grid.31143.34</span><span class="institution" title="institution">Centre de Recherche en Génomique des Pathologies Humaines (GENOPATH), Faculté de Médecine et de Pharmacie, </span><span class="institution" title="institution">Mohammed V University of Rabat, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">000000041936754X</span><span class="institution-id" title="institution-id">grid.38142.3c</span><span class="institution" title="institution">Department of Pediatrics, Boston Children’s Hospital, </span><span class="institution" title="institution">Harvard Medical School, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0482 5331</span><span class="institution-id" title="institution-id">grid.411984.1</span><span class="institution" title="institution">Institute of Human Genetics, </span><span class="institution" title="institution">University Medical Center Goettingen, </span></div></span><span class="citation_author_institution" id="Aff6">[6], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0378 8438</span><span class="institution-id" title="institution-id">grid.2515.3</span><span class="institution" title="institution">Boston Children’s Hospital and Harvard Medical School, </span></div></span><span class="citation_author_institution" id="Aff7">[7], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2297 5165</span><span class="institution-id" title="institution-id">grid.94365.3d</span><span class="institution" title="institution">Unit on Ocular &amp;amp; Stem Cell Translational Research, National Eye Institute, </span><span class="institution" title="institution">NIH, </span></div></span><span class="citation_author_institution" id="Aff8">[8], </span><span class="citation_author_institution" id="Aff9">[9], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.418480.1</span><span class="institution" title="institution">Département de génétique médicale, </span><span class="institution" title="institution">Institut National d’Hygiène, </span></div></span><span class="citation_author_institution" id="Aff10">[10], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2168 4024</span><span class="institution-id" title="institution-id">grid.31143.34</span><span class="institution" title="institution">Service d’Ophtalmologie B, Hôpital des Spécialités, CHU Rabat, Faculté de médecine et de Pharmacie, </span><span class="institution" title="institution">Mohammed V University of Rabat, </span></div></span><span class="citation_author_institution" id="Aff11">[11], </span><span class="citation_author_institution" id="Aff12">[12], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.412725.7</span><span class="institution" title="institution">Pediatrics Clinic and Institute for Molecular Medicine "A. Nocivelli", Department of Clinical and Experimental Sciences, </span><span class="institution" title="institution">University of Brescia and ASST-Spedali Civili of Brescia, </span></div></span><span class="citation_author_institution" id="Aff13">[13], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0000 8580 3777</span><span class="institution-id" title="institution-id">grid.6190.e</span><span class="institution" title="institution">Cologne Center for Genomics University of Cologne, </span></div></span><span class="citation_author_institution" id="Aff14">[14], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0000 8580 3777</span><span class="institution-id" title="institution-id">grid.6190.e</span><span class="institution" title="institution">Center for Molecular Medicine Cologne (CMMC), </span><span class="institution" title="institution">University of Cologne, </span></div></span><span class="citation_author_institution" id="Aff15">[15], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0000 8580 3777</span><span class="institution-id" title="institution-id">grid.6190.e</span><span class="institution" title="institution">Institute of Human Genetics, </span><span class="institution" title="institution">University of Cologne, </span></div></span><span class="citation_author_institution" id="Aff16">[16], </span><span class="citation_author_institution" id="Aff17">[17], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2221 2926</span><span class="institution-id" title="institution-id">grid.17788.31</span><span class="institution" title="institution">Department of Human Genetics and Metabolic Diseases, </span><span class="institution" title="institution">Hadassah-Hebrew University Medical Center, </span></div></span><span class="citation_author_institution" id="Aff18">[18], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2355 7002</span><span class="institution-id" title="institution-id">grid.4367.6</span><span class="institution" title="institution">Department of Developmental Biology, </span><span class="institution" title="institution">Washington University School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff19">[19], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.462336.6</span><span class="institution" title="institution">Laboratory of embryology and genetics of human malformation, Institut National de la Santé et de la Recherche Médicale (INSERM) UMR 1163, Paris Descartes-Sorbonne Paris Cité University, </span><span class="institution" title="institution">Institut Imagine, </span></div></span><span class="citation_author_institution" id="Aff20">[20], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2175 4109</span><span class="institution-id" title="institution-id">grid.50550.35</span><span class="institution" title="institution">INSERM UMR-781, Département de Génétique, Hôpital Necker-Enfants Malades, </span><span class="institution" title="institution">Assistance Publique Hôpitaux de Paris (APHP), </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-3-3</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1180</span><span class="history" title="history"><span class="received" title="received">received: 2018-2-26</span><span class="accepted" title="accepted">accepted: 2019-1-9</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">A failure in optic fissure fusion during development can lead to blinding malformations of the eye. Here, we report a syndrome characterized by facial dysmorphism, colobomatous microphthalmia, ptosis and syndactyly with or without nephropathy, associated with homozygous frameshift mutations in <i>FAT1</i>. We show that <i>Fat1</i> knockout mice and zebrafish embryos homozygous for truncating <i>fat1a</i> mutations exhibit completely penetrant coloboma, recapitulating the most consistent developmental defect observed in affected individuals. In human retinal pigment epithelium (RPE) cells, the primary site for the fusion of optic fissure margins, FAT1 is localized at earliest cell-cell junctions, consistent with a role in facilitating optic fissure fusion during vertebrate eye development. Our findings establish <i>FAT1</i> as a gene with pleiotropic effects in human, in that frameshift mutations cause a severe multi-system disorder whereas recessive missense mutations had been previously associated with isolated glomerulotubular nephropathy.</p> </div> <div class="abstract" title="abstract"> <p id="Par2">Loss of the cadherin FAT1 has been associated with nephropathy and epithelial cell adhesion defects. Here, the authors report five families with a syndromic form of coloboma associated with homozygous frameshift variants in <i>FAT1</i> and recapitulate the phenotype in mutant mice and zebrafish.</p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par3">The eye develops as an evagination of the neural plate, which subsequently invaginates to form a dual-layered optic cup. This invagination is asymmetric, and a ventral opening (optic fissure) forms around the 5th week of human gestation<sup><a href="#CR1">1</a></sup>. For the eye to develop normally, the two edges of the fissure must approximate and fuse. If the optic fissure fails to fuse, uveal coloboma, a potentially blinding congenital malformation, results. Uveal coloboma accounts for up to 10% of childhood blindness worldwide, affecting between 0.5 and 2.6 per 10,000 births<sup><a href="#CR1">1</a></sup>. Mutations in several developmentally regulated genes, including <i>CHD7</i>, <i>CHX10/VSX2</i>, <i>GDF3</i>, <i>GDF6</i>, <i>MAF</i>, <i>OTX2</i>, <i>PAX2</i>, <i>PAX6</i>, <i>RAX</i>, <i>SHH</i>, <i>SIX3</i>, <i>SOX2</i>, <i>FOXE3, STRA6, BCOR</i>, <i>BMP4, YAP1</i>, and <i>MITF</i>, have been reported in patients with uveal coloboma<sup><a href="#CR2">2</a></sup>. To our knowledge the <i>FAT1</i> gene has not been previously associated with microphthalmia and coloboma.</p> <p id="Par4">The <i>FAT</i> cadherins are involved in fundamental developmental processes including cell–cell contact<sup><a href="#CR3">3</a></sup>, planar cell polarity<sup><a href="#CR4">4</a></sup>, cell migration<sup><a href="#CR5">5</a></sup>, and maintenance of apical–basal polarity<sup><a href="#CR6">6</a></sup> in epithelial cells. Loss of FAT1 function causes decreased epithelial cell adhesion and podocyte foot process effacement, resulting in abnormal glomerular filtration and nephropathy in humans and mouse, and cystic kidney in zebrafish<sup><a href="#CR7">7</a>,<a href="#CR8">8</a></sup>. <i>Fat1</i><sup>−/−</sup> mice also display partially penetrant anterior neural tube closure defects due to reduced actin accumulation, leading to apical constriction defects in the neural epithelium<sup><a href="#CR9">9</a></sup>. These developmental observations in epithelial cell types suggest that <i>FAT1</i> plays an important role in epithelial cell–cell adhesion and/or sheet fusion. Epithelial sheet fusion is one of the most critical morphogenetic events occurring during embryonic development, failure of which causes clinically well-characterized congenital defects including, neural tube closure defects (e.g. spina bifida), secondary palatal epithelial fusion defects (e.g. cleft palate), defective fusion of bilateral urogenital primordia (e.g. hypospadias), and optic fissure closure defects (e.g. coloboma)<sup><a href="#CR10">10</a></sup>.</p> <p id="Par5">We here report five unrelated families exhibiting a syndromic form of coloboma associated with homozygous frame-shift mutations in the <i>FAT1</i> gene. We demonstrate that <i>Fat1</i> knock-out mice and zebrafish homozygous for truncating <i>fat1a</i> mutations exhibit coloboma, supporting the causality of these mutations and pointing to an evolutionary conserved role of <i>Fat1</i> in eye development and optic fissure closure. Furthermore, studies conducted in human primary retinal pigment epithelium (RPE) cells point to a defect in optic fissure margin fusion likely caused by loss of FAT1 at the earliest cell–cell contacts that mediate optic fissure fusion.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="fat1mutationscauseasyndromicformofcolobomatousmicrophthalmia" title="sec"> <div class="title" tagx="title" title="title"><i>FAT1</i> mutations cause a syndromic form of colobomatous microphthalmia</div> <p id="Par6">We identified homozygous frameshift variants in the atypical protocadherin <i>FAT1</i> by whole exome sequencing (WES) and Sanger sequencing confirmation in 10 affected individuals from five unrelated consanguineous families of Middle-Eastern, Turkish, Pakistani, and North-African descent (Fig. <a href="#Fig1">1a, b</a>, Table <a href="#Tab1">1</a>). Patients presented with a previously undescribed syndrome including ocular abnormalities, nephropathy, syndactyly of the toes, and facial dysmorphism (Fig. <a href="#Fig1">1c–i</a>, Table <a href="#Tab1">1</a>). Seven patients presented with bilateral ptosis and two patients had unilateral ptosis (9/10, Fig. <a href="#Fig1">1c</a>). Ocular abnormalities included amongst others microphthalmia (4/10, Fig. <a href="#Fig1">1d</a>) iris coloboma (3/10, Fig. <a href="#Fig1">1e</a>), retinal coloboma (6/10, Fig. <a href="#Fig1">1f, g</a>), and severe amblyopia (5/10). The size of the eye was determined by measuring the axial length of the eye with an echo-biometer. Optical coherence tomography (OCT) images of individual F2-IV-1 are provided in Supplementary Fig. <a href="#MOESM2">1</a>. Syndactyly of the toes was seen in 8 out of 10 patients and affected predominantly the 3rd and 4th digits (Fig. <a href="#Fig1">1h</a>). X-ray of the feet demonstrated cutaneous syndactyly (Fig. <a href="#Fig1">1i</a>) in patient F2-IV-1. Patients F3-IV-1 and F3-IV-3 presented with bone fusion of phalanges 3–4 on the right foot and hypotrophy of phalanx 2 of the left foot (Fig. <a href="#Fig1">1h</a>). Dysmorphic facial features included high arched eyebrows, a long philtrum, long nose, and elongated appearance of the face (Fig. <a href="#Fig1">1c</a>). Affected individuals from families 1 and 2 had normal intellectual development corresponding to their age whereas patients F3-IV-3, F4-II-3, and F5-II-1 presented with intellectual disability. Patient F3-IV-1 presented with stage 5 chronic kidney disease at the age of 20 and a biopsy showed focal segmental glomerulosclerosis. His brother, patient F3-IV-3, developed intermittent proteinuria with normal kidney function at the age of 20 years. Patient F5-II-1 was hospitalized at the age of 15 years with proteinuria and hematuria and renal biopsy displayed glomerulotubular nephropathy (Supplementary Fig. <a href="#MOESM2">2</a>)<sup><a href="#CR8">8</a></sup>. Clinical follow-up of the other patients revealed asymptomatic proteinuria in two siblings from family 1 (patients IV-1 and IV-5). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Recessive frameshift mutations in <i>FAT1</i> cause a new clinical syndrome. Pedigree of family 1–5 (<b>a</b>). A schematic of human <i>FAT1</i> start/stop codon, exons, location of mutations previous published and identified in this study (top panel). The FAT1 protein is 4588 amino acids long and contains 34 cadherin repeats (CA), followed by five epidermal growth factor (EGF)-like repeat domains (E), a laminin G domain (LamG), a transmembrane domain (green), and an intracellular domain (bottom panel). Family 1 and 2 were found to carry the same homozygous frameshift variant c.2207dupT (p.I737NfsX7) in <i>FAT1 (</i>NM_005245). In Families 3–5 we identified, respectively, the following homozygous frameshift <i>FAT1</i> variants: c.2600_2601delCA (p.T867IfsX4), c.9729del p.(V3245LfsX25), and c.3093_3096del (p.P1032CgsX11). H homozygous, h heterozygous (<b>b</b>). Ophthalmic features observed in patients included ptosis; bilateral in patients F1-IV-1, F1-IV-3, F1-IV-5, F2-III-3, F2-IV-3, F3-IV-1 and unilateral in F2-IV-1, F4-II-3 (<b>c</b>), microphthalmia (<b>d</b>), iris coloboma (<b>e</b>), large chorioretinal coloboma containing the papilla/optic nerve with two other smaller circumscribed chorioretinal colobomas localized above the papilla (<b>f</b>), and retinal coloboma (<b>g</b>). Skeletal abnormalities included syndactyly in the majority of patients and bone fusion of phalanges 3–4 on the right foot and hypotrophy of phalanx 2 of the left foot in patient F3-II-2 (<b>h</b>). X-ray of the feet demonstrating cutaneous syndactyly in patient F2-IV-1 (<b>i</b>)</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> <p>Clinical characteristics of homozygous <i>FAT1</i> mutations carriers</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="2" /> <th colspan="3">Family 1</th> <th colspan="3">Family 2</th> <th colspan="2">Family 3</th> <th>Family 4</th> <th>Family 5</th> </tr> <tr> <th>IV:1</th> <th>IV:3</th> <th>IV:5</th> <th>III:2</th> <th>IV:1</th> <th>IV:3</th> <th>IV:1</th> <th>IV:3</th> <th>II:3</th> <th>II:1</th> </tr> </thead> <tbody> <tr> <td><i>FAT1</i> mutation (All homozygous)</td> <td colspan="3">c.2207dupT (p.I737NfsX7)</td> <td colspan="3">c.2207dupT (p.I737NfsX7)</td> <td colspan="2">c.2600_2601delCA (p.T867IfsX4)</td> <td>c.9729del (p.V3245LfsX25)</td> <td>c.3093_3096del (p.P1032CfsX11)</td> </tr> <tr> <td>Ethnicity</td> <td colspan="3">Morocco</td> <td colspan="3">Morocco</td> <td colspan="2">Middle-East</td> <td>Pakistan</td> <td>Turkey</td> </tr> <tr> <td>Consanguinity</td> <td colspan="3">+</td> <td colspan="3">+</td> <td colspan="2">+</td> <td>+</td> <td>+</td> </tr> <tr> <td>Sex</td> <td>F</td> <td>M</td> <td>M</td> <td>F</td> <td>F</td> <td>M</td> <td>M</td> <td>M</td> <td>M</td> <td>M</td> </tr> <tr> <td>Age (years)</td> <td>39</td> <td>34</td> <td>18</td> <td>36</td> <td>8</td> <td>2</td> <td>27</td> <td>24</td> <td>8</td> <td>8</td> </tr> <tr> <td>Intellectual disability</td> <td>−</td> <td>−</td> <td>−</td> <td>−</td> <td>−</td> <td>−</td> <td>−</td> <td>+</td> <td>+</td> <td>+</td> </tr> <tr> <td colspan="11"><i>Ocular features</i></td> </tr> <tr> <td> Iris coloboma</td> <td>−</td> <td>−</td> <td>B</td> <td>U</td> <td>−</td> <td>B</td> <td>−</td> <td>−</td> <td>−</td> <td>−</td> </tr> <tr> <td> Retinal coloboma</td> <td>B</td> <td>−</td> <td>B</td> <td>B</td> <td>B</td> <td>B</td> <td>−</td> <td>U</td> <td>−</td> <td>−</td> </tr> <tr> <td> Ptosis</td> <td>B</td> <td>B</td> <td>B</td> <td>B</td> <td>U</td> <td>B</td> <td>−</td> <td>B</td> <td>U</td> <td>B</td> </tr> <tr> <td> Microphtalmia</td> <td>−</td> <td>−</td> <td>B</td> <td>−</td> <td>U</td> <td>U</td> <td>−</td> <td>U</td> <td>–</td> <td>−</td> </tr> <tr> <td colspan="11"><i>Feet abnormalities</i></td> </tr> <tr> <td> Syndactyly</td> <td>−</td> <td>3rd−4th RF, 3rd−4th LF</td> <td>3rd−4th RF, 3rd-5th LF</td> <td>3rd−4th RF</td> <td>2nd–3rd RF</td> <td>1st–2nd LF, 3rd–4th RF</td> <td>3rd–4th RF; Bone fusion</td> <td>3rd–4th RF; Bone fusion; phalanx hypotrophy</td> <td>4th−5th bilaterally</td> <td>−</td> </tr> <tr> <td colspan="11"><i>Renal manifestations</i></td> </tr> <tr> <td> Nephropathy</td> <td>+</td> <td>−</td> <td>+</td> <td>−</td> <td>−</td> <td>−</td> <td>+</td> <td>+</td> <td>−</td> <td>+</td> </tr> <tr> <td> Biopsy (at age)</td> <td>n/a</td> <td>n/a</td> <td>n/a</td> <td>n/a</td> <td>n/a</td> <td>n/a</td> <td>FSGS (20 years of age)</td> <td>n/a</td> <td>n/a</td> <td>TIN, MS, thin GBM (12 years)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> For a more detailed phenotype description see Supplementary Table <a href="#MOESM2">2</a></p> <p><i>B</i> bilateral, <i>F</i> female, <i>FSGS</i> focal segmental glomerulosclerosis, <i>GBM</i> glomerular basement membrane, <i>LF</i> left foot, <i>M</i> male, <i>MS</i> mesangial sclerosis, <i>n/a</i> not assessed, <i>U</i> unilateral, <i>RF</i> right foot, <i>TIN</i> tubular interstitial nephritis; + phenotype present, − phenotype absent</p> </div> </div> <p /> <p id="Par7">In all five families mutated alleles were inherited recessively from each of the unaffected consanguineous parents (Fig. <a href="#Fig1">1a</a>). Identified variants were absent in 123,136 exomes and 15,496 genomes from the Genome Aggregation Database (gnomAD, accessed November 2017)<sup><a href="#CR11">11</a></sup> and 2497 individuals from the Greater Middle East (GME, accessed November 2017) Variome Project<sup><a href="#CR12">12</a></sup>. Furthermore, variant c.2207dupT (p.I737NfsX7) found in Moroccan families 1 and 2 was absent in 400 alleles of individuals of Moroccan descent. All four variants are predicted to results in a premature truncation of the FAT1 protein at amino acids 744, 871, 1043, and 3270, respectively (Fig. <a href="#Fig1">1b</a>). The gnomAD database contains only 46 LoF variants (expected number of LoF variants: 144) and none of them in homozygous condition, in comparison to 2601 missense variants (expected number of missense variants: 2585). We performed two-point linkage analysis for the <i>FAT1</i> variants using genotype and pedigree information (Fig. <a href="#Fig1">1</a>). Assuming an autosomal recessive inheritance and a penetrance of 0.99, we obtained a combined maximal LOD score of 5.3 for the <i>FAT1</i> variants. Patients F5-II:1 and F4-II-3 had been previously published, of whom the latter was diagnosed with Dubowitz syndrome (OMIM: 223370), although sequencing of <i>NSUN2</i> was negative<sup><a href="#CR13">13</a></sup>.</p> </div> <div class="deletionoffat1leadstocolobomainmouseandzebrafish" title="sec"> <div class="title" tagx="title" title="title">
Deletion of <i>Fat1</i> leads to coloboma in mouse and zebrafish</div> <p id="Par8">Given the important role of FAT1 in forming the earliest cell–cell contacts in epithelia<sup><a href="#CR3">3</a></sup> and in maintaining epithelial junctional integrity<sup><a href="#CR3">3</a>,<a href="#CR7">7</a>,<a href="#CR8">8</a></sup>, we hypothesized that FAT1 is involved in optic fissure fusion via epithelial cell-mediated fusion and thereby underlies the coloboma observed in patients. We analyzed the spatio-temporal mRNA expression of <i>Fat1</i> in the developing wild type (WT) mouse eye at E10.5, E11.5, and E12.5, approximately corresponding to "before", "during", and "after" optic fissure closure. <i>Fat1</i> had a dynamic expression at the mouse optic fissure margins, periocular mesenchyme, and optic cup with specific spatio-temporal patterns during optic fissure closure (Fig. <a href="#Fig2">2</a>). <i>Fat1</i> was expressed in the optic cup, including apposing edges of the optic fissure, in the lens vesicle, and periocular mesenchyme (POM) (Fig. <a href="#Fig2">2a–c</a>). <i>Fat1</i> was more highly expressed in the nasal than in the temporal portion of presumptive neural retina (Fig. <a href="#Fig2">2b</a>). Expression of <i>Fat1</i> in POM was elevated in the ventral compartment as compared to the dorsal compartment (Fig. <a href="#Fig2">2d–f</a>). The spatio-temporal pattern of <i>Fat1</i> expression supports the hypothesis of Fat1 involvement in optic fissure closure. Next, we performed morphological and histological analysis of <i>Fat1</i><sup>−/−</sup> mouse embryos at embryonic day (E) 14.5 which demonstrated completely penetrant coloboma (<i>n</i> &amp;gt; 20), recapitulating the prominent developmental eye defect observed in affected individuals. At E14.5, when the optic fissure is completely fused in WT (Fig. <a href="#Fig2">2g–i</a>), <i>Fat1</i><sup>−/−</sup> mice showed unfused margins (Fig. <a href="#Fig2">2n, o</a>) and/or persistence of POM—including early hyaloid vessel precursors (Fig. <a href="#Fig2">2k, l</a>, arrow) interposing between the closing edges of the fissure. Compared to wild-type (WT) embryos (Fig. <a href="#Fig2">2g</a>) <i>FAT1</i><sup>−/−</sup> mouse exhibited microphthalmia (Fig. <a href="#Fig2">2j, m</a>). The coloboma phenotype was observed to be completely penetrant in the homozygous mutant mice and was never observed in heterozygous mutant mice. Occurrence of unfused optic fissure margins apposed to each other and presence of POM tissue in the choroid fissure obstructing the fusion of optic fissure margins, suggests multiple mechanisms like fusion and morphogenetic defects contributing towards the coloboma phenotype. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p><i>FAT1</i><sup>-/-</sup> mouse exhibit microphthalmia and completely penetrant coloboma. <i>FAT1</i> mRNA expression pattern in sagittal (<b>a</b>–<b>c</b>) and coronal (<b>d</b>–<b>f</b>) mouse optic cup sections at E10.5 (<b>a</b>, <b>d</b>), E11.5 (<b>b</b>, <b>e</b>), and E12.5 (<b>c</b>, <b>f</b>). Compared to wild-type (WT) embryos (<b>g</b>–<b>i</b>, E14.5) <i>FAT1</i><sup>−/−</sup> mouse exhibited microphthalmia (<b>j</b>, <b>m</b>). Fused optic fissure margins of WT mouse embryos (E14.5, <b>h</b>, <b>i</b>; sagittal section) and unfused margins were seen in <i>Fat1</i><sup>−/−</sup> mice (<b>n</b>, <b>o</b>) and/or persistence of POM—including early hyaloid vessel precursors (<b>k</b>, <b>l</b>, arrow) interposing between the closing edges of the fissure (E14.5). D dorsal, V ventral, T temporal, N nasal</p>   </div> <p /> <p id="Par9">In zebrafish, <i>fat1a</i> expression was observed to be restricted to the rostral end of the embryo (Supplementary Fig. <a href="#MOESM2">3a–d</a>) during the time of optic fissure closure (24-h post-fertilization), as has been reported previously<sup><a href="#CR7">7</a></sup>. Morpholino-mediated knockdown of the zebrafish <i>fat1a</i> using two different concentrations, consistently resulted in coloboma (Supplementary Fig. <a href="#MOESM2">4a–d</a>), which was further confirmed by detailed histological analysis of day 3 post-fertilization larvae (Supplementary Fig. <a href="#MOESM2">4e–h</a>). At lower doses of morpholino the coloboma phenotype observed was similar to that observed for mouse, where the two optic fissure margins were very close to each other but failed to fuse. The variability in morphant phenotype observed at high doses of morpholino are depicted in Supplementary Fig. <a href="#MOESM2">5</a>.</p> </div> <div class="fat1isrequiredforjunctionalintegrityinhumanrpecells" title="sec"> <div class="title" tagx="title" title="title">
FAT1 is required for junctional integrity in human RPE cells</div> <p id="Par10">To further investigate the role of <i>FAT1</i> in optic fissure closure, we studied human primary RPE, the cell type present at the leading edge of optic fissure margins that mediates fissure closure<sup><a href="#CR14">14</a></sup>. The fusion of optic fissure margins is initiated by cellular processes emanating from the apposing edges of optic fissure margins, observed using transmission electron microscopy (TEM) (Supplementary Fig. <a href="#MOESM2">6a</a>), and forming "simple appositional type," contacts<sup><a href="#CR15">15</a></sup>. The failure to fuse optic fissure margins represents a classical epithelial fusion defect during eye development<sup><a href="#CR10">10</a></sup>. Using confocal microscopy, we observed FAT1 immunostaining with F-ACTIN fibers at the leading edges and filipodia of isolated RPE cells (Supplementary Fig. <a href="#MOESM2">6b–d</a>), as has been reported for other epithelial cell types. The specificity of the FAT1 antibody used was validated using mouse embryonic fibroblast (MEF) isolated from WT and <i>Fat1</i><sup>−/−</sup> mouse embryos (Supplementary Fig. <a href="#MOESM2">6c</a>), where FAT1 immuno-staining was observed at the earliest cell–cell contacts in WT cells but was absent in FAT1<sup>−/−</sup> cells. In sub-confluent RPE cultures, FAT1 localized with F-ACTIN fibers at leading cell edges (Supplementary Fig. <a href="#MOESM2">6b, d</a>) and with ZO-1 at the earliest cell–cell interactions (Supplementary Fig. <a href="#MOESM2">6e</a>), respectively.</p> <p id="Par11">We usually culture RPE cells on a 2D trans-well culture system where they form a polarized monolayer, to recapitulate in vivo electro-physiological properties of RPE monolayer. In confluent monolayer polarized RPE cultures, FAT1 localized at the cell borders similar to the organized F-ACTIN filaments and ZO-1 staining (Fig. <a href="#Fig3">3b, c</a> top panels) suggesting that it might be involved in formation or maintenance of cell–cell junctional complexes. Short hairpin RNA (shRNA)-mediated knockdown of <i>FAT1</i> in sub-confluent RPE cultures resulted in loss of FAT1 from filopodia and disruption of ZO-1 immuno-staining at the earliest cell–cell junctions (Supplementary Fig. <a href="#MOESM2">6d, e</a>). Knockdown of <i>FAT1</i> in RPE cells before monolayer formation (Fig. <a href="#Fig3">3a</a>) resulted in failure to organize F-ACTIN fibers (Fig. <a href="#Fig3">3b</a>), loss of ZO-1 staining from cell–cell junctions (Fig. <a href="#Fig3">3c</a>), and significantly reduced trans-epithelial barrier potential, suggesting compromised junctional integrity (Fig. <a href="#Fig3">3d</a>). As such, FAT1 might play an important role in the earliest RPE cell–cell junctions and F-ACTIN filament organization during RPE monolayer formation. To further confirm our assumption that junctional integrity is compromised due to FAT1 knockdown we studied β-CATENIN staining which is known to localize at adherens junctions. Although not as dramatic as loss of ZO-1 staining, we observed a significant disruption of β-CATENIN staining (Supplementary Fig. <a href="#MOESM2">7</a>), confirming our assumption that FAT1 might be involved in establishing junctional integrity of RPE monolayer. These observations were further supported by TEM studies where shRNA-mediated knockdown of FAT1 resulted in a failure to organize RPE cells into an epithelial monolayer (Fig. <a href="#Fig4">4a, b</a>) and disrupted epithelial junctions (Fig. <a href="#Fig4">4c, d</a>). We currently do not know if the failure to organize RPE cells into an epithelial monolayer is primarily due to loss of FAT1 or a secondary defect resulting from disrupted junctions. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>FAT1 knockdown disrupts F-ACTIN and ZO-1 localization in RPE. ShRNA-mediated knockdown of FAT1 in RPE cells was confirmed using Western blotting (<b>a</b>, <i>N</i> = 3). Disorganization of F-ACTIN fibers (<b>b</b>) and loss of ZO-1 staining pattern (<b>c</b>) was observed in RPE cells upon knockdown of FAT1. Trans-epithelial electrical resistance was measured at week one and two post-FAT1 knockdown to determine junctional integrity (<b>d</b>, <i>N</i> = 6). Scale bar is 20 µm. Error bars represent standard error of mean, and statistical significance is at <i>P</i> &amp;lt; 0.05 (two tailed <i>t</i>-test) denoted by "*". Raw data is provided in supplementary section</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>FAT1 knockdown results in a failure to organize RPE cells into a monolayer. Transmission electron miscroscopy (TEM) of RPE cells cultured on trans-wells for 2 weeks following treatment with scrambled (<b>a</b>, <b>c</b>) and FAT1 shRNA (<b>b</b>, <b>d</b>). A transverse section depicting RPE cells (*, scale bar is 2 µm) sitting on top of trans-well membrane (<b>a</b>, arrow). A higher magnification showing epithelial tight junction (<b>c</b>, <b>d</b>, arrow, scale bar is 500 nm) between two neighboring cells</p>   </div> <p /> <p id="Par12">Interestingly, in vivo differentiation of RPE from neuro-ectodermal to an epithelial cell type is not affected by loss of FAT1 as observed during mouse eye development (Fig. <a href="#Fig2">2g–m</a>). No defects were observed in the organization of F-Actin fibers and ZO-1 staining pattern in the RPE flat-mounts from WT and <i>Fat1</i><sup>−/−</sup> mouse embryos (Supplementary Fig. <a href="#MOESM2">8</a>), suggesting a specific role of FAT1 in epithelial sheet fusion by mediating earliest cell–cell junctions, rather than differentiation towards epithelial lineage.</p> </div> <div class="thefat1cytoplasmictailisinvolvedinopticfissureclosure" title="sec"> <div class="title" tagx="title" title="title">
The Fat1 cytoplasmic tail is involved in optic fissure closure</div> <p id="Par13">We also studied the expression of the remaining <i>Fat</i> gene family members, <i>Fat2</i>, <i>Fat3</i>, and <i>Fat4</i>, in the developing WT mouse eye at E10.5, E11.5, and E12.5. <i>Fat2</i>, and <i>Fat3</i> were not expressed in the developing eye. The <i>Fat4</i> mRNA expression pattern was overlapping with that of <i>Fat1</i> (Supplementary Fig. <a href="#MOESM2">9a–e</a>). The <i>Fat4</i> mRNA expression was observed around the optic cup in peri-ocular mesenchyme and RPE. Although, we observed that both <i>Fat1</i> and <i>Fat4</i> had overlapping expression pattern in the developing mouse eye, we only found <i>Fat1</i> to be essential for optic fissure closure in mouse and zebrafish. Unlike <i>Fat1</i><sup>−/−</sup> mice, no developmental defects, including microphthalmia or coloboma were observed in <i>Fat4</i>−/− mice eyes by E12.5 (<i>n</i> = 20, Supplementary Fig. <a href="#MOESM2">9g–l</a>). Similarly, morpholino-mediated knockdown of the zebrafish <i>fat4</i> showed no apparent defects during eye development. A similar observation was made during zebrafish pronephros and lens epithelia development, where morpholino-mediated <i>fat1</i> knockdown, but not <i>fat4</i>, resulted in cystic pronephros<sup><a href="#CR7">7</a></sup> and loss of apical–basal polarity of <i>Fat1</i><sup>−/−</sup> mouse lens epithelia<sup><a href="#CR6">6</a></sup>. This is in line with the observation that recessive mutations in <i>FAT4</i> result in Van Maldergem syndrome (MIM615546) and Hennekam lymphangiectasia-lymphedema syndrome 2 (MIM616006), neither of which include coloboma or any other eye defects.</p> <p id="Par14">Since FAT1 and FAT4 have similar extra-cellular domains, but diverge significantly in the cytoplasmic region, we hypothesized that the FAT1 cytoplasmic tail could be involved during optic fissure closure. We therefore targeted the cytoplasmic tail region of zebrafish <i>fat1a</i> using CRISPR/<i>cas9</i> and selected for frameshift truncating alleles, resulting in loss of the two preferred VASP/MENA and the PDZ-binding domains located at the C-terminal end, due to a premature stop codon (Supplementary Fig. <a href="#MOESM2">10a, b</a>). We observed optic fissure fusion defects in embryos with homozygous alleles of truncated <i>fat1a</i> (Fig. <a href="#Fig5">5a, b, e</a>, f). This was further confirmed by detailed histological analysis of optic cup during the time of optic fissure fusion (Fig. <a href="#Fig5">5c, d, g</a> and h). Sagittal section through the zebrafish optic cups at the time of optic fissure fusion revealed abnormal organization of the optic fissure margins and migration and/or delayed differentiation of the RPE cells at the optic fissure margins (Fig. <a href="#Fig5">5d, h</a>, black arrows) in homozygous <i>fat1a</i> mutants. Compromised structural integrity of RPE monolayer around the optic cup of homozygous <i>fat1a</i> mutants was also observed, pointing towards a combination of morphogenetic and structural organizational defects as the cause of fissure margins fusion failure. These observations also suggest that VASP/MENA and PDZ-binding domains located in the cytoplasmic tail of FAT1 are involved in optic fissure closure. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Zebrafish embryos with homozygous alleles of truncated <i>fat1a</i> display coloboma. CRISPR/Cas9-mediated introduction of frame-shift mutations in FAT1 C-terminal resulted in optic fissure closure defects (<b>a</b> and <b>e</b>, scale bar is 0.1 mm). A higher magnification of eye depicting fused margins in WT and unfused margins in homozygous mutant (*, <b>b</b> and <b>f</b>, scale bar is 0.05 mm). Sagittal sections of zebrafish embryos (24–30 hpf) followed by toludene blue staining showing organization of the optic cup (<b>c</b> and <b>g</b>, scale bar is 50 µm). Higher magnification of the optic cup shows morphology of optic fissure margins in WT and homozygous mutant (<b>d</b>, <b>h</b>, scale bar is 20 µm)</p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par15">We here identified five unrelated families presenting with a new syndrome consisting of colobomatous microphthalmia, ptosis, and cutaneous syndactyly with or without glomerulotubular nephropathy, associated with homozygous frame-shift mutations in the <i>FAT1</i> gene. The causal relation between loss of function mutations in <i>FAT1</i> and coloboma was established through studies in mice and fish loss-of-function models, underscoring a highly conserved role of <i>FAT1</i> during vertebrate eye development. We demonstrated for the first time that FAT1 is expressed in human primary RPE cells, the primary site for optic fissure margins fusion, placing it exquisitely at the precise cell type and location for facilitating optic fissure fusion during vertebrate eye development. Using an in vitro culture system, we showed that FAT1 is essential for establishment of the earliest cell to cell junctions, F-Actin organization, and maintenance of junctional integrity of RPE monolayer.</p> <p id="Par16">The novel clinical syndrome we describe here joins the growing list of human conditions involving both coloboma and renal disease. The differential diagnosis includes amongst others CHARGE, Papillorenal, Temtamy, and Klippel–Feil syndromes. Mutations in <i>SALL1/SALL2, YAP1</i> and <i>STRA6</i> also result in a syndromic form of coloboma with a renal phenotype<sup><a href="#CR8">8</a>,<a href="#CR16">16</a></sup>. Although we found a strikingly high penetrance of ocular and limb defects in affected individuals, renal dysfunction was less prominent. We found a wide spectrum of renal involvement ranging from asymptomatic proteinuria (3/10) to severe nephrotic syndrome (2/10). In addition, in some of the affected individuals, proteinuria was only present intermittently. We therefore recommend that asymptomatic <i>FAT1</i> patients should undergo clinical evaluation of kidney function, which should be repeated on a regular basis. <i>FAT1</i> is another example of a gene wherein genetic variation has pleiotropic effects, with biallellic missense variants leading to non-syndromic glomerulotubular nephropathy and homozygous frameshift variants causing the broad multi-system disorder that we describe here.</p> <p id="Par17">Our mechanistic studies in mice, zebrafish, and RPE cell cultures support the causality of <i>FAT1</i> biallellic frameshift variants in the pathogenesis of coloboma, and for the first time implicate <i>FAT1</i> in optic fissure fusion during eye development. During optic fissure closure apposing margins of the presumptive neural retina that are lined by presumptive RPE adhere initially by cellular protrusions (Supplementary Fig. <a href="#MOESM2">6a</a>), then proceed to form more stable appositional junctions<sup><a href="#CR17">17</a></sup> and subsequently undergo differentiation of the RPE and neuroepithelium to yield a complete eye where RPE is overlying neural retina. In our previous study we showed that genes that are dynamically expressed at the apposing edges of the optic fissure margins during the process of optic fissure closure are important in this developmental process<sup><a href="#CR18">18</a></sup>. Tissue collected from optic fissure margins in wild-type mouse embryos at E10.5, E11.5, and E12.5—approximately corresponding to "before", "during", and "after" optic fissure closure—identified the important role of <i>Nlz</i> genes in optic fissure closure in mouse and zebrafish<sup><a href="#CR18">18</a></sup>. In the same screen we independently identified <i>FAT</i> atypical cadherins as potential candidates for optic fissure closure<sup><a href="#CR18">18</a></sup>. Though we were not able to show FAT1 immunoreactivity at the earliest cellular processes that initiate the optic fissure fusion process, due to technical difficulties, we did observe the presence of Fat1 mRNA at the leading edges of the optic fissure margins (Fig. <a href="#Fig2">2b, c, e, f</a>). In cell culture experiments we observed FAT1 immunoreactivity at leading edges, filopodia, and earliest cell–cell contacts of RPE cells (Supplementary Fig. <a href="#MOESM2">6b–e</a>). Neural tube closure, like optic fissure fusion, is a classical epithelial sheet adhesion and fusion morphogenetic event. <i>Fat1</i><sup>−/−</sup> mice display partially penetrant anterior neural tube closure defects due to reduced actin accumulation, leading to aberrant apical constriction in the neural epithelium cells<sup><a href="#CR9">9</a></sup>. The presence of optic fissure and neural tube closure defects supports the role of <i>FAT1</i> in epithelial cell–cell adhesion and/or fusion</p> <p id="Par18">We demonstrate that FAT1 plays an essential role in formation and maintenance of junctional integrity. The observations that support this finding include disruption of ZO-1 and β-CATENIN immuno-staining at the earliest cell–cell junctions, failure to organize F-ACTIN fibers and failure of RPE cells to organize an epithelial monolayer resulting in reduced trans-epithelial barrier potential. Disrupted epithelial junctions suggests a critical role of FAT1 in formation and maintenance of earliest cell–cell junctions. In line with the ocular and kidney involvement in affected individuals, loss of FAT1 results in decreased cell adhesion in podocytes (an epithelial cell type), and disrupted lumen formation in renal tubular cells<sup><a href="#CR8">8</a></sup>.</p> <p id="Par19">Our work in zebrafish where we targeted the cytoplasmic tail region of zebrafish <i>fat1a</i> using CRISPR/<i>cas9</i> provides evidence that VASP/MENA and PDZ-binding domains located in the cytoplasmic tail of FAT1 are involved in optic fissure closure. The FAT1 cytoplasmic tail acts as a binding partner for β-CATENIN, RERE, ENA/VASP, and PDZ domain containing proteins including SCRIBBLE. Previous studies have suggested the association of coloboma with β-CATENIN<sup><a href="#CR19">19</a></sup>, RERE<sup><a href="#CR20">20</a></sup>, and SCRIBBLE<sup><a href="#CR21">21</a></sup>; currently we do not know which of these proteins are involved in <i>FAT1</i>-mediated coloboma. The ENA/VASP proteins have been implicated in F-ACTIN polymerization/reorganization during epithelial adhesion<sup><a href="#CR22">22</a></sup>, and establishing FAT1-mediated initial cell–cell contact<sup><a href="#CR3">3</a>,<a href="#CR23">23</a></sup>.</p> <p id="Par20">While a failure to fuse the fissure margins during the process of optic fissure fusion is a commonly appreciated cause for coloboma formation, and our data supports such a mechanism, it could also be due in part to morphogenetic events. Our finding that intervening peri-ocular mesenchyme can be observed obstructing fusion of fissure margins (Fig. <a href="#Fig2">2k, l</a> panels) also provides evidence for an underlying morphogenic event that may at least in part explain the coloboma phenotype. We acknowledge that further work is needed to provide more detailed insight into the latter mechanism.</p> <p id="Par21">Heterozygous loss of function mutations in human <i>YAP1</i> and loss of <i>Cdc42</i> in mouse have been associated with ocular coloboma<sup><a href="#CR24">24</a></sup>. These two genes have both been implicated in Hippo signaling<sup><a href="#CR25">25</a></sup>. Yes-associated protein (YAP) is a downstream transcriptional co-activator inhibited by Hippo signaling has a crucial role in nephron induction and morphogenesis<sup><a href="#CR25">25</a></sup>, and specification of RPE cell fate during eye development<sup><a href="#CR14">14</a></sup>. Cdc42 has been shown to act upstream of YAP in nephron progenitor cells to promote YAP-dependent gene expression necessary to form functioning nephrons<sup><a href="#CR25">25</a></sup>. Tissue-specific deletion of Yap1 as well as Cdc42 in mouse results in strikingly similar and severe defects in kidney development<sup><a href="#CR25">25</a></sup>. The PDZ-binding domain of FAT1 has been shown to recruit Scribble for mediating Hippo signaling that leads to Yap inhibition, during pronephros development in zebrafish<sup><a href="#CR7">7</a></sup>. Recently, loss of FAT1 has been shown to decrease Cdc42 activity<sup><a href="#CR8">8</a></sup>. Furthermore, in vitro studies in HEK293 cells showed that FAT1 knockdown results in nuclear accumulation of YAP1<sup><a href="#CR26">26</a></sup>. Together, these data suggest that the ophthalmic phenotype due to loss of FAT1 may also result from deregulated YAP1 activity. However, shRNA-mediated knock-down of FAT1 in primary human RPE monolayer on trans-wells showed no significant differences in YAP/TAZ localization pattern compared to scrambled shRNA control (Supplementary Fig. <a href="#MOESM2">11a</a>). Also, protein levels (<i>n</i> = 3) of total YAP/TAZ and phosphorylated form of YAP (pYAP-Ser127) were unchanged in scrambled and <i>FAT1</i> shRNA-treated RPE monolayers (Supplementary Fig. <a href="#MOESM2">11b</a>). Similarly, no differences were observed in YAP/TAZ-staining patterns between MEF cells isolated from WT and <i>Fat1</i><sup>−/−</sup> mouse embryos (Supplementary Fig. <a href="#MOESM2">11c</a>). Moreover, YAP immunostaining in mouse E11.5–12.5 optic cup sections (sagittal) identified no differences in the pattern of YAP staining between WT and <i>Fat1</i><sup>−/−</sup> mouse presumptive neural retina and RPE cells (Supplementary Fig. <a href="#MOESM2">11d</a>), While we did not observe any significant changes in YAP levels and localization upon loss of FAT1 in our experiments, additional studies on mechanisms by which FAT1 regulates YAP1 during eye development are warranted. Since we observed <i>FAT1</i> RNA labeling in POM it is possible that FAT1 could be involved in regulating YAP levels in the POM but not in the RPE.</p> <p id="Par22">In conclusion, our study associates <i>FAT1</i> frameshift mutations to a previously undescribed clinically recognizable syndrome consisting of ophthalmic, facial, renal, and skeletal abnormalities. <i>Fat1</i> ablation in mice and zebrafish showed a similar ophthalmic phenotype, for the first time underscoring the fundamental and evolutionary conserved role of the atypical cadherin <i>FAT1</i> during optic fissure closure. FAT1 also seems to be involved for earliest cell–cell contacts, apical junction formation, and F-ACTIN re-organization in primary human RPE cells. Our study identified <i>FAT1</i> as another example of a gene wherein genetic variation has pleiotropic effects, where biallellic missense variants lead to non-syndromic glomerulotubular nephropathy and homozygous frameshift variants lead to a broad multi-system disorder.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="clinicalevaluationandwes" title="sec"> <div class="title" tagx="title" title="title">
Clinical evaluation and WES</div> <p id="Par23">Patients were referred to the Department of Clinical Genetics for further comprehensive genetic analyses and counseling. The study protocol was approved by the local Institutional Review Boards where the patients were followed and signed informed consent was obtained from the patients or their parents. The authors affirm that human research participants provided informed consent, for publication of the images in Fig. <a href="#Fig1">1</a>. Sequencing protocols and mean coverage for each family is summarized in Supplementary Table <a href="#MOESM2">1</a>.</p> <p id="Par24">For family 1, genome-wide homozygosity mapping was performed in patients IV-1, IV-3, IV-5, and their parents, by using the Affymetrix 250K <i>Nsp</i>I SNP genotyping microarray. Multipoint LOD scores across the whole genome were calculated using the MERLIN software (<a href="http://www.sph.umich.edu/csg/abecasis/Merlin">http://www.sph.umich.edu/csg/abecasis/Merlin</a>), assuming recessive inheritance with complete penetrance. Three regions of homozygosity were shared by the three affected siblings: on chromosome 4 with a size of 9.5 Mb, on chromosome 13 with a size of 1 Mb, and on chromosome 17 with a size of 2.2 Mb. Because of the high number of genes present in these regions, WES was then conducted in patients IV-3 and IV-5, and their parents using Agilent SureSelect libraries and sheared with a Covaris S2 Ultrasonicator. Exome capture was performed with the 51 Mb SureSelect Human All Exon kit v5 (Agilent Technologies). Sequencing was carried out on a pool of barcoded exome libraries using a HiSeq 2500 instrument (Illumina), generating 100 + 100 bp paired-end reads, with a mean depth of coverage of ×140. After demultiplexing, paired-end sequences were mapped to the reference human genome (GRCh37/hg19 assembly, NCBI) using Burrows–Wheeler aligner (BWA). Downstream processing was performed using the Genome Analysis Toolkit (GATK)<sup><a href="#CR17">17</a></sup>, SAMtools<sup><a href="#CR27">27</a></sup>, and Picard. Variant calls were made with the GATK UnifiedGenotyper. Variant annotation was based on Ensembl release 71. Variants were filtered against publicly available SNPs plus variant data from more than 7000 in-house exomes (Institut <i>Imagine</i>).</p> <p id="Par25">In family 2, the coding exons were captured using the Agilent SureSelect Human All Exon v5 and sequenced on the Hiseq2000 sequencer (Illumina) at a target mean depth of 50 reads per target base. SOAPaligner (version 2.21) was used to align sequence reads to the human reference genome (NCBI build 37/hg19). For single nucleotide variants, functional annotation was performed with SOAPsnp. For insertion/deletion detection, we aligned sequence reads by BWA and performed annotation with GATK.</p> <p id="Par26">In family 3, patient genomic DNA was isolated, followed by exome capture using the SureSelect Human All Exon kit V4 (Agilent technologies Inc). Sequencing of captured fragments was performed on the HiSeq2500 platform with a mean depth of coverage of 84X After alignment of sequence reads, the DNAnexus software (Palo Alto, CA, release October 2011) was used to perform variant calling with the human reference genome (NCBI build 37/hg19) as reference. Variant filtering was subsequently performed against public databases (dbSNP129, HapMap, 1000 Genomes).</p> <p id="Par27">In family 4, exome capture for WES was performed using the NimbleGen SeqCap EZ Human Exome Library (Roche NimbleGen Inc., Madison, WI). Sequencing of paired-end 100 bp fragment reads was performed on the Illumina HiSeq2000 with a mean depth of coverage of ×90. The VarBank v.2.1 data analysis pipeline was used for mapping, alignment, and variant calling. Alignment of sequenced reads, indexing of the reference genome, variant calling, and annotation was conducted using a pipeline consisting of BWA, Samtools, Picard, and Annovar<sup><a href="#CR28">28</a></sup>.</p> <p id="Par28">In family 5, genomic DNA was isolated from blood lymphocyte or saliva samples. Exome capture for WES was performed using the SureSelect Human All Exon kit V5 (Agilent technologies Inc). Subsequently, captured fragments were sequenced on the llumina HiSeq2000 with a mean depth of coverage of ×50. The CLC Genomics Workbench™ (version 6.5.2) software (CLC bio, Aarhus, Denmark) was used to align sequence reads to the human reference genome (NCBI build 38/hg19). Variant filtering was then performed against public databases (dbSNP147 and 1000 Genomes). We excluded synonymous and intronic variants that were not located at splice sites. WES data was evaluated for potential disease-causing mutations in &amp;gt;50 known monogenic genes associated to nephrotic syndrome. A second round of filtering of the remaining variants was conducted using public databases (EVS server, ExAC, gnomAD and 1000 Genomes). Variants identified in families 1–5 were validated using Sanger sequencing and whenever parental DNA was available, segregation analysis was performed. A combined LOD score across the families was calculated using phenotype, genotype, and pedigree information of each family by two-point linkage analysis with the use of the Superlink-Online software package<sup><a href="#CR29">29</a></sup> with the assumption of a recessive pattern of inheritance, a disease-allele frequency of 0.001 and a penetrance of 0.99.</p> </div> <div class="genotypingoffat1c.2207duptinmoroccancontrols" title="sec"> <div class="title" tagx="title" title="title">
Genotyping of <i>FAT1</i> c.2207dupT in Moroccan controls</div> <p id="Par29">Blood samples from the umbilical cord were collected from 200 unrelated newborns. Individuals originated from different regions in Morocco and the Moroccan origin of parents and grandparents was established. DNA was extracted from 3 mL blood with standard salting-out method. Informed consent was obtained from the parents. We developed a realtime polymerase chain reaction (PCR; Applied Biosystems 7500 Fast Real-Time PCR Systems) assay using TaqMan probes for the <i>FAT1</i> c.2207dupT mutation (Table <a href="#Tab1">1</a>), and validated the assay using homozygous and heterozygous members of Family 1.</p> </div> <div class="animalbreedingandmaintenance" title="sec"> <div class="title" tagx="title" title="title">
Animal breeding and maintenance</div> <p id="Par30">All animal studies were performed in accordance with the NEI/NIH animal ethics committee guidelines (NEI-605 and NEI-648). C57BL/6J mice were obtained from Jackson Laboratories (Bar Harbor, ME). <i>Fat1</i> and <i>Fat4</i> knockout mice have been described previously<sup><a href="#CR6">6</a></sup> and were housed and bred according to an NEI-approved animal study protocol (NEI-605). Embryo harvesting was performed according to standard protocol, with E0.5 marking the day after the appearance of a cervical mucus plug. Adult zebrafish were maintained in an automated fish housing systems following a standard protocol and zebrafish embryos were maintained at 28.5 °C in fish embryo medium as previously described<sup><a href="#CR30">30</a></sup>.</p> </div> <div class="histologicalandinsitustainingofmousetissue" title="sec"> <div class="title" tagx="title" title="title">
Histological and in situ staining of mouse tissue</div> <p id="Par31">Mice were euthanized using carbon dioxide followed by cervical dislocation. Isolated mouse embryos for histopathology were dissected on ice-cold phosphate buffered saline (PBS) and fixed overnight in phosphate-buffered 4% paraformaldehyde at 4 °C. Methacrylate sections through the optic fissure (eyes) or in appropriate sagittal orientation, stained with hematoxylin and eosin (H&amp;amp;E), were used for histopathology. In situ hybridization was performed using a Fat4 probe that included the 3′ UTR cloned into a plasmid as previously described<sup><a href="#CR4">4</a>,<a href="#CR31">31</a></sup>. The plasmid for generating the Fat1 probe (ID 6841037) was from Open Biosystems (Lafayette, CO, USA). In situ and immunofluorescence staining of mouse cryo-sections was performed as previously described<sup><a href="#CR32">32</a></sup>. For RPE flat-mounts, mouse embryonic (E14.5–15.5) eye cups were isolated and dissected to remove presumptive neural-retina. The surrounding choroidal tissue was not removed completely to provide structural support to the RPE monolayer. The dissected out RPE was immuno-stained with ZO-1 and F-Actin as described below. The plane of RPE during confocal imaging of flat-mounts was determined by adding a transmitted PMT (T-PMT) to visualize the location of melanosomes present on the apical side of RPE.</p> </div> <div class="zebrafishembryomanipulationbymorpholinoandcrisprcas9" title="sec"> <div class="title" tagx="title" title="title">
Zebrafish embryo manipulation by morpholino and CRISPRcas9</div> <p id="Par32">Previously described zebrafish <i>fat1a</i> translation-blocking morpholino oligonucleotides<sup><a href="#CR7">7</a>,<a href="#CR8">8</a></sup> (Gene Tools LLC., Philomath, OR), diluted in 0.1 M KCl, nuclease-free water and phenol red, were injected (0.45 and 0.52 pMol) in freshly fertilized zebrafish embryos at the one cell stage. The knockdown experiments were repeated using a different morpholino <i>fat1a</i> (TTTGCAGCGCACTCCTCTCTGAAAC) to validate the results. The zebrafish <i>fat4</i> morpholino (CCGGGTTTTCCCGAGCCTCATACAT) which has been previously reported<sup><a href="#CR7">7</a></sup>, and the injection control morpholino (CCTCTTACCTCAGTTACAATTTATA), were used as described above for the <i>fat1a</i> morpholino.</p> <p id="Par33">CRISPR targets and primers spanning the targets were selected using the online tool CHOPCHOP<sup><a href="#CR33">33</a></sup>. The selected target had no off-target sequences of homology. The oligonucleotides containing T7 promoter, target, guide RNA scaffold, and termination signal sequences were purchased from IDT (Coralville, IA) in the form of gBlocks<sup>®</sup> fragments. Guide RNA were synthesized in vitro by driving transcription mediated by T7 promoter using the HiScribe™ T7 High Yield RNA Synthesis Kit (NEB, Ipswich, MA). For targeting the cytoplasmic tail of zebrafish <i>fat1a</i>, ~1 nL of in vitro synthesized guide RNA (200 ng/µL) and Cas9 protein (500 ng/µL) were injected in one-cell stage embryos (F0 embryos). F0 embryos were maintained until adulthood and then outcrossed to WT ABTL to isolate F1 progenies carrying frame-shift mutations in the germline. The selected F1 progenies were inbred to generate F2 embryos that were screened for coloboma phenotype and confirmation by genotyping. Zebrafish <i>fat1a</i> mutants were maintained as heterozygotes and inbreed to generate homozygous mutants.</p> </div> <div class="cellcultureexperiments" title="sec"> <div class="title" tagx="title" title="title">
Cell culture experiments</div> <p id="Par34">Mouse embryonic fibroblast cells were derived from embryonic day 12.5 (E12.5) embryos as described previously<sup><a href="#CR34">34</a></sup>. Primary human fetal RPE cells kindly provided by Dr. Sheldon Miller, derived from cadaver eyes obtained from Advanced Bioscience Resources (Alameda, CA, USA). RPE cells were cultured in Primaria<sup>®</sup> tissue culture flasks (BD Biosciences, Franklin Lakes, NJ) in culture medium based on MEM-α, as described previously<sup><a href="#CR35">35</a></sup>. Once confluence was reached, cells were seeded onto vitronectin-coated polystyrene-based trans-wells placed in the wells of a 12-well plate or four-well chamber slides for shRNA-mediated knockdown, immuno-fluorescence staining, and confocal microscopy, as described previously<sup><a href="#CR36">36</a></sup>. <i>FAT1</i> (SHCLNV-NM_005245) and scrambled (SHC016H) shRNA lentiviral transduction particles were purchased from Sigma Aldrich (St. Louis, MO, USA). Lentiviral transduction was performed 48 h post seeding on four-well chamber slides followed by fixation and immuno-fluorescence staining 48 h post transduction. In trans-well culture system, shRNA lentiviral transduction was performed 48 h and again at 120 h post seeding, followed by electron microscopy and immunofluorescence staining 2 weeks post transduction. Trans-epithelial resistance of RPE monolayers on trans-well cultures was measured using EVOM2 (World Precision Instruments, Sarasota, FL, USA) at 1 and 2 weeks post-shRNA transduction, as described by manufacturer (<a href="https://www.wpiinc.com/var-2754-epithelial-volt-ohm-teer-meter">https://www.wpiinc.com/var-2754-epithelial-volt-ohm-teer-meter</a>). For TEM, mouse eyes (E11.5) and RPE monolayers on trans-wells (2 weeks post transduction) were processed as previously described<sup><a href="#CR37">37</a></sup>. Briefly, specimens were doubly fixed in gluteraldehyde (2.5% in PBS) and osmium tetroxide (0.5% in PBS), dehydrated, and embedded in Spurr’s epoxy resin. Ultrathin sections (90 nm) were prepared and double-stained with uranyl acetate and lead citrate, and viewed with JEOL JEM-1010 (Peabody, MA, USA) and photographed.</p> </div> <div class="immunofluorescencestainingofcellsandconfocalimaging" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence staining of cells and confocal imaging</div> <p id="Par35">Cultured cells were fixed for 15 min in 4% paraformaldehyde (PFA) in PBS. After washing with 1× PBS and permeabilization and blocking in ICC buffer (0.5% BSA, 0.5% Tween, and 0.1% triton X100 1× PBS). Cells were then incubated overnight at 4 °C with the primary antibody in ICC buffer (1:100 dilution). After multiple washes in PBS, the cells were incubated for 1 h at room temperature in Alexa488 or 555 conjugated goat anti-rabbit and/or ant-mouse antibody or Phalloidin (Abcam ab176756, ab176753) and Hoechst33342 (1:1000 dilution in ICC buffer). Cells were then washed in PBST before mounting with Fluoromount-G<sup>®</sup> (SouthernBiotech, Birmingham, AL, USA) imaging. Primary antibodies used were FAT1 (Abcam: ab190242 and EMD Millipore: MABC612), YAP/TAZ (Cell Signaling Technology: 8418), FITC-tagged ZO-1 (Invitrogen: 33–9111). FAT1 antibody specificity was confirmed in mouse embryonic fibroblast derived from WT and <i>Fat1</i><sup>−/−</sup> mouse.</p> <p id="Par36">Zeiss confocal microscopes 700 and 880 coupled with Airyscan<sup>®</sup> detector was used for confocal imaging. The images were analyzed using ZEN Software (Carl Zeiss Microscopy LLC, Thornwood, NY). The cell culture experiments were repeated three times for each scrambled and FAT1 shRNA conditions.</p> </div> <div class="westernblotting" title="sec"> <div class="title" tagx="title" title="title">
Western blotting</div> <p id="Par37">For Western blotting cells were lysed with RIPA lysis buffer (Sigma-Aldrich, St. Louis, MO) containing protease inhibitor cocktail and Halt™ phosphatase inhibitor cocktail (Pierce Biotechnology), centrifuged at 14,000 × <i>g</i>, and supernatant was collected. Total protein concentration was determined by BCA protein assay kit (Pierce Biotechnology, Foster City, CA, USA). Samples (40 µg/each well) were SDS–PAGE electrophoresed using Criterion XT system for high molecular weight proteins (XT-running buffer, XT-sample loading dye and 3–8% Tris–acetate pre-casted gels Bio-Rad, Hercules, CA, USA), blotted onto polyvinylidene fluoride (PVDF) membranes (Bio-Rad) and immunoreacted with antibodies to FAT1 (Abcam: ab190242), YAP/TAZ (Cell Signaling Technology: 8418), phosphor YAP- Ser127 (Cell Signaling Technology:13008), and β-ACTIN (Cell Signaling Technology: 3700) for normalization (1:1000 dilution). The membranes were then incubated with secondary antibodies (LI-COR, goat anti-mouse IRDye 800CW or donkey anti-rabbit IRDye 680RD, 1: 20,000 dilution) for 45 min at room temperature followed by washings, scanned using the Odyssey infrared scanner and analyzed using Image Studio Lite Ver.4.0 (LI-COR Inc., Lincoln, NE, USA). Uncropped scan of the blots is provided in the Supplementary Information section.</p> </div> <div class="urls" title="sec"> <div class="title" tagx="title" title="title">
URLs</div> <p id="Par38">For dbSNP, see <a href="http://www.ncbi.nlm.nih.gov/SNP/">http://www.ncbi.nlm.nih.gov/SNP/</a>; for CHOPCHOP, see <a href="http://chopchop.cbu.uib.no">http://chopchop.cbu.uib.no</a>; for 1000 Genomes Project, see <a href="http://www.1000genomes.org/">http://www.1000genomes.org/</a>; for NHLBI ESP6500 databases, see <a href="http://evs.gs.washington.edu/EVS/">http://evs.gs.washington.edu/EVS/</a>; for VariantDB, see http://www.biomina.be/app/variantdb; for ANNOVAR, see <a href="http://www.openbioinformatics.org/annovar/">http://www.openbioinformatics.org/annovar/</a>; for NCBI Map Viewer, see <a href="http://www.ncbi.nlm.nih.gov/mapview/">http://www.ncbi.nlm.nih.gov/mapview/</a>; for Exome Aggregation Consortium (ExAC), Cambridge, MA, see <a href="http://exac.broadinstitute.org">http://exac.broadinstitute.org</a>; for Genome Aggregation Database (gnomAD), Cambridge, MA, see <a href="http://gnomad.broadinstitute.org">http://gnomad.broadinstitute.org</a>.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41467_2019_8547_MOESM1_ESM.pdf">LINK</a> <p>Peer Review File</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41467_2019_8547_MOESM2_ESM.pdf">LINK</a> <p>Supplementary Information</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41467_2019_8547_MOESM3_ESM.xlsx">LINK</a> <p>Source Data File</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Journal peer review information</b>: <i>Nature Communications</i> thanks Stephan Heermann, and the other anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available.</p> <p><b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>These authors contributed equally: Najim Lahrouchi, Aman George, Ilham Ratbi.</p> </div> <div class="fn-type-" title=""> <p>These authors jointly supervised the work: Connie R. Bezzina, Brian P. Brooks, Abdelaziz Sefiani.</p> </div> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p><b>Supplementary Information</b> accompanies this paper at 10.1038/s41467-019-08547-w.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank the families for their participation and collaboration. We thank Tiziana Cogliati (NEI/NIH) for critical review of the manuscript; Drs. G. Walz and Michelle Dawn (University Freiburg Medical Center) for zebrafish <i>fat1</i> constructs; Dr. S. Miller and Omar Memon (NEI/NIH) for RPE cells. We thank Dr. Robert Farris (NEI, Biological Imaging Core) for expert assistance with confocal laser scanning microscopy, and Chris Ardeljan and Heba Mohammed (NEI/NIH) for technical assistance with electron microscopy. Andrew Wegerski and Tyler Fayard (NEI/NIH) for zebrafish breeding and maintenance. We thank Dr. Ahron Bloch, Prof. Michal Elhalel-Dranitzky, Prof. Dvora Rubinger, Dr. Amir Haze, Dr. Samer Khateb (Hadassah-Hebrew University Hospital, Jerusalem, Israel), and Prof. Francesco Semararo (Eye Clinic, Department of Neurological and Vision Sciences, University of Brescia, Italy) for their clinical assistance. We thank Leander Beekman for technical assistance (Academic Medical Center, Amsterdam). This work was supported by the Agence Nationale de la Recherche (ANR-10-IAHU-01), France and intramural program of the National Eye Institute/National Institutes of Health, USA (DK076683) to F.H. F.H. is the William E. Harmon Professor.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>C.R.B., B.P.P., and A.S. designed and supervised the study. N.L. and A.G. compiled results, wrote the manuscript, and prepared all the figures. N.L., I.R., A.H.-C., F.-Z.L., E.M.L., H.K., V.M., and S.M. performed sequencing data analysis. I.R., N.L., R.S., S.C.E., N.A., Y.L., H.E., I.C., V.L., A.P., J.A., S.L., B.W., A.B., and F.H. recruited patients and gathered detailed clinical information for the study. H.M. provided mouse strains and molecular biology reagents. S.B., K.B., F.O., and A.G. performed mouse studies. A.V.P. performed LOD score calculation and critically reviewed the manuscript. K.B. supervised and provided cell lines, reagents and funds for cell culture experiments. A.G. and R.S. performed cell culture experiments. A.G. performed zebrafish experiments. M.A. performed electron microscopy studies. All the authors read and agreed on the final draft of the manuscript.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blain</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertuzzi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Uveal coloboma: clinical and basic science update</span><span class="source" tagx="source" title="source">Curr. Opin. Ophthalmol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">447</span><span class="lpage" tagx="lpage" title="lpage">470</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/01.icu.0000243020.82380.f6">10.1097/01.icu.0000243020.82380.f6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16932062">16932062</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Patel, A. &amp;amp; Sowden, J. C. Genes and pathways in optic fissure closure. <i>Semin. Cell. Dev. Biol</i>. 10.1016/j.semcdb.2017.10.010 (2017).</span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanoue</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeichi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mammalian Fat1 cadherin regulates actin dynamics and cell–cell contact</span><span class="source" tagx="source" title="source">J. Cell. Biol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">165</span><span class="fpage" tagx="fpage" title="fpage">517</span><span class="lpage" tagx="lpage" title="lpage">528</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200403006">10.1083/jcb.200403006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15148305">15148305</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saburi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hester</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodrich</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNeill</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional interactions between Fat family cadherins in tissue morphogenesis and planar polarity</span><span class="source" tagx="source" title="source">Development</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">139</span><span class="fpage" tagx="fpage" title="fpage">1806</span><span class="lpage" tagx="lpage" title="lpage">1820</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/dev.077461">10.1242/dev.077461</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22510986">22510986</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sibinga</span><span class="given-names" tagx="given-names" title="given-names">NES</span></span></span><span class="mixed-article-title" title="mixed-article-title">Atrophin proteins interact with the Fat1 cadherin and regulate migration and orientation in vascular smooth muscle cells</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">284</span><span class="fpage" tagx="fpage" title="fpage">6955</span><span class="lpage" tagx="lpage" title="lpage">6965</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M809333200">10.1074/jbc.M809333200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131340">19131340</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugiyama</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shelley</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badouel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNeill</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McAvoy</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Atypical cadherin Fat1 is required for lens epithelial cell polarity and proliferation but not for fiber differentiation</span><span class="source" tagx="source" title="source">Investig. Opthalmology Vis. Sci.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">4099</span><span class="pub-id"><a href="https://dx.doi.org/10.1167/iovs.15-17008">10.1167/iovs.15-17008</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skouloudaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Scribble participates in Hippo signaling and is required for normal zebrafish pronephros development</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">8579</span><span class="lpage" tagx="lpage" title="lpage">8584</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0811691106">10.1073/pnas.0811691106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19439659">19439659</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gee</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">FAT1 mutations cause a glomerulotubular nephropathy</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">10822</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms10822">10.1038/ncomms10822</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26905694">26905694</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badouel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Fat1 interacts with Fat4 to regulate neural tube closure, neural progenitor proliferation and apical constriction during mouse brain development</span><span class="source" tagx="source" title="source">Development</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">142</span><span class="fpage" tagx="fpage" title="fpage">2781</span><span class="lpage" tagx="lpage" title="lpage">2791</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/dev.123539">10.1242/dev.123539</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26209645">26209645</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pai</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epithelial fusion during neural tube morphogenesis</span><span class="source" tagx="source" title="source">Birth Defects Res. Part A</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">817</span><span class="lpage" tagx="lpage" title="lpage">823</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/bdra.23072">10.1002/bdra.23072</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lek</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of protein-coding genetic variation in 60,706 humans</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">536</span><span class="fpage" tagx="fpage" title="fpage">285</span><span class="lpage" tagx="lpage" title="lpage">291</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature19057">10.1038/nature19057</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27535533">27535533</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of Greater Middle Eastern genetic variation for enhanced disease gene discovery</span><span class="source" tagx="source" title="source">Nat. Genet.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">1071</span><span class="lpage" tagx="lpage" title="lpage">1076</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.3592">10.1038/ng.3592</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27428751">27428751</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lougaris</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Profound T-cell defects in Dubowitz syndrome</span><span class="source" tagx="source" title="source">Pediatr. Allergy Immunol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">511</span><span class="lpage" tagx="lpage" title="lpage">513</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/pai.12238">10.1111/pai.12238</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24899539">24899539</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miesfeld</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Yap and Taz regulate retinal pigment epithelial cell fate</span><span class="source" tagx="source" title="source">Development</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">142</span><span class="fpage" tagx="fpage" title="fpage">3021</span><span class="lpage" tagx="lpage" title="lpage">3032</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/dev.119008">10.1242/dev.119008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26209646">26209646</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hero</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Optic fissure closure in the normal cinnamon mouse</span><span class="source" tagx="source" title="source">Ultrastruct. Study Invest. Ophthalmol. Vis. Sci.</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="lpage" tagx="lpage" title="lpage">216</span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Williamson, K. A. et al. <i>Heterozygous Loss-of-function Mutations in YAP1 Cause Both Isolated and Syndromic Optic Fissure Closure Defects</i>. 10.1016/j.ajhg.2014.01.001 (2014).</span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKenna</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">1297</span><span class="lpage" tagx="lpage" title="lpage">1303</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.107524.110">10.1101/gr.107524.110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199">20644199</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expression profiling during ocular development identifies 2 Nlz genes with a critical role in optic fissure closure</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">1462</span><span class="lpage" tagx="lpage" title="lpage">1467</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0812017106">10.1073/pnas.0812017106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19171890">19171890</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westenskow</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piccolo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuhrmann</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">-catenin controls differentiation of the retinal pigment epithelium in the mouse optic cup by regulating Mitf and Otx2 expression</span><span class="source" tagx="source" title="source">Development</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">136</span><span class="fpage" tagx="fpage" title="fpage">2505</span><span class="lpage" tagx="lpage" title="lpage">2510</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/dev.032136">10.1242/dev.032136</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19553286">19553286</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fregeau</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">De novo mutations of RERE cause a genetic syndrome with features that overlap those associated with proximal 1p36 deletions</span><span class="source" tagx="source" title="source">Am. J. Hum. Genet.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">963</span><span class="lpage" tagx="lpage" title="lpage">970</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajhg.2016.03.002">10.1016/j.ajhg.2016.03.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27087320">27087320</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dauber</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SCRIB and PUF60 are primary drivers of the multisystemic phenotypes of the 8q24.3 copy-number variant</span><span class="source" tagx="source" title="source">Am. J. Hum. Genet.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">798</span><span class="lpage" tagx="lpage" title="lpage">811</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajhg.2013.09.010">10.1016/j.ajhg.2013.09.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24140112">24140112</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasioukhin</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuchs</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Directed actin polymerization is the driving force for epithelial cell–cell adhesion</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">219</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(00)81559-7">10.1016/S0092-8674(00)81559-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10660044">10660044</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moeller</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Protocadherin FAT1 binds Ena/VASP proteins and is necessary for actin dynamics and cell polarization</span><span class="source" tagx="source" title="source">EMBO J.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">3769</span><span class="lpage" tagx="lpage" title="lpage">3779</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.emboj.7600380">10.1038/sj.emboj.7600380</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15343270">15343270</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathak</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katrina</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hofstetter</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuntz</span><span class="given-names" tagx="given-names" title="given-names">Jonathon</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dylan Burnette</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cellular interactions during closure of the optic fissure in the embryonic mouse eye. in</span><span class="source" tagx="source" title="source">Invest. Ophthalmol. Vis. Sci.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">3005</span><span class="lpage" tagx="lpage" title="lpage">3005</span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reginensi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Yap- and Cdc42-dependent nephrogenesis and morphogenesis during mouse kidney development</span><span class="source" tagx="source" title="source">PLoS Genet.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e1003380</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1003380">10.1371/journal.pgen.1003380</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555292">23555292</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Assembly and activation of the Hippo signalome by FAT1 tumor suppressor</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">2372</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-04590-1">10.1038/s41467-018-04590-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29985391">29985391</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The sequence alignment/map format and SAMtools</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2078</span><span class="lpage" tagx="lpage" title="lpage">2079</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hakonarson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">e164</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkq603">10.1093/nar/gkq603</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20601685">20601685</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silberstein</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A system for exact and approximate genetic linkage analysis of SNP data in large pedigrees</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="lpage" tagx="lpage" title="lpage">205</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bts658">10.1093/bioinformatics/bts658</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23162081">23162081</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babcock</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">aldh7a1 regulates eye and limb development in Zebrafish</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e101782</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0101782">10.1371/journal.pone.0101782</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25004007">25004007</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nakayama, M. et al. Identification of high-molecular-weight proteins with multiple EGF-like motifs by motif-trap screening. <i>Genomics</i><b>51</b>, 27–34 (1998).</span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodgkinson</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mutations at the mouse microphthalmia locus are associated with defects in a gene encoding a novel basic-helix-loop-helix-zipper protein</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">74</span><span class="fpage" tagx="fpage" title="fpage">395</span><span class="lpage" tagx="lpage" title="lpage">404</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(93)90429-T">10.1016/0092-8674(93)90429-T</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8343963">8343963</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montague</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruz</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gagnon</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Church</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">CHOPCHOP: a CRISPR/Cas9 and TALEN web tool for genome editing</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">W401</span><span class="lpage" tagx="lpage" title="lpage">W407</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku410">10.1093/nar/gku410</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24861617">24861617</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garfield</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Derivation of primary mouse embryonic fibroblast (PMEF) cultures. in</span><span class="source" tagx="source" title="source">Methods Mol. Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">633</span><span class="fpage" tagx="fpage" title="fpage">19</span><span class="lpage" tagx="lpage" title="lpage">27</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-59745-019-5_2">10.1007/978-1-59745-019-5_2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20204617">20204617</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maminishkis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Confluent monolayers of cultured human fetal retinal pigment epithelium exhibit morphology and physiology of native tissue</span><span class="source" tagx="source" title="source">Investig. Opthalmol. Vis. Sci.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">3612</span><span class="pub-id"><a href="https://dx.doi.org/10.1167/iovs.05-1622">10.1167/iovs.05-1622</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May-Simera</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Primary cilium-mediated retinal pigment epithelium maturation is disrupted in ciliopathy patient cells</span><span class="source" tagx="source" title="source">Cell Rep.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">189</span><span class="lpage" tagx="lpage" title="lpage">205</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2017.12.038">10.1016/j.celrep.2017.12.038</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29298421">29298421</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogilvy</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abu-Asab</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Implications of DNA leakage in eyes of mutant mice</span><span class="source" tagx="source" title="source">Ultrastruct. Pathol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">335</span><span class="lpage" tagx="lpage" title="lpage">343</span><span class="pub-id"><a href="https://dx.doi.org/10.3109/01913123.2014.927406">10.3109/01913123.2014.927406</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24963552">24963552</a></span></span></li> </ul> </div> </div>  </body></html>