<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Oncogene</span><span class="iso-abbrev" title="iso-abbrev">Oncogene</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Oncogene</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0950-9232</span><span class="issn-epub" tagx="issn" title="issn-epub">1476-5594</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5031503">5031503</a></span><span class="pii" title="pii">onc201647</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/onc.2016.47">10.1038/onc.2016.47</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26996666">26996666</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Lamellipodin promotes invasive 3D cancer cell migration via regulated interactions with Ena/VASP and SCAR/WAVE</div> <div class="alt-title" title="alt-title">
Lamellipodin promotes breast cancer metastasis</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carmona</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><a href="#aff1">1</a><a href="#note1"><sup>11</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perera</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><a href="#aff2">2</a><a href="#note1"><sup>11</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gillett</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naba</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Law</span><span class="given-names" tagx="given-names" title="given-names">A-L</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">V P</span></span><a href="#aff4">4</a><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyckoff</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balsamo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosis</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Piano</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><a href="#aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monypenny</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><a href="#aff2">2</a><a href="#aff3">3</a><a href="#aff8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McConnell</span><span class="given-names" tagx="given-names" title="given-names">R E</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouneimne</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><a href="#aff9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Hemelrijck</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><a href="#aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condeelis</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><a href="#aff4">4</a><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hynes</span><span class="given-names" tagx="given-names" title="given-names">R O</span></span><a href="#aff1">1</a><a href="#aff10">10</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F B</span></span><a href="#aff1">1</a><a href="#caf1">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><a href="#aff2">2</a><a href="#caf2">*</a></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Koch Institute for Integrative Cancer Research at MIT, Massachusetts Institute of Technology</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">King's College London, Randall Division of Cell and Molecular Biophysics</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">King's College London, Research Oncology, Division of Cancer Studies, Faculty of Life Sciences and Medicine</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Department of Anatomy and Structural Biology, Albert Einstein College of Medicine</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="aff5">[5], <span class="institution" title="institution">Gruss Lipper Biophotonics Center, Albert Einstein College of Medicine</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="aff6">[6], <span class="institution" title="institution">Department of Microbiology, Tumor and Cell Biology, Karolinska Institute</span><span class="country" tagx="country" title="country">Sweden</span></span><span class="citation_author_institution" id="aff7">[7], <span class="institution" title="institution">King's College London, Division of Cancer Studies, Cancer Epidemiology Group</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="aff8">[8], <span class="institution" title="institution">King's College London, Division of Cancer Studies, Richard Dimbleby Department of Cancer Research</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="aff9">[9], <span class="institution" title="institution">University of Arizona Cancer Center,</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="aff10">[10], <span class="institution" title="institution">Howard Hughes Medical Institute, Massachusetts Institute of Technology</span><span class="country" tagx="country" title="country">USA</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><span class="institution" title="institution">Koch Institute for Integrative Cancer Research</span>, MIT, BLDG 76-361, 77 Massachusetts Avenue, Cambridge, 02139-4307 MA, <span class="country" tagx="country" title="country">USA</span> E-mail: <span class="email" tagx="email" title="email">fgertler@mit.edu</span></div> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><span class="institution" title="institution">King's College London, Randall Division of Cell and Molecular Biophysics</span>, New Hunt's House, Guy's Campus, London SE1 1UL, <span class="country" tagx="country" title="country">UK</span>. E-mail: <span class="email" tagx="email" title="email">Matthias.Krause@kcl.ac.uk</span></div> <div class="fn-type-equal" title="equal"><span class="label" tagx="label" title="label">11</span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2016-09-09</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-03-03</span></span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">39</span><span class="fpage" tagx="fpage" title="fpage">5155</span><span class="lpage" tagx="lpage" title="lpage">5169</span><span class="history" title="history"><span class="received" title="received">received: 2015-04-27</span><span class="rev-recd" title="rev-recd">rev-recd: 2016-01-20</span><span class="accepted" title="accepted">accepted: 2016-02-08</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Macmillan Publishers Limited</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span></span><span class="license" title="license"><span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a></span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Cancer invasion is a hallmark of metastasis. The mesenchymal mode of cancer cell invasion is mediated by elongated membrane protrusions driven by the assembly of branched F-actin networks. How deregulation of actin regulators promotes cancer cell invasion is still enigmatic. We report that increased expression and membrane localization of the actin regulator Lamellipodin correlate with reduced metastasis-free survival and poor prognosis in breast cancer patients. In agreement, we find that Lamellipodin depletion reduced lung metastasis in an orthotopic mouse breast cancer model. Invasive 3D cancer cell migration as well as invadopodia formation and matrix degradation was impaired upon Lamellipodin depletion. Mechanistically, we show that Lamellipodin promotes invasive 3D cancer cell migration via both actin-elongating Ena/VASP proteins and the Scar/WAVE complex, which stimulates actin branching. In contrast, Lamellipodin interaction with Scar/WAVE but not with Ena/VASP is required for random 2D cell migration. We identified a phosphorylation-dependent mechanism that regulates selective recruitment of these effectors to Lamellipodin: Abl-mediated Lamellipodin phosphorylation promotes its association with both Scar/WAVE and Ena/VASP, whereas Src-dependent phosphorylation enhances binding to Scar/WAVE but not to Ena/VASP. Through these selective, regulated interactions Lamellipodin mediates directional sensing of epidermal growth factor (EGF) gradients and invasive 3D migration of breast cancer cells. Our findings imply that increased Lamellipodin levels enhance Ena/VASP and Scar/WAVE activities at the plasma membrane to promote 3D invasion and metastasis.</p> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Breast cancer metastasis is one of the leading causes of cancer-associated mortality in women worldwide.<sup><a href="#bib1">1</a></sup> Metastasis is a multistep process.<sup><a href="#bib2">2</a></sup> After breaching, the basement membrane metastasizing cancer cells migrate through the dense extracellular matrix (ECM) of the tumor stroma in order to intravasate.<sup><a href="#bib2">2</a>, <a href="#bib3">3</a></sup> Carcinoma cells that migrate in a mesenchymal mode form elongated membrane protrusions driven by the assembly of branched F-actin networks. Actin polymerization-driven migration and invasion is coordinated by the proto-oncogenes c-Src and c-Abl kinases and cytoskeletal regulatory proteins including Rac GTPase, the Scar/WAVE complex and Ena/VASP proteins.<sup><a href="#bib4">4</a>, <a href="#bib5">5</a>, <a href="#bib6">6</a>, <a href="#bib7">7</a></sup></p> <p>Ena/VASP proteins (Mena, EVL and VASP) enhance processive filament elongation.<sup><a href="#bib8">8</a>, <a href="#bib9">9</a>, <a href="#bib10">10</a>, <a href="#bib11">11</a>, <a href="#bib12">12</a>, <a href="#bib13">13</a>, <a href="#bib14">14</a></sup> Mena is upregulated in breast cancer and promotes invasion.<sup><a href="#bib15">15</a>, <a href="#bib16">16</a></sup> We identified Lamellipodin (Lpd) as a binding partner of Ena/VASP proteins.<sup><a href="#bib5">5</a>, <a href="#bib17">17</a>, <a href="#bib18">18</a></sup> Lpd localizes to lamellipodia, thin membrane protrusions at the leading edge of migrating cells.<sup><a href="#bib17">17</a></sup> The Lpd-Ena/VASP interaction is positively regulated by Abl kinase-mediated Lpd phosphorylation, which drives Ena/VASP recruitment to lamellipodia by Lpd.<sup><a href="#bib19">19</a></sup></p> <p>Lpd is required for lamellipodium formation<sup><a href="#bib17">17</a></sup> and binds directly to the Scar/WAVE complex.<sup><a href="#bib20">20</a></sup> Scar/WAVE activates the Arp2/3 complex to nucleate branched actin networks during lamellipodia formation.<sup><a href="#bib4">4</a>, <a href="#bib5">5</a>, <a href="#bib6">6</a>, <a href="#bib7">7</a></sup> Surprisingly, Lpd-driven random cell migration in 2D requires Lpd binding to Scar/WAVE, but not to Ena/VASP.<sup><a href="#bib20">20</a></sup></p> <p>The mechanisms by which actin regulators coordinate the interplay between actin-elongation and actin-branching factors to promote cancer cell invasion remain incompletely understood. Here, we report that Lamellipodin mediates invasive 3D migration of cancer cells via selective, regulated interactions with Ena/VASP and Scar/WAVE. Our findings point to key roles for increased Lpd levels in breast cancer invasion and metastasis.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>We observed higher Lpd levels in invasive and metastatic basal cell lines compared with noninvasive, luminal tumor cell lines (<a href="#fig1">Figure 1a</a>). Therefore, we analyzed publicly available data sets<sup><a href="#bib21">21</a></sup> to examine whether Lpd mRNA levels correlated with occurrence of distant metastases in breast cancer patients. Lpd was overexpressed in several types of breast tumors compared with matched healthy tissue (<a href="#sup1">Supplementary Figure 1A</a>). High levels of Lpd mRNA correlated with reduced metastasis-free and disease-free survival of breast cancer patients in three separate cohorts (<a href="#fig1">Figures 1b and c</a>; <a href="#sup1">Supplementary Figures 1B and C</a>).<sup><a href="#bib22">22</a>, <a href="#bib23">23</a>, <a href="#bib24">24</a></sup> In addition, we explored whether Lpd protein expression levels correlate with clinical outcome for breast cancer patients by staining a tumor microarray (TMA)<sup><a href="#bib25">25</a></sup> generated from 312 patients with invasive breast cancer with anti-Lpd antibodies. Moderately, but not highly, increased abundance of Lpd in the cytoplasm (Histoscore 2; Hazard ratio (HR) (95% confidence interval (CI)): 1.765 (1.026–3.036); <a href="#sup1">Supplementary Figures 1D and 2A,B</a>) and at the plasma membrane (Histoscore 2: HR, (95% CI): 2.231 (1.26–3.949); <a href="#fig1">Figures 1d and e</a>; compared with respective histoscore 1) was significantly associated with increased risk for breast cancer-associated mortality. Furthermore, we observed an inverse correlation between Lpd intensity at the plasma membrane and Her2 expression (<a href="#sup1">Supplementary Figure 1E</a>). Consistent with Lpd's predominant role at the plasma membrane in promoting cell motility and migration,<sup><a href="#bib17">17</a>, <a href="#bib19">19</a>, <a href="#bib20">20</a></sup> we observed a significant association between highly, but not moderately, increased Lpd staining intensity at the plasma membrane and reduced disease-free (Histoscore 3: HR (95% CI): 1.652 (1.24–2.428)) and metastasis-free survival of breast cancer patients (Histoscore 3: HR (95% CI): 1.515 (1.054–2.178); <a href="#fig1">Figure 1e</a> compared with respective histoscore 1).</p> <p>To investigate the requirement for Lpd in metastasis, we tested the effect of reducing Lpd expression in MDA-MB-231-LM2 cells (further referred to as LM2), a highly metastatic derivative of MDA-MB-231 breast cancer cells,<sup><a href="#bib26">26</a></sup> on their ability to metastasize from an orthotopic mammary tumor to the lungs. We generated stable LM2 cell lines with two independent Lpd-targeting shRNAs or a non-targeting control shRNA, all three retroviruses also conferring the cis-linked marker ZsGreen (<a href="#sup1">Supplementary Figure 2C</a>). Lpd knockdown and control cells were injected orthotopically into the mammary fat pad of immunodeficient mice. Primary tumors formed 6 weeks after injection from Lpd-deficient cells were similar in size to those arising from control LM2 cells, suggesting that the loss of Lpd did not affect primary tumor growth (<a href="#fig2">Figure 2a</a>). Importantly, only 3 out of 20 mice bearing Lpd-depleted tumors developed macroscopic lung metastases, compared with 9 out of 10 control tumor-bearing mice (<a href="#fig2">Figure 2b</a>). In addition, animals with tumors generated from Lpd-depleted cells that metastasized displayed significantly reduced numbers of pulmonary ZsGreen-positive metastases compared with the metastatic burden of animals with control tumors (<a href="#fig2">Figures 2c and d</a>).</p> <p>We then explored whether Lpd promotes cancer cell invasion during metastasis. In fixed samples, tumors generated from control shRNA LM2 cells prominently invaded the surrounding stroma. However, tumors from Lpd knockdown cells were markedly less invasive (<a href="#fig2">Figure 2e</a>). We investigated Lpd function in cancer invasion in more detail by intravital imaging. Compared with control shRNA LM2 tumors, Lpd knockdown tumors had fewer motile cells, which migrated less directionally and extended protrusions less frequently (<a href="#fig2">Figures 2f–i</a>; <a href="#sup1">Supplementary Video 1</a>), indicating that Lpd is required for invasive cancer cell phenotypes <i>in vivo</i>.</p> <p>To examine Lpd-driven breast cancer intravasation and dissemination, we implanted fluorescently labeled MDA-MB-231 breast cancer cells into the perivitelline cavities of zebrafish embryos. In this assay, the injected cancer cells intravasate, and then infiltrate the trunk of the fish.<sup><a href="#bib27">27</a>, <a href="#bib28">28</a></sup> Overexpression of GFP-Lpd in MDA-MB-231 cells enhanced the frequency of seeding of these breast cancer cells compared with GFP-expressing control cells (<a href="#fig2">Figures 2j and k</a>). We then performed tail vein injections of the Lpd knockdown and control LM2 cell lines into immunocompromised mice and quantified lung metastases after 28 days to test whether Lpd influenced the later stages of the metastatic cascade. Lpd depletion did not reduce the number of metastatic foci in the lungs of the mice compared with controls (<a href="#sup1">Supplementary Figures 2D and E</a>). Taken together, our results reveal that Lpd promotes local tumor invasion, intravasation and metastasis <i>in vivo,</i> but is not required for extravasation.</p> <p>Breast cancer cell migration toward blood vessels is guided by cues from the tumor microenvironment, such as epidermal growth factor (EGF).<sup><a href="#bib29">29</a></sup> We reasoned that the effect of Lpd depletion on EGF-induced 3D invasion might arise from defects in lamellipodial dynamics. Depletion of Lpd in MDA-MB-231 breast cancer cells decreased lamellipodia size (<a href="#sup1">Supplementary Figure 3A</a>), similar to B16-F1 mouse melanoma cells, in which Lpd depletion also reduces protrusion speed under steady-state conditions.<sup><a href="#bib17">17</a></sup> EGF-stimulated MDA-MB-231 Lpd knockdown cells displayed reduced protrusion persistence and distance, without affecting protrusion speed (<a href="#sup1">Supplementary Figure 3B</a>). We chose MTLn3 cells, a mammary adenocarcinoma cancer cell line, in which protrusion responses to EGF have been extensively characterized, to examine EGF-elicited protrusion in more detail as lamellipodial size is least affected by Lpd knockdown in this cell line (<a href="#sup1">Supplementary Figure 3A</a>). In agreement with our findings in MDA-MB-231 cells, in EGF-stimulated MTLn3 cells reduced Lpd levels significantly decreased protrusion persistence and distance (<a href="#fig3">Figures 3a and b</a>) but did not affect protrusion speed (<a href="#sup1">Supplementary Figure 3D</a>), compared with controls. Lpd was diffusely distributed throughout the cytoplasm of serum-starved cells, but was rapidly recruited to the cell edge following bath application of EGF (<a href="#fig3">Figures 3c and d</a>). Lamellipodial initiation was detected 30 s after EGF stimulation in Ctrl-shRNA-expressing cells but was delayed significantly when Lpd levels were reduced (<a href="#fig3">Figures 3b, e and f</a>; <a href="#sup1">Supplementary Figure 3E</a>; <a href="#sup1">Supplementary Video 3</a>). Taken together, our data suggest that, in breast cancer cells, Lpd depletion reduces EGF-elicited lamellipodial protrusion formation and persistence, but not speed.</p> <p>Membrane extension during lamellipodial protrusion is driven by actin polymerization.<sup><a href="#bib4">4</a>, <a href="#bib5">5</a>, <a href="#bib6">6</a></sup> To determine how Lpd depletion influences actin polymerization, we used a G-actin incorporation assay<sup><a href="#bib30">30</a></sup> to measure the abundance and distribution of polymerization-competent, free (uncapped) F-actin barbed-ends in lamellipodia of living cells. Silencing of Lpd significantly reduced free barbed-end formation 1 min after EGF stimulation, relative to Ctrl-shRNA-expressing cells (<a href="#fig4">Figures 4a and b</a>). Collectively, these data indicate that Lpd promotes lamellipodial protrusion by increasing actin polymerization downstream of EGFR activation.</p> <p>EGF-dependent membrane protrusion in MTLn3 cells requires Ena/VASP proteins and Arp2/3-mediated dendritic nucleation;<sup><a href="#bib16">16</a>, <a href="#bib31">31</a></sup> and Lpd binds both Ena/VASP proteins and the Arp2/3 activating Scar/WAVE complex.<sup><a href="#bib17">17</a>, <a href="#bib19">19</a>, <a href="#bib20">20</a></sup> Consistent with this, membrane recruitment of Mena (<a href="#fig4">Figures 4c and d</a>) and Arp2/3 complex to the protruding edge (<a href="#fig4">Figures 4e and f</a>) was significantly reduced in Lpd-depleted cells after EGF stimulation.</p> <p>We hypothesized that the requirement for Lpd in EGF-induced protrusion reflects Lpd-mediated initiation of chemotactic responses. The initial step of chemotaxis is directional sensing.<sup><a href="#bib32">32</a>, <a href="#bib33">33</a></sup> We used a micropipette to generate a spatially restricted EGF gradient. Ctrl-shRNA MTLn3 cells formed new protrusions toward the pipette, demonstrating their ability to sense the EGF gradient. However, those of Lpd-reduced cells were random relative to the micropipette (<a href="#fig4">Figures 4g and h</a>; <a href="#sup1">Supplementary Figures 4A and B</a>; <a href="#sup1">Supplementary Video 5</a>). Chemotactic indices confirmed the lack of directional bias in Lpd-deficient MTLn3 protrusions (<a href="#fig4">Figure 4h</a>), highlighting an essential role for Lpd in the initial steps of chemotaxis toward EGF.</p> <p>Lpd is concentrated at the edges of lamellipodia that protrude in response to uniform EGF stimulation. Correlative differential interference contrast microscopy and immunofluorescence imaging in live cells revealed enrichment of Lpd at the edges of cells oriented toward the micropipettes containing EGF (<a href="#sup1">Supplementary Figure 4C</a>), confirming that Lpd was enriched in membranes exposed to the highest concentration of the EGF gradient and supporting a function for Lpd in linking gradient sensing to directed membrane protrusion.</p> <p>We hypothesized that EGF chemosensing might involve Lpd-mediated recruitment of Ena/VASP proteins. We tested whether Lpd-depleted cells, expressing either a GFP-Lpd or an Lpd mutant in which all Ena/VASP binding sites had been rendered non-functional by mutation (GFP-Lpd<sup>EVmut</sup>), could respond to EGF gradients in the micropipette assay and discovered that, while GFP-Lpd effectively rescued the chemosensing defects in Lpd-depleted cells, GFP-Lpd<sup>EVmut</sup> conferred no significant phenotypic rescue (<a href="#fig4">Figure 4i</a> and <a href="#sup1">Supplementary Figure 5C</a>). In line with this finding, a function-perturbing approach<sup><a href="#bib34">34</a></sup> revealed that Ena/VASP proteins were required for chemosensing (<a href="#sup1">Supplementary Figures 5A and B</a>). Thus, extension of lamellipodia toward EGF during chemosensing by breast carcinoma cells requires the Lpd-dependent recruitment of Ena/VASP proteins, despite the dispensability of Ena/VASP for Lpd-driven random 2D cell migration.</p> <p>We next analyzed the requirements for Lpd during EGF-dependent 3D invasion. In 3D inverted chemotaxis assays toward EGF with MDA-MB-231 or SUM-159 invasive breast cancer lines<sup><a href="#bib35">35</a>, <a href="#bib36">36</a></sup> knockdown of Lpd significantly decreased invasion through matrigel (<a href="#fig5">Figures 5a and b</a>; <a href="#sup1">Supplementary Figure 6A</a>) and collagen (<a href="#sup1">Supplementary Figures 6B–D</a>) compared with control shRNA expressing cells. Conversely, Lpd overexpression significantly increased invasion toward EGF (<a href="#fig5">Figures 5d and e</a>).</p> <p>As invasion is known to be partially dependent on matrix metalloproteinase (MMP) digestion of ECM, we tested whether Lpd increases invasion via MMP-dependent or migration-dependent mechanisms. MMP inhibitor treatment of GFP-expressing control cells reduced invasion, as expected. Similarly, MMP inhibitor treatment reduced invasion of Lpd overexpressing MDA-MB-231 cells. However, Lpd overexpressing cells treated with MMP inhibitors invaded significantly further compared with GFP control cells treated with the inhibitor, suggesting that Lpd functions to increase invasion by increasing migration and potentially MMP-dependent ECM degradation (<a href="#fig5">Figure 5c</a> and <a href="#sup1">Supplementary Figure 6E</a>).</p> <p>The aforementioned findings prompted us to investigate Lpd's role in MMP-dependent degradation. Carcinoma cells can utilize protrusive invadopodia, sites of MMP exocytosis, to invade through the basement membrane and ECM-rich interstitial stroma.<sup><a href="#bib37">37</a>, <a href="#bib38">38</a>, <a href="#bib39">39</a></sup> We observed that Lpd co-localized with the invadopodial marker cortactin at invadopodia, at sites of matrix degradation (<a href="#fig5">Figure 5f</a>). Previously, we reported that Mena promotes invadopodium stabilization and matrix degradation,<sup><a href="#bib16">16</a></sup> and here we observed that Lpd depletion appeared to reduce the amount of Mena within invadopodia considerably, potentially reflecting a role for Lpd in Mena recruitment to invadopodia (<a href="#sup1">Supplementary Figure 6H</a>). Silencing of Lpd in MDA-MB-231 cells decreased the number of precursors, mature invadopodia and the total number of invadopodia in comparison with control cells (<a href="#fig5">Figures 5i and j</a>). Furthermore, Lpd-depleted MDA-MB-231 cells exhibited a significant decrease in their ability to degrade matrix relative to control cells (<a href="#fig5">Figures 5g and h</a>). Taken together, these results suggest that Lpd is required for invadopodial precursor formation or for both precursor formation and subsequent stabilization/maturation, and for invadopodia-mediated matrix degradation.</p> <p>To test the relative contribution of Lpd interactions with Ena/VASP proteins or with the Scar/WAVE complex during MDA-MB-231 3D invasion, we overexpressed a panel of Lpd mutants in which all Ena/VASP (GFP-Lpd<sup>EVmut</sup>), all Scar/WAVE-binding sites (GFP-Lpd<sup>S/Wmut</sup>), or all Ena/VASP and all Scar/WAVE-binding sites (GFP-Lpd<sup>EV+S/Wmut</sup>) had been mutated. All of these mutants localized to the leading edge of MDA-MB-231 cells (<a href="#sup1">Supplementary Figures 6F and G</a>). At steady state when embedded in 3D matrigel, MDA-MB-231 cells overexpressing GFP-Lpd displayed significantly more protrusions, which protruded faster compared with GFP control cells. However, expression of Lpd-Ena/VASP-, Lpd-Scar/WAVE- and double-binding mutants all failed to increase protrusion numbers and speed (Figure 7h and <a href="#sup1">Supplementary Figure 7G</a>). Similarly, these mutants did not support invasion through matrigel toward EGF, suggesting that Lpd promotes 3D chemotactic invasion via both Ena/VASP and Scar/WAVE (<a href="#fig5">Figures 5d and e</a>). These findings, combined with our previous observation that Lpd interaction with Scar/WAVE but not with Ena/VASP is required for random 2D migration, prompted us to hypothesize that interactions between Lpd and these actin regulators may be differentially regulated.</p> <p>Previously, we found that Lpd is phosphorylated by Abl kinases upon growth factor stimulation, and this positively regulates its interaction with Ena/VASP proteins and their recruitment to the leading edge of cells.<sup><a href="#bib19">19</a></sup> As Src kinases are also activated upon growth factor stimulation and increased Src activity promotes cancer cell invasion,<sup><a href="#bib40">40</a></sup> we explored whether Lpd interactions with downstream partners are regulated by Src phosphorylation.</p> <p>We expressed GFP-Lpd with wild-type c-Src or a kinase-inactive mutant of c-Src and, after immunoprecipitation of Lpd, observed that it was tyrosine phosphorylated in cells expressing wild-type but not kinase-inactive c-Src (<a href="#fig6">Figure 6a</a>). Both Src and Abl kinases are activated downstream of growth factor receptors, and Src phosphorylation of Abl kinases contributes to their activation.<sup><a href="#bib40">40</a>, <a href="#bib41">41</a>, <a href="#bib42">42</a></sup> To distinguish between Src and Abl phosphorylation of Lpd and to investigate whether endogenous Lpd is phosphorylated by Src tyrosine kinases, we made use of Abl/Arg double knockout mouse embryonic fibroblasts.<sup><a href="#bib43">43</a></sup> After PDGF-BB stimulation, Lpd was robustly tyrosine phosphorylated in the absence of both Abl kinases, Abl and Arg. This was blocked by the Src inhibitor Bosutinib (<a href="#fig6">Figures 6b and c</a>), indicating that Src kinase activity leads to Lpd phosphorylation upon growth factor receptor activation.</p> <p>We first investigated whether Src phosphorylation controls Lpd-Ena/VASP interaction, since this is positively regulated by Abl phosphorylation.<sup><a href="#bib19">19</a></sup> Surprisingly, c-Src-induced Lpd phosphorylation did not affect Lpd-Ena/VASP binding (<a href="#sup1">Supplementary Figures 7A and B</a>). In contrast, co-immunoprecipitation between GFP-Lpd and Myc-tagged Scar/WAVE complex revealed that significantly more Scar/WAVE complex co-precipitated with Lpd when either c-Src or c-Abl was co-expressed (<a href="#fig6">Figures 6d–g</a>). We also tested whether the interaction between endogenous Lpd and Scar/WAVE is positively regulated by c-Src by using ectopically expressed GST-tagged Abi (which reduces endogenous Abi and thereby replaces it<sup><a href="#bib44">44</a></sup>) to efficiently pull down the Scar/WAVE complex and associated proteins. GST-Abi pulldowns from cells co-expressing GFP-Src contained higher levels of endogenous Scar/WAVE2 co-precipitating with endogenous Lpd compared with GFP controls (<a href="#sup1">Supplementary Figures 7C and D</a>). Conversely, GST-Abi pulldowns from cells co-expressing GFP-Src treated with the c-Src inhibitor (KB SRC 4) contained lower levels of endogenous Scar/WAVE2 co-precipitating with endogenous Lpd compared with GST-Abi pulldowns from DMSO-treated control cells (<a href="#sup1">Supplementary Figures 7E and F</a>). Together, these findings indicate that the Lpd-Ena/VASP interaction is regulated by c-Abl phosphorylation, whereas the Lpd-Scar/WAVE interaction is positively regulated by both c-Abl and c-Src-dependent phosphorylation.</p> <p>To better understand this differential regulation, we identified potential direct c-Src phosphorylation sites in Lpd, using purified c-Src kinase to phosphorylate an immobilized peptide array covering all putative tyrosine phosphorylation sites in Lpd. This analysis revealed that Lpd harbors two robustly and four weakly phosphorylated c-Src tyrosine phosphorylation sites (<a href="#fig7">Figure 7a</a>). We have previously mapped eight c-Abl tyrosine phosphorylation sites in Lpd,<sup><a href="#bib19">19</a></sup> which partly overlap with these newly identified c-Src sites (<a href="#fig7">Figure 7a</a>). To verify that the Lpd phosphorylation sites identified <i>in vitro</i> were phosphorylated in cells, we mutated the tyrosines in the six c-Src (GFP-Lpd<sup>Y6F</sup>) and the eight c-Abl phosphorylation sites (GFP-Lpd<sup>Y8F</sup>) to phenylalanine, rendering them non-phosphorylatable. Overexpression of GFP-Lpd induced low levels of Lpd tyrosine phosphorylation, which was markedly enhanced by stimulation with 100 ng/ml EGF for 5 min. In contrast, neither GFP-Lpd<sup>Y6F</sup> nor GFP-Lpd<sup>Y8F</sup> tyrosine phosphorylation was enhanced when cells expressing these constructs were stimulated with EGF (<a href="#fig7">Figures 7b and c</a>). We found that the GFP-Lpd<sup>Y6F</sup> and GFP-Lpd<sup>Y8F</sup> mutants interacted significantly less with the Scar/WAVE complex (<a href="#fig7">Figures 7f and g</a>; <a href="#sup1">Supplementary Figure 8D</a>). Nevertheless, GFP-Lpd<sup>Y6F</sup> and -Lpd<sup>Y8F</sup> mutants localized to the leading edge of MDA-MB-231 cells similar to wild-type Lpd (<a href="#sup1">Supplementary Figures 6F and G</a>). Taken together, our data indicate that full-length Lpd can be phosphorylated at these sites in cells upon EGFR activation, likely as a consequence of activated Abl and Src kinases.</p> <p>To verify that both Src and Abl kinases are required for breast cancer invasion toward EGF, we tested the invasiveness of MDA-MB-231 cells with and without incubation with the Src and Abl inhibitors Dasatinib or STI571. As expected, we found that breast cancer cell invasiveness is impaired when Src or Abl kinases were inhibited (<a href="#sup1">Supplementary Figures 8A and B</a>).</p> <p>To investigate the functional significance of Lpd tyrosine phosphorylation for breast cancer invasion, we compared the effects of overexpressing the non-phosphorylatable mutants, GFP-Lpd<sup>Y6F</sup> or GFP-Lpd<sup>Y8F</sup> with control GFP-Lpd. We observed that, in contrast to GFP-Lpd, neither of the mutants increased protrusion numbers or speed in cells embedded in matrigel at steady state, or the invasiveness of breast cancer cells through matrigel toward EGF (<a href="#fig7">Figures 7d, e and h</a> and <a href="#sup1">Supplementary Figure 8C</a>), suggesting that phosphorylation by both Abl and Src kinases is required for Lpd-mediated breast cancer invasion.</p> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Here, we reveal that Lpd is required for metastasis in an orthotopic breast cancer mouse model, and that increased Lpd levels correlate with reduced metastasis-free survival in breast cancer patients. Lpd promotes metastasis <i>in vivo</i> by supporting tumor invasion and intravasation. Lpd function in metastasis may be mediated via both Ena/VASP and the Scar/WAVE complex since we observed that Lpd mediates breast cancer invasion via both actin effectors. Both Ena/VASP and Scar/WAVE are implicated in breast cancer metastasis by multiple lines of evidence.<sup><a href="#bib15">15</a>, <a href="#bib16">16</a>, <a href="#bib45">45</a>, <a href="#bib46">46</a></sup> Our results suggest that the pro-metastatic function of Lpd may, in part, involve coordinating the activities of these two distinct types of actin regulators to optimize chemotactic invasion and matrix degradation by invading tumor cells.</p> <p>Here, we provide evidence that Lpd is a substrate of Src kinases and that phosphorylation of Lpd by Src positively regulates the Lpd-Scar/WAVE complex interaction, but not the Lpd-Ena/VASP interaction, whereas c-Abl-mediated phosphorylation of Lpd positively regulates both Lpd-Ena/VASP and Lpd-Scar/WAVE interaction. This differential regulation of Lpd-Ena/VASP or Lpd-Scar/WAVE recruitment may allow Lpd to fine-tune actin cytoskeletal dynamics via Ena/VASP-mediated actin filament elongation and Scar/WAVE-Arp2/3-mediated nucleation/branching, consistent with the context-dependent differential requirements for these interactions we observed. Lpd-driven random 2D cell migration requires Scar/WAVE, but not Ena/VASP,<sup><a href="#bib20">20</a></sup> Lpd-dependent chemosensing in 2D requires Ena/VASP, and interactions with both SCAR/WAVE and Ena/VASP are required for 3D chemotaxis and migration. These findings lead to the interesting possibility that Lpd balances actin nucleation/branching and filament elongation activities to optimize protrusion morphology and dynamics during cellular responses to growth factors and ECM composition and organization.</p> <p>Surprisingly, we observed that only moderately, but not highly increased levels of Lpd correlate with increased risk of breast cancer-associated mortality, suggesting that not all of Lpd functions (cell invasion, cell proliferation and EGFR endocytosis)<sup><a href="#bib47">47</a>, <a href="#bib48">48</a></sup> induced by high but not medium levels of Lpd may be beneficial for tumors cells. However, in agreement with its role in metastasis, we found that highly increased Lpd abundance at the plasma membrane of cancer cells in breast tumors correlates with reduced disease- and metastasis-free interval. The increased membrane accumulation of Lpd protein observed in our TMA analysis may reflect the known role of Lpd in regulating membrane protrusion in migrating cells.<sup><a href="#bib17">17</a>, <a href="#bib20">20</a></sup> Furthermore, consistent with the TMA analysis, we found that increased Lpd mRNA levels correlate with reduced metastasis-free survival of breast cancer patients.</p> <p>On the basis of our findings, we propose that Lpd functions as an essential component of a pro-metastatic signaling pathway composed of Src and Abl kinases, Lpd, Ena/VASP and Scar/WAVE that promotes metastatic progression.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="plasmidsandshrnas" title="sec"> <div class="title" tagx="title" title="title">
Plasmids and shRNAs</div> <p>GFP-VASP, pMSCV-mRFP1-FP4/AP4-mito,<sup><a href="#bib34">34</a></sup> Myc-Pir121, -Nap1, -Abi1d, -WAVE2 in pRK5-Myc-DEST<sup><a href="#bib20">20</a></sup> and Lpd<sup><a href="#bib17">17</a></sup> in pENTR (Invitrogen) mutated using Quikchange (Agilent), transferred into pCAG-DEST-EGFP (Gateway) (pCAG-EGFP; C Cepko, Addgene,11150) and pCB6-Src-WT-EGFP, pCB6-Src-KI (kinase-inactive)-EGFP<sup><a href="#bib49">49</a></sup> (M Way).</p> <p>GFP-Lpd<sup>S/Wmut</sup>: <sup><a href="#bib20">20</a></sup>(AAS82582.1) Site 1: (aa 968–978) GKKP(P&amp;gt;A) (P&amp;gt;A) T(P&amp;gt;A)Q(R&amp;gt;A)N; Site 2: (aa 1119–1129) APP(P&amp;gt;A)TR(P&amp;gt;A)K(R&amp;gt;A)ND; Site 3: (aa 1230–1244) RRGP(P&amp;gt;A)A(P&amp;gt;A)(P&amp;gt;A)(K&amp;gt;A)(R&amp;gt;A)DQNT.</p> <p>GFP-Lpd<sup>EVmut</sup>: <sup><a href="#bib20">20</a></sup>(AAS82582.1) FP4-1: aa 869 F&amp;gt;A; FP4-2: aa 916 F&amp;gt;A; FP4-3: aa927 F&amp;gt;A; FP4-4: aa 1064 F&amp;gt;A; FP4-5: aa1073 F&amp;gt;A; FP4-6: aa1082 L&amp;gt;A; FP4-7: aa1202 L&amp;gt;A.</p> <p>GFP-Lpd<sup>Y4F</sup>: <sup><a href="#bib19">19</a></sup>(AAS82582.1) Y&amp;gt;F: aa 426, 456, 513, 1226; GFP-Lpd<sup>Y6F</sup>: (AAS82582.1) Y&amp;gt;F: aa 366, 426, 456, 466, 481, 510; GFP-Lpd<sup>Y8F</sup>: (AAS82582.1) Y&amp;gt;F: aa 366, 426, 456, 466, 481, 510, 513, 1226.</p> <p>pLJM1-H2BK-mCherry: histone H2BK amplified from A431 cell by RT–PCR cloned into pLJM1-mCherry (D Sabatini; Addgene plasmid #19319).</p> <p>shRNAs were cloned into pLL3.7-Puro, miR30-MLS EGFP/mCherry<sup><a href="#bib50">50</a></sup> or MSCV-ZsG-2A-Puro-miR30 retroviral vector.<sup><a href="#bib51">51</a></sup> Cells were FACS sorted or puromycin (Invitrogen) selected and knockdown tested by western blotting. shRNA (5′–3′) used:</p> <p>Human Lpd-shRNA1: TTTCCCCAAAAGATAATTCTG, humanLpd-shRNA2: TTCCCATACTTTGCAATGCGG, ratLpd-shRNA2: TAGAGCTCACAGTACTTTGGG, ratLpd-shRNA3: AAGAGGTCCAATCATAAGCTG and Control-shRNA (targeting luciferase): TTAATTAAAGACTTCAAGCGG (<a href="#fig2">Figures 2</a>, <a href="#fig3">3</a>, <a href="#fig4">4</a> and <a href="#sup1">Supplementary Figures S2–S6</a>). Human Lpd-shRNA-1:GCGTCAAATCACAGAAACG, Human Lpd-shRNA-2:GCTCTGAATCAGGGAGAGA and Control-shRNA (Lpd-scrambled): GCCGATAACCGAGAATACC (<a href="#fig2">Figures 2</a> and <a href="#fig5">5</a>; <a href="#sup1">Supplementary Figures S2 and S6</a>).</p> </div> <div class="cellcultureandtransfection" title="sec"> <div class="title" tagx="title" title="title">
Cell culture and transfection</div> <p>HEK293, BT549, MCF7 and T47D cells obtained from ATCC and maintained according to ATCC's protocol. SUM-159 and MTLn3 cells cultured as described.<sup><a href="#bib52">52</a>, <a href="#bib53">53</a></sup> HEK cells expressing GST-Abi2 and GFP-Src were treated for 1 h with 100 μ <div class="sc_UNKNOWN">
m</div> KB SRC 4 (Tocris) before lysis. Abl1/Abl2 double knockout MEFs (gift of T Koleske, Yale), MDA-MB-231 cells (gift of A Ridley) and MDA-MB-231 LM2 cells<sup><a href="#bib26">26</a></sup> (gift from J Massague, MSKCC) were cultured in high-glucose DMEM, penicillin, streptomycin, 10% FBS. MTLn3 and MDA-MB-231 transfection was done using Lipofectamine 2000 (Invitrogen). Abl1/2DKO MEFs were serum starved (18 h) and treated with 10 μ <div class="sc_UNKNOWN">
m</div> Bosutinib (Cambridge Biosciences) (2 h) in DMSO or DMSO (control), then stimulated with 20 ng/ml PDGF-BB (2 min).</p> </div> <div class="antibodies" title="sec"> <div class="title" tagx="title" title="title">
Antibodies</div> <p>Lpd pab 3917,<sup><a href="#bib17">17</a></sup> Mena mab A351F7D9,<sup><a href="#bib54">54</a></sup> Wave2 (Cell Signaling Technology), p34Arc (Millipore, 07-227), Tubulin (DM1A), Hsc70 mab (Santa Cruz), GFP mab (Roche), Myc mab (Sigma, 9E10), pTyr mab (Millipore, 4G10) and Vimentin (550513, Biosciences). Alexa-conjugated secondary antibodies, phalloidin (Invitrogen, Biotium) diluted 1:50–1000.</p> </div> <div class="immunoprecipitation,gst-pulldownsandwesternblotting" title="sec"> <div class="title" tagx="title" title="title">
Immunoprecipitation, GST-pulldowns and western blotting</div> <p>Immunoprecipitation, GST-pulldowns and western blotting were performed as described.<sup><a href="#bib20">20</a></sup></p> </div> <div class="peptidearrayassay" title="sec"> <div class="title" tagx="title" title="title">
Peptide array assay</div> <p>Peptide array assay was done using Src kinase (NEB) as described.<sup><a href="#bib19">19</a></sup></p> </div> <div class="immunofluorescencemicroscopy" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence microscopy</div> <p>Cells were plated on glass coverslips, fixed and stained as described.<sup><a href="#bib19">19</a></sup> Imaged using Deltavision microscope (Applied Precision, Olympus IX71, × 60/1.4NA, Softworx software) (SGI, Mountain View, CA, USA). Olympus IX-81 microscope (Metamorph, Photometrics CascadeII 512B camera, × 40 UPlanFL, × 60 PlanApoNA1.45 or × 100 UPlanApoS NA1.4 objectives) was used. Leading edge localization (<a href="#sup1">Supplementary Figure 6G</a>) was quantified by two blinded observers from independent data sets (<i>n</i>=32–38 cells for each mutant).</p> </div> <div class="invertedinvasionassay" title="sec"> <div class="title" tagx="title" title="title">
Inverted invasion assay</div> <p>In all, 5 × 10<sup>5</sup> MDA-MB-231 cells stably expressing mCherry-H2B were seeded onto underside of 8 μm pore-size transwell inserts (Greiner Bio-One Ltd) containing matrigel (BD Biosciences, UK; polymerized 30 min (37 °C)). Inserts were inverted after cells adhered (4 h), placed in serum-free medium, and normal growth medium containing EGF (25 ng/ml) placed on top. Inhibitors used were GM6001 μ <div class="sc_UNKNOWN">
m</div>; STI571 10 μ <div class="sc_UNKNOWN">
m</div> and Dasatinib 10 n <div class="sc_UNKNOWN">
m</div>. Seventy-two hours later, cells that did not cross the transwell filter were removed; invading cells visualized by confocal microscopy; 2.5 μm sections. The number of nuclei of invading cells above 40 or 80 μm was automatically quantified using the Volocity software.</p> </div> <div class="membraneprotrusionassays" title="sec"> <div class="title" tagx="title" title="title">
Membrane protrusion assays</div> <p>Membrane protrusion assay for EGF-treated cells was performed as described.<sup><a href="#bib16">16</a></sup> Kymographic analysis was performed to analyze the protrusion parameters including: protrusion persistence, distance, velocity and protrusion initiation after EGF stimulation.</p> <p>For membrane protrusion assay in 3D matrigel, MDA-MB-231 cells were stained with CellTracker Green dye (ThermoFisher, UK) embedded in matrigel (BD Biosciences) in μ-slide chambers (81506; Ibidi, Germany). Four hours after plating, 5 min movies, one frame every 15 s; × 40; Olympus IX-81 were generated and protrusive activity around the entire circumference between frames automatically quantified from thresholded movies using ImageJ plugin ADAPT.<sup><a href="#bib55">55</a></sup></p> </div> <div class="micropipetteassay" title="sec"> <div class="title" tagx="title" title="title">
Micropipette assay</div> <p>The micropipette assay was performed as described.<sup><a href="#bib53">53</a>, <a href="#bib56">56</a></sup></p> </div> <div class="barbed-endassay" title="sec"> <div class="title" tagx="title" title="title">
Barbed-end assay</div> <p>G-actin was extracted from rabbit muscle acetone powder and standard techniques used to gel-filter over a Superdex-200 gel filtration column. The G-actin was polymerized to F-actin in F-actin buffer (1 m <div class="sc_UNKNOWN">
m</div> ATP, 5 m <div class="sc_UNKNOWN">
m</div> MgCl<sub>2</sub>, 50 m <div class="sc_UNKNOWN">
m</div> KCl, 50 m <div class="sc_UNKNOWN">
m</div> Tris/HCl, pH 8.0), labeled with Rhodamine-X succinimidyl ester (Invitrogen; following manufacturer's instructions), depolymerized in G-actin buffer (0.2 m <div class="sc_UNKNOWN">
m</div> ATP, 0.5 m <div class="sc_UNKNOWN">
m</div> DTT, 0.2 m <div class="sc_UNKNOWN">
m</div> CaCl<sub>2</sub>, 2 m <div class="sc_UNKNOWN">
m</div> Tris/HCl, pH 8.0) to G-actin, and passed through PD-10 columns (GE Healthcare) to eliminate free rhodamine. The barbed-end assay in MTLn3 cells was performed as described.<sup><a href="#bib16">16</a></sup> Images taken with deconvolution microscope; the ratio of the barbed-end intensity to phalloidin intensity along the edge (the zone between 0 and 0.66 μm inside the cell) quantified as described above.</p> </div> <div class="invadopodiumdegradationandimmunofluorescence" title="sec"> <div class="title" tagx="title" title="title">
Invadopodium degradation and immunofluorescence</div> <p>MDA-MB-231 cells used in this study were cultured on FN/gelatin matrix (for 8 h) or thin gelatin matrix (for 4 h), and treated as described.<sup><a href="#bib57">57</a></sup></p> </div> <div class="zebrafishtumorcelldisseminationassay" title="sec"> <div class="title" tagx="title" title="title">
Zebrafish tumor cell dissemination assay</div> <p>The zebrafish tumor cell dissemination assay was done as described.<sup><a href="#bib28">28</a></sup></p> </div> <div class="mousemodels" title="sec"> <div class="title" tagx="title" title="title">
Mouse models</div> <p>Tumor growth and spontaneous metastasis formation assayed by injecting tumor cells orthotopically into inguinal mammary fat pads (6- to 8-week-old female NOD/SCID/IL2Rγ-null mice) (Jackson Laboratory, Bar Harbor, ME, USA). Mice anesthetized with isoflurane, injected with 1.5 × 10<sup>5</sup> cells in Hank's Balanced Salt Solution (Gibco); killed 6±0.5 weeks post injection; tumors dissected, weighed, flash frozen, stored (−80 °C) or fixed: 3.8% formaldehyde, imaged with a fluorescence microscope, and embedded in paraffin and sectioned. Lungs were collected, fixed: 3.8% formaldehyde, embedded in paraffin, sectioned and stained with hematoxylin and eosin (H&amp;amp;E). ZsGreen-positive foci were counted in left pulmonary lobe using ImageJ.</p> </div> <div class="intravitalimaging" title="sec"> <div class="title" tagx="title" title="title">
Intravital imaging</div> <p>Female NOD/SCID/IL2Rγ-null mice (6–8 weeks old) injected in the mammary fat pad with Ctrl-shRNA or LpdhsRNA2-expressing LM2 cells. Experiments were performed as described previously.<sup><a href="#bib58">58</a></sup> Five mice per group were used. Collagen I fibers were imaged by second harmonic generated polarized light. Cell motility observed by time-lapse imaging: 30 min in 2-min cycles. Three-dimensional time-lapse videos were analyzed using Image J. Tumor cell motility was quantified manually. A cell was scored as motile if the translocation of the cell body was visible over the course of a 30-min video within a visual field that is defined in three dimensions as 50 μm by 512 × 512 pixels. Protrusion was defined as tumors cells showing a dynamic lamellipodia-like morphology. A protrusion was defined to be at least 5 μm long, but less than half the length of the cell. Directionality was calculated as described.<sup><a href="#bib58">58</a></sup></p> </div> <div class="statistics" title="sec"> <div class="title" tagx="title" title="title">
Statistics</div> <p>Statistical analysis performed by ANOVA with appropriate <i>post hoc</i> tests (see figure legends), or Student's <i>t</i>-test using Prism 5 (GraphPAD Software). <i>P</i>-values &amp;lt;0.05 considered as significant.</p> </div> <div class="clinicaldatasetsanalysis" title="sec"> <div class="title" tagx="title" title="title">
Clinical data sets analysis</div> <p>Oncomine (<a href="http://www.oncomine.org">www.oncomine.org</a>) used for Lpd mRNA expression from microarray data.<sup><a href="#bib21">21</a></sup> Statistical survival analysis (Kaplan–Meier plots) was done with ROCK (<a href="http://www.rock.icr.ac.uk">http://www.rock.icr.ac.uk</a>). Expression value of Lpd was upper quartile (25% against rest) for the NKI295<sup><a href="#bib24">24</a></sup> and Loi<sup><a href="#bib22">22</a></sup> data sets or upper tertile (33% against rest) for the Miller<sup><a href="#bib23">23</a></sup> data set. The log-rank <i>P</i>-value was used for statistical significance.</p> </div> <div class="tissuemicroarrays" title="sec"> <div class="title" tagx="title" title="title">
Tissue microarrays</div> <p>TMAs were performed with 0.6 mm<sup>2</sup> cores of formalin-fixed, paraffin-embedded invasive breast tumor samples (312 consecutive patients) with clinicopathological data (King's Health Partners Cancer Biobank, London, UK). Automated immunohistochemistry (VENTANA Discovery UTLRA) of 3 μm TMA sections was performed, deparaffinized (EZ prep; Ventana Medical) (30 min, 72 °C), antigen retrieval in automated slide stainer (ULTRA CC2 solution, Ventana Medical) (68 min, 91 °C). Affinity purified polyclonal rabbit Lpd antibody in PBS (1:250) (32 min, room temperature) was used. Slides counterstained with hematoxylin II and bluing reagent (Ventana Medical) (4 min each), dehydrated: 1 × 70% IMS, 1 min, 2 × 100% IMS, 1.5 min, 3 × xylene, 1 min, mounted (Eukitt) and imaged (Leica microscope, × 40).</p> <p>Intensity of Lpd in cytoplasm and membrane assessed on TMAs using weighted histoscore (H-Score) method: intensity in majority of cells assessed as negative and weak (1), or moderate and strong (2), then multiplied by the percentage of cells within this category. The weighted histoscore: 0–200 divided into thirtiles: Cytoplasma: histoscore 1 (0–88.75), histoscore 2 (88.75–170) and histoscore 3 (170–200); Membrane: histoscore 1 (0–25), histoscore 2 (25–95) and histoscore 3 (95–200). TMAs were assessed by two independent observers (UP and CG). Intensity scores that varied by more than a factor of one or a proportion by more than 20% were jointly reassessed and consensus reached. For all other cases, the mean score was used. Disease-specific survival curves generated using the Kaplan–Meier method. The log-rank test was used to compare statistically significant differences between subgroups. Univariate and multivariate analyses Cox proportional hazards regression models used to evaluate overall and breast cancer-specific death by histoscore of Lpd intensity in cytoplasm and at membrane. All analyses were performed using Statistical Analysis Systems (SAS) 9.4 (SAS Institute, Cary, NC, USA).</p> </div> <div class="studyapproval" title="sec"> <div class="title" tagx="title" title="title">
Study approval</div> <p>The study approval was obtained for TMA from NHS Research Ethics Committee, King's Health Partners Cancer Biobank, London, UK. Mice: all animal experiments and husbandry were approved by MIT's Department of Comparative Medicine and Committee on Animal Care. The study approval was obtained for Zebrafish from Northern Stockholm Experimental Animal Ethical Committee.</p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank members of the Gertler and Krause laboratories for helpful discussions and technical assistance; R Weinberg for his helpful discussions; J Lamar, S Hansen, D Mullins, M Way, C Cepko, D Sabatini, T Koleske and M Hemann for reagents; the Koch Institute Swanson Biotechnology Center – specifically M Griffin (Flow Cytometry Core Facility), E Vasile (Microscopy Core Facility) and K Cormier (Hope Babette Tang Histology Facility) for technical support. D Barry and M Way (LRI, CRICK, UK) for adapting the Image J plugin ADAPT for us to V1.33. Patient samples and data were provided by King's Health Partners Cancer Biobank, London, UK, which is supported by the Experimental Cancer Medicine Centre at King's College London and the Department of Health via the National Institute for Health Research comprehensive Biomedical Research Centre award. GC received a postdoctoral fellowship from the Ludwig Fund for Cancer Research and UP received a King's College Overseas Research PhD Studentship (KORS). This work was supported by grants from the National Cancer Institute (U54-CA112967 and U54-CA163109), funds from the Ludwig Center for Metastasis Research at MIT (FBG, GC, AN and ROH), the Koch Institute Support Grant P30-CA14051 from the National Cancer Institute, the Biotechnology and Biological Science Research Council, UK (BB/F011431/1; BB/J000590/1; BB/N000226/1) and the Wellcome Trust (082907/Z/07/Z) (MK).</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><a href="#sup1">Supplementary Information</a> accompanies this paper on the Oncogene website (http://www.nature.com/onc)</p> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors declare no conflict of interest.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="bib1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Eccles SA, Aboagye EO, Ali S, Anderson AS, Armes J, Berditchevski F et al. <span class="mixed-article-title" title="mixed-article-title">Critical research gaps and translational priorities for the successful prevention and treatment of breast cancer</span>. <span class="source" tagx="source" title="source">Breast Cancer Res</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">R92</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24286369">24286369</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Friedl P, Alexander S. <span class="mixed-article-title" title="mixed-article-title">Cancer invasion and the microenvironment: plasticity and reciprocity</span>. <span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">147</span>: <span class="fpage" tagx="fpage" title="fpage">992</span>–1009.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22118458">22118458</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Roussos ET, Condeelis JS, Patsialou A. <span class="mixed-article-title" title="mixed-article-title">Chemotaxis in cancer</span>. <span class="source" tagx="source" title="source">Nat Rev Cancer</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">573</span>–587.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21779009">21779009</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ridley AJ. <span class="mixed-article-title" title="mixed-article-title">Life at the leading edge</span>. <span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">145</span>: <span class="fpage" tagx="fpage" title="fpage">1012</span>–1022.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21703446">21703446</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Krause M, Gautreau A. <span class="mixed-article-title" title="mixed-article-title">Steering cell migration: lamellipodium dynamics and the regulation of directional persistence</span>. <span class="source" tagx="source" title="source">Nat Rev Mol Cell Biol</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">577</span>–590.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25145849">25145849</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Insall RH, Machesky LM. <span class="mixed-article-title" title="mixed-article-title">Actin dynamics at the leading edge: from simple machinery to complex networks</span>. <span class="source" tagx="source" title="source">Dev Cell</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">310</span>–322.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19758556">19758556</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bisi S, Disanza A, Malinverno C, Frittoli E, Palamidessi A, Scita G. <span class="mixed-article-title" title="mixed-article-title">Membrane and actin dynamics interplay at lamellipodia leading edge</span>. <span class="source" tagx="source" title="source">Curr Opin Cell Biol</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">565</span>–573.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23639310">23639310</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Breitsprecher D, Kiesewetter AK, Linkner J, Vinzenz M, Stradal TE, Small JV et al. <span class="mixed-article-title" title="mixed-article-title">Molecular mechanism of Ena/VASP-mediated actin-filament elongation</span>. <span class="source" tagx="source" title="source">EMBO J</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">456</span>–467.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217643">21217643</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bear JE, Svitkina TM, Krause M, Schafer DA, Loureiro JJ, Strasser GA et al. <span class="mixed-article-title" title="mixed-article-title">Antagonism between Ena/VASP proteins and actin filament capping regulates fibroblast motility</span>. <span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">109</span>: <span class="fpage" tagx="fpage" title="fpage">509</span>–521.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12086607">12086607</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Barzik M, Kotova TI, Higgs HN, Hazelwood L, Hanein D, Gertler FB et al. <span class="mixed-article-title" title="mixed-article-title">Ena/VASP proteins enhance actin polymerization in the presence of barbed end capping proteins</span>. <span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">280</span>: <span class="fpage" tagx="fpage" title="fpage">28653</span>–28662.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15939738">15939738</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chereau D, Dominguez R. <span class="mixed-article-title" title="mixed-article-title">Understanding the role of the G-actin-binding domain of Ena/VASP in actin assembly</span>. <span class="source" tagx="source" title="source">J Struct Biol</span><span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">155</span>: <span class="fpage" tagx="fpage" title="fpage">195</span>–201.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16684607">16684607</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pasic L, Kotova T, Schafer DA. <span class="mixed-article-title" title="mixed-article-title">Ena/VASP proteins capture actin filament barbed ends</span>. <span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">283</span>: <span class="fpage" tagx="fpage" title="fpage">9814</span>–9819.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18283104">18283104</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Breitsprecher D, Kiesewetter AK, Linkner J, Urbanke C, Resch GP, Small JV et al. <span class="mixed-article-title" title="mixed-article-title">Clustering of VASP actively drives processive, WH2 domain-mediated actin filament elongation</span>. <span class="source" tagx="source" title="source">EMBO J</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">2943</span>–2954.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18923426">18923426</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hansen SD, Mullins RD. <span class="mixed-article-title" title="mixed-article-title">VASP is a processive actin polymerase that requires monomeric actin for barbed end association</span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">191</span>: <span class="fpage" tagx="fpage" title="fpage">571</span>–584.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21041447">21041447</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Roussos ET, Wang Y, Wyckoff JB, Sellers RS, Wang W, Li J et al. <span class="mixed-article-title" title="mixed-article-title">Mena deficiency delays tumor progression and decreases metastasis in polyoma middle-T transgenic mouse mammary tumors</span>. <span class="source" tagx="source" title="source">Breast Cancer Res</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">R101</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21108830">21108830</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Philippar U, Roussos ET, Oser M, Yamaguchi H, Kim HD, Giampieri S et al. <span class="mixed-article-title" title="mixed-article-title">A Mena invasion isoform potentiates EGF-induced carcinoma cell invasion and metastasis</span>. <span class="source" tagx="source" title="source">Dev Cell</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">813</span>–828.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19081071">19081071</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Krause M, Leslie JD, Stewart M, Lafuente EM, Valderrama F, Jagannathan R et al. <span class="mixed-article-title" title="mixed-article-title">Lamellipodin, an Ena/VASP ligand, is implicated in the regulation of lamellipodial dynamics</span>. <span class="source" tagx="source" title="source">Dev Cell</span><span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">571</span>–583.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15469845">15469845</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Krause M, Dent EW, Bear JE, Loureiro JJ, Gertler FB. <span class="mixed-article-title" title="mixed-article-title">ENA/VASP proteins: regulators of the actin cytoskeleton and cell migration</span>. <span class="source" tagx="source" title="source">Annu Rev Cell Dev Biol</span><span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">541</span>–564.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14570581">14570581</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Michael M, Vehlow A, Navarro C, Krause M. <span class="mixed-article-title" title="mixed-article-title">c-Abl, Lamellipodin and Ena/VASP proteins cooperate in dorsal ruffling of fibroblasts and axonal morphogenesis</span>. <span class="source" tagx="source" title="source">Curr Biol</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">783</span>–791.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20417104">20417104</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Law AL, Vehlow A, Kotini M, Dodgson L, Soong D, Theveneau E et al. <span class="mixed-article-title" title="mixed-article-title">Lamellipodin and the Scar/WAVE complex cooperate to promote cell migration <i>in vivo</i></span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">203</span>: <span class="fpage" tagx="fpage" title="fpage">673</span>–689.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24247431">24247431</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rhodes DR, Yu J, Shanker K, Deshpande N, Varambally R, Ghosh D et al. <span class="mixed-article-title" title="mixed-article-title">ONCOMINE: a cancer microarray database and integrated data-mining platform</span>. <span class="source" tagx="source" title="source">Neoplasia</span><span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–6.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15068665">15068665</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Loi S, Haibe-Kains B, Desmedt C, Wirapati P, Lallemand F, Tutt AM et al. <span class="mixed-article-title" title="mixed-article-title">Predicting prognosis using molecular profiling in estrogen receptor-positive breast cancer treated with tamoxifen</span>. <span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">239</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18498629">18498629</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Miller LD, Smeds J, George J, Vega VB, Vergara L, Ploner et al. <span class="mixed-article-title" title="mixed-article-title">An expression signature for p53 status in human breast cancer predicts mutation status, transcriptional effects, and patient survival</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">102</span>: <span class="fpage" tagx="fpage" title="fpage">13550</span>–13555.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141321">16141321</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">van de Vijver MJ, He YD, van't Veer LJ, Dai H, Hart AA, Voskuil DW et al. <span class="mixed-article-title" title="mixed-article-title">A gene-expression signature as a predictor of survival in breast cancer</span>. <span class="source" tagx="source" title="source">N Engl J Med</span><span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">347</span>: <span class="fpage" tagx="fpage" title="fpage">1999</span>–2009.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12490681">12490681</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Youssef G, Gillett C, Agbaje O, Crompton T, Montano X. <span class="mixed-article-title" title="mixed-article-title">Phosphorylation of NTRK1 at Y674/Y675 induced by TP53-dependent repression of PTPN6 expression: a potential novel prognostic marker for breast cancer</span>. <span class="source" tagx="source" title="source">Mod Pathol</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">361</span>–374.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23948750">23948750</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Minn AJ, Gupta GP, Siegel PM, Bos PD, Shu W, Giri DD et al. <span class="mixed-article-title" title="mixed-article-title">Genes that mediate breast cancer metastasis to lung</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">436</span>: <span class="fpage" tagx="fpage" title="fpage">518</span>–524.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16049480">16049480</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Arjonen A, Kaukonen R, Mattila E, Rouhi P, Hognas G, Sihto H et al. <span class="mixed-article-title" title="mixed-article-title">Mutant p53-associated myosin-X upregulation promotes breast cancer invasion and metastasis</span>. <span class="source" tagx="source" title="source">J Clin Invest</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">124</span>: <span class="fpage" tagx="fpage" title="fpage">1069</span>–1082.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24487586">24487586</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lee SL, Rouhi P, Dahl Jensen L, Zhang D, Ji H, Hauptmann G et al. <span class="mixed-article-title" title="mixed-article-title">Hypoxia-induced pathological angiogenesis mediates tumor cell dissemination, invasion, and metastasis in a zebrafish tumor model</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">106</span>: <span class="fpage" tagx="fpage" title="fpage">19485</span>–19490.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19887629">19887629</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Goswami S, Sahai E, Wyckoff JB, Cammer M, Cox D, Pixley FJ et al. <span class="mixed-article-title" title="mixed-article-title">Macrophages promote the invasion of breast carcinoma cells via a colony-stimulating factor-1/epidermal growth factor paracrine loop</span>. <span class="source" tagx="source" title="source">Cancer Res</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">5278</span>–5283.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15958574">15958574</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Symons MH, Mitchison TJ. <span class="mixed-article-title" title="mixed-article-title">Control of actin polymerization in live and permeabilized fibroblasts</span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">1991</span>; <span class="volume" tagx="volume" title="volume">114</span>: <span class="fpage" tagx="fpage" title="fpage">503</span>–513.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1860882">1860882</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bailly M, Macaluso F, Cammer M, Chan A, Segall JE, Condeelis JS. <span class="mixed-article-title" title="mixed-article-title">Relationship between Arp2/3 complex and the barbed ends of actin filaments at the leading edge of carcinoma cells after epidermal growth factor stimulation</span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">1999</span>; <span class="volume" tagx="volume" title="volume">145</span>: <span class="fpage" tagx="fpage" title="fpage">331</span>–345.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10209028">10209028</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Iglesias PA, Devreotes PN. <span class="mixed-article-title" title="mixed-article-title">Navigating through models of chemotaxis</span>. <span class="source" tagx="source" title="source">Curr Opin Cell Biol</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">35</span>–40.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18207721">18207721</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Janetopoulos C, Firtel RA. <span class="mixed-article-title" title="mixed-article-title">Directional sensing during chemotaxis</span>. <span class="source" tagx="source" title="source">FEBS Lett</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">582</span>: <span class="fpage" tagx="fpage" title="fpage">2075</span>–2085.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18452713">18452713</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bear JE, Loureiro JJ, Libova I, Fassler R, Wehland J, Gertler FB. <span class="mixed-article-title" title="mixed-article-title">Negative regulation of fibroblast motility by Ena/VASP proteins</span>. <span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">101</span>: <span class="fpage" tagx="fpage" title="fpage">717</span>–728.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10892743">10892743</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hennigan RF, Hawker KL, Ozanne BW. <span class="mixed-article-title" title="mixed-article-title">Fos-transformation activates genes associated with invasion</span>. <span class="source" tagx="source" title="source">Oncogene</span><span class="year" tagx="year" title="year">1994</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">3591</span>–3600.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7970719">7970719</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sanz-Moreno V, Gadea G, Ahn J, Paterson H, Marra P, Pinner S et al. <span class="mixed-article-title" title="mixed-article-title">Rac activation and inactivation control plasticity of tumor cell movement</span>. <span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">135</span>: <span class="fpage" tagx="fpage" title="fpage">510</span>–523.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18984162">18984162</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lu P, Weaver VM, Werb Z. <span class="mixed-article-title" title="mixed-article-title">The extracellular matrix: a dynamic niche in cancer progression</span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">196</span>: <span class="fpage" tagx="fpage" title="fpage">395</span>–406.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22351925">22351925</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Beaty BT, Condeelis J. <span class="mixed-article-title" title="mixed-article-title">Digging a little deeper: The stages of invadopodium formation and maturation</span>. <span class="source" tagx="source" title="source">Eur J Cell Biol</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">93</span>: <span class="fpage" tagx="fpage" title="fpage">438</span>–444.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25113547">25113547</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Poincloux R, Lizarraga F, Chavrier P. <span class="mixed-article-title" title="mixed-article-title">Matrix invasion by tumour cells: a focus on MT1-MMP trafficking to invadopodia</span>. <span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">122</span>: <span class="fpage" tagx="fpage" title="fpage">3015</span>–3024.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19692588">19692588</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kim LC, Song L, Haura EB. <span class="mixed-article-title" title="mixed-article-title">Src kinases as therapeutic targets for cancer</span>. <span class="source" tagx="source" title="source">Nat Rev Clin Oncol</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">587</span>–595.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19787002">19787002</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Greuber EK, Smith-Pearson P, Wang J, Pendergast AM. <span class="mixed-article-title" title="mixed-article-title">Role of ABL family kinases in cancer: from leukaemia to solid tumours</span>. <span class="source" tagx="source" title="source">Nat Rev Cancer</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">559</span>–571.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23842646">23842646</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bradley WD, Koleske AJ. <span class="mixed-article-title" title="mixed-article-title">Regulation of cell migration and morphogenesis by Abl-family kinases: emerging mechanisms and physiological contexts</span>. <span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">122</span>: <span class="fpage" tagx="fpage" title="fpage">3441</span>–3454.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19759284">19759284</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Koleske AJ, Gifford AM, Scott ML, Nee M, Bronson RT, Miczek KA et al. <span class="mixed-article-title" title="mixed-article-title">Essential roles for the Abl and Arg tyrosine kinases in neurulation</span>. <span class="source" tagx="source" title="source">Neuron</span><span class="year" tagx="year" title="year">1998</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">1259</span>–1272.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9883720">9883720</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Derivery E, Lombard B, Loew D, Gautreau A. <span class="mixed-article-title" title="mixed-article-title">The Wave complex is intrinsically inactive</span>. <span class="source" tagx="source" title="source">Cell Motil Cytoskeleton</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">66</span>: <span class="fpage" tagx="fpage" title="fpage">777</span>–790.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19206172">19206172</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Roussos ET, Balsamo M, Alford SK, Wyckoff JB, Gligorijevic B, Wang Y et al. <span class="mixed-article-title" title="mixed-article-title">Mena invasive (MenaINV) promotes multicellular streaming motility and transendothelial migration in a mouse model of breast cancer</span>. <span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">124</span>: <span class="fpage" tagx="fpage" title="fpage">2120</span>–2131.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21670198">21670198</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Di Modugno F, Mottolese M, Di Benedetto A, Conidi A, Novelli F, Perracchio L et al. <span class="mixed-article-title" title="mixed-article-title">The cytoskeleton regulatory protein hMena (ENAH) is overexpressed in human benign breast lesions with high risk of transformation and human epidermal growth factor receptor-2-positive/hormonal receptor-negative tumors</span>. <span class="source" tagx="source" title="source">Clin Cancer Res</span><span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">1470</span>–1478.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16533770">16533770</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Vehlow A, Soong D, Vizcay-Barrena G, Bodo C, Law AL, Perera U et al. <span class="mixed-article-title" title="mixed-article-title">Endophilin, Lamellipodin, and Mena cooperate to regulate F-actin-dependent EGF-receptor endocytosis</span>. <span class="source" tagx="source" title="source">Embo J</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">2722</span>–2734.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24076656">24076656</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lyulcheva E, Taylor E, Michael M, Vehlow A, Tan S, Fletcher et al. <span class="mixed-article-title" title="mixed-article-title">Drosophila pico and its mammalian ortholog lamellipodin activate serum response factor and promote cell proliferation</span>. <span class="source" tagx="source" title="source">Dev Cell</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">680</span>–690.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19000833">19000833</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Newsome TP, Weisswange I, Frischknecht F, Way M. <span class="mixed-article-title" title="mixed-article-title">Abl collaborates with Src family kinases to stimulate actin-based motility of vaccinia virus</span>. <span class="source" tagx="source" title="source">Cell Microbiol</span><span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">233</span>–241.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16441434">16441434</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dickins RA, Hemann MT, Zilfou JT, Simpson DR, Ibarra I, Hannon GJ et al. <span class="mixed-article-title" title="mixed-article-title">Probing tumor phenotypes using stable and regulated synthetic microRNA precursors</span>. <span class="source" tagx="source" title="source">Nat Genet</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">1289</span>–1295.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16200064">16200064</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lamar JM, Stern P, Liu H, Schindler JW, Jiang ZG, Hynes RO. <span class="mixed-article-title" title="mixed-article-title">The Hippo pathway target, YAP, promotes metastasis through its TEAD-interaction domain</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">109</span>: <span class="fpage" tagx="fpage" title="fpage">E2441</span>–E2450.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22891335">22891335</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ignatoski KM, Ethier SP. <span class="mixed-article-title" title="mixed-article-title">Constitutive activation of pp125fak in newly isolated human breast cancer cell lines</span>. <span class="source" tagx="source" title="source">Breast Cancer Res Treat</span><span class="year" tagx="year" title="year">1999</span>; <span class="volume" tagx="volume" title="volume">54</span>: <span class="fpage" tagx="fpage" title="fpage">173</span>–182.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10424408">10424408</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Mouneimne G, Soon L, DesMarais V, Sidani M, Song X, Yip SC et al. <span class="mixed-article-title" title="mixed-article-title">Phospholipase C and cofilin are required for carcinoma cell directionality in response to EGF stimulation</span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">166</span>: <span class="fpage" tagx="fpage" title="fpage">697</span>–708.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15337778">15337778</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lebrand C, Dent EW, Strasser GA, Lanier LM, Krause M, Svitkina TM et al. <span class="mixed-article-title" title="mixed-article-title">Critical role of Ena/VASP proteins for filopodia formation in neurons and in function downstream of netrin-1</span>. <span class="source" tagx="source" title="source">Neuron</span><span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">42</span>: <span class="fpage" tagx="fpage" title="fpage">37</span>–49.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15066263">15066263</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Barry DJ, Durkin CH, Abella JV, Way M. <span class="mixed-article-title" title="mixed-article-title">Open source software for quantification of cell migration, protrusions, and fluorescence intensities</span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">209</span>: <span class="fpage" tagx="fpage" title="fpage">163</span>–180.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25847537">25847537</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Mouneimne G, DesMarais V, Sidani M, Scemes E, Wang W, Song X et al. <span class="mixed-article-title" title="mixed-article-title">Spatial and temporal control of cofilin activity is required for directional sensing during chemotaxis</span>. <span class="source" tagx="source" title="source">Curr Biol</span><span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">2193</span>–2205.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17113383">17113383</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sharma VP, Eddy R, Entenberg D, Kai M, Gertler FB, Condeelis J. <span class="mixed-article-title" title="mixed-article-title">Tks5 and SHIP2 regulate invadopodium maturation, but not initiation, in breast carcinoma cells</span>. <span class="source" tagx="source" title="source">Curr Biol</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">2079</span>–2089.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24206842">24206842</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gligorijevic B, Wyckoff J, Yamaguchi H, Wang Y, Roussos ET, Condeelis J. <span class="mixed-article-title" title="mixed-article-title">N-WASP-mediated invadopodium formation is involved in intravasation and lung metastasis of mammary tumors</span>. <span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">125</span>: <span class="fpage" tagx="fpage" title="fpage">724</span>–734.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22389406">22389406</a></span></span></li> </ul> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Information</span><div class="media" title="media"><a href="onc201647x1.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Video 1</span><div class="media" title="media"><a href="onc201647x2.avi">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Video 2</span><div class="media" title="media"><a href="onc201647x3.avi">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Video 3</span><div class="media" title="media"><a href="onc201647x4.mov">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Video 4</span><div class="media" title="media"><a href="onc201647x5.mov">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Video 5</span><div class="media" title="media"><a href="onc201647x6.mov">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Video 6</span><div class="media" title="media"><a href="onc201647x7.mov">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Increased Lpd expression correlates with poor prognosis for breast cancer patients. (<b>a</b>) Western blot analysis of Lpd expression in human breast cancer cell lines with varying metastatic potential. Loading control: Tubulin. (<b>b</b>) Kaplan–Meier analysis of metastasis-free survival in the NKI295 data set. Patients were stratified by expression of Lpd. The <i>P</i>-value was calculated by a log-rank test. (<b>c</b>) Kaplan–Meier analysis of disease-free survival in the NKI295 data set. Patients were stratified by expression of Lpd. The <i>P-</i>value was calculated by a log-rank test. (<b>d</b>) Kaplan–Meier plots of breast cancer-associated mortality of histoscores 1–3 for Lpd intensity at the plasma membrane. Histoscore 2: HR (95% CI): 2.23 (1.26–3.95). (<b>e</b>) Representative examples of Lpd immunohistochemistry staining for histoscores 1–3 for Lpd staining intensity at the plasma membrane. Scale bar, 5 μm. See also <a href="#sup1">Supplementary Figures S1 and S2</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Lpd is required for lung metastasis from orthotopic mammary tumors. (<b>a–e</b>) NOD/SCID/IL2Rγ-null mice were injected orthotopically with LM2 cells stably expressing Ctrl-shRNA or Lpd-shRNA1 or Lpd-shRNA2. Tumors were allowed to grow for 6±0.5 weeks. (<b>a</b>) Primary tumor weights at the time of killing of individual mice are shown. Data are represented as mean±s.e.m. Number of animals per group is indicated in (<b>b</b>). (<b>b</b>) The number of mice that presented with visible metastases in the lung is indicated (mice with lung metastases/total number of mice analyzed). (<b>c</b>) Representative images of whole left pulmonary lobe from LM2 (control or knockdown) tumor-bearing mice with ZsGreen-positive metastatic foci (top panel). Scale bar, 5 mm. Representative lung sections stained with H&amp;amp;E; arrowhead indicates the presence of metastatic foci (bottom panel). Scale bar, 20 μm. (<b>d</b>) Numbers of ZsGreen-positive metastatic foci in the left pulmonary lobe were counted. Quantification of data shown in (<b>c</b>, top panel). Data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; *<i>P</i>⩽0.05. Number of animals per group is indicated in (<b>b</b>). (<b>e</b>) Representative images of paraffin tissue sections stained with Masson's trichrome of primary tumors to show local invasion. Scale bar, 20 μm. (<b>f–i</b>) Tumor cell motility <i>in vivo</i> monitored by multi-photon confocal imaging. (<b>f</b>) Image shows a Ctrl-shRNA ZsGreen tumor. Cyan=ZsGreen-positive cells, red=collagen fibers. One motile Ctrl-shRNA-expressing tumor cell is outlined. Scale bar, 40 μm. (<b>g</b>) The average numbers of motile cells per field were determined. Data are represented as mean±s.e.m.. Unpaired <i>t</i>-test; *<i>P</i>⩽0.05, <i>n</i>=5 mice per group. (<b>h</b>) The average numbers of cells extending protrusions per field were determined. Data are represented as mean±s.e.m.; unpaired <i>t</i>-test; *<i>P</i>⩽0.05; <i>n</i>=5 mice per group. (<b>i</b>) Directionality of the motile cells (net path/total path) was determined. Data are represented as mean±s.e.m.; unpaired <i>t</i>-test; *<i>P</i>⩽0.05; <i>n</i>=5 mice per group. (<b>j, k</b>) Red fluorescently labeled MDA-MB-231 cells overexpressing GFP-Lpd or GFP as a control were implanted into the perivitelline cavity of zebrafish embryos and dissemination to the trunk region was quantified 2 days after injection. (<b>j</b>) Image of representative zebrafish trunks 2 days after injection. Arrows point to seeded tumor cells. Scale bar, 500 μm. (<b>k</b>) Quantified data from (<b>j</b>) represented as mean number of seeded tumor cells per mm of trunk length. Mean±s.e.m; data from 53 fish embryos for GFP and from 49 fish embryos for GFP-Lpd from 3 independent experiments; <i>t</i>-test; **<i>P</i>⩽0.01. See also <a href="#sup1">Supplementary Figure S2</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Lpd is required for EGF-induced membrane protrusion. (<b>a</b>) Representative kymographs of Ctrl-shRNA and Lpd-shRNA2 MTLn3 cells. A line drawn perpendicular to the cell surface is shown for each frame of a time-lapse movie to depict temporal dynamics of cell edge. X axis: time (arrow length: 20 s); Y axis: distance (arrow length: 3.1 μm). (<b>b</b>) Quantification of protrusion parameters from kymographic analysis of Ctrl-shRNA and Lpd-shRNA2 MTLn3 cells. Data represented as mean±s.e.m. Unpaired <i>t</i>-test; *<i>P</i>⩽0.05. (<b>c</b>) Micrographs showing immunofluorescence for endogenous Lpd in MTLn3 cells stimulated with 5 n <div class="sc_UNKNOWN">
m</div> EGF (post-stimulation time is indicated). Scale bar, 10 μm. (<b>d</b>) Quantification of data shown in (<b>c</b>): mean fluorescence intensity of Lpd within a 0.66-μm zone at the lamellipodial edge is plotted as a function of time; &amp;gt;30 cells analyzed from at least three independent experiments. Error bars indicate s.e.m. (<b>e</b>) Representative micrographs from time-lapse movies of Ctrl-shRNA (control non-targeting shRNA) and Lpd-shRNA2-expressing MTLn3 cells stimulated with 5 n <div class="sc_UNKNOWN">
m</div> EGF. Dashed line shows cell edge. Scale bar, 10 μm. (<b>f</b>) Quantification of membrane protrusions on Ctrl-shRNA and Lpd-shRNA-treated cells. Cell area was determined after EGF stimulation and normalized to the pretreatment cell area; &amp;gt;30 cells analyzed from three independent experiments. Error bars indicate s.e.m. See also <a href="#sup1">Supplementary Figure S3</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Lpd is required for chemosensing. (<b>a</b>) Barbed-end incorporation after 5 n <div class="sc_UNKNOWN">
M</div> EGF stimulation in Ctrl-shRNA- and Lpd-shRNA2-MTLn3 cells. Fixed cells expressing rhodamine-labeled actin were co-stained with phalloidin. Scale bar, 20 μm. (<b>b</b>) Relative number of barbed-ends incorporation at the lamellipodium edge at 1 min after 5 n <div class="sc_UNKNOWN">
M</div> EGF stimulation; over 60 cells analyzed. (<i>n</i>=3). Data are represented as mean±s.e.m. Unpaired <i>t</i>-test; *<i>P</i>⩽0.05. (<b>c</b>) Mena immunofluorescence in Ctrl-shRNA and Lpd-shRNA2 MTLn3 cells. Cells were stimulated for 1 min with 5 n <div class="sc_UNKNOWN">
M</div> EGF. Insets show enlarged image of Mena staining. Scale bar, 10 μm. (<b>d</b>) Quantification of data shown in (<b>c</b>); mean fluorescence intensity of Mena at the lamellipodium edge (within 0.66 μm of leading edge); over 45 cells analyzed. (<i>n</i>=3). Data are represented as mean±s.e.m. Unpaired <i>t</i>-test; *<i>P</i>⩽0.05. (<b>e</b>) p34Arc immunofluorescence in Ctrl-shRNA- and Lpd-shRNA2-MTLn3 cells, 1 min after 5 n <div class="sc_UNKNOWN">
M</div> EGF stimulation. Insets show enlarged image of p34Arc staining. Scale bar, 10 μm. (<b>f</b>) Quantification of data shown in (<b>e</b>); mean fluorescence intensity of p34Arc at the lamellipodium edge (within 0.66 μm of the leading edge); over 45 cells analyzed. (<i>n</i>=3). Data are represented as mean±s.e.m. Unpaired <i>t</i>-test; *<i>P</i>⩽0.05. (<b>g</b>) Representative micrographs from time-lapse movies of Ctrl-shRNA- and Lpd-shRNA2-MTLn3 cells stimulated with an EGF-filled micropipette (position indicated by asterisk). White arrows on the 480-s frames indicate the directions of protrusion overtime. Scale bar, 10 μm. Colored lines indicate cell contour. (<b>h</b>) Quantification of chemotactic index of Ctrl-shRNA- and Lpd-shRNA2-MTLn3 cells. Over 25 cells analyzed from at least three independent experiments. Data are represented as mean±s.e.m. Unpaired <i>t</i>-test; *<i>P</i>⩽0.05. (<b>i</b>) Quantification of chemotactic index of Ctrl-shRNA MTLn3 cells transfected with GFP-vector (<i>n</i>=13 cells); and Lpd-shRNA2 MTLn3 cells transfected with either GFP-vector (<i>n</i>=22 cells), GFP-Lpd (<i>n</i>=17 cells) or GFP-Lpd<sup>EVmut</sup> (<i>n</i>=17 cells). Data are represented as mean±s.e.m. One-way ANOVA; Bonferroni's test; *<i>P</i>⩽0.05 vs Ctrl-shRNA+GFP; **<i>P</i>⩽0.05 vs Lpd-shRNA2+GFP; ***<i>P</i>⩽0.05 vs Lpd-shRNA2+GFP-Lpd. The difference between Lpd-shRNA2+GFP and Lpd-shRNA2+GFP-Lpd<sup>EVmut</sup> was not significant. See also <a href="#sup1">Supplementary Figures S4 and S5</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Lpd is required for 3D invasion of cancer cells. (<b>a, b</b>) Inverted invasion assays were performed using MDA-MB-231 cells stably expressing mCherry-H2B (labeling the nucleus) transfected with Ctrl-shRNA, Lpd-shRNA1 or Lpd-shRNA2. Additionally, cells were co-transfected with empty Blasticidin vector as well and transfected cells were selected. The nuclei of the cells were visualized using confocal microscopy. (<b>a</b>) The image stacks were processed by Volocity software to make a 3D reconstruction. (<b>b</b>) Quantification of the number of nuclei of invading cells above 40 μm from the data shown in (<b>a</b>), <i>n</i>=4 (with approximately 4000 cells per experiment). Data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; *<i>P</i>⩽0.05. (<b>c</b>) Inverted invasion assays with GFP or GFP-Lpd expressing MDA-MB-231 cells treated with the MMP inhibitor 10 μ <div class="sc_UNKNOWN">
M</div> GM6001 or just the solvent DMSO as a control. Quantification of the number of nuclei of invading cells above 40 μm from the data shown in <a href="#sup1">Supplementary Figure 6E</a>. <i>n</i>=3. Data are represented as mean±s.e.m. One-way ANOVA; Tukey's; *<i>P</i>⩽0.05, **<i>P</i>⩽0.01, ***<i>P</i>⩽0.001. (<b>d</b>) Inverted invasion assays were performed using MDA-MB-231 breast cancer cells stably expressing mCherry-H2B (labeling the nucleus) transfected with GFP-Lpd, GFP-Lpd<sup>EVmut</sup>, GFP-Lpd<sup>S/Wmut</sup>, GFP-Lpd<sup>EV+S/Wmut</sup> or GFP empty vector as a control. The nuclei of the cells were visualized using confocal microscopy. The image stacks were processed by Volocity software to make a 3D reconstruction. (<b>e</b>) Quantification of the number of nuclei of invading cells above 40 μm from the data shown in (<b>d</b>), <i>n</i>=3 (with approximately 4000 cells per experiment). Data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; *<i>P</i>⩽0.05, **<i>P</i>⩽0.01, ***<i>P</i>⩽0.001. (<b>f</b>) MDA-MB-231 plated on Alexa 488-gelatin/fibronectin matrix, fixed and stained for Lpd, and the invadopodia marker cortactin. White boxes: enlarged images shown in insets. Scale bar: 10 μm. (<b>g</b>) Steady-state assay for invadopodial matrix degradation. MDA-MB-231 cells were plated for 8 h on Alexa 488-gelatin/fibronectin matrices, fixed and stained with phalloidin. Scale bar, 10 μm. (<b>h</b>) Quantification of data shown in (<b>g</b>): invadopodial degradation area/cells in the steady-state matrix degradation assay, normalized to number of cells/field. Data are represented as mean±s.e.m. (<i>n</i>=3). Mann–Whitney test; *<i>P</i>&amp;lt;0.05. (<b>i</b>) MDA-MB-231 stably expressing Ctrl-shRNA or Lpd-shRNA cells were plated on 405-gelatin and immunostained with cortactin and Tks5 antibodies to identify invadopodia. Scale bar, 10 μm. (<b>j</b>) Quantification of data shown in (<b>i</b>); number of total invadopodia, invadopodia precursors and mature invadopodia per cell were determined; Ctrl-shRNA cells (<i>n</i>=52) or Lpd-shRNA cells (<i>n</i>=57). Data are represented as mean±s.e.m. Mann–Whitney test; *<i>P</i>&amp;lt;0.05. See also <a href="#sup1">Supplementary Figure S6</a>.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>c-Src phosphorylates Lpd and the Lpd-Scar/WAVE interaction is positively regulated by c-Abl and c-Src. (<b>a</b>) HEK293FT cells were transfected with either GFP as a control or GFP-Lpd and co-transfected with Src-WT (wild type) or Src-KI (kinase inactive). Immunoprecipitation was performed from cell lysates using Lpd-specific antibodies or rabbit IgG as a control followed by western blotting with anti-Lpd and anti-phosphotyrosine (pTyr) antibodies. <i>n</i>=3. (<b>b</b>) Abl and Arg double knockout MEFs (Abl/Arg DKO) were serum starved overnight and treated with 10 μ <div class="sc_UNKNOWN">
m</div> Bosutinib (c-Src kinase inhibitor) for 2 h before stimulating with 20 ng/ml PDGF-BB for 2 min. Immunoprecipitation was performed from cell lysates using Lpd-specific antibodies or rabbit IgG as a control followed by western blotting with anti-Lpd and anti-phosphotyrosine (pTyr) antibodies. (<b>c</b>) Quantified band intensities of chemiluminescence blots from (<b>b</b>) of Lpd and pTyr imaged with a CCD camera. pTyr was normalized against the immunoprecipitated Lpd. The pTyr signal from Rabbit IgG control lanes was subtracted from the pTyr signal from the immunoprecipitated Lpd lanes. <i>n</i>=3, data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; ***<i>P</i>⩽0.001, ****<i>P</i>⩽0.0001, NS, not significant. (<b>d</b>) HEK293FT cells were transfected with GFP-Lpd, Myc-tagged components of the Scar/WAVE complex and either Src-WT (wild type) or Src-KI (kinase inactive). Immunoprecipitation was performed from cell lysates using Lpd-specific antibody or rabbit IgG as a control followed by western blotting with anti-GFP, anti-Myc and anti-phosphotyrosine (pTyr) antibodies. (<b>e</b>) Quantified band intensities of chemiluminescence blots (<b>d</b>) of GFP-Lpd and Myc-tagged components of the Scar/WAVE complex imaged with a CCD camera. Individual Scar/WAVE components were normalized against the immunoprecipitated Lpd. <i>n</i>=4, data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; *<i>P</i>⩽0.05, **<i>P</i>⩽0.01. (<b>f</b>) HEK293FT cells were transfected with GFP-Lpd, Myc-tagged components of the Scar/WAVE complex and either GFP-c-Abl or GFP. Immunoprecipitation was performed from cell lysates using Lpd-specific antibody or rabbit IgG as a control followed by western blotting with anti-GFP, anti-Myc and anti-phosphotyrosine (pTyr) antibodies. (<b>g</b>) Quantified band intensities of chemiluminescence blots (<b>f</b>) of GFP-Lpd and Myc-tagged components of the Scar/WAVE complex imaged with a CCD camera. Individual Scar/WAVE components were normalized against the immunoprecipitated Lpd. <i>n</i>=4, data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; **<i>P</i>⩽0.01, ***<i>P</i>⩽0.001.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Phosphorylation of Lpd by c-Src and c-Abl is required for cancer cell invasion. (<b>a</b>) Peptides harboring all the tyrosine residues in Lpd were directly synthesized onto a membrane. An <i>in vitro</i> kinase assay was performed: the membranes were incubated with purified c-Src kinase and γ-<sup>32</sup>P-ATP. Phosphorylation was detected using a phosphorimager and visualized as high intensity spots. Increasing levels of c-Abl phosphorylation of respective peptides as identified by Michael <i>et al.</i><sup><a href="#bib19">19</a></sup> are indicated by (+), (++), (+++) and (−) for not phosphorylated. Straight rectangles represent the common phosphorylation sites for both c-Src and c-Abl, and dotted rectangles represent c-Abl specific phosphorylation sites.<sup><a href="#bib19">19</a></sup> (<b>b</b>) HeLa cells were transfected with GFP-Lpd<sup>Y6F</sup>, GFP-Lpd<sup>Y8F</sup> or GFP-Lpd as a control. HeLa cells were serum starved overnight and stimulated with 100 ng/ml EGF for 5 min. Immunoprecipitation was performed from cell lysates using Lpd-specific antibodies or rabbit IgG as a control followed by western blotting with anti-Lpd and anti-phosphotyrosine (pTyr) antibodies. (<b>c</b>) Quantified band intensities of chemiluminescence blots (<b>b</b>) of GFP-Lpd, GFP-Lpd phospho-mutants and pTyr imaged with a CCD camera. pTyr normalized against the immunoprecipitated Lpd. Baseline phosphorylation in the absence of EGF was subtracted from the corresponding EGF+ samples. <i>n</i>=6, data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; *<i>P</i>⩽0.0001. (<b>d, e</b>) Inverted invasion assays were performed using MDA-MB-231 breast cancer cells stably expressing mCherry-H2B (labeling the nucleus) transfected with GFP-Lpd, GFP-Lpd<sup>Y6F</sup>, GFP-Lpd<sup>Y8F</sup> or GFP empty vector as a control. The nuclei of the cells were visualized using confocal microscopy. (<b>d</b>) The image stacks were processed by Volocity software to make a 3D reconstruction. (<b>e</b>) Quantification of the number of nuclei of invading cells above 80 μm. <i>n</i>=6 (with approximately 4000 cells per experiment). Data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; *<i>P</i>⩽0.001, **<i>P</i>⩽0.0001. Error bars represent s.e.m. (<b>f</b>) HEK293FT cells were transfected with GFP-Lpd, GFP-Lpd<sup>Y4F</sup>, GFP-Lpd<sup>Y6F</sup>, GFP-Lpd<sup>Y8F</sup> and Myc-tagged components of the Scar/WAVE complex. Immunoprecipitation was performed from cell lysates using GFP-specific antibody or rabbit IgG as a control followed by western blotting with anti-GFP, anti-Myc and anti-phosphotyrosine (pTyr) antibodies. (<b>g</b>) Quantified band intensities of chemiluminescence blots (<b>f</b>) of GFP-Lpd and Myc-tagged components of the Scar/WAVE complex imaged with a CCD camera. Scar/WAVE2 was normalized against the immunoprecipitated Lpd. <i>n</i>=4, data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; **<i>P</i>⩽0.01, ***<i>P</i>⩽0.001. (<b>h</b>) Quantification of the number of protrusion of MDA-MB-231 transfected with GFP-Lpd, GFP-Lpd<sup>EVmut</sup>, GFP-Lpd<sup>S/Wmut</sup>, GFP-Lpd<sup>EV+S/Wmut</sup>, GFP-Lpd<sup>Y6F</sup>, GFP-Lpd<sup>Y8F</sup> or GFP empty vector as a control plated in 3D matrigel. <i>n</i>=35–46 cells for each mutant; from 5 experiments. Data are represented as mean±s.e.m. One-way ANOVA; Dunnett's; *<i>P</i>⩽0.05, ***<i>P</i>&amp;lt;0.001. See also <a href="#sup1">Supplementary Figures S7 and S8</a>.</p>   </div> </div>  </body></html>