<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Int J Health Geogr</span><span class="iso-abbrev" title="iso-abbrev">Int J Health Geogr</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">International Journal of Health Geographics</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1476-072X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7149848">7149848</a></span><span class="publisher-id" title="publisher-id">207</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12942-020-00207-3">10.1186/s12942-020-00207-3</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Detecting geospatial patterns of <i>Plasmodium falciparum</i> parasite migration in Cambodia using optimized estimated effective migration surfaces</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Yao</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">liyao@umd.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shetty</span><span class="given-names" tagx="given-names" title="given-names">Amol C.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">ashetty@som.umaryland.edu</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lon</span><span class="given-names" tagx="given-names" title="given-names">Chanthap</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">ChanthapL.ca@afrims.org</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spring</span><span class="given-names" tagx="given-names" title="given-names">Michele</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">Michele.spring.ctr@afrims.org</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saunders</span><span class="given-names" tagx="given-names" title="given-names">David L.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">david.l.saunders.mil@mail.mil</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukuda</span><span class="given-names" tagx="given-names" title="given-names">Mark M.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">Mark.Fukuda.mil@afrims.org</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hien</span><span class="given-names" tagx="given-names" title="given-names">Tran Tinh</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">hientt@oucru.org</span></span><a href="#Aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pukrittayakamee</span><span class="given-names" tagx="given-names" title="given-names">Sasithon</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">sasithon.puk@mahidol.ac.th</span></span><a href="#Aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fairhurst</span><span class="given-names" tagx="given-names" title="given-names">Rick M.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">rickfairhurst1@gmail.com</span></span><a href="#Aff8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dondorp</span><span class="given-names" tagx="given-names" title="given-names">Arjen M.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">arjen@tropmedres.ac</span></span><a href="#Aff9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plowe</span><span class="given-names" tagx="given-names" title="given-names">Christopher V.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">chris.plowe@duke.edu</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Connor</span><span class="given-names" tagx="given-names" title="given-names">Timothy D.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">timothydoconnor@gmail.com</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takala-Harrison</span><span class="given-names" tagx="given-names" title="given-names">Shannon</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">stakala@medicine.umaryland.edu</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">Kathleen</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">stewartk@umd.edu</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.164295.d</span><span class="institution-id" title="institution-id">0000 0001 0941 7177</span><span class="institution" title="institution">Center for Geospatial Information Science, Department of Geographical Sciences, </span><span class="institution" title="institution">University of Maryland, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.26009.3d</span><span class="institution-id" title="institution-id">0000 0004 1936 7961</span><span class="institution" title="institution">Duke Global Health Institute, </span><span class="institution" title="institution">Duke University, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.411024.2</span><span class="institution-id" title="institution-id">0000 0001 2175 4264</span><span class="institution" title="institution">Institute for Genome Sciences, </span><span class="institution" title="institution">University of Maryland School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.411024.2</span><span class="institution-id" title="institution-id">0000 0001 2175 4264</span><span class="institution" title="institution">Center for Vaccine Development and Global Health, </span><span class="institution" title="institution">University of Maryland School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.413910.e</span><span class="institution-id" title="institution-id">0000 0004 0419 1772</span><span class="institution" title="institution">Armed Forces Research Institute of Medical Sciences, </span></div></span><span class="citation_author_institution" id="Aff6">[6], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.412433.3</span><span class="institution-id" title="institution-id">0000 0004 0429 6814</span><span class="institution" title="institution">Oxford University Clinical Research Unit, </span></div></span><span class="citation_author_institution" id="Aff7">[7], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.10223.32</span><span class="institution-id" title="institution-id">0000 0004 1937 0490</span><span class="institution" title="institution">Department of Clinical Tropical Medicine, </span><span class="institution" title="institution">Mahidol University, </span></div></span><span class="citation_author_institution" id="Aff8">[8], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.94365.3d</span><span class="institution-id" title="institution-id">0000 0001 2297 5165</span><span class="institution" title="institution">National Institutes of Health, </span></div></span><span class="citation_author_institution" id="Aff9">[9], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.501272.3</span><span class="institution-id" title="institution-id">0000 0004 5936 4917</span><span class="institution" title="institution">Mahidol-Oxford Tropical Medicine Research Unit, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">19</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">13</span><span class="history" title="history"><span class="received" title="received">received: 2019-11-12</span><span class="accepted" title="accepted">accepted: 2020-4-1</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">Understanding the genetic structure of natural populations provides insight into the demographic and adaptive processes that have affected those populations. Such information, particularly when integrated with geospatial data, can have translational applications for a variety of fields, including public health. Estimated effective migration surfaces (EEMS) is an approach that allows visualization of the spatial patterns in genomic data to understand population structure and migration. In this study, we developed a workflow to optimize the resolution of spatial grids used to generate EEMS migration maps and applied this optimized workflow to estimate migration of <i>Plasmodium falciparum</i> in Cambodia and bordering regions of Thailand and Vietnam.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p id="Par2">The optimal density of EEMS grids was determined based on a new workflow created using density clustering to define genomic clusters and the spatial distance between genomic clusters. Topological skeletons were used to capture the spatial distribution for each genomic cluster and to determine the EEMS grid density; i.e., both genomic and spatial clustering were used to guide the optimization of EEMS grids. Model accuracy for migration estimates using the optimized workflow was tested and compared to grid resolutions selected without the optimized workflow. As a test case, the optimized workflow was applied to genomic data generated from <i>P. falciparum</i> sampled in Cambodia and bordering regions, and migration maps were compared to estimates of malaria endemicity, as well as geographic properties of the study area, as a means of validating observed migration patterns.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par3">Optimized grids displayed both high model accuracy and reduced computing time compared to grid densities selected in an unguided manner. In addition, EEMS migration maps generated for <i>P. falciparum</i> using the optimized grid corresponded to estimates of malaria endemicity and geographic properties of the study region that might be expected to impact malaria parasite migration, supporting the validity of the observed migration patterns.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par4">Optimized grids reduce spatial uncertainty in the EEMS contours that can result from user-defined parameters, such as the resolution of the spatial grid used in the model. This workflow will be useful to a broad range of EEMS users as it can be applied to analyses involving other organisms of interest and geographic areas.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"><i>Plasmodium falciparum</i></div> <div class="kwd" title="kwd">
Estimated effective migration surfaces</div> <div class="kwd" title="kwd">
Parasite migration</div> <div class="kwd" title="kwd">
Malaria elimination</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases of the National Institutes of Health</span></div><span class="award-id" tagx="award-id" title="award-id">U19AI129386</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plowe</span><span class="given-names" tagx="given-names" title="given-names">Christopher V.</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01-AI125579</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takala-Harrison</span><span class="given-names" tagx="given-names" title="given-names">Shannon</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par12">Understanding the genetic structure of natural populations provides insight into the demographic and adaptive processes that have affected those populations, such as migration or natural selection. Such information can have important applications in fields such as conservation biology or public health, particularly when integrated with geographic data. For example, geospatial modeling methods have been used to analyze pathogen genetic or genomic data to understand spatial transmission patterns of influenza virus [<a href="#CR1">1</a>–<a href="#CR4">4</a>] and typhoid fever [<a href="#CR5">5</a>], the sources of imported malaria [<a href="#CR6">6</a>] and dengue infections [<a href="#CR7">7</a>], and malaria parasite landscape genetics [<a href="#CR8">8</a>].</p> <p id="Par13">Often estimates of population structure are made without regard to the geographic coordinates of sampling locations and then later interpreted in the context of the geographic information. However, approaches have been developed that model both the spatial and genomic data. One such approach, called estimated effective migration surfaces (EEMS) [<a href="#CR9">9</a>, <a href="#CR10">10</a>], uses genomic data for a species to visualize the spatial contours of migration and diversity for this species for a given study area. The model broadly assumes isolation-by-distance, whereby genetic similarity and geographic distance are negatively correlated, and identifies areas where genetic similarity decays faster than expected for a given geographic distance (low effective migration) and areas where genetic similarity decays more slowly than expected for a given geographic distance (high effective migration). The model output is a map of areas of high and low effective migration or diversity for the study region. An assumption underlying EEMS is that the population structure is consistent with isolation-by-distance and as such, EEMS results represent effective (i.e., relative) rather than absolute migration rates and are likely to capture patterns associated with more historic timescales. It should be noted that the EEMS toolkit has also been expanded to support identity-by-descent approaches and estimations of migration and population-size surfaces for more recent time scales with the MAPS toolkit [<a href="#CR11">11</a>]. For a historic understanding of migration patterns in a region, EEMS is a useful tool and has been used, for example, to understand the population structure of human populations in southern [<a href="#CR12">12</a>] and eastern [<a href="#CR13">13</a>] Africa and in Europe [<a href="#CR14">14</a>], and to visualize barriers and corridors of gene flow associated with human migration in Scandinavia [<a href="#CR15">15</a>] and Peru [<a href="#CR16">16</a>]. EEMS has also been applied to simulate historical gene flow patterns for the gray wolf (<i>Canis lupus</i>) [<a href="#CR17">17</a>] and the blunt-nosed leopard lizard <i>Gambelia sila</i> [<a href="#CR18">18</a>], and to investigate the genetic diversity of Atlantic Bluefin tuna in the Mediterranean Sea [<a href="#CR19">19</a>]. In this paper, we use EEMS to estimate migration surfaces for <i>Plasmodium falciparum,</i> the deadliest human malaria species.</p> <p id="Par14">In our previous research, we have applied EEMS as well as approaches based on identity-by-descent to investigate migration patterns and population structure of <i>Plasmodium falciparum</i> in the Greater Mekong Subregion [<a href="#CR20">20</a>], an area of emerging multidrug resistance being targeted for malaria elimination [<a href="#CR21">21</a>]. EEMS maps are visually–intuitive and may be useful to malaria elimination programs by identifying defined geographic areas that can be targeted with interventions. However, to be useful for this purpose, it will be important to reduce spatial uncertainty in the EEMS contours that can arise from user-defined parameters, such as the resolution of the spatial grid used in the model. The spatial grid is a grid of regular triangles that covers the study area. Each vertex in the grid represents a <i>deme</i>, and EEMS uses the number of demes selected by a user to generate the spatial resolution of its spatial grids. The random selection of the number of demes can result in a high standard deviation among posterior distributions estimated using EEMS and in turn, higher levels of spatial uncertainty in the migration contours generated by EEMS. For example, if the grid is too sparse (i.e., relatively fewer demes), then many sampling locations may be assigned to a single deme, reducing model accuracy through excessive smoothing of genomic differences. On the other hand, if the grid is too dense (i.e. relatively high numbers of demes) spatial uncertainty may result from estimation of parameters for many demes lacking genomic data [<a href="#CR20">20</a>]. In addition, the number of demes included in the analysis has a substantial impact on computing time, with computing time scaling cubically with the number of demes. Researchers typically employ either an average of the results obtained from running multiple MCMC iterations using different numbers of demes to infer migration patterns [<a href="#CR9">9</a>, <a href="#CR17">17</a>, <a href="#CR22">22</a>, <a href="#CR23">23</a>], or apply maximum likelihood values to guide selection of the number of demes [<a href="#CR20">20</a>]. In both cases, users must run EEMS at multiple grid resolutions, which can be time-consuming.</p> <p id="Par15">Here we present an approach that utilizes a density clustering algorithm to define genomic clusters, which are then used to determine the optimal maximum length of triangle edges and grid resolution. This workflow provides a systematic method to select the optimal number of demes that will maximize model accuracy and minimize computing time. We tested the optimized workflow by applying it to estimate geospatial patterns of <i>Plasmodium falciparum</i> migration in Cambodia and bordering regions of Thailand and Vietnam, and found that migration contours corresponded to estimates of malaria endemicity and geographic properties of the region that might be expected to impact malaria parasite migration.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="studyareaanddatacollection" title="sec"> <div class="title" tagx="title" title="title">
Study area and data collection</div> <p id="Par16">Our approach was tested on a subset of the <i>P. falciparum</i> genomic data from our previous publication [<a href="#CR20">20</a>], including 28,496 biallelic, genome-wide SNPs from 1007 samples collected in 35 districts in Cambodia and 8 bordering districts of Thailand and Vietnam between 2008 and 2013 (Fig. <a href="#Fig1">1</a>) [<a href="#CR24">24</a>–<a href="#CR28">28</a>]. SNPs were either called from whole genome sequences generated as part of the MalariaGEN <i>Plasmodium falciparum</i> Community Project [<a href="#CR29">29</a>], or, for samples that did not meet quality control criteria for whole genome sequencing or were not part of the Community Project, were genotyped using a <i>P. falciparum</i>-specific Nimblegen DNA microarray [<a href="#CR30">30</a>] (NIH Gene Expression Omnibus, Accession number: GSE100704. European Variant Archive, Accession PRJEB28530). The same nucleotide positions typed on the microarray were extracted from whole-genome data for analysis, with missingness cut-offs applied as previously described [<a href="#CR20">20</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Sampling locations in Cambodia, Thailand and Vietnam</p>   </div> <p /> </div> <div class="computingestimatedeffectivemigrationsurfaces" title="sec"> <div class="title" tagx="title" title="title">
Computing estimated effective migration surfaces</div> <p id="Par17">EEMS utilizes a grid of regular triangles that cover the study area. The grid is created using two user-defined parameters, a bounding box that defines the geographic area where gene flow will be modeled, and the number of <i>demes</i>, where a deme represents a vertex in the grid. The EEMS toolkit allows the number of demes to vary up to a maximum value of 1000, allowing for different grid resolutions. Genomic data from a given sampling location is assigned to the nearest deme, and the model uses the deme locations to estimate and map effective migration surfaces. EEMS assumes individuals migrate locally between subpopulations (at demes) and that migration rates vary by location. The model also assumes that each subpopulation exchanges migrants only with its neighbors (i.e., a stepping-stone model). For every triangle in the grid, EEMS assigns diversity estimates to demes and migration estimates to triangle edges. Markov chain Monte Carlo (MCMC) methods are employed to estimate both migration and diversity parameters by sampling from their posterior distributions given observed genetic dissimilarities. The matrix of average pairwise genetic dissimilarities is computed using <i>bed2diffs</i>, and EEMS is then run using <i>runeems_snps</i>: a C ++ implementation of EEMS for SNP data.</p> <p id="Par18">For each iteration, two sets of Voronoi tessellations are generated, representing spatial patterns of migration and diversity respectively. These tessellations are generated based on a user-defined value <i>nseeds</i> that represents the number of Voronoi cells and that is also assigned to the grids. The two Voronoi tessellations are independent of each other and are updated with a birth/death process since the number of Voronoi cells is initially unknown. A maximum likelihood method is commonly used to adjust estimates of diversity and migration so that simulated genetic dissimilarity rates fit observed genetic dissimilarity in both cases.</p> </div> <div class="clusteringbasedonthedistributionofp.falciparumgenomicdata" title="sec"> <div class="title" tagx="title" title="title">
Clustering based on the distribution of <i>P. falciparum</i> genomic data</div> <p id="Par19">To optimize the grid, we first applied a density clustering algorithm to define clusters based on parasite genomic data [<a href="#CR31">31</a>]. Clustering was performed using <i>densityClust</i>, an algorithm that is an improvement on K-means clustering, and is available as an open source package in R [<a href="#CR32">32</a>]. This clustering method did not require prior knowledge of the desired number of clusters and assumed that cluster centers were distinct from points with higher local density and were surrounded by points with low local density. Clustering was performed using the matrix of pairwise genetic dissimilarities generated through the EEMS toolkit, as a measure of genetic distance. A decision graph (Fig. <a href="#Fig2">2</a>a) and a multidimensional scaling graph (Fig. <a href="#Fig2">2</a>b) were generated for all the samples, where the x-axis represented the local density p<sub><i>i</i></sub> of sample <i>i</i> and the y-axis represented the genetic distance from the nearest points with a higher density δ<sub>i</sub>. The local density p<sub><i>i</i></sub> was defined as: </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p><b>a</b> Decision graphs generated from clustering by fast search and find of density peaks [<a href="#CR31">31</a>] and <b>b</b> multidimensional scaling graph generated from extracting density peaks</p>   </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">1</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {\text{p}}_{i} = \mathop \sum \limits_{j} \chi \left( {d_{ij} - d_{c} } \right) $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mi_UNKNOWN">
χ</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ij</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
c</div> </div> </div> </div> </div></div><div class="graphic_UNKNOWN" position="anchor" /> </div> </div>where <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \chi \left( x \right) = 1 $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
χ</div> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span> if <i>x </i>&amp;lt; 0 and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ \chi \left( x \right) = 0 $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
χ</div> <div class="mfenced_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span> otherwise. <i>d</i><sub>c</sub> was the cutoff distance and p<sub><i>i</i></sub> was equal to the number of samples that were closer than <i>d</i><sub>c</sub> to sample <i>i</i>. δ<sub>i</sub> was calculated by measuring the minimum distance between the sample <i>i</i> and other samples with higher density: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">2</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$ {{\delta }}_{i} = \mathop {\text{min} }\limits_{{j:p_{j} &amp;gt; p_{i} }} \left( {d_{ij} } \right) $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mtext_UNKNOWN">
min</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
:</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ij</div> </div> </div> </div> </div></div><div class="graphic_UNKNOWN" position="anchor" /> </div> </div> <p /> <p id="Par21">For the sample with the highest density, we assumed δ<sub>i</sub> = max<sub>j</sub>(<i>d</i><sub>ij</sub>), where δ<sub>i</sub> would be much greater than the typical nearest neighbor distance only for samples that with the local or global maxima density and cluster centers were recognized as samples for which the δ<sub>i</sub> value was anomalously large.</p> <p id="Par22">The output of the decision graph was used to confirm the number of genomic clusters. Points located in the upper-right quadrant distant from the other points are more likely to be cluster centers (Fig. <a href="#Fig2">2</a>a). All points were treated as cluster centers as long as their <i>p</i><sub><i>i</i></sub> and δ<sub>i</sub> were higher than the mean <i>p</i><sub><i>i</i></sub> and δ<sub>i</sub> value. After the number of cluster centers and hence, the number of clusters was determined, all the samples were assigned to a cluster based on genetic similarity (Fig. <a href="#Fig2">2</a>b).</p> </div> <div class="computingthelengthoftriangleedgesanddeterminingthenumberofdemes" title="sec"> <div class="title" tagx="title" title="title">
Computing the length of triangle edges and determining the number of demes</div> <p id="Par23">The genomic clustering results were used to determine the optimal maximum length of triangle edges. For each cluster, a kernel density map was generated based on sample sizes and locations. A natural breaks classification was used to group each kernel density map into binary categories of high and low sample density, respectively. This classification method minimized the average deviation from the class mean and maximized the deviation from the means of the other groups. It also reduced the variance within classes while maximizing the variance between classes. The category representing the highest sample density was selected to determine the spatial distribution of boundaries that represent each cluster and from which triangle lengths could be established. Since the cluster polygons were often irregularly shaped, the topological skeletons of polygons were used to capture polygon shapes. To represent the spatial distance between clusters, the nearest distance between each pair of topological skeletons was computed. Finally, the shortest distance between each pair of clusters was determined and selected to represent the <i>maximum length</i> of a triangle edge in the grid. To determine the number of demes that optimizes the grid, this optimized triangle edge length was used as input in an inverse function to the EEMS grid generation function with the result that the edge length of the generated EEMS grid is shorter than the optimized triangle edge length. In this way, both genomic and spatial clustering were used to guide the optimization of triangle sizes and the density of demes, i.e., grid resolution, used for generating <i>P. falciparum</i> parasite migration maps.</p> </div> <div class="evaluationofmodelaccuracy" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of model accuracy</div> <p id="Par24">While the EEMS toolkit can be used to generate both migration and diversity maps for a bounded region [<a href="#CR9">9</a>], in this study, we focused particularly on migration maps, using genomic data from <i>P. falciparum</i>. The EEMS toolkit allows the generation of scatterplots to visualize the correlation between observed and fitted genetic dissimilarity between demes to determine model accuracy for migration maps. Such scatterplots have been used by researchers to evaluate the model accuracy of EEMS contours [<a href="#CR33">33</a>–<a href="#CR36">36</a>]. For each grid density, R<sup>2</sup> was estimated and compared to determine how migration model accuracy varied by the number of demes.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par25">Applying density clustering to the <i>P. falciparum</i> genomic data from Cambodia and bordering sites in Thailand and Vietnam, the decision graph identified five genomic clusters (Fig. <a href="#Fig3">3</a>a). Kernel density analysis was applied to generate a map of these clusters and showed the five genomic clusters occupied six different locations (Fig. <a href="#Fig3">3</a>b). The six locations included, (1) northwestern Cambodia bordering Thailand (Oddar Meanchey and Preah Vihear Provinces), (2) western Pailin Province on the eastern border with Thailand, (3) south of Tonle Sap Lake in Pursat Province, (4) the adjacent region in southeastern Koh Kong Province, southwestern Kampong Speu Province and the northern part of Kampot Province, (5) eastern Cambodia in an area that overlapped southern Ratanakiri Province and northern Mondulkiri Province, and (6) Bu Dop district, Vietnam (Fig. <a href="#Fig3">3</a>b). In Fig. <a href="#Fig3">3</a>a, one genomic cluster (colored red) was identified in all six locations, while another cluster (cyan) was found in only two locations, namely the Pailin District in western Cambodia and northern Bu Gia Map National Park in Vietnam. All five genomic clusters were found in Pailin District, whereas only one genomic cluster (red) was found in eastern Cambodia in the area where Ratanakiri Province borders Mondulkiri Province (Fig. <a href="#Fig3">3</a>b). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Density-based clustering of genomic data where <b>a</b> five genomic clusters that were detected using the decision graph corresponded geographically to <b>b</b> six geographic locations (five locations in Cambodia and one in Vietnam)</p>   </div> <p /> <div class="generatinganoptimizedgridforthestudyarea" title="sec"> <div class="title" tagx="title" title="title">
Generating an optimized grid for the study area</div> <p id="Par26">The shortest distance between cluster centers computed using the topological skeletons of the clusters was 32.6 km, representing the maximum edge length for each triangle in the EEMS grid. The corresponding number of demes was calculated by setting the longest edge length to this value and using the inverse function as described above. Using our workflow, the optimized grid contained 350 demes (Additional file <a href="#MOESM1">1</a>: Figure S1). The MCMC iteration using this grid resolution was 30 million; burn-in was 29 million; and thinning iteration was 9999. The running time was approximately 13 h using 64 CPUs on a Linux high-performance network.</p> </div> <div class="evaluatingtheoptimizedgrid" title="sec"> <div class="title" tagx="title" title="title">
Evaluating the optimized grid</div> <p id="Par27">We investigated migration model accuracy for the optimized grid by examining scatterplots of the observed genetic dissimilarity between demes versus the fitted genetic dissimilarity between demes. The fitted genetic dissimilarly was calculated based on the computed migration between pairs of deme locations. The scatter plots showed a strong linear relationship (R<sup>2</sup> value was 0.757) between observed and fitted dissimilarity (Fig. <a href="#Fig4">4</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>EEMS model accuracy for between-demes using the optimized grid</p>   </div> <p /> <p id="Par28">To evaluate our optimization strategy, we used EEMS to generate migration surfaces for a range of grid resolutions from 200 to 750 demes, and compared the model accuracy and computing times for these grids with the optimized grid (Fig. <a href="#Fig5">5</a>). Using a grid of 200 demes had the poorest performance with R<sup>2</sup> = 0.38 (Fig. <a href="#Fig5">5</a>a). A grid of 400 demes also had a slightly lower R<sup>2</sup> value of 0.748 compared to the optimized grid (R2 = 0.757). And while R<sup>2</sup> values appeared to increase for grids with more than 550 demes, these cases were associated with excessive computing times (Fig. <a href="#Fig5">5</a>b). The computing time for the 350-demes case (approximately 13 h) was much less than for 400 demes (28 h), 500 demes (39 h), 600 demes (73 h) and 700 demes (112 h). Running the analyses with different grid resolutions indicated that optimizing the number of demes offered the best performance for migration estimates with a significantly reduced computing time. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Model performance for 200 to 700 demes for <b>a</b> model accuracy (R<sup>2</sup> value for migration model accuracy) and <b>b</b> computation time</p>   </div> <p /> </div> <div class="effectivemigrationsurfacesusingtheoptimizedgrid" title="sec"> <div class="title" tagx="title" title="title">
Effective migration surfaces using the optimized grid</div> <p id="Par29">The optimized triangular grid was used to generate an estimated effective migration surface using parasite genomic data generated from isolates collected in Cambodia and surrounding locations. The migration contours in the resulting migration map (where blue indicates relative high migration and brown indicates lower migration) showed the lowest migration index value was near Tonle Sap Lake, which is the largest inland lake in Cambodia with an area of over 12,876 sq.km, while Koh Kong Province in the southwest showed the highest migration (Fig. <a href="#Fig6">6</a>). Southwest Cambodia (southern Koh Kong, southern Kampong Speu, Sihanoukville, Kampot, and Takeo Provinces) in general showed high migration relative to other locations, and the border area between eastern Cambodia and Vietnam was also associated with higher migration while locations in the border area of northwest Cambodia and Thailand showed lower parasite migration estimates. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Estimated migration surface of <i>P. falciparum</i> parasites in Cambodia using the optimized grid</p>   </div> <p /> </div> <div class="comparisonbetweenestimatedeffectivemigrationsurfacesandp.falciparumendemicityandannualparasiteincidenceincambodia" title="sec"> <div class="title" tagx="title" title="title">
Comparison between estimated effective migration surfaces and <i>P. falciparum</i> endemicity and annual parasite incidence in Cambodia</div> <p id="Par30">We compared the migration surface results generated using the spatially optimized grid (Fig. <a href="#Fig6">6</a>) with a <i>P. falciparum</i> endemicity map based on <i>P. falciparum</i> parasite rate (PfPR) data from 2010 made available through the Malaria Atlas Project [<a href="#CR37">37</a>] (Fig. <a href="#Fig7">7</a>), and also compared migration estimates with estimates of annual parasite incidence (API) per 1000 for 2013 [<a href="#CR38">38</a>]. The area of low migration near Tonle Sap Lake coincided with low P. <i>falciparum</i> endemicity. Areas with high <i>P. falciparum</i> migration in both southwestern and northeastern Cambodia were found to match regions with a relatively high prevalence of <i>P. falciparum</i>. This close relationship between <i>P. falciparum</i> migration and endemicity may imply the conditions in these locations are suitable for transmission of <i>P. falciparum</i> within these regions in Cambodia. High <i>P. falciparum</i> migration in northeastern Cambodia was consistent with high API values greater than 20% in the Steung Treng and Ratanakiri Provinces. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p><i>P. falciparum</i> endemicity patterns within Cambodia from the Malaria Atlas Project (Data downloaded from <a href="https://map.ox.ac.uk/">https://map.ox.ac.uk/</a>) (2010)</p>   </div> <p /> <p id="Par31">We also compared <i>P. falciparum</i> migration contours with data available from OpenDevelopment Cambodia on natural protected areas [<a href="#CR39">39</a>] and found that our results using EEMS corresponded to landcover features in a way we might expect, i.e., contours of high migration coincided with areas having mostly forest landcover (Additional file <a href="#MOESM1">1</a>: Figure S1). For example, high <i>P. falciparum</i> migration in the northeastern and eastern regions of Cambodia (Fig. <a href="#Fig7">7</a>), corresponded to heavily forested areas, including a large National Park in Ratanakiri Province as well as four large wildlife sanctuaries in Mondulkiri and Ratanakiri provinces that could have served as habitats for <i>Anopheles</i> mosquitoes. Another area of high <i>P. falciparum</i> migration was located along the border between northwestern Cambodia and Thailand, and within northeastern Kampong Thom and south of Preah Vihear Province (Fig. <a href="#Fig7">7</a>) that were also forested with wildlife sanctuaries and protected natural habitats (Additional file <a href="#MOESM2">2</a>: Figure S2). Another area of higher migration was observed in Battambang and Pursat provinces, just west of Tonle Sap Lake. Low migration patterns were more notable in western Cambodia, and a region of low <i>P. falciparum</i> migration extended from southern Laos, across the center of Cambodia, to southern Vietnam.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par32">In this study, we developed a framework based on both genomic and spatial clustering to select the optimal number of demes to reduce spatial uncertainty in EEMS migration maps, in the presence of irregular sampling. EEMS migration contours can vary–sometimes substantially–based on the selected number of demes. Therefore, having a systematic, rational approach to determine grid density will likely be helpful to EEMS users. We were able to test and show that optimized grids displayed both high model accuracy and reduced processing time compared to grid densities selected in an unguided manner. In addition, when we utilized an optimized grid to generate EEMS migration maps for <i>P. falciparum,</i> we found that migration contours reflecting the parasite population structure corresponded to estimates of malaria endemicity and geographic properties of the study region (e.g., landcover and large waterbodies) that might be expected to impact malaria parasite migration.</p> <p id="Par33">The results of our genomic clustering approach indicated the presence of multiple genomic clusters based on malaria parasite genomic data generated from isolates collected in provinces in western, northwestern, and southwestern Cambodia. This finding is consistent with previous analyses of subsets of these data that found multiple sympatric genetic subpopulations of parasites that were hypothesized to have originated as founder populations resulting from the emergence of artemisinin resistance [<a href="#CR40">40</a>–<a href="#CR42">42</a>]. The congruence of these findings suggests that the density clustering approach applied in our optimization framework is accurately capturing known patterns of parasite genetic diversity in the study area.</p> <p id="Par34">Our results indicated overlap between areas of high <i>P. falciparum</i> migration and hotspots of malaria incidence in eastern Cambodia [<a href="#CR43">43</a>] [<a href="#CR44">44</a>–<a href="#CR46">46</a>], as well as other areas of high malaria endemicity [<a href="#CR37">37</a>, <a href="#CR47">47</a>]. In Battambang and Pursat Provinces, high <i>P. falciparum</i> migration could result from flooding of the forests around Tonle Sap Lake during the wet season [<a href="#CR48">48</a>], providing habitats for malaria vectors [<a href="#CR49">49</a>]. In Pursat and Preah Vihear Provinces, high <i>P. falciparum</i> migration areas coincided with a high prevalence of multidrug resistance that is known to have emerged and spread in the area during this time frame [<a href="#CR50">50</a>].</p> <p id="Par35">Migration maps generated in this study corroborated major migration barriers for <i>P. falciparum</i> identified in our previous study [<a href="#CR20">20</a>]. However, use of the optimized grid allowed detection of a migration barrier in Pailin Province that was not identified in our previous analysis that is consistent with malaria elimination efforts in this area that have contributed to a dramatic decline in clinical malaria incidence [<a href="#CR51">51</a>–<a href="#CR53">53</a>]. <i>P. falciparum</i> migration barriers in northern Cambodia may have been due to higher urbanization (lower vegetation coverage) north of Tonle Sap Lake as well as Tonle Sap Lake itself, which is a large enough waterbody that it may have served as a barrier to <i>P. falciparum</i> migration. Deforestation of cardamom forests and large-scale land acquisitions in the area corresponded to the southern part of the ring-like contour of low migration and may also have been a contributor to this migration barrier [<a href="#CR54">54</a>, <a href="#CR55">55</a>]. The Mekong River running through southeastern Cambodia as well as the urbanized area of Phnom Penh, may both have contributed to reduced parasite migration in this part of Cambodia. The fact that detected migration hotspots and barriers were geographically related with landcover and hydrologic features underscores the role that geography plays in shaping parasite population structure, which is consistent with findings from other studies, for example, a major migration barrier was detected around the Andes Mountains in Peru [<a href="#CR56">56</a>].</p> </div> <div class="limitationsandfuturework" title="sec"> <div class="title" tagx="title" title="title">
Limitations and future work</div> <p id="Par36">Cambodia may be unique with respect to its patterns of parasite genetics due to multiple selection events of antimalarial drug resistance in the GMS region. This phenomenon is apparent through the overlapping genomic clusters, and ongoing work is addressing how selection of drug resistance may impact migration patterns observed in this area.</p> <p id="Par37">Future research will investigate how the spatial granularity of sampling may contribute to uncertainty in EEMS migration maps. For example, data from parasite isolates in this study were geolocated at the district level, which could lead to spatial uncertainty based on aggregation of multiple locations into a single location. Improving local geographical granularity may aid in detecting more detailed migration patterns. Further investigation is also required to improve our understanding of any boundary effects in EEMS analyses, as well as the impact of the assumption of geographic uniformity across a study area implied by use of a uniform grid, since geographic uniformity is not assumed for all studies [<a href="#CR57">57</a>], and is an assumption that is likely violated in many settings, including in studies of the malaria parasite as presented here.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par38">We have developed a semi-automatic workflow that used both genomic and spatial clustering to guide the optimization of triangle sizes and the density of demes, i.e., grid resolution, to generate effective migration surfaces for <i>P. falciparum</i> migration. Computing the analyses using different grid resolutions indicated that optimizing the number of demes offered the best performance for producing migration estimates with a significantly reduced computing time, an important consideration if maps are to be used to guide intervention strategies. We tested the optimized EEMS workflow on data generated from parasite isolates collected in Cambodia and bordering regions of Thailand and Vietnam, and found that migration contours corresponded to estimates of malaria endemicity and geographic features in the region that might be expected to impact malaria parasite migration (e.g., landcover, large waterbodies), supporting the validity of EEMS migration estimates. While in this study, our optimization framework was applied to malaria parasites, we believe this workflow is generalizable for other study areas and pathogens and can be used to guide the generation of migration maps based on available genomic sample distributions.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="12942_2020_207_MOESM1_ESM.jpg">LINK</a> <p><b>Additional file</b><a href="#MOESM1"><b>1</b></a><b>: Figure S1.</b> Optimized grid generated using the computed maximum triangle edge length.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="12942_2020_207_MOESM2_ESM.jpg">LINK</a> <p><b>Additional file</b><a href="#MOESM2"><b>2</b></a><b>: Figure S2.</b> Environmental features including protected areas, national parks, wildlife sanctuaries, and locations of waterbodies in Cambodia (2013).</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
EEMS</div> <div class="def" title="def"> <p id="Par5">Estimated effective migration surfaces</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GMS</div> <div class="def" title="def"> <p id="Par6">Greater Mekong Subregion</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LISA</div> <div class="def" title="def"> <p id="Par7">Local indicators of spatial autocorrelation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SNPs</div> <div class="def" title="def"> <p id="Par8">Single nucleotide polymorphisms</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MCMC</div> <div class="def" title="def"> <p id="Par9">Markov chain Monte Carlo</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
P.f</div> <div class="def" title="def"> <p id="Par10"><i>Plasmodium falciparum</i></p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
P.v</div> <div class="def" title="def"> <p id="Par11"><i>Plasmodium vivax</i></p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p><b>Supplementary information</b> accompanies this paper at 10.1186/s12942-020-00207-3.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We would like to thank the participants in the MalariaGEN <i>P. falciparum</i> Community Project, including study participants and investigators who took part in the ARC3, ARCE, and TRAC collaborations. We would also like to thank Christopher Jacob and Sonia Agrawal for generation of SNP array data and for quality control of the data.</p> <div class="disclaimer" title="sec"> <div class="title" tagx="title" title="title">
Disclaimer</div> <p id="Par39">The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. Material has been reviewed by the Walter Reed Army Institute of Research. There is no objection to its presentation and/or publication. The opinions or assertions contained herein are the private views of the author, and are not to be construed as official, or as reflecting true views of the Department of the Army or the Department of Defense. Research was conducted under an approved animal use protocol in an AAALACi accredited facility in compliance with the Animal Welfare Act and other federal statutes and regulations relating to animals and experiments involving animals and adheres to principles stated in the Guide for the Care and Use of Laboratory Animals, NRC Publication, 2011 edition. The investigators have adhered to the policies for protection of human subjects as prescribed in AR 70–25.</p> </div> </div> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>Designed the experiments: YL and KS. Collected samples: DLS, CL, MS, MMF, TTH, SP, RMF and AMD. Analyzed data: YL, STH, ACS and TDO. Interpreted the data: YL, KS, STH, CVP and TDO. Wrote the manuscript: YL, KS, STH and TDO. All authors read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>Research reported in this publication was supported by the National Institute of Allergy and Infectious Diseases of the National Institutes of Health under Award Number U19AI129386 for KES and CVP, NIH R01-AI101713 and R01-AI125579 for ST-H, and NIH R01AI145852 for ST-H, TDO, and KES.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>Genotyping data are publicly available through the MalariaGEN website (<a href="https://www.malariagen.net/data/p-falciparum-community-project-jan-2016-data-release">https://www.malariagen.net/data/p-falciparum-community-project-jan-2016-data-release</a>) or through the NIH Gene Expression Omnibus (<a href="http://www.ncbi.nlm.gov/geo/">www.ncbi.nlm.gov/geo/</a>) (Accession number: GSE100704) and European Variation Archive (Accession number: PRJEB28530) [<a href="#CR20">20</a>].</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par40">All samples from which parasite genomic data were generated were collected with prior approval from the local ethical review boards for the institutions that conducted the studies. Parasite genotyping and genomic analyses were undertaken after prior approval of the University of Maryland School of Medicine Institutional Review Board.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par41">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par42">The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrel</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jobe</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moody</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">X-F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spatiotemporal structure of molecular evolution of H5N1 highly pathogenic avian influenza viruses in Vietnam</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e8631</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0008631">10.1371/journal.pone.0008631</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20072619">20072619</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TTY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ip</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halpin</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stockwell</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Migratory flyway and geographical distance are barriers to the gene flow of influenza virus among North American birds</span><span class="source" tagx="source" title="source">Ecol Lett</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">24</span><span class="lpage" tagx="lpage" title="lpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2011.01703.x">10.1111/j.1461-0248.2011.01703.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22008513">22008513</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza A H5N1 immigration is filtered out at some international borders</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e1697</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0001697">10.1371/journal.pone.0001697</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18301773">18301773</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">X-F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Highly pathogenic H5N1 avian influenza viruses exhibit few barriers to gene flow in Vietnam</span><span class="source" tagx="source" title="source">Eco Health</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">60</span><span class="lpage" tagx="lpage" title="lpage">69</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22350419">22350419</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clements</span><span class="given-names" tagx="given-names" title="given-names">ACA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karkey</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arjyal</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Combined high-resolution genotyping and geospatial analysis reveals modes of endemic urban typhoid fever transmission</span><span class="source" tagx="source" title="source">Open Biol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">110008</span><span class="lpage" tagx="lpage" title="lpage">110008</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsob.110008">10.1098/rsob.110008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22645647">22645647</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">H-H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wesolowski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinha</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacob</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahmud</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mapping imported malaria in Bangladesh using parasite genetic and human mobility data</span><span class="source" tagx="source" title="source">eLife</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e43481</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.43481">10.7554/eLife.43481</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30938289">30938289</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raghwani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hang</span><span class="given-names" tagx="given-names" title="given-names">VT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hien</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrar</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Endemic dengue associated with the co-circulation of multiple viral lineages and localized density-dependent transmission</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e1002064</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002064">10.1371/journal.ppat.1002064</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21655108">21655108</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janko</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mwandagalirwa</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tshefu</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The geography of malaria genetics in the Democratic Republic of Congo: a complex and fragmented landscape</span><span class="source" tagx="source" title="source">Soc Sci Med</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">133</span><span class="fpage" tagx="fpage" title="fpage">233</span><span class="lpage" tagx="lpage" title="lpage">241</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.socscimed.2014.10.037">10.1016/j.socscimed.2014.10.037</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25459204">25459204</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petkova</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novembre</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Visualizing spatial population structure with estimated effective migration surfaces</span><span class="source" tagx="source" title="source">Nat Genet</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">94</span><span class="lpage" tagx="lpage" title="lpage">100</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.3464">10.1038/ng.3464</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26642242">26642242</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><a href="https://github.com/dipetkov/eems">https://github.com/dipetkov/eems</a>. 2018.</span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Asadi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petkova</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novembre</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimating recent migration and population-size surfaces</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">e1007908</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1007908">10.1371/journal.pgen.1007908</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30640906">30640906</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uren</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bobo</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gignoux</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Helden</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Fine-scale human population structure in southern Africa reflects ecogeographic boundaries</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">204</span><span class="fpage" tagx="fpage" title="fpage">303</span><span class="lpage" tagx="lpage" title="lpage">314</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.116.187369">10.1534/genetics.116.187369</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27474727">27474727</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brucato</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandes</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazières</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusuma</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wainaina Nganga</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The Comoros show the earliest Austronesian gene flow into the Swahili corridor</span><span class="source" tagx="source" title="source">Am J Hum Genet</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">58</span><span class="lpage" tagx="lpage" title="lpage">68</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajhg.2017.11.011">10.1016/j.ajhg.2017.11.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29304377">29304377</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathieson</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alpaslan-Roodenberg</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posth</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szécsényi-Nagy</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohland</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The genomic history of southeastern Europe</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">555</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature25778">10.1038/nature25778</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29466330">29466330</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karczewski</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerminen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurki</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarin</span><span class="given-names" tagx="given-names" title="given-names">A-P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Artomov</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Haplotype sharing provides insights into fine-scale population history and disease in Finland</span><span class="source" tagx="source" title="source">Am J Hum Genet</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">760</span><span class="lpage" tagx="lpage" title="lpage">775</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajhg.2018.03.003">10.1016/j.ajhg.2018.03.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29706349">29706349</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">DN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shetty</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levano</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cáceres</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padilla</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary genomic dynamics of Peruvians before, during, and after the Inca Empire</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">e6526</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1720798115">10.1073/pnas.1720798115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29946025">29946025</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rick</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moen</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erb</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strasburg</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Population structure and gene flow in a newly harvested gray wolf (<i>Canis lupus)</i> population</span><span class="source" tagx="source" title="source">Conserv Genet</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10592-017-0961-7">10.1007/s10592-017-0961-7</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richmond</span><span class="given-names" tagx="given-names" title="given-names">JQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westphal</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandergast</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leaché</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saslaw</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Persistence of historical population structure in an endangered species despite near-complete biome conversion in California’s San Joaquin desert</span><span class="source" tagx="source" title="source">Mol Ecol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">3618</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.14125">10.1111/mec.14125</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28370723">28370723</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antoniou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasapidis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotoulas</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mylonas</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magoulas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic diversity of Atlantic Bluefin tuna in the Mediterranean Sea: insights from genome-wide SNPs and microsatellites</span><span class="source" tagx="source" title="source">J Biol Res-Thessalon</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40709-017-0062-2">10.1186/s40709-017-0062-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28239596">28239596</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shetty</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacob</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agrawal</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saunders</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic structure and diversity of <i>Plasmodium falciparum</i> in Southeast Asia reveal recent parasite migration patterns</span><span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">2665</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-019-10121-3">10.1038/s41467-019-10121-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31209259">31209259</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">World Health Organization. Strategy for Malaria Elimination in the Greater Mekong Subregion (2015–2030). 2015.</span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsuda</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stocks</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Källman</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sønstebø</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parducci</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The extent and meaning of hybridization and introgression between Siberian spruce (<i>Picea obovata</i>) and Norway spruce (<i>Picea abies</i>): cryptic refugia as stepping stones to the west?</span><span class="source" tagx="source" title="source">Mol Ecol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2773</span><span class="lpage" tagx="lpage" title="lpage">2789</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.13654">10.1111/mec.13654</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27087633">27087633</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gopalan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berl</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belbin</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gignoux</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feldman</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hewlett</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hunter-gatherer genomes reveal diverse demographic trajectories following the rise of farming in East Africa</span><span class="source" tagx="source" title="source">Biorxiv</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/517730">10.1101/517730</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dondorp</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nosten</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phyo</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tarning</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Artemisinin resistance in <i>Plasmodium falciparum</i> malaria</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">361</span><span class="fpage" tagx="fpage" title="fpage">455</span><span class="lpage" tagx="lpage" title="lpage">467</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa0808859">10.1056/NEJMoa0808859</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19641202">19641202</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashley</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhorda</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fairhurst</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaratunga</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Spread of artemisinin resistance in <i>Plasmodium falciparum</i> malaria</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">371</span><span class="fpage" tagx="fpage" title="fpage">411</span><span class="lpage" tagx="lpage" title="lpage">423</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa1314981">10.1056/NEJMoa1314981</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25075834">25075834</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaorattanakawee</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saunders</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sea</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chanarat</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yingyuen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sundrakes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ex vivo drug susceptibility and molecular profiling of clinical Plasmodium falciparum isolates from Cambodia in suggest emerging piperaquine resistance</span><span class="source" tagx="source" title="source">Antimicrob Agents Chemother.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">59</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">4631</span><span class="lpage" tagx="lpage" title="lpage">4643</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AAC.00366-15">10.1128/AAC.00366-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26014942">26014942</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hien</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thuy-Nhien</span><span class="given-names" tagx="given-names" title="given-names">NT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phu</span><span class="given-names" tagx="given-names" title="given-names">NH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thanh</span><span class="given-names" tagx="given-names" title="given-names">NV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nha-Ca</span><span class="given-names" tagx="given-names" title="given-names">NT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">In vivo susceptibility of <i>Plasmodium falciparum</i> to artesunate in Binh Phuoc Province</span><span class="source" tagx="source" title="source">Vietnam Malar J</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">355</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2875-11-355">10.1186/1475-2875-11-355</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23101492">23101492</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bethell</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Se</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lon</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyner</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saunders</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sriwichai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Artesunate dose escalation for the treatment of uncomplicated malaria in a region of reported artemisinin resistance: a randomized clinical trial</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e19283</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0019283">10.1371/journal.pone.0019283</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21603629">21603629</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">MalariaGEN Plasmodium falciparum Community Project. Genomic epidemiology of artemisinin resistant malaria. elife. 2016;5:e08714.</span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacob</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takala-Harrison</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferdig</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A microarray platform and novel SNP calling algorithm to evaluate <i>Plasmodium falciparum</i> field samples of low DNA quantity</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">719</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-15-719">10.1186/1471-2164-15-719</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25159520">25159520</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laio</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clustering by fast search and find of density peaks</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">344</span><span class="fpage" tagx="fpage" title="fpage">1492</span><span class="lpage" tagx="lpage" title="lpage">1496</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1242072">10.1126/science.1242072</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24970081">24970081</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Thomas Lin Pedersen. densityClust [Internet]. 2015. <a href="https://github.com/thomasp85/densityClust">https://github.com/thomasp85/densityClust</a>.</span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sherpa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blum</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capblancq</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cumer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rioux</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Després</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Unravelling the invasion history of the Asian tiger mosquito in Europe</span><span class="source" tagx="source" title="source">Mol Ecol</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">2360</span><span class="lpage" tagx="lpage" title="lpage">2377</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30849200">30849200</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">KO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Linking patterns of genetic variation to processes of diversification in Malaysian torrent frogs (Anura: Ranidae: Amolops): a landscape genomics approach</span><span class="source" tagx="source" title="source">BioRxiv</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/628891">10.1101/628891</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berv</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campagna</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feo</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro-Astor</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribas</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prum</span><span class="given-names" tagx="given-names" title="given-names">RO</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic phylogeography of the White Crowned Manakin <i>Pseudopipra pipra</i> (Aves: Pipridae) illuminates a continental-scale radiation out of the Andes</span><span class="source" tagx="source" title="source">BioRxiv</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/713081">10.1101/713081</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pimenta</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopes</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carracedo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arenas</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amorim</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comas</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spatially explicit analysis reveals complex human genetic gradients in the Iberian Peninsula</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-44121-6">10.1038/s41598-019-44121-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30626917">30626917</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gething</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patil</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guerra</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elyazar</span><span class="given-names" tagx="given-names" title="given-names">IR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnston</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A new world malaria map: <i>Plasmodium falciparum</i> endemicity in 2010</span><span class="source" tagx="source" title="source">Malar J</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">378</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2875-10-378">10.1186/1475-2875-10-378</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22185615">22185615</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maude</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguon</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ly</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bunkea</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ngor</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torre</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Spatial and temporal epidemiology of clinical malaria in Cambodia 2004–2013</span><span class="source" tagx="source" title="source">Malar J</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">385</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2875-13-385">10.1186/1475-2875-13-385</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25266007">25266007</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">OpenDevelopment Cambodia. OpenDevelopment Cambodia. 2016.</span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miotto</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almagro-Garcia</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manske</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacInnis</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campino</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rockett</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple populations of artemisinin-resistant <i>Plasmodium falciparum</i> in Cambodia</span><span class="source" tagx="source" title="source">Nat Genet</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">648</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.2624">10.1038/ng.2624</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624527">23624527</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miotto</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amato</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashley</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacInnis</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almagro-Garcia</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaratunga</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic architecture of artemisinin-resistant <i>Plasmodium falciparum</i></span><span class="source" tagx="source" title="source">Nat Genet</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">226</span><span class="lpage" tagx="lpage" title="lpage">234</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.3189">10.1038/ng.3189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25599401">25599401</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agrawal</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moser</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morton</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parihar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwivedi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Association of a novel mutation in the <i>Plasmodium falciparum</i> chloroquine resistance transporter with decreased piperaquine sensitivity</span><span class="source" tagx="source" title="source">J Infect Dis</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">216</span><span class="fpage" tagx="fpage" title="fpage">468</span><span class="lpage" tagx="lpage" title="lpage">476</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jix334">10.1093/infdis/jix334</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28931241">28931241</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sattabongkot</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Malaria in the Greater Mekong Subregion: heterogeneity and complexity</span><span class="source" tagx="source" title="source">Tackling Malar End Game Reg Needs Chall Success Malar Elimin</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">121</span><span class="fpage" tagx="fpage" title="fpage">227</span><span class="lpage" tagx="lpage" title="lpage">239</span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sluydts</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coosemans</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Roey</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gryseels</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canier</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Spatial clustering and risk factors of malaria infections in Ratanakiri Province</span><span class="source" tagx="source" title="source">Cambodia Malar J</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">387</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2875-13-387">10.1186/1475-2875-13-387</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25269827">25269827</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Organization WH. Malaria in the Greater Mekong subregion: regional and country profiles. 2010.</span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steenkeste</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">WO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okell</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeanne</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Incardona</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duval</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sub-microscopic malaria cases and mixed malaria infection in a remote area of high malaria endemicity in Rattanakiri province, Cambodia: implication for malaria elimination</span><span class="source" tagx="source" title="source">Malar J</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">108</span><span class="lpage" tagx="lpage" title="lpage">108</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2875-9-108">10.1186/1475-2875-9-108</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20409349">20409349</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Autino</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noris</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castelli</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology of malaria in endemic areas</span><span class="source" tagx="source" title="source">Mediterr J Hematol Infect Dis</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">e2012060</span><span class="lpage" tagx="lpage" title="lpage">e2012060</span><span class="pub-id"><a href="https://dx.doi.org/10.4084/mjhid.2012.060">10.4084/mjhid.2012.060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23170189">23170189</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sawada H, Araki M, Chappell NA, LaFrankie JV, Shimizu A. Forest environments in the Mekong River basin. Springer; 2007.</span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obsomer</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Defourny</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coosemans</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The <i>Anopheles dirus</i> complex: spatial distribution and environmental drivers</span><span class="source" tagx="source" title="source">Malar J</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2875-6-26">10.1186/1475-2875-6-26</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17341297">17341297</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amato</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miotto</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amaratunga</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dek</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pearson</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic markers associated with dihydroartemisinin–piperaquine failure in <i>Plasmodium falciparum</i> malaria in Cambodia: a genotype–phenotype association study</span><span class="source" tagx="source" title="source">Lancet Infect Dis</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">164</span><span class="lpage" tagx="lpage" title="lpage">173</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(16)30409-1">10.1016/S1473-3099(16)30409-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27818095">27818095</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imwong</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">TN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tripura</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peto</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lwin</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The epidemiology of subclinical malaria infections in South-East Asia: findings from cross-sectional surveys in Thailand-Myanmar border areas, Cambodia, and Vietnam</span><span class="source" tagx="source" title="source">Malar J</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">381</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12936-015-0906-x">10.1186/s12936-015-0906-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26424000">26424000</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cook</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Speybroeck</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sochanta</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somony</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokny</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claes</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sero-epidemiological evaluation of changes in <i>Plasmodium falciparum</i> and <i>Plasmodium vivax</i> transmission patterns over the rainy season in Cambodia</span><span class="source" tagx="source" title="source">Malar J</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">86</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1475-2875-11-86">10.1186/1475-2875-11-86</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22443375">22443375</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoyer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Habib</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khim</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sum</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Focused screening and treatment (FSAT): a pcr-based strategy to detect malaria parasite carriers and contain drug resistant <i>P. falciparum</i>, Pailin, Cambodia</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">45797</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0045797">10.1371/journal.pone.0045797</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milne</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Grounding forest carbon: property relations and avoided deforestation in Cambodia</span><span class="source" tagx="source" title="source">Hum Ecol</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">693</span><span class="lpage" tagx="lpage" title="lpage">706</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10745-012-9526-z">10.1007/s10745-012-9526-z</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">KF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rulli</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pichdara</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D’Odorico</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Accelerated deforestation driven by large-scale land acquisitions in Cambodia</span><span class="source" tagx="source" title="source">Nat Geosci</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">772</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ngeo2540">10.1038/ngeo2540</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Borda V, Alvim I, Aquino MM, Silva C, Soares-Souza GB, Leal TP, et al. The genetic structure and adaptation of Andean highlanders and Amazonian dwellers is influenced by the interplay between geography and culture. BioRxiv. 2020</span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ringbauer H, Kolesnikov A, Field DL, Barton NH. Estimating barriers to gene flow from distorted isolation by distance patterns. Genetics. 2018;genetics. 300638.2017.</span></li> </ul> </div> </div>  </body></html>