<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6976573">6976573</a></span><span class="publisher-id" title="publisher-id">57970</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-020-57970-3">10.1038/s41598-020-57970-3</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Two genomic regions encoding exopolysaccharide production systems have complementary functions in <i>B. cereus</i> multicellularity and host interaction</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caro-Astorga</span><span class="given-names" tagx="given-names" title="given-names">Joaquin</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Álvarez-Mena</span><span class="given-names" tagx="given-names" title="given-names">Ana</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hierrezuelo</span><span class="given-names" tagx="given-names" title="given-names">Jesús</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guadix</span><span class="given-names" tagx="given-names" title="given-names">Juan Antonio</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heredia-Ponce</span><span class="given-names" tagx="given-names" title="given-names">Zahira</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arboleda-Estudillo</span><span class="given-names" tagx="given-names" title="given-names">Yohanna</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8291-1518</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">González-Munoz</span><span class="given-names" tagx="given-names" title="given-names">Elena</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2716-9861</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vicente</span><span class="given-names" tagx="given-names" title="given-names">Antonio</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2052-6515</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero</span><span class="given-names" tagx="given-names" title="given-names">Diego</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">diego_romero@uma.es</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2298 7828</span><span class="institution-id" title="institution-id">grid.10215.37</span><span class="institution" title="institution">Instituto de Hortofruticultura Subtropical y Mediterránea "La Mayora" –Departamento de Microbiología, Universidad de Málaga, Bulevar Louis Pasteur 31 (Campus Universitario de Teatinos), </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2298 7828</span><span class="institution-id" title="institution-id">grid.10215.37</span><span class="institution" title="institution">Departamento de Biología Animal, Facultad de Ciencias, Universidad de Málaga - IBIMA, Campus de Teatinos s/n, </span></div></span><span class="citation_author_institution" id="Aff3">[3], </span><span class="citation_author_institution" id="Aff4">[4], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1000</span><span class="history" title="history"><span class="received" title="received">received: 2019-7-29</span><span class="accepted" title="accepted">accepted: 2020-1-9</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Bacterial physiology and adaptation are influenced by the exopolysaccharides (EPS) they produce. These polymers are indispensable for the assembly of the biofilm extracellular matrix in multiple bacterial species. In a previous study, we described the profound gene expression changes leading to biofilm assembly in <i>B. cereus</i> ATCC14579 (CECT148). We found that a genomic region putatively dedicated to the synthesis of a capsular polysaccharide (<i>eps2</i>) was overexpressed in a biofilm cell population compared to in a planktonic population, while we detected no change in the transcript abundance from another genomic region (<i>eps1</i>) also likely to be involved in polysaccharide production. Preliminary biofilm assays suggested a mild role for the products of the <i>eps2</i> region in biofilm formation and no function for the products of the <i>eps1</i> region. The aim of this work was to better define the roles of these two regions in <i>B. cereus</i> multicellularity. We demonstrate that the <i>eps2</i> region is indeed involved in bacterial adhesion to surfaces, cell-to-cell interaction, cellular aggregation and biofilm formation, while the <i>eps1</i> region appears to be involved in a kind of social bacterial motility. Consistent with these results, we further demonstrate using bacterial-host cell interaction experiments that the <i>eps2</i> region is more relevant to the adhesion to human epithelial cells and the zebrafish intestine, suggesting that this region encodes a bacterial factor that may potentiate gut colonization and enhance pathogenicity against humans.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Biofilms</div> <div class="kwd" title="kwd">
Microbial ecology</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100003329</span><span class="institution" title="institution">Ministerio de Economía y Competitividad (Ministry of Economy and Competitiveness)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">AGL2012-31968</span><span class="award-id" tagx="award-id" title="award-id">AGL2016-78662-R</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero</span><span class="given-names" tagx="given-names" title="given-names">Diego</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/100010663</span><span class="institution" title="institution">EC | EU Framework Programme for Research and Innovation H2020 | H2020 Priority Excellent Science | H2020 European Research Council (H2020 Excellent Science - European Research Council)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">BacBio 637971</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero</span><span class="given-names" tagx="given-names" title="given-names">Diego</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">The biofilm extracellular matrix mainly consists of proteins, extracellular DNA and exopolysaccharides (EPSs). The relative importance of each component for biofilm formation depends on the species, but these three elements generally confer the diverse physico-chemical features defining the extracellular matrix<sup><a href="#CR1">1</a></sup>. EPSs are large molecules with complex structures and diverse chemical properties that cover a variety of functions relevant for bacterial physiology and multicellular lifestyles. Some of their functions include maintaining biofilm architecture, establishing colony morphology, and providing robustness to biofilms<sup><a href="#CR2">2</a>–<a href="#CR5">5</a></sup>. EPSs are also involved in the adhesion of cells to biotic and abiotic surfaces which, for instance, determines the ability of the bacterial cells to efficiently colonize natural niches, such as the plant rhizosphere<sup><a href="#CR6">6</a>–<a href="#CR10">10</a></sup>. Interactions among bacterial cells have also been shown to be supported by EPSs, which can induce cellular aggregation<sup><a href="#CR11">11</a>,<a href="#CR12">12</a></sup>. Thus, it is not surprising to find that EPSs are involved in the migration and mobility of bacterial colonies, functioning either as rails over which the bacteria move or in reducing friction and facilitating the displacement of either single cells or the entire community in the modes of social movement known as swarming and sliding, depending on what other bacterial factors are involved<sup><a href="#CR13">13</a>–<a href="#CR17">17</a></sup>. Roles for EPSs in stress resistance have also been widely reported, including conferring resistance to osmotic, desiccation, and oxidative stress and in cryoprotection<sup><a href="#CR18">18</a>–<a href="#CR22">22</a></sup>. In some cases, EPSs offer protection against phage attack, even though specific EPSs are sometimes the targets recognized and enzymatically degraded by phage capsid proteins to trigger infection<sup><a href="#CR23">23</a>–<a href="#CR25">25</a></sup>. In addition to their roles in conferring resistance to sanitizers, detergents, and antimicrobials and their implications in human disease, EPSs can also contribute to the corrosion of stainless steel, a material that is very commonly used in the food industry, tanks, pipes and ship hulls, on which <i>B. cereus</i> is prone to establish biofilms<sup><a href="#CR26">26</a></sup>.</p> <p id="Par3">The variety of EPS functions reflect differences in chemical composition, particularly in terms of the different sugar residues present, chemical bond types, ramifications, sugar modifications and chain length, all of which are determined via complex biosynthetic processes<sup><a href="#CR27">27</a></sup>. Therefore, common polysaccharides, like cellulose, can be produced by bacterial species in different genera, while others are strain specific<sup><a href="#CR28">28</a>,<a href="#CR29">29</a></sup>. The study of the bacterial factors involved in biofilm formation is useful for understanding the functions of the biofilm in bacterial physiology and ecology, including interactions between bacterial cells or their hosts. Furthermore, such studies can reveal potential bacterial targets for the development of strategies for minimizing the negative impacts of bacterial biofilms. Studies on <i>B. cereus</i> have highlighted the relevant contributions of EPSs to the total composition of the extracellular matrix, although their origins still remain uncertain<sup><a href="#CR30">30</a></sup>. Some reports have focused on the characterization of spore polysaccharides<sup><a href="#CR31">31</a></sup> or secondary cell wall polysaccharide, both of which seem to be strain dependent<sup><a href="#CR32">32</a></sup>. In the phylogenetically-related species <i>B. subtilis</i>, the <i>epsA-O</i> operon encodes a group of enzymes required for the synthesis of biofilm EPS, and deletion of this operon leads to impaired biofilm formation<sup><a href="#CR33">33</a>,<a href="#CR34">34</a></sup>. <i>B. cereus</i> ATCC14579 possesses a homologous region (<i>Bc5279-Bc5263</i>, henceforth <i>eps1</i>), which we have confirmed to be irrelevant for biofilm formation in this strain, a finding that is also supported by studies of the strain <i>B. cereus</i> 905 in a pellicle model of biofilm<sup><a href="#CR35">35</a></sup>. However, it appears that deletion of some of the genes in this region impairs biofilm formation in the strain ATCC10987, using PEG plates as a model of biofilm<sup><a href="#CR28">28</a></sup>, a discordancy that reveals the heterogeneity that can exist among bacterial strains of the same species.</p> <p id="Par4">Prompted by an interest in the influences of EPS on bacterial physiology and ecology and the divergence in terms of their exact chemical composition and function, in this work we studied the structures of two genomic regions that putatively encode proteins that synthesize two different polysaccharides. As biofilm model, we used the ring of biomass adhered to the wall of the well of culture plates given that <i>B. cereus</i> 14579 only produce pellicles in specific conditions and long periods of incubation. We then explored the implications of these regions in <i>B. cereus</i> multicellularity and host interactions. We previously reported a transcriptomic analysis in which we showed that an additional region of the <i>B. cereus</i> ATCC14579 genome (containing the genes <i>Bc1583-Bc1591</i> and referred to as <i>eps2</i>) was overexpressed in biofilm cells compared to its expression level in planktonic cells<sup><a href="#CR36">36</a></sup>. This region has been annotated to be involved in capsular polysaccharide production, although <i>B. cereus ATCC14579</i> lacks a capsule. Our analysis indicates that the two putative polysaccharides play complementary roles in <i>B. cereus</i> multicellularity: while <i>eps1</i> promotes bacterial social mobility, <i>eps2</i> is involved in biofilm maturation, cell-to-cell interaction and aggregation. Interestingly, and consistent with these findings, EPS2 seems to be more important for the adhesion of <i>B. cereus</i> cells to human epithelial cells and to the zebrafish gut, a model used to study bacteria-gut interactions.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="theeps1andeps2regionsaredifferentiallyexpressedinb.cereusbiofilm" title="sec"> <div class="title" tagx="title" title="title">
The <i>eps1</i> and <i>eps2</i> regions are differentially expressed in <i>B. cereus</i> biofilm</div> <p id="Par5">A previous transcriptomic analysis of <i>B. cereus</i> cells grown under static conditions demonstrated that there were no statistically significant differences in the expression levels of a region homologous to the <i>B. subtilis eps</i>A-<i>O</i> operon (which is dedicated to EPS production) between biofilm and planktonic cells 24 and 48 h post-inoculation (Fig. Suppl. <a href="#MOESM1">1A</a>). However, we found that a group of genes (<i>BC1583-BC1591</i>) absent in the <i>B. subtilis</i> group and annotated as "capsular polysaccharide biosynthesis" was upregulated in biofilm cells. Consistent with these findings, a <i>B. cereus eps1</i> mutant strain was not defective in biofilm formation, as demonstrated by crystal violet staining of adhered biomass. However, biofilm formation seemed to be subtly affected in a strain lacking the entire <i>eps2</i> region<sup><a href="#CR36">36</a></sup>.</p> <p id="Par6">Before initiating specific studies designed to determine the functions of each region, we first studied their genetic organization. Before performing any experiment, we confirmed by PCR analysis the genotype of the <i>eps</i> mutants using specific pair of primers (Table <a href="#MOESM1">S2</a> and Fig. Suppl. <a href="#MOESM1">1B</a>). The <i>B. cereus eps1</i> region was compared with the <i>B. subtilis epsA-O</i> region to confirm previously reported similarities<sup><a href="#CR35">35</a></sup>. Comparison of the genetic regions showed poor homology, with species-specific genes and duplications. Therefore, the different operon arrangement might explain the lack of a phenotype in a knock-out mutant strain (Fig. <a href="#Fig1">1A</a>). To determine if this region is organized as a single operon, we performed RT-PCR with total RNA isolated from a liquid culture of <i>B. cereus</i> grown at 30 °C for 24 h (Fig. <a href="#Fig1">1B</a>). The results indicated that the <i>eps1</i> region appears to consist of three transcriptional units and an orphan tyrosine-protein kinase: (i) <i>BC5279</i>, (ii) <i>BC5278-BC5277</i>, (iii) <i>BC5276-BC5267</i>, and (iv) <i>BC5266-BC5263</i>. An <i>in silico</i> analysis of this region also predicted three putative promoter regions at the beginning of each of the transcriptional units. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>The <i>eps1</i> region is organized in multiple transcriptional units. (<b>A</b>) Comparison of the genetic structures of the <i>B. cereus eps1</i> region and the <i>B. subtilis epsA-O</i> operon. Conserved genes are in black and weave and unique genes are marked in grey. Top, a gene cluster predicted to be involved in the production of an EPS in <i>B. cereus</i>. Bottom, the operon involved in the synthesis of the biofilm EPS in <i>B. subtilis</i>. Automatic annotation of the genes in the <i>eps1</i> region is shown. (<b>B</b>) qPCR with cDNA obtained by reverse transcribing RNA isolated from a broth culture of <i>B. cereus</i> grown for 24 h at 30 °C.</p>   </div> <p /> <p id="Par7">The same analysis performed on the <i>eps2</i> region revealed that the nine genes are organized in a single transcriptional unit (Fig. <a href="#Fig2">2</a>). To confirm that any additional genes were excluded from the operon, we performed further <i>in silico</i> analysis of the 843 base pairs of the intergenic region upstream of the <i>BC1583</i> locus using the software ORFfinder. RT-PCR analysis confirmed that the poly-cistronic mRNA also includes the three putative ORFs; however, these additional ORFs were excluded from the knock-out mutation of this genomic region, which did not affect to the loss-of-function phenotype of the <i>eps2</i> mutant strain<sup><a href="#CR36">36</a></sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>The <i>eps2</i> region is organized in a single transcriptional unit. (<b>A</b>) qPCR with cDNA obtained by reverse transcribing RNA isolated from a broth culture of <i>B. cereus</i> grown for 24 h at 30 °C. The arrow indicates the hypothetical promoter region of the operon. (<b>B</b>) The genetic structure determined from the RT-PCR results and the automatic annotation of the genes included in the operon. (<b>C</b>) Upstream putative ORFs found via ORF-Finder analysis and included as an integrant of the operon.</p>   </div> <p /> </div> <div class="theeps2region,butnottheeps1region,islinkedtobiofilmformation" title="sec"> <div class="title" tagx="title" title="title">
The <i>eps2</i> region, but not the <i>eps1</i> region, is linked to biofilm formation</div> <p id="Par8">Previous data led us to propose that <i>eps2</i> makes a subtle contribution to biofilm formation, as revealed by crystal violet staining of biomass adhered to an abiotic surface. However, we wanted to decipher the real implications of these two regions in biofilm formation, either alone or in combination. Therefore, we constructed a <i>Δeps1 Δeps2</i> double mutant strain (Fig. Suppl. <a href="#MOESM1">1B</a>) to search for collaborative functions. Biofilm assays in shaken liquid cultures of the <i>Δeps</i>2 strain showed clear defects in the quantity, consistency and continuity of the air-liquid interphase biofilm ring adhered to the glass (Fig. <a href="#Fig3">3A</a>). Reflecting the differences in the expression patterns of <i>eps1</i> in biofilm and planktonic cells (Fig. Suppl. <a href="#MOESM1">1</a>), the <i>Δeps1</i> strain was not impaired in biofilm formation compared to the wild type (WT) strain, and the <i>Δeps1, Δeps2</i> double-mutant strain showed the same phenotype as the <i>Δeps2</i> single mutant strain (Fig. <a href="#Fig3">3A</a>). In parallel to evaluating the ring-adhered biomass, we also observed significant bacterial cell sedimentation in cultures of the WT and <i>Δeps1</i> strains (Fig. <a href="#Fig3">3A</a>). Consistently, imaging of aliquots of the supernatants from cultures of the WT and <i>Δeps1</i> strains with a phase contrast microscope revealed bacterial clumps that were absent in spent medium from the <i>Δeps2</i> single-mutant strain and from the <i>Δeps1, Δeps2</i> double-mutant strain (Fig. <a href="#Fig3">3B</a>). In a previous study<sup><a href="#CR37">37</a></sup>, we showed the contribution made by the TasA protein to adhesion to abiotic surfaces; therefore, we wondered whether the reduced adhesion phenotype of the <i>Δeps2</i> mutant strain might be caused by the absence of this protein. First, we observed no significant changes in the <i>tasA</i> expression levels between the <i>Δeps1</i> or <i>Δeps2</i> mutant strains and the WT strain in RT-PCR analyses of RNA extracted from biofilm-associated cells (Fig. Suppl. <a href="#MOESM1">2</a>). Second, immunogold labeling with anti-TasA antibodies and subsequent electron microscopy imaging revealed the presence of similar amounts of TasA fibers decorating the surfaces of the WT, <i>Δeps1</i> and <i>Δeps2</i> cells, with no significant differences in their appearance (Fig. <a href="#Fig3">3C</a>). Based on these data, we excluded the possibility that the reduced adhesion of the <i>Δeps2</i> strain to abiotic surfaces was due to the absence of TasA or its inability to assemble fibers. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Biofilm phenotypes of the WT and <i>eps-</i>mutant strains. (<b>A</b>) Liquid cultures in TY medium incubated for 24 h at 30 °C with 150 rpm shaking showed that the WT strain yielded the largest amount of biomass attached to the walls of the flask compared to the strain mutated in the <i>eps2</i> region. (<b>B</b>) Phase contrast micrographs of the spent medium from liquid cultures show the presence of bacterial clumps in the media from the WT and <i>Δeps1</i> strains. The bottom pictures are closer views of the squared areas in the original pictures (<b>C</b>) Transmission electron micrographs of negatively stained cells decorated with fibers reactive to anti-TasA antibodies. (<b>D</b>) Congo Red biofilm assays in: Top, TY liquid medium supplemented with Congo Red and incubated without shaking at 30 °C for 6 days, and bottom, TY medium solidified with 1.5% agar supplemented with Congo Red and incubated at 30 °C for 72 h. The bars in D = 1 μm.</p>   </div> <p /> <p id="Par9">Prompted by the well-known staining properties of certain EPSs, we performed biofilm assays on agar plates and in static liquid cultures in the presence of Congo Red, a dye commonly used to reveal the presence of certain types of EPS (Fig. <a href="#Fig3">3D</a>)<sup><a href="#CR13">13</a>,<a href="#CR38">38</a>,<a href="#CR39">39</a></sup>. As previously reported<sup><a href="#CR36">36</a></sup>, the <i>Δeps1</i> single mutant cells formed a biomass on the walls of the wells in static cultures, similar to that formed by the WT cells; however, subtle differences could be appreciated in the <i>Δeps2</i> strain, and major defects were observed in the double-mutant strain. These assays also clearly defined a red zone visible in the bottom part of the adhered biomass of the WT and <i>Δeps1</i> strains that was absent in the biomass of the <i>Δeps2</i> single- and <i>Δeps1 Δeps2</i> double-mutant strains (Fig. <a href="#Fig3">3D</a>, top). Complementary to these findings, two distinct phenotypes emerged from the same experiments performed instead on agar plates: the absence of reddish staining and robust spreading of colonies of the <i>Δeps2</i> single- and <i>Δeps1 Δeps2</i> double-mutant strains (Fig. <a href="#Fig3">3D</a>, bottom). Taken together, these findings support two primary conclusions: (i) The two EPSs have divergent functions, i.e., while EPS2 seems to be relevant for establishing the sedentary lifestyle associated with biofilm formation, EPS1 appears to be more important for colony mobility; and (ii) the two putative EPS species likely have different chemical compositions, as indicated by the binding of EPS2, but not EPS1, to Congo Red. To evaluate differences at chemical level, EPSs from these strains were cold-ethanol precipitated, separated in size exclusion chromatography (SEC) and fractions subjected to the phenol-sulfuric acid method<sup><a href="#CR40">40</a></sup> (Fig. Suppl. <a href="#MOESM1">3A–D</a>, insets). Fractions that firstly eluted in the SEC column, and thus attributed to larger polysaccharide structures, were initially analyzed in Attenuated total reflectance-Fourier transform infrared spectroscopy (ATR-FTIR) to confirm the presence of functional groups characteristics of EPS<sup><a href="#CR41">41</a></sup> (Fig. Supp. <a href="#MOESM1">3E</a>). As previously reported, in the range of 3600-3200 cm<sup>−1</sup> the samples exhibit a broad peak for the O-H stretching vibration of the polysaccharide<sup><a href="#CR42">42</a></sup>. The two weak bands between 2930-2850 cm<sup>−1</sup> are assigned to C-H stretching vibrations (CH<sub>2</sub> and CH<sub>3</sub> groups). Peaks at 2360 cm<sup>−1</sup> and 1658 cm<sup>−1</sup> might be respectively attributed to NH stretching and C=O stretching vibration of N-acetyl group that might be protecting OH groups. The symmetrical deformations of CH<sub>2</sub> and C-OH deformations are weak bands that appear between 1440-1460 cm<sup>−1</sup>. For polysaccharide identification, the strong peak in the range 1200-1000 cm<sup>−1</sup> is caused by C-O-C and C-O groups in pyranose ring<sup><a href="#CR43">43</a></sup>. The coexistence of α and β anomeric form is confirmed by the presence of bands at 860 and 915 cm<sup>−1</sup>. These fractions were next chemically studied by hydrolysis and subsequent derivatization for further separation in gas chromatography and mass spectrometry. The major component monosaccharides of all extracts were found to be glucose, galactose and mannose together with other minor components like ribose, rhamnose or xylose, indicating that the EPS produced is a heteropolysaccharide (Figs. Suppl. <a href="#MOESM1">3A–D</a> and Suppl. 4). Interestingly, purified samples of the <i>Δeps2</i> mutant strains exhibit fucose units in the analysis. Based on this findings, one can assume that these differences at sugar composition might reflect the divergent behavior in binding devices features of each strain.</p> </div> <div class="theproductsoftheeps1andeps2genomicregionsarenotdirectlyinvolvedincellularauto-aggregation" title="sec"> <div class="title" tagx="title" title="title">
The products of the <i>eps1</i> and <i>eps2</i> genomic regions are not directly involved in cellular auto-aggregation</div> <p id="Par10">EPSs are known to possess adhesive properties that promote cell-to-cell interactions, an early step in the formation and maturation of bacterial biofilms<sup><a href="#CR44">44</a></sup>. The observation of bacterial clumps in the spent media from agitated cultures of the WT and <i>Δeps1</i> strains led us to consider the importance of EPS2 in cell aggregation (Fig. <a href="#Fig1">1</a>). To definitively address this phenotype, we performed auto-aggregation assays using two complementary methods: (i) recording time-dependent variations in the optical density (OD) of the upper part of liquid cultures<sup><a href="#CR45">45</a></sup>, and (ii) measuring the OD at a defined depth in liquid cultures at the end of the experiment<sup><a href="#CR46">46</a></sup>. To perform these assays, bacterial cells were grown overnight at 28 °C, and then, after adjusting the culture ODs to a uniform value, cell suspensions were incubated at room temperature (RT) with no agitation. Samples were taken every 15 min for the first method (Fig. <a href="#Fig4">4A</a>), and for the second method, the ODs were measured at the same depth in all of the cell suspensions after 9 h of incubation (Fig. <a href="#Fig4">4B</a>). Our findings indicated that absence of either of the <i>eps</i> regions led to faster aggregation kinetics compared with that of the WT strain, and this effect was amplified in the <i>Δeps1 Δeps2</i> double-mutant strain during the first 5 h of stationary incubation (Fig. <a href="#Fig4">4A</a>). Consistently, the OD values obtained with the second method were lower in the mutant strains compared with that of the WT strain after 9 h of incubation (Fig. <a href="#Fig4">4B</a>). Furthermore, after 24 h of incubation, sediments were visible in the bottoms of the tubes in the cultures of the mutant strains, but not in that of the WT strain (Fig. Suppl. 5). These results suggested that <i>eps1</i> and <i>eps2</i> are irrelevant for cellular auto-aggregation and indicated that other cell surface factors (possibly hidden by the presence of EPS) might be directly involved in this cellular process. Thus, in the absence of either EPS species, the cells interact more efficiently and increase their sedimentation rate, suggesting a mechanism through which EPS could indirectly control cell auto-aggregation. We next hypothesized that either TasA or CalY (extracellular proteins involved in <i>B. cereus</i> biofilm formation) might function with the EPS to regulate bacterial auto-aggregation at the structural level. Strains mutant for either <i>tasA</i> or <i>calY</i> aggregated similarly to the WT strain or to either of the <i>eps</i> mutant strains (Fig. Suppl. 6). The <i>Δeps1</i>, <i>eps2</i>, <i>tasA</i> triple-mutant strain showed aggregation kinetics similar to those of the WT strain; in contrast with the phenotype of the <i>Δeps1</i>, <i>eps2</i>, <i>calY</i> triple-mutant strain that was severely defective in cellular auto-aggregation. These findings supported the functionality of CalY in bacterial cell-to-cell interaction and demonstrated that CalY is a bacterial factor involved in auto-aggregation whose functionality may be hidden or modulated by the presence of EPS (although other such factors may also exist). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Eps1 and Eps2 prevent cellular auto-aggregation. The optical densities (ODs) of cultures grown in TY medium incubated at 30 °C for 16 h with shaking at 150 rpm were adjusted to OD = 3 before the experiments were initiated. (<b>A</b>) Top, representative images of the tubes after 5 h of incubation; bottom, OD values of the air-liquid interface in the cell suspensions under static conditions at room temperature. (<b>B</b>) Top, representative images of the cultures after 9 h of incubation; bottom, OD values of the liquid cultures after 9 h of incubation, with sampling at the height of the grade bar. Means compared using t-test (P &amp;lt; 0.05).</p>   </div> <p /> </div> <div class="theeps1andeps2regionsantagonisticallyinfluencecellmotility" title="sec"> <div class="title" tagx="title" title="title">
The <i>eps1</i> and <i>eps2</i> regions antagonistically influence cell motility</div> <p id="Par11">Several EPSs have been reported to play roles in motility in different bacterial species, and we noticed differences in the colony sizes of the <i>eps</i> mutant strains grown on typical solid agar plates (Fig. <a href="#Fig3">3D</a>, bottom). To determine if EPS1 or EPS2 is involved in bacterial motility, we seeded suspensions of WT or mutant cells on 0.3% or 0.7% agar plates to assay swimming or swarming motility, respectively (Fig. <a href="#Fig5">5A,B</a>). The <i>Δeps2</i> cells swam more efficiently than did the WT or <i>Δeps1</i> cells, and we observed a reversion to the <i>eps1</i> phenotype of the double mutant <i>eps1, eps2</i> (Fig. <a href="#Fig5">5A</a>). This result suggested that EPS2 has a negative effect on swimming and was consistent with the propensity for cellular clumping observed in liquid cultures 24 h post-inoculation (Fig. <a href="#Fig3">3A</a>). On swarming agar plates, the <i>Δeps1</i> and double-mutant colonies spread less than the WT colonies, and <i>eps2</i> deletion did not result in any noticeable change in colony spreading (Fig. <a href="#Fig5">5B</a>). Taken together, these findings indicate that EPS2 likely antagonizes cell motility, while EPS1 seems to promote social movement. To really decipher the involvement of EPS1 in motility different from swimming and swarming, we did mutate the <i>flag</i> allele in all the strains. No significant differences were observed in colony mobility, showing no swimming or swarming motility of all strains lacking the <i>flag</i> allele, confirming that EPS1 is involved in motility dependent on flagella and that the previous results are not a result of a different type of mobility (Fig. <a href="#Fig5">5A,B</a>, bottom). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>EPS1 and EPS2 contribute differently to bacterial cell motility. Cell suspensions were spotted in the centers of the plates, which were then incubated at 28 °C before examining the motility. (<b>A</b>) Images taken from the swimming agar plates (0.3% TY agar plates, 24 h). (<b>B</b>) Images taken from the swarming agar plates (0.7% TY agar plates, 72 h). The graphs represent the values of the colony diameters at the end of each experiment. A Tukey statistical analysis was realized (p-value &amp;lt; 0.01) and the significant differences are indicated with letters. (<b>C</b>,<b>D)</b> Bacterial strains constitutively expressing YFP or CFP were mixed at a 1:1 ratio and then spotted in the centers of the plates. (<b>C</b>) The motility and Congo Red binding phenotypes were analyzed on TY swarming agar plates supplemented with Congo Red. The drawing indicates the zone occupied by each strain in the corresponding complementation experiment and marked with squares in the original picture (bottom). (<b>D</b>) Fluorescence microscopy images showing the presence of each bacterial strain in the same section indicated in panel C, bottom.</p>   </div> <p /> <p id="Par12">Based on these findings, we reasoned that mixing cells from each strain might restore the WT phenotype. To test this hypothesis, we performed strain complementation experiments on swarming agar plates supplemented with Congo Red and tested for reversion of the swarming motility and dye staining phenotypes (Fig. <a href="#Fig5">5C</a>). When both tested strains possessed the <i>eps2</i> region, the colony color was homogeneously reddish. When one of the strains lacked the <i>eps2</i> region, colored sectors appeared. In colonies in which neither or both strains possessed <i>eps1</i>, the sectors reached the colony border; however, these sectors decayed from the center to the border of the colony when only one strain lacked <i>eps1</i> (Fig. <a href="#Fig5">5C</a>). These results support our view that EPS1 is necessary for robust social motility, given that the strains lacking <i>eps1</i> showed delayed movement and remained in the colony centers. Two strain mixtures, WT + <i>Δeps1</i> (reddish) and <i>Δeps2 + Δeps1Δeps2</i> (pinkish) showed no sectors, given that the color phenotypes of both strains were similar. To clearly visualize the cellular distribution of each strain in the complete community, these strains were transformed with replicative plasmids constitutively expressing either <i>yfp</i> or <i>cfp</i> (Fig. <a href="#Fig5">5D</a>). Fluorescent microscopy imaging confirmed the formation of sectors in the mixtures and supported the previously-stated hypothesis regarding the positive contribution of EPS1 to social movement on swarming agar plates. A mixture of WT and <i>Δeps1</i> cells showed reduced fitness in the cells lacking <i>eps1</i>, which remained close to the original spot area, and the colony borders were mainly dominated by WT cells (Fig. <a href="#Fig5">5D</a>). Similarly, the <i>Δeps1 Δeps2</i> double-mutant cells were less competitive when combined with <i>Δeps2</i> cells. These results suggest that only cells producing EPS1 show enhanced movement; thus, the function of EPS1 in motility is likely on the level of individual cells, not shared with the neighbors, rather on the level of the entire community, the latter of which has been reported for the secretion of surfactants and other EPSs that can facilitate the motility of non-producer cells<sup><a href="#CR47">47</a></sup>.</p> </div> <div class="eps2ismorerelevantfortheadhesionofb.cereuscellstohumanepithelialcellsandthezebrafishgut" title="sec"> <div class="title" tagx="title" title="title">
EPS2 is more relevant for the adhesion of <i>B. cereus</i> cells to human epithelial cells and the zebrafish gut</div> <p id="Par13">The previous <i>in vitro</i> experiments suggested a major contribution of <i>eps2</i> to the adhesion to abiotic surfaces and to biofilm formation. Thus, we wondered if these findings could be extended to biotic surfaces. In light of the pathological implications of <i>B. cereus</i> in humans, we initially tested the adhesion abilities of these strains to HeLa and MDA epithelial cells as a model for human infection. As expected, based on the results of the <i>in vitro</i> experiments, the <i>Δeps2</i> strain was severely impaired in adhesion, while the <i>Δeps1</i> strain behaved like the WT strain. Like the <i>Δ</i>eps2 strain, the double-mutant strain showed reduced adhesion to both epithelial cell lines (Fig. <a href="#Fig6">6A</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>EPS2 is more important for the adhesion of <i>B. cereus</i> cells to human epithelial cells and to the zebrafish gut. (<b>A</b>) Adhesion assays in cultures of the human HeLa and MDA epithelial cell lines. Means compared using t-student (P &amp;lt; 0.05). (<b>B</b>) Top left, representative image of a zebrafish larva 6 days post-fertilization (dpf). Bottom left graph, rates (±SD) of 6-dpf zebrafish larvae harboring fluorescent bacteria in the GI tract 24 h after feeding with the WT, <i>Δeps1, Δeps2</i> or double-mutant <i>B. cereus</i> strains (1 E8 ucf/ml). Means compared using t-student (P &amp;lt; 0.05). Right panel, bacterial localization in the gut of the zebrafish larvae after infection. The bacterial strains were transformed with a plasmid expressing the <i>yfp</i> gene under the control of a constitutive promoter. a-d´: Fluorescent Z-stack confocal microscopy images (the bacteria marked with asterisks in a-d are magnified in a´-d´). Scale bars = 200 μm in a,b,c,d, 100 μm in a-d, and 1 μm in a´-d´.</p>   </div> <p /> <p id="Par14">After confirming the adhesive properties of EPS2 to epithelial cells, we evaluated the implication of this EPS in bacterial adhesion to a complex biotic surface, the zebrafish (<i>Danio rerio</i>) intestine, a model that anatomically mimics the conditions of the mammalian gut due to the presence of finger-like protrusions (villi) on the intestinal epithelium and to the presence of a diverse cell population, including immune cells<sup><a href="#CR48">48</a></sup>. After feeding zebrafish larvae with WT, single- or double-mutant bacteria harboring a reporter plasmid that constitutively expressed YFP, the presence of the bacteria in the fish gut was evaluated by confocal fluorescence microscopy (Fig. Suppl. 7). Examination of fish showed that 24 h post-feeding, 67% of the larvae retained fluorescent bacteria in their gut. Larvae fed with <i>Δeps1, Δeps2</i> or double-mutant cells showed decreased retention rates of 54%, 37% and 32%, respectively. Based on these differences in the dynamics of the bacterial populations, we expected different distribution patterns of the cells in the intestinal tract. A detailed examination by confocal microscopy using triple labelling with phalloidin to stain the epithelial cells, DAPI to stain the nuclei, and anti-GFP, confirmed the presence of bacterial cells of each specific strain in different regions of the intestinal tract (labeled with asterisks) (Fig. <a href="#Fig6">6Ba–d</a>, right panel). Representative images of zebrafish intestines showed that more WT, <i>Δeps2</i> and double-mutant (<i>Δeps1, Δeps2</i>) cells accumulated in the mid- and posterior-gut compared with the amount of <i>Δeps1</i> cells in the same regions.</p> <p id="Par15">The experiments with the epithelial cell lines (37 °C) and zebrafish (RT) were performed at different temperatures; thus, we asked if different culture temperatures could lead to different expression levels of the two EPSs, thus altering their functionality. An RT-PCR analysis revealed that both <i>eps</i> regions were more efficiently expressed at 30 °C than at 37 °C, especially <i>eps2</i>, which showed a nearly 5-fold higher expression level at 30 °C (Fig. Suppl. 8). A comparison of the changes in the <i>eps1</i> and <i>eps2</i> expression levels indicated that <i>eps1</i> expression is less sensitive to temperature changes. Thus, the lower importance of EPS1 in the adhesion to biotic surfaces could reflect the chemical differences between EPS1 and EPS2 rather than differences in their expression levels.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par16">Bacteria normally possess a repertoire of genetic regions devoted to the synthesis of different polysaccharides<sup><a href="#CR22">22</a></sup>. These polymers can play various structural functions, including coating of the cell wall, capsule formation, spore surface decoration, and providing adhesion or structural support for biofilms; furthermore, they also offer protection against various stresses, provide a storage reservoir for carbon and protect the bacterial cells against host defenses<sup><a href="#CR49">49</a>,<a href="#CR50">50</a></sup>. <i>B. cereus</i> has been shown to produce several different polysaccharides, including (i) exopolysaccharides associated with sporulation (genes <i>BC0484-90)</i><sup><a href="#CR51">51</a></sup>, (ii) two spore-decorating sugars (genes <i>BC3358-61</i>)<sup><a href="#CR31">31</a></sup>, and (iii) two secondary cell wall polysaccharides (SCWP) (genes unknown)<sup><a href="#CR52">52</a></sup>. In this work, we have investigated the roles of two <i>B. cereus</i> genomic regions putatively dedicated to the production of two different polysaccharides, referred to as <i>eps1</i> and <i>eps2</i>. The data obtained from our study indicate that the two polysaccharides play different, but complementary, roles in <i>B. cereus</i> multicellularity, with EPS1 mainly contributing to motility and EPS2 mainly supporting adhesion, biofilm maturation and interaction with host cells.</p> <p id="Par17">One of the cellular processes related to biofilm formation is bacterial cell auto-aggregation<sup><a href="#CR44">44</a></sup>. The data from our auto-aggregation assays showed that the lack of either <i>eps</i> region resulted in increased sedimentation rates, suggesting that the EPS may mask bacterial surface structures responsible for cell-cell interactions, a phenomena also reported in other bacterial species<sup><a href="#CR6">6</a>,<a href="#CR53">53</a></sup>. This finding was, however, contradictory to the cellular aggregation observed in flasks, a discrepancy that is due to the different timing of both experiments resulting in different ages of the cultures and, consequently, the physiological states of the cells, given that the auto-aggregation assays were performed after 16 h of incubation, before the formation of a visible biofilm ring, and the flask aggregation assays were performed after 24 h of incubation with vigorous shaking and after well-developed biofilm rings were visible. Therefore, these findings should be considered complementary: one set shows the auto-aggregation properties of planktonic cells, while the other set shows the aggregation in mature cultures, in which the bacteria have entered a physiological state in which they are particularly prone to form biofilms. Our results also suggest that additional elements beyond the EPSs might be involved in the cellular attractive and repulsive forces regulated in response to culture conditions and responsible for the cellular auto-aggregation and cell-cell interactions<sup><a href="#CR54">54</a></sup>. For instance, in <i>B. thuringiensis</i>, the S-layer is released at the onset of stationary phase, changing the surface characteristics of the bacteria<sup><a href="#CR55">55</a></sup>. The observation that the cellular auto-aggregation of the <i>eps1, eps2, calY</i> triple-mutant strain was affected suggests a direct role for CalY in this process and that the regulation of this cellular process depends on interactions between different structural components. Nevertheless, the biofilm assays performed in shaking flasks clearly showed that the loss of <i>eps2</i> resulted in reduced aggregation and reduced biofilm formation, consistent with previous works that demonstrated that auto-aggregation is directly linked to biofilm formation<sup><a href="#CR44">44</a></sup>.</p> <p id="Par18">Motility is an important bacterial feature that contributes to adaptation, interaction, survival and biofilm formation<sup><a href="#CR56">56</a>–<a href="#CR58">58</a></sup>. In addition to the single-cell movement produced by flagella, bacteria have evolved sophisticated social movements in which certain molecules produced by a sub-population of cells are used to facilitate the movement of the entire bacterial community<sup><a href="#CR59">59</a></sup>. Consistent with the implication of EPS2 in the development of a sessile lifestyle, the results obtained from our motility assays indicated a negative effect of EPS2 on swimming motility, probably because of the adhesive properties that mediate cell-cell interactions and lead to the formation of cell clumps<sup><a href="#CR5">5</a></sup>. Contrary to this finding, EPS1 seemed to be more important for the development of efficient movement. Indeed, complementation experiments in which two strains were co-seeded revealed reduced fitness in <i>eps1</i> mutant cells placed in competition with the other strains, i.e., the movement of these cells was consistently delayed and they were ultimately restricted to the colony inner areas. The difference in the functions of EPS1 and EPS2 can be considered positive in terms of the need for <i>B. cereus</i> to adapt to the variety of environmental conditions that it may encounter during its life cycle, i.e., (i) promoting motility for the efficient colonization of highly competitive habitats, such as vegetable tissue or mammal and insect guts<sup><a href="#CR17">17</a></sup>, or (ii) switching to a sessile lifestyle in less competitive, but more harsh habitats, such as on the surfaces of industrial or medical devices<sup><a href="#CR60">60</a></sup>. The observation that the <i>eps1</i> mutant is impaired, but not entirely deprived of motility on swarming agar plates suggests that the products of the <i>eps1</i> region must be involved in social, collective movement, but not directly in swarming motility<sup><a href="#CR61">61</a></sup>. In <i>B. subtilis</i>, swarming is not affected by the deletion of the <i>eps</i> operon, while flagellum independent sliding depends on the produced EPS<sup><a href="#CR62">62</a></sup>. Our results in <i>B. cereus</i> suggest the requirement of both flagella and EPS1 for efficient surface colonization. This is reminiscence to the mechanism required for the wandering colonies of <i>Paenibacillus</i> that depends on functional flagella and the secreted protein CmoA, speculated to be a water recruiting protein<sup><a href="#CR63">63</a></sup>. <i>B. cereus</i> EPS1 might have similar function to surfactin swarming in <i>B. subtilis</i>, with the particularity that it is not shared for the community.</p> <p id="Par19">We observed that the <i>eps1</i> expression level is less sensitive to temperature variation, while the <i>eps2</i> expression level is very sensitive to temperature shifts, becoming downregulated at 37 °C. In mammalian niches, <i>B. cereus</i> cells are more likely to attack the host cells, remaining in a planktonic state<sup><a href="#CR36">36</a>,<a href="#CR64">64</a></sup>. Unless other factors influence <i>eps2</i> expression, in a scenario in which <i>B. cereus</i> is predisposed to attack its mammalian host, small amounts of EPS2 could support efficient adhesion of the <i>B. cereus</i> cells to the host’s epithelial cells. Our results in the zebrafish gut are consistent with the results of our <i>in vitro</i> adhesion assays and support the contribution of EPS2 to <i>B. cereus</i> persistence in a natural environment. However, the differences in adhesion were less evident <i>in vivo</i>, probably due to the higher complexity of the gut environment, where a mucus layer covers the lumen and other resident microorganisms may interact with <i>B. cereus</i><sup><a href="#CR48">48</a></sup>.</p> <p id="Par20">In summary, our results reveal that <i>eps2</i> is involved in adhesion, biofilm formation, cell-cell cohesion and host interaction, while <i>eps1</i> might play a role in promoting a kind of social motility. The <i>eps1</i> region shows similarities with the <i>B. subtilis epsA-O</i> operon, which encodes the factors required for the synthesis of the structural biofilm EPSs. Some previous work has demonstrated that this EPS stimulates anti-inflammatory M2 macrophages, inhibiting T cell activation<sup><a href="#CR65">65</a></sup>. Thus, we tentatively propose that the <i>B. cereus eps1</i> region might have evolved to exert enhanced immune modulatory effects, while losing most of its structural functions; however, this suggestion requires further investigation.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods and Materials</div> <div class="bacterialstrainsandcultureconditions" title="sec"> <div class="title" tagx="title" title="title">
Bacterial strains and culture conditions</div> <p id="Par21">The bacteria used in this study are listed in Annex Table <a href="#MOESM1">S1</a>. Bacteria were routinely grown in TY broth (1% tryptone, OXOID), 0.5% yeast extract (OXOID), 0.5% NaCl, 10 mM MgSO<sub>4</sub>, and 0.1 mM MnSO<sub>4</sub>). Biofilm assays were performed in TY both or on the same medium solidified with 1.5% agar<sup><a href="#CR37">37</a></sup>.</p> </div> <div class="constructionofb.cereusmutants" title="sec"> <div class="title" tagx="title" title="title">
Construction of <i>B. cereus</i> mutants</div> <p id="Par22"><i>B. cereus</i> mutants were obtained by electroporation using the plasmid pMAD<sup><a href="#CR66">66</a></sup>. The primers used to generate the mutagenesis constructs are listed in Annex Table <a href="#MOESM1">S2</a>. The constructs were created by joining PCR. In the first step, regions flanking the target genes were amplified separately, purified, and used for the joining PCRs. These PCR products were digested and cloned into the pMAD vector digested with the same enzymes. The resulting suicide plasmids were then used to transform <i>B. cereus</i> electrocompetent cells as described previously, with some modifications<sup><a href="#CR37">37</a></sup>. Electroporation was performed with 10 µg of plasmid in 100 µL of electrocompetent <i>B. cereus</i> cells in 0.2-cm cuvettes using the following electroporation parameters: voltage 1400 kV, capacitance 25 µF, resistance 400 Ω. After electroporation, the cells were incubated in LB for 5 h and then seeded on LB plates supplemented with X-Gal and erythromycin, followed by incubation for 72 h at 30 °C. Blue colonies were selected and streaked to trigger allele replacement. Finally, white, MLS-sensitive colonies were selected, and deletion of the target gene was verified by colony PCR analysis and sequencing of the amplicons.</p> </div> <div class="plasmidsforfluorescencemicroscopyanalysis" title="sec"> <div class="title" tagx="title" title="title">
Plasmids for fluorescence microscopy analysis</div> <p id="Par23">The plasmid pHCMC02 was slightly modified to constitutively express a fluorescent protein (either YFP or CFP)<sup><a href="#CR67">67</a></sup>. The original promoter in the plasmid was replaced by the constitutively expressed promoter of the <i>upp</i> gene from <i>B. cereus</i> using the primers listed in Annex Table <a href="#MOESM1">S3</a>. The WT strain and the <i>Δeps1, Δeps2</i> and double-mutant strains were transformed with the constructed vectors, and positive clones were isolated and checked for suitability. The images were collected using a Leica SP5 HyD MP Multiphoton/Confocal Microscope equipped with a tunable titanium-sapphire infrared laser capable of emitting between 700 and 1040 nm.</p> </div> <div class="biofilmassays" title="sec"> <div class="title" tagx="title" title="title">
Biofilm assays</div> <p id="Par24">The <i>B. cereus</i> biofilm assays were performed by assessing the bacterial adhesion to abiotic surfaces in 24-well plates and 250 mL flasks<sup><a href="#CR68">68</a></sup>. The cultures were grown in liquid TY medium at 30 °C with or without agitation. To visualize the population dynamics in the biofilm, the well adhered biomass (24-well plates) of each strain was recovered at different time points and resuspended in TY. The optical density at 600 nm was measured in a spectrophotometer and total number of viable cell bacteria was estimated and expressed as CFU/mL.</p> <p id="Par25">The Congo Red (CR) staining of the <i>B. cereus</i> colonies was performed as described above but using TY agar medium supplemented with CR and Coomassie Brilliant Blue G at final concentrations of 20 µg/ml and 10 µg/ml, respectively; the dyes were filtered and added to sterilized TY medium<sup><a href="#CR69">69</a></sup>.</p> </div> <div class="auto-aggregationassays" title="sec"> <div class="title" tagx="title" title="title">
Auto-aggregation assays</div> <p id="Par26">Flasks containing 25 ml of TY medium were inoculated with <i>B. cereus</i> and incubated with agitation at 200 rpm at 28 °C overnight. The cultures were divided into two fractions. The OD<sub>600</sub> values of the 10 ml culture aliquots for each strain were adjusted to 3 using the supernatant obtained from centrifugation of the other fraction. Auto-aggregation was measured using a final point measurement<sup><a href="#CR44">44</a></sup> and a kinetic method<sup><a href="#CR70">70</a></sup>.The tubes were then incubated vertically at RT without agitation or movement. At each sampling time, 10 µL aliquots were carefully taken from the air-liquid interface of the medium and diluted in 90 µL of TY medium to measure the OD<sub>600</sub> values in a plate reader (Omega).</p> </div> <div class="swarmingandswimmingmotilityassays" title="sec"> <div class="title" tagx="title" title="title">
Swarming and swimming motility assays</div> <p id="Par27">Petri dishes with TY medium containing 0.7% agar (for swarming) and 0.3% agar (for swimming) and 20 µg/ml of CR were used to assess colony expansion<sup><a href="#CR71">71</a></sup>. Bacterial cell suspensions at OD<sub>600</sub> = 1 were seeded in the centers of the plates in 5-µl drops. The plates were then incubated at 28 °C for 24, 48 and 72 h. The colony diameters were measured, and the data were statistically treated to obtain the means and SDs, n = 6–9.</p> </div> <div class="rt-pcrandqrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
RT-PCR and qRT-PCR</div> <p id="Par28">Total RNA was isolated following the protocol previously described<sup><a href="#CR36">36</a></sup>. The quality of the RNA extraction was tested by spectrometry using NanoVue GE equipment and electrophoresis in agarose gel. cDNA was obtained from the total RNA samples using the Titan One RT-PCR System (Roche). To determine the mRNA levels of the products expressed from the operons containing the <i>eps1</i> and <i>eps2</i> regions, PCRs were performed using the cDNA as a template and the primers listed in Annex Table <a href="#MOESM1">S4</a>. Positive controls for each primer pair were included using genomic DNA as the template, and for negative controls, the original RNA extraction was used as a template to ensure that the extracted RNA was not contaminated with genomic DNA.</p> <p id="Par29">For the quantitative real-time PCR, we measured the transcript levels of the <i>eps1</i> (<i>Bc5274</i>) and <i>eps2</i> (<i>Bc1583</i>) sequences at 28 and 37 °C using qRT-PCR. The reaction was performed using the Power SYBR Green Master Mix (Bio-RP) following the manufacturer’s recommendations. The reactions were performed in triplicate in 96-well plates in a reaction volume of 20 μL containing 1 μL of cDNA, 10 μL of SYBR Green Master Mix, 0.6 μL of a 10 nM solution of each primer, and 7.8 μL of ddH<sub>2</sub>O. The reaction was started with an initial denaturation at 95 °C for 3 min followed by 40 cycles of amplification: 95 °C for 20 s, 56 °C for 20 s and 72 °C for 30 s. The statistical significance was assessed by repeated measures ANOVA with post-hoc paired Student’s t test. Results with p &amp;lt; 0.05 were considered statistically significant.</p> </div> <div class="immunocytochemistryandtransmissionelectronmicroscopy" title="sec"> <div class="title" tagx="title" title="title">
Immunocytochemistry and transmission electron microscopy</div> <p id="Par30">Biofilm cells were harvested 48 h post-inoculation and gently resuspended in 1 ml of phosphate-buffered saline (PBS). Carbon-coated grids were placed on top of 20-μl samples for 1 h. The grid was washed in PBS for 5 min, fixed with a solution of 2% paraformaldehyde for 10 min, washed in PBS and blocked with bovine serum albumin (BSA)-PBS for 30 min. The monoclonal anti-TasA primary antibody was raised against a peptide (KGISAGKSDKFK) from the TasA sequence, which is different from that used by Caly (NovoPro). The anti-TasA antibody was used at a dilution of 1:50 in 2% BSA-PBS for 1 h at RT. The samples were washed three times with TBS (50 mM Tris-HCl, 150 mM NaCl, pH 7.5) -Tween20 (0.1%) for 5 min. The grids were then exposed to the secondary goat antirabbit antibody conjugated to 10-nm gold nanoparticles at a dilution 1:1000 for 1 h. The samples were then washed two times with TBS-Tween20 and in a drop of water for 5 min. Next, the grids were treated with 2% glutaraldehyde for 10 min, washed in water for 5 min, and then negatively stained with a 1% uranyl acetate solution. The samples were dried and examined using a FEI Tecnai G2 20 TWIN transmission electron microscope.</p> <p id="Par31">The immunofluorescence analysis was performed by blocking non-specific binding sites with 16% sheep serum, 1% BSA and 0.5% Triton X-100 in PBS (SBT) and incubating the tissue slides overnight at 4 °C in the primary antibody (GFP, 1:250 dilution). The samples were subsequently washed in PBS (3 × 5 min), incubated (2 h) in the proper FITC-conjugated secondary antibody (anti-chicken IgG, Jackson, 1:100 dilution) + Alexa Fluor 647 Phalloidin (Life technologies, 1:200 dilution) + DAPI (1:2000 dilution) and washed in PBS (3 × 5 min). All of the samples were mounted in a 1:1 PBS/glycerol solution and analyzed under a SP5 laser confocal microscope (LEICA).</p> </div> <div class="epsextraction" title="sec"> <div class="title" tagx="title" title="title">
EPS extraction</div> <p id="Par32">Exopolysaccharides substances were extracted following procedures previously published with modifications<sup><a href="#CR73">73</a></sup>. <i>B. cereus</i> WT and related mutants in the different EPS were growth overnight in Ty liquid culture and one milliliter was inoculated into 200 ml of fresh Ty liquid medium and incubated with no agitation at 30 °C for 5 days. Cells were harvested by centrifugation at 8000 × g for 10 min and washed with distilled water. After centrifugation, the cell pellets were resuspended in PBS buffer and were mildly sonicated (twice at 30% amplitude for 1 minute) to separate exopolysaccharides from cells. The soluble fractions were stirred on ice and a 100% TCA solution was added in order to precipitate proteins from the EPS extract. The mixture was left at 4 °C overnight and the proteins were removed at 10000 × g for 20 min. Exopolysaccharide from the supernatant were precipitated adding 5 volumes of chilled ethanol under continuous stirring and kept overnight for precipitation at 4 °C. EPS crude was collected by centrifugation (10000 × g, 20 min). The pellets were dialyzed against distilled water overnight (Spectra/Por<sup>®</sup> dialysis membrane molecular weight cutoff 3.5 kD) and the resulting solution was filtered and applied onto a size exclusion chromatography column (Hiprep<sup>TM</sup> 16/60 Sephacryl<sup>TM</sup> S-300HR) eluting with MQ water. Aiming the sugar presence into the fractions, aliquots of each fraction were analyzed by the pheno sulphuric acid method and monitored the absorption at 490 nm in a microplate reader (FLUOstar Omega reader, BMG LabTech). Fractions containing carbohydrates were pooled and lyophilized for further analysis.</p> </div> <div class="sugarcompositionanalysis(gc-ms)" title="sec"> <div class="title" tagx="title" title="title">
Sugar composition analysis (GC-MS)</div> <p id="Par33">Extracted samples (1-2 mg) of EPS were hydrolyzed with 1 mL of HCl/MeOH 3 N (sigma Aldrich) into a 3 ml reacti-vial (Thermo) at 80 °C for 24 h. The mixture was evaporated under N<sub>2</sub> stream at 50 °C. The residue was washed three times with MeOH and dried again under N<sub>2</sub> stream. Sililation of hydrolyzed samples were carried out with 300 µl of Tri-Sil reactive (Pierce, Thermo) for 1 h at 80 °C. Excess of the reactive were eliminated by N<sub>2</sub> stream and 1.5 ml of Hexane was added followed by centrifugation. Recovered supernatant was filtered, dried under N<sub>2</sub> stream and 150 µl of Hexane (LC-MS grade, Sigma) was added for Gas chromatography analysis. A Zebrón ZB-5 capillary column (30 m × 0.25 mm ID × 0.25 µm df, Phenomenex) was used to separate monosaccharides. Carrier gas flow was set at 1.2 ml/min and injection volume at 1 µl in split mode. The initial column temperature was set at 80 °C for 2 min followed by a temperature gradient of 10 °C/min until 180 °C and a 5 °C/min gradient until 250 °C and then the temperature was kept for 2 min. Ionization in mass spectrometer was performed by electron impact (EI) at 70 eV and ion source temperature at 230 °C. Full scan was set between 50-650 m/z.</p> </div> <div class="atr-ftir" title="sec"> <div class="title" tagx="title" title="title">
ATR-FTIR</div> <p id="Par34">In order to investigate the functional groups in the EPSs samples of <i>B. cereus</i> and mutants Attenuated Total Reflection Fourier Transform Infrared spectroscopy (ATR-FTIR) was used. Spectra of solid samples were recorded from 4000 cm<sup>−1</sup> to 500 cm<sup>−1</sup> on a Golden Gate Single Reflection Diamond ATR System (Graseby Specac) fitted into a Bruker Vertex 70 FT-IR spectrometer. Standard spectral resolution of 4 cm<sup>−1</sup> was used and 64 scans were selected for both background and sample measurements. Spectral contributions from residual water vapor were reduced using the atmospheric compensation filter built in the Bruker OPUS software.</p> </div> <div class="cellcultureadhesion" title="sec"> <div class="title" tagx="title" title="title">
Cell culture adhesion</div> <p id="Par35">Bacterial adhesion to human cells cultures was performed using a slightly modified protocol described before<sup><a href="#CR72">72</a></sup>. MDA-MB-231 breast adenocarcinoma cells and HeLa cervical cancer cells were obtained from the ATCC and grown in RPMI 1640 medium or DMEM supplemented with glucose (4.5 g/L), respectively, both containing glutamine (2 mM), penicillin (50 IU/mL), streptomycin (50 mg/L), amphotericin (1.25 mg/L), and 10% FBS at 37 °C in a 5% CO<sub>2</sub> atmosphere. Cells were seeded at 2000 and 1500 cells/well in 96-well plates and incubated for 72 h at 37 °C in a 5% CO<sub>2</sub> atmosphere to achieve confluence with a cell density of 1·10<sup>4</sup> cells/well. The medium from the HeLa and MDA cell cultures was replaced with "assay culture" medium (supplemented with glutamine and FBS, without antibiotics or antifungals), and the cells were incubated for 2 h. Next, the culture medium was replaced once again with assay culture medium before the bacterial inoculation. <i>B. cereus</i> ATCC14579 and the EPS mutants were streaked on LB agar plates and incubated for 24 h at 28 °C. Tubes containing 5 ml of TY medium were inoculated with <i>B. cereus</i> ATCC14579 and the EPS mutants and incubated overnight at 28 °C with vigorous shaking. The bacteria were washed three times with sterile PBS and the OD<sub>600</sub> values were adjusted to 1 (approx. 10<sup>7</sup> UCF/ml). Cultures of the human cells in 96-well plates were inoculated with 10 µl of bacterial suspensions at an MOI of 10:1. The plates were centrifuged at 2000 g for 1 min to force the bacteria into contact with the human cells and to avoid bias introduced by motility differences. After incubation for 45 min, the plates were washed 5 times with sterile PBS to remove the non-adhered bacteria. The cells were lysed with 0.1% Triton X-100 for 10 min, and serial dilutions were made from each well and plated on LB agar to titer the number of bacteria that adhered to the cells.</p> </div> <div class="danioreriogutadhesionassay" title="sec"> <div class="title" tagx="title" title="title">
Danio rerio <i>gut adhesion assay</i></div> <p id="Par36">Zebrafish larvae (6 dpf, days post-fertilization) were used for the experiment to assess the proportion of larvae maintaining <i>B. cereus</i> cells (introduced into the gut by feeding) 24 h post-feeding. In 50 ml tubes containing 30 ml of E3 medium without disinfectant, 30-40 larvae were fed with bacteria at a final concentration of 1.0e8 UCF/ml (approx. OD<sub>600</sub> = 1). The larvae were incubated at RT for 12 min. Next, the tubes were cooled on ice until the larvae settled to the bottoms to allow removal of the medium containing excess bacteria (repeated six times). The larvae were then microscopically observed individually, and 10-21 specimens with a visible mass of bacteria in the anterior intestine (intestinal bulb) were selected to continue the experiment. These larvae were incubated at 28 °C for 24 h. To prepare the larvae for confocal microscopy with a Nikon Eclipse Ti Fluorescence Microscope, they were incubated on ice for 3 min and then fixed in cold 4% PFA for 1 h. Microscopic observation was used to determine the rates of bacterial retention in the gut, and the retention rates were reported as the number bacteria-positive guts divided by the total number of observations. For the image analysis, ImageJ Fiji software was used to construct Z-stacks of the fluorescent images merged with the corresponding bright field images. The results were obtained from 5 replicates (each one using 10-21 larvae per treatment). Statistical analysis (ANOVA) showed significant differences between the groups. Experiments were carried out in accordance with Spanish Institutional Animal Use and Care Committee regulations and with the approval through the Regional Andalusian Government (code A/ES/14/43).</p> </div> <div class="ethicalapprovalandinformedconsent" title="sec"> <div class="title" tagx="title" title="title">
Ethical approval and informed consent</div> <p id="Par37">Experiments were carried out in accordance with Spanish Institutional Animal Use and Care Committee regulations and with the approval through the Regional Andalusian Government (code A/ES/14/43).</p> <p id="Par38">The methods were carried out in accordance with the relevant guidelines and regulations.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_57970_MOESM1_ESM.pdf">LINK</a> <p>Supplementary info.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p>is available for this paper at 10.1038/s41598-020-57970-3.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We would like to thank John Pearson (Bionand) for guidance and assistance with the confocal microscopy, Juan Felix (Bionand) for electron microscopy guidance, Miguel Ángel Medina Torres for providing the human cell lines, Reyes Molina García (Bionand) for managing and feeding the zebrafish, María Luisa Pola (SCI-UMA) for GC-MS analysis, and José Luis Zafra (SCI-UMA) for the ATR-FTIR analysis. Joaquín Caro-Astorga is the recipient of a FPI contract [BES-2013-064134] from the Ministerio de Economía y Competitividad. Juan Antonio Guadix acknowledges financial support from Plan Propio-Universidad de Málaga. This work was supported by grants [AGL2012-31968 and AGL2016-78662-R] from the Ministerio de Economía y Competitividad, the Spanish Government and the European Research Council Starting Grant under Grant [BacBio 637971].</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>D.R. conceived the study; D.R. and J.C.A. designed the experiments; J.C.A. performed the main experimental work; A.A.M. did motility experiments, transmission electron microscopy and participate in the rest of experimental work; Z.H.P. constructed some strains and participate in some experimental work; Y.A. and E.G.M. performed Zebra-fish experiments; J.A.G. performed the microscopy analysis of Zebra-Fish-<i>Bacillus</i> interaction; D.R. and J.C.A. wrote the manuscript; J.H. performed the chemical characterization of EPS; D.R., J.C.A., A.d.V., J.H. and J.A.G. contributed critically to writing the final version of the manuscript.</p> <div class="title" tagx="title" title="title"><b>Data availability</b></div> <p>All data generated or analyzed during this study are included in this published article (and its Supplementary Information files).</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par39">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">López, D., Vlamakis, H. &amp;amp; Kolter, R. Biofilms. <i>Cold Spring Harb Perspect Biol</i><b>2</b>, (2010).</span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irie</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Pseudomonas aeruginosa</i> biofilm matrix polysaccharide Psl is regulated transcriptionally by RpoS and post-transcriptionally by RsmA</span><span class="source" tagx="source" title="source">Mol. Microbiol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">158</span><span class="lpage" tagx="lpage" title="lpage">172</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20735777">20735777</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altman</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beveridge</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Speert</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Colonial Morphology of <i>Burkholderia cepacia</i> Complex Genomovar III: Implications in Exopolysaccharide Production, Pilus Expression, and Persistence in the Mouse</span><span class="source" tagx="source" title="source">Infect. Immun.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">904</span><span class="lpage" tagx="lpage" title="lpage">909</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/IAI.71.2.904-909.2003">10.1128/IAI.71.2.904-909.2003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12540572">12540572</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wang, C., Saito, M., Ogawa, M. &amp;amp; Yoshida, S. Colony types and virulence traits of <i>Legionella feeleii</i> determined by exopolysaccharide materials. <i>FEMS Microbiol. Lett.</i><b>363</b>, (2016).</span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Limoli, D. H., Jones, C. J. &amp;amp; Wozniak, D. J. Bacterial Extracellular Polysaccharides in Biofilm Formation and Function. <i>Microbiol Spectr</i><b>3</b>, (2015).</span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dertli, E., Mayer, M. J. &amp;amp; Narbad, A. Impact of the exopolysaccharide layer on biofilms, adhesion and resistance to stress in <i>Lactobacillus johnsonii</i> FI9785. <i>BMC Microbiol.</i><b>15</b>, (2015).</span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shea</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nunley</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williamson</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith-Somerville</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of the adhesion properties of <i>Deleya marina</i> and the exopolysaccharide-defective mutant strain DMR</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">3107</span><span class="lpage" tagx="lpage" title="lpage">3113</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.57.11.3107-3113.1991">10.1128/AEM.57.11.3107-3113.1991</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1781675">1781675</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Labbate</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Quorum-Sensing Regulation of Adhesion in <i>Serratia marcescens</i> MG1 Is Surface Dependent</span><span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">189</span><span class="fpage" tagx="fpage" title="fpage">2702</span><span class="lpage" tagx="lpage" title="lpage">2711</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.01582-06">10.1128/JB.01582-06</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237163">17237163</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva</span><span class="given-names" tagx="given-names" title="given-names">VO</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biofilm Formation on Biotic and Abiotic Surfaces in the Presence of Antimicrobials by <i>Escherichia coli</i> Isolates from Cases of Bovine Mastitis</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">80</span><span class="fpage" tagx="fpage" title="fpage">6136</span><span class="lpage" tagx="lpage" title="lpage">6145</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.01953-14">10.1128/AEM.01953-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25063668">25063668</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Berne, C., Ducret, A., Hardy, G. G. &amp;amp; Brun, Y. V. Adhesins involved in attachment to abiotic surfaces by Gram-negative bacteria. <i>Microbiol Spectr</i><b>3</b>, (2015).</span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorken</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">GP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">French</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">WCK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Aggregation by depletion attraction in cultures of bacteria producing exopolysaccharide</span><span class="source" tagx="source" title="source">J. R. Soc. Interface</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">3490</span><span class="lpage" tagx="lpage" title="lpage">3502</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2012.0498">10.1098/rsif.2012.0498</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22896568">22896568</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutherland</span><span class="given-names" tagx="given-names" title="given-names">IW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exopolysaccharides in biofilms, flocs and related structures</span><span class="source" tagx="source" title="source">Water Sci. Technol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">77</span><span class="lpage" tagx="lpage" title="lpage">86</span><span class="pub-id"><a href="https://dx.doi.org/10.2166/wst.2001.0345">10.2166/wst.2001.0345</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11381975">11381975</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Exopolysaccharide biosynthesis genes required for social motility in <i>Myxococcus xanthus</i></span><span class="source" tagx="source" title="source">Mol. Microbiol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">206</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2958.2004.04369.x">10.1111/j.1365-2958.2004.04369.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15612929">15612929</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berleman</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Exopolysaccharide microchannels direct bacterial motility and organize multicellular behavior</span><span class="source" tagx="source" title="source">ISME J.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">2620</span><span class="lpage" tagx="lpage" title="lpage">2632</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ismej.2016.60">10.1038/ismej.2016.60</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27152937">27152937</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibiansky</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahmen</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">GCL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Earthquake-like dynamics in <i>Myxococcus xanthus</i> social motility</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">110</span><span class="fpage" tagx="fpage" title="fpage">2330</span><span class="lpage" tagx="lpage" title="lpage">2335</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1215089110">10.1073/pnas.1215089110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23341622">23341622</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hu, W. <i>et al</i>. Interplay between type IV pili activity and exopolysaccharides secretion controls motility patterns in single cells of <i>Myxococcus xanthus</i>. <i>Sci Rep</i><b>6</b>, (2016).</span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Liu, A. <i>et al</i>. Exopolysaccharides Play a Role in the Swarming of the Benthic Bacterium <i>Pseudoalteromonas</i> sp. SM9913. <i>Front Microbiol</i><b>7</b>, (2016).</span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casillo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure-activity relationship of the exopolysaccharide from a psychrophilic bacterium: a strategy for cryoprotection</span><span class="source" tagx="source" title="source">Carbohydr. Polym.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">156</span><span class="fpage" tagx="fpage" title="fpage">364</span><span class="lpage" tagx="lpage" title="lpage">371</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.carbpol.2016.09.037">10.1016/j.carbpol.2016.09.037</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27842835">27842835</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberson</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Firestone</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Relationship between Desiccation and Exopolysaccharide Production in a Soil <i>Pseudomonas</i> sp</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">1284</span><span class="lpage" tagx="lpage" title="lpage">1291</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.58.4.1284-1291.1992">10.1128/AEM.58.4.1284-1291.1992</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16348695">16348695</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnider-Keel</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lejbølle</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baehler</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">The sigma factor AlgU (AlgT) controls exopolysaccharide production and tolerance towards desiccation and osmotic stress in the biocontrol agent <i>Pseudomonas fluorescens</i> CHA0</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">5683</span><span class="lpage" tagx="lpage" title="lpage">5693</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.67.12.5683-5693.2001">10.1128/AEM.67.12.5683-5693.2001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11722923">11722923</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ionescu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belkin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Overproduction of Exopolysaccharides by an <i>Escherichia coli</i> K-12 rpoS Mutant in Response to Osmotic Stress</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">483</span><span class="lpage" tagx="lpage" title="lpage">492</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.01616-08">10.1128/AEM.01616-08</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18997023">18997023</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Two novel exopolysaccharides from <i>Bacillus amyloliquefaciens</i> C-1: antioxidation and effect on oxidative stress</span><span class="source" tagx="source" title="source">Curr. Microbiol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">298</span><span class="lpage" tagx="lpage" title="lpage">306</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00284-014-0717-2">10.1007/s00284-014-0717-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25362510">25362510</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roach</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sjaarda</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castle</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svircev</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Host Exopolysaccharide Quantity and Composition Impact <i>Erwinia amylovora</i> Bacteriophage Pathogenesis</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">3249</span><span class="lpage" tagx="lpage" title="lpage">3256</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.00067-13">10.1128/AEM.00067-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23503310">23503310</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drulis-Kawa</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Majkowska-Skrobek</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maciejewska</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delattre</span><span class="given-names" tagx="given-names" title="given-names">A-S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavigne</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Learning from Bacteriophages - Advantages and Limitations of Phage and Phage-Encoded Protein Applications</span><span class="source" tagx="source" title="source">Curr. Protein Pept. Sci.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">699</span><span class="lpage" tagx="lpage" title="lpage">722</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/138920312804871193">10.2174/138920312804871193</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23305359">23305359</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scholl</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adhya</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merril</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>Escherichia coli</i> K1’s Capsule Is a Barrier to Bacteriophage T7</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">4872</span><span class="lpage" tagx="lpage" title="lpage">4874</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.71.8.4872-4874.2005">10.1128/AEM.71.8.4872-4874.2005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085886">16085886</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">S. Bragadeeswaran. Exopolysaccharide production by <i>Bacillus cereus</i> GU812900, a fouling marine bacterium. <i>African Journal of Microbiology Research</i><b>5</b>, (2011).</span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Whitfield, G. B., Marmont, L. S. &amp;amp; Howell, P. L. Enzymatic modifications of exopolysaccharides enhance bacterial persistence. <i>Front Microbiol</i><b>6</b>, (2015).</span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okshevsky</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A transposon mutant library of <i>Bacillus cereus</i> ATCC 10987 reveals novel genes required for biofilm formation and implicates motility as an important factor for pellicle‐biofilm formation</span><span class="source" tagx="source" title="source">MicrobiologyOpen</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">0</span><span class="fpage" tagx="fpage" title="fpage">e00552</span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Römling</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galperin</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial cellulose biosynthesis: diversity of operons, subunits, products and functions</span><span class="source" tagx="source" title="source">Trends Microbiol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">545</span><span class="lpage" tagx="lpage" title="lpage">557</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2015.05.005">10.1016/j.tim.2015.05.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26077867">26077867</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houry</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial swimmers that infiltrate and take over the biofilm matrix</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">13088</span><span class="lpage" tagx="lpage" title="lpage">13093</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1200791109">10.1073/pnas.1200791109</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22773813">22773813</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Li, Z. <i>et al</i>. A Four-gene Operon in <i>Bacillus cereus</i> Produces Two Rare Spore-decorating Sugars. <i>Journal of Biological Chemistry</i> jbc.M117.777417, 10.1074/jbc.M117.777417 (2017).</span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forsberg</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Secondary cell wall polysaccharides from <i>Bacillus cereus</i> strains G9241, 03BB87 and 03BB102 causing fatal pneumonia share similar glycosyl structures with the polysaccharides from <i>Bacillus anthracis</i></span><span class="source" tagx="source" title="source">Glycobiol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">934</span><span class="lpage" tagx="lpage" title="lpage">948</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/glycob/cwr026">10.1093/glycob/cwr026</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elsholz</span><span class="given-names" tagx="given-names" title="given-names">AKW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wacker</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Losick</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Self-regulation of exopolysaccharide production in <i>Bacillus subtilis</i> by a tyrosine kinase</span><span class="source" tagx="source" title="source">Genes. Dev.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">1710</span><span class="lpage" tagx="lpage" title="lpage">1720</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gad.246397.114">10.1101/gad.246397.114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25085422">25085422</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Branda</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez-Pastor</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben-Yehuda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Losick</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolter</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fruiting body formation by <i>Bacillus subtilis</i></span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">11621</span><span class="lpage" tagx="lpage" title="lpage">11626</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.191384198">10.1073/pnas.191384198</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11572999">11572999</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foulston</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Losick</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Alternative modes of biofilm formation by plant-associated <i>Bacillus cereus</i></span><span class="source" tagx="source" title="source">MicrobiologyOpen</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">452</span><span class="lpage" tagx="lpage" title="lpage">464</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/mbo3.251">10.1002/mbo3.251</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25828975">25828975</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Caro-Astorga, J. <i>et al</i>. Biofilm formation displays intrinsic offensive and defensive features of <i>Bacillus cereus</i>. <i>bioRxiv</i> 676403, 10.1101/676403 (2019).</span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Caro-Astorga, J., Pérez-García, A., Vicente, A. &amp;amp; Romero, D. A genomic region involved in the formation of adhesin fibers in <i>Bacillus cereus</i> biofilms. <i>Front Microbiol</i><b>5</b>, (2015).</span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKinney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Staining Bacterial Polysaccharides1</span><span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">1953</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">453</span><span class="lpage" tagx="lpage" title="lpage">454</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.66.4.453-454.1953">10.1128/JB.66.4.453-454.1953</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13096503">13096503</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">Y-G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S-K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Embedded Biofilm, a New Biofilm Model Based on the Embedded Growth of Bacteria</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">211</span><span class="lpage" tagx="lpage" title="lpage">219</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.02311-14">10.1128/AEM.02311-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25326307">25326307</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DuBois</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilles</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamilton</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rebers</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Colorimetric Method for Determination of Sugars and Related Substances</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">1956</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">350</span><span class="lpage" tagx="lpage" title="lpage">356</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ac60111a017">10.1021/ac60111a017</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Omoike</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chorover</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spectroscopic study of extracellular polymeric substances from <i>Bacillus subtilis</i>: aqueous chemistry and adsorption effects</span><span class="source" tagx="source" title="source">Biomacromolecules</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1219</span><span class="lpage" tagx="lpage" title="lpage">1230</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/bm034461z">10.1021/bm034461z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15244434">15244434</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sardari</span><span class="given-names" tagx="given-names" title="given-names">RRR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation of the production of exopolysaccharides by two strains of the thermophilic bacterium <i>Rhodothermus marinus</i></span><span class="source" tagx="source" title="source">Carbohydr. Polym.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">156</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.carbpol.2016.08.062">10.1016/j.carbpol.2016.08.062</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27842803">27842803</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijayabaskar</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babinastarlin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shankar</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivakumar</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anandapandian</span><span class="given-names" tagx="given-names" title="given-names">KTK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantification and Characterization of Exopolysaccharides from <i>Bacillus subtilis</i> (MTCC 121)</span><span class="source" tagx="source" title="source">Adv. Biol. Res.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">71</span><span class="lpage" tagx="lpage" title="lpage">76</span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sorroche</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spesia</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zorreguieta</span><span class="given-names" tagx="given-names" title="given-names">Á</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giordano</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">A Positive Correlation between Bacterial Autoaggregation and Biofilm Formation in Native <i>Sinorhizobium meliloti</i> Isolates from Argentina</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">4092</span><span class="lpage" tagx="lpage" title="lpage">4101</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.07826-11">10.1128/AEM.07826-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22492433">22492433</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chauhan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakamoto</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghigo</span><span class="given-names" tagx="given-names" title="given-names">J-M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beloin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Did I Pick the Right Colony? Pitfalls in the Study of Regulation of the Phase Variable Antigen 43 Adhesin</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e73568</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0073568">10.1371/journal.pone.0073568</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24039985">24039985</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beloin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghigo</span><span class="given-names" tagx="given-names" title="given-names">J-M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Combined Inactivation and Expression Strategy To Study Gene Function under Physiological Conditions: Application to Identification of New <i>Escherichia coli</i> Adhesins</span><span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">187</span><span class="fpage" tagx="fpage" title="fpage">1001</span><span class="lpage" tagx="lpage" title="lpage">1013</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.187.3.1001-1013.2005">10.1128/JB.187.3.1001-1013.2005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659678">15659678</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Garcia-Betancur, J. C., Yepes, A., Schneider, J. &amp;amp; Lopez, D. Single-cell analysis of <i>Bacillus subtilis</i> biofilms using fluorescence microscopy and flow cytometry. <i>J Vis Exp</i>10.3791/3796. (2012)</span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brugman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">The zebrafish as a model to study intestinal inflammation</span><span class="source" tagx="source" title="source">Developmental Comp. Immunology</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">82</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.dci.2016.02.020">10.1016/j.dci.2016.02.020</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hidalgo-Cantabrana</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic Overview and Biological Functions of Exopolysaccharide Biosynthesis in <i>Bifidobacterium</i> spp</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">80</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.02977-13">10.1128/AEM.02977-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24123746">24123746</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sims</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure and functions of exopolysaccharide produced by gut commensal <i>Lactobacillus reuteri</i> 100-23</span><span class="source" tagx="source" title="source">ISME J.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1115</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ismej.2010.201">10.1038/ismej.2010.201</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21248858">21248858</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bar-Peled</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of a Unique Extracellular Polysaccharide in Members of the Pathogenic <i>Bacillus</i> That Can Co-form with Spores</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">291</span><span class="fpage" tagx="fpage" title="fpage">19051</span><span class="lpage" tagx="lpage" title="lpage">19067</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M116.724708">10.1074/jbc.M116.724708</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27402849">27402849</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Candela</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Environmental and Biofilm-dependent Changes in a <i>Bacillus cereus</i> Secondary Cell Wall Polysaccharide</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">286</span><span class="fpage" tagx="fpage" title="fpage">31250</span><span class="lpage" tagx="lpage" title="lpage">31262</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M111.249821">10.1074/jbc.M111.249821</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21784857">21784857</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horn</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Spontaneous mutation reveals influence of exopolysaccharide on <i>Lactobacillus johnsonii</i> surface characteristics</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e59957</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0059957">10.1371/journal.pone.0059957</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23544114">23544114</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aslim</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onal</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beyatli</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Factors influencing autoaggregation and aggregation of <i>Lactobacillus delbrueckii</i> subsp. bulgaricus isolated from handmade yogurt</span><span class="source" tagx="source" title="source">J. Food Prot.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">223</span><span class="lpage" tagx="lpage" title="lpage">227</span><span class="pub-id"><a href="https://dx.doi.org/10.4315/0362-028X-70.1.223">10.4315/0362-028X-70.1.223</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17265886">17265886</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luckevich</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beveridge</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of a dynamic S layer on <i>Bacillus thuringiensis</i></span><span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">1989</span><span class="volume" tagx="volume" title="volume">171</span><span class="fpage" tagx="fpage" title="fpage">6656</span><span class="lpage" tagx="lpage" title="lpage">6667</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.171.12.6656-6667.1989">10.1128/JB.171.12.6656-6667.1989</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2592346">2592346</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belas</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biofilms, flagella, and mechanosensing of surfaces by bacteria</span><span class="source" tagx="source" title="source">Trends Microbiol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">517</span><span class="lpage" tagx="lpage" title="lpage">527</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2014.05.002">10.1016/j.tim.2014.05.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24894628">24894628</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moens</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vanderleyden</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functions of bacterial flagella</span><span class="source" tagx="source" title="source">Crit. Rev. Microbiol.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">67</span><span class="lpage" tagx="lpage" title="lpage">100</span><span class="pub-id"><a href="https://dx.doi.org/10.3109/10408419609106456">10.3109/10408419609106456</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8817078">8817078</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ottemann</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Roles for motility in bacterial-host interactions</span><span class="source" tagx="source" title="source">Mol. Microbiol.</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1109</span><span class="lpage" tagx="lpage" title="lpage">1117</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2958.1997.4281787.x">10.1046/j.1365-2958.1997.4281787.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9218761">9218761</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghelardi</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Contribution of Surfactin and SwrA to Flagellin Expression, Swimming, and Surface Motility in <i>Bacillus subtilis</i></span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">6540</span><span class="lpage" tagx="lpage" title="lpage">6544</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.01341-12">10.1128/AEM.01341-12</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22773650">22773650</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagorska</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostrowski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hinc</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holland</span><span class="given-names" tagx="given-names" title="given-names">IB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obuchowski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Importance of eps genes from <i>Bacillus subtilis</i> in biofilm formation and swarming</span><span class="source" tagx="source" title="source">J. Appl. Genet.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">369</span><span class="lpage" tagx="lpage" title="lpage">381</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF03208867">10.1007/BF03208867</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20720312">20720312</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovács</span><span class="given-names" tagx="given-names" title="given-names">ÁT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grau</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pollitt</span><span class="given-names" tagx="given-names" title="given-names">EJG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Surfing of bacterial droplets: <i>Bacillus subtilis</i> sliding revisited</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">E8802</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1710371114">10.1073/pnas.1710371114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29073022">29073022</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Grau, R. R. <i>et al</i>. A Duo of Potassium-Responsive Histidine Kinases Govern the Multicellular Destiny of <i>Bacillus subtilis</i>. <i>mBio</i><b>6</b>, (2015).</span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanesaki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshikawa</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic Analysis of Collective Motility of <i>Paenibacillus</i> sp. NAIST15-1</span><span class="source" tagx="source" title="source">PLOS Genet.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">e1006387</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1006387">10.1371/journal.pgen.1006387</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27764113">27764113</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pathogenic Potential of <i>Bacillus cereus</i> Strains as Revealed by Phenotypic Analysis</span><span class="source" tagx="source" title="source">J. Clin. Microbiol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">320</span><span class="lpage" tagx="lpage" title="lpage">323</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.02848-12">10.1128/JCM.02848-12</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23135929">23135929</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paynich</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones-Burrage</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knight</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exopolysaccharide from <i>Bacillus subtilis</i> Induces Anti-Inflammatory M2 Macrophages That Prevent T Cell-Mediated Disease</span><span class="source" tagx="source" title="source">J. Immunol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">198</span><span class="fpage" tagx="fpage" title="fpage">2689</span><span class="lpage" tagx="lpage" title="lpage">2698</span><span class="pub-id"><a href="https://dx.doi.org/10.4049/jimmunol.1601641">10.4049/jimmunol.1601641</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28202619">28202619</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnaud</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chastanet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Débarbouillé</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">New Vector for Efficient Allelic Replacement in Naturally Nontransformable, Low-GC-Content, Gram-Positive Bacteria</span><span class="source" tagx="source" title="source">Appl. Env. Microbiol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">6887</span><span class="lpage" tagx="lpage" title="lpage">6891</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.70.11.6887-6891.2004">10.1128/AEM.70.11.6887-6891.2004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15528558">15528558</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">HD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Construction of plasmid-based expression vectors for <i>Bacillus subtilis</i> exhibiting full structural stability</span><span class="source" tagx="source" title="source">Plasmid</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">241</span><span class="lpage" tagx="lpage" title="lpage">248</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plasmid.2005.05.001">10.1016/j.plasmid.2005.05.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16005967">16005967</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Toole</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic approaches to study of biofilms</span><span class="source" tagx="source" title="source">Meth. Enzymol.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">310</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">109</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0076-6879(99)10008-9">10.1016/S0076-6879(99)10008-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547784">10547784</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vlamakis</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Losick</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolter</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional analysis of the accessory protein TapA in <i>Bacillus subtilis</i> amyloid fiber assembly</span><span class="source" tagx="source" title="source">J. Bacteriol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">196</span><span class="fpage" tagx="fpage" title="fpage">1505</span><span class="lpage" tagx="lpage" title="lpage">1513</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.01363-13">10.1128/JB.01363-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24488317">24488317</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chauhan, P. S., Goradia, B. &amp;amp; Saxena, A. Bacterial laccase: recent update on production, properties and industrial applications. <i>3 Biotech</i><b>7</b>, (2017).</span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harshey</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial Motility on a Surface: Many Ways to a Common Goal</span><span class="source" tagx="source" title="source">Annu. Rev. Microbiology</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">57</span><span class="fpage" tagx="fpage" title="fpage">249</span><span class="lpage" tagx="lpage" title="lpage">273</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.micro.57.030502.091014">10.1146/annurev.micro.57.030502.091014</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Salehi, S. <i>et al</i>. <i>Salmonella enterica</i> Serovar Kentucky Flagella Are Required for Broiler Skin Adhesion and Caco-2 Cell Invasion. <i>Appl. Environ. Microbiol</i>. <b>83</b>, (2017).</span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauregard</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vlamakis</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Losick</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolter</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Galactose Metabolism Plays a Crucial Role in Biofilm Formation by <i>Bacillus subtilis</i></span><span class="source" tagx="source" title="source">MBio.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e00184</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00184-12">10.1128/mBio.00184-12</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22893383">22893383</a></span></span></li> </ul> </div> </div>  </body></html>