<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7142146">7142146</a></span><span class="publisher-id" title="publisher-id">62997</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-020-62997-7">10.1038/s41598-020-62997-7</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Dynamic spherical harmonics approach for shape classification of migrating cells</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medyukhina</span><span class="given-names" tagx="given-names" title="given-names">Anna</span></span><a href="#Aff1">1</a><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blickensdorf</span><span class="given-names" tagx="given-names" title="given-names">Marco</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cseresnyés</span><span class="given-names" tagx="given-names" title="given-names">Zoltán</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruef</span><span class="given-names" tagx="given-names" title="given-names">Nora</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stein</span><span class="given-names" tagx="given-names" title="given-names">Jens V.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figge</span><span class="given-names" tagx="given-names" title="given-names">Marc Thilo</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">thilo.figge@leibniz-hki.de</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a><a href="#Aff4">4</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0143 807X</span><span class="institution-id" title="institution-id">grid.418398.f</span><span class="institution" title="institution">Applied Systems Biology, Leibniz Institute for Natural Product Research and Infection Biology – Hans Knöll Institute (HKI), </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 1939 2794</span><span class="institution-id" title="institution-id">grid.9613.d</span><span class="institution" title="institution">Institute of Microbiology, Faculty of Biological Sciences, </span><span class="institution" title="institution">Friedrich Schiller University Jena, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0478 1713</span><span class="institution-id" title="institution-id">grid.8534.a</span><span class="institution" title="institution">Department of Oncology, Microbiology and Immunology, </span><span class="institution" title="institution">University of Fribourg, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0000 8517 6224</span><span class="institution-id" title="institution-id">grid.275559.9</span><span class="institution" title="institution">Center for Sepsis Control and Care (CSCC), </span><span class="institution" title="institution">Jena University Hospital, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0224 711X</span><span class="institution-id" title="institution-id">grid.240871.8</span><span class="institution" title="institution">Present Address: Center for Bioimage Informatics, St. Jude Children’s Research Hospital, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">6072</span><span class="history" title="history"><span class="received" title="received">received: 2019-9-19</span><span class="accepted" title="accepted">accepted: 2020-3-24</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Cell migration involves dynamic changes in cell shape. Intricate patterns of cell shape can be analyzed and classified using advanced shape descriptors, including spherical harmonics (SPHARM). Though SPHARM have been used to analyze and classify migrating cells, such classification did not exploit SPHARM spectra in their dynamics. Here, we examine whether additional information from dynamic SPHARM improves classification of cell migration patterns. We combine the static and dynamic SPHARM approach with a support-vector-machine classifier and compare their classification accuracies. We demonstrate that the dynamic SPHARM analysis classifies cell migration patterns more accurately than the static one for both synthetic and experimental data. Furthermore, by comparing the computed accuracies with that of a naive classifier, we can identify the experimental conditions and model parameters that significantly affect cell shape. This capability should – in the future – help to pinpoint factors that play an essential role in cell migration.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Computational biology and bioinformatics</div> <div class="kwd" title="kwd">
Computational biophysics</div> <div class="kwd" title="kwd">
Signal processing</div> <div class="kwd" title="kwd">
Time series</div> <div class="kwd" title="kwd">
Cell migration</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">A cell’s migration behavior depends on the state of the cell, extracellular environment, and signals from other cells<sup><a href="#CR1">1</a></sup>. We can study the mechanisms of cell migration by, e.g., knocking out a certain gene or altering the structure of the extracellular matrix (ECM) and testing whether these changes affect cell migration patterns, such as cell trajectory, shape, or shape dynamics (Fig. <a href="#Fig1">1</a>). To compare migration patterns in an objective and statistically sound way, they have to be automatically analyzed and quantified<sup><a href="#CR2">2</a></sup>. Whereas both cell trajectories<sup><a href="#CR3">3</a>,<a href="#CR4">4</a></sup> and cell shape<sup><a href="#CR5">5</a>,<a href="#CR6">6</a></sup> can be quantified with a multitude of available methods, the analysis of shape dynamics – especially in 3D – received considerably less attention. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Analysis and quantification of migration patterns can help to study cell migration mechanisms. First, a migration experiment is performed, which involves altering the migrating cells (genetic or chemical modification, choosing cells of a different type), altering the extracellular matrix (ECM), or adding chemical signal to the extracellular environment. Next, cell migration patterns are analyzed and quantified. Finally, one or more quantitative measures derived from these patterns are statistically compared between conditions to detect significant differences.</p>   </div> <p /> <p id="Par3">When analyzing cell shape in a static fashion, we look at just one snapshot of the cell’s migration history. Depending on how we choose this snapshot, we may either miss important differences in cell shape – e.g., if cells transiently appear similar but have different migration patterns – or detect spurious differences – e.g., if cells occur in different phases of the same migration pattern. Even averaging cell shape descriptors over time<sup><a href="#CR7">7</a></sup> may not always be sufficient to distinguish some migration patterns, for example when all cells evolve through similar phases of cell shape but different cells do this with different frequencies (Fig. <a href="#MOESM1">S1</a>)<sup><a href="#CR8">8</a></sup>. To distinguish such details of migration behavior we need dynamic shape analysis that takes into account relative changes in cell shape between consecutive time points.</p> <p id="Par4">While such dynamic shape analysis has been done in 2D<sup><a href="#CR8">8</a>–<a href="#CR11">11</a></sup>, 3D shape descriptors have not been applied to characterize and compare the full dynamic migration patterns of cells. The ultimate goal, however, is to understand how cells migrate in living organisms<sup><a href="#CR12">12</a></sup>. Due to advances in intravital microscopy<sup><a href="#CR13">13</a>–<a href="#CR15">15</a></sup>, we have increasingly more 3D + time data of cells migrating <i>in vivo</i> and we should exploit the potential of 3D methods to analyze these data<sup><a href="#CR16">16</a></sup>.</p> <p id="Par5">Although there are many simple shape descriptors that can be applied in 3D (e.g., solidity, ellipticity, prolateness), only relatively complex ones can reveal the fine details of the cell shape and classify between relevant spatial patterns while ignoring random shape variations<sup><a href="#CR7">7</a></sup>. One especially popular and promising approach involves spherical harmonics (SPHARM)<sup><a href="#CR17">17</a>–<a href="#CR19">19</a></sup>. SPHARM is a 3D extension of a Fourier analysis, where an arbitrary shape function is expanded on a sphere using a set of orthogonal spherical functions as a basis. This approach was shown to be effective for characterizing the shape of proteins<sup><a href="#CR20">20</a>,<a href="#CR21">21</a></sup>, red blood cells<sup><a href="#CR22">22</a>,<a href="#CR23">23</a></sup>, brain structures<sup><a href="#CR19">19</a>,<a href="#CR24">24</a>,<a href="#CR25">25</a></sup>, as well as migrating cells<sup><a href="#CR7">7</a>,<a href="#CR26">26</a>–<a href="#CR28">28</a></sup>. In the contexts of cell migration analysis, SPHARM have been applied to identify phases of amoeboid cell motion<sup><a href="#CR26">26</a>–<a href="#CR28">28</a></sup> and to classify shapes of migrating cells based on SPHARM spectra averaged over time<sup><a href="#CR7">7</a></sup>. Therefore, SPHARM descriptors represent an ideal first candidate to be extended for dynamic 3D shape analysis.</p> <p id="Par6">In this proof-of-principle study, we investigate, whether the use of dynamic shape descriptors can improve classification between migration patterns of cells. We extend the SPHARM analysis by computing dynamic SPHARM descriptors, combine both descriptors with a support-vector-machine classifier, and compare their ability to distinguish between migration patterns of cells in synthetic and experimental data.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <p id="Par7">To study the use of dynamic SPHARM for classifying migrating cells, we analyzed two types of input data: synthetic cells generated with an in-house developed cell migration simulator (CMS), and T cells visualized with intravital microscopy. For each cell, we extracted cell surfaces at various time points and transformed them into a static or dynamic SPHARM feature vector. We then used the computed feature vectors to classify cells according to their migration behavior.</p> <div class="cellmigrationsimulator" title="sec"> <div class="title" tagx="title" title="title">
Cell migration simulator</div> <p id="Par8">To generate synthetic migrating cells, we used our previously developed cell migration simulator (CMS)<sup><a href="#CR29">29</a></sup>. In CMS, each cell consists of a set of grid-based spatial units (SU), and the cell migration in 3D is simulated by iteratively moving SU from the rear of the cell to the front (Fig. <a href="#Fig2">2a</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Synthetic and real migrating cells analyzed in this study. (<b>a</b>) Schematic overview of a 2D version of the cell migration simulator; the simulation starts with a spherical cell consisting of the pixel-based spatial units (SU); for each SU, we compute a position vector <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\overrightarrow{{v}_{P}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div> relative to the center of mass of the cell; we randomly choose the migration direction <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\overrightarrow{{v}_{D}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span> and define the cell’s front and rear perpendicularly to <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\overrightarrow{{v}_{D}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span>; at each iteration, we take one SU from the rear surface and move it to the front, which causes the cell to shift its center of mass; pale yellow SU indicate the candidates for removal, whereas pale purple SU indicate the free positions at the cell front. (<b>b</b>) Example of a cell generated with the 3D version of the cell migration simulator; gray shades designate the cell positions at previous time points. (<b>c</b>) Upper row: Deconvolved images of CD8<sup>+</sup> T cells migrating in a lymph node (LN), submandibular salivary gland (SMG) and skin shown in Maximum Intensity Projection (MIP) mode; Lower row: individual representative cells in higher magnification zoom-in from the same groups as in the upper row in 3D Surface Rendering mode in Imaris.<p />   </div> <p /> <p id="Par9">The simulation starts with a spherical cell with the center of mass at position <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${p}_{init}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div> and a randomly chosen migration direction <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\overrightarrow{{v}_{D}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span> (Fig. <a href="#Fig2">2a</a>). Each SU of the cell receives a position vector <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\overrightarrow{\,{v}_{P}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span>, which is used to compute the relative SU position <i>P</i> as the dot product between the normal vectors <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\overrightarrow{{v}_{P}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\overrightarrow{\,{v}_{D}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span>: <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$P=\overrightarrow{{v}_{D}}\cdot \overrightarrow{{v}_{P}}\in [-1,1]$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
=</div> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div> <div class="mo_UNKNOWN">
⋅</div> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
→</div> </div> </div> <div class="mo_UNKNOWN">
∈</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
]</div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span>. The value of <i>P</i> defines whether the corresponding SU belongs to the cell’s rear or front: for all front SU, <i>P</i> must be greater than a pre-defined front-rear threshold <i>FR</i>, whereas all SU with <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M19">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\,P\le FR\,$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
R</div> <div class="mspace_UNKNOWN" /></div><span class="inline-graphic" title="inline-graphic" /></div></span>belong to the cell’s rear. The parameter <i>FR</i> defines the fraction of the cell volume considered as the front. Thus, for <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M21">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$FR=0$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div></div><span class="inline-graphic" title="inline-graphic" /></div></span> the front and the back have equal weights; for a negative <i>FR</i> the front is wider than the back, and for a positive <i>FR</i> the front is narrower than the back (Fig. <a href="#MOESM1">S2a</a>).<p /> <p id="Par10">In addition to <i>FR</i>, the model has three further parameters: neighbor weight (NW), position weight (PW), and distance weight (DW), which define the probability of moving a specific SU from the rear to the front surface. In each iteration, an SU from the rear surface and a free position at the cell’s front are chosen randomly using Monte Carlo acceptance-rejection sampling. The acceptance probability for each SU depends on the number of neighbors <i>N</i>, position <i>P</i>, and the distance to the center of mass <i>D</i> (Fig. <a href="#MOESM1">S2b–d</a>) and is computed based on the rear and front scores <i>S</i><sub><i>r</i></sub> and <i>S</i><sub><i>f</i></sub>: <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M23">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\,{S}_{r}={|P|}^{{\rm{PW}}}\cdot {(6-N)}^{{\rm{NW}}}\cdot {D}^{{\rm{DW}}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
PW</div> </div> </div> <div class="mo_UNKNOWN">
⋅</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
6</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
N</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
NW</div> </div> </div> <div class="mo_UNKNOWN">
⋅</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
DW</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div>, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M25">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${S}_{f}={|P|}^{{\rm{PW}}}\cdot {N}^{{\rm{NW}}}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
PW</div> </div> </div> <div class="mo_UNKNOWN">
⋅</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
NW</div> </div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span>. Thus, those rear SU and free front positions have a higher probability of being selected that are closer to the cell’s migration axis (large <i>P</i>). Additionally, those SU are more likely to be removed that are further away from the cells’ center of mass (large <i>D</i>) and have fewer neighbors (small <i>N</i>), whereas the free positions at the front are preferred when they have more neighbors (large <i>N</i>).<p /> <p id="Par11">In each iteration of the model, only one SU moves from the cell’s rear to the front. Thus, the cell shape changes only slightly even after 1000 iterations, and the cell moves a fraction of its diameter (Fig. <a href="#Fig2">2b</a>, Supplement Video <a href="#MOESM3">S1</a>). This means that our cell migration simulator allows us to study the changes in cell shape with high spatio-temporal resolution. Therefore, we focused on resolving the changes in cell shape rather than on following the cells for extended time intervals. We simulated each cell for 100 000 iterations and saved the coordinates of all SU every 500 iterations. We analyzed the cells starting at iteration 60 000 when the cell shape appeared sufficiently different from its initial spherical shape; this resulted in 80 time points per cell. To investigate diverse patterns of cell migration, we generated four sets of cells – one for each parameter of the model. Each set consisted of three classes corresponding to three different values of the examined parameter, and each class contained 70 cells (see section 3.1).</p> </div> <div class="experimentaldata" title="sec"> <div class="title" tagx="title" title="title">
Experimental data</div> <p id="Par12">Besides synthetic cells, we applied our analysis to intravital two-photon microscopy images of T cells migrating in the popliteal lymph node (LN), submandibular salivary gland (SMG), and skin (Fig. <a href="#Fig2">2c</a>, Supplementary Videos <a href="#MOESM3">S2</a>–<a href="#MOESM3">S4</a>). We isolated T cells from the spleen and the peripheral lymph nodes of GFP<sup>+</sup> or dsRed<sup>+</sup> donor mice and transferred them to recipient mice, which were subsequently infected with a virus to generate memory T cells in various organs (see Supplementary Methods for details). We visualized the T cells migrating in the LN, SMG, and skin with two-photon microscopy, and acquired six time series for each condition. All experiments were approved by the Cantonal Committee for Animal Experimentation (License 2018_22_FR) and performed in accordance with federal guidelines.</p> </div> <div class="surfaceextraction" title="sec"> <div class="title" tagx="title" title="title">
Surface extraction</div> <p id="Par13">To obtain cell surfaces for SPHARM analysis, we reconstructed the surface of each cell as a triangular mesh and extracted the x, y, z coordinates of the mesh vertices. For synthetic cells, we converted the SU coordinates of each cell into a binary mask and reconstructed the cell surface using the marching cube algorithm implemented in the scikit-image library of Python<sup><a href="#CR30">30</a></sup>. To reconstruct the surfaces of T cells, we first deconvolved the microscopy images using HuygensPro 19.04 (SVI, Hilversum, Netherlands) and then segmented and tracked them using Imaris 9.3.1 (Bitplane, Zürich, Switzerland; see Supplementary Methods for details). The resulting cell shapes were saved as VRML files for further analysis. To unify the different time intervals between frames (20 seconds for LN and SMG vs. 60 seconds for the skin), we analyzed only every third time point in the tracks from LN and SMG.</p> </div> <div class="featureextractionwithsphericalharmonics" title="sec"> <div class="title" tagx="title" title="title">
Feature extraction with spherical harmonics</div> <p id="Par14">To analyze the cell shape and its dynamics, we decomposed the cell surface at each time point into rotation-invariant spherical harmonics. The SPHARM transform was calculated using the Python library SHTools<sup><a href="#CR31">31</a></sup>, which requires a regular N x N spherical grid as input. Such grid was obtained by converting the surface coordinates from the Cartesian to the spherical coordinate system and then interpolating them onto a 120 ×120 grid of the polar angle <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M27">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\theta $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div></div><span class="inline-graphic" title="inline-graphic" /></div> and azimuthal angle <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M29">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\varphi $$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
φ</div></div><span class="inline-graphic" title="inline-graphic" /></div></span> (Fig. <a href="#Fig3">3</a>, left). The spherical grid was expanded into complex spherical harmonics using Driscoll and Healy’s sampling theorem<sup><a href="#CR32">32</a></sup> implemented within SHTools. In order to obtain a rotation-invariant shape feature <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M31">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$F(l)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
)</div></div><span class="inline-graphic" title="inline-graphic" /></div></span>, the power <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M33">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$A{(l,m)}^{2}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mn_UNKNOWN">
2</div> </div></div><span class="inline-graphic" title="inline-graphic" /></div></span> of each complex harmonic was computed and summed up for all orders <i>m</i> of each degree <i>l</i> (Fig. <a href="#Fig3">3</a>, middle)<sup><a href="#CR33">33</a></sup>. The first <i>l</i><sub><i>max</i></sub> degrees of each rotation-invariant spectrum were used to represent the shape of an individual cell at a single time point. <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Schematic overview of the surface analysis and classification. We convert surface coordinates to a regular spherical grid, transform them with SPHARM, and compute a rotation-invariant spectrum. We use the spectrum of a single time point either as a feature vector by its own (static features) or combine it with other time points into a time map (dynamic time features). We can further convert the time map into a frequency map by Fourier transform of each degree <i>l</i> (dynamic frequency features). We then use one of the three feature vectors to classify cells according to their shape.</p>   </div> <p /> <p id="Par15">The rotation-invariant spectra <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M35">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$F(l)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
)</div></div><span class="inline-graphic" title="inline-graphic" /></div> were either used to characterize the static cell shape or combined with spectra from other time points to characterize the shape dynamics (Fig. <a href="#Fig3">3</a>, middle). For both static and dynamic analysis, individual migrating cells – rather than individual time points of the same cell – served as independent observations. Thus, to obtain static shape features, we used <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M37">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$F(l)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
)</div></div><span class="inline-graphic" title="inline-graphic" /></div></span> from the first time point of each cell and discarded the remaining time points. Information from different time points was combined by computing two types of dynamic features in the time and the frequency domain. To compute the features in the time domain (dynamic time features), we combined <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M39">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$F(l)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
)</div></div><span class="inline-graphic" title="inline-graphic" /></div></span> from the first <i>T</i> time points of each cell into a time map <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M41">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\,F(l,t)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div></div><span class="inline-graphic" title="inline-graphic" /></div></span>. Being concatenated one after another, the time-ordered spectra served as an extended feature vector of size <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M43">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\,{l}_{max}\cdot T$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
x</div> </div> </div> <div class="mo_UNKNOWN">
⋅</div> <div class="mi_UNKNOWN">
T</div></div><span class="inline-graphic" title="inline-graphic" /></div></span>. The features in the frequency domain (dynamic frequency features) were revealed by a Fourier transform of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M45">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$F(l,t)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div></div><span class="inline-graphic" title="inline-graphic" /></div></span> calculated for each <i>l</i>. This transform resulted in a frequency map <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M47">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$F(l,f)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
)</div></div><span class="inline-graphic" title="inline-graphic" /></div></span> of the same size, whose values constituted another dynamic feature vector. The sizes of the three feature vectors were defined by the parameters <i>l</i><sub><i>max</i></sub> and <i>T</i>, which we adjusted to maximize the classification accuracy (Supplementary Methods, Fig. <a href="#MOESM1">S3</a>, <a href="#MOESM1">S4</a>).<p /> </div> <div class="shapeclassification" title="sec"> <div class="title" tagx="title" title="title">
Shape classification</div> <p id="Par16">After extracting static and dynamic shape features, we classified the cells using the linear-kernel support vector machine (SVM) classifier implemented in the scikit-learn library of Python<sup><a href="#CR34">34</a></sup> (Fig. <a href="#Fig3">3</a>, right). In order to compare the shape between different classes of cells and to identify significant shape differences, we computed the classifier accuracy between each pair of classes and compared it to the accuracy of a control classifier.</p> <p id="Par17">The classifier accuracy was evaluated for each pair of classes by performing 150 rounds of cross-validation with stratified shuffle split for synthetic cells and stratified group shuffle split for T cells. For synthetic data, 50 cells from each class were randomly chosen in each cross-validation round to serve as the training dataset, while the remaining cells (20 cells per class) were used to test the classifier performance (Fig. <a href="#Fig4">4</a>). For experimental data, we had to combine the cells into groups according to time series (six groups per class). In each cross-validation round, one time series from each class was left out for testing, while the cells from the remaining time series served to train the classifier (Fig. <a href="#Fig5">5</a>). In this way, T cells from the same time series were either in the training or testing dataset, but not in both. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Schematic of the cross-validation scheme for synthetic data. We compute the classifier accuracy for each pair of classes using two-class classification with stratified shuffle split cross-validation. For the control group, we apply stratified shuffle split cross-validation after randomly shuffling class labels.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Schematic of the cross-validation scheme for experimental data. We compute the classifier accuracy for each pair of classes using two-class classification with stratified group shuffle split cross-validation. If the data is balanced (none of the classes contains more than 60% of cells), the control accuracy is computed by randomly shuffling the class labels and performing cross-validation with stratified group shuffle split. For unbalanced data, the majority classifier serves as a control.</p>   </div> <p /> <p id="Par18">After evaluating the classifier accuracy, we compared it to a naive control classifier, which was chosen depending on whether the classified dataset was balanced or unbalanced. We considered a dataset to be balanced if none of the classes contained more than 60% of all cells. In this case, we used a random classifier as the best naive classifier. As a control for unbalanced data – where more than 60% of cells belonged to one class – we used the majority classifier. This classifier assigns all cells to the class with the most observation and thus performs better than a random classifier.</p> <p id="Par19">Since the dataset of synthetic cells was balanced (70 cells per class), we used a random classifier as control (Fig. <a href="#Fig4">4</a>). To compute the accuracy of the random classifier, we randomly shuffled class labels and carried out ten rounds of cross-validation with stratified-shuffle-split with 20 cells from each class serving as test data. We repeated the shuffling of class labels five times for each pair of classes, which produced 50 control accuracy values for each pair. We combined the accuracy values from all three pairs into a single control set with 150 values.</p> <p id="Par20">In contrast to synthetic data, each class of T cells contained different cell numbers (though the number of time series was the same). Hence, for each pair of classes, we determined whether the data was balanced or unbalanced and computed the control accuracy with either a random or a majority classifier (Fig. <a href="#Fig5">5</a>). For balanced data, we randomly shuffled class labels (repeated 15 times) and carried out cross-validation with stratified group shuffle split (10 rounds). For unbalanced data, we randomly selected one time series from each class (repeated 150 times), assigned all selected cells to the majority class, and computed the resulting accuracy. Thus, for each pair of classes, we used an individual control group with 150 accuracy values.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par21">We used the described cross-validation scheme to classify migration patterns in both synthetic cells and T cells. We examined how the applied feature vector (static, dynamic time-domain, or dynamic frequency-domain features) affected the accuracy of the classifier. The classifier accuracy was further compared to that of a control classifier (random or majority) to identify experimental conditions and model parameters that significantly change the cell migration patterns.</p> <div class="dynamicspharm-basedclassificationofsyntheticcells" title="sec"> <div class="title" tagx="title" title="title">
Dynamic SPHARM-based classification of synthetic cells</div> <p id="Par22">By detecting significant differences in migration patterns, our approach can be used to study how various parameters of the cell migration simulator (CMS) affect the shape of generated cells. To illustrate this use, we applied static and dynamic SPHARM-based classification to examine the behavior of the four parameters of the CMS. For each parameter, we examined three different values, while setting the other parameters to their default values. Thus, we tested the neighbor-weight parameter NW, which determines the cell surface roughness (default value NW = 4), the parameter for position weight PW, which affects the cell elongation (default value PW = 4), the distance weight parameter DW, which governs the size of cell protrusions (default value DW = 6), and the front-rear threshold FR, which is associated with the cell volume fraction considered as the front (default value FR = 0). We then quantified the classification accuracy between each pair of classes and compared it to a random control classifier to identify significant differences (Fig. <a href="#Fig4">4</a>).</p> <p id="Par23">Interestingly, different parameters affected cell shape to a different extent (Fig. <a href="#Fig6">6</a>). Thus, changing the value of the PW parameter from 1 to 2 to 4 resulted in visually distinct cells (Figs. <a href="#Fig6">6a</a>, <a href="#MOESM1">S5a</a>) and large quantitative differences between the cell shapes (Fig. <a href="#Fig6">6b</a>). Changing the value of NW from 1 to 2 to 4 resulted in a similarly large quantitative difference (Fig. <a href="#Fig6">6d</a>), but the cells’ 3D surface reconstructions were hard to distinguish visually (Fig. <a href="#Fig6">6c</a>). Only looking at the cells’ maximum projections revealed that – indeed – lower NW values resulted in a more structured cell surface (Fig. <a href="#MOESM1">S5b</a>). For the DW parameter, both 3D surfaces and maximum projections were similar (Figs. <a href="#Fig6">6e</a>, <a href="#MOESM1">S5c</a>) for different parameter values (2, 4, and 6), and no significant differences in cell shape could be identified by the static classifier (Fig. <a href="#Fig6">6f</a>). In contrast, when shape dynamics was included, the classifier accuracy was significantly higher than random, indicating that the visually similar cells might evolve their shape in different ways. Interestingly, different value ranges of the <i>FR</i> parameter had different effects on the cell shape, even though the parameter values were equally spaced. Thus, cell shape did not change when the value of <i>FR</i> increased from 0 to 0.3, which was both observed visually (Figs. <a href="#Fig6">6g</a>, <a href="#MOESM1">S5d</a>) and reflected by the close-to-random classifier accuracy (Fig. <a href="#Fig6">6h</a>). In contrast, cell shape for <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M49">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$FR=0.6$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.6</div></div><span class="inline-graphic" title="inline-graphic" /></div> was significantly different from both other classes, which was confirmed by visual inspection and by the classifier accuracy for all three feature vectors (Fig. <a href="#Fig6">6g,h</a>). <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Classification of synthetic cells. Surfaces of representative cells generated with three different values of the position weight (PW) (<b>a</b>), neighbor weight (NW) (<b>c</b>), distance weight (DW) (<b>e</b>), and front-rear (FR) (<b>g</b>) parameters. Classifier accuracy for different pairs of classes relative to control for different values of the PW (<b>b</b>), NW (<b>d</b>), DW (<b>f</b>), and FR (<b>h</b>) parameters. Unless indicated, the default parameter values were used: FR = 0, NW = 4, PW = 4, DW = 6. Significantly higher than control: *p &amp;lt; 0.05, **p &amp;lt; 0.01, ***p &amp;lt; 0.001; significantly higher than static features: <sup>$$$</sup>p &amp;lt; 0.001; one-sided Mann-Whitney test, n=150 per group.</p>   </div> <p /> <p id="Par24">Although we examined only three values for each parameter, our approach should bring the most benefit when many parameter values and their combinations have to be scanned and it is hard to visually control the outcome of every combinations. Adjusting parameter values of a model can be a daunting task, and our approach can facilitate this endeavor by quantifying the shape differences that result from different parameter values and thus support the identification of meaningful parameter ranges. For instance, for our CMS, it would probably make little sense to further examine <i>FR</i> values between 0 and 0.3, since these values result in very similar cell shape, and one should rather focus on the <i>FR</i> values between 0.3 and 0.6.</p> <p id="Par25">When comparing the accuracy of the static and dynamic classifiers, dynamic features outperformed static features in nearly all comparisons (Fig. <a href="#Fig6">6</a>). No improvement was achieved only in those cases, where the static classifier accuracy was close to 100% (Fig. <a href="#Fig6">6b,d</a>), or the dynamic classifier detected no significant shape differences (Fig. <a href="#Fig6">6h</a>, <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M51">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$FR=0$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div></div><span class="inline-graphic" title="inline-graphic" /></div>.0 vs. <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M53">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$FR=0.3$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.3</div></div><span class="inline-graphic" title="inline-graphic" /></div></span>). The use of dynamic frequency domain did not further improve classifier performance and resulted in similar accuracy values as for dynamic time domain (Fig. <a href="#Fig6">6</a>).<p /> <p id="Par26">Importantly, the higher accuracy of the dynamic classifiers was not due to the larger dataset used to train them. As described in Materials and Methods, we used only the first time point for extracting static SPHARM features in order to keep the number of independent observations equal between the static and dynamic classifiers. Effectively, this resulted in a much larger feature vector for the dynamic classifiers while most of the information for the static classifier was discarded. Using all time points as independent observations, however, does not improve the performance of the static classifier because these data points are highly correlated. In fact, a static classifier based on all time points was inferior to the one based on just the first time point (Fig. <a href="#MOESM1">S6</a>). It also detected significant differences in cell shape where the actual classifier accuracy was only marginally higher than random (Fig. <a href="#MOESM1">S6</a>, <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M55">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$FR=0$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div></div><span class="inline-graphic" title="inline-graphic" /></div> vs. <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M57">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$FR=0.3$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.3</div></div><span class="inline-graphic" title="inline-graphic" /></div></span>). Thus, simply gathering cell shapes from all time points into a static classifier not only doesn’t help to improve the classifier accuracy, but also can result in overfitting and misleading conclusions. Only a truly dynamic way to handle time information can boost the classifier performance by including the shapes at different time points into feature vectors of individual cells.<p /> </div> <div class="dynamicspharm-basedclassificationoftcells" title="sec"> <div class="title" tagx="title" title="title">
Dynamic SPHARM-based classification of T cells</div> <p id="Par27">We further examined whether our approach can also be used to classify migration patterns in experimental data. We analyzed the three groups of T cells migrating in various tissues (LN, skin, and SMG). As in synthetic data, we quantified the classification accuracy between each pair of tissues and compared it to a control classifier (Fig. <a href="#Fig5">5</a>).</p> <p id="Par28">The classifier accuracy significantly differed from control for each comparison and each feature vector (Fig. <a href="#Fig7">7a–c</a>). This difference was most apparent for the comparison of environments LN versus skin (Fig. <a href="#Fig7">7b</a>) because these tissues induce two extremes in the shape of T cells (Figs. <a href="#Fig2">2c</a>, <a href="#Fig7">7d</a>). The T cells migrating in LN had a more regular close-to-spherical shape, whereas the T cells from skin were irregular and made many protrusions. T cells migrating in SMG had an intermediate shape, which also strongly varied from time series to time series. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Classification of T cells for various pairs of classes relative to control. (<b>a</b>) LN vs SMG. (<b>b</b>) LN vs Skin. (<b>c</b>) SMG vs Skin. (<b>d</b>) Surfaces and SPHARM reconstructions of representative cells from the three analyzed classes. Significantly higher than control: **p &amp;lt; 0.01, ***p &amp;lt; 0.001; significantly higher than static features: <sup>$$$</sup>p &amp;lt; 0.001; significantly higher than dynamic time features: <sup>###</sup>p &amp;lt; 0.001; one-sided Mann-Whitney test, n = 150 per group. Numbers below the box plots indicate the number (ratio) of cells in each class for the corresponding class pair. For ratios between 0.4 and 0.6, the random classifier was used as control. In all other cases, the majority classifier was used, whose accuracy is defined by the ratio of cells. In the dynamic classifiers, fewer cells were analyzed because not all cell tracks were sufficiently long (20 time points) to be included. This resulted in different ratios between cell numbers and hence different control accuracies for the static and dynamic classifiers.</p>   </div> <p /> <p id="Par29">The relative accuracy of the static and dynamic classifiers differed for each comparison, but the dynamic classifiers generally performed better (Fig. <a href="#Fig7">7a–c</a>). Thus, the dynamic classifiers both in the time- and frequency-domain were superior when comparing LN to the other two environments, whereby in the LN vs. skin comparison, the accuracy further increased with the use of dynamic frequency features. When comparing SMG and skin, the use of dynamic time features did not bring any benefit, whereas the dynamic frequency features outperformed both dynamic time and static features. As a common trend for all comparisons, the classifier accuracy with dynamic frequency features was always significantly higher than the accuracy of the static classifier. This trend confirms our hypothesis that not only static shape features but also the dynamic shape patterns play an essential role in defining unique modes of cell migration in different environments.</p> </div> <div class="comparisonofspharm-basedclassificationto2dshapedescriptors" title="sec"> <div class="title" tagx="title" title="title">
Comparison of SPHARM-based classification to 2D shape descriptors</div> <p id="Par30">We evaluated how SPHARM-based descriptors perform in comparison to 2D shape descriptors. As SPHARM is a 3D extension of a Fourier analysis, we applied a method that is based on the classification of a set of Fourier components produced by Discrete Fourier Transform (DFT) after carrying out a 3D-to-2D projection of surface-rendered cells<sup><a href="#CR35">35</a></sup>. The advantage of the latter approach is two-fold: (i) the more complex 3D shape characterization by SPHARM is reduced to 2D DFT and (ii) the combination of various random 3D-to-2D projections allows a seamless transition from a pure 2D shape descriptor to a quasi-3D description of cell shapes. We combined these 2D descriptors obtained from various numbers of projections with the same classification scheme that we used for SPHARM and compared the classification accuracy to a static SPHARM classifier.</p> <p id="Par31">The results of this comparison for synthetic cells are summarized in Figs. <a href="#MOESM1">S7</a>–<a href="#MOESM1">S10</a> and reveal that the SPHARM-based approach always outcompetes the 2D shape descriptors, except for variations in the DW parameter (see Fig. <a href="#MOESM1">S9</a>) and for a specific comparison of FR parameters (see Fig. <a href="#MOESM1">S10a</a>). This does not come as a surprise, because we noted before that for these parameters the SPHARM-based classifier does not perform better than the random classifier in the static case (see Fig. <a href="#Fig6">6f,h</a>). These observations suggest that the DFT-based approach is more sensitive to details in the outline of the shape projections than the SPHARM-based approach. On the other hand, increasing the number of projections gives rise to a quasi-3D description of cell shapes that is approaching but not superseding the accuracy of the SPHARM-based classification for variations in the model parameters PW, NW and FR. For T cells, the performance of the static SPHARM-based classification compared with the DFT-based revealed that the accuracy of the former is always significantly higher, even if the latter approach includes numerous projections for a quasi-3D representation of the cell shapes. In fact, as can be seen in Fig. <a href="#Fig8">8</a>, the classification accuracy of the DFT-based approach is comparable to that of a random classifier. We conclude that the complexity of the T cell shape determines the minimum number of projections for which the DFT-based approach of T cell representation performs as well as the SPHARM-based approach does by 3D reconstruction. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>Static SPHARM-based classifier accuracy for T cells of different pairs of environments in comparison to the DFT-based with projection numbers between 1 and 12. The environments. (<b>a</b>) SMG versus Skin, (<b>b</b>) LN versus Skin, and (<b>c</b>) LN versus SMG are compared.</p>   </div> <p /> <p id="Par32">We conclude from this comparison that the SPHARM-based approach is generally superior to using 2D shape descriptors.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par33">In our proof-of-principle study, we confirmed that adding dynamics to shape analysis can improve the classification of migration patterns of cells. We illustrated this improvement by combining the dynamic SPHARM approach with an SVM classifier to identify conditions with significantly different cell shapes. This approach, however, is not restricted to SPHARM and SVM. We believe that adding dynamics to other shape descriptors, such as Fourier-based descriptors<sup><a href="#CR35">35</a></sup> or wavelets<sup><a href="#CR36">36</a>,<a href="#CR37">37</a></sup>, could also improve their classification accuracy. The performance of other classifiers – such as random forest or neural networks – should also be studied in the future.</p> <p id="Par34">Other implementations of the SPHARM analysis should also be tested, e.g. the SPHARM-MAT toolbox available in MATLAB. Here we focused on an open-source Python alternative SHTools<sup><a href="#CR31">31</a></sup> making this analysis freely available to the community. The problem of the SHTools implementation, however, is that it is based on converting the cell shape to spherical coordinates and is therefore accurate only for cells with the center of mass inside the cell body. While this requirement was always met for synthetic cells, some T cells – especially those migrating in the skin – had a complex shape with the center of mass temporarily appearing outside the cell body. Such shape could not be accurately represented in spherical coordinates, which resulted in erroneous shape reconstructions after SPHARM transform (Fig. <a href="#MOESM1">S11</a>, bottom row). Since such complex-shaped cells constituted only a fraction of the total T cell number (9% LN, 7% in SMG, and 28% in the skin), we neglected this inaccuracy in the current study, but this problem will have to be addressed in the future. One solution to this could be to create open-source implementations of other SPHARM variants (like SPHARM-MAT) that calculate the SPHARM transform directly from the Cartesian coordinates of the surface and thus do not suffer from the center-of-mass problem.</p> <p id="Par35">Despite the limitations of our approach for cells with highly irregular shape, most of the cells analyzed in our study (all synthetic cells, as well as&amp;gt; 90% of LN and SMG T cells) had their center of mass inside the cell and thus could be accurately analyzed by our method. In all comparisons that involved these accurately represented cells (Figs. <a href="#Fig6">6</a>, <a href="#Fig7">7a</a>), dynamic SPHARM outperformed its static counterpart and – in one case – could even reveal shape differences not detected by the static SPHARM analysis (Fig. <a href="#Fig6">6f</a>). This suggests that including dynamic shape features is especially relevant when shape differences between the analyzed groups are small, whereas for visually distinct shapes, static analysis – or even simpler shape descriptors – may be sufficient.</p> <p id="Par36">When looking at dynamic SPHARM in the frequency domain, in most cases it did not bring any improvement relative to the time domain. Interestingly, the only comparisons where the dynamic frequency features were superior involved the T cells migrating in skin (Fig. <a href="#Fig7">7b,c</a>), which included a relatively high fraction (28%) of cells with center of mass transiently appearing outside the cell body. This event resulted in a dramatic change in shape representation for some of the time points (Fig. <a href="#MOESM1">S11</a>, bottom row) and apparently created a unique frequency pattern that was picked up by the dynamic frequency classifier and used as a feature to distinguish skin T cells from other T cells. This observation confirms that the use of frequency domain can indeed help to detect interesting frequency patterns<sup><a href="#CR8">8</a></sup>, either relevant for cell migration, or irrelevant artifacts as in our case.</p> <p id="Par37">By recognizing differences in dynamic cell shape, dynamic shape analysis methods like ours can be used to study how cell migration patterns change under various experimental conditions. Here we showed that the extracellular environment affects the dynamic shape of migrating T cells: even though we looked at the same type of cells, their migratory behavior dramatically changed depending on the surrounding tissue. In the future, a similar approach can be used to study other factors that affect cell migration<sup><a href="#CR1">1</a></sup> (Fig. <a href="#Fig1">1</a>), which could involve various cell types migrating in different intact or altered extracellular environments both <i>in vivo</i> and in 3D artificial matrices<sup><a href="#CR38">38</a></sup>.</p> <p id="Par38">Dynamic SPHARM analysis can also be applied to study cell migration <i>in silico</i>. Classifying between cells generated with different parameters of migration models will help to better understand the meaning of these parameters both in our CMS and other models of cell migration. Furthermore, comparing the shape of synthetic migrating cells to that of real cells will enable the use of the image-based systems biology approach<sup><a href="#CR39">39</a>,<a href="#CR40">40</a></sup> to create cell migration models that are realistic and data-driven. With these models, we can identify and interpret the meaning of the model parameters that induce cell shapes observed in a particular experiment<sup><a href="#CR41">41</a></sup>.</p> <p id="Par39">Taken together, dynamic SPHARM-based classification can contribute to our understanding of cell migration in two ways. On the one hand, it can help to investigate various experimental settings and their effect on cell migration patterns. On the other hand, it should enable identifying parameters of realistic migration models that produce specific experimentally observed cell shapes. Both of these applications can help to pinpoint the factors that affect the shape of migrating cells and thus bring us closer to understanding the mechanisms underlying cell migration.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62997_MOESM1_ESM.pdf">LINK</a> <p>supplementary information.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62997_MOESM2_ESM.pdf">LINK</a> <p>supplementary information 2.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62997_MOESM3_ESM.pdf">LINK</a> <p>supplementary information 3.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62997_MOESM4_ESM.mpg">LINK</a> <p>supplementary information 4.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62997_MOESM5_ESM.mp4">LINK</a> <p>supplementary information 5.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62997_MOESM6_ESM.mp4">LINK</a> <p>supplementary information 6.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_62997_MOESM7_ESM.mp4">LINK</a> <p>supplementary information 7.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>These authors contributed equally: Anna Medyukhina and Marco Blickensdorf.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p>is available for this paper at 10.1038/s41598-020-62997-7.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We want to thank Carl-Magnus Svensson for valuable hints on the choice of control classifiers. This work was financially supported by the International Leibniz Research School (ILRS) and by the Deutsche Forschungsgemeinschaft (DFG) through the Excellence Graduate School Jena School for Microbial Communication (JSMC) and CRC 1278 PolyTarget under DFG project number 316213987 (project Z01 to M.T.F.).</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>A.M. and M.T.F. conceived and designed the study. M.B. developed the migration model and generated synthetic cells. N.R. and J.S. performed microscopy experiments. Z.C. performed deconvolution and image analysis of the microscopy data, and carried out the 3D-to-2D projection-based DFT analysis. A.M. developed and performed the SPHARM-based shape analysis and classification. A.M., M.B., Z.C. and M.T.F. interpreted the analysis results. A.M., M.B. and Z.C. prepared the figures. All authors were involved in writing, critically revising, and approving the manuscript.</p> <div class="title" tagx="title" title="title">
Data and Code availability</div> <p>The authors declare that all relevant data supporting the findings of this study are available within the paper (and its Supplementary Information file). Any raw data can be downloaded from <a href="https://asbdata.hki-jena.de/publidata/MedyukhinaEtAL_SPHARM/">https://asbdata.hki-jena.de/publidata/MedyukhinaEtAL_SPHARM/</a>. The source code for the dynamic SPHARM analysis is available at <a href="https://github.com/applied-systems-biology/Dynamic_SPHARM">https://github.com/applied-systems-biology/Dynamic_SPHARM</a>. The source code for the cell migration simulator is available at <a href="https://github.com/applied-systems-biology/cell-migration-simulator">https://github.com/applied-systems-biology/cell-migration-simulator</a>.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par40">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ridley</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cell Migration: Integrating Signals from Front to Back</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">302</span><span class="fpage" tagx="fpage" title="fpage">1704</span><span class="lpage" tagx="lpage" title="lpage">1709</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1092053">10.1126/science.1092053</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14657486">14657486</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masuzzo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Troys</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ampe</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martens</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Taking Aim at Moving Targets in Computational Cell Migration</span><span class="source" tagx="source" title="source">Trends Cell Biol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">88</span><span class="lpage" tagx="lpage" title="lpage">110</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tcb.2015.09.003">10.1016/j.tcb.2015.09.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26481052">26481052</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mokhtari</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Automated Characterization and Parameter-Free Classification of Cell Tracks Based on Local Migration Behavior</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e80808</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0080808">10.1371/journal.pone.0080808</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24324630">24324630</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svensson</span><span class="given-names" tagx="given-names" title="given-names">C-M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medyukhina</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belyaev</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al‐Zaben</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figge</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Untangling cell tracks: Quantifying cell migration by time lapse image data analysis</span><span class="source" tagx="source" title="source">Cytometry A</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">370</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cyto.a.23249">10.1002/cyto.a.23249</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28976646">28976646</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pincus</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theriot</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of quantitative methods for cell-shape analysis</span><span class="source" tagx="source" title="source">J. Microsc.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">227</span><span class="fpage" tagx="fpage" title="fpage">140</span><span class="lpage" tagx="lpage" title="lpage">156</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2818.2007.01799.x">10.1111/j.1365-2818.2007.01799.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17845709">17845709</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tangelder</span><span class="given-names" tagx="given-names" title="given-names">JWH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veltkamp</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span></span><span class="mixed-article-title" title="mixed-article-title">A survey of content based 3D shape retrieval methods</span><span class="source" tagx="source" title="source">Multimed. Tools Appl.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">441</span><span class="lpage" tagx="lpage" title="lpage">471</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11042-007-0181-0">10.1007/s11042-007-0181-0</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dufour</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Signal Processing Challenges in Quantitative 3-D Cell Morphology: More than meets the eye</span><span class="source" tagx="source" title="source">IEEE Signal Process. Mag.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">30</span><span class="lpage" tagx="lpage" title="lpage">40</span><span class="pub-id"><a href="https://dx.doi.org/10.1109/MSP.2014.2359131">10.1109/MSP.2014.2359131</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maeda</span><span class="given-names" tagx="given-names" title="given-names">YT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inose</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuo</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwaya</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sano</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ordered Patterns of Cell Shape and Orientational Correlation during Spontaneous Cell Migration</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e3734</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0003734">10.1371/journal.pone.0003734</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19011688">19011688</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Driscoll</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fourkas</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Losert</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Local and global measures of shape dynamics</span><span class="source" tagx="source" title="source">Phys. Biol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">055001</span><span class="pub-id"><a href="https://dx.doi.org/10.1088/1478-3975/8/5/055001">10.1088/1478-3975/8/5/055001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21832799">21832799</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell dynamic morphology classification using deep convolutional neural networks</span><span class="source" tagx="source" title="source">Cytometry A</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">628</span><span class="lpage" tagx="lpage" title="lpage">638</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cyto.a.23490">10.1002/cyto.a.23490</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29762901">29762901</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Li, H., Liu, Z., Pang, F. &amp;amp; Shi, Y. Characterization of single cell dynamic morphology by local deformation pattern modeling. In <i>2017 39th Annual International Conference of the IEEE Engineering in Medicine and Biology Society (EMBC)</i> 329–332, 10.1109/EMBC.2017.8036829 (2017).</span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mogilner</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keren</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Shape of Motile Cells</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">R762</span><span class="lpage" tagx="lpage" title="lpage">R771</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2009.06.053">10.1016/j.cub.2009.06.053</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19906578">19906578</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coombes</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robey</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dynamic imaging of host–pathogen interactions <i>in vivo</i></span><span class="source" tagx="source" title="source">Nat. Rev. Immunol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">353</span><span class="lpage" tagx="lpage" title="lpage">364</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nri2746">10.1038/nri2746</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20395980">20395980</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Konjufca</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Two-photon microscopy of host–pathogen interactions: acquiring a dynamic picture of infection <i>in vivo</i></span><span class="source" tagx="source" title="source">Cell. Microbiol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">551</span><span class="lpage" tagx="lpage" title="lpage">559</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-5822.2009.01289.x">10.1111/j.1462-5822.2009.01289.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19170686">19170686</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Secklehner</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Celso</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlin</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Intravital microscopy in historic and contemporary immunology</span><span class="source" tagx="source" title="source">Immunol. Cell Biol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">506</span><span class="lpage" tagx="lpage" title="lpage">513</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/icb.2017.25">10.1038/icb.2017.25</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28366932">28366932</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Driscoll</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danuser</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantifying Modes of 3D Cell Migration</span><span class="source" tagx="source" title="source">Trends Cell Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">749</span><span class="lpage" tagx="lpage" title="lpage">759</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tcb.2015.09.010">10.1016/j.tcb.2015.09.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26603943">26603943</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brechbühler</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerig</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kübler</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="mixed-article-title" title="mixed-article-title">Parametrization of Closed Surfaces for 3-D Shape Description</span><span class="source" tagx="source" title="source">Comput. Vis. Image Underst.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">154</span><span class="lpage" tagx="lpage" title="lpage">170</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/cviu.1995.1013">10.1006/cviu.1995.1013</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saykin</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fourier method for large-scale surface modeling and registration</span><span class="source" tagx="source" title="source">Comput. Graph.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">299</span><span class="lpage" tagx="lpage" title="lpage">311</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cag.2009.03.002">10.1016/j.cag.2009.03.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20161536">20161536</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Styner, M. <i>et al</i>. Framework for the Statistical Shape Analysis of Brain Structures using SPHARM-PDM. <i>Insight J</i>. 242–250 (2006).</span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Najmanovich</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kahraman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thornton</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Real spherical harmonic expansion coefficients as 3D shape descriptors for protein binding pocket and ligand comparisons</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">2347</span><span class="lpage" tagx="lpage" title="lpage">2355</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bti337">10.1093/bioinformatics/bti337</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728116">15728116</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venkatraman</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sael</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kihara</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potential for Protein Surface Shape Analysis Using Spherical Harmonics and 3D Zernike Descriptors</span><span class="source" tagx="source" title="source">Cell Biochem. Biophys.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">23</span><span class="lpage" tagx="lpage" title="lpage">32</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12013-009-9051-x">10.1007/s12013-009-9051-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19521674">19521674</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khairy</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Shapes of Red Blood Cells: Comparison of 3D Confocal Images with the Bilayer-Couple Model</span><span class="source" tagx="source" title="source">Cell. Mol. Bioeng.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">173</span><span class="lpage" tagx="lpage" title="lpage">181</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12195-008-0019-5">10.1007/s12195-008-0019-5</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khairy</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Minimum-energy vesicle and cell shapes calculated using spherical harmonics parameterization</span><span class="source" tagx="source" title="source">Soft Matter</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">2138</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/c0sm01193b">10.1039/c0sm01193b</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gerig, G., Styner, M., Shenton, M. E. &amp;amp; Lieberman, J. A. Shape versus Size: Improved Understanding of the Morphology of Brain Structures. in <i>Medical Image Computing and Computer-Assisted Intervention – MICCAI 2001</i> (eds. Niessen, W. J. &amp;amp; Viergever, M. A.) 24–32 (Springer Berlin Heidelberg, 2001).</span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ford</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makedon</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saykin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">A surface-based approach for classification of 3D neuroanatomic structures</span><span class="source" tagx="source" title="source">Intell. Data Anal.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">519</span><span class="lpage" tagx="lpage" title="lpage">542</span><span class="pub-id"><a href="https://dx.doi.org/10.3233/IDA-2004-8602">10.3233/IDA-2004-8602</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">C-J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawkins</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bretschneider</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">3D time series analysis of cell shape using Laplacian approaches</span><span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">296</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-14-296">10.1186/1471-2105-14-296</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24090312">24090312</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ducroz, C., Olivo-Marin, J. &amp;amp; Dufour, A. Characterization of cell shape and deformation in 3D using Spherical Harmonics. In <i>2012 9th IEEE International Symposium on Biomedical Imaging (ISBI)</i> 848–851, 10.1109/ISBI.2012.6235681 (2012).</span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ducroz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olivo-Marin</span><span class="given-names" tagx="given-names" title="given-names">J-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dufour</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spherical Harmonics based extraction and annotation of cell shape in 3D time-lapse microscopy sequences</span><span class="source" tagx="source" title="source">Conf. Proc. Annu. Int. Conf. IEEE Eng. Med. Biol. Soc. IEEE Eng. Med. Biol. Soc. Annu. Conf.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">2011</span><span class="fpage" tagx="fpage" title="fpage">6619</span><span class="lpage" tagx="lpage" title="lpage">6622</span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Blickensdorf, M. Agent based modeling of cell migration and shape analysis. (Friedrich Schiller University Jena, 2015).</span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walt</span><span class="given-names" tagx="given-names" title="given-names">Svander</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">scikit-image: image processing in Python</span><span class="source" tagx="source" title="source">PeerJ</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e453</span><span class="pub-id"><a href="https://dx.doi.org/10.7717/peerj.453">10.7717/peerj.453</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25024921">25024921</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieczorek</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meschede</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">SHTools: Tools for Working with Spherical Harmonics</span><span class="source" tagx="source" title="source">Geochem. Geophys. Geosystems</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">2574</span><span class="lpage" tagx="lpage" title="lpage">2592</span><span class="pub-id"><a href="https://dx.doi.org/10.1029/2018GC007529">10.1029/2018GC007529</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Driscoll</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Healy</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Computing Fourier Transforms and Convolutions on the 2-Sphere</span><span class="source" tagx="source" title="source">Adv. Appl. Math.</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">202</span><span class="lpage" tagx="lpage" title="lpage">250</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/aama.1994.1008">10.1006/aama.1994.1008</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kazhdan, M., Funkhouser, T. &amp;amp; Rusinkiewicz, S. Rotation Invariant Spherical Harmonic Representation of 3D Shape Descriptors. In <i>Proceedings of the 2003 Eurographics/ACM SIGGRAPH Symposium on Geometry Processing</i> 156–164 (Eurographics Association, 2003).</span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedregosa</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Scikit-learn: Machine Learning in Python</span><span class="source" tagx="source" title="source">J. Mach. Learn. Res.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">2825</span><span class="lpage" tagx="lpage" title="lpage">2830</span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kriegel</span><span class="given-names" tagx="given-names" title="given-names">FL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cell shape characterization and classification with discrete Fourier transforms and self-organizing maps</span><span class="source" tagx="source" title="source">Cytometry A</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">323</span><span class="lpage" tagx="lpage" title="lpage">333</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cyto.a.23279">10.1002/cyto.a.23279</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29077263">29077263</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antoine</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandergheynst</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Wavelets on the 2-Sphere: A Group-Theoretical Approach</span><span class="source" tagx="source" title="source">Appl. Comput. Harmon. Anal.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">262</span><span class="lpage" tagx="lpage" title="lpage">291</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/acha.1999.0272">10.1006/acha.1999.0272</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Tournemenne, R., Ducroz, C., Olivo-Marin, J. &amp;amp; Dufour, A. 3D shape analysis using overcomplete spherical wavelets: Application to BLEB detection in cell biology. In <i>2014 IEEE 11th International Symposium on Biomedical Imaging (ISBI)</i> 365–368, 10.1109/ISBI.2014.6867884 (2014).</span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffith</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swartz</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Capturing complex 3D tissue physiology <i>in vitro</i></span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">211</span><span class="lpage" tagx="lpage" title="lpage">224</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm1858">10.1038/nrm1858</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16496023">16496023</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figge</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Image-based systems biology</span><span class="source" tagx="source" title="source">Cytometry A</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">459</span><span class="lpage" tagx="lpage" title="lpage">461</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cyto.a.22663">10.1002/cyto.a.22663</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26033857">26033857</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medyukhina</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timme</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mokhtari</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figge</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Image-based systems biology of infection</span><span class="source" tagx="source" title="source">Cytometry A</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">462</span><span class="lpage" tagx="lpage" title="lpage">470</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cyto.a.22638">10.1002/cyto.a.22638</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25641512">25641512</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figge</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantitative bioimage analysis of cell characteristics</span><span class="source" tagx="source" title="source">Cytometry A</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">278</span><span class="lpage" tagx="lpage" title="lpage">280</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cyto.a.23349">10.1002/cyto.a.23349</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29579368">29579368</a></span></span></li> </ul> </div> </div>  </body></html>