<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Dev Cell</span><span class="iso-abbrev" title="iso-abbrev">Dev. Cell</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Developmental Cell</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1534-5807</span><span class="issn-epub" tagx="issn" title="issn-epub">1878-1551</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cell Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6983945">6983945</a></span><span class="publisher-id" title="publisher-id">S1534-5807(19)31032-9</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.devcel.2019.12.011">10.1016/j.devcel.2019.12.011</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
F-Actin Interactome Reveals Vimentin as a Key Regulator of Actin Organization and Cell Mechanics in Mitosis</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serres</span><span class="given-names" tagx="given-names" title="given-names">Murielle P.</span></span><a href="#aff1">1</a><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samwer</span><span class="given-names" tagx="given-names" title="given-names">Matthias</span></span><a href="#aff3">3</a><a href="#fn2">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Truong Quang</span><span class="given-names" tagx="given-names" title="given-names">Binh An</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">Geneviève</span></span><a href="#aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perera</span><span class="given-names" tagx="given-names" title="given-names">Upamali</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Görlich</span><span class="given-names" tagx="given-names" title="given-names">Dirk</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charras</span><span class="given-names" tagx="given-names" title="given-names">Guillaume</span></span><a href="#aff5">5</a><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petronczki</span><span class="given-names" tagx="given-names" title="given-names">Mark</span></span><a href="#aff2">2</a><a href="#fn2">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">Philippe P.</span></span><span class="email" tagx="email" title="email">philippe.roux@umontreal.ca</span><a href="#aff4">4</a><a href="#aff7">7</a><a href="#cor1">∗</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">Ewa K.</span></span><span class="email" tagx="email" title="email">ekp25@cam.ac.uk</span><a href="#aff1">1</a><a href="#fn1">8</a><a href="#fn3">10</a><a href="#cor2">∗∗</a></span></div><span class="citation_author_institution" id="aff1">[1], </span><span class="citation_author_institution" id="aff2">[2], </span><span class="citation_author_institution" id="aff3">[3], </span><span class="citation_author_institution" id="aff4">[4], </span><span class="citation_author_institution" id="aff5">[5], </span><span class="citation_author_institution" id="aff6">[6], </span><span class="citation_author_institution" id="aff7">[7], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">∗</span>Corresponding author <span class="email" tagx="email" title="email">philippe.roux@umontreal.ca</span></div> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">∗∗</span>Corresponding author <span class="email" tagx="email" title="email">ekp25@cam.ac.uk</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">8</span><p id="ntpara0010a">Present address: Department of Physiology, Development, and Neuroscience, University of Cambridge, Downing Street, Cambridge CB2 3DY, UK</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">9</span><p id="ntpara0010">Present address: Boehringer Ingelheim, RCV GmbH &amp;amp;Co KG, Vienna, Austria</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">10</span><p id="ntpara0015">Lead Contact</p> </div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-1-1</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-1-1</span></span><span class="volume" tagx="volume" title="volume">52</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">210</span><span class="lpage" tagx="lpage" title="lpage">222.e7</span><span class="history" title="history"><span class="received" title="received">received: 2019-2-8</span><span class="rev-recd" title="rev-recd">rev-recd: 2019-9-23</span><span class="accepted" title="accepted">accepted: 2019-12-12</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Summary</div> <p>Most metazoan cells entering mitosis undergo characteristic rounding, which is important for accurate spindle positioning and chromosome separation. Rounding is driven by contractile tension generated by myosin motors in the sub-membranous actin cortex. Recent studies highlight that alongside myosin activity, cortical actin organization is a key regulator of cortex tension. Yet, how mitotic actin organization is controlled remains poorly understood. To address this, we characterized the F-actin interactome in spread interphase and round mitotic cells. Using super-resolution microscopy, we then screened for regulators of cortex architecture and identified the intermediate filament vimentin and the actin-vimentin linker plectin as unexpected candidates. We found that vimentin is recruited to the mitotic cortex in a plectin-dependent manner. We then showed that cortical vimentin controls actin network organization and mechanics in mitosis and is required for successful cell division in confinement. Together, our study highlights crucial interactions between cytoskeletal networks during cell division.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Highlights</div> <p> </p><ul> <li><span class="label" tagx="label" title="label">•</span><p id="p0010">Comparison of the F-actin interactome in spread interphase and round mitotic cells</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0015">Proteomics identifies vimentin and plectin as key regulators of the mitotic cortex</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0020">Vimentin intermediate filaments localize under the actin cortex in mitosis</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0025">Sub-cortical vimentin regulates actin cortex organization and mechanics in mitosis</p> </li> </ul> <p /> </div> <div class="abstract" title="abstract"> <p>Mitotic rounding, which is essential for accurate spindle positioning, is controlled by the sub-membranous cortical actin network. Using proteomics, Serres et al. identify the intermediate filament vimentin as a key regulator of the mitotic cortex. Vimentin forms a sub-cortical layer in mitosis and controls actin cortex organization and mechanics.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
actin organization</div> <div class="kwd" title="kwd">
cortex</div> <div class="kwd" title="kwd">
cell shape</div> <div class="kwd" title="kwd">
mitosis</div> <div class="kwd" title="kwd">
mitotic rounding</div> <div class="kwd" title="kwd">
vimentin</div> <div class="kwd" title="kwd">
proteomics</div> <div class="kwd" title="kwd">
plectin</div> <div class="kwd" title="kwd">
CDK1</div> <div class="kwd" title="kwd">
phosphorylation</div> </div> </div> <p id="misc0010">Published: January 9, 2020</p> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="p0030">A precise control of cell shape is central to a wealth of physiological processes, including tissue morphogenesis, cell migration, and cell division. One of the first events of cell division in most animal cells, both in culture and <i>in vivo</i>, is cellular rounding at mitosis entry. Mitotic rounding generates the intracellular space required for accurate spindle assembly and metaphase plate formation (reviewed in <a href="#bib31">Lancaster and Baum, 2014</a>). This is of particular importance for cells dividing in confined environments. In packed epithelia, mitotic rounding defects lead to abnormal spindle orientation and division asymmetries (<a href="#bib7">Chanet et al., 2017</a>, <a href="#bib36">Luxenburg et al., 2011</a>). In cells cultured in 3-dimensional (3D) confining devices, physically preventing rounding leads to spindle assembly defects and delayed mitotic progression (<a href="#bib32">Lancaster et al., 2013</a>). It is thus essential to understand how mitotic rounding is controlled.</p> <p id="p0035">Multiple studies converge on identifying the reorganization of cellular actin into a uniform, contractile actomyosin cortex at the cell surface as a key driver of mitotic rounding (reviewed in <a href="#bib6">Champion et al., 2017</a>, <a href="#bib14">Clark and Paluch, 2011</a>, <a href="#bib31">Lancaster and Baum, 2014</a>). Cortical contractility, which increases through mitosis and peaks in metaphase, creates tension at the cell surface and promotes rounding (<a href="#bib14">Clark and Paluch, 2011</a>). At an upstream level, rounding appears to be controlled by the mitotic master regulator CDK1 and its substrate the RhoA-GEF Ect2 (<a href="#bib21">Gavet and Pines, 2010</a>, <a href="#bib38">Maddox and Burridge, 2003</a>, <a href="#bib39">Matthews et al., 2012</a>). However, how exactly these pathways effect the changes in actin organization and cell surface mechanics driving cell shape change remains unclear.</p> <p id="p0040">In <i>Drosophila</i> cells, an increase in membrane-to-cortex attachment and cortex stiffness via the ezrin-radixin-moesin (ERM) family protein moesin is essential for rounding (<a href="#bib4">Carreno et al., 2008</a>, <a href="#bib30">Kunda et al., 2008</a>). However, in mammalian cells, although ezrin depletion slightly decreases mitotic tension (<a href="#bib55">Toyoda et al., 2017</a>), ERMs do not appear to be required for rounding (<a href="#bib37">Machicoane et al., 2014</a>). Instead, for many years, cortex tension in mammalian cells had been thought to be primarily controlled by the levels and activity of cortical myosin (<a href="#bib40">Mayer et al., 2010</a>, <a href="#bib46">Ramanathan et al., 2015</a>, <a href="#bib54">Tinevez et al., 2009</a>). However, recent studies, including a screen for regulators of cortex tension (<a href="#bib55">Toyoda et al., 2017</a>), have shown that proteins controlling actin filament length and actin cross-linkers affect cortical tension (<a href="#bib11">Chugh et al., 2017</a>, <a href="#bib16">Ding et al., 2017</a>, <a href="#bib35">Logue et al., 2015</a>, <a href="#bib55">Toyoda et al., 2017</a>). Taken together, it is increasingly clear that the organization of cortical actin is a key regulator of cortex tension (reviewed in <a href="#bib27">Koenderink and Paluch, 2018</a>). Yet, a systematic investigation of how cortex organization in mitosis is controlled has been missing. As a result, our understanding of the regulation of mitotic actin architecture remains fragmented and limited to the role of a handful of selected proteins.</p> <p id="p0045">Here, we took an unbiased approach to identify proteins controlling actin cortex organization in mitosis. First, we established a phalloidin affinity matrix and mass spectrometry-based method to determine the proteins that bind filamentous actin in interphase and mitosis. This identified F-actin binding proteins (F-ABPs) specifically enriched in metaphase, when cortical tension is highest. We then conducted a quantitative small interfering RNA (siRNA) screen to test the effect of candidate regulators on mitotic cortex architecture, using thickness, measured by super-resolution microscopy, as a readout. Our screen revealed that the intermediate filament protein vimentin is a key regulator of mitotic cortex thickness. We then found that in mitosis, vimentin relocalizes to the actin cortex in a plectin-dependent manner, while mitotic phosphorylation appears to control vimentin network organization. The sub-cortical vimentin layer appears to mechanically resist the contractility, and possibly the expansion toward the cytoplasm, of the actomyosin cortex. We then show that the sub-cortical vimentin network controls cortical tension and is required for effective rounding and successful division in 3D confinement. Taken together, our study shows that the organization and mechanics of the actin cortex cannot be studied in isolation but must be investigated in the context of the entire cell.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="f-actininteractomeininterphaseandmitosis" title="sec"> <div class="title" tagx="title" title="title">
F-Actin Interactome in Interphase and Mitosis</div> <p id="p0050">To identify ABPs potentially regulating mitotic cortex organization, we compared which proteins bind actin filaments in mitotic and interphase cells. We developed a pull-down assay to isolate F-ABPs from cellular extracts (<a href="#fig1">Figure 1</a>A) and compared HeLa cells synchronized in mitosis (arrested in metaphase), when cells are rounded with a uniform actin cortex, and in interphase (arrested in G2 phase), when cells adopt spread shapes with prominent lamellipodia and stress fibers (<a href="#fig1">Figures 1</a>A and 1B). We verified that the synchronized cells displayed the expected actin and microtubule distributions (<a href="#fig1">Figures 1</a>B and <a href="#mmc1">S1</a>A). We also validated synchronization efficiency by checking the levels of both cyclinB1, a subunit of the cyclin B-CDK1 complex expressed at high levels from G2 to anaphase onset, and phospho-Histone H3, a marker of mitosis (<a href="#mmc1">Figure S1</a>B). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Determining the Interphase and Mitotic F-Actin Interactomes</p> <p>(A) Schematic representation of the synchronization procedure and the experimental pipeline. Cells were first arrested in S phase by thymidine treatment. For synchronization in interphase, thymidine-pre-treated cells were then released and treated with the CDK1 inhibitor RO3306, yielding a population of cells in G2 phase. For synchronization in metaphase, thymidine-pre-treated cells were submitted to a short treatment with nocodazole, followed by a mitotic shake off and metaphase arrest with the proteasome inhibitor MG132. The drugs used for synchronization were washed out prior to extract preparation.</p> <p>(B) Representative images of HeLa cells after synchronization following the procedure described in (A). Cells were stained with DAPI (red) to visualize DNA and phalloidin (cyan) to visualize F-actin. Scale bars, 10 μm.</p> <p>(C) Immunoblot of actin, cyclin B, and phospho-Histone H3 (P-HH3) levels in the different synchronized cell extracts. S, supernatant; P, Pellet.</p> <p>(D) Colloidal Coomassie staining of an SDS-PAGE gel of synchronized cell extracts and of the control and phalloidin-bead elutions for mass spectrometry (corresponding to one of the mass spectrometry experiments, MS2). The x marks an unrelated lane. The asterisk marks the position of the actin band on the Coomassie gel.</p> <p>(E) Gene Ontology Panther protein classification of the two mass spectrometry data sets (contaminants and proteins found more enriched in the control bead-bound fractions were removed).</p> <p>(F) Number of proteins enriched in the phalloidin-bead-bound fractions in the two mass spectrometry analyses (MS1 and MS2) for different phases of the cell cycle (G2, metaphase).</p> <p>(G) Total number of proteins enriched in the phalloidin-bound fractions and overlap between the two mass spectrometry experiments.</p> <p>See also <a href="#mmc1">Figure S1</a>.</p>   </div> <p /> <p id="p0055">To isolate F-ABPs, we used a phalloidin affinity matrix for mapping F-actin interactomes (<a href="#bib49">Samwer et al., 2013</a>). First, we recovered soluble fractions containing maximal amounts of actin. Cells were lysed in the presence of low doses of the actin depolymerizing drug Latrunculin A and the supernatant was collected. After optimization of lysis conditions, our protocol yielded more than 70% of the cellular actin in the supernatant (<a href="#mmc1">Figure S1</a>C). We then applied this lysis protocol to each synchronized cell population and recovered cellular extracts with a comparable amount of actin in each phase (<a href="#fig1">Figure 1</a>C). In order to isolate F-ABPs, we incubated the solubilized extracts with ATP and MgCl<sub>2</sub>, to induce actin re-polymerization, together with custom-made superparamagnetic beads conjugated with phalloidin, a toxin that binds and stabilizes F-actin (<a href="#fig1">Figure 1</a>A). We verified that the post-translational modifications appeared maintained and that the incubation did not affect the cell-cycle state of the extracts (<a href="#mmc1">Figures S1</a>D and <a href="#mmc1">S2</a>E). We also confirmed by scanning electron microscopy that our protocol induced actin polymerization in the extracts (<a href="#mmc1">Figure S1</a>F). Control and phalloidin-bead-bound fractions were then eluted and subjected to SDS-PAGE and Coomassie staining to assess protein contents (<a href="#fig1">Figures 1</a>D and <a href="#mmc1">S1</a>G). As expected, we observed a strong enrichment of actin in the phalloidin-bead elution compared to controls (<a href="#fig1">Figures 1</a>D and <a href="#mmc1">S1</a>G). The gels were then trypsin digested and the protein composition of the elutions was analyzed using liquid chromatography-tandem mass spectrometry (LC-MS/MS). We performed two independent experiments (referred to as MS1 and MS2), which identified 37,221 and 30,502 spectra, respectively (corresponding to 396 and 317 proteins) (<a href="#mmc10">Table S1</a>). This list was reduced to 290 and 211 proteins, respectively, after removing contaminants and proteins more represented in the control bead elutions (<a href="#mmc10">Table S1</a>). As expected, β-actin was the most abundant protein identified (with a mean of 1,967 and 1,902 spectra in MS1 and MS2, respectively). A Gene Ontology analysis revealed that "cytoskeletal proteins" and the "actin family" represented the most enriched molecular classes (<a href="#fig1">Figure 1</a>E). Importantly, the protein class distributions were similar between replicate experiments, suggesting that our assay yielded highly reproducible data (<a href="#fig1">Figure 1</a>E). The number of F-actin binders identified was comparable between the two phases of the cell cycle in both experiments (<a href="#fig1">Figure 1</a>F), with ∼51% of the proteins identified being present in both phases in at least one of the mass spectrometry experiments (<a href="#mmc10">Table S1</a>). Overall, 168 F-actin binders were reproducibly identified between replicate experiments (<a href="#fig1">Figure 1</a>G).</p> </div> <div class="changesinf-actininteractomebetweeninterphaseandmitosis" title="sec"> <div class="title" tagx="title" title="title">
Changes in F-Actin Interactome between Interphase and Mitosis</div> <p id="p0060">We then explored how the F-actin interactome (thereafter FAI) changes between interphase and mitosis (<a href="#mmc10">Table S1</a>). We analyzed changes in protein levels in the F-actin-associated fractions (thereafter FAI abundance), as estimated by total spectral counts, between interphase and metaphase (<a href="#fig2">Figures 2</a>A and 2B). Importantly, we verified that changes in FAI abundance did not simply reflect differences in protein levels in the solubilized extracts (<a href="#mmc1">Figure S2</a>). For 67% of the factors identified, FAI abundance changed by less than 2-fold between the two cell cycle phases in one or both experiments (<a href="#fig2">Figure 2</a>A, green), suggesting that actin network composition is relatively similar in interphase and mitosis. Only six proteins (3.5% of total) displayed inconsistent changes, with FAI abundance increasing between interphase and metaphase in one experiment and decreasing in the other; these proteins were excluded from further analysis (<a href="#fig2">Figure 2</a>A, orange). The remaining 48 proteins reproducibly displayed at least 2-fold changes in FAI abundance between interphase and mitosis (<a href="#fig2">Figures 2</a>A (blue) and 2C; <a href="#tbl1">Tables 1</a> and <a href="#mmc10">S1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Changes in F-Actin Interactome between Interphase and Metaphase</p> <p>(A) Numbers of proteins displaying increased (&amp;gt;2-fold increase), decreased (&amp;gt;2-fold decrease), or unchanged (&amp;lt;2-fold change) amounts, as estimated by the total spectral count, in the interphase and metaphase FAIs in the two mass spectrometry experiments. Blue: proteins displaying consistent changes of at least 2-fold in the two experiments. Green: proteins displaying less than 2-fold change in one or both experiments. Orange: proteins displaying inconsistent changes between the two experiments.</p> <p>(B) Graphic representation of the changes in total spectral count, as a readout of protein abundance, between interphase and metaphase for proteins found to bind F-actin in the two mass spectrometry experiments (MS1, light blue and MS2, dark blue). Each line corresponds to an individual protein. Proteins that changed by more than 2-fold in one experiment but not the other are not represented.</p> <p>(C) FAI abundance changes between interphase and metaphase. The graph displays the total spectral count as a function of the ratio of spectral counts between metaphase and interphase for the 48 proteins that display at least a 2-fold change in abundance in both mass spectrometry experiments (MS1, light blue and MS2, dark blue). Total spectral counts for each protein were normalized to the spectral count of actin-β (ACTB) and multiplied by the mean spectral count of ACTB (used as a reference) for each mass spectrometry experiment. When a protein was only present in interphase or metaphase, its spectral count was set to 1 instead of 0, so that the log of the ratio could be computed; the corresponding proteins fall on the red dashed line in the graph; all spectral counts are reported in <a href="#tbl1">Table 1</a>. The gray shaded region highlights the area with less than a 2-fold change in spectral count between interphase and metaphase.</p> <p>See also <a href="#mmc1">Figure S2</a> and <a href="#mmc10">Table S1</a>.</p>   </div> <div class="table-wrap_UNKNOWN" position="float" id="tbl1"><span class="label" tagx="label" title="label">Table 1</span> <p>List of Proteins Displaying at Least 2-Fold Changes in F-Actin Interactome Abundance between Interphase and Metaphase</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="3">Protein name</th> <th colspan="4">Normalized total Spectral Count <hr /> </th> <th>Mean PAI <hr /> </th> </tr> <tr> <th colspan="2">MS1 <hr /> </th> <th colspan="2">MS2 <hr /> </th> <th rowspan="2">MS1-MS2</th> </tr> <tr> <th>G2</th> <th>Metaphase</th> <th>G2</th> <th>Metaphase</th> </tr> </thead> <tbody> <tr> <td colspan="5"><b>Enriched in interphase</b></td> <td>G2 PAI</td> </tr> <tr> <td colspan="6"> <hr /> </td> </tr> <tr> <td>TPM4</td> <td>87</td> <td>0</td> <td>253</td> <td>103</td> <td>515</td> </tr> <tr> <td>TPM2</td> <td>79</td> <td>0</td> <td>143</td> <td>0</td> <td>335</td> </tr> <tr> <td>MYH9</td> <td>627</td> <td>99</td> <td>549</td> <td>15</td> <td>259</td> </tr> <tr> <td>TBB3</td> <td>106</td> <td>0</td> <td>72</td> <td>0</td> <td>178</td> </tr> <tr> <td>CALD1</td> <td>103</td> <td>1</td> <td>96</td> <td>1</td> <td>163</td> </tr> <tr> <td>TBA4A</td> <td>92</td> <td>27</td> <td>52</td> <td>26</td> <td>145</td> </tr> <tr> <td>TBB6</td> <td>48</td> <td>0</td> <td>35</td> <td>0</td> <td>83</td> </tr> <tr> <td>PPR18</td> <td>32</td> <td>14</td> <td>29</td> <td>11</td> <td>45</td> </tr> <tr> <td>MYH10</td> <td>94</td> <td>0</td> <td>95</td> <td>0</td> <td>41</td> </tr> <tr> <td>SPTB2</td> <td>125</td> <td>2</td> <td>75</td> <td>23</td> <td>36</td> </tr> <tr> <td>VASP</td> <td>17</td> <td>3</td> <td>10</td> <td>4</td> <td>35</td> </tr> <tr> <td>ANLN</td> <td>61</td> <td>0</td> <td>25</td> <td>0</td> <td>35</td> </tr> <tr> <td>RLA0</td> <td>11</td> <td>4</td> <td>6</td> <td>1</td> <td>25</td> </tr> <tr> <td>MYH14</td> <td>52</td> <td>0</td> <td>48</td> <td>0</td> <td>22</td> </tr> <tr> <td>ENAH</td> <td>10</td> <td>0</td> <td>9</td> <td>1</td> <td>14</td> </tr> <tr> <td>SVIL</td> <td>29</td> <td>6</td> <td>18</td> <td>0</td> <td>12</td> </tr> <tr> <td>EF1G</td> <td>8</td> <td>1</td> <td>2</td> <td>0</td> <td>10</td> </tr> <tr> <td>NUCL</td> <td>12</td> <td>3</td> <td>2</td> <td>0</td> <td>9</td> </tr> <tr> <td>IF4A1</td> <td>5</td> <td>2</td> <td>3</td> <td>0</td> <td>9</td> </tr> <tr> <td>SERA</td> <td>3</td> <td>1</td> <td>6</td> <td>0</td> <td>8</td> </tr> <tr> <td>DNJA1</td> <td>4</td> <td>0</td> <td>2</td> <td>1</td> <td>7</td> </tr> <tr> <td>XPO2</td> <td>8</td> <td>0</td> <td>4</td> <td>0</td> <td>6</td> </tr> <tr> <td>DOCK7</td> <td>17</td> <td>1</td> <td>8</td> <td>0</td> <td>5</td> </tr> <tr> <td>SYEP</td> <td>11</td> <td>1</td> <td>6</td> <td>0</td> <td>5</td> </tr> <tr> <td>MY18A</td> <td>9</td> <td>0</td> <td>10</td> <td>0</td> <td>4</td> </tr> <tr> <td>PRKDC</td> <td>4</td> <td>0</td> <td>3</td> <td>0</td> <td>1</td> </tr> <tr> <td colspan="6"> <hr /> </td> </tr> <tr> <td colspan="5"><b>Enriched in metaphase</b></td> <td>M PAI</td> </tr> <tr> <td colspan="6"> <hr /> </td> </tr> <tr> <td>PLEC</td> <td>249</td> <td>1,405</td> <td>0</td> <td>638</td> <td>197</td> </tr> <tr> <td>COF1</td> <td>0</td> <td>25</td> <td>5</td> <td>12</td> <td>99</td> </tr> <tr> <td>GTF2I</td> <td>15</td> <td>83</td> <td>0</td> <td>68</td> <td>70</td> </tr> <tr> <td>VIME</td> <td>15</td> <td>39</td> <td>8</td> <td>27</td> <td>60</td> </tr> <tr> <td>ARPC2</td> <td>1</td> <td>14</td> <td>2</td> <td>18</td> <td>47</td> </tr> <tr> <td>SRC8</td> <td>5</td> <td>21</td> <td>6</td> <td>34</td> <td>45</td> </tr> <tr> <td>ARPC5</td> <td>5</td> <td>10</td> <td>0</td> <td>3</td> <td>39</td> </tr> <tr> <td>WDR1</td> <td>11</td> <td>26</td> <td>2</td> <td>22</td> <td>36</td> </tr> <tr> <td>ARC1B</td> <td>2</td> <td>13</td> <td>0</td> <td>11</td> <td>29</td> </tr> <tr> <td>HORN</td> <td>9</td> <td>115</td> <td>6</td> <td>23</td> <td>24</td> </tr> <tr> <td>RL13</td> <td>1</td> <td>5</td> <td>1</td> <td>3</td> <td>17</td> </tr> <tr> <td>DDX5</td> <td>7</td> <td>18</td> <td>1</td> <td>4</td> <td>16</td> </tr> <tr> <td>TRI27</td> <td>0</td> <td>3</td> <td>0</td> <td>9</td> <td>14</td> </tr> <tr> <td>EPIPL</td> <td>0</td> <td>118</td> <td>0</td> <td>39</td> <td>14</td> </tr> <tr> <td>EF2</td> <td>0</td> <td>22</td> <td>0</td> <td>5</td> <td>14</td> </tr> <tr> <td>RSSA</td> <td>2</td> <td>6</td> <td>0</td> <td>3</td> <td>14</td> </tr> <tr> <td>CS021</td> <td>4</td> <td>10</td> <td>0</td> <td>4</td> <td>9</td> </tr> <tr> <td>TPX2</td> <td>0</td> <td>10</td> <td>0</td> <td>6</td> <td>9</td> </tr> <tr> <td>C4BPA</td> <td>0</td> <td>7</td> <td>0</td> <td>4</td> <td>8</td> </tr> <tr> <td>HNRPU</td> <td>2</td> <td>6</td> <td>0</td> <td>3</td> <td>5</td> </tr> <tr> <td>AFAP1</td> <td>1</td> <td>4</td> <td>0</td> <td>4</td> <td>5</td> </tr> <tr> <td>MICA3</td> <td>1</td> <td>3</td> <td>0</td> <td>5</td> <td>4</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Normalized total spectral count (normalization as in <a href="#fig2">Figure 2</a>C) for proteins enriched in the interphase (top part) and metaphase (bottom part) FAIs for the two mass spectrometry repeats (MS1 and MS2). The last column displays the mean protein abundance index (PAI), which is calculated as the ratio of the spectral count of the protein of interest, averaged between the two mass spectrometry repeats, and of the molecular weight of the protein. The PAI corrects for differences in the number of observable peptides due to protein size and is thus an adjusted readout of protein abundance.</p> </div> </div> </div> <p /> <p id="p0065">Consistent with previous observations (<a href="#bib28">Kordowska et al., 2006</a>, <a href="#bib59">Yamashiro et al., 1990</a>, <a href="#bib60">Yamashiro et al., 1991</a>), Caldesmon 1 (CALD1), a myosin II activity regulator, was associated with F-actin in interphase but not in metaphase (<a href="#fig2">Figure 2</a>C and <a href="#tbl1">Table 1</a>). As expected, we also found that Ena/VASP, a key stress fiber regulator, was more associated with F-actin in extracts from interphase cells, where stress fibers dominate actin organization (<a href="#bib12">Ciobanasu et al., 2012</a>, <a href="#bib29">Krause et al., 2002</a>). Our analysis further revealed that myosin heavy chains IIA (MYH9), IIB (MYH10), IIC (MYH14), and myosin XVIII (MY18A), as well as different tropomyosins (TPM2 and TPM4), displayed a higher FAI abundance in interphase than in metaphase. Finally, we observed that several subunits of the Arp2/3 complex (ARC1B, ARPC2, and ARPC5), as well as the intermediate filament protein vimentin and the actin-vimentin binding factor plectin, were enriched in the FAI of metaphase cells. Overall, our FAI mass spectrometry analysis uncovered several potential regulators of actin architecture during the cell cycle.</p> </div> <div class="regulationofcorticalactinorganization" title="sec"> <div class="title" tagx="title" title="title">
Regulation of Cortical Actin Organization</div> <p id="p0070">We then investigated if proteins displaying increased FAI abundance in mitosis (<a href="#tbl1">Table 1</a>) affected mitotic actin cortex architecture. For this, we depleted each identified mitotic hit (<a href="#fig2">Figure 2</a>C and <a href="#tbl1">Table 1</a>, excluding two ribosomal proteins) and investigated the effect on cellular actin organization. Target proteins were depleted using siRNA pools and mRNA transcript depletion efficiency was confirmed using qPCR (<a href="#mmc1">Figure S3</a>A). We observed that none of the factors tested individually led to an overall disruption of the cortex, suggesting that the formation of a cortical actin network in mitosis is robust to perturbations (<a href="#mmc1">Figure S3</a>B). We then measured cortical thickness, a key readout of cortical architecture (<a href="#bib11">Chugh et al., 2017</a>) in the different depletion conditions. As cortical thickness is generally below the resolution limit of conventional light microscopy (<a href="#bib13">Clark et al., 2013</a>), we used direct stochastic optical reconstruction microscopy (dSTORM) (<a href="#fig3">Figures 3</a>A and 3B). As dSTORM resolution is ∼40 nm when imaging at the cell center, significantly lower than cortical thickness, we measured the full width at half maximum (FWHM) of actin profiles across the cortex as a readout of thickness (<a href="#fig3">Figures 3</a>B, 3C, and <a href="#mmc1">S3</a>C). Depletion of the actin severing protein cofilin (CFL1) led to an increase in thickness (<a href="#fig3">Figures 3</a>B, 3C, and <a href="#mmc1">S3</a>C), consistent with previous observations (<a href="#bib11">Chugh et al., 2017</a>, <a href="#bib13">Clark et al., 2013</a>). We further identified several additional regulators of cortex thickness: the transcription factor GTF2I, the oxydoreductase MICAL3, and the spindle assembly factor TPX2 (<a href="#fig3">Figures 3</a>B and <a href="#mmc1">S3</a>C). Surprisingly, we also found that depletion of the intermediate filament protein vimentin, and of the F-actin intermediate filament linker plectin, led to increased cortical thickness, suggesting that intermediate filaments play a role in controlling actin architecture in mitosis (<a href="#fig3">Figures 3</a>B, 3C, and <a href="#mmc1">S3</a>C). To our knowledge, how intermediate filaments affect mitotic actin organization has not been investigated. Therefore, we focused on vimentin for the rest of the study. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Regulation of Cortical Actin Organization</p> <p>(A) Cortex thickness measurements by super-resolution microscopy. Top: STORM image of a mitotic cell stained with phalloidin-Alexa647. Bottom: the cell was automatically segmented and the mean full width at half maximum (FWHM) of intensity profiles across the actin cortex was measured as a readout of actin cortex thickness. Scale bar, 10 μm.</p> <p>(B) Actin FWHM after depletion of protein candidates. Relative FWHM values for siRNA-depleted cells (target, black data points) were obtained by dividing the FWHM of individual depleted cells by the median of the corresponding non-targeting control siRNA (NTC, gray datapoints). Individual data points as well as median and interquartile range are plotted. N ≥ 3 independent experiments. Gene names are used, instead of protein names as in <a href="#tbl1">Tables 1</a>, <a href="#mmc10">S1</a> and <a href="#fig2">Figure 2</a>. Depletion levels were checked by qPCR (<a href="#mmc1">Figure S3</a>A). Unpaired Welch’s t test; p values: <sup>∗∗∗</sup>p &amp;lt; 0.001; <sup>∗∗</sup>p &amp;lt; 0.01.</p> <p>(C) Examples of STORM images of the actin cortex for treatments leading to increased FWHM. Scale bars: top panel, 10 μm; bottom panel, 2 μm.</p> <p>See also <a href="#mmc1">Figure S3</a> and <a href="#mmc1">Table S2</a>.</p>   </div> <p /> </div> <div class="vimentinlocalizestothecellcortexinmitosisinaplectin-dependentmannerandcontrolsactincortexorganizationandmechanics" title="sec"> <div class="title" tagx="title" title="title">
Vimentin Localizes to the Cell Cortex in Mitosis in a Plectin-Dependent Manner and Controls Actin Cortex Organization and Mechanics</div> <p id="p0075">We first characterized vimentin localization and dynamics during mitosis. Previous studies suggested that vimentin networks disassemble during mitosis in some cell types but not in others (<a href="#bib8">Charras et al., 2009</a>, <a href="#bib9">Chou et al., 2003</a>, <a href="#bib17">Duarte et al., 2019</a>). We thus monitored vimentin during the cell cycle in HeLa cells (<a href="#mmc2">Video S1</a>). We observed that vimentin distribution changed from a prominent cytoplasmic network in interphase to cytoplasmic aggregates and a clear cortical layer in metaphase (<a href="#mmc2">Video S1</a>; <a href="#fig4">Figure 4</a>A). In some cells, the cortical vimentin layer appeared somewhat patchy (<a href="#fig4">Figures 4</a>A and <a href="#mmc1">S4</a>A, left panel). 3D visualization revealed that mitotic vimentin formed a network of thick cables at the cell periphery, with a faint vimentin meshwork between the cables (<a href="#mmc1">Figure S4</a>A). Finally, we explored the relative localization of vimentin and actin using 2-color dSTORM (<a href="#fig4">Figures 4</a>B and <a href="#mmc1">S4</a>B). As also reported in a recent study in several other cell types (<a href="#bib17">Duarte et al., 2019</a>), the vimentin layer was located immediately underneath, and might thus interact with, the actin cortex (<a href="#fig4">Figures 4</a>B and <a href="#mmc1">S4</a>B). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Vimentin Forms a Sub-cortical Layer in Mitosis and Controls Actin Cortex Organization and Mechanics</p> <p>(A) Representative time lapse of HeLa cells expressing mCherry-vimentin through the cell cycle (images from <a href="#mmc2">Video S1</a>). Time in minutes. Scale bar, 20 μm.</p> <p>(B) Top: representative two-color dSTORM image of vimentin (magenta) and F-actin (cyan) in a mitotic cell. Scale bars, 5 and 1 μm. Bottom: distance between the peaks of the actin and vimentin fluorescence profiles across the cortex (actin-VIM; see <a href="#sec5">STAR Methods</a> for details). To rule out any systematic shift, we also measured peak-to-peak distance for actin profiles where actin was labeled with two differentially labeled phalloidin probes (actin-actin; <a href="#mmc1">Figure S4</a>B). Graph: mean ± standard deviation (actin-actin, n = 10 and actin-VIM, n = 14; at least 3 independent experiments). Mann-Whitney test.</p> <p>(C) Left: representative images of endogenous vimentin localization in mitotic HeLa cells transfected with non-targeting control siRNA (siNTC), siRNA against plectin (siPLEC), or vimentin (siVIM). Scale bars, 5 μm. Right: quantification of vimentin enrichment at the cell cortex. Graph: mean ± standard deviation; 3 independent experiments. siNTC, n = 30 and siPLEC, n = 29. Mann-Whitney test.</p> <p>(D) Cortex tension in mitotic (prometaphase) cells measured by atomic force microscopy (top schematic, see <a href="#sec5">STAR Methods</a> for details on tension measurement) for cells treated with non-targeting control siRNA (siNTC, n = 47), siRNA against vimentin (siVIM, n = 28), or siRNA against plectin (siPLEC, n = 18). Graph: median and interquartile range; at least 3 independent experiments. Kruskal-Wallis test.</p> <p>See also <a href="#mmc1">Figure S4</a>.</p>   </div> <p /> <p id="p0080"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S1. HeLa Cells Transfected with mCherry-Vimentin Going through the Cell Cycle, Related to Figure 4A</div> <p>Frames were acquired every 2 min. Scale bar, 20 μm.</p>  <div class="media" title="media"><a href="mmc2.mp4">LINK</a></div> </div> <p /> <p id="p0090">We then asked whether plectin, an F-actin-vimentin linker (<a href="#bib57">Wiche et al., 2015</a>) and a key hit in our screen (<a href="#fig3">Figure 3</a>), contributed to recruiting vimentin to the F-actin cortex in mitosis. We observed that plectin formed cytoplasmic networks in interphase cells and reorganized into a cortical layer in mitosis (<a href="#mmc1">Figure S4</a>C). We then explored how depleting plectin and vimentin affected their respective localizations (<a href="#fig4">Figures 4</a>C, <a href="#mmc1">S4</a>D, and S4E). Interestingly, in plectin-depleted mitotic cells, vimentin did not localize to the cortex but instead accumulated in cytoplasmic bundles (<a href="#fig4">Figure 4</a>C). In contrast, vimentin depletion did not affect mitotic plectin localization (<a href="#mmc1">Figure S4</a>E). Thus, plectin appears to be instrumental in recruiting vimentin to the mitotic cortex.</p> <p id="p0095">Finally, we explored how the plectin-dependent formation of a vimentin sub-cortical layer affects functional properties of the mitotic actin cortex. The key function of the cortex in mitosis is the generation of high tension driving mitotic rounding. We have shown recently that cortical thickness decreases in mitosis, and this cortex thinning contributes to increasing cortical tension (<a href="#bib11">Chugh et al., 2017</a>). Therefore, we measured cortical tension in vimentin- and plectin-depleted cells using atomic force microscopy. Both treatments resulted in a strong decrease in mitotic cortex tension (<a href="#fig4">Figure 4</a>D), with vimentin depletion bringing cortex tension to levels comparable to the low tensions measured in rounded interphase cells (<a href="#mmc1">Figure S4</a>F) (<a href="#bib11">Chugh et al., 2017</a>). As cortical tension also strongly depends on myosin II activity, we verified that vimentin depletion did not significantly affect the levels of total or phosphorylated myosin regulatory light chain (<a href="#mmc1">Figures S4</a>G and S4H). Together, these results suggest that in mitotic cells, plectin recruits a vimentin network to the cortex, which results in thinning of the cortical actin layer and contributes to increasing cortical tension.</p> </div> <div class="phosphorylationcontrolstheorganizationofthesub-corticalvimentinnetworkinmitosis" title="sec"> <div class="title" tagx="title" title="title">
Phosphorylation Controls the Organization of the Sub-cortical Vimentin Network in Mitosis</div> <p id="p0100">We then further explored how the organization of the mitotic sub-cortical vimentin network is regulated. The formation of a sub-cortical vimentin layer in mitosis is not simply a consequence of mitotic rounding, as artificially rounding interphase cells did not lead to recruitment of vimentin to the actin cortex (<a href="#mmc1">Figure S5</a>A). This suggests that vimentin reorganization in mitosis is the result of a mitosis-specific signal. The mitotic kinases CDK1 and PLK1 were previously shown to phosphorylate vimentin on Ser56 and Ser83, respectively (<a href="#bib56">Tsujimura et al., 1994</a>, <a href="#bib58">Yamaguchi et al., 2005</a>, <a href="#bib61">Yasui et al., 2001</a>). We thus asked how mitotic kinase activity affects vimentin organization and found that both CDK1 inhibition using RO-3306 and PLK1 inhibition using BI2536 disrupted vimentin organization in mitosis, converting the network of cortical vimentin cables into a more diffuse cortical meshwork (<a href="#mmc1">Figure S5</a>B; <a href="#mmc3">Video S2</a>; <a href="#mmc4">Video S3</a>). Furthermore, staining with phospho-specific antibodies showed that vimentin phosphorylated on Ser56 and Ser83 localized to the cortex in mitosis, at levels similar to but not higher than total vimentin (<a href="#mmc1">Figure S5</a>C). Together, these experiments suggest that phosphorylation is unlikely to target vimentin to the cortex but that it could regulate the organization of the sub-cortical vimentin network.</p> <p id="p0085"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S2. HeLa Cells Stably Expressing GFP-Vimentin-WT Treated with CDK1 Inhibitor, Related to Figure 5</div> <p>The video displays projected z-stacks for each time point. Frames were acquired every 1 min. Cells were synchronized in pro-metaphase using STLC and treated with the CDK1 inhibitor RO3306 immediately before the movie. Scale bar, 10 μm.</p>  <div class="media" title="media"><a href="mmc3.mp4">LINK</a></div> </div> <p /> <p id="p0105"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S3. Representative 3D Visualization of Vimentin Stainings in HeLa Cells Treated with DMSO, CDK1 Inhibitor, or PLK1 Inhibitor, Related to Figure S5B</div>  <div class="media" title="media"><a href="mmc4.mp4">LINK</a></div> </div> <p /> <p id="p0110">To test this, we generated cell lines stably overexpressing GFP-tagged wild-type vimentin (WT-VIM) or vimentin with single point mutations that either mimic phosphorylation by CDK1 and PLK1 (VIM-56E and VIM-83E, respectively) or abolish CDK1 phosphorylation (VIM-56A) (<a href="#mmc1">Figure S5</a>D). All three mutants formed cytoplasmic networks in interphase, suggesting that the mutant vimentins normally integrate into filaments, but VIM-56E-GFP and VIM-83E-GFP also often displayed cytoplasmic aggregates, indicating that the phosphomimetic mutations affect vimentin network organization in interphase (<a href="#mmc1">Figure S5</a>E). In mitosis, expression of VIM-WT, -56A, or -83E did not significantly alter cortical vimentin localization (<a href="#fig5">Figure 5</a>A). In contrast, expression of VIM-56E reduced the cortical accumulation of vimentin in mitosis (<a href="#fig5">Figure 5</a>A). Analysis of 3D stacks indicated that phosphomimetic VIM-56E induced the collapse of vimentin cables into aggregates (<a href="#fig5">Figure 5</a>B; <a href="#mmc5">Video S4</a>); the aggregates were localized close to the cortex, explaining the patchy appearance of the vimentin staining in single plane images (<a href="#fig5">Figure 5</a>A). Finally, the localization of the mutant vimentins themselves mostly recapitulated the distribution of total vimentin, with the exception of VIM-83E, which displayed a slight decrease in cortical accumulation compared to total vimentin (<a href="#fig5">Figures 5</a>A and <a href="#mmc1">S5</a>F). Together, these results indicate that vimentin phosphorylation on Ser56, and to a milder degree on Ser83, controls the organization of the vimentin network in mitosis leading to the formation of a bundle-like meshwork, with excessive phosphorylation resulting in aggregation of vimentin into localized spots. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Sub-cortical Vimentin Directly Controls Cortical Actin Organization in Mitosis</p> <p>(A) Left: representative images of total vimentin (immunostaining) in mitotic Hela cells stably expressing GFP-vimentin mutants. Right: corresponding quantification of vimentin enrichment at the cell cortex. Scale bar, 5 μm. Right: graph, mean ± standard deviation; 3 independent experiments. WT, n = 31; 56A, n = 31; 56E, n = 33; and 83E, n = 29. Kruskal-Wallis test; ns, non-significant.</p> <p>(B) Representative maximum z projections of confocal stacks across mitotic HeLa cells stably expressing vimentin mutants and stained for total vimentin. Scale bar, 10 μm. See <a href="#mmc4">Video S3</a> for 3D visualizations.</p> <p>(C) Left: examples of STORM images of the actin cortex in cells expressing the different vimentin mutants. Scale bars, 0.5 and 5 μm. Right: values of full width at half maximum (FWHM) of the cortical actin signal in cells expressing the different vimentin mutants. NT, n = 16; WT, n = 26; 56E, n = 20; 56A, n = 19; 83E, and n = 21. 3 independent experiments. Graph: mean ± standard deviation. ANOVA Tukey’s multiple comparison.</p> <p>(D) Cortex tension in mitotic (prometaphase) cells expressing VIM-WT, VIM-56E, or VIM-83E. Graph: median with interquartile range; 3 independent experiments. WT, n = 60; 56E, n = 29; and 83E, n = 26. Kruskal-Wallis test.</p> <p>(E) Left: representative time lapses of curvature maps along the cell contour (see <a href="#sec5">STAR Methods</a> for details) during ablation experiments in HeLa cells stably expressing GFP-VIM-WT or GFP-VIM-56E (displayed in insets; yellow circles, ablation region). Right: quantification of the effects of laser ablation in HeLa cells expressing GFP-VIM-WT or GFP-VIM-56E. N = 4 independent experiments for VIM-WT (n = 52 cells) and N = 3 for VIM-56E (n = 57 cells). In some cells (15 cells out of 52 for VIM-WT and 26 cells out of 57 for VIM-56E), ablation elicited bleb formation at the site of ablation (<a href="#mmc8">Video S7</a>, right panel); these cells were excluded from the quantifications, as bleb formation indicated ablation disrupted the actin cortex (<a href="#bib54">Tinevez et al., 2009</a>). Scale bars, 5 μm.</p> <p>See also <a href="#mmc1">Figure S5</a>.</p>   </div> <p /> <p id="p0115"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S4. Representative 3D Visualization of Total Vimentin Stainings in HeLa Cells Stably Expressing GFP-Vimentin-WT, -56A, -56E, or -83E, Related to Figure 5B</div>  <div class="media" title="media"><a href="mmc5.mp4">LINK</a></div> </div> <p /> </div> <div class="thesub-corticalvimentinlayerdirectlycontrolscorticalactinorganization" title="sec"> <div class="title" tagx="title" title="title">
The Sub-cortical Vimentin Layer Directly Controls Cortical Actin Organization</div> <p id="p0120">We then asked whether expression of the phosphomimetic vimentin mutants affected the actin cortex. Interestingly, we found that actin cortex thickness was increased in cells expressing VIM-56E and VIM-83E (<a href="#fig5">Figure 5</a>C). Furthremore, cortical tension was decreased in cells expressing VIM-56E, though not VIM-83E (<a href="#fig5">Figure 5</a>D), possibly because of the weaker effect of this mutant on actin organization compared to the VIM-56E mutant (<a href="#fig5">Figures 5</a>A and 5B). These results suggest that the sub-cortical vimentin layer directly controls the arrangement of the actin cortex in mitosis.</p> <p id="p0125">Because vimentin forms stiff elastic networks, we speculated the sub-cortical vimentin layer might mechanically counteract the cortex. To test this hypothesis, we used laser ablation to locally disrupt the vimentin layer in mitotic cells. We first verified that with our settings, laser ablation did not disrupt the plasma membrane (<a href="#mmc6">Video S5</a>). Given the proximity of the vimentin and actin layers, the ablation of the vimentin layer occasionally led to the disruption of the actin cortex, resulting in bleb formation (<a href="#mmc7">Video S6</a>, lower panel), consistent with previous observations (<a href="#bib54">Tinevez et al., 2009</a>). When no bleb was observed (indicating that the actin cortex was not disrupted, <a href="#mmc7">Video S6</a>), we found that in cells expressing VIM-WT, local ablation focused on sub-cortical vimentin led to local flattening of the cell surface, suggesting a local contraction of the cortex (18 cells out of 35, <a href="#fig5">Figure 5</a>E; <a href="#mmc8">Video S7</a>). In contrast, cortex flattening was only seldom observed upon similar laser ablation in cells expressing VIM-56E, where the levels of vimentin at the cortex are reduced (9 cells out of 31, <a href="#fig5">Figure 5</a>E; <a href="#mmc8">Video S7</a>). Finally, inhibiting actomyosin contractility with the ROCK inhibitor Y27632 (<a href="#bib54">Tinevez et al., 2009</a>) strongly decreased occurrences of cortex flattening upon sub-cortical ablation (<a href="#mmc1">Figure S5</a>G). These experiments suggest that disrupting the sub-cortical vimentin layer locally releases mechanical resistance to cortical forces, leading to a contraction of the overlaying actomyosin cortex and cell surface flattening. We thus propose that the sub-cortical vimentin network could contribute to mitotic cortex thinning by physically resisting the growth of the cortical actin meshwork toward the cytoplasm.</p> <p id="p0130"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S5. Example of Ablation Experiments Leading to Flattening of the Cell Surface in Presence of a Membrane Dye, Related to Figure 5</div> <p>Ablation was performed in HeLa cells stably expressing GFP-vimentin-WT and in presence of Cell Mask to monitor the plasma membrane during ablation (left panel) or in presence of fluorescent dextran in the medium (right panel). The yellow circle represents the site of ablation.</p>  <div class="media" title="media"><a href="mmc6.mp4">LINK</a></div> </div> <p /> <p id="p0135"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S6. Example of Actin Behavior during Ablation Experiments Leading to Flattening of the Cell Surface or Triggering Bleb Formation, Related to Figure 5</div> <p>Ablation was performed in HeLa cells stably expressing GFP-vimentin-WT (left panel) and transfected with mCherry-Lifeact to monitor the actin cortex during ablation (right panel). The yellow circle represents the site of ablation.</p>  <div class="media" title="media"><a href="mmc7.mp4">LINK</a></div> </div> <p /> <p id="p0140"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S7. Example of Ablation Experiments Leading to Flattening of the Cell Surface (Left Panel); Not Eliciting Changes in Cell Surface Curvature (Middle Panel); or Triggering a Bleb (Right Panel), Related to Figure 5F</div> <p>Ablation was performed in HeLa cells stably expressing GFP-vimentin-WT or -56E. The yellow circle represents the site of ablation. Frames were acquired every 3.26 s and the ablation was performed at 25 s (left panel) and at 9s (middle panel and right panels). Scale bars, 5 μm.</p>  <div class="media" title="media"><a href="mmc8.mp4">LINK</a></div> </div> <p /> </div> <div class="vimentinpromotesmitoticcellrounding" title="sec"> <div class="title" tagx="title" title="title">
Vimentin Promotes Mitotic Cell Rounding</div> <p id="p0145">Finally, we asked if the changes in cortex organization and tension upon vimentin depletion had functional consequences on mitotic rounding. We observed that vimentin-depleted cells were less round in mitosis than control cells (<a href="#fig6">Figure 6</a>A) and that the rounding-up process was slower compared to control cells (<a href="#fig6">Figure 6</a>B; <a href="#mmc9">Video S8</a>). This shows that vimentin plays a role in mitotic rounding, possibly by promoting cortex thinning and the concomitant cortex tension increase. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Vimentin Is Important for Mitotic Cell Rounding</p> <p>(A) Representative images of control and vimentin-depleted cells in metaphase and quantification of cell roundness (see <a href="#sec5">STAR Methods</a>); R on the images is the roundness coefficient of the cell depicted. Scale bars, 10 μm. Graph: mean ± standard deviation (siNTC, n = 21 and siVIM, n = 23; 3 independent experiments). Non-parametric Mann-Whitney test.</p> <p>(B) Representative time lapses of mitotic rounding in control and vimentin-depleted HeLa cells stably expressing H2B-mCherry (red) and Lifeact-GFP (cyan). Scale bars, 20 μm. Right: quantification of rounding timing. Graph: mean ± standard deviation (NT, n = 16; siNTC, n = 22; and siVIM, n = 63. 2 independent experiments). NT = non-transfected. ANOVA Tukey’s multiple comparison; ns, non-significant.</p> <p>(C) Representative images of HeLa cells stably expressing H2B-mCherry to mark the chromosomes, and aplasma membrane marker (membrane targeting domain of LYN tagged with GFP), dividing unconfined or in confinement under a 5 kPa polyacrylamide gel. Scale bars, 10 μm.</p> <p>(D) Quantification of chromosome segregation defects in control and vimentin-depleted cells during division without confinement (no gel) and in confinement under 5 kPa gels. Percentages of cells with lagging chromosomes, chromosomal bridges or multipolar spindles were quantified in time lapse movies of cells stably expressing H2B-mCherry (N ≥ 3 independent experiments; no gel, n = 64 and n = 65 and 5 kPa gel, n = 112 and n = 101 for siNTC and siVIM, respectively).</p> <p>(E) Representative images of lagging chromosomes, chromosomal bridges, and multipolar spindles in HeLa cells stably expressing H2B-mCherry (inverted contrast) to mark the chromosomes, dividing in confinement under a 5-kPa polyacrylamide gel. Scale bar, 10 μm.</p> <p>See also <a href="#mmc1">Figure S5</a>H.</p>   </div> <p /> <p id="p0150"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video S8. Examples of Cell Division of a Control Cell or a Vimentin-Depleted Cell, Related to Figure 6B</div> <p>Frames were acquired every 2 min. DNA (red); F-actin (cyan); z-projections are diaplayed. Scale bar, 20 μm.</p>  <div class="media" title="media"><a href="mmc9.mp4">LINK</a></div> </div> <p /> <p id="p0155">Finally, we investigated whether the rounding defects observed upon vimentin depletion interfered with cell division. We did not observe any apparent defects for cells dividing in a dish (<a href="#fig6">Figures 6</a>C, 6D, and <a href="#mmc1">S5</a>H; "no gel" conditions). However, mitotic rounding is particularly crucial for cells dividing in confinement, where external pressure interferes with spindle assembly and chromosome segregation if rounding is defective (<a href="#bib32">Lancaster et al., 2013</a>). We thus confined the cells using 5 kPa polyacrylamide gels mimicking the mechanical environment of a tissue (<a href="#fig6">Figure 6</a>C). This range of stiffness allowed for normal rounding and limited mitotic defects in control cells (<a href="#fig6">Figures 6</a>D and 6E), consistent with previous observations (<a href="#bib32">Lancaster et al., 2013</a>). Strikingly, confined vimentin-depleted cells displayed increased occurrences of chromosome missegregation and multipolar spindles (<a href="#fig6">Figures 6</a>C–6E). Taken together, our experiments indicate that by affecting actin cortex organization and mechanics, vimentin is involved in controlling mitotic cell rounding, essential for division accuracy in 3D confinement.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="p0160">To systematically investigate the regulation of cortical actin organization in mitosis, we established a method to identify proteins that bind filamentous actin in interphase and mitotic cells. Of note, in our protocol, F-actin is first depolymerized for extract preparation and then re-polymerized for recovery of the F-actin binding fraction on phalloidin-coated beads (<a href="#fig1">Figure 1</a>). As a result, the organization of actin filaments on the beads is unlikely to recapitulate actin network organization in the original, spread or rounded cells. Thus, while our assay recovers proteins present and able to bind F-actin in the specific cell-cycle phases investigated, differences in F-actin binding due to the actual organization of cellular networks might be missed. Importantly, our assay cannot disentangle differences in actin binding from differences in protein abundance in the soluble extracts, though comparing differences in protein levels and FAI abundance for a subset of proteins did not reveal any clear correlation (<a href="#fig2">Figure 2</a>). Nonetheless, our FAI analysis identified a list of F-ABP candidates for the regulation of mitotic cortex organization (<a href="#tbl1">Table 1</a>).</p> <p id="p0165">We then performed a quantitative super-resolution screen to test the effects of the candidate regulators on cortex architecture (<a href="#fig3">Figure 3</a>). Strikingly, we found that the depletion of the intermediate filament protein vimentin and the actin-vimentin linker plectin increase mitotic cortex thickness (<a href="#fig3">Figure 3</a>B). This was surprising, as some studies had suggested that the vimentin network is disassembled in mitosis (<a href="#bib10">Chou et al., 1989</a>, <a href="#bib26">Inagaki et al., 1987</a>, <a href="#bib53">Takai et al., 1996</a>). In the cells used in our study, we observed a clear mitotic vimentin layer directly underneath the actin cortex (<a href="#fig4">Figures 4</a>A and 4B), in agreement with early observations suggesting a cortical vimentin layer in mitosis (<a href="#bib5">Celis et al., 1983</a>), and a recent study showing that vimentin and actin interact in mitosis (<a href="#bib17">Duarte et al., 2019</a>). We further show that the formation of a sub-cortical vimentin layer in mitosis depends on the vimentin-actin linker plectin (<a href="#fig4">Figures 4</a>C and <a href="#mmc1">S4</a>E). Mitosis-specific phosphorylation of plectin has been shown to reduce its affinity for vimentin, with a large proportion of cellular plectin becoming soluble (<a href="#bib19">Foisner et al., 1996</a>). Our immunostainings may indicate a large cytoplasmic pool (<a href="#mmc1">Figure S4</a>E), which would imply that plectin-driven recruitment to the cortex in mitosis relies on only a portion of cellular plectin.</p> <p id="p0170">Our screen identified a number of regulators for which knock down led to actin cortex thickening. In a previous study, we had shown that the actin cortex becomes thinner in mitosis and that this thinning is important for increasing cortical tension (<a href="#bib11">Chugh et al., 2017</a>). Thus, our assay identifies potential regulators of mitotic cortex mechanics. Consistently, the increase in cortical thickness upon vimentin and plectin depletion correlated with a decrease in cortical tension (<a href="#fig4">Figure 4</a>D). Interestingly, a previous study of vimentin contribution to cell mechanics had shown that vimentin does not affect cortical stiffness in spread interphase cells (<a href="#bib22">Guo et al., 2013</a>); this suggests that vimentin cortical localization and function in cortex mechanics could be specific to mitosis. We further found that impairing the formation of the sub-cortical vimentin layer using phosphomimetic vimentin mutants also led to increased cortical thickness and decreased cortical tension (<a href="#fig5">Figures 5</a>C and 5D). Vimentin may control mitotic actin biochemically (<a href="#bib25">Hubert et al., 2011</a>). However, our ablation experiments suggest that the effect of the sub-cortical vimentin layer on the cortical actin network could be at least partly mechanical, with the vimentin network physically resisting the growth of the actin cortex toward the cytoplasm. It will be interesting to investigate how exactly vimentin and actin interact, biochemically and mechanically, at the cortex.</p> <p id="p0175">Our study identified vimentin as a key regulator of mitotic cortex tension. High cortical tension in mitosis is thought to be instrumental to support a rounded cell shape, generating space for accurate spindle positioning and chromosome segregation (<a href="#bib47">Ramkumar and Baum, 2016</a>). This is particularly important in confinement, when cells must push against surrounding structures in order to round up (<a href="#bib32">Lancaster et al., 2013</a>). Consistent with this, we observed that vimentin-depleted cells displayed rounding defects (<a href="#fig6">Figures 6</a>A and 6B) and chromosome segregation defects when placed in 3D confinement (<a href="#fig6">Figures 6</a>C–6E). Interestingly, vimentin overexpression is a hallmark of various cancers (<a href="#bib23">Heerboth et al., 2015</a>, <a href="#bib50">Satelli and Li, 2011</a>). Vimentin function in cancer has so far been associated with epithelial-to-mesenchymal transitions and cell migration (<a href="#bib24">Huber et al., 2015</a>, <a href="#bib33">Leduc and Etienne-Manneville, 2017</a>, <a href="#bib41">Mendez et al., 2010</a>). Our data suggest that high vimentin levels might also facilitate cell division in strongly confined and stiff environments, such as those experienced by proliferating cancer cells (<a href="#bib34">Levental et al., 2009</a>).</p> <p id="p0180">Taken together, our unbiased approach reveals a key function for vimentin in controlling cortex architecture and mechanics in mitosis. It is also possible that vimentin contributes to mitotic rounding by regulating focal adhesions (<a href="#bib3">Burgstaller et al., 2010</a>, <a href="#bib41">Mendez et al., 2010</a>) or by affecting other mechanical properties of the actin cortex, such as viscosity or turnover. It will be exciting to investigate how the mechanical and biochemical integration of actin networks with intermediate filaments, and possibly other intracellular structures, controls actin organization during mitosis and cell morphogenetic events in general.</p> </div> <div class="star★methods" title="sec"> <div class="title" tagx="title" title="title">
STAR★Methods</div> <div class="keyresourcestable" title="sec"> <div class="title" tagx="title" title="title">
Key Resources Table</div> <p id="p0190"> <div class="table-wrap_UNKNOWN" position="float" id="undtbl1"> </div></p><table frame="hsides" rules="groups"> <thead> <tr> <th>REAGENT or RESOURCE</th> <th>SOURCE</th> <th>IDENTIFIER</th> </tr> </thead> <tbody> <tr> <td colspan="3"><b>Antibodies and Dyes</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>mouse anti-cyclin B</td> <td>Santa Cruz Biotechnology</td> <td>Santa Cruz Biotechnology sc-245; WB 1/1000; RRID:<a href="nif-antibody:AB_627338">AB_627338</a></td> </tr> <tr> <td>mouse anti-β-actin</td> <td>Santa Cruz Biotechnology</td> <td>Santa Cruz Biotechnology sc-47778; WB 1/1000; RRID:<a href="nif-antibody:AB_2714189">AB_2714189</a></td> </tr> <tr> <td>mouse anti-GAPDH</td> <td>Abcam</td> <td>Abcam ab8245; WB 1/4000; RRID:<a href="nif-antibody:AB_2107448">AB_2107448</a></td> </tr> <tr> <td>rabbit phospho-Histone H3 S10</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 9701; WB 1/1000; RRID:<a href="nif-antibody:AB_331535">AB_331535</a></td> </tr> <tr> <td>mouse anti-MRLC pS19</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 3675; WB 1/1000; RRID:<a href="nif-antibody:AB_2250969">AB_2250969</a></td> </tr> <tr> <td>rabbit anti-MRLC</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 3672; WB 1/1000; RRID:<a href="nif-antibody:AB_10692513">AB_10692513</a></td> </tr> <tr> <td>rabbit anti-vimentin</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 3932; WB 1/1000 – IF 1/100; RRID:<a href="nif-antibody:AB_2288553">AB_2288553</a></td> </tr> <tr> <td>mouse anti-α-tubulin</td> <td>ThermoFisher Scientific</td> <td>ThermoFisher Scientific DM1A; WB 1/1000; RRID:<a href="nif-antibody:AB_1954824">AB_1954824</a></td> </tr> <tr> <td>rabbit-anti phospho (Ser)-CDK1 substrate</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 9477; WB 1/1000; RRID:<a href="nif-antibody:AB_2714143">AB_2714143</a></td> </tr> <tr> <td>rat anti-GFP</td> <td>Chromoteck</td> <td>Chromotek; WB 1/1000; RRID:<a href="nif-antibody:AB_10773374">AB_10773374</a></td> </tr> <tr> <td>rabbit anti-VASP</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 3132S; WB 1/1000; RRID:<a href="nif-antibody:AB_2213393">AB_2213393</a></td> </tr> <tr> <td>rabbit anti-MENA</td> <td>Santa Cruz Biotechnology</td> <td>Santa Cruz Biotechnology sc-135988; WB 1/1000; RRID:<a href="nif-antibody:AB_2098633">AB_2098633</a></td> </tr> <tr> <td>mouse anti-MYH9</td> <td>Abnova</td> <td>Abnova; WB 1/1000</td> </tr> <tr> <td>mouse anti-caldesmon</td> <td>Abcam</td> <td>Abcam ab183146; WB 1/1000</td> </tr> <tr> <td>mouse anti-fascin</td> <td>Santa Cruz Biotechnology</td> <td>Santa Cruz Biotechnology sc-21743; WB 1/1000; RRID:<a href="nif-antibody:AB_627580">AB_627580</a></td> </tr> <tr> <td>rabbit anti-filamin</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 4762S; WB 1/1000; RRID:<a href="nif-antibody:AB_2106408">AB_2106408</a></td> </tr> <tr> <td>rabbit anti-cofilin</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech 3318S; WB 1/1000; RRID:<a href="nif-antibody:AB_2080595">AB_2080595</a></td> </tr> <tr> <td>Rabbit anti Phospho-Vimentin (Ser83)</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech #12569; RRID:<a href="nif-antibody:AB_2797957">AB_2797957</a></td> </tr> <tr> <td>Rabbit anti Phospho-Vimentin (Ser56)</td> <td>Thermo Fisher Scientific</td> <td>Thermo Fisher Scientific<br /><br />711212; RRID:<a href="nif-antibody:AB_2633001">AB_2633001</a></td> </tr> <tr> <td>Rabbit anti Vimentin conjugated with Alexa Fluor 647</td> <td>Cell Signaling Tech</td> <td>Cell Signaling Tech #5741; RRID:<a href="nif-antibody:AB_10695459">AB_10695459</a></td> </tr> <tr> <td>Goat anti-WDR1</td> <td>Santa Cruz Biotechnology</td> <td>Santa Cruz Biotechnology sc-160907; WB 1/500; RRID:<a href="nif-antibody:AB_2215093">AB_2215093</a></td> </tr> <tr> <td>rabbit anti-plectin</td> <td>Thermo Fisher Scientific</td> <td>Thermo Fisher Scientific<br /><br />#PA556292 IF 1/100; RRID:<a href="nif-antibody:AB_2645646">AB_2645646</a></td> </tr> <tr> <td>mouse anti-α-tubulin</td> <td>Sigma</td> <td>Sigma T5168; IF 1/2000; RRID:<a href="nif-antibody:AB_477579">AB_477579</a></td> </tr> <tr> <td>Phalloidin conjugated with Alexa Fluor 568</td> <td>Life Technology</td> <td>Life Technology A12380</td> </tr> <tr> <td>Phalloidin conjugated with Alexa Fluor 647</td> <td>Life Technology</td> <td>Life Technology A22287</td> </tr> <tr> <td>DAPI</td> <td>Thermo Fisher Scientific</td> <td>Molecular Probes D1306</td> </tr> <tr> <td>Alexa Fluor 680</td> <td>Thermo Fisher Scientific</td> <td>Life Technology A21058</td> </tr> <tr> <td>Alexa Fluor 790</td> <td>Thermo Fisher Scientific</td> <td>Life Technology A11369</td> </tr> <tr> <td>Cell Mask membrane stain</td> <td>Thermo Fisher Scientific</td> <td>Invitrogen C10046</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>RNAi max lipofectamine</td> <td>Thermo Fisher Scientific</td> <td>Thermo Fisher Scientific<br /><br />13778075</td> </tr> <tr> <td>Fugene HD</td> <td>Promega</td> <td>Promega E2311</td> </tr> <tr> <td>Thymidine</td> <td>Sigma</td> <td>Sigma T1895</td> </tr> <tr> <td>RO3306</td> <td>Merck Millipore</td> <td>Calbiochem 217699</td> </tr> <tr> <td>BI 2536</td> <td>Santa Cruz Biotechnology</td> <td>Santa Cruz Biotechnology sc-364431</td> </tr> <tr> <td>Y27632</td> <td>Tocris</td> <td>Tocris 1254</td> </tr> <tr> <td>Dextran</td> <td>Thermo Fisher Scientific</td> <td>Thermo Fisher Scientific<br /><br />D22914</td> </tr> <tr> <td>Non conjugated Phalloidin</td> <td>Sigma</td> <td>Sigma – P2141</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Experimental Models: Cell Lines</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>HeLa Kyoto cells</td> <td>Research Institute of Molecular Pathology (Vienna, Austria)</td> <td>N/A</td> </tr> <tr> <td>Hela Kyoto cells - H2B-mCherry</td> <td><a href="#bib52">Su et al., 2011</a></td> <td>N/A</td> </tr> <tr> <td>Hela Kyoto cells - H2B-mCherry - GFP-Lifeact</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td>Hela Kyoto cells - GFP-vimentin WT and mutants cell lines</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Oligonucleotides</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>SiRNA used in this study</td> <td>This paper</td> <td><a href="#mmc1">Table S2</a></td> </tr> <tr> <td>Primers used in this study</td> <td>This paper</td> <td><a href="#mmc1">Table S2</a></td> </tr> <tr> <td>point mutations in the human vimentin cDNA 5′- gc agc ctc tac gcc tcg GCC ccg ggc ggc gtg tat gcc a -3′</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td>point mutations in the human vimentin cDNA 5′- t ggc ata cac gcc gcc cgg GGC cga ggc gta gag gct gc - 3′</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td>point mutations in the human vimentin cDNA 5′- gc agc ctc tac gcc tcg GAA ccg ggc ggc gtg tat gcc a -3′</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td>point mutations in the human vimentin cDNA 5′- t ggc ata cac gcc gcc cgg TTC cga ggc gta gag gct gc - 3′</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td>point mutations in the human vimentin cDNA 5′- g gtg cgg ctc ctg cag gac GAG gtg gac ttc tcg ctg gcc -3′</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td>point mutations in the human vimentin cDNA 5′- ggc cag cga gaa gtc cac CTC gtc ctg cag gag ccg cac c - 3′</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Recombinant DNA</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>Lifeact-GFP plasmid</td> <td>Matthieu Piel (Institut Curie, France)</td> <td>N/A</td> </tr> <tr> <td>pIRESpuro3 Flag–AcGFP- vimentin mutant plasmids</td> <td>This paper</td> <td>N/A</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Software and Algorithms</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>Fiji</td> <td>N/A</td> <td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td> </tr> <tr> <td>Matlab</td> <td>Mathworks</td> <td>MathWorks</td> </tr> <tr> <td>Prism</td> <td>GraphPad</td> <td>GraphPad</td> </tr> <tr> <td>Icy</td> <td>N/A</td> <td><a href="http://icy.bioimageanalysis.org/">http://icy.bioimageanalysis.org/</a></td> </tr> </tbody> </table>  <p /> </div> <div class="leadcontactandmaterialsavailability" title="sec"> <div class="title" tagx="title" title="title">
Lead Contact and Materials Availability</div> <p id="p0195">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Ewa Paluch (<a href="mailto:ekp25@cam.ac.uk">ekp25@cam.ac.uk</a>).</p> <p id="p0200">All unique/stable reagents generated in this study are available from the Lead Contact without restriction.</p> </div> <div class="experimentalmodelandsubjectdetails" title="sec"> <div class="title" tagx="title" title="title">
Experimental Model and Subject Details</div> <div class="celllinesandculture" title="sec"> <div class="title" tagx="title" title="title">
Cell Lines and Culture</div> <p id="p0205">All cell lines were grown at 37°C with 5% CO<sub>2</sub> in DMEM GlutaMAX, 4.5 g/l glucose (Gibco, Invitrogen/ Life Technologies) supplemented with 10% fetal bovine serum (Sigma), 1% penicillin-streptomycin (Gibco, Invitrogen/ Life Technologies) and regularly tested for mycoplasma. HeLa Kyoto cells were obtained from the Research Institute of Molecular Pathology (Vienna, Austria). For the stable Lifeact cell line, a Lifeact-GFP plasmid kindly provided by Matthieu Piel (Institut Curie) was transfected into HeLa Kyoto cells stably expressing H2B-mCherry (kindly provided by Kuan-Chung Su (<a href="#bib52">Su et al., 2011</a>)) using FuGENE 6 (Roche) according to the manufacturer’s instructions. HeLa Kyoto cells stably expressing H2B-mCherry and GFP-Lifeact were grown in DMEM supplemented with 500 μg/ml G418 (Invitrogen), and 0.35 μg/ml puromycine to maintain selection. For the stable GFP-vimentin cell lines, GFP-vimentin mutant plasmids were transfected into HeLa Kyoto cells using FuGENE HD (Roche) according to the manufacturer’s instructions. Hela Kyoto cells stably expressing GFP-Vimentin were grown in DMEM supplemented with 0.35 μg/ml puromycine to maintain selection. Cell lines have not been authenticated.</p> </div> </div> <div class="methoddetails" title="sec"> <div class="title" tagx="title" title="title">
Method Details</div> <div class="sirnatransfection" title="sec"> <div class="title" tagx="title" title="title">
siRNA Transfection</div> <p id="p0210">For siRNA transfection, RNAi max lipofectamine (Invitrogen) reverse transfection was performed according to the manufacturer’s instructions. Final concentrations of 30 nM siRNA were used for all the experiments. SiRNAs used in this study are listed in <a href="#mmc1">Table S2</a>.</p> </div> <div class="preparationofgfp-vimentinmutants" title="sec"> <div class="title" tagx="title" title="title">
Preparation of GFP-Vimentin Mutants</div> <p id="p0215">All vimentin mutants (S56A, S56E, S83E) were generated using the QuikChange methodology (Stratagene, La Jolla, CA). The following oligonucleotides were used to create the different point mutations in the human vimentin cDNA (capital letters indicate the codon mutated). 56A: Forward: 5’- gc agc ctc tac gcc tcg GCC ccg ggc ggc gtg tat gcc a -3’; Reverse: 5’- t ggc ata cac gcc gcc cgg GGC cga ggc gta gag gct gc - 3’. 56E: Forward: 5’- gc agc ctc tac gcc tcg GAA ccg ggc ggc gtg tat gcc a -3’; Reverse: 5’- t ggc ata cac gcc gcc cgg TTC cga ggc gta gag gct gc - 3’. 83E: Forward: 5’- g gtg cgg ctc ctg cag gac GAG gtg gac ttc tcg ctg gcc -3’; Reverse: 5’- ggc cag cga gaa gtc cac CTC gtc ctg cag gag ccg cac c - 3’. The mutated cDNAs were fully sequenced to confirm the sole presence of intended mutations and inserted into pIRESpuro3 Flag–AcGFP-tagged plasmid.</p> </div> <div class="cellsynchronizationformassspectrometry" title="sec"> <div class="title" tagx="title" title="title">
Cell Synchronization for Mass Spectrometry</div> <p id="p0220">To prepare cell extracts from specific phases of the cell cycle, HeLa cells were first synchronized in early S phase with thymidine (Sigma) at 2 mM for 22 hours. For synchronization in G2 phase, cells were released for 6hrs and treated with CDK1 inhibitor RO3306 (Calbiochem) at 10 μM for 3 hours. To synchronize cells in metaphase, the cells were released from a single thymidine block for 6hrs, then treated with nocodazole 50 μg/ml for 4 h, and then with 10 μM of MG132 for 2 h as previously described (<a href="#bib44">Petronczki et al., 2007</a>). In all cases, cells were washed twice with DMEM between each treatment and wash with PBS before processed for experiment.</p> </div> <div class="extractspreparation" title="sec"> <div class="title" tagx="title" title="title">
Extracts Preparation</div> <p id="p0225">Around 20 millions of cells were first mechanically lysed for 30min by vortexing and pipetting-up-and-down in 300 μL of buffer containing 10 mM HEPES pH 7.5, 83 mM KCl, 17 mM NaCl and 0.1% Triton X-100 supplemented with proteases and phosphatases inhibitors: Microcystin, 10 μg/ml of Leupeptin, Pepstatin and 10 mM of β-glycerophosphate. To solubilize F-actin network, 200 nM of LatrunculinA was added to the lysis. After centrifugation at 28 000 rpm with TLA-120.1 rotor (Beckman Coulter) for 20 min, pellets were resuspended in 300 μL of same lysis buffer without Triton X-100 and mechanically lysed again for 15 min. After a second centrifugation, pellets were sonicated, both supernatants were pooled and frozen in liquid nitrogen. Latrunculin A was present throughout extract preparation, limiting the possibility of actin loss through repolymerization. Lysis conditions were optimized using cells enriched in mitosis (overnight nocodazole treatment). Protein concentrations were determined using Bradford protein assay reagent (Biorad). For western blotting, supernatant and pellets from the preparation were then resuspended in 5X loading buffer (312.5 mM Tris-HCl pH 6.8, 50% glycerol, 5% SDS, 5% β-mercapto ethanol, 0.25% Bromophenol blue), and boiled for 5 min. Pellets amounts loaded for immunoblotting were adjusted to be the same amount as the supernatant.</p> </div> <div class="phalloidinpull-downassay" title="sec"> <div class="title" tagx="title" title="title">
Phalloidin Pull-Down Assay</div> <p id="p0230">Extracts were cleared by centrifugation at 30,000 g for 5 min and 700 μg of each extract were used per condition. We verified that no significant amount of actin was lost from the extract at this step. Pull-down assay was performed using a phalloidin affinity matrix (phalloidin coupled through a flexible polyethylene glycol linker to superparamagnetic beads) (<a href="#bib49">Samwer et al., 2013</a>). Control and Phalloidin matrix were pre-washed 2 times with 10 mM HEPES pH 7.5, 83 mM KCl, 17 mM NaCl and 0.05% Triton-100. Extracts and beads were incubated for 1 hours at room temperature with 2 mM MgCl<sub>2</sub> and 1 mM ATP to induce actin polymerization (<a href="#bib49">Samwer et al., 2013</a>). Of note, the concentration of phalloidin on the beads was sufficient to effectively induce actin polymerization even though a small amount of Latrunculin was present in the extract (see section above). After the incubation, beads were washed 2 or 3 times for 2 min with wash buffer (10 mM HEPES pH 7.5, 83 mM KCl, 17 mM NaCl and 0.05% Triton-100) and eluted using 25 μl of loading buffer (125 mM Tris-HCl pH 6.8, 3% SDS, 50 mM DTT, 1 M sucrose, 0.1% bromophenol blue). Samples were boiled for 3 min, separated by SDS-PAGE (NuPage Invitrogen) and stained by colloidal Coomassie (<a href="#bib43">Neuhoff et al., 1988</a>, <a href="#bib49">Samwer et al., 2013</a>).</p> </div> <div class="massspectrometryanddataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Mass Spectrometry and Data Analysis</div> <p id="p0235">Coomassie-stained gel bands were excised and subjected to in-gel trypsin digestion, as described previously in (<a href="#bib20">Galan et al., 2014</a>). The resulting peptides were extracted and subjected to capillary LC-MS/MS using a high-resolution Orbitrap Fusion Tribrid (Thermo Scientific, San Jose, CA). Raw files were searched with MaxQuant (<a href="#bib15">Cox and Mann, 2008</a>) software version 1.3.0.3 and searched against the Uniprot Human database (<a href="http://www.uniprot.org/">http://www.uniprot.org/</a>) release 2016_02 (17-Feb-2016) including the reversed database with 74508 entries. General false discovery rate (FDR) for peptides was at 1.0% with decoy removal. Searches were performed at a 25-ppm precursor ion tolerance and 1.0 Da fragment ion tolerance, assuming full tryptic digestion with up to three missed cleavages. Correct peptide identifications were distinguished from incorrect using the target-decoy approach, coupled with linear discriminant analysis as described previously. Once peptides were filtered to an initial FDR of 1%, peptides were assembled into proteins and further filtered to a final protein-level FDR of 1%. All proteins of the same family were grouped and isoforms of the same protein were considered as one. Finally, contaminants (e.g. keratins) and proteins more represented in control beads eluates were removed (see <a href="#mmc10">Table S1</a>). To estimate the abundance of proteins within eluates, we rely on total spectral counts and a modified Protein Abundance Index (PAI) (<a href="#bib1">Biro et al., 2013</a>, <a href="#bib2">Bovellan et al., 2014</a>, <a href="#bib48">Rappsilber et al., 2002</a>). PAI was calculated based on total spectral counts as follows: PAI = total spectral count/MW, where MW corresponds to the protein molecular weight, which was used to adjust for differences between proteins in the number of observable peptides. Mass spectrometry data were also analyzed using Gene ontology enrichment analysis and panther protein classification system (<a href="#bib42">Mi et al., 2017</a>)</p> </div> <div class="immunoblotting" title="sec"> <div class="title" tagx="title" title="title">
Immunoblotting</div> <p id="p0240">Protein samples were subjected to SDS-PAGE and transferred onto polyvinylidene difluoride membranes (Millipore). After blocking with PBS containing 0.1% Tween-20 and 5% milk, membranes were incubated overnight with primary antibodies, washed in PBS containing 0.1% Tween-20 and then incubated 1 h with HRP-conjugated secondary antibodies (GE Healthcare; 1/5000). After washes with PBS containing 0.1% Tween-20, immunodectection was performed with Amersham ECL Western Blotting Detection Reagent and Amersham Hyperfilm (GE Healthcare). Membrane stripping was performed using stripping buffer containing glycine, 0.01% SDS and 0.01% Tween pH 2.2. For fluorescent immunoblotting, nitrocellulose membranes (Thermo Scientific) and secondary antibodies conjugated to IRDye 800 (LI-COR) or Alexa Fluor 680 and Alexa Fluor 790 (Life Technology) were used. Immunodetection was performed using an Odyssey infrared imaging system (LI-COR). The following primary antibodies were used: mouse anti-cyclin B (Santa Cruz Biotechnology sc-245; 1/1000); mouse anti-β-actin (Santa Cruz Biotechnology sc-47778; 1/1000); mouse anti-GAPDH (Abcam ab8245; 1/4000); phosphoHistone H3 S10 (Cell Signaling Tech 9701; 1/1000); mouse anti-MRLC pS19 (Cell Signaling Tech 3675; 1/1000); rabbit anti-MRLC (Cell Signaling Tech 3672; 1/1000); rabbit anti-vimentin (Cell Signaling Tech 3932; 1/1000); mouse anti-plectin (Thermo Fisher Scientific PA556292 1/1000); mouse anti-α-tubulin (ThermoFisher Scientific DM1A; 1/1000); rabbit-anti phospho (Ser) CDK1 substrate (Cell Signaling Tech 9477; 1/1000); rat anti-GFP (Chromotek; 1/1000); rabbit anti-VASP (Cell Signaling Tech 3132S; 1/1000); rabbit anti-MENA (Santa Cruz Biotechnology sc-135988; 1/1000); mouse anti-MYH9 (Abnova; 1/1000) ; mouse anti-caldesmon (Abcam ab183146; 1/1000); mouse anti-fascin (Santa Cruz Biotechnology sc-21743; 1/1000), rabbit anti-filamin (Cell Signaling Tech 4762S; 1/1000), rabbit anti-cofilin (Cell Signaling Tech 3318S; 1/1000) and WDR1 (Santa Cruz Biotechnology sc-135988; 1/500).</p> </div> <div class="immunofluorescence" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence</div> <p id="p0245">Cells were seeded in labteck chambers (Thermo Scientific) or on coverslips (No 1.5, 170 nm). For CDK1 or PLK1 inhibition experiments, STLC treated cells were treated with DMSO, RO3306 (Calbiochem) at 10 μM for 20 min or BI2536 (Santa Cruz Biotechnology) at 100 nM for 2 h. Cells were fixed in 4% PFA in PBS containing 0.01 Triton X-100 (PBST) for 15 min at room temperature. After washes in PBST, cells were permeabilized in PBS 0.2% Triton X-100 for 10 min and incubated with primary antibodies for 1h at room temperature. After washes in PBST, cells were then incubated with secondary antibodies and/or phalloidin conjugated with Alexa fluorescent dyes. The following primary antibodies were used: rabbit anti-vimentin (Cell signaling Tech 3932; 1/100); mouse anti-plectin (Thermo Fisher Scientific #PA556292 1/100); Rabbit anti Phospho-vimentin (Ser83) (Cell Signaling Tech 2569; 1/100); Rabbit anti Phospho-vimentin (Ser56) (Thermo Fisher Scientific 711212; 1/100); mouse anti-α-tubulin (Sigma T6024; 1/2000); rabbit anti-vimentin conjugated with Alexa Fluor 647 (Cell Signaling Tech 5741; 1/100). Fluorescent dyes used were Phalloidin conjugated with Alexa Fluor 488 (Life Technology A12379); Phalloidin conjugated with Alexa Fluor 568 (Life Technology A12380); Phalloidin conjugated with Alexa Fluor 647 (Life Technology A22287) and DAPI (Molecular Probes) to stain DNA. Secondary antibodies conjugated to Alexa Fluor 488 or 568 (Life Technology; 1/500) were used for immunofluorescence microscopy detection. Coverslips were mounted using Prolong Gold Antifade (Invitrogen). For cortex localization analysis, a single plane in the middle of the cell was used for the quantification of mean cortex intensity. Cell cortex intensity was calculated using Fiji software as the mean intensity of 0.4 μm<sup>2</sup> circles placed all along the cell cortex, normalized by the mean intensity of the entire cell.</p> <p id="p0250">For one-color STORM experiments (measurements of cortical thickness screen), cells were synchronized using thymidine (Sigma) at 2 mM for 22 h. Then, cells were rinsed twice and fresh media was added for the release for 9 h. For the one-color STORM experiments on GFP-vimentin mutants, cells were pre-treated with STLC for 6h to enrich for mitotic cells. Cells were then permeabilized and fixed at the same time in 4% PFA, 0.2% Triton X-100 in Cytoskeleton buffer pH to 7.4 (10 mM MES, 138 mM KCl, 3 mM MgCl<sub>2</sub>, 2 mM EGTA) supplemented with sucrose (Cytoskeleton buffer with sucrose; CBS) for 6 min and then 14 min in 4% PFA in CBS. After 3 washes in CBS, cells were incubated 20 min with phalloidin conjugated with Alexa-647 (Life Technology A22287). Coverslips were mounted using Vectashield (Vectorlabs).</p> <p id="p0255">For two-color STORM experiments, cells were seeded onto glass-bottom petri dishes (Ibidi) and fixed, permeabilized and washed as described above for one-color STORM experiments. Cells were then incubated with vimentin antibody conjugated with Alexa 647 (Cell signaling Tech 9856; 1/1000) and phalloidin conjugated with Alexa 568 (Life Technology A12380) for 30 min. For imaging, samples were kept in STORM buffer (Tris base 50 mM, NaCl 100 mM, MEA 1M, supplemented with Glucose, Catalase and Oxidase).</p> </div> <div class="stormimaging,rendering,andanalysis" title="sec"> <div class="title" tagx="title" title="title">
STORM Imaging, Rendering, and Analysis</div> <p id="p0260">One-color STORM imaging: For the thickness screen, raw images were acquired on an N-STORM microscope (Nikon) using a x100 objective (1.49 numerical aperture CFI Apo TIRF). DAPI staining was used to select cells in metaphase. For actin cortex thickness measurements in cells expressing GFP-vimentin and GFP-vimentin mutant constructs, raw images were acquired on a Zeiss Elyra microscope in STORM mode using a x100 objective (1.46 numerical aperture alpha Plan-Apochromat). Prometaphase cells were selected based on DNA appearance in bright field microscopy. Coordinates of single molecules in the raw STORM images were detected using the Nikon microscope software (thickness screen) or the Zen software (GFP-vimentin mutants). Drift, chromatic aberration correction and super-resolution rendering were performed using a custom-written script in MATLAB (MathWorks).</p> <p id="p0265">Two-color STORM: raw images were acquired on a Zeiss Elyra microscope in STORM mode using a x100 (1.46 numerical aperture alpha Plan-Apochromat) objective. To correct for chromatic aberration and drift during the acquisition, multi-color fluorescent beads (TetraSpeck 0.1 μm diamater, Invitrogen T7279) were added to the samples. Coordinates of single molecules in raw STORM images were detected using the open source software ThunderSTORM (<a href="#bib62">Ovesný et al., 2014</a>). Drift, chromatic aberration correction and super-resolution rendering were done using a custom-written script in MATLAB (MathWorks).</p> </div> <div class="stormimageanalysis" title="sec"> <div class="title" tagx="title" title="title">
STORM Image Analysis</div> <p id="p0270">The cell cortex in each rendered STORM image was detected semi-automatically. First, portions of cell periphery where the cortex is well defined, i.e., without too many microvilli or aggregation of fiducial beads, were manually selected using a custom-written GUI in MATLAB. The cortex was then automatically detected by finding the maximum intensity peak of transverse profiles across the cell contour at every pixel along the regions selected. The segmented cortex was straightened up using a spline and individual intensity linescans across the cortex were obtained.</p> <p id="p0275">In one-color STORM experiments, the full width at half maximum (FWHM) of cortex intensity profiles was used to assess cortical thickness. For two-color STORM experiments, the intensity peaks for each protein pair (phalloidin in two colors, or phalloidin and vimentin) were localized and peak-to-peak distance was measured using a custom-written MATLAB script.</p> </div> <div class="cellconfinement" title="sec"> <div class="title" tagx="title" title="title">
Cell Confinement</div> <p id="p0280">Confinement experiments were done as previously described (<a href="#bib32">Lancaster et al., 2013</a>) with small alterations. Briefly, polyacrylamide gels (5 kPa rigidity) were polymerized onto 19 mm glass coverslips and incubated in imaging media at 37°C. Gel-covered coverslips were then gently positioned on top of the siRNA treated cells seeded on glass bottom petri dishes (Ibidi), and pressure to confine the cells was exerted using a PDMS column. Time lapse images were taken every 5 min with 4 z-stack of 3 μm step with a Zeiss Observer Z1 microscope widefield timelapse microscope equipped with a 12bit Hamamatsu Orca ER camera and a 60X objective.</p> </div> <div class="livecellimagingandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Live Cell Imaging and Analysis</div> <p id="p0285">All cells were imaged in in phenol-red free and CO<sub>2</sub>-independent medium (Invitrogen) at 37°C. Hela Kyoto cells stably expressing H2B-mCherry and Lifeact-GFP were imaged using a Nikon TE2000 microscope equipped with a Plan Fluor x60/1.4 DIC H objective (Nikon), a PerkinElmer ERS Spinning disk system, a Digital CCD C4742-80-12AG camera (Hamamatsu), and controlled by Volocity 6.0.1 software. For vimentin dynamics during the cell cycle, CDK1 inhibitor time lapses, and cell rounding experiments, stacks of 7 z-planes 3 μm apart every 2 min. Cell roundness (4<sup>∗</sup>area/pi<sup>∗</sup>sqr(major axis)) was calculated using Fiji software. Hela Kyoto cells stably expressing the different GFP-vimentin mutants were imaged with an Olympus FluoView FV1200 Confocal Laser Scanning Microscope using a 60X objective (UPLSAPO60XS) and the FV1200 software. Cortex localization analysis of GFP-vimentin mutants were done the same way as explained in the Immunofluorescence section.</p> </div> <div class="laserablationexperiments" title="sec"> <div class="title" tagx="title" title="title">
Laser Ablation Experiments</div> <p id="p0290">Ablation experiments were performed on an Olympus FluoView FV1200 Confocal Laser Scanning Microscope using a 60X objective (UPLSAPO60XS) as previously described (<a href="#bib54">Tinevez et al., 2009</a>). A 405 nm laser (PicoQuant) delivering picosecond pulses with nominal power 3 mW was used with the maximal speed setting corresponding to ∼3 s between time points. Pulses were delivered for ∼2 s in a spiraling movement within a circular region with a diameter of ∼400 nm focused on the sub-cortical vimentin layer. To confirm that our ablation settings did not result in puncture of the cell membrane, CellMaskTM Deep Red Plasma membrane stain (Thermo Fisher Scientific) or 10 KDa non-cell permeable dextran conjugated to Alexa 647 was added to the media prior to the experiment. For the ROCK inhibitor experiment, cells were pre-treated with 10 μM Y27632 for 15 min. For vimentin ablation, the laser was focused on sub-cortical the vimentin layer; areas of bright and low intensities of vimentin were chosen randomly and no difference in cell behavior was observed between the different areas.</p> <p id="p0295">To compute cell curvature (<a href="#fig5">Figure 5</a>E), the cell was segmented by adding a Gaussian filter to the image and then automatically thresholding with Otsu's method. Contour points of the cell from the binary image were extracted (∼1000 points/cell) and for each point a circle through that point was fitted. To reduce local noise that highly affects curvature measurements, a circle to each point was fitted using the 60th point before and after the point. Extracted local curvature (1/radius of the fitted circles) was then plotted along the cell contour.</p> </div> <div class="afmmeasurements" title="sec"> <div class="title" tagx="title" title="title">
AFM Measurements</div> <p id="p0300">For cortical tension measurements, cells were transfected with siRNA 48 hours prior to the experiment. Cells were then treated with STLC 4 h before the first measurement to enrich for mitotic cells. Interphase cells were pre-treated with thymidine for 22h and detached using trypsin before the measurement. Cells were placed in glass bottom dishes (Mattek) in phenol-red free and CO<sub>2</sub>-independent medium (Invitrogen) and incubated with 0.01% CellMaskTM Deep Red Plasma membrane Stain (Thermo Fisher Scientific). Mitotic and interphase cells were then selected based on DNA appearance in bright field microscopy. Atomic force microscopy tension measurements were performed using Tipless silicon cantilevers (ARROW-TL1Au-50) with a nominal spring constant of 0.03 N/m, mounted on a JPK CellHesion module (JPK Instruments) on an IX81 inverted confocal microscope (Olympus). Sensitivity was calibrated by acquiring a force curve on a glass coverslip, and the spring constant was calibrated by the thermal noise fluctuation method. Constant height mode was selected and a Z-length parameter of 35 μm was used with a setpoint force set at 25 nN. The measurement was carried out by first lowering the tipless cantilever on the glass in an empty area next to the cell to detect the position of the substrate. The cantilever was then positioned above the cell and a compression was applied for 250 seconds. During the constant height compression, the force acting on the cantilever was recorded. After initial force relaxation, the resulting force value was used to extract surface tension.</p> <p id="p0305">The calculation of cortex tension is based on (<a href="#bib18">Fischer-Friedrich et al., 2014</a>) and described in (<a href="#bib11">Chugh et al., 2017</a>). Briefly, neglecting the angle of the cantilever with respect to the dish (∼8°) and assuming negligible adhesion between cell, dish and cantilever, the force balance at the contact point reads: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
F</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
r</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
mid</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
r</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
πr</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
mid</div> </div> </div> </div> </div> </div></div></div> <p /> <p id="p0310">where r<sub>mid</sub> is the radius of the maximum cross-sectional area of the selected cell, r<sub>c</sub> is the radius of the contact area of the cell with the cantilever and F is the force exerted by the cell on the cantilever. To avoid errors due to direct measurement of r<sub>c</sub>, the contact radius was calculated using the following formula (<a href="#bib51">Stewart et al., 2012</a>): </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
c</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
mid</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
π</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
4</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
h</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
cell</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div></div></div> <p /> <p id="p0315">where A<sub>c</sub> = π r<sub>c</sub><sup>2</sup> is the contact area between the cell and the cantilever, A<sub>mid</sub> is the cell maximum cross-sectional area and h<sub>cell</sub> is the cell height.</p> </div> <div class="scanningelectronmicroscopy(sem)" title="sec"> <div class="title" tagx="title" title="title">
Scanning Electron Microscopy (SEM)</div> <p id="p0320">Synchronized extracts were supplemented with 20 μM soluble phalloidin (Sigma), 2 mM MgCl<sub>2</sub> and 1 mM ATP to promote F-actin polymerization. The mixture (10 μl) was sandwiched between coverslips spaced with parafilm strips as previously described (<a href="#bib45">Plastino et al., 2004</a>). Samples were incubated 1 h at room temperature and then fixed with glutaraldehyde 1%. Preparation for scanning electron microscopy was performed as previously described in (<a href="#bib2">Bovellan et al., 2014</a>, <a href="#bib11">Chugh et al., 2017</a>). Briefly, after the first fixation samples were fixed a second time with 0.1% Tannic Acid for 5 min, washed 5 times with water and incubated with 0.1 % uranyl-acetate for 10 min. After 5 washes with water, samples were then dehydrated with serial ethanol dilutions, dried in a critical point dryer, coated with a thin layer of approximately 7 nm thick of Gold using a Quorum sputter coater and then viewed and images recorded using a FEI Quanta 200 FEG ESEM, operated at 5 kV in the high vacuum mode.</p> </div> <div class="rnaextractionandrt-qpcr" title="sec"> <div class="title" tagx="title" title="title">
RNA Extraction and RT-qPCR</div> <p id="p0325">Total RNA from Hela Kyoto cells was extracted using RNeasy Mini Kit (Qiagen) 48h after siRNA transfection. Total RNA was reverse-transcribed using a cDNA Reverse Transcription Kit (Applied Biosystems, Waltham, MA, United States of America) following the manufacturer’s instructions. Gene expression levels of endogenous controls GAPDH and beta-actin (ACTB) were determined using pre-validated TaqMan Gene Expression Assays (Applied Biosystems). Expression levels of queried and control genes were determined using assays designed with the Universal Probe Library from Roche. The PCR reactions were carried out on an ABI Real Time 7900HT cycler and analyzed with SDS 2.2 software. qPCR was carried out with TaqMan Gene Expression Master Mix (Applied Biosystems), and all samples were tested in duplicate. mRNA levels of queried genes were normalized to the averaged levels of GAPDH and ACTB, and relative abundance was calculated by normalizing values from cells treated with control and targeted siRNA pools by dividing by the control value. All the genes tested and the corresponding primers are listed in <a href="#mmc1">Table S2</a>.</p> </div> </div> <div class="quantificationandstatisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Quantification and Statistical Analysis</div> <p id="p0330">3D visualization videos were generated using Icy software. dSTORM sata were analyased using Matlab (MathWorks). All the other image analysis was carried out in Fiji. Graphs and statistical tests were produced in GraphPad Prism. Normally distributed data sets were analyzed using the unpaired t test, or ANOVA. Non-normal data sets were analyzed using Mann-Whitney or Kruskal-Wallis test. Details of the statistical tests used, exact value of n, definition error bars on graphs and number of experiments performed are all detailed in the legends of the figures.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data and Code Availability</div> <p id="p0335">Data and custom-written codes developed for data analysis are available upon request from the lead contact.</p> <p id="p0340">The software used for STORM rendering and analysis is described in (Truong Quang et al., submitted).</p> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biro</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romeo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kroschwald</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bovellan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boden</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tcherkezian</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">P.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charras</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">E.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell cortex composition and homeostasis resolved by integrating proteomics and quantitative imaging</span><span class="source" tagx="source" title="source">Cytoskeleton (Hoboken)</span><span class="volume" tagx="volume" title="volume">70</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">741</span><span class="lpage" tagx="lpage" title="lpage">754</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24136886">24136886</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bovellan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romeo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biro</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boden</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chugh</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yonis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaghela</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fritzsche</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moulding</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thorogate</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cellular control of cortical actin nucleation</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">24</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1628</span><span class="lpage" tagx="lpage" title="lpage">1635</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25017211">25017211</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burgstaller</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregor</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winter</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiche</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Keeping the vimentin network under control: cell-matrix adhesion-associated plectin 1f affects cell shape and polarity of fibroblasts</span><span class="source" tagx="source" title="source">Mol. Biol. Cell</span><span class="volume" tagx="volume" title="volume">21</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">3362</span><span class="lpage" tagx="lpage" title="lpage">3375</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20702585">20702585</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carreno</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouranti</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glusman</span><span class="given-names" tagx="given-names" title="given-names">E.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuller</span><span class="given-names" tagx="given-names" title="given-names">M.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Echard</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Payre</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moesin and its activating kinase Slik are required for cortical stability and microtubule organization in mitotic cells</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">180</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">739</span><span class="lpage" tagx="lpage" title="lpage">746</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18283112">18283112</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Celis</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">P.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fey</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Celis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phosphorylation of keratin and vimentin polypeptides in normal and transformed mitotic human epithelial amnion cells: behavior of keratin and vimentin filaments during mitosis</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">97</span><span class="year" tagx="year" title="year">1983</span><span class="fpage" tagx="fpage" title="fpage">1429</span><span class="lpage" tagx="lpage" title="lpage">1434</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6195164">6195164</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Champion</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linder</span><span class="given-names" tagx="given-names" title="given-names">M.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kutay</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cellular reorganization during mitotic entry</span><span class="source" tagx="source" title="source">Trends Cell Biol.</span><span class="volume" tagx="volume" title="volume">27</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">26</span><span class="lpage" tagx="lpage" title="lpage">41</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27528558">27528558</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chanet</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin 2-induced mitotic rounding enables columnar epithelial cells to interpret cortical spindle positioning cues</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">27</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">3350</span><span class="lpage" tagx="lpage" title="lpage">3358.e3</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29107549">29107549</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charras</span><span class="given-names" tagx="given-names" title="given-names">G.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchison</span><span class="given-names" tagx="given-names" title="given-names">T.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahadevan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Animal cell hydraulics</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="volume" tagx="volume" title="volume">122</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">3233</span><span class="lpage" tagx="lpage" title="lpage">3241</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19690051">19690051</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chou</span><span class="given-names" tagx="given-names" title="given-names">Y.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khuon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrmann</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nestin promotes the phosphorylation-dependent disassembly of vimentin intermediate filaments during mitosis</span><span class="source" tagx="source" title="source">Mol. Biol. Cell</span><span class="volume" tagx="volume" title="volume">14</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">1468</span><span class="lpage" tagx="lpage" title="lpage">1478</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12686602">12686602</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chou</span><span class="given-names" tagx="given-names" title="given-names">Y.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosevear</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phosphorylation and disassembly of intermediate filaments in mitotic cells</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">86</span><span class="year" tagx="year" title="year">1989</span><span class="fpage" tagx="fpage" title="fpage">1885</span><span class="lpage" tagx="lpage" title="lpage">1889</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2648386">2648386</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chugh</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">M.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassani</span><span class="given-names" tagx="given-names" title="given-names">D.A.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dierkes</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragab</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">P.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charras</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salbreux</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">E.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Actin cortex architecture regulates cell surface tension</span><span class="source" tagx="source" title="source">Nat. Cell Biol.</span><span class="volume" tagx="volume" title="volume">19</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">689</span><span class="lpage" tagx="lpage" title="lpage">697</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28530659">28530659</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciobanasu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faivre</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Clainche</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Actin dynamics associated with focal adhesions</span><span class="source" tagx="source" title="source">Int. J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">2012</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">941292</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22505938">22505938</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dierkes</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">E.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Monitoring actin cortex thickness in live cells</span><span class="source" tagx="source" title="source">Biophys. J.</span><span class="volume" tagx="volume" title="volume">105</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">570</span><span class="lpage" tagx="lpage" title="lpage">580</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23931305">23931305</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanics and regulation of cell shape during the cell cycle</span><span class="source" tagx="source" title="source">Results Probl. Cell Differ.</span><span class="volume" tagx="volume" title="volume">53</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">31</span><span class="lpage" tagx="lpage" title="lpage">73</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21630140">21630140</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</span><span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="volume" tagx="volume" title="volume">26</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">1367</span><span class="lpage" tagx="lpage" title="lpage">1372</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029910">19029910</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">W.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">H.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hara</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wongsantichon</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyama</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">R.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nédélec</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaidel-Bar</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plastin increases cortical connectivity to facilitate robust polarization and timely cytokinesis</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">216</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">1371</span><span class="lpage" tagx="lpage" title="lpage">1386</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28400443">28400443</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duarte</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viedma-Poyatos</span><span class="given-names" tagx="given-names" title="given-names">Á.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Navarro-Carrasco</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martínez</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pajares</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pérez-Sala</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vimentin filaments interact with the actin cortex in mitosis allowing normal cell division</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">10</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">4200</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31519880">31519880</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer-Friedrich</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyman</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jülicher</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helenius</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Quantification of surface tension and internal pressure generated by single mitotic cells</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="volume" tagx="volume" title="volume">4</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">6213</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25169063">25169063</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foisner</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malecz</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dressel</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiche</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">M-phase-specific phosphorylation and structural rearrangement of the cytoplasmic cross-linking protein plectin involve p34cdc2 kinase</span><span class="source" tagx="source" title="source">Mol. Biol. Cell</span><span class="volume" tagx="volume" title="volume">7</span><span class="year" tagx="year" title="year">1996</span><span class="fpage" tagx="fpage" title="fpage">273</span><span class="lpage" tagx="lpage" title="lpage">288</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8688558">8688558</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galan</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geraghty</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavoie</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanshin</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tcherkezian</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calabrese</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeschke</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">B.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ballif</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blenis</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phosphoproteomic analysis identifies the tumor suppressor PDCD4 as a RSK substrate negatively regulated by 14–3-3</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">111</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">E2918</span><span class="lpage" tagx="lpage" title="lpage">E2927</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25002506">25002506</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gavet</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pines</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Progressive activation of CyclinB1-Cdk1 coordinates entry to mitosis</span><span class="source" tagx="source" title="source">Dev. Cell</span><span class="volume" tagx="volume" title="volume">18</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">533</span><span class="lpage" tagx="lpage" title="lpage">543</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20412769">20412769</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehrlicher</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahammad</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fabich</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fredberg</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weitz</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of vimentin intermediate filaments in cortical and cytoplasmic mechanics</span><span class="source" tagx="source" title="source">Biophys. J.</span><span class="volume" tagx="volume" title="volume">105</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">1562</span><span class="lpage" tagx="lpage" title="lpage">1568</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24094397">24094397</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heerboth</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Housman</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leary</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longacre</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Byler</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapinska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willbanks</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarkar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">EMT and tumor metastasis</span><span class="source" tagx="source" title="source">Clin. Transl. Med.</span><span class="volume" tagx="volume" title="volume">4</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25852822">25852822</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boire</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López</span><span class="given-names" tagx="given-names" title="given-names">M.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koenderink</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cytoskeletal crosstalk: when three different personalities team up</span><span class="source" tagx="source" title="source">Curr. Opin. Cell Biol.</span><span class="volume" tagx="volume" title="volume">32</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">39</span><span class="lpage" tagx="lpage" title="lpage">47</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25460780">25460780</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hubert</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandekerckhove</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gettemans</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Unconventional actin conformations localize on intermediate filaments in mitosis</span><span class="source" tagx="source" title="source">Biochem. Biophys. Res. Commun.</span><span class="volume" tagx="volume" title="volume">406</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">101</span><span class="lpage" tagx="lpage" title="lpage">106</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21295548">21295548</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inagaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishizawa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuyama</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Site-specific phosphorylation induces disassembly of vimentin filaments in vitro</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">328</span><span class="year" tagx="year" title="year">1987</span><span class="fpage" tagx="fpage" title="fpage">649</span><span class="lpage" tagx="lpage" title="lpage">652</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3039376">3039376</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koenderink</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">E.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Architecture shapes contractility in actomyosin networks</span><span class="source" tagx="source" title="source">Curr. Opin. Cell Biol.</span><span class="volume" tagx="volume" title="volume">50</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">79</span><span class="lpage" tagx="lpage" title="lpage">85</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29482169">29482169</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kordowska</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hetrick</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adam</span><span class="given-names" tagx="given-names" title="given-names">L.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phosphorylated l-caldesmon is involved in disassembly of actin stress fibers and postmitotic spreading</span><span class="source" tagx="source" title="source">Exp. Cell Res.</span><span class="volume" tagx="volume" title="volume">312</span><span class="year" tagx="year" title="year">2006</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">110</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16289153">16289153</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loureiro</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Ena/VASP enigma</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="volume" tagx="volume" title="volume">115</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">4721</span><span class="lpage" tagx="lpage" title="lpage">4726</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12432060">12432060</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunda</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelling</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baum</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moesin controls cortical rigidity, cell rounding, and spindle morphogenesis during mitosis</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">18</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">101</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18207738">18207738</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancaster</span><span class="given-names" tagx="given-names" title="given-names">O.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baum</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Shaping Up to divide: coordinating actin and microtubule cytoskeletal remodelling during mitosis</span><span class="source" tagx="source" title="source">Semin. Cell Dev. Biol.</span><span class="volume" tagx="volume" title="volume">34</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">109</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24607328">24607328</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lancaster</span><span class="given-names" tagx="given-names" title="given-names">O.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Berre</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimitracopoulos</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonazzi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zlotek-Zlotkiewicz</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Picone</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duke</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baum</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mitotic rounding alters cell geometry to ensure efficient bipolar spindle formation</span><span class="source" tagx="source" title="source">Dev. Cell</span><span class="volume" tagx="volume" title="volume">25</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">283</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23623611">23623611</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leduc</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Etienne-Manneville</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Intermediate filaments join the action</span><span class="source" tagx="source" title="source">Cell Cycle</span><span class="volume" tagx="volume" title="volume">16</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">1389</span><span class="lpage" tagx="lpage" title="lpage">1390</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28722513">28722513</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levental</span><span class="given-names" tagx="given-names" title="given-names">K.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kass</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lakins</span><span class="given-names" tagx="given-names" title="given-names">J.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egeblad</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erler</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fong</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Csiszar</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giaccia</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weninger</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Matrix crosslinking forces tumor progression by enhancing integrin signaling</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">139</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">891</span><span class="lpage" tagx="lpage" title="lpage">906</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19931152">19931152</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Logue</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cartagena-Rivera</span><span class="given-names" tagx="given-names" title="given-names">A.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baird</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davidson</span><span class="given-names" tagx="given-names" title="given-names">M.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chadwick</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Erk regulation of actin capping and bundling by Eps8 promotes cortex tension and leader bleb-based migration</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">4</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">e08314</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26163656">26163656</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luxenburg</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasolli</span><span class="given-names" tagx="given-names" title="given-names">H.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">S.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuchs</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Developmental roles for Srf, cortical cytoskeleton and cell shape in epidermal spindle orientation</span><span class="source" tagx="source" title="source">Nat. Cell Biol.</span><span class="volume" tagx="volume" title="volume">13</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">203</span><span class="lpage" tagx="lpage" title="lpage">214</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21336301">21336301</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Machicoane</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Frutos</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fink</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocancourt</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombardi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garel</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Echard</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">SLK-dependent activation of ERMs controls LGN-NuMA localization and spindle orientation</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">205</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">791</span><span class="lpage" tagx="lpage" title="lpage">799</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24958772">24958772</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maddox</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burridge</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RhoA is required for cortical retraction and rigidity during mitotic cell rounding</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">160</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">255</span><span class="lpage" tagx="lpage" title="lpage">265</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12538643">12538643</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matthews</span><span class="given-names" tagx="given-names" title="given-names">H.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delabre</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohn</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guck</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunda</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baum</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Changes in Ect2 localization couple actomyosin-dependent cell shape changes to mitotic progression</span><span class="source" tagx="source" title="source">Dev. Cell</span><span class="volume" tagx="volume" title="volume">23</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">371</span><span class="lpage" tagx="lpage" title="lpage">383</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22898780">22898780</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Depken</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bois</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jülicher</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grill</span><span class="given-names" tagx="given-names" title="given-names">S.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Anisotropies in cortical tension reveal the physical basis of polarizing cortical flows</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">467</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">617</span><span class="lpage" tagx="lpage" title="lpage">621</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20852613">20852613</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendez</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kojima</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vimentin induces changes in cell shape, motility, and adhesion during the epithelial to mesenchymal transition</span><span class="source" tagx="source" title="source">FASEB J.</span><span class="volume" tagx="volume" title="volume">24</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">1838</span><span class="lpage" tagx="lpage" title="lpage">1851</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20097873">20097873</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muruganujan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mills</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">P.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">PANTHER version 11: expanded annotation data from Gene Ontology and Reactome pathways, and data analysis tool enhancements</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">45</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">D183</span><span class="lpage" tagx="lpage" title="lpage">D189</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899595">27899595</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neuhoff</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arold</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taube</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehrhardt</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improved staining of proteins in polyacrylamide gels including isoelectric focusing gels with clear background at nanogram sensitivity using Coomassie brilliant blue G-250 and R-250</span><span class="source" tagx="source" title="source">Electrophoresis</span><span class="volume" tagx="volume" title="volume">9</span><span class="year" tagx="year" title="year">1988</span><span class="fpage" tagx="fpage" title="fpage">255</span><span class="lpage" tagx="lpage" title="lpage">262</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2466658">2466658</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ovesný</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Křížek</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borkovec</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svindrych</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagen</span><span class="given-names" tagx="given-names" title="given-names">G.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">ThunderSTORM: a comprehensive ImageJ plug-in for PALM and STORM data analysis and super-resolution imaging</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">30</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">2389</span><span class="lpage" tagx="lpage" title="lpage">2390</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24771516">24771516</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petronczki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glotzer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraut</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peters</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Polo-like kinase 1 triggers the initiation of cytokinesis in human cells by promoting recruitment of the RhoGEF Ect2 to the central spindle</span><span class="source" tagx="source" title="source">Dev. Cell</span><span class="volume" tagx="volume" title="volume">12</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">713</span><span class="lpage" tagx="lpage" title="lpage">725</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488623">17488623</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plastino</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olivier</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sykes</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Actin filaments align into hollow comets for rapid VASP-mediated propulsion</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">14</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">1766</span><span class="lpage" tagx="lpage" title="lpage">1771</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15458649">15458649</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramanathan</span><span class="given-names" tagx="given-names" title="given-names">S.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helenius</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">M.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cattin</span><span class="given-names" tagx="given-names" title="given-names">C.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyman</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cdk1-dependent mitotic enrichment of cortical myosin II promotes cell rounding against confinement</span><span class="source" tagx="source" title="source">Nat. Cell Biol.</span><span class="volume" tagx="volume" title="volume">17</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">148</span><span class="lpage" tagx="lpage" title="lpage">159</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25621953">25621953</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramkumar</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baum</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coupling changes in cell shape to chromosome segregation</span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="volume" tagx="volume" title="volume">17</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">511</span><span class="lpage" tagx="lpage" title="lpage">521</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27353479">27353479</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rappsilber</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryder</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamond</span><span class="given-names" tagx="given-names" title="given-names">A.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Large-scale proteomic analysis of the human spliceosome</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="volume" tagx="volume" title="volume">12</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">1231</span><span class="lpage" tagx="lpage" title="lpage">1245</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176931">12176931</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samwer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehne</span><span class="given-names" tagx="given-names" title="given-names">H.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spira</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollmar</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerlich</span><span class="given-names" tagx="given-names" title="given-names">D.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urlaub</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Görlich</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The nuclear F-actin interactome of Xenopus oocytes reveals an actin-bundling kinesin that is essential for meiotic cytokinesis</span><span class="source" tagx="source" title="source">EMBO J.</span><span class="volume" tagx="volume" title="volume">32</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">1886</span><span class="lpage" tagx="lpage" title="lpage">1902</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23727888">23727888</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satelli</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vimentin in cancer and its potential as a molecular target for cancer therapy</span><span class="source" tagx="source" title="source">Cell. Mol. Life Sci.</span><span class="volume" tagx="volume" title="volume">68</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">3033</span><span class="lpage" tagx="lpage" title="lpage">3046</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21637948">21637948</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">M.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyoda</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyman</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tracking mechanics and volume of globular cells with atomic force microscopy using a constant-height clamp</span><span class="source" tagx="source" title="source">Nat. Protoc.</span><span class="volume" tagx="volume" title="volume">7</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">143</span><span class="lpage" tagx="lpage" title="lpage">154</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22222789">22222789</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">K.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takaki</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petronczki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Targeting of the RhoGEF Ect2 to the equatorial membrane controls cleavage furrow formation during cytokinesis</span><span class="source" tagx="source" title="source">Dev. Cell</span><span class="volume" tagx="volume" title="volume">21</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">1104</span><span class="lpage" tagx="lpage" title="lpage">1115</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22172673">22172673</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogawara</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tomono</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moritoh</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imajoh-Ohmi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsutsumi</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taketani</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inagaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mitosis-specific phosphorylation of vimentin by protein kinase C coupled with reorganization of intracellular membranes</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">133</span><span class="year" tagx="year" title="year">1996</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">149</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8601602">8601602</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tinevez</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulze</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salbreux</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roensch</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joanny</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of cortical tension in bleb growth</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">106</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">18581</span><span class="lpage" tagx="lpage" title="lpage">18586</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19846787">19846787</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyoda</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cattin</span><span class="given-names" tagx="given-names" title="given-names">C.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stewart</span><span class="given-names" tagx="given-names" title="given-names">M.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poser</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theis</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurzchalia</span><span class="given-names" tagx="given-names" title="given-names">T.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchholz</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hyman</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-scale single-cell mechanical phenotyping reveals disease-related genes involved in mitotic rounding</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">8</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">1266</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29097687">29097687</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsujimura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogawara</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeuchi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imajoh-Ohmi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ha</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inagaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Visualization and function of vimentin phosphorylation by cdc2 kinase during mitosis</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">269</span><span class="year" tagx="year" title="year">1994</span><span class="fpage" tagx="fpage" title="fpage">31097</span><span class="lpage" tagx="lpage" title="lpage">31106</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7983050">7983050</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiche</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osmanagic-Myers</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castañón</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Networking and anchoring through plectin: a key to IF functionality and mechanotransduction</span><span class="source" tagx="source" title="source">Curr. Opin. Cell Biol.</span><span class="volume" tagx="volume" title="volume">32</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">29</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25460778">25460778</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamaguchi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yokoyama</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silljé</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanisch</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uldschmid</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oguri</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nigg</span><span class="given-names" tagx="given-names" title="given-names">E.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inagaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phosphorylation by Cdk1 induces Plk1-mediated vimentin phosphorylation during mitosis</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">171</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">431</span><span class="lpage" tagx="lpage" title="lpage">436</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16260496">16260496</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamashiro</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakita</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishikawa</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsumura</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mitosis-specific phosphorylation causes 83K non-muscle caldesmon to dissociate from microfilaments</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">344</span><span class="year" tagx="year" title="year">1990</span><span class="fpage" tagx="fpage" title="fpage">675</span><span class="lpage" tagx="lpage" title="lpage">678</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2157986">2157986</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamashiro</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamakita</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hosoya</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsumura</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phosphorylation of non-muscle caldesmon by p34cdc2 kinase during mitosis</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">349</span><span class="year" tagx="year" title="year">1991</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">172</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1986309">1986309</a></span></span></li> <li tag="ref"><a name="bib61" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yasui</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsui</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manser</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagata Ki</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inagaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protein kinases required for segregation of vimentin filaments in mitotic process</span><span class="source" tagx="source" title="source">Oncogene</span><span class="volume" tagx="volume" title="volume">20</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">2868</span><span class="lpage" tagx="lpage" title="lpage">2876</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11420699">11420699</a></span></span></li> </ul> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplemental Information</div> <p id="p0365"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Document S1. Figures S1–S5 and Table S2</div>  <div class="media" title="media"><a href="mmc1.pdf">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S1. Mass Spectrometry Data on the F-actin Interactome (Raw Data and Overlay between Experiments), Related to Figures 1 and 2</div>  <div class="media" title="media"><a href="mmc10.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Document S2. Article plus Supplemental Information</div>  <div class="media" title="media"><a href="mmc11.pdf">LINK</a></div> </div> <p /> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We thank Dennis Bray, Kevin Chalut, and members of the Paluch lab, in particular Dani Bodor, for feedback on the work and manuscript. We thank Wolfram Pönisch for help with analyzing cell curvatures; Mark Turmaine for help with scanning electron microscopy; Andrew Vaughan, Ki Hng, and Catherine Scherer for support with light and STORM microscopy; Davide Cassani for technical help with AFM; and Jürgen Schünemann (MPI for Biophysical Chemistry) for HPLC-purifying phalloidin derivatives. We thank Éric Bonneil and Pierre Thibault for expert technical help and advice on mass spectrometry analysis and peptide quantification. We acknowledge support from the </p><div class="funding-source" title="funding-source">
Medical Research Council</div> UK (MRC programme award MC_UU_00012/5, M.P.S., B.A.T.Q., U.P., and E.K.P.), the <div class="funding-source" title="funding-source">
Human Frontier Science Program</div> (Young Investigator grant RGY 66/2013 to G.C., P.P.R., and E.K.P.), the <div class="funding-source" title="funding-source">
European Research Council</div> (starting grant 311637-MorphoCorDiv to E.K.P.), <div class="funding-source" title="funding-source">
Cancer Research UK</div> (M.P. and M.P.S.), a <div class="funding-source" title="funding-source">
BBSRC</div> ALERT16 equipment grant (BB/R000042 to G.C.) for purchase of AFM instrumentation, and an operating grant from the <div class="funding-source" title="funding-source">
Canadian Institute of Health Research</div> to P.P.R. (MOP-142374). P.P.R. is supported by a <div class="funding-source" title="funding-source">
FRQS</div> Senior investigator career award.<p /> <div class="authorscontributions" title="sec"> <div class="title" tagx="title" title="title">
Authors Contributions</div> <p id="p0350">M.P.S., M.P., P.P.R., and E.K.P. designed the research; M.P.S. carried out most of the experiments and analyzed the data. M.S. and D.G. developed and provided the phalloidin matrix; B.A.T.Q. developed the analysis tools for STORM and analyzed the 2-color STORM experiment; G.L. carried out the qPCR, helped with mass spectrometry, and performed the mutagenesis; U.P. carried out SEM experiments and helped with other experiments and data analysis; M.P.S. and E.K.P. wrote the paper; D.G., G.C., and P.P.R. provided technical support and conceptual advice. Funding Acquisition, G.C., M.P., P.P.R., and E.K.P. All authors discussed the results and manuscript.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Declaration of Interests</div> <p id="p0355">The authors declare no competing interests.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-supplementary-material" title="supplementary-material"> <p id="p0360">Supplemental Information can be found online at <a href="https://doi.org/10.1016/j.devcel.2019.12.011">https://doi.org/10.1016/j.devcel.2019.12.011</a>.</p> </div> </div> </div>  </body></html>