<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Cell Dev Biol</span><span class="iso-abbrev" title="iso-abbrev">Front Cell Dev Biol</span><span class="publisher-id" title="publisher-id">Front. Cell Dev. Biol.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Cell and Developmental Biology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2296-634X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6935720">6935720</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fcell.2019.00304">10.3389/fcell.2019.00304</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell and Developmental Biology</span>: <span class="subject" title="subject">Original Research</span></div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Thymosin β4 Regulates Focal Adhesion Formation in Human Melanoma Cells and Affects Their Migration and Invasion</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowiecka</span><span class="given-names" tagx="given-names" title="given-names">Aleksandra</span></span><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/771528/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malek</span><span class="given-names" tagx="given-names" title="given-names">Natalia</span></span><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/349548/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazurkiewicz</span><span class="given-names" tagx="given-names" title="given-names">Ewa</span></span><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/772929/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mrówczyńska</span><span class="given-names" tagx="given-names" title="given-names">Ewa</span></span><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/850628/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">Dorota</span></span><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/506839/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">Antonina Joanna</span></span><a href="#c001"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/771347/overview" /></span></span></div><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Department of Cell Pathology, Faculty of Biotechnology, University of Wrocław</span><span class="addr-line" title="addr-line">Wrocław</span><span class="country" tagx="country" title="country">Poland</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Vladimir Sytnyk, University of New South Wales, Australia</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Nicola Smart, University of Oxford, United Kingdom; Nagaraj Balasubramanian, Indian Institute of Science Education and Research, Pune, India</p> </div> <div class="corresp" title="corresp">
*Correspondence: Antonina Joanna Mazur, <span class="email" tagx="email" title="email">antonina.mazur@uwr.edu.pl</span></div> <div class="fn-type-other" title="other"> <p>This article was submitted to Cell Adhesion and Migration, a section of the journal Frontiers in Cell and Developmental Biology</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-12-12</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">304</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-05</span><span class="accepted" title="accepted">accepted: 2019-11-11</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Makowiecka, Malek, Mazurkiewicz, Mrówczyńska, Nowak and Mazur</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Thymosin β4 (Tβ4), a multifunctional 44-amino acid polypeptide and a member of actin-binding proteins (ABPs), plays an important role in developmental processes and wound healing. In recent years an increasing number of data has been published suggesting Tβ4’s involvement in tumorigenesis. However, Tβ4’s role in melanoma tumor development still remains to be elucidated. In our study we demonstrate that Tβ4 is crucial for melanoma adhesion and invasion. For the purpose of our research we tested melanoma cell lines differing in invasive potential. Moreover, we applied shRNAs to silence <i>TMSB4X</i> (gene encoding Tβ4) expression in a cell line with high <i>TMSB4X</i> expression. We found out that Tβ4 is not only a component of focal adhesions (FAs) and interacts with several FAs components but also regulates FAs formation. We demonstrate that Tβ4 level has an impact on FAs’ number and morphology. Moreover, manipulation with <i>TMSB4X</i> expression resulted in changes in cells’ motility on non-coated and Matrigel<sup>TM</sup> (resembling basement membrane composition)-coated surfaces and drastically decreased invasion abilities of the cells. Additionally, a correlation between Tβ4 expression level and exhibition of mesenchymal-like [epithelial-mesenchymal transition (EMT)] features was discovered. Cells with lowered <i>TMSB4X</i> expression were less EMT-progressed than control cells. Summarizing, obtained results show that Tβ4 by regulating melanoma cells’ adhesion has an impact on motility features and EMT. Our study not only contributes to a better understanding of the processes underlying melanoma cells’ capacity to create metastases but also highlights Tβ4 as a potential target for melanoma management therapy.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
melanoma</div> <div class="kwd" title="kwd">
thymosin β4</div> <div class="kwd" title="kwd">
focal adhesion</div> <div class="kwd" title="kwd">
epithelial-mesenchymal transition</div> <div class="kwd" title="kwd">
invasion</div> <div class="kwd" title="kwd">
migration</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Fundacja na rzecz Nauki Polskiej<span class="named-content" title="named-content">10.13039/501100001870</span></div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Narodowe Centrum Nauki<span class="named-content" title="named-content">10.13039/501100004281</span></div> </div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The crucial event during melanoma progression is a transition from the superficial spreading stage to invasive propagation to the dermis (<a href="#B35">Miller and Mihm, 2006</a>). In order to invade, melanoma cells need to acquire the ability to cross the basement membrane through alterations in adhesion and motility properties. During tumorigenesis epithelial cells are transformed into mesenchymal-like cells in a process called the epithelial-mesenchymal transition (EMT). Melanoma cells derive from melanocytes, cells of neural crest origin, which during embryogenesis undergo a process resembling EMT (<a href="#B27">Kulesa et al., 2013</a>). Melanocytes are tightly attached to the basement membrane and their survival and function depend on interactions with extracellular matrix (ECM) and keratinocytes (<a href="#B9">Cichorek et al., 2013</a>). The main cell-matrix adhesion structures are focal adhesions (FAs), that are composed of clusters of transmembrane proteins – integrins interacting with their ECM ligands and intracellular multiprotein complexes providing connection to the actin cytoskeleton. These structures are responsible for mechanical anchorage to ECM and extracellular cues transmission into a cell’s interior (<a href="#B53">Zaidel-Bar et al., 2004</a>).</p> <p>During tumor progression vast number of cellular properties and processes undergo alteration, which means that along with the adhesive properties, the cells’ motility abilities change. There are two cell movement types described as canonical–mesenchymal, dependent on Rac1 activation, characterized by degradation of ECM and formation of lamellipodia and invadopodia, and ameboidal, dependent on Rho/ROCK1 activity but not on ECM degradation (<a href="#B26">Krakhmal et al., 2015</a>). Both types of cell’s motility base on dynamic rearrangements of the actin cytoskeleton, which depend on the activity of actin-binding proteins (ABPs), which according to their functions can be divided into several groups. To one of these groups belong proteins sequestering monomeric actin (G-actin) and one example of this set is thymosin β4 (Tβ4) (<a href="#B46">Shekhar and Carlier, 2017</a>). Several studies showed that Tβ4 affects cells’ migration (reviewed in <a href="#B15">Goldstein et al., 2005</a>; <a href="#B49">Sribenja et al., 2013</a>). However, the role of Tβ4 in neoplasia is still unclear. Although overexpression of <i>TMSB4X</i> (gene encoding Tβ4) is correlated with patients’ poor prognosis in some types of tumors (<a href="#B7">Chi et al., 2017</a>), Tβ4 exhibits a suppressive effects in others (<a href="#B5">Caers et al., 2010</a>). An increasing number of data shows that Tβ4 is involved in EMT and cell differentiation in normal and tumor cells (<a href="#B17">Ho et al., 2007</a>; <a href="#B36">Mollinari et al., 2009</a>; <a href="#B52">Wirsching et al., 2014</a>). It was reported that <i>in vivo</i> selected melanoma cell lines expressed Tβ4 at high level (<a href="#B11">Clark et al., 2000</a>), what was connected with their metastatic potential. However, the role of Tβ4 in melanoma progression has not been thoroughly investigated yet. Therefore in our studies, we decided to unveil the role of Tβ4 in melanoma cells’ motility and EMT progression. We performed experiments on melanoma cells differing in invasion abilities and on cells with lowered expression of <i>TMSB4X</i> by application of shRNA. We discovered that Tβ4 level regulates the number and morphology of FAs and probably through that has an impact on adhesion and thus motility of melanoma cells. Moreover, we found out that manipulating with <i>TMSB4X</i> expression EMT progression can be influenced.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="hightβ4expressionispositivelycorrelatedwithinvasivenessofmelanomacells" title="sec"> <div class="title" tagx="title" title="title">
High Tβ4 Expression Is Positively Correlated With Invasiveness of Melanoma Cells</div> <p>According to Oncomine database<sup><a href="#footnote1">1</a></sup> (<a href="#F1">Figure 1A</a>; <a href="#B41">Ramaswamy et al., 2001</a>) the <i>TMSB4X</i> expression level varies depending on tumor type. Some of them, including melanoma, are characterized by a very wide range of expression level in patients’ samples. Intrigued by this finding we decided to test four melanoma cell lines in terms of Tβ4 level and its subcellular localization. Here we have to state that validation of two commercially available antibodies recognizing Tβ4 revealed their non-specificity, as two homologous polypeptides to Tβ4 present in humans: Tβ10 and Tβ15 (<a href="#B15">Goldstein et al., 2005</a>) were recognized by used antibodies (<a href="#DS1">Supplementary Figure S1</a>). We cloned all three thymosins β (Tβs) under a HA-tag and after transfection of the cells with DNA constructs coding for HA-Tβs we fixed and immunostained the cells with antibodies. As it can be seen on micrographs all three Tβs are recognized by two used commercially available antibodies directed against Tβ4. Because of that, starting from now on, whenever antibodies recognizing Tβs are used, we write Tβs instead of Tβ4. Due to the lack of specific antibodies it was also impossible to perform Western blot analysis to verify the level of Tβ4 in studied cells. That is why we checked <i>TMSB4X</i> expression level at mRNA level. Analysis of amplification curves (qRT-PCR) showed that among three Tβs present in human in WM1341D cells Tβ4 is a dominant version of Tβs, although Tβ15 is expressed at a relatively high level too (<a href="#DS1">Supplementary Figure S2</a>). In the case of A375 cells the differences between amplification curves for Tβ4 and Tβ15 are much bigger in comparison to WM1341D cell line. On contrary in both cell lines <i>TMSB10</i> was expressed at a very low level.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Evaluation of <i>TMSB4X</i> expression in different neoplasms and human melanoma cell lines differing in invasion abilities. <b>(A)</b><i>TMSB4X</i> expression depending on neoplasia type; adapted from <a href="https://www.oncomine.org">oncomine.org</a>; <b>(B)</b> Immunostainings of four human melanoma cell lines to visualize F-actin and Tβs. Left column shows lower magnification, whereas the right one present microphotographs of single cells. <b>(C)</b> qRT-PCR analysis of <i>TMSB4X</i> expression in cell lines obtained from primary tumor sites and metastases (<i>n</i> = 3). <b>(D)</b> 3D-migration/invasion analysis of four melanoma cell lines (<i>n</i> = 3). Arrows point at stress fibers tips. The significance level was set at <sup>∗</sup><i>P</i> &amp;lt; 0.05, <sup>∗∗</sup><i>P</i> &amp;lt; 0.01, <sup>∗∗∗</sup><i>P</i> &amp;lt; 0.001, and <sup>∗∗∗∗</sup><i>P</i> &amp;lt; 0.0001 (<a href="http://www.oncomine.org/">www.oncomine.org</a>, February 2018, Thermo Fisher Scientific, Ann Arbor, MI, United States).</p>   </div> <p>We then examined Tβs subcellular localization. In tested cell lines thymosins β were localized in cell nucleus, cytoplasm and on the cell’s perimeter and they colocalized with filamentous actin (F-actin) at the tips of stress fibers (<a href="#F1">Figure 1B</a>). Next we found out that metastatic cell lines expressed <i>TMSB4X</i> at higher level when compared to cell lines isolated from primary tumor sites (<a href="#F1">Figure 1C</a>). Interestingly in every pair (primary sites and metastases) there was a cell line with significantly lower <i>TMSB4X</i> expression. These observations correlated with invasion capacities of tested cells. The invasion assay was performed using Transwell<sup>TM</sup> filters coated with Matrigel<sup>TM</sup> (a mixture of ECM proteins which mimic the basement membrane composition). This assay imitates penetration of the basement membrane by cells during melanoma progression from RPG to VPG. The cells with invasion potential are capable to protrude through Matrigel<sup>TM</sup> and overcome the pores in Transwell filter’s membrane. The percent of invading A375 cells was three times higher comparing to WM1341D cells, whereas among Hs294T cells there were seven times more invading cells than in the case of WM9 cells (<a href="#F1">Figure 1D</a>). For further experiments we chose WM1341D and A375 cell lines because they represent cells isolated from primary tumor sites.</p> </div> <div class="tβ4isacomponentoffocaladhesionsandinteractswithfascomponents" title="sec"> <div class="title" tagx="title" title="title">
Tβ4 Is a Component of Focal Adhesions and Interacts With FAs Components</div> <p>Due to the lack of specific Tβ4-recognizing antibodies we obtained DNA constructs (<a href="#F2">Figure 2A</a>), which were used in following experiments. Intrigued by Tβs presence at the tips of stress fibers (<a href="#F1">Figure 1C</a>) we performed additional immunostainings on WM1341D and A375 cells and stated that Tβs co-localized within FAs with vinculin and αVβ3 integrin (FAs’ constituents; <a href="#B19">Horzum et al., 2014</a>) in studied cell lines (<a href="#F2">Figure 2B</a>). Further analysis revealed that HA-Tβ4 co-localized with vinculin in FAs (<a href="#F2">Figure 2C</a>). Moreover, we noted that only after overexpression of Tβ4, but not after Tβ10 or Tβ15 overexpression morphology of FAs was changed (<a href="#DS1">Supplementary Figure S3</a>). In both cell lines FAs were bigger than in non-transfected cells. Proximity ligation assay (PLA) with FAs’ structural and signaling proteins showed that HA-Tβ4 formed complexes with vinculin, integrin-linked (ILK), focal adhesion kinase (FAK), pinch 2, α parvin, and integrin αVβ3 (<a href="#F2">Figure 2D</a>). Interestingly, HA-Tβ4 is in close proximity to vinculin, ILK and FAK in the whole cell body, whereas putative complexes between HA-Tβ4 and pinch 2, α parvin and integrin αVβ3 are localized rather submembranous, possibly within FAs. Negative controls for PLA assay are shown in <a href="#DS1">Supplementary Figure S4A</a>. All tested here proteins are present at FAs in both cell lines (<a href="#DS1">Supplementary Figure S4B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Thymosin β4 is a constituent of focal adhesions (FAs) and forms complexes with FAs constituents in WM1341D and A375 cell lines. <b>(A)</b> Schematic representation of DNA constructs used in this study. <b>(B)</b> Immunocytochemical stainings visualizing FAs components: vinculin and αVβ3 integrin and co-localizing with them Tβs. <b>(C)</b> Visualization of vinculin and exogenous HA-Tβ4 in transfected cells. <b>(D)</b> PLA of HA-Tβ4 vs. vinculin, ILK, FAK, Pinch 2, α parvin, and αVβ3 integrin in transfected cells. Cell nuclei are shown in dark gray. Arrows point at FAs, while red arrowheads indicate PLA complexes.</p>   </div> </div> <div class="tβ4influencesformationoffasandadhesionofmelanomacells" title="sec"> <div class="title" tagx="title" title="title">
Tβ4 Influences Formation of FAs and Adhesion of Melanoma Cells</div> <p>Quantitative analysis of FAs visualized by vinculin staining (<a href="#F3">Figure 3A</a>) revealed significantly reduced number (two times less) of FAs in A375 cells when compared to WM1341D cells at 24 h (<a href="#F3">Figure 3B</a>). At 24, 48, and 72 h average FAs number in WM1341D cells was 43, 80, 95, respectively, meaning that new FAs were additionally formed over time. However, in A375 cells at every time point the FAs number was the same c.a. 20 (<a href="#F3">Figures 3A,B</a>). Though the number of FAs in WM1341D cells was high, FAs were significantly smaller than those observed in A375 cells (<a href="#F3">Figures 3A,C</a>). The differences in FAs’ morphology were not caused by changes in vinculin protein level (<a href="#DS1">Supplementary Figure S5</a>). We performed similar analyses on cells with visualized α parvin to check if data obtained on the basis of vinculin staining was replicable (<a href="#DS1">Supplementary Figure S6</a>). Results acquired by analysis of the cells with visualized α parvin corroborated analogous previous outcomes. Next we looked at the area covered by a single cell. On average A375 cell’s area was about 35% smaller than of WM1341D cell (<a href="#F3">Figure 3D</a>), but in detached cells the differences in median of relative size (FSC parameter measured using flow cytometry) between two cell lines was below 2.5% (<a href="#DS1">Supplementary Figure S7</a>). Therefore the differences observed in cell’s area of A375 and WM1341D cells were caused by distinct adhesion abilities of these cell lines. Because of that, we decided to perform an adhesion assay. Higher number of WM1341D cells adhered within 1 h to both non-coated and Matrigel<sup>TM</sup>-coated surface in comparison to A375 cells (<a href="#F3">Figure 3E</a>). Intriguingly, while more WM1341D cells attached to Matrigel<sup>TM</sup> than to non-coated surface, A375 cells adhered to both types of surfaces at the same extent.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Focal adhesion formation in WM1341D and A375 cells and effect of <i>TMSB4X</i> expression silencing on formation of FAs in melanoma cells. <b>(A,F)</b> Immunocytochemical analysis of cells in order to visualize vinculin. Cells’ borders were marked for better orientation with white lines. Quantitation of FAs’ number <b>(B,G)</b>, size of FA’s <b>(C,H),</b> and cell’s area <b>(D,I)</b> (<i>n</i> = 20). <b>(E,J)</b> Adhesion assays of investigated cell lines and clones (<i>n</i> = 3). Arrows point at FAs. The significance level was set at <sup>∗</sup><i>P</i> &amp;lt; 0.05, <sup>∗∗</sup><i>P</i> &amp;lt; 0.01, <sup>∗∗∗</sup><i>P</i> &amp;lt; 0.001, and <sup>∗∗∗∗</sup><i>P</i> &amp;lt; 0.0001 (<a href="http://www.oncomine.org/">www.oncomine.org</a>, February 2018, Thermo Fisher Scientific, Ann Arbor, MI, United States).</p>   </div> <p>To check the role of Tβ4 in FAs formation we obtained five A375 stable clones with decreased <i>TMSB4X</i> expression (<a href="#DS1">Supplementary Figure S8</a>). We noted that <i>TMSB4X</i> silencing did not affect <i>TMSB10</i> and <i>TMSB15</i> expression at the mRNA levels (<a href="#DS1">Supplementary Figure S8</a>). Due to unspecifity of antibodies we were not able to observe changes at protein level using immunocytochemical analysis of A375-sh<i>TMSB4X</i> and A375-shscramble clones with anti-Tβ4 antibodies (<a href="#DS1">Supplementary Figure S1C</a>). We performed all following analyses for five biological replicates of A375-sh<i>TMSB4X</i> clones and then we merged obtained data and described them as shTβ4 on the graphs to simplify the results presentation. First we immunostained the cells in order to visualize vinculin and thus FAs. In A375-sh<i>TMSB4X</i> clones we noted higher number but decreased size of FAs when compared to A375-shscramble cells (<a href="#F3">Figures 3F–H</a>). Again we did analogous analyses on micrographs of cells stained for α parvin and we obtained similar results (<a href="#DS1">Supplementary Figure S9</a>) as we did on the basis of vinculin visualization. Simultaneously, we observed a cell’s area increased by about 33% in A375-sh<i>TMSB4X</i> clones (<a href="#F3">Figure 3I</a>). Next, we checked adhesion abilities of tested clones. Unexpectedly, we noted that cells with lowered <i>TMSB4X</i> expression adhered to the non-coated surface in lower numbers than the control cells (<a href="#F3">Figure 3J</a>). We suspect that the difference between cell adhesion vs. spreading on the surface may be the result of the duration of experiments. In the case of adhesion studies, the cells’ binding was checked after 1 h, while the spreading on the surface after 24 h. On the other hand, cells with lowered level of Tβ4 attached to the Matrigel<sup>TM</sup>-coated surface better than control cells. Generally, both cell types adhered better to Matrigel<sup>TM</sup> than to non-coated surface. We planned to obtain stable clones from WM1341D cells with increased expression of Tβ4. Unfortunately, we were not able to receive these clones. WM1341D cells turned out to be highly resistant for any kind of genetic manipulation. Transfection efficiency was too low to perform any experiments after transient transfection. We tried lipofection, magnetic transfection and electroporation. We were able to obtain very few transfected cells enabling us to prepare <a href="#DS1">Supplementary Figure S3</a>. However, any attempt to obtain stable clones failed.</p> </div> <div class="tβ4affectsmelanomacells’invasionandmigrationabilities" title="sec"> <div class="title" tagx="title" title="title">
Tβ4 Affects Melanoma Cells’ Invasion and Migration Abilities</div> <p>Because adhesion is directly connected to motility (<a href="#B21">Huttenlocher et al., 1995</a>), we evaluated migratory abilities of studied cells. Spontaneous 2D cell movement (2D migration) showed that A375 cells covered significantly longer distances (twofold change) compared to WM1341D cells (<a href="#F4">Figures 4A,B</a>). However, there were no differences between two cell lines in terms of directionality (<a href="#F4">Figure 4C</a>). The way we calculated this parameter is presented in <a href="#DS1">Supplementary Figure S10</a>. A similar observation was made for collective migration capacity measured by a wound healing assay. The scratch was closed significantly faster by A375 cells when compared to WM1341D cells (<a href="#F4">Figures 4D,E</a>). We also checked 2D migration of tested cells on Matrigel<sup>TM</sup>-coated surface (<a href="#F4">Figures 4F–J</a>). In the case of scratch assay we checked first if Matrigel<sup>TM</sup> coating remained upon making a scratch. Visualization of laminin (main component of Matrigel<sup>TM</sup>) revealed that the layer of Matrigel<sup>TM</sup> stays after scratching (<a href="#DS1">Supplementary Figure S11</a>). Surprisingly discrepancies between WM1341D and A375 cells were much smaller than in the case of non-coated surface (<a href="#F4">Figures 4F–J</a> vs. <a href="#F4">4A–E</a>). The only statistically significant difference we noted was in the amount of covered distance. A375 cells covered c.a. 1.2 times longer distance than WM1341D cells, whilst on non-coated surface A375 cells moved over c.a. 3.8 times longer distances when compared to WM1341D cells (<a href="#F4">Figures 4B,G</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Characterization of 2D motility on non-coated and Matrigel<sup>TM</sup>-coated surface of WM1341D and A375 cells. The cells were seeded either on non-coated surface <b>(A–E)</b> or on Matrigel<sup>TM</sup>-coated surface <b>(F–J)</b>. <b>(A–C,F–H)</b> Spontaneous 2D-migration of tested cells. The cells were recorded over 72 h (<i>n</i> = 30): <b>(A,F)</b> trajectories of single cells migration, <b>(B,G)</b> calculated migrated distances of the cells, <b>(C,H)</b> estimation of directionality of cells’ migration. Wound healing assay performed on tested cells (<i>n</i> = 3): <b>(D,I)</b> representative pictures of cells migrating collectively and <b>(E,J)</b> graphs representing scratch closure expressed in % of closed scratch. The significance level was set at <sup>∗</sup><i>P</i> &amp;lt; 0.05, <sup>∗∗</sup><i>P</i> &amp;lt; 0.01, <sup>∗∗∗</sup><i>P</i> &amp;lt; 0.001, and <sup>∗∗∗∗</sup><i>P</i> &amp;lt; 0.0001 (<a href="http://www.oncomine.org/">www.oncomine.org</a>, February 2018, Thermo Fisher Scientific, Ann Arbor, MI, United States).</p>   </div> <p>To determine if the differences in 2D migration potential of two tested melanoma cell lines are connected with Tβ4 level, we further tested A375-sh<i>TMSB4X</i> and A375-shscramble clones. 2D migration assay showed surprisingly that A375-sh<i>TMSB4X</i> clones covered significantly longer distances compared to control clone (<a href="#F5">Figures 5A,B</a>). Directionality of protrusion on non-coated surface was unaffected by lowered Tβ4 level (<a href="#F5">Figure 5C</a>). Interestingly, similarly to covered distances, collective migration was improved in A375-sh<i>TMSB4X</i> clones compared to control cells (<a href="#F5">Figures 5D,E</a> and <a href="#DS1">Supplementary Figure S12</a>). Again we performed 2D migration analyses on Matrigel<sup>TM</sup>-coated surface. We found out that though the covered distances were indistinguishable for tested clones (<a href="#F5">Figures 5F,G</a>), directionality of cells with lowered Tβ4 level was statistically significantly altered in comparison to control cells (<a href="#F5">Figure 5H</a>). On the contrary, scratch was closed similarly fast by both tested types of clones (<a href="#F5">Figures 5I,J</a> and <a href="#DS1">Supplementary Figure S13</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 5</span></a></div>   <p>Decreasing of Tβ4 level affects A375 cells’ 2D motility. The cells were seeded either on non-coated surface <b>(A–E)</b> or on Matrigel<sup>TM</sup> surface <b>(F–J)</b>. <b>(A–C,F–H)</b> Spontaneous 2D-migration of tested cells. The cells were recorded over 72 h (<i>n</i> = 30): <b>(A,F)</b> trajectories of single cells migration, <b>(B,G)</b> calculated migrated distances of the cells, <b>(C,H)</b> estimation of directionality of cells’ migration. Wound healing assay performed on tested cells (<i>n</i> = 3): <b>(D,I)</b> representative pictures of cells migrating collectively and <b>(E,J)</b> graphs representing scratch closure expressed in % of closed scratch. The significance level was set at <sup>∗</sup><i>P</i> &amp;lt; 0.05, <sup>∗∗</sup><i>P</i> &amp;lt; 0.01, <sup>∗∗∗</sup><i>P</i> &amp;lt; 0.001, and <sup>∗∗∗∗</sup><i>P</i> &amp;lt; 0.0001 (<a href="http://www.oncomine.org/">www.oncomine.org</a>, February 2018, Thermo Fisher Scientific, Ann Arbor, MI, United States).</p>   </div> </div> <div class="tβ4influencesepithelial-mesenchymaltransitioninmelanomacells" title="sec"> <div class="title" tagx="title" title="title">
Tβ4 Influences Epithelial-Mesenchymal Transition in Melanoma Cells</div> <p>We analyzed the type of cell movement of both cell lines. WM1341D and A375 cells motility was impaired by addition of inhibitors such as GM6001 (matrix metalloproteinases inhibitor; <a href="#B45">Schultz et al., 1992</a>), NSC23766 (inhibitor of Rac1; <a href="#B14">Gao et al., 2004</a>) and Y-27632 a selective, ATP-competitive inhibitor of Rho-associated protein kinase (ROCK1) (<a href="#B50">Uehata et al., 1997</a>). After treatment with GM6001 the number of invading cells was decreased by 50 and 85% for WM1341D and A375 cells, respectively. In the case of NSC23766 addition the number of cells with invasive potential was diminished by 25 and 30% for WM1341D and A375 cells, respectively. Addition of Y-27632 caused a drop in the number of invading cells by 75 and 50% for WM1341D and A375 cells, respectively (<a href="#F6">Figure 6A</a>). Visualization of F-actin and cortactin (invadopodia marker; <a href="#B2">Artym et al., 2006</a>) revealed invadopodia formation only in A375 cells (<a href="#F6">Figure 6B</a>), while in 3D Matrigel<sup>TM</sup> matrix we observed rounded morphology for WM1341D cells and more elongated shape for A375 cells (<a href="#F6">Figure 6B</a>). These observations show that WM1341D cells move in a more ameboidal way, whereas A375 cells protrude in a more mesenchymal way. Generally, A375 cells were more prone to invade through the Matrigel<sup>TM</sup> gel toward a chemotactic gradient than WM1341D cells (<a href="#F1">Figure 1D</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 6</span></a></div>   <p>A375 cells are more progressed in epithelial-mesenchymal transition (EMT) than WM1341D cells. <b>(A)</b> Analysis of 3D motility type exhibited by studied cell lines by inhibitors application: 25 μM GM6001, 10 μM NSC23766, 10 μM Y-27632. The cells were seeded onto Matrigel<sup>TM</sup> gel placed in the upper compartment of a Transwell<sup>TM</sup> in medium with added inhibitors. 24 h later the assay was terminated (<i>n</i> = 3). <b>(B)</b> Immunostainings performed on cells growing under 2D or 3D (the cells were embedded in Matrigel<sup>TM</sup> gel) conditions. F-actin and cortactin were visualized by application of fluorescently labeled phalloidin and antibodies recognizing cortactin. Arrowheads point at invadopodia. <b>(C)</b> Visualization of N-cadherin and F-actin in studied cells by immucytochemical staining. <b>(D)</b> Representative immunoblots of EMT markers levels in tested cell lines. 30 μg of protein was loaded on every lane. Membranes prior to incubation with antibodies were stained with Ponceau S to show equal protein loading on lanes. The significance level was set at <sup>∗</sup><i>P</i> &amp;lt; 0.05, <sup>∗∗</sup><i>P</i> &amp;lt; 0.01, <sup>∗∗∗</sup><i>P</i> &amp;lt; 0.001, and <sup>∗∗∗∗</sup><i>P</i> &amp;lt; 0.0001 (<a href="http://www.oncomine.org/">www.oncomine.org</a>, February 2018, Thermo Fisher Scientific, Ann Arbor, MI, United States).</p>   </div> <p>Next, we checked by immunostainings the presence of N-cadherin in studied cells, what corroborated our presumption that WM1341D and A375 underwent EMT (<a href="#F6">Figure 6C</a>). N-cadherin is expressed in both cell lines in low and high density, however, we observed changes in localization. In a more confluent cell culture this protein is localized in adherens/tight junctions and in single cells the signal from staining is more dispersed (<a href="#DS1">Supplementary Figure S14</a>). The same situation we observed in β-catenin staining, however, in any case we did not notice translocation to nucleus. While we did not observe the presence of ZO-1 in adherens/tight junctions. Western blot analysis (WB) was performed to further estimate the levels of several EMT markers (proteins involved in cell’s epithelial or mesenchymal phenotype) such as vimentin, β-catenin, SNAI1/Snail, SNAI2/Slug, ZO-1, and Zeb-1 (<a href="#B28">Lamouille et al., 2014</a>). Elevated levels of vimentin, SNAI1/Snail, ZO-1, and Zeb1 were observed in A375 cells when compared to WM1341D cells (<a href="#F6">Figure 6D</a>). Interestingly, SNAI2/Slug was expressed at high level only in WM1341D cells (<a href="#F6">Figure 6D</a>). On the other hand expression of β-catenin was similar in both cell lines (<a href="#F6">Figure 6D</a>). Obtained results imply that tested cell lines are at different stages of EMT. In order to control the amount of loaded protein on lanes we performed total protein analysis (TPA) (<a href="#B1">Aldridge et al., 2008</a>). The reasons for that are discussed elsewhere (<a href="#B33">Mazur et al., 2016a</a>). In <a href="#T1">Table 1</a> we compared results obtained for WM1341D and A375 cell lines.</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Comparison of WM1341D and A375 melanoma cell lines studied features.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><b>Feature</b></td> <td valign="top" align="center" rowspan="1" colspan="1"><b>WM1341D</b></td> <td valign="top" align="center" rowspan="1" colspan="1"><b>A375</b></td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><i>TMSB4X</i> expression level</td> <td valign="top" align="center" rowspan="1" colspan="1">Low</td> <td valign="top" align="center" rowspan="1" colspan="1">High</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Invasion potential</td> <td valign="top" align="center" rowspan="1" colspan="1">Low</td> <td valign="top" align="center" rowspan="1" colspan="1">High</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Type of cell movement</td> <td valign="top" align="center" rowspan="1" colspan="1">More ameboidal</td> <td valign="top" align="center" rowspan="1" colspan="1">More mesenchymal</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">2D Migration</td> <td valign="top" align="center" rowspan="1" colspan="1">Short distances</td> <td valign="top" align="center" rowspan="1" colspan="1">Long distances</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Average number of focal adhesion (per cell)</td> <td valign="top" align="center" rowspan="1" colspan="1">24 h – 43 FAs<br /><br />48 h – 80 FAs<br /><br />72 h – 93 FAs</td> <td valign="top" align="center" rowspan="1" colspan="1">24 h – 20 FAs<br /><br />48 h – 20 FAs<br /><br />72 h – 20 FAs</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Spreading area (average cell’s area)</td> <td valign="top" align="center" rowspan="1" colspan="1">49 AU</td> <td valign="top" align="center" rowspan="1" colspan="1">31 AU</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Number of cells adhering to a substrate within a given time</td> <td valign="top" align="center" rowspan="1" colspan="1">High</td> <td valign="top" align="center" rowspan="1" colspan="1">Low</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Advancement in EMT</td> <td valign="top" align="center" rowspan="1" colspan="1">Less progressed</td> <td valign="top" align="center" rowspan="1" colspan="1">More progressed</td> </tr> </tbody> </table> </div> <p>Furthermore we decided to check what effect lowered Tβ4 levels had on melanoma cells in terms of invasion potential and EMT progression. We noted that decreased <i>TMSB4X</i> expression caused a reduction in invasion by more than 50% in A375-sh<i>TMSB4X</i> clones compared to control (<a href="#F7">Figure 7A</a>). Simultaneously, we observed transition in cells’ shape in 3D matrix from a elongated one for A375-shscramble cells to a rounded one for A375-sh<i>TMSB4X</i> cells (<a href="#F7">Figure 7B</a>). We noted also decreased expression level of SNAI1/Snail and vimentin at protein level (<a href="#F7">Figures 7C,D</a>) and on mRNA level but only for SNAI1 (<a href="#DS1">Supplementary Figure S15</a>). Expression of other EMT markers remained unchanged (data not shown). Using microarrays data we show that both <i>SNAI1</i> (gene encoding SNAI/Snail) and <i>VIM</i> (gene encoding vimentin) expression levels exhibited positive correlation with <i>TMSB4X</i> expression in human melanoma patients (<a href="#F7">Figure 7E</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 7</span></a></div>   <p>Silencing of <i>TMSB4X</i> expression has an impact on EMT in A375 cells. <b>(A)</b> Invasive potential of A375-sh<i>TMSB4X</i> clones compared to A375-<i>shscr</i> (<i>n</i> = 3). <b>(B)</b> Immunocytochemical analysis of A375-<i>shscr</i> and A375-sh<i>TMSB4X</i> clones embedded in Matrigel<sup>TM</sup> gel. Fluorescently labeled phalloidin and antibodies were used to visualize F-actin and cortactin. <b>(C,D)</b> Western blot analysis of SNAI1/Snail and vimentin levels in A375-<i>shscr</i> and A375-sh<i>TMSB4X</i> clones: <b>(C)</b> densitometry (<i>n</i> = 3) and <b>(D)</b> representative immunoblots. 30 μg of protein was loaded on every lane. Membranes prior to incubation with antibodies were stained with Ponceau S to show equal protein loading on lanes. <b>(E)</b> Microarray analysis of <i>TMSB4X</i> and <i>SNAI1</i> or <i>VIM</i> expression’s correlation in patients samples (<i>n</i> = 168). The significance level was set at <sup>∗</sup><i>P</i> &amp;lt; 0.05, <sup>∗∗</sup><i>P</i> &amp;lt; 0.01, <sup>∗∗∗</sup><i>P</i> &amp;lt; 0.001, and <sup>∗∗∗∗</sup><i>P</i> &amp;lt; 0.0001 (<a href="http://www.oncomine.org/">www.oncomine.org</a>, February 2018, Thermo Fisher Scientific, Ann Arbor, MI, United States).</p>   </div> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Melanoma cells are characterized by high heterogeneity due to its neuroectodermal origin and high plasticity. Cells during melanoma progression undergo phenotype switching, which manifests itself by the acquisition of mesenchymal and stem cells features (<a href="#B24">Kemper et al., 2014</a>). We wanted to elucidate the role of Tβ4 in melanoma cell’s biology, since it was shown that in glioblastoma Tβ4 regulates invasion and stemness (<a href="#B52">Wirsching et al., 2014</a>). In our study, we analyzed microarray data from melanoma patients and melanoma cell lines, which were characterized by a wide range of <i>TMSB4X</i> expression. For more detailed analysis we finally chose two cell lines originating from primary melanomas in vertical growth phase, i.e., WM1341D and A375 cells. To further analyse the functions of Tβ4 we obtained A375 stable clones with silenced <i>TMSB4X</i> expression.</p> <p>Melanocytes and melanoma cells at early progression stages reside at the basement membrane, composition of which is resembled here in a Matrigel<sup>TM</sup> form. Attachment to the basement membrane is loosened with disease progression. At the core of this process lay changes in the expression level of proteins involved in cell’s adhesion (<a href="#B8">Christofori, 2003</a>; <a href="#B43">Saladi et al., 2010</a>). We show here for the first time that Tβ4 is a component of FAs and is able to create complexes with FAs’ structural and signaling proteins, i.e., vinculin, ILK, FAK, pinch 2, α parvin and integrin αVβ3. It has been proven earlier that Tβ4 directly interacts with ILK and forms a functional complex with ILK-PINCH-Parvin in the cardiac cells (<a href="#B4">Bock-Marquette et al., 2004</a>). It is for future studies to check if Tβ4 can directly interact with vinculin, FAK and integrin αVβ3. <a href="#B48">Sosne et al. (2002)</a> presented data showing Tβ4 impact on FAs morphology in human conjunctival epithelial cells. Here, in comparison to WM1341D cells, the A375 cell line, characterized with high <i>TMSB4X</i> expression level, exhibited differences in FAs’ number, morphology and size. The lower number of big-sized FAs in A375 resulted in poorer cells’ spreading and adhesion ability to non-coated and Matrigel<sup>TM</sup>-coated surfaces in comparison to WM1341D cells exhibiting many small-sized FAs. We show for the first time that down-regulation of Tβ4 expression in melanoma cells resulted in an increased number of small-size FAs, which were smaller when compared to control cells. As a consequence A375-sh<i>TMSB4X</i> cells flattened and adhered better to Matrigel<sup>TM</sup>-coated surface. What is intriguing in this story is the fact that HA-Tβ4:vinculin, HA-Tβ4:ILK and HA-Tβ4:FAK complexes were detected in the whole body of analyzed cells differently to the localization of HA-Tβ4:pinch 2, HA-Tβ4:α parvin and HA-Tβ4:integrin αVβ3 complexes. It cannot be excluded that high Tβ4 abundance in cytoplasm prevents vinculin, ILK and FAK from targeting plasma membrane or participation in FAs assembly. In cell’s movement the important role in force generation plays adhesion to ECM proteins. We show that A375 cells characterized by high <i>TMSB4X</i> expression were more advanced in melanoma progression than WM1341D cells. It was supported by greater motility of A375 cells in 2D and 3D Matrigel<sup>TM</sup> matrix. Although the differences between WM1341D and A375 cells moving on Matrigel<sup>TM</sup>-coated surface were more modest when compared to motility of the cells on non-coated surface. WM1341D line is characterized by a higher number of FAs and it seems that for these cells interaction with surface is more important. Their migration speed as well adhesion abilities are increased when surface is coated with Matrigel<sup>TM</sup>. It highlights again the fact that choosing proper conditions for performing experiments is very important, and in some cases plastic, as a non-physiological material, may influence results. We observed also other mesenchymal features such as invadopodia formation and elongated morphology in 3D Matrigel<sup>TM</sup> matrix.</p> <p>Surprisingly, decreased Tβ4 level resulted in increased 2D migration on non-coated surface. Nevertheless, motility on Matrigel<sup>TM</sup>-coated surface of A375 cells with lowered Tβ4 level was impaired. Distances covered by control and A375-sh<i>TMSB4X</i> clones were similar but directionality of A375-sh<i>TMSB4X</i> cells was severely affected. The directional cell movement is based on two main mechanisms, i.e., chemotaxis and haptotaxis. The former depends on the chemical gradient and is quite well known. The latter, responsible for directionality of movement in response to adhesive substrates such as ECM is far less understood. The comparison of migratory behavior of mesenchymal stem cells and fibroblasts revealed that reducing cell’s interactions with the surface lead to haptotactic directional cell movement. Moreover, too strong or too weak binding to surface prevented this type of movement. Additionally, it was shown that cells ability to spread on a surface may play a role in limiting haptotaxis (<a href="#B51">Wen et al., 2015</a>). Here we observed a similar phenomenon. In A375-sh<i>TMSB4X</i> cells we observed increased spreading ability, followed by reduced directionality of movement. In breast cancer cells, this type of migration is based on interaction between α5β3 integrin and actin regulator protein – Mena. Expression of this protein in cancer cells is responsible for directional cell movement via its interaction with α5 integrin subunit and F-actin (<a href="#B38">Oudin et al., 2016</a>). While in melanoma cells integrin αVβ3 plays a role in haptotactic motility (<a href="#B3">Aznavoorian et al., 1996</a>). What’s intriguing we show here that Tβ4 probably forms complexes with αVβ3 integrin. However, further studies are required to understand the role of this polypeptide in directionality of cell migration. We did not observe statistically significant differences between WM1341D and A375 cells in terms of directionality. Cells with lowered Tβ4 level shared some aspects of phenotype with WM1341D cells but not all. For instance they differed in manifestation of EMT markers. Possibly they may also differ in activation of signaling pathways. We have not looked at signaling in tested cells as it is beyond the scope of this study. But it could be interesting to check, in the future, the involvement of two axes playing a role in cells motility directionality, i.e., Rac/Par3, Par6, and Cdc42, LPA receptor/Rab5 or N-cadherin recycling (<a href="#B44">Scarpa and Mayor, 2016</a>). These observations imply as well that surface coating (thin layer of polymerized Matrigel<sup>TM</sup> spread over a coverslip or well bottom) is a key factor in motility assays and finally Tβ4 influences cells protrusion on resembling basement membrane composition. That is why it is not surprising that invasion of cells with lowered Tβ4 level was impaired. Diminished 3D migration is hence plausible, since it was previously shown that increased Tβ4 level in endothelial cells induced expression of matrix metalloproteinases (<a href="#B10">Cierniewski et al., 2007</a>), which are crucial for mesenchymal type of movement in 3D matrix.</p> <p>A375 cells were more advanced in EMT than WM1341D cells, which was corroborated by high vimentin, Zeb1, and SNAI1/Snail (EMT markers) expression. At the same time we observed a higher level of ZO-1 in A375 cells than in WM1341D cells. Recent studies suggest that EMT in cancer cells is a dynamic and reversible process manifested by the spectrum of intermediate states between epithelial and mesenchymal phenotypes. It was shown that tumor cells with "hybrid" epithelial/mesenchymal phenotype are more invasive than fully mesenchymal (<a href="#B39">Pastushenko et al., 2018</a>). Furthermore, ZO-1 (a marker of tight junctions in epithelial cells) and ADAM12 (a marker of mesenchymal cell differentiation) are co-expressed and interact with each other in invasive breast cancer cells. ZO-1 was localized in invadopodia-like structures and decreased expression of this protein suppressed matrix degradation and invasion of cells (<a href="#B12">Dekky et al., 2018</a>). Here, visualization of ZO-1 in tested melanoma cell lines revealed a similar phenomenon. In A375 cells characterized by high invasion potential we did not observe ZO-1 in adherens/tight junctions. However, ZO-1 colocalized with F-actin in cellular protrusions involved in cell movement such as invadopodia and lamellipodia. In our studies we focused on melanoma cells which are tumor-transformed melanocytes, cells of neuroectodermal origin. In these cells the EMT may proceed differently compared to epithelial cells. To our current knowledge, there has not been performed any research about "hybrid" epithelial/mesenchymal phenotype in melanoma. However, there are reports showing highly heterogeneous expression of proteins associated with EMT in series of early passage human melanoma cell lines and melanocytes (<a href="#B25">Kim et al., 2013</a>) as well as in histological studies on human melanoma tissue (<a href="#B47">Shirley et al., 2012</a>). Intermediate EMT states in melanoma cells may play an important role in metastasis and should be more deeply investigated. Though SNAI1/Snail and SNAI2/Slug are considered to play the same biological role in EMT triggering in epithelial tumors (<a href="#B28">Lamouille et al., 2014</a>), it has been shown for melanoma cells that SNAI2/Slug is responsible for melanocytic differentiation and thus suppresses EMT (<a href="#B6">Caramel et al., 2013</a>). That is why it is not surprising that SNAI2/Slug was barely detectable in A375 cells. Lowering of Tb4 level resulted in partially reversed EMT in A375-sh<i>TMSB4X</i> clones, since we observed decreased level of only two EMT markers tested by us.</p> <p>It was shown for hepatoblastoma and oral squamous cell carcinoma that Tβ4 induced EMT transition (<a href="#B13">Fu et al., 2015</a>; <a href="#B18">Hong et al., 2016</a>). Here, we demonstrate that also in melanoma cells there is a connection between <i>TMSB4X</i> expression and EMT progression. In colorectal carcinoma it was demonstrated that Tβ4 triggers EMT transition by ILK upregulation (<a href="#B20">Huang et al., 2007</a>). Recent studies indicate that SNAI1/Snail regulates the expression of αV integrin and reduces the expression of ECM proteins in normal and tumor cells (<a href="#B16">Haraguchi et al., 2008</a>), whereas FAK was identified in embryonic cells undergoing EMT as a regulator of SNAI1/Snail (<a href="#B30">Li X. Y. et al., 2011</a>). These findings indicate the dependence of EMT on adhesion abilities and our results suggest a linking role of Tβ4 in it.</p> <p><a href="#B42">Ray et al. (2017)</a> demonstrated that mesenchymal cells are characterized by large FAs and directional movement, while ameboid cells by small FAs and random migration. In our study we observed that cells with silenced <i>TMSB4X</i> expression were characterized with withdrawal of some properties of mesenchymal cells and dramatic alterations in FAs formation and adhesion abilities. Finally, analysis of patients’ microarray data supported some of our observations, there is a positive correlation between expression of <i>VIM</i> and <i>TMSB4X</i> and <i>SNAI1</i> and <i>TMSB4X</i>. We propose that Tβ4, by direct regulation of FAs formation, alters adhesion abilities of melanoma cells (<a href="#F8">Figure 8</a>). This could lead to acquisition of mesenchymal features resulting in increased metastatic potential. Further studies should focus on unveiling Tβ4’s role in FAs dynamics, elucidation of character of Tβ4 interactions with FAs’ compounds and Tβ4 impact on the parameters studied here in aspect of different ECM proteins. It appears that Tβ4 is an important player in melanoma progression and could be potentially a target for anti-melanoma treatment.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 8</span></a></div>   <p>Influence of Tβ4 on focal adhesions’ formation and its impact on melanoma progression. Increased <i>TMSB4X</i> expression is positively correlated with FAs size, cells’ invasion abilities and expression of EMT markers. On the other hand we observed negative correlation between <i>TMSB4X</i> expression and number of FAs, cell’s adhesion area and ability of studied cells to adhere to ECM (Matrigel<sup>TM</sup>). Motility of cells with low expression of <i>TMSB4X</i> was more depended on Rac1 action than Rho/ROCK1 activity.</p>   </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="celllinesandcultureconditions" title="sec"> <div class="title" tagx="title" title="title">
Cell Lines and Culture Conditions</div> <p>All tested cell lines were of human origin and were authenticated by ATCC<sup>®</sup> in 2017 and 2018. A375 (CRL-1619<sup>TM</sup>) and Hs294T (HTB-140<sup>TM</sup>) cell lines were obtained from the ATCC<sup>®</sup>. WM1341D and WM9 cell lines were bought from Rockland Immunochemicals. The cell culture conditions are described elsewhere (<a href="#B31">Makowiecka et al., 2016</a>).</p> </div> <div class="invasionassay" title="sec"> <div class="title" tagx="title" title="title">
Invasion Assay</div> <p>The assay is described elsewhere (<a href="#B31">Makowiecka et al., 2016</a>). In some experiments 25 μM of a non-selective inhibitor of metalloproteases (GM6001, Santa Cruz Biotechnology Inc.), 10 μM Rac1 inhibitor (NSC 23766, Santa Cruz Biotechnology Inc.) or 10 μM ROCK1 inhibitor (Y-27632, Santa Cruz Biotechnology Inc.) were added to the medium, in which cells were seeded.</p> </div> <div class="cellmigrationassay" title="sec"> <div class="title" tagx="title" title="title">
Cell Migration Assay</div> <p>2D spontaneous migration was monitored using IncuCyte<sup>®</sup> ZOOM System (Essen BioScience). 1000 cells per well were seeded into 96-well IncuCyte<sup>®</sup> ImageLock plates (Essen BioScience) and image sets have been collected every 2 h for 72 h. The single cell distance and trajectory have been analyzed using Manual Tracking plug-in for ImageJ (F. Cordelieres, Institute Curie, Paris, France) for 10 cells from three separate experiments. Directionality was calculated as it is described elsewhere (<a href="#B32">Masuzzo et al., 2017</a>).</p> <p>Wound healing assay was performed as described. Cells were seeded onto Essen Bioscience 96-well ImageLock plates and cultured until confluence was reached. With the use of IncuCyte<sup>®</sup> WoundMaker (96-pin woundmaking tool) unified scratches were made. Collective cell migration was performed with the use of IncuCyte<sup>®</sup> Live Cell Analysis Imaging System (with data collection every 2 h over 48 h) and analyzed with IncuCyte<sup>®</sup> software. Results were presented as a percent of scratch overgrown area in time. To establish if Wound Maker is able to remove Matrigel<sup>TM</sup> from coated surface, a confluent monolayer of cells growing on Matrigel<sup>TM</sup> coated wells was scratched. Cells and surfaces of wells were immunostained using anti-pan-laminin antibodies and fluorescently labeled phalloidin in order to visualize laminin and F-actin.</p> </div> <div class="immunocytochemistryandconfocalmicroscopy" title="sec"> <div class="title" tagx="title" title="title">
Immunocytochemistry and Confocal Microscopy</div> <p>For details see <a href="#B31">Makowiecka et al. (2016)</a>. Used antibodies and their dilutions are listed in <a href="#DS1">Supplementary Table S1</a>, Alexa Fluor<sup>®</sup> 568-labeled phalloidin and Hoechst 33342 were purchased from Invitrogen. Photos and Z-stacks were taken using the Olympus FluoView 500 and Zeiss LSM510 confocal laser scanning microscopes. The number of FAs was semi-automatically calculated using ImageJ software, according to a procedure described elsewhere (<a href="#B19">Horzum et al., 2014</a>). The cell area was analyzed using ImageJ. A super-resolution confocal microscope (Zeiss LSM880 with Airyscan module) was applied to collect some of the data. Immunostaining in 3D matrix was performed on cells seeded into 96-well plates in 2 mg/ml Matrigel<sup>TM</sup> solution in medium without FBS.</p> </div> <div class="westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Western Blot Analysis</div> <p>The procedure describing the preparation of cell lysates and WB is described elsewhere (<a href="#B31">Makowiecka et al., 2016</a>). 30 μg of protein was loaded on every lane. Prior to the blocking step the membranes were stained in 0.2% Ponceau S solution for 10 min for TPA. Used antibodies and their dilutions are listed in <a href="#DS1">Supplementary Table S1</a>. Densitometric analysis of the membranes was performed with the Image Lab 4.0 software (Bio-Rad). From three separate WBs performed for each protein the volume intensity of band (the intensity of luminescence signal in the whole volume of the detected protein band) was measured.</p> </div> <div class="quantitativepolymerasechainreaction(qpcr)" title="sec"> <div class="title" tagx="title" title="title">
Quantitative Polymerase Chain Reaction (qPCR)</div> <p>RNA from cells was isolated using GenElute<sup>TM</sup> Mammalian Total RNA Miniprep Kit (Sigma-Aldrich) and the DNA was digested with DNase I (Sigma-Aldrich) according to manufacturer’s protocol. cDNA was obtained by reversed transcription of 0.5 μg total RNA using High Capacity cDNA Reverse Transcription Kit (Applied Biosystems). For qPCR reaction PowerUp<sup>TM</sup> SYBR<sup>TM</sup> Green Master Mix (Thermo Fisher Scientific) was used according to manufacturer’s protocol. Applied Biosystems StepOne<sup>TM</sup> was used for qPCR performance. For quantification, the samples were normalized against the expression of a housekeeping gene – <i>HPRT1</i> (hypoxanthine phosphoribosyltransferase 1). Primers are listed in <a href="#DS1">Supplementary Table S2</a>.</p> </div> <div class="dnaconstructs" title="sec"> <div class="title" tagx="title" title="title">
DNA Constructs</div> <p>PCR products were cloned into a vector using NEBuilder HiFi DNA Assembly Master Mix (New England Biolabs). Primers and vectors are listed in <a href="#DS1">Supplementary Table S3</a>. Plasmid pLVX-IRES-tdTomato-FlagAkt1 was a gift from Eva Gonzalez (Addgene plasmid # 64831) (<a href="#B23">Kajno et al., 2015</a>) and plasmid p3xHA-C1 (<a href="#B37">Müller et al., 2012</a>). The accuracy of DNA constructs was verified by sequencing. Plasmids for <i>TMSB4X</i> gene silencing were obtained from Sigma-Aldrich – five MISSION pLKO.1-puro <i>TMSB4X</i> shRNA vectors and a control MISSION pLKO.1 puro Non-Target shRNA. Sequences are listed in <a href="#DS1">Supplementary Table S4</a>.</p> </div> <div class="celltransfection" title="sec"> <div class="title" tagx="title" title="title">
Cell Transfection</div> <p>The cells were transfected with expression vectors using 2 mg/ml polyethyleneimine (PEI) solution (Sigma-Aldrich). In order to introduce shRNA vectors into cells Neon Transfection System (Thermo Fisher Scientific) was used. Applied pulse parameters were: 1100 V, 40 ms. Stable cell lines were obtained by 1 μg/ml puromycin selection.</p> </div> <div class="proximityligationassay" title="sec"> <div class="title" tagx="title" title="title">
Proximity Ligation Assay</div> <p>The assay was performed according to manufacturer’s protocol (Sigma-Aldrich). For details see <a href="#B34">Mazur et al. (2016b)</a>. Photos were taken with the use of Zeiss LSM510 confocal microscope. Number of positive signals were quantitatively analyzed in 20 cells per group using ImageJ software equipped with appropriate plugin (<a href="#B19">Horzum et al., 2014</a>). The protocol was modified as follows: Log3D plugins parameters sigma <i>X</i> = 3 and sigma <i>Y</i> = 3 were used and for ANALYZE PARTICLES command following parameters were used: size = 0–infinity and circularity = 0–1.0.</p> </div> <div class="adhesionassay" title="sec"> <div class="title" tagx="title" title="title">
Adhesion Assay</div> <p>The assay was performed as described elsewhere (<a href="#B40">Radwanska et al., 2012</a>). 35000 cells were seeded into 96-well plates with 2 μg/cm<sup>2</sup> Matrigel<sup>TM</sup> or without coating surface in serum-free medium containing 0.5% BSA, 2 mM MgCl<sub>2</sub> and 2 mM CaCl<sub>2</sub>. After incubation (1 h at 37°C) unbound cells were removed by five D-PBS washes and the adhered cells were analyzed with a MTT assay (Sigma-Aldrich) according to manufacturer’s protocol.</p> </div> <div class="datapreprocessingofmicroarraygeneexpression" title="sec"> <div class="title" tagx="title" title="title">
Data Preprocessing of Microarray Gene Expression</div> <p>Gene expression data of melanoma patients on three Affymetrix human genome microarray platforms (U133A, U133A2, U133APlus2) were obtained from Gene Omnibus (GEO). The datasets included: GSE3189 (<i>n</i> = 70), GSE4587 (<i>n</i> = 15), and GSE8401 (<i>n</i> = 83). Each data set was background corrected and normalized using Robust Multichip Average algorithm (RMA) and subsequently compiled and adjusted for batch effect using ComBat (<a href="#B22">Johnson et al., 2007</a>). Probes for <i>TMSB4X</i>, <i>SNAI1</i>, and <i>VIM</i> were selected with Jetset algorithm (<a href="#B29">Li Q. et al., 2011</a>). Preprocessed data sets consisted totally of nine normal skin samples, 22 nevi-like skin samples, 82 from primary, and 55 from metastatic melanoma biopsies. Non-parametric test (Spearman’s rank correlation) was used to estimate the correlation and significance of <i>TMSB4X</i> with other genes co-expression.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analysis</div> <p>All data are given as means ± standard deviations (SD) are representative of at least three independent experiments. Their significance was determined using either the two-tailed, unpaired Student’s <i>t</i>-test or ANOVA (one-way or two-way) with <i>post hoc</i> Tukey HSD were applicable, which were performed in GraphPad Prism 7. The significance level was set at <sup>∗</sup><i>P</i> &amp;lt; 0.05, <sup>∗∗</sup><i>P</i> &amp;lt; 0.01, <sup>∗∗∗</sup><i>P</i> &amp;lt; 0.001, or <sup>∗∗∗∗</sup><i>P</i> &amp;lt; 0.0001. Graphs were plotted in GraphPad Prism 7.</p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Thymosin β4 is a predominantly expressed thymosin β in human melanoma cells and is a constituent of FA. This polypeptide regulates formation of FAs in melanoma cells by influencing their morphology and number. Moreover, it has significant impact on adhesion and protrusive abilities under 2D and 3D conditions of melanoma cells. Tβ4 level decides about progression of EMT of melanoma cells.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>The raw data supporting the conclusions of this article will be made available by the authors, without undue reservation, to any qualified researcher.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>AM partially designed the study, performed the experiments, contributed to the preparation of the figures, and took part in writing the manuscript. NM, EMa, and EMr performed some experiments and contributed to the preparation of the figures. AJM designed the study, gained funding for the study, performed the experiments, contributed to the preparation of the figures, and took part in writing the manuscript. DN discussed the manuscript. All authors reviewed the results and approved the final version of the manuscript.</p> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This work was supported by the Foundation for Polish Science within the HOMING PLUS project (HOMING PLUS/2010-2/8) (granted to AJM) and the National Center for Science within ETIUDA 5 project (2017/24/T/NZ3/00068) (granted to AM). NM thanks the Foundation for Polish Science (FNP) for START scholarship. The funders had no role in the design of the study, in the collection, analyses, or interpretation of data, in the writing of the manuscript, and in the decision to publish the results.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">1</span><p><a href="http://www.oncomine.org">www.oncomine.org</a></p> </div> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fcell.2019.00304/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fcell.2019.00304/full#supplementary-material</a></p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="Data_Sheet_1.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aldridge</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Podrebarac</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenough</span><span class="given-names" tagx="given-names" title="given-names">W. T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiler</span><span class="given-names" tagx="given-names" title="given-names">I. J.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">The use of total protein stains as loading controls: an alternative to high-abundance single-protein controls in semi-quantitative immunoblotting.</span><span class="source" tagx="source" title="source"><i>J. Neurosci. Methods</i></span><span class="volume" tagx="volume" title="volume">172</span><span class="fpage" tagx="fpage" title="fpage">250</span>–<span class="lpage" tagx="lpage" title="lpage">254</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jneumeth.2008.05.003">10.1016/j.jneumeth.2008.05.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18571732">18571732</a></span></span></li> <li tag="ref"><a name="B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Artym</span><span class="given-names" tagx="given-names" title="given-names">V. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seillier-Moiseiwitsch</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">K. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mueller</span><span class="given-names" tagx="given-names" title="given-names">S. C.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Dynamic interactions of cortactin and membrane type 1 matrix metalloproteinase at invadopodia: defining the stages of invadopodia formation and function.</span><span class="source" tagx="source" title="source"><i>Cancer Res.</i></span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">3034</span>–<span class="lpage" tagx="lpage" title="lpage">3043</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-05-2177">10.1158/0008-5472.CAN-05-2177</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16540652">16540652</a></span></span></li> <li tag="ref"><a name="B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aznavoorian</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stracke</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parsons</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McClanahan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liotta</span><span class="given-names" tagx="given-names" title="given-names">L. A.</span></span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">Integrin mediates chemotactic and haptotactic motility in human melanoma cells through different signaling pathways.</span><span class="source" tagx="source" title="source"><i>J. Biol. Chem.</i></span><span class="volume" tagx="volume" title="volume">271</span><span class="fpage" tagx="fpage" title="fpage">3247</span>–<span class="lpage" tagx="lpage" title="lpage">3254</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.271.6.3247">10.1074/jbc.271.6.3247</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8621727">8621727</a></span></span></li> <li tag="ref"><a name="B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bock-Marquette</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saxena</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">M. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimaio</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin beta4 activates integrin-linked kinase and promotes cardiac cell migration, survival and cardiac repair.</span><span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">432</span><span class="fpage" tagx="fpage" title="fpage">466</span>–<span class="lpage" tagx="lpage" title="lpage">472</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature03000">10.1038/nature03000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15565145">15565145</a></span></span></li> <li tag="ref"><a name="B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caers</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hose</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuipers</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bos</span><span class="given-names" tagx="given-names" title="given-names">T. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Valckenborgh</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menu</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin β4 has tumor suppressive effects and its decreased expression results in poor prognosis and decreased survival in multiple myeloma.</span><span class="source" tagx="source" title="source"><i>Haematologica</i></span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">163</span>–<span class="lpage" tagx="lpage" title="lpage">167</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molonc.2015.04.005">10.1016/j.molonc.2015.04.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19833631">19833631</a></span></span></li> <li tag="ref"><a name="B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caramel</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papadogeorgakis</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Browne</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richard</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wierinckx</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">A switch in the expression of embryonic EMT-inducers drives the development of malignant melanoma.</span><span class="source" tagx="source" title="source"><i>Cancer Cell</i></span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">466</span>–<span class="lpage" tagx="lpage" title="lpage">480</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ccr.2013.08.018">10.1016/j.ccr.2013.08.018</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24075834">24075834</a></span></span></li> <li tag="ref"><a name="B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chi</span><span class="given-names" tagx="given-names" title="given-names">L.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">W.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">Y.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">Y.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tai</span><span class="given-names" tagx="given-names" title="given-names">C.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">K.-W.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Global proteomics-based identification and validation of thymosin beta-4 x-linked as a prognostic marker for head and neck squamous cell carcinoma.</span><span class="source" tagx="source" title="source"><i>Sci. Rep.</i></span><span class="volume" tagx="volume" title="volume">7</span>:<span class="issue" tagx="issue" title="issue">9031</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-09539-w">10.1038/s41598-017-09539-w</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28831179">28831179</a></span></span></li> <li tag="ref"><a name="B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christofori</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">New embo member’s review: changing neighbours, changing behaviour: cell adhesion molecule-mediated signalling during tumour progression.</span><span class="source" tagx="source" title="source"><i>EMBO J.</i></span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">2318</span>–<span class="lpage" tagx="lpage" title="lpage">2323</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/emboj/cdg228">10.1093/emboj/cdg228</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12743026">12743026</a></span></span></li> <li tag="ref"><a name="B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cichorek</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wachulska</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stasiewicz</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tymińska</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Skin melanocytes: biology and development.</span><span class="source" tagx="source" title="source"><i>Postępy dermatologii i Alergol.</i></span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">30</span>–<span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"><a href="https://dx.doi.org/10.5114/pdia.2013.33376">10.5114/pdia.2013.33376</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24278043">24278043</a></span></span></li> <li tag="ref"><a name="B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cierniewski</span><span class="given-names" tagx="given-names" title="given-names">C. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malinowski</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bednarek</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cierniewska-Cieslak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Adhesive and proteolytic phenotype of migrating endothelial cells induced by thymosin beta-4.</span><span class="source" tagx="source" title="source"><i>Ann. N. Y. Acad. Sci.</i></span><span class="volume" tagx="volume" title="volume">1112</span><span class="fpage" tagx="fpage" title="fpage">123</span>–<span class="lpage" tagx="lpage" title="lpage">139</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1196/annals.1415.019">10.1196/annals.1415.019</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17495245">17495245</a></span></span></li> <li tag="ref"><a name="B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">E. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golub</span><span class="given-names" tagx="given-names" title="given-names">T. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lander</span><span class="given-names" tagx="given-names" title="given-names">E. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hynes</span><span class="given-names" tagx="given-names" title="given-names">R. O.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Genomic analysis of metastasis reveals an essential role for RhoC.</span><span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">406</span><span class="fpage" tagx="fpage" title="fpage">532</span>–<span class="lpage" tagx="lpage" title="lpage">535</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/35020106">10.1038/35020106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10952316">10952316</a></span></span></li> <li tag="ref"><a name="B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dekky</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruff</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonnier</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Legagneux</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Théret</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Proteomic screening identifies the zonula occludens protein ZO-1 as a new partner for ADAM12 in invadopodia-like structures.</span><span class="source" tagx="source" title="source"><i>Oncotarget</i></span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">21366</span>–<span class="lpage" tagx="lpage" title="lpage">21382</span>. <span class="pub-id"><a href="https://dx.doi.org/10.18632/oncotarget.25106">10.18632/oncotarget.25106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29765546">29765546</a></span></span></li> <li tag="ref"><a name="B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin β4 promotes hepatoblastoma metastasis via the induction of epithelial-mesenchymal transition.</span><span class="source" tagx="source" title="source"><i>Mol. Med. Rep.</i></span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">127</span>–<span class="lpage" tagx="lpage" title="lpage">132</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3892/mmr.2015.3359">10.3892/mmr.2015.3359</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25695679">25695679</a></span></span></li> <li tag="ref"><a name="B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dickerson</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Rational design and characterization of a Rac GTPase-specific small molecule inhibitor.</span><span class="source" tagx="source" title="source"><i>Proc. Natl. Acad. Sci. U.S.A.</i></span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">7618</span>–<span class="lpage" tagx="lpage" title="lpage">7623</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0307512101">10.1073/pnas.0307512101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15128949">15128949</a></span></span></li> <li tag="ref"><a name="B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldstein</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hannappel</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleinman</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin β: actin-sequestering protein moonlights to repair injured tissues.</span><span class="source" tagx="source" title="source"><i>Trends Mol. Med.</i></span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">421</span>–<span class="lpage" tagx="lpage" title="lpage">429</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molmed.2005.07.004">10.1016/j.molmed.2005.07.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16099219">16099219</a></span></span></li> <li tag="ref"><a name="B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haraguchi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okubo</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyashita</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyamoto</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayashi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crotti</span><span class="given-names" tagx="given-names" title="given-names">T. N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Snail regulates cell-matrix adhesion by regulation of the expression of integrins and basement membrane proteins.</span><span class="source" tagx="source" title="source"><i>J. Biol. Chem.</i></span><span class="volume" tagx="volume" title="volume">283</span><span class="fpage" tagx="fpage" title="fpage">23514</span>–<span class="lpage" tagx="lpage" title="lpage">23523</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M801125200">10.1074/jbc.M801125200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18593711">18593711</a></span></span></li> <li tag="ref"><a name="B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">J. H.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuang</span><span class="given-names" tagx="given-names" title="given-names">C.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">C.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shih</span><span class="given-names" tagx="given-names" title="given-names">Y.-R. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">O. K.-S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Internalization is essential for the antiapoptotic effects of exogenous thymosin β-4 on human corneal epithelial cells.</span><span class="source" tagx="source" title="source"><i>Investig. Opthalmol. Vis. Sci.</i></span><span class="volume" tagx="volume" title="volume">48</span>:<span class="issue" tagx="issue" title="issue">27</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1167/iovs.06-0826">10.1167/iovs.06-0826</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17197512">17197512</a></span></span></li> <li tag="ref"><a name="B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">K.-O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.-I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">S.-P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">S.-D.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin β4 induces proliferation, invasion, and epithelial-to-mesenchymal transition of oral squamous cell carcinoma.</span><span class="source" tagx="source" title="source"><i>Amino Acids</i></span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">117</span>–<span class="lpage" tagx="lpage" title="lpage">127</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00726-015-2070-6">10.1007/s00726-015-2070-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26276576">26276576</a></span></span></li> <li tag="ref"><a name="B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horzum</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozdil</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pesen-Okvur</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Step-by-step quantitative analysis of focal adhesions.</span><span class="source" tagx="source" title="source"><i>MethodsX</i></span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">56</span>–<span class="lpage" tagx="lpage" title="lpage">59</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mex.2014.06.004">10.1016/j.mex.2014.06.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26150935">26150935</a></span></span></li> <li tag="ref"><a name="B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">H.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">C.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">M.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.-S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">P.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin beta4 triggers an epithelial-mesenchymal transition in colorectal carcinoma by upregulating integrin-linked kinase.</span><span class="source" tagx="source" title="source"><i>Oncogene</i></span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">2781</span>–<span class="lpage" tagx="lpage" title="lpage">2790</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.onc.1210078">10.1038/sj.onc.1210078</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17072345">17072345</a></span></span></li> <li tag="ref"><a name="B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huttenlocher</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandborg</span><span class="given-names" tagx="given-names" title="given-names">R. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horwitz</span><span class="given-names" tagx="given-names" title="given-names">A. F.</span></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Adhesion in cell migration.</span><span class="source" tagx="source" title="source"><i>Curr. Opin. Cell Biol.</i></span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">697</span>–<span class="lpage" tagx="lpage" title="lpage">706</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0955-0674(95)80112-X">10.1016/0955-0674(95)80112-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8573345">8573345</a></span></span></li> <li tag="ref"><a name="B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">W. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabinovic</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Adjusting batch effects in microarray expression data using empirical Bayes methods.</span><span class="source" tagx="source" title="source"><i>Biostatistics</i></span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">118</span>–<span class="lpage" tagx="lpage" title="lpage">127</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/biostatistics/kxj037">10.1093/biostatistics/kxj037</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632515">16632515</a></span></span></li> <li tag="ref"><a name="B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kajno</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGraw</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Development of a new model system to dissect isoform specific Akt signaling in adipocytes.</span><span class="source" tagx="source" title="source"><i>Biochem. J.</i></span><span class="volume" tagx="volume" title="volume">468</span>(<span class="issue" tagx="issue" title="issue">Pt 3</span>), <span class="fpage" tagx="fpage" title="fpage">425</span>–<span class="lpage" tagx="lpage" title="lpage">434</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1042/BJ20150191">10.1042/BJ20150191</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25856301">25856301</a></span></span></li> <li tag="ref"><a name="B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kemper</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Goeje</span><span class="given-names" tagx="given-names" title="given-names">P. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peeper</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Amerongen</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Phenotype switching: tumor cell plasticity as a resistance mechanism and target for therapy.</span><span class="source" tagx="source" title="source"><i>Cancer Res.</i></span><span class="volume" tagx="volume" title="volume">74</span><span class="fpage" tagx="fpage" title="fpage">5937</span>–<span class="lpage" tagx="lpage" title="lpage">5941</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-14-1174">10.1158/0008-5472.CAN-14-1174</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25320006">25320006</a></span></span></li> <li tag="ref"><a name="B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baguley</span><span class="given-names" tagx="given-names" title="given-names">B. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finlay</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Heterogeneity of expression of epithelial–mesenchymal transition markers in melanocytes and melanoma cell lines.</span><span class="source" tagx="source" title="source"><i>Front. Genet.</i></span><span class="volume" tagx="volume" title="volume">4</span>:<span class="issue" tagx="issue" title="issue">97</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fgene.2013.00097">10.3389/fgene.2013.00097</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23755070">23755070</a></span></span></li> <li tag="ref"><a name="B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krakhmal</span><span class="given-names" tagx="given-names" title="given-names">N. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zavyalova</span><span class="given-names" tagx="given-names" title="given-names">M. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denisov</span><span class="given-names" tagx="given-names" title="given-names">E. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vtorushin</span><span class="given-names" tagx="given-names" title="given-names">S. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perelmuter</span><span class="given-names" tagx="given-names" title="given-names">V. M.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Cancer invasion: patterns and mechanisms.</span><span class="source" tagx="source" title="source"><i>Acta Nat.</i></span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">17</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>.</span></li> <li tag="ref"><a name="B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulesa</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The neural crest and cancer: a developmental spin on melanoma.</span><span class="source" tagx="source" title="source"><i>Cells Tissues Organs</i></span><span class="volume" tagx="volume" title="volume">198</span><span class="fpage" tagx="fpage" title="fpage">12</span>–<span class="lpage" tagx="lpage" title="lpage">21</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1159/000348418">10.1159/000348418</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23774755">23774755</a></span></span></li> <li tag="ref"><a name="B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamouille</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Derynck</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular mechanisms of epithelial-mesenchymal transition.</span><span class="source" tagx="source" title="source"><i>Nat. Rev. Mol. Cell Biol.</i></span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">178</span>–<span class="lpage" tagx="lpage" title="lpage">196</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm3758">10.1038/nrm3758</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24556840">24556840</a></span></span></li> <li tag="ref"><a name="B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birkbak</span><span class="given-names" tagx="given-names" title="given-names">N. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gyorffy</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szallasi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eklund</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Jetset: selecting the optimal microarray probe set to represent a gene.</span><span class="source" tagx="source" title="source"><i>BMC Bioinform.</i></span><span class="volume" tagx="volume" title="volume">12</span>:<span class="issue" tagx="issue" title="issue">474</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-12-474">10.1186/1471-2105-12-474</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22172014">22172014</a></span></span></li> <li tag="ref"><a name="B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rowe</span><span class="given-names" tagx="given-names" title="given-names">R. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlaepfer</span><span class="given-names" tagx="given-names" title="given-names">D. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ilić</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Snail1 controls epithelial–mesenchymal lineage commitment in focal adhesion kinase–null embryonic cells.</span><span class="source" tagx="source" title="source"><i>J. Cell Biol.</i></span><span class="volume" tagx="volume" title="volume">195</span><span class="fpage" tagx="fpage" title="fpage">729</span>–<span class="lpage" tagx="lpage" title="lpage">738</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201105103">10.1083/jcb.201105103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22105351">22105351</a></span></span></li> <li tag="ref"><a name="B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowiecka</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simiczyjew</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Varying effects of EGF, HGF and TGFβ on formation of invadopodia and invasiveness of melanoma cell lines of different origin.</span><span class="source" tagx="source" title="source"><i>Eur. J. Histochem.</i></span><span class="volume" tagx="volume" title="volume">60</span>:<span class="issue" tagx="issue" title="issue">2728</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4081/ejh.2016.2728">10.4081/ejh.2016.2728</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28076931">28076931</a></span></span></li> <li tag="ref"><a name="B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masuzzo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huyck</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simiczyjew</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ampe</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martens</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Troys</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">An end-to-end software solution for the analysis of high-throughput single-cell migration data.</span><span class="source" tagx="source" title="source"><i>Sci. Rep.</i></span><span class="volume" tagx="volume" title="volume">7</span>:<span class="issue" tagx="issue" title="issue">42383</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/srep42383">10.1038/srep42383</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28205527">28205527</a></span></span></li> <li tag="ref"><a name="B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morosan-Puopolo</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowiecka</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malicka-Błaszkiewicz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brand-Saberi</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2016a</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of gelsolin expression pattern in developing chicken embryo reveals high GSN expression level in tissues of neural crest origin.</span><span class="source" tagx="source" title="source"><i>Brain Struct. Funct.</i></span><span class="volume" tagx="volume" title="volume">221</span><span class="fpage" tagx="fpage" title="fpage">515</span>–<span class="lpage" tagx="lpage" title="lpage">534</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00429-014-0923-5">10.1007/s00429-014-0923-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25352156">25352156</a></span></span></li> <li tag="ref"><a name="B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radaszkiewicz</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowiecka</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malicka-Błaszkiewicz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mannherz</span><span class="given-names" tagx="given-names" title="given-names">H. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2016b</span>). <span class="mixed-article-title" title="mixed-article-title">Gelsolin interacts with LamR, hnRNP U, nestin, Arp3 and β-tubulin in human melanoma cells as revealed by immunoprecipitation and mass spectrometry.</span><span class="source" tagx="source" title="source"><i>Eur. J. Cell Biol.</i></span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">26</span>–<span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejcb.2015.11.001">10.1016/j.ejcb.2015.11.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26598132">26598132</a></span></span></li> <li tag="ref"><a name="B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mihm</span><span class="given-names" tagx="given-names" title="given-names">M. C.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Melanoma.</span><span class="source" tagx="source" title="source"><i>N. Engl. J. Med.</i></span><span class="volume" tagx="volume" title="volume">355</span><span class="fpage" tagx="fpage" title="fpage">51</span>–<span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMra052166">10.1056/NEJMra052166</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16822996">16822996</a></span></span></li> <li tag="ref"><a name="B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mollinari</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ricci-Vitiani</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pieri</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucantoni</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rinaldi</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Racaniello</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Downregulation of thymosin β4 in neural progenitor grafts promotes spinal cord regeneration.</span><span class="source" tagx="source" title="source"><i>J. Cell Sci.</i></span><span class="volume" tagx="volume" title="volume">122</span>(<span class="issue" tagx="issue" title="issue">Pt 22</span>), <span class="fpage" tagx="fpage" title="fpage">4195</span>–<span class="lpage" tagx="lpage" title="lpage">4207</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.056895">10.1242/jcs.056895</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19861493">19861493</a></span></span></li> <li tag="ref"><a name="B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behrmann</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diensthuber</span><span class="given-names" tagx="given-names" title="given-names">R. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radke</span><span class="given-names" tagx="given-names" title="given-names">M. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Functional characterization of the human α-cardiac actin mutations Y166C and M305L involved in hypertrophic cardiomyopathy.</span><span class="source" tagx="source" title="source"><i>Cell. Mol. Life Sci.</i></span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">3457</span>–<span class="lpage" tagx="lpage" title="lpage">3479</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00018-012-1030-5">10.1007/s00018-012-1030-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22643837">22643837</a></span></span></li> <li tag="ref"><a name="B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oudin</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonas</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosciuk</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broye</span><span class="given-names" tagx="given-names" title="given-names">L. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guido</span><span class="given-names" tagx="given-names" title="given-names">B. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyckoff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Tumor cell–driven extracellular matrix remodeling drives haptotaxis during metastatic progression.</span><span class="source" tagx="source" title="source"><i>Cancer Disc.</i></span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">516</span>–<span class="lpage" tagx="lpage" title="lpage">531</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1158/2159-8290.CD-15-1183">10.1158/2159-8290.CD-15-1183</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26811325">26811325</a></span></span></li> <li tag="ref"><a name="B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pastushenko</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brisebarre</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sifrim</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fioramonti</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Revenco</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boumahdi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of the tumour transition states occurring during EMT.</span><span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">556</span><span class="fpage" tagx="fpage" title="fpage">463</span>–<span class="lpage" tagx="lpage" title="lpage">468</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-018-0040-3">10.1038/s41586-018-0040-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29670281">29670281</a></span></span></li> <li tag="ref"><a name="B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radwanska</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Litwin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baczynska</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wegrowski</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maquart</span><span class="given-names" tagx="given-names" title="given-names">F.-X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Overexpression of lumican affects the migration of human colon cancer cells through up-regulation of gelsolin and filamentous actin reorganization.</span><span class="source" tagx="source" title="source"><i>Exp. Cell Res.</i></span><span class="volume" tagx="volume" title="volume">318</span><span class="fpage" tagx="fpage" title="fpage">2312</span>–<span class="lpage" tagx="lpage" title="lpage">2323</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.yexcr.2012.07.005">10.1016/j.yexcr.2012.07.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22814255">22814255</a></span></span></li> <li tag="ref"><a name="B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramaswamy</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamayo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rifkin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherjee</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeang</span><span class="given-names" tagx="given-names" title="given-names">C. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Angelo</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Multiclass cancer diagnosis using tumor gene expression signatures.</span><span class="source" tagx="source" title="source"><i>Proc. Natl. Acad. Sci. U.S.A.</i></span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">15149</span>–<span class="lpage" tagx="lpage" title="lpage">15154</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.211566398">10.1073/pnas.211566398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11742071">11742071</a></span></span></li> <li tag="ref"><a name="B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Win</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alford</span><span class="given-names" tagx="given-names" title="given-names">P. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">D.-H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Anisotropic forces from spatially constrained focal adhesions mediate contact guidance directed cell migration.</span><span class="source" tagx="source" title="source"><i>Nat. Commun.</i></span><span class="volume" tagx="volume" title="volume">8</span>:<span class="issue" tagx="issue" title="issue">14923</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms14923">10.1038/ncomms14923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28401884">28401884</a></span></span></li> <li tag="ref"><a name="B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saladi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keenen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marathe</span><span class="given-names" tagx="given-names" title="given-names">H. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chin</span><span class="given-names" tagx="given-names" title="given-names">K.-V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de la Serna</span><span class="given-names" tagx="given-names" title="given-names">I. L.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Modulation of extracellular matrix/adhesion molecule expression by BRG1 is associated with increased melanoma invasiveness.</span><span class="source" tagx="source" title="source"><i>Mol. Cancer</i></span><span class="volume" tagx="volume" title="volume">9</span>:<span class="issue" tagx="issue" title="issue">280</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1476-4598-9-280">10.1186/1476-4598-9-280</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20969766">20969766</a></span></span></li> <li tag="ref"><a name="B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scarpa</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayor</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Collective cell migration in development.</span><span class="source" tagx="source" title="source"><i>J. Cell Biol.</i></span><span class="volume" tagx="volume" title="volume">212</span><span class="fpage" tagx="fpage" title="fpage">143</span>–<span class="lpage" tagx="lpage" title="lpage">155</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201508047">10.1083/jcb.201508047</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26783298">26783298</a></span></span></li> <li tag="ref"><a name="B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schultz</span><span class="given-names" tagx="given-names" title="given-names">G. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strelow</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stern</span><span class="given-names" tagx="given-names" title="given-names">G. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chegini</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">M. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galardy</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Treatment of alkali-injured rabbit corneas with a synthetic inhibitor of matrix metalloproteinases.</span><span class="source" tagx="source" title="source"><i>Invest. Ophthalmol. Vis. Sci.</i></span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">3325</span>–<span class="lpage" tagx="lpage" title="lpage">3331</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1385350">1385350</a></span></span></li> <li tag="ref"><a name="B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shekhar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlier</span><span class="given-names" tagx="given-names" title="given-names">M.-F.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Enhanced depolymerization of actin filaments by ADF/Cofilin and monomer funneling by capping protein cooperate to accelerate barbed-end growth.</span><span class="source" tagx="source" title="source"><i>Curr. Biol.</i></span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">1990.e5</span>–<span class="lpage" tagx="lpage" title="lpage">1998.e5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2017.05.036">10.1016/j.cub.2017.05.036</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28625780">28625780</a></span></span></li> <li tag="ref"><a name="B47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirley</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greene</span><span class="given-names" tagx="given-names" title="given-names">V. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duncan</span><span class="given-names" tagx="given-names" title="given-names">L. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres Cabala</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grimm</span><span class="given-names" tagx="given-names" title="given-names">E. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusewitt</span><span class="given-names" tagx="given-names" title="given-names">D. F.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Slug expression during melanoma progression.</span><span class="source" tagx="source" title="source"><i>Am. J. Pathol.</i></span><span class="volume" tagx="volume" title="volume">180</span><span class="fpage" tagx="fpage" title="fpage">2479</span>–<span class="lpage" tagx="lpage" title="lpage">2489</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajpath.2012.02.014">10.1016/j.ajpath.2012.02.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22503751">22503751</a></span></span></li> <li tag="ref"><a name="B48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sosne</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hafeez</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenberry</span><span class="given-names" tagx="given-names" title="given-names">A. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurpakus-Wheater</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin beta4 promotes human conjunctival epithelial cell migration.</span><span class="source" tagx="source" title="source"><i>Curr. Eye Res.</i></span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">268</span>–<span class="lpage" tagx="lpage" title="lpage">273</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1076/ceyr.24.4.268.8414">10.1076/ceyr.24.4.268.8414</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12324865">12324865</a></span></span></li> <li tag="ref"><a name="B49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sribenja</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wongkham</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wongkham</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Roles and mechanisms of β-thymosins in cell migration and cancer metastasis: an update.</span><span class="source" tagx="source" title="source"><i>Cancer Invest.</i></span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">103</span>–<span class="lpage" tagx="lpage" title="lpage">110</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3109/07357907.2012.756111">10.3109/07357907.2012.756111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23320791">23320791</a></span></span></li> <li tag="ref"><a name="B50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uehata</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishizaki</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satoh</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ono</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawahara</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morishita</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Calcium sensitization of smooth muscle mediated by a Rho-associated protein kinase in hypertension.</span><span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">389</span><span class="fpage" tagx="fpage" title="fpage">990</span>–<span class="lpage" tagx="lpage" title="lpage">994</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/40187">10.1038/40187</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9353125">9353125</a></span></span></li> <li tag="ref"><a name="B51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor-Weiner</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuhrmann</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karpiak</span><span class="given-names" tagx="given-names" title="given-names">J. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almutairi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Haptotaxis is cell type specific and limited by substrate adhesiveness.</span><span class="source" tagx="source" title="source"><i>Cell. Mol. Bioeng.</i></span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">530</span>–<span class="lpage" tagx="lpage" title="lpage">542</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s12195-015-0398-3">10.1007/s12195-015-0398-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26640598">26640598</a></span></span></li> <li tag="ref"><a name="B52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wirsching</span><span class="given-names" tagx="given-names" title="given-names">H. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Florea</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frei</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krayenbühl</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasenbach</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Thymosin beta 4 gene silencing decreases stemness and invasiveness in glioblastoma.</span><span class="source" tagx="source" title="source"><i>Brain</i></span><span class="volume" tagx="volume" title="volume">137</span><span class="fpage" tagx="fpage" title="fpage">433</span>–<span class="lpage" tagx="lpage" title="lpage">448</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/brain/awt333">10.1093/brain/awt333</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24355709">24355709</a></span></span></li> <li tag="ref"><a name="B53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaidel-Bar</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Addadi</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geiger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Hierarchical assembly of cell–matrix adhesion complexes.</span><span class="source" tagx="source" title="source"><i>Biochem. Soc. Trans.</i></span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">416</span>–<span class="lpage" tagx="lpage" title="lpage">420</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1042/BST0320416">10.1042/BST0320416</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15157150">15157150</a></span></span></li> </ul> </div> </div>  </body></html>