<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6041326">6041326</a></span><span class="publisher-id" title="publisher-id">28717</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-018-28717-y">10.1038/s41598-018-28717-y</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Myosin X is required for efficient melanoblast migration and melanoma initiation and metastasis</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokuo</span><span class="given-names" tagx="given-names" title="given-names">Hiroshi</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">tokuo@bu.edu</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhawan</span><span class="given-names" tagx="given-names" title="given-names">Jag</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coluccio</span><span class="given-names" tagx="given-names" title="given-names">Lynne M.</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0367 5222</span><span class="institution-id" title="institution-id">grid.475010.7</span><span class="institution" title="institution">Department of Physiology &amp;amp; Biophysics, </span><span class="institution" title="institution">Boston University School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0367 5222</span><span class="institution-id" title="institution-id">grid.475010.7</span><span class="institution" title="institution">Department of Dermatology, </span><span class="institution" title="institution">Boston University School of Medicine, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-7-7</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-7-7</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">10449</span><span class="history" title="history"><span class="received" title="received">received: 2017-11-23</span><span class="accepted" title="accepted">accepted: 2018-6-15</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Myosin X (Myo10), an actin-associated molecular motor, has a clear role in filopodia induction and cell migration <i>in vitro</i>, but its role <i>in vivo</i> in mammals is not well understood. Here, we investigate the role of Myo10 in melanocyte lineage and melanoma induction. We found that Myo10 knockout (Myo10KO) mice exhibit a white spot on their belly caused by reduced melanoblast migration. Myo10KO mice crossed with available mice that conditionally express in melanocytes the BRAF<sup>V600E</sup> mutation combined with Pten silencing exhibited reduced melanoma development and metastasis, which extended medial survival time. Knockdown of Myo10 (Myo10kd) in B16F1 mouse melanoma cell lines decreased lung colonization after tail-vein injection. Myo10kd also inhibited long protrusion (LP) formation by reducing the transportation of its cargo molecule vasodilator-stimulated phosphoprotein (VASP) to the leading edge of migrating cells. These findings provide the first genetic evidence for the involvement of Myo10 not only in melanoblast migration, but also in melanoma development and metastasis.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/100007265</span><span class="institution" title="institution">Boston University School of Medicine (BUSM)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">1UL1TR001430</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokuo</span><span class="given-names" tagx="given-names" title="given-names">Hiroshi</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/100000057</span><span class="institution" title="institution">U.S. Department of Health &amp;amp; Human Services | NIH | National Institute of General Medical Sciences (NIGMS)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">GM111615</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coluccio</span><span class="given-names" tagx="given-names" title="given-names">Lynne M.</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Myo10 is one of several actin-based motor molecules in the myosin superfamily. It has a motor or head domain with a nucleotide-binding site and an actin-binding site, an IQ or neck domain, which binds three molecules of calmodulin, and a C-terminal tail domain that has a single α-helix (SAH) region followed by a coiled-coil region presumably involved in dimerization, 3 PEST sequences, which are rich in proline, glutamate, serine and threonine and confer sensitivity to certain proteases, 3 pleckstrin homology (PH) domains, a Myosin Tail Homology 4 (MyTH4) domain, which binds to microtubules, and a band 4.1, Ezrin, Radixin, Merlin (FERM) domain<sup><a href="#CR1">1</a></sup>.</p> <p id="Par3">Myo10 localizes to the tips of filopodia, actin-rich finger-like protrusions found at the leading edge of cells<sup><a href="#CR2">2</a></sup> and believed to be involved in numerous cellular processes including cell migration, wound healing, adhesion to the extracellular matrix, guidance towards chemoattractants, neuronal growth-cone path finding and embryonic development<sup><a href="#CR3">3</a></sup>. Presumably, Myo10 exists chiefly as a folded monomer in the cytoplasm; however, contact with phosphatidylinositol (3,4,5)-triphosphate (PIP<sub>3</sub>) induces its unfolding and dimerization to act as a processive motor able to transport cargo<sup><a href="#CR4">4</a></sup>. Previous studies have shown that Myo10 promotes filopodia formation by delivering specific cargos to the cell periphery, e.g., Mena/VASP, which binds to the tail domain<sup><a href="#CR5">5</a>,<a href="#CR6">6</a></sup>, and integrins<sup><a href="#CR7">7</a></sup>, netrin receptors<sup><a href="#CR8">8</a></sup>, and VE-cadherin<sup><a href="#CR9">9</a></sup>, which bind to the FERM domain. Also, the PH domain of Myo10 binds to PIP<sub>3</sub> and targets Myo10 to the cell membrane of the leading edge for filopodia formation<sup><a href="#CR10">10</a></sup>.</p> <p id="Par4">Myo10 is expressed ubiquitously in vertebrate tissues<sup><a href="#CR2">2</a>,<a href="#CR11">11</a>,<a href="#CR12">12</a></sup>, and its functional importance has been found in a variety of cells/tissues, e.g., phagocytosis cup formation in leukocytes<sup><a href="#CR13">13</a></sup>, nuclear anchoring and spindle assembly in <i>Xenopus laevis</i><sup><a href="#CR14">14</a></sup>, orientation of the mitotic spindle in cultured cells<sup><a href="#CR15">15</a></sup>, endothelial cell migration and angiogenesis<sup><a href="#CR16">16</a></sup>, axonal path-finding regulated by netrin<sup><a href="#CR8">8</a></sup>, cranial neural crest cell migration in <i>Xenopus</i><sup><a href="#CR17">17</a>,<a href="#CR18">18</a></sup>, sealing zone patterning<sup><a href="#CR19">19</a></sup> and differentiation<sup><a href="#CR20">20</a></sup> in osteoclasts, tight junction barrier formation in polarized epithelial cells<sup><a href="#CR21">21</a></sup>, axon outgrowth in cortical neurons<sup><a href="#CR22">22</a></sup>, formation of <i>Shigella</i>-induced membrane protrusions and cell-to-cell spreading<sup><a href="#CR23">23</a></sup>, dendritic spine development by VASP trafficking<sup><a href="#CR6">6</a></sup>, and leukocyte extravasation through cultured endothelial cells<sup><a href="#CR24">24</a></sup>. Most recently, reduced filopodia formation was observed in isolated macrophages from Myo10KO mice<sup><a href="#CR25">25</a></sup> and in retinal angiogenesis in the eyes of Myo10KO mice<sup><a href="#CR26">26</a></sup>. In spite of these studies, the physiological and pathological functions of Myo10 in murine or human organs is still largely unknown.</p> <p id="Par5">Recently, information regarding the role of Myo10 in cancer has been accumulating, e.g., the expression level of Myo10 correlates with aggressiveness and metastasis of breast cancer<sup><a href="#CR27">27</a>,<a href="#CR28">28</a></sup>; Myo10 is up-regulated and participates in lung adenocarcinoma metastasis<sup><a href="#CR29">29</a></sup>; Myo10 was found as a target of microRNA-124, which was suppressed in aggressive non-small cell lung cancer<sup><a href="#CR30">30</a></sup>; Myo10 was also a target of microRNA-340, which inhibits the metastasis of breast cancer<sup><a href="#CR31">31</a></sup>; MYO10 expression was higher in prostate cancer than in normal tissue; and Myo10kd reduced the migration speed and directional persistence of a prostate cancer cell line<sup><a href="#CR32">32</a></sup>.</p> <p id="Par6">In this study, we found that Myo10KO mice (<i>Myo10</i><sup><i>tm2/tm2</i></sup>) exhibit a white belly spot and syndactyly, striking defects that were also reported with other types of Myo10KO mice (<i>Myo10</i><sup><i>tm1d/tm1d</i></sup>, <i>Myo10</i><sup><i>tm1a/tm1a</i></sup>, and <i>Myo10</i><sup><i>m1J/m1J</i></sup>) while this paper was in review<sup><a href="#CR26">26</a></sup>. Our focus is on the white belly spot (which is observed in all of the Myo10KO mice) for the following reasons: (i) Natural mouse mutants with white belly spots have been instrumental to understanding melanoblast (precursors to melanocytes) migration. Studies of <i>lethal spotting</i> (<i>Edn-3</i>), <i>piebald</i> (<i>Ednrb</i>), <i>dominant white spotting</i> (<i>Kit</i>), and <i>microphthalmia</i> (<i>Mitf</i>) mutants led to the identification of genes and proteins essential for melanoblast migration. For example, the <i>endothelin-3</i> (<i>Edn-3</i>) ligand, produced by the ectoderm, binds to the <i>endothelin receptor-B</i> (<i>Ednrb</i>), which is expressed in melanoblasts. Signaling through this complex is essential for migrating melanoblasts to reach their final destination<sup><a href="#CR33">33</a></sup>. The kit ligand produced in the dermomyotome, dermis, and hair follicles binds to the c-kit receptor tyrosine kinase, which is expressed by melanoblasts. Only melanoblasts expressing the c-kit and Ednrb receptors respond to the attractive and repulsive signals controlling cell migration<sup><a href="#CR34">34</a></sup>. The <i>Mitf</i> (microphthalmia-associated transcription factor) gene and the <i>M-Mitf</i> isoform (specific to the melanocyte lineage) play a key role in the biology of melanoblasts and melanocytes<sup><a href="#CR35">35</a></sup>. The function of this transcription factor is not only to ensure specification and survival of the melanocyte lineage, but also to contribute to their migration through regulation of numerous target genes. Among them a transcription factor, Slug (<i>Snai2</i>), is directly regulated by M-Mitf and implicated in melanoblast migration<sup><a href="#CR36">36</a></sup>. The transcription factors <i>Pax3</i> and <i>Sox10</i> synergistically regulate <i>M-Mitf</i> expression, and heterozygous mutations in these genes give rise to a white-spotted phenotype<sup><a href="#CR37">37</a>,<a href="#CR38">38</a></sup>. Thus, Myo10 may have an important role in melanoblast migration resulting in a white belly spot phenotype. (ii) Recent findings suggest that migration of melanoblasts during development is highly correlative with melanoma metastasis. The imprint of past migratory behavior of melanoblasts has been suggested to confer a propensity of primary melanomas to establish remote metastases<sup><a href="#CR39">39</a>–<a href="#CR41">41</a></sup>. Nonetheless, nothing is known about the function of Myo10 in melanoma.</p> <p id="Par7">Here, we demonstrate that Myo10KO mice exhibit a white belly spot and show that in cultured melanoblasts Myo10kd decreases the formation of LPs and cell migration. These data indicate that the white belly spot in Myo10KO mice may be a consequence of reduced migration of melanoblasts in the absence of Myo10. We also show that Myo10KO delayed onset and development of melanoma, and reduced metastasis in a mouse melanoma model. We present evidence that Myo10kd in a melanoma cell line greatly impaired pseudopod formation and lung colonization in mice following tail injection. We also determined that elevated expression of the <i>MYO10</i> gene was associated with inferior survival outcomes in melanoma patients and Myo10 expression increased in human melanoma. Together, we uncovered for the first time that Myo10-induced protrusions drive melanoblast migration and melanoma initiation and metastasis.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="genomicstructureofmyo10tm2(komp)wtsimiceandmyo10expression" title="sec"> <div class="title" tagx="title" title="title">
Genomic structure of <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> mice and Myo10 expression</div> <p id="Par8">To assess Myo10 function <i>in vivo</i>, <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> (Myo10KO) mice were obtained from the Wellcome Trust Sanger Research Institute, and the mouse line was maintained in a C57BL/6 N background. KO was performed by insertion of the L1L2_Bact_P cassette, which created a deletion of 9595 nucleotides starting at position 25743874 and ending at position 25753469 of chromosome 15 (Genome Build GRCm38) (Fig. <a href="#Fig1">1A</a>). This deletion resulted in the removal of exon 19, which is functionally critical, and a part of intron 19, but the alternative transcription start sites and first exons specific for <i>headless Myo10</i> remained intact. The precise structure of the <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> allele is shown in Fig. <a href="#Fig1">1A</a>. The presence and zygosity of the <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> allele were verified by PCR using genomic DNA isolated from tail snips (Fig. <a href="#Fig1">1B</a>). An image of the whole gel is shown in Supplementary Fig. <a href="#MOESM1">S1</a>. To confirm the KO effect on protein expression, first we performed immunoblot analysis of whole kidney lysates from one-month old mice and found that only the control mice expressed full-length Myo10 (Fig. <a href="#Fig1">1C</a>, arrow with F). Several extra bands were detected below the molecular weight of full-length Myo10 in wild-type samples only, which we speculate to be degradation products of full-length Myo10 (Fig. <a href="#Fig1">1C</a>, bracket with D). A band (~50 KDa) was found in both genotypes, which is likely a non-specific signal (Fig. <a href="#Fig1">1C</a>, asterisk). Next, to check for expression of headless Myo10, we performed immunoblot analysis of whole brain lysates from one-month old mice, which reportedly express both full-length and headless Myo10<sup><a href="#CR42">42</a></sup>. As observed in kidney, full-length Myo10 was detected in brain samples of control mice; however, headless Myo10 was found in brain extracts from both control (+/+) and Myo10KO (<i>tm2/tm2</i>) mice (Fig. <a href="#Fig1">1D</a>, arrow with H). A nonspecific band was detected in both genotypes at ~50 KDa as observed in kidney lysates (Fig. <a href="#Fig1">1D</a>, asterisk). These results indicate that the <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> allele nullifies full-length Myo10, but not headless Myo10. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Genomic structure of <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> mice and expression of Myo10. (<b>A</b>) Allele structure of mouse <i>full-length Myo10</i> (NM_019472.2) and three alternative splicing variants called "<i>headless Myo10</i>" (XM_006520025.3, XM_006520024.3 and XM_006520026.3) (top). Vertical bars on the gene map represent exons, and purple bars represent first exons specific for <i>headless Myo10</i>. The lesion from exon 18 to 20 is magnified and primers for genotyping are indicated as red arrows. Genomic structure of <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> (Myo10KO) mice (bottom). The <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> allele is shown to illustrate the mutated <i>Myo10</i> gene in which the <i>lacZ</i> and <i>neomycin (neo)</i> expression cassette (L1L2_Bact_P) was inserted to create a deletion 9595 nucleotides in size starting at position 25743874 and ending at position 25753469 of chromosome 15 (Genome Build GRCm38). This deletion results in the removal of functionally critical exon 19 and a portion of intron 19. This cassette contained the splice acceptor of mouse Engrailed 2 exon 2 (En2 SA), an internal ribosome entry sequence (IRES) to initiate <i>lacZ</i> translation, and SV40 polyadenylation (pA) to terminate transcription after the <i>lacZ</i> gene. The <i>neo</i> gene is driven by the human beta actin promoter (hBactP) and contains its own pA. Additionally, L1L2_Bact_P is flanked by FRT (flippase recognition target) sites, and hBactP-neo-pA is flanked by loxP sites. Red arrows indicate the positions of primers for genotyping. The specific details and the complete DNA sequence of the <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> allele are available on the International Mouse Phenotyping Consortium (IMPC) website (<a href="http://www.mousephenotype.org/data/search/allele2?kw=Myo10">http://www.mousephenotype.org/data/search/allele2?kw=Myo10</a>). (<b>B</b>) Characterization of Myo10KO mice by genomic PCR. Genomic DNA samples isolated from WT mice (+/+), mice heterozygous for the Myo10 mutant allele (+/<i>tm2</i>), and homozygous mice (<i>tm2/tm2</i>) were used as a template for PCR with primer pairs Myo10F/Myo10R, Myo10F/MutR, and LacZF/LacZR. PCR with the primer pair Myo10F/Myo10R produced a 400-bp band (top panel) only in mice that carried at least one copy of the WT allele; PCR with the primer pair Myo10F/MutR resulted in a 295-bp product (middle panel),; and PCR with the primer pair LacZF/LacZR resulted in a 108-bp band (bottom panel) in mice that carried at least one copy of the mutant allele. An image of the whole gel is shown in Supplementary Fig. <a href="#MOESM1">S1</a>. (<b>C</b>,<b>D</b>) Immunoblots of tissue samples from WT (+/+) and Myo10 KO (<i>tm2/tm2</i>) mice. Kidney (<b>C</b>) and brain (<b>D</b>) samples were probed using anti-Myo10 antibody (right panels). The probed membranes were stained with CBB (left panels). The asterisk indicates a non-specific band.</p>   </div> <p /> </div> <div class="myo10-deficientmicehaveawhitebellyspotandsyndactyly" title="sec"> <div class="title" tagx="title" title="title">
Myo10-deficient mice have a white belly spot and syndactyly</div> <p id="Par9">When heterozygous <i>Myo10</i><sup>+<i>/tm2</i></sup> mice were crossed with one another, we obtained the following genotypes: 30 <i>Myo10</i><sup>+<i>/</i>+</sup> pups, 80 <i>Myo10</i><sup>+<i>/tm2</i></sup> pups, and 16 <i>Myo10</i><sup><i>tm2/tm2</i></sup> pups. Thus, <i>Myo10</i><sup><i>tm2/tm2</i></sup> pups were obtained ~50% as often as wild-type homozygote births, almost the same as the <i>Myo10</i><sup><i>tm1d/tm1d</i></sup> and <i>Myo10</i><sup><i>tm1a/tm1a</i></sup> pups, which were obtained 40% and 45%, respectively, as often as wild-type homozygote births<sup><a href="#CR26">26</a></sup> (Table <a href="#Tab1">1</a>). We observed a white belly phenotype with 100% penetrance in <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice (Fig. <a href="#Fig2">2A</a>), the same rate as in <i>Myo10</i><sup><i>tm1a/tm1a</i></sup>, <i>Myo10</i><sup><i>tm1d/tm1d</i></sup>, and <i>Myo10</i><sup><i>m1J/m1J</i></sup> mice. Less than 5% of the <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice also had white spots along the dorsal midline. Histological analysis of black areas of the <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice at P14 showed melanocytes in hair follicles (not shown), whereas the number of melanocytes present in white patches of the <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice was reduced by 87% vs. dark patches on control mice (Fig. <a href="#Fig2">2B,C</a>). These results suggest that Myo10 deficiency in the melanocyte lineage causes a pigmentation defect due to the lack of melanocytes in the ventral areas distal to their origin of migration. We also found that <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice have webbed digits, a form of syndactyly, in the forelimbs or hind legs with a 72% penetrance rate (Fig. <a href="#Fig2">2D</a>). This rate is slightly higher than that observed in <i>Myo10</i><sup><i>tm1a/tm1a</i></sup> (50%), <i>Myo10</i><sup><i>tm1d/tm1d</i></sup> (50%), and <i>Myo10</i><sup><i>m1J/m1J</i></sup> (43%) mice. We found that only two in 126 (1.6%) <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice had a kinked tail, similar to <i>Myo10</i><sup><i>m1J/m1J</i></sup> mice, which did not have kinked tails, but a much lower rate than the 75% of <i>Myo10</i><sup><i>tm1a/tm1a</i></sup> and the 50% of <i>Myo10</i><sup><i>tm1d/tm1d</i></sup> mice that exhibited a visible kink near the tip of the tail. Although we did not examine eye abnormalities, the International Mouse Phenotyping Consortium (IMPC) reports that 13/14 <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice had persistent hyaloid vasculature (bilateral in 8/13 cases), 9/14 had fused cornea and lens (7/8 presented in both eyes), 9/14 had corneal opacity (7/9 presented in both eyes), and 3/14 had an abnormal pupil light response. These eye phenotypes are similar to the 11/14 of <i>Myo10</i><sup><i>tm1d/tm1d</i></sup>, 5/5 of <i>Myo10</i><sup><i>tm1a/tm1a</i></sup>, and 5/5 of <i>Myo10</i><sup><i>m1J/m1J</i></sup> mice that had persistent hyaloid vasculature that was pigmented and bilateral<sup><a href="#CR26">26</a></sup>. Other phenotypes associated with the <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> allele including fused phalanges are described on the IMPC website (<a href="http://www.mousephenotype.org/data/genes/MGI:107716">http://www.mousephenotype.org/data/genes/MGI:107716</a>). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Comparison of <i>Myo10</i><sup><i>tm2/tm2</i></sup> with <i>Myo10</i><sup><i>tm1d/tm1d</i></sup>, <i>Myo10</i><sup><i>tm1a/tm1a</i></sup><i>, and Myo10</i><sup><i>m1J/m1J</i></sup>.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Traits</th> <th><i>Myo10</i><sup><i>tm1d/tm1d</i></sup><i>and Myo10</i><sup><i>tm1a/tm1a</i></sup><br /><br />Data from Heimsath <i>et al</i>.<sup><a href="#CR26">26</a></sup></th> <th><i>Myo10</i><sup><i>m1J/m1J</i></sup><br /><br />Data from Heimsath <i>et al</i>.<sup><a href="#CR26">26</a></sup></th> <th><i>Myo10</i><sup><i>tm2/tm2</i></sup><br /><br />Data from this report and the IMPC (<a href="http://www.mousephenotype.org/data/genes/MGI:107716">http://www.mousephenotype.org/data/genes/MGI:107716</a>)</th> </tr> </thead> <tbody> <tr> <td>Protein Expression</td> <td>(−) full-length Myo10<br /><br />(−) headless Myo10</td> <td>(−) full-length Myo10<br /><br />(−) headless Myo10</td> <td>(−) full-length Myo10<br /><br />(+) headless Myo10</td> </tr> <tr> <td>Birth Rate<br /><br />(Homozygous/wild type)</td> <td><i>tm1d</i>; 40%<br /><br /><i>tm1a</i>; 45%</td> <td>29%</td> <td>50%</td> </tr> <tr> <td>White Belly Spot in Homozygote</td> <td>100%</td> <td>100%</td> <td>100%</td> </tr> <tr> <td>Syndactyly in Homozygote</td> <td>50%</td> <td>43%</td> <td>72%</td> </tr> <tr> <td>Kinked Tail in Homozygote</td> <td><i>tm1d</i>; 75%<br /><br /><i>tm1a</i>; 50%</td> <td>0%</td> <td>1.6%</td> </tr> <tr> <td>Eye Abnormalities</td> <td>11/14 <i>Myo10</i><sup><i>tm1d/tm1d</i></sup> mice and 5/5 <i>Myo10</i><sup><i>tm1a/tm1a</i></sup> mice had persistent hyaloid vasculature that was pigmented and bilateral</td> <td>5/5 <i>Myo10</i><sup><i>m1J/m1J</i></sup> mice had persistent hyaloid vasculature that was pigmented and bilateral</td> <td>13/14 <i>Myo10</i><sup><i>tm2/tm2</i></sup> mice had persistent hyaloid vasculature (bilateral in 8/13 cases), 9/14 had fused cornea and lens (7/8 presented in both eyes), 9/14 had corneal opacity (7/9 presented in both eyes), and 3/14 had an abnormal pupil light response</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Myo10-deficient mice have a white belly spot and syndactyly. (<b>A</b>) Representative coat color of a Myo10KO mouse (right) with control littermate (left, Myo10WT). (<b>B</b>) Staining of hair follicle bulbs in the abdominal skin from wild-type and Myo10KO mice for Trp2, a melanocyte marker. Scale bar is 50 μm. (<b>C</b>) Quantification of Trp2-positive melanocytes. The average number of Trp2-positive melanocytes in hair bulbs in the white belly spots of Myo10KO mice was 1.2 ± 0.7 (mean ± SEM, 30 hair bulbs per mouse × 3 mice, <i>n</i> = 90), which was significantly lower (<i>P</i> = 0.0004) than 9.2 ± 2.1 (<i>n</i> = 90) for Myo10WT mice. (<b>D</b>) Representative image of fused toes (arrows), syndactyly, in Myo10KO mouse.</p>   </div> <p /> </div> <div class="myo10kdinthemousemelanoblastcelllinemelb-areducedthenumberoflpsandmigration" title="sec"> <div class="title" tagx="title" title="title">
Myo10kd in the mouse melanoblast cell line Melb-a reduced the number of LPs and migration</div> <p id="Par10">The white belly spot phenotype is caused by reduced migration of melanoblasts during embryonic development, and proper formation of LPs (also referred to as long pseudopods) is the driving force of melanoblast migration <i>in vivo</i><sup><a href="#CR43">43</a></sup>. To determine the functional consequences of <i>MYO10</i> loss in melanoblasts, we used shRNA to silence <i>MYO10</i> in Melb-a cells. The Melb-a cell line was established from mouse melanoblasts, and Melb-a cells retain the character of melanoblasts versus that of mature melanocytes or melanoma cells<sup><a href="#CR44">44</a></sup>. Myo10kd was confirmed by immunoblotting of cell lysates from control or Myo10-specific shRNA transfected Melb-a cell lines (Fig. <a href="#Fig3">3A</a>). Images of the whole immunoblots are shown in Supplementary Fig. <a href="#MOESM1">S2</a>. Myo10kd did not disturb the rate of cell proliferation on culture dishes (not shown). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Myo10kd in the mouse melanoblast cell line Melb-a reduced the number of cells with LPs and cell migration. (<b>A</b>) Immunoblotting of Myo10 in cell lysates from Melb-a cells transfected with control or a Myo10-specific shRNA. Tubulin served as a loading control. Whole blots are shown in Supplementary Fig. <a href="#MOESM1">S2</a>. (<b>B</b>) Representative phenotype of live Melb-a cells cultured on thick collagen matrices. Control cells show LPs, elongated structures protruding in the direction of cell migration (left panel, arrows), whereas Myo10kd reduced the number of cells with LPs (right panel). Scale bar is 50 μm. (<b>C</b>) Melb-a cells in B were fixed and stained for F-actin with fluorescent phalloidin. Control cells have long F-actin-rich protrusions (left panel; arrows), but Myo10 kd cells do not (right panel). Scale bar is 10 μm. (<b>D</b>) The number of cells with LPs was counted manually and divided by the total number of cells. The percentage of Myo10kd cells with LPs was 11.1% ± 2.7 (mean ± SEM, 4 different random fields × 3 experiments, <i>n</i> = 12, shRNA-1), 12.2% ± 3.9 (<i>n</i> = 12, shRNA-2), and 14.1% ± 3.1 (<i>n</i> = 12, shRNA-3), which were significantly lower (<i>P</i> &amp;lt; 0.0001) than 48.4% ± 3.8 (<i>n</i> = 12) in control shRNA cells. (E) Quantification of migrating cells in transwell migration assays. The average number of Myo10kd cells that migrated onto the lower surface of the filter was 33.8 ± 6.0 (mean ± SEM, 4 different random fields × 3 experiments, <i>n</i> = 12, shRNA-1), 35.6 ± 7.0 (<i>n</i> = 12, shRNA-2), and 46.9 ± 7.6 (<i>n</i> = 12, shRNA-3), which were significantly fewer (<i>P</i> = 0.0002; shRNA-1, <i>P</i> = 0.0004; shRNA-2 and <i>P</i> = 0.0061; shRNA-3) than 81.9 ± 8.6 (<i>n</i> = 12) in control shRNA cells.</p>   </div> <p /> <p id="Par11">First, to investigate the effect of Myo10kd on the cellular phenotype of melanoblasts, cells were cultured on thick collagen matrices, and live-cell images were captured. The number of cells with LPs (Fig. <a href="#Fig3">3B</a>, arrows) was 77% lower in Myo10 shRNA-1 cells, 73% lower in Myo10 shRNA-2 cells, and 71% lower in Myo10 shRNA-3 cells vs. cells transfected with scrambled shRNA (Fig. <a href="#Fig3">3B,D</a>). To investigate the F-actin structure in these cells, Melb-a cells were fixed and stained with fluorescent phalloidin (Fig. <a href="#Fig3">3C</a>). In control cells, F-actin accumulated in LPs (Fig. <a href="#Fig3">3C</a>, arrows), but in Myo10kd cells, F-actin was diffusely localized on the surface and in fine spikes, and no LPs were observed. These results indicate that Myo10 is important for induction of LPs in melanoblasts. Next, we performed transwell migration assays and found that the number of cells successfully migrating through the membrane pores was 59% lower in Myo10 shRNA-1 cells, 57% lower in Myo10 shRNA-2 cells, and 43% lower in Myo10 shRNA-3 cells than in cells transfected with scrambled shRNA (Fig. <a href="#Fig3">3E</a>). These results are consistent with a previous report showing that Myo10kd inhibits transwell migration of breast cancer cells<sup><a href="#CR27">27</a></sup>. These results suggest a link between LP formation and migration of melanoblasts and that Myo10 may play important roles in these processes.</p> </div> <div class="lossofmyo10delayedonsetanddevelopmentofmelanoma" title="sec"> <div class="title" tagx="title" title="title">
Loss of Myo10 delayed onset and development of melanoma</div> <p id="Par12">We speculate that Myo10 plays an important role not only in melanoblast migration but also in melanoma metastasis. Several key genetic lesions governing human melanoma initiation and progression have been identified. Among them, the earliest and most common is a point mutation (T1779A) in the <i>Braf</i> proto-oncogene, which is detected in ~65% of affected individuals<sup><a href="#CR45">45</a></sup>. <i>Braf</i><sup><i>T1799A</i></sup> encodes Braf<sup>V600E</sup> (BrafCA), a constitutively active serine kinase that elicits sustained activation of the Braf/MEK1/2/ERK1/2 MAP kinase pathway. BrafCA stimulates constitutive cell signaling, growth factor-independent proliferation, and transformation of immortalized melanocytes, allowing these cells to grow as tumors in nude mice. Although it contributes to the aberrant pathophysiological characteristics of the melanoma cell, oncogenic activation of <i>Braf</i> in melanocytes is insufficient for full malignant conversion. Hence, progression to malignant melanoma is invariably accompanied by silencing of one or more tumor suppressor genes, most commonly <i>Pten</i> or <i>CDKN2A</i><sup><a href="#CR46">46</a>–<a href="#CR48">48</a></sup>. The combination of <i>Braf</i><sup><i>T1799A</i></sup> and <i>Pten</i> silencing is common in human melanoma (~20%)<sup><a href="#CR49">49</a></sup>, and genetic evidence for <i>Braf</i> and <i>Pten</i> cooperation in this disease has been obtained by mouse experiments. Upon induction of BrafCA in melanocyte-specific expression, mice develop benign melanocytic hyperplastic growths that fail to progress to melanoma<sup><a href="#CR50">50</a></sup>. By contrast, expression of BrafCA combined with Pten tumor suppressor gene silencing elicits development of melanoma and remote metastasis with 100% penetrance<sup><a href="#CR50">50</a></sup>.</p> <p id="Par13">We therefore crossed Myo10KO mice with a genetically modified model of metastatic malignant melanoma, Tyr-CreER/PtenKO/BrafCA mice<sup><a href="#CR50">50</a></sup>, to assess whether Myo10 is important for primary melanoma development and/or metastasis. Local administration of 4-hydroxytamoxifen (4-HT) to adult animals allowed tumor formation to be monitored at the point of application. Tyr-CreER/PtenKO/BrafCA/Myo10WT mice developed primary melanomas with a similar penetrance and latency to that previously described<sup><a href="#CR50">50</a></sup> (Fig. <a href="#Fig4">4A</a>). To investigate the expression of Myo10 in these induced melanoma, we performed immunoblot analysis of whole tumor lysates and found that only the WT mice expressed full-length Myo10, and neither WT nor mutant mice expressed headless Myo10 (Fig. <a href="#Fig4">4B</a>). Tumor cells in both genotypes were positive for S100, a marker with excellent sensitivity for pigmented and amelanotic melanoma (Fig. <a href="#Fig4">4C</a>). Onset of tumors was delayed by Myo10 inactivation, and the average tumor volumes were also markedly reduced from day 30 in Tyr-CreER/PtenKO/BrafCA/Myo10KO mice (Fig. <a href="#Fig4">4A,D</a>). These findings provide the first genetic evidence for the involvement of <i>MYO10</i> in melanoma development as reflected by the longer disease latency and reduced tumor size in the Myo10KO mice. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Loss of Myo10 delayed the onset and development of melanoma. (<b>A</b>) Representative images of induced melanoma in Myo10WT and Myo10KO mice under the genetic background of Tyr-CreER/PtenKO/BrafCA at the indicated number of days after local 4-HT administration. (<b>B</b>) Immunoblotting of tumor samples from WT (+/+) and Myo10 KO (<i>tm2/tm2</i>) mice with Tyr-CreER/PtenKO/BrafCA genotype. No expression of full-length Myo10 (F) was observed in the KO tumor. The bands labeled with D are breakdown products; the band labeled with an asterisk is non-specific. Membrane staining with CBB (left panel) after probing showed similar protein loading. (<b>C</b>) Detection of S100-positive cells in tumors 40 days (Myo10 WT) and 85 days (Myo10KO) after 4-HT induction. Tumor cells were S100-positive in both genotypes. Inserts are 2.5 × enlargements of boxed areas. Arrows point to S100-positive cells. Scale bar is 200 μm. (<b>D</b>) Tumor growth curves for Tyr-CreER/PtenKO/BrafCA/Myo10WT (<i>n</i> = 10) and Tyr-CreER/PtenKO/BrafCA/Myo10KO (<i>n</i> = 11) genotypes. Average tumor size (mm<sup>3</sup>, mean ± SEM; <i>y</i> axis) was plotted as a function of the number of days after 4-HT treatment (<i>x</i> axis). Average tumor size in Myo10KO mice was 6 ± 5 mm<sup>3</sup>, which was significantly smaller (<i>P</i> = 0.0045) than the average tumor size of 419 ± 122 mm<sup>3</sup> in Myo10WT mice at day 30; 28 ± 20 mm<sup>3</sup> in KO and 1045 ± 157 mm<sup>3</sup> in WT (<i>P</i> &amp;lt; 0.0001) at day 40; and 75 ± 45 mm<sup>3</sup> in KO and 1750 ± 358 mm<sup>3</sup> in WT (<i>P</i> = 0.0003) at day 50.</p>   </div> <p /> </div> <div class="geneticinactivationofmyo10inhibitedmelanomametastasisandincreasedsurvival" title="sec"> <div class="title" tagx="title" title="title">
Genetic inactivation of <i>MYO10</i> inhibited melanoma metastasis and increased survival</div> <p id="Par14">To study Myo10 function in melanoma metastasis, we performed necropsy of Tyr-CreER/PtenKO/BrafCA/Myo10KO and Tyr-CreER/PtenKO/BrafCA/Myo10WT mice when tumor volumes reached ~1,000 cm<sup>3</sup>. Mice that received 4-HT on the left side of the dorsal skin showed evidence that the melanoma had spread into the left inguinal lymph node based on the swelling and pigmented appearance observed in Tyr-CreER/PtenKO/BrafCA/Myo10WT mice (Fig. <a href="#Fig5">5A</a>). The number of S100-positive cells in lymph nodes was 68% fewer in the Tyr-CreER/PtenKO/BrafCA/Myo10KO animals than in the Tyr-CreER/PtenKO/BrafCA/Myo10WT animals (Fig. <a href="#Fig5">5A,B</a>). The medial survival time of Tyr-CreER/PtenKO/BrafCA mice was extended ~264% (from 35.0 to 92.5 days) by inactivation of <i>MYO10</i> (Fig. <a href="#Fig5">5C</a>). Log-rank (Mantel-Cox) test of survival plots revealed a significant difference between the Myo10WT and KO genotypes under the genetic background of Tyr-CreER/PtenKO/BrafCA, evidence of Myo10’s role in melanoma progression. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Genetic inactivation of <i>MYO10</i> inhibited melanoma metastasis and increased survival. (<b>A</b>) Representative images of inguinal lymph nodes and immunohistochemistry of S100 in Tyr-CreER/PtenKO/BrafCA/Myo10KO and Tyr-CreER/PtenKO/BrafCA/Myo10WT mice. Scale bar is 100 μm. (<b>B</b>) Quantification of S100-positive melanocytes in regional lymph nodes. The average number of S100-positive cells in lymph nodes in Tyr-CreER/PtenKO/BrafCA/Myo10KO mice was 18.0 ± 6.3 (mean ± SEM, 6 random fields × 3 slices × 10 mice, <i>n</i> = 180), which was significantly fewer (<i>P</i> = 0.0003) than 55.9 ± 8.5 (<i>n</i> = 198) in lymph nodes from Tyr-CreER/PtenKO/BrafCA/Myo10WT mice. (<b>C</b>) Kaplan-Meier survival curve of mice with the indicated genotypes, Myo10WT (<i>n</i> = 10) or KO (<i>n</i> = 11) under the genetic background of Tyr-CreER/PtenKO/BrafCA following local administration of 4-HT. Log-rank (Mantel-Cox) test of survival plots revealed a significant difference (<i>P</i> &amp;lt; 0.0001) between the Myo10 WT and KO genotypes.</p>   </div> <p /> </div> <div class="myo10kdinb16f1cellsreducedlpformationin3dcollagenmatricesandlungcolonizationafterintravenousinjection" title="sec"> <div class="title" tagx="title" title="title">
Myo10kd in B16F1 cells reduced LP formation in 3D collagen matrices and lung colonization after intravenous injection</div> <p id="Par15">To determine whether Myo10KO in melanoma cells is responsible for the observed inhibition of metastasis in mice, we employed B16F1 mouse melanoma cells in lung colony formation assays. The migration of B16F1cells in 3D collagen lattices is accompanied by dynamic changes in the morphology of a prominent actin-rich protrusion at the leading edge<sup><a href="#CR51">51</a></sup>. First, we established B16F1 Myo10kd cell lines and confirmed that the knockdown effect was &amp;gt;90% as compared to B16F1 cells expressing control shRNA (Fig. <a href="#Fig6">6A</a>). Images of the whole immunoblots are shown in Supplementary Fig. <a href="#MOESM1">S3</a>. Myo10kd did not change the proliferation rate of B16F1 cells cultured in complete medium on culture dishes (data not shown). The number of cells with LPs (Fig. <a href="#Fig6">6B</a>, arrows) cultured in collagen matrix was 47% fewer in Myo10 shRNA-1 cells than in control shRNA cells (Fig. <a href="#Fig6">6B,C</a>). Next, we analyzed lung colony formation after intravenous injection of control shRNA and Myo10 shRNA-1 cells into the tail vein of mice and found that the number of lung colonies after two weeks was ~55% fewer in Myo10kd vs. control cells (Fig. <a href="#Fig6">6D,E</a>). These results suggest that attenuation of LP formation by Myo10kd inhibited the extravasation of B16F1 cells and colony formation in the lung and indicate that the observed inhibition of melanoma metastasis in the previous experiments was due to reduced Myo10 expression in melanoma. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Myo10kd in B16F1 cells reduced the number of cells with LPs in 3D collagen matrices and lung colonization after intravenous injection. (<b>A</b>) Immunoblots of Myo10 in cell lysates from B16F1 control cells or B16F1 cells transfected with Myo10-specific shRNA. Tubulin served as a loading control. Whole blots are shown in Supplementary Fig. <a href="#MOESM1">S3</a>. (<b>B</b>) Representative phenotype of cells cultured in 3D collagen matrices. Control shRNA cells showed LPs (upper panel, arrows), whereas Myo10kd (Myo10 shRNA-1, lower panel) inhibited protrusion formation. Scale bar is 50 μm. (<b>C</b>) The average percentage of cells with LPs in Myo10 shRNA-1 cells was 19.1% ± 3.4 (mean ± SEM, 4 different random fields × 3 experiments, <i>n</i> = 12), which was significantly lower (<i>P</i> = 0.0062) than 36.3% ± 4.6 (<i>n</i> = 12) in control shRNA cells. (<b>D</b>) Representative images of lungs isolated 2 weeks after intravenous injection of B16F1 control or Myo10kd cells. Arrows indicate colonies of B16F1 cells. (<b>E</b>) Quantification of the number of lung colonies detected per animal. The average number of lung colonies in mice injected with Myo10 shRNA-1 cells was 28.0 ± 8.6 (mean ± SEM, <i>n</i> = 10), which was significantly fewer (<i>P</i> = 0.015) than 62.0 ± 9.3 (<i>n</i> = 10) found in lungs of mice injected with control shRNA cells.</p>   </div> <p /> </div> <div class="myo10localizedatthetipsofprotrusionsin3dmigratingb16f1cells,andmyo10kdreducedvasplocalizationatthesesites" title="sec"> <div class="title" tagx="title" title="title">
Myo10 localized at the tips of protrusions in 3D migrating B16F1 cells, and Myo10kd reduced VASP localization at these sites</div> <p id="Par16">Migrating B16F1 cells in collagen lattices form pseudopods with multiple finger-like termini<sup><a href="#CR51">51</a></sup>. The kinetics of the leading edge and the traction of collagen fibers suggested a new perspective on 3D migration of melanoma cells in which the dynamics of filopodia mediate force generation and translocation of the leading edge. At the front of the pseudopod, multiple filopodia transiently bind to and pull on collagen fibrils, causing a reversible change in the position of both the filopodia and fibrils<sup><a href="#CR51">51</a></sup>. Thus, we hypothesized that Myo10 localizes at the tips of filopodia projecting from pseudopods and mediates membrane dynamics at the leading edge of migrating melanoma cells. To investigate this possibility, first we localized Myo10 in B16F1 cells cultured in 3D collagen matrices. Figure <a href="#Fig7">7A</a> shows the typical phenotype of a B16F1 cell migrating near the bottom of a 3D collagen matrix. A broad (~5 μm) and long (&amp;gt;20 μm) protrusion, or pseudopod, was observed at the front of the migrating cell and consisted of two parts: a shaft (left of the dashed line, Fig. <a href="#Fig7">7A</a>) and front (right of the dashed line). Several (sometimes dozens) filopodia were observed at the front of the pseudopod. Longer and stable ones were filled with F-actin (Fig. <a href="#Fig7">7A</a>, arrows), but it was difficult to detect F-actin in shorter ones (Fig. <a href="#Fig7">7A</a>, arrow heads). Filopodia existed also at the shaft of the pseudopods. Myo10 was accumulated at the tips of almost all of the filopodia. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Myo10 localized at the front of pseudopods in 3D migrating cells, and Myo10kd reduced VASP localization at these sites. (<b>A</b>) An image showing the typical phenotype of B16F1 cells that migrate near the bottom of a 3D collagen matrix. Cells were fixed and stained with anti-Myo10 antibody (green), fluorescent phalloidin (red), and DAPI (blue). A broad (~5 μm) and long (&amp;gt;20 μm) pseudopod defines the direction of the migrating cell and consisted of two parts: a shaft (left of the dotted line) and a front (right of the dotted line). Several filopodia (sometimes dozens) extended from the front of the pseudopod. Longer and stable filopodia were filled with F-actin (arrows), but F-actin was difficult to detect in the shorter ones (arrowheads). Myo10 accumulated at the tips of almost all the filopodia. Scale bar is 10 μm. (<b>B</b>) A representative time-lapse image of a B16F1 cell that has migrated into the 3D collagen matrix. GFP-Myo10 (left) and mCherry-Lifeact (right) were co-expressed in B16F1 cells and monitored in 5 min intervals over a 30 min period, and movies were generated from these images (Supplementary Movies <a href="#MOESM2">S1</a> and <a href="#MOESM3">S2</a>). The lower panel shows a magnified view (box, 10′00″) of the front of the pseudopod at 10 min. White arrows indicate Myo10 punctae in the shaft of the pseudopod; arrowheads indicate Myo10 at the front of the pseudopod; and blue arrows indicate where Myo10 appears to form a continuous track due to its movement back and forth along actin filaments in filopodia within a single movie frame. (<b>C</b>) Knockdown of Myo10 reduced VASP localization at the front of pseudopods. B16F1 cells transfected with control shRNA (upper panels) or Myo10 shRNA-1 (lower panels) were cultured in 3D collagen matrices, and stained with anti-VASP antibody (green) and fluorescent phalloidin (red). Scale bar is 5 μm.</p>   </div> <p /> <p id="Par17">Next, we transfected GFP-Myo10 and mCherry-Lifeact to monitor Myo10 dynamics and actin accumulation, respectively, in 3D migrating live cells. GFP signals were diffuse in the cytoplasm and more highly concentrated in puncta, and mCherry signals were also diffuse in the cytoplasm and accumulated in multiple finger-like termini at the leading edge (Fig. <a href="#Fig7">7B</a>, upper panel). In a magnified view of a pseudopod excerpted from the area in the time lapse images denoted by a dotted box (Fig. <a href="#Fig7">7B</a>, lower panel), two different types of GFP-positive puncta were detected: small ones (arrowheads) that localized at the front of the pseudopod, and larger ones (arrows) that localized at the shaft of the pseudopod. The smaller puncta located mainly at the tips of the filopodia projecting from the front of the pseudopod, and these filopodia dynamically elongated and retracted (Supplementary Movies <a href="#MOESM2">S1</a> and <a href="#MOESM3">S2</a>) so that several of them appeared to form a continuous line because they were captured several times in one time frame (lower panel, blue arrows). Bigger puncta behind the leading edge were stable and did not change location during the recorded period. These results suggest that Myo10 induces filopodia projecting from pseudopods and mediates dynamics at the front of pseudopods in 3D migrating melanoma cells.</p> <p id="Par18">In addition to actin, finger-like protrusions at the leading edge contained foci of VASP<sup><a href="#CR51">51</a></sup>. VASP, which exhibits anti-capping activity during actin assembly, is a major inducer of filopodia and lamellipodia<sup><a href="#CR52">52</a>,<a href="#CR53">53</a></sup>. We previously reported that Myo10 transports VASP to the tips of filopodia<sup><a href="#CR5">5</a></sup>. From these results, we speculated that Myo10 transports VASP to the tips of filopodia protruding from pseudopods in melanoma cells. To study this possibility, VASP localization at the tips of 3D pseudopods was compared between control (upper panels) and Myo10 shRNA-1 (lower panels) transfected B16F1 cells (Fig. <a href="#Fig7">7C</a>). In control cells, VASP localized in foci at the tips and along the shafts of filopodia (arrows) and at the tips of membrane ruffles (arrowheads). On the other hand, pseudopods rarely formed in Myo10kd cells. Fewer filopodia were evident and little or no VASP localized in filopodia (arrow) and ruffles (arrowheads), although some VASP were accumulated in short spikes (green arrows). The localization of VASP in focal adhesions was the same as in control cells (blue arrows). These results are consistent with a model in which Myo10 transports VASP to the leading edge of 3D migrating melanoma cells, which accelerates actin polymerization required for invasion.</p> </div> <div class="myo10expressionincreasedinhumanmelanomaanddecreasedinmelanomawithtp53mutation" title="sec"> <div class="title" tagx="title" title="title"><i>MYO10</i> expression increased in human melanoma and decreased in melanoma with <i>TP53</i> mutation</div> <p id="Par19">To investigate the role of Myo10 in human melanoma, we first analyzed the expression of <i>MYO10</i> mRNA in human skin, benign nevi, and melanoma samples by <i>in silico</i> analysis of Oncomine datasets. <i>MYO10</i> mRNA was significantly higher in melanoma than in normal skin in the Riker melanoma data set (Fig. <a href="#Fig8">8A</a>) and in the Talantov melanoma data set (Fig. <a href="#Fig8">8B</a>). Also, <i>MYO10</i> mRNA expression was significantly higher in melanoma than in nevi in the Talantov melanoma data set (Fig. <a href="#Fig8">8B</a>). These results suggest that <i>MYO10</i> mRNA expression is higher in human melanoma than in normal skin and nevi. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>Increased <i>MYO10</i> expression in human melanoma and relationship with <i>TP53</i> mutation. Oncomine analysis of <i>MYO10</i> expression in malignant melanoma compared with normal tissues. (<b>A</b>) Box-and-whisker plots show that <i>MYO10</i> mRNA expression was significantly higher (<i>P</i> = 0.014) in melanoma (<i>n</i> = 45) than in normal skin (<i>n</i> = 7) in the Riker melanoma data set GSE7553. (<b>B</b>) <i>MYO10</i> mRNA expression was significantly higher in nevi (<i>n</i> = 18) than in normal skin (<i>n</i> = 7, <i>P</i> = 0.0022), in melanoma (<i>n</i> = 45) than in nevi (P &amp;lt; 0.001), and in melanoma than in normal skin (P &amp;lt; 0.001) in the Talantov melanoma data set GSE3189. (<b>C</b>) <i>MYO10</i> expression and survival (Kaplan-Meier curves) were analyzed in 163 patients with malignant melanoma (TCGA-SKCM). Log-rank test <i>P</i> value, <i>P</i> = 0.00096. (<b>D</b>) <i>MYO10</i> mRNA expression was significantly lower (<i>P</i> &amp;lt; 0.0001) in samples with <i>TP53</i> mutants (Mut, <i>n</i> = 55) than in wild type (WT, <i>n</i> = 405) in the skin cutaneous melanoma data set (TCGA, Provisional) downloaded from the cBioPortal database.</p>   </div> <p /> <p id="Par20">Next, we used the PROGgeneV2 prognostic biomarker identification tool<sup><a href="#CR54">54</a></sup> to study the implications of <i>MYO10</i> gene expression on overall survival of 163 melanoma patients in The Cancer Genome Atlas-Skin Cutaneous Melanoma (TCGA-SKCM) data sets. Using median gene expression values as bifurcation points, Cox proportional hazards analyses showed that elevated expression of the <i>MYO10</i> gene was associated with inferior survival outcomes (Fig. <a href="#Fig8">8C</a>). Kaplan-Meier plots indicated significant segregation in survival outcomes for patients with high vs. low <i>MYO10</i> expression (hazard ratio, HR = 1.43; confidence interval, CI, 1.16–1.77; <i>P</i> = 0.00096).</p> <p id="Par21">A recent study showed that <i>MYO10</i> mRNA expression was significantly higher in breast tumors with mutant p53<sup><a href="#CR28">28</a></sup>. To determine if this same correlation exists for melanoma, we compared <i>MYO10</i> mRNA expression relative to the <i>TP53</i> status in melanoma samples. Interestingly, <i>MYO10</i> mRNA expression was significantly lower in samples expressing <i>TP53</i> mutations vs. wild-type <i>TP53</i>, the opposite of that observed in breast cancers (Fig. <a href="#Fig8">8D</a>). In prostate cancer cell lines, there was no link between Myo10 overexpression and expression of mutant p53<sup><a href="#CR32">32</a></sup>. These results suggest that regulation of <i>MYO10</i> mRNA expression is different among melanoma, breast cancer, and prostate cancer.</p> </div> <div class="myo10expressionishigherinmelanomathaninnevi" title="sec"> <div class="title" tagx="title" title="title">
Myo10 expression is higher in melanoma than in nevi</div> <p id="Par22">To confirm Myo10 protein expression in human melanoma, we stained randomly selected tissue samples of diagnosed benign nevi (<i>n</i> = 5) and malignant melanoma (<i>n</i> = 9) archived in the SkinPath Laboratory of Boston University School of Medicine. Figure <a href="#Fig9">9A</a> shows examples of low Myo10 expression in nevus (left panel) and high expression in melanoma (right panel) by immunostaining. There was a tendency for higher Myo10 expression in melanoma (<i>n</i> = 5, high; 3, medium; and 1, low) vs. nevi (<i>n</i> = 2, medium; and 3, low), but the sample number was not large enough to determine whether there was a statistical difference in expression levels between the two groups. Thus, we used a tissue microarray (TMA; ME1004e; US Biomax, Inc.) consisting of both melanomas and nevi on one slide allowing for staining of all cores at the same time (Fig. <a href="#Fig9">9B</a>). We performed immunohistochemistry on the TMA with anti-Myo10 antibody and anti-S100 antibody, and the intensity of Myo10 staining was captured only in S100-positive regions and calculated per pixel. Box-and-whiskers plots demonstrated that Myo10 expression was significantly higher in melanoma lesions than in nevi, consistent with the data of <i>MYO10</i> mRNA expression (Fig. <a href="#Fig8">8B</a>). These results suggest that Myo10 expression level correlates with melanocytic neoplastic progression in humans and overall survival of melanoma patients. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>Increased Myo10 expression in human melanoma. (<b>A</b>) Examples of Myo10 immunostaining in human nevus (left) and melanoma (right) samples. Scale bar is 200 μm. (<b>B</b>) Expression of Myo10 was studied in a tissue microarray consisting of primary and metastatic melanomas and nevi. Examples of Myo10 and S100 staining in melanoma and nevus are shown. (<b>C</b>) Box-and-whiskers plots demonstrated differences in Myo10 expression in the nevi (<i>n</i> = 10) and melanoma (<i>n</i> = 43) samples. Myo10 expression was significantly higher in melanoma lesions than in nevi (<i>P</i> = 0.0011).</p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="myo10kophenotype" title="sec"> <div class="title" tagx="title" title="title">
Myo10KO phenotype</div> <p id="Par23">We show that the Myo10KO mice used in this study, <i>Myo10</i><sup><i>tm2/tm2</i></sup>, express headless Myo10 in brain, but do not express full-length Myo10 in brain or kidney. These results are consistent with the allele structure of <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup>, which has intact alternative transcription start sites and first exons specific for <i>headless Myo10</i>. After this study was submitted, a new article describing Myo10KO mice generated with a different KO strategy was published<sup><a href="#CR26">26</a></sup>. In that study, the <i>Myo10</i><sup><i>tm1a</i></sup> and <i>Myo10</i><sup><i>tm1d</i></sup> mutations target exon 27, and the <i>Myo10</i><sup><i>m1J</i></sup> mutation deletes 8 bp in exon 25. None of those three KO mice (<i>Myo10</i><sup><i>tm1a/tm1a</i></sup>, <i>Myo10</i><sup><i>tm1d/tm1d</i></sup>, and <i>Myo10</i><sup><i>m1J/m1J</i></sup>) expresses either full-length <i>or</i> headless Myo10 in brain. Interestingly, we found that the phenotype of the total KO mouse (i.e., knockout of both <i>full-length Myo10</i> and <i>headless Myo10</i>; <i>Myo10</i><sup><i>tm1a/tm1a</i></sup>, <i>Myo10</i><sup><i>tm1d/tm1d</i></sup>, and <i>Myo10</i><sup><i>m1J/m1J</i></sup>) and the <i>full-length Myo10</i> only KO mouse (<i>Myo10</i><sup><i>tm2/tm2</i></sup>) is almost the same. This conclusion is reasonable because most tissues express only full-length Myo10, whereas neural stem cells, neurons, and astrocytes express both full-length and headless Myo10<sup><a href="#CR22">22</a>,<a href="#CR42">42</a></sup>. Thus, the difference between the total KO and the full-length only Myo10 KO may be restricted to these neuronal cells. One idea is that headless Myo10 is a negative regulator of full-length Myo10<sup><a href="#CR22">22</a></sup>, but whether headless Myo10 plays other functions is largely unknown. It is plausible that new insight into the function of headless Myo10 <i>in vivo</i> will result from comparion studies of neuronal function in <i>total Myo10</i> KO mice vs. <i>full-length Myo10 only</i> KO mice.</p> </div> <div class="myo10functioninmelanoblastmigration" title="sec"> <div class="title" tagx="title" title="title">
Myo10 function in melanoblast migration</div> <p id="Par24">We found that Myo10KO induces white belly spots in mice and that Myo10kd in melanoblast cells reduces LP formation, evidence that Myo10 controls melanoblast migration by mediating LP formation. Rac1 positively regulates the frequency of initiation of LPs in melanoblasts, which control migration speed and directional plasticity<sup><a href="#CR43">43</a></sup>; and Rac1KO in melanocytes results in a white belly spot phenotype. Loss of the Rac exchange factor P-Rex1 also causes white belly spots and inhibition of melanoma metastasis<sup><a href="#CR55">55</a></sup>. Thus, the PIP<sub>3</sub>/P-Rex1/Rac1/Scar/WAVE/Arp2/3 signaling pathway is speculated to be a major pathway for LP-driven migration of the melanocyte lineage<sup><a href="#CR55">55</a></sup>. A recent study demonstrates that Cdc42 null mice display a severe loss of pigmentation, a white patch covering half to most of the underside, and that melanoblasts show cell-cycle progression, migration, and cytokinesis defects<sup><a href="#CR56">56</a></sup>. Cdc42KO melanoblasts migrate slowly and inefficiently in the epidermis with bulky elongated LPs. These results suggest that proper length and dynamics of LPs are important for normal migration of melanoblasts <i>in vivo</i>. Recent findings suggest that PIP<sub>3</sub> binding is important for the localization of Myo10 at the plasma membrane in order to induce filopodia formation<sup><a href="#CR10">10</a></sup>. Also, Myo10 is thought to exist as a monomer whose tail is folded back onto the head/neck region in an auto-inhibited "off" state<sup><a href="#CR4">4</a></sup>. Binding to PIP<sub>3</sub> induces unfolding of the molecule and promotes dimerization of Myo10 to form an active motor able to promote actin fiber convergence at the cell periphery, which initiates filopodia formation<sup><a href="#CR57">57</a></sup>. Thus, Myo10 is possibly involved in the PIP<sub>3</sub>-mediated signaling pathway that induces LPs in melanoblasts.</p> <p id="Par25">Melanoblasts, which derive from neural crest cells (NCCs), are precursors to melanocytes. NCCs first differentiate into founder melanoblasts in the trunk region on mouse embryonic day 8.5–9.5 (E8.5-E9.5). These founder melanoblasts proliferate in a wedge-shaped area (migration staging area) in the ectoderm, the neural tube and the somites. On E10.5 these melanoblast precursors, which are still proliferating, begin to migrate in between the somites and the ectoderm along a rostro-caudal temporal gradient. Some of the migrating melanoblasts begin to cross the basement membrane (BM) separating the dermis from the epidermis on E11.5. The remaining cells continue to migrate dorsolaterally in the dermis until E15.5 while continuing to cross the BM<sup><a href="#CR58">58</a></sup>. Loss of Myo10 reduces the number of filopodia and junction formation in cultured kidney epithelial cells<sup><a href="#CR21">21</a></sup>. There are no available data about Myo10 function in junction formation in skin keratinocytes, but it is possible that melanoblast migration is regulated by the surrounding epidermal cells.</p> </div> <div class="myo10inmelanomadevelopmentandmetastasis" title="sec"> <div class="title" tagx="title" title="title">
Myo10 in melanoma development and metastasis</div> <p id="Par26">In the current study we found that Myo10KO significantly reduced melanoma development and metastasis in Tyr-CreER/PtenKO/BrafCA mice. These results suggest that Myo10KO attenuates the PtenKO and/or BrafCA effects in melanocyte neoplastic progression. PtenKO increases the concentration of PIP<sub>3</sub> in melanocytes, and PIP<sub>3</sub> binding activates Myo10 in one of two ways: (1) by targeting Myo10 to the plasma membrane, and (2) by inducing a conformational change in the Myo10 molecule itself as discussed above. Thus, if the main target of the Pten/PIP<sub>3</sub> signaling pathway is to activate Myo10 so that it induces pseudopod formation required for melanoma induction/metastasis, Myo10KO could diminish the PtenKO phenotype. Using "Matrigel on top" 3D culture models, the formation of actin-rich filopodial-like protrusions (FLPs) was identified as critical for extravasation of melanoma cells<sup><a href="#CR59">59</a></sup>. When mice are inoculated with aggressive cancer cells, abundant formation of FLPs is found at the inoculation site and in lung metastases. The number of FLPs was decreased by blocking either the Rif/mDia2 or ILK/β-parvin/cofilin signaling pathways resulting in a reduction in proliferation and metastasis. Pseudopods are thought to be similar to FLPs structurally and functionally and required not only for metastatic outgrowth, but also for primary tumor formation by interacting with the surrounding extracellular matrix<sup><a href="#CR59">59</a></sup>. Myo10 transports integrin-β to the tips of filopodia for their stability and elongation<sup><a href="#CR7">7</a></sup>. FLPs contain integrin-β1 for outside-in signaling transduction for proliferation. Thus, reducing the formation of actin-based protrusions by Myo10KO may decrease the proliferation signaling induced by BrafCA mutation.</p> </div> <div class="melanomacellmigrationinvivoandin3dculturemodels" title="sec"> <div class="title" tagx="title" title="title">
Melanoma cell migration <i>in vivo</i> and in 3D culture models</div> <p id="Par27">The mechanism of melanoma migration/metastasis <i>in vivo</i> is still controversial, and cancer cells use different modes of migration, which are influenced by a number of intrinsic and extrinsic cues<sup><a href="#CR60">60</a></sup>. Using a combination of single and multiphoton confocal microscopy, migration of melanoma cells expressing a fluorescent marker can be monitored <i>in vivo</i><sup><a href="#CR61">61</a></sup>. Even in an aggressive mouse line (B16F2) only 6–7% of the cells are motile <i>in vivo</i>, and most motile cells use an amoeboid-like single-cell mode of motility. It is possible that these amoeboid-like cells have fine protrusions such as FLPs or pseudopods, but that they are not detected because they are below the limit of resolution.</p> <p id="Par28">To overcome the limitation of <i>in vivo</i> observation, we used a 3D collagen culture system to monitor the morphology of live cells during migration. In this system, migrating mouse and human melanoma cell lines both showed similar morphodynamics of the finger-like filopodia at the front of the cells<sup><a href="#CR51">51</a></sup>. Our results suggest that Myo10 binds and transports the actin-polymerization machinery (including VASP) to the tips of pseudopods, where actin assembly occurs. It is possible that in addition to Myo10 other factors are also important for VASP targeting to the tips of pseudopods. Actin filaments then bundle into thin protrusions to form filopodia that push out the cell membrane in the direction of migration<sup><a href="#CR57">57</a></sup>. In 3D migration, integrins also play an important role by attaching the cell to the surrounding extracellular matrix (ECM). Activated β1 integrin localizes at every contact point the cell makes along its length with the fibers comprising the ECM<sup><a href="#CR62">62</a></sup>, and recycling of α5β1 integrin enhances the invasive migration of cancer cells in the three-dimensional ECM, which is driven by filopodial structures, which resemble pseudopods with filopodia at their front<sup><a href="#CR63">63</a></sup>. Myo10 binds (through its FERM domain) and transports β1 integrin to the tip of filopodia, which may serve as adhesive structures that support filopodial extension<sup><a href="#CR7">7</a>,<a href="#CR64">64</a></sup>. Induced filopodia in cells expressing the FERM-deletion mutant of full-length Myo10 do not stably extend<sup><a href="#CR65">65</a>,<a href="#CR66">66</a></sup>. These results suggest the possibility that Myo10 transports integrins into the leading front of pseudopods, which enable cells to adhere to the ECM fibers and move forward. Myo10 may control a balance between adhesion to ECM fibers driven by integrins and actin polymerization driven by VASP at the leading front of pseudopods.</p> <p id="Par29">For primary melanoma cells to establish remote metastases, they must invade other tissues/organs by breaching barriers such as the BM and stromal and/or parenchymal regions of target organs. Myo10 is expressed ubiquitously in almost all organs so these barriers may be altered in Myo10KO mice. Myo10kd in B16F1 cells reduced lung colonization after tail vein injection indicating that the effect is autonomous to melanoma cells, although it is still possible that Myo10KO in another organ may positively or negatively contribute to melanoma metastasis in Myo10KO mice.</p> </div> <div class="myo10expressioninhumanmelanoma" title="sec"> <div class="title" tagx="title" title="title">
Myo10 expression in human melanoma</div> <p id="Par30">Recent studies suggest that Myo10 has important functions in various types of cancer, e.g., breast cancer<sup><a href="#CR27">27</a>,<a href="#CR28">28</a>,<a href="#CR31">31</a></sup>, lung adenocarcinoma<sup><a href="#CR29">29</a></sup>, non-small cell lung cancer<sup><a href="#CR30">30</a></sup>, and prostate cancer<sup><a href="#CR32">32</a></sup>. In this study, we present the first genetic evidence that Myo10KO delays the onset and development of melanoma and reduces its metastasis <i>in vivo</i>. In human melanoma, increased expression of Myo10 decreases patient survival, which is the same effect observed in breast cancer. It is not clear whether Myo10 mediates the same mechanism in the progression of these types of cancer, but our findings in melanoma may provide a new direction to tackle other types of cancer.</p> <p id="Par31">In summary, we propose a Myo10-mediated LP induction model in melanoblast migration and melanoma induction and metastasis. Melanoblast and melanoma share a similar mechanism for migration and proliferation, although they are not completely the same. If the majority of intrinsic and extrinsic cues converge at LP formation, Myo10 may play a pivotal role in these phenotypes. Thus, Myo10 may be a therapeutic target for melanocyte-related diseases.</p> </div> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="mousemodels" title="sec"> <div class="title" tagx="title" title="title">
Mouse models</div> <p id="Par32"><i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> (Myo10KO) C57BL/6 N mice were obtained from the Wellcome Trust Sanger Research Institute. The inducible <i>Tyr::CreER</i><sup><i>T2</i></sup><i>;Pten</i><sup><i>LoxP/LoxP</i></sup><i>;Braf</i><sup><i>CA/</i>+</sup> melanoma model (Tyr-Cre/PtenKO/BrafCA) has been described before<sup><a href="#CR50">50</a></sup>, and we obtained this line from The Jackson Laboratory (Stock No:013590). By crossing these two lines, we generated <i>Tyr::CreER</i><sup><i>T2</i></sup><i>;Pten</i><sup><i>LoxP/LoxP</i></sup><i>;Braf</i><sup><i>CA/</i>+</sup><i>;Myo10</i><sup><i>−/−</i></sup> (Tyr-Cre/PtenKO/BrafCA/Myo10KO) and <i>Tyr::CreER</i><sup><i>T2</i></sup><i>;Pten</i><sup><i>LoxP/LoxP</i></sup><i>;Braf</i><sup><i>CA/</i>+</sup><i>;Myo10</i><sup>+<i>/</i>+</sup> (Tyr-Cre/PtenKO/BrafCA/Myo10WT) mice. All the animal experiments were performed in accordance to the NIH Guide for the Care and Use of Laboratory Animals and were approved by the Institutional Animal Care and Use Committee (IACUC) of Boston University School of Medicine.</p> </div> <div class="genotyping" title="sec"> <div class="title" tagx="title" title="title">
Genotyping</div> <p id="Par33">Primer sequences for genotyping <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup> mice were also provided by the Wellcome Trust Sanger Research Institute as follows:</p> <p id="Par34">Myo10F: 5′-ATCTGTTTCCCCTTAAGCGAAAAT-3′,</p> <p id="Par35">Myo10R: 5′-CTCTGTGGGGCCCAGAGCT-3′,</p> <p id="Par36">MutR: 5′-TCGTGGTATCGTTATGCGCC-3′,</p> <p id="Par37">LacZF: 5′-ATCACGACGCGCTGTATC-3′,</p> <p id="Par38">LacZR: 5′-ACATCGGGCAAATAATATCG-3′.</p> <p id="Par39">Genomic DNA samples isolated from tail snips were used as a template for PCR with primer pairs Myo10F/Myo10R, Myo10F/MutR and LacZF/LacZR. PCR with the primer pair Myo10F/Myo10R produced a 400-bp product only in mice that carried at least one copy of the WT allele. PCR with the primer pair Myo10F/MutR resulted in a 295-bp product, and PCR with the primer pair LacZF/LacZR resulted in a 108-bp product in mice that carried at least one copy of the mutant allele. The method for genotyping Tyr-Cre/PtenKO/BrafCA mice has been previously described<sup><a href="#CR50">50</a></sup>.</p> </div> <div class="activationofthetyr::creert2transgeneandmousesurvivalcohorts" title="sec"> <div class="title" tagx="title" title="title">
Activation of the Tyr::CreER<sup>T2</sup> transgene and mouse survival cohorts</div> <p id="Par40">The dorsal hair of three week old mice was shaved and further depilated with depilatory cream (Veet) before topical application to the left side of the dorsal skin of 2 μl 10 mM 4-HT in ethanol (Fisher Scientific). A total of 10 Myo10WT and 11 Myo10KO mice under the background of Tyr-Cre/PtenKO/BrafCA was monitored for up to 120 days for the development of melanoma and signs of metastasis. All mice were checked twice weekly for the development of malignant melanoma, and tumor growth was measured using calipers. Tumor size was estimated using the ellipsoid volume formula: 4/3π(length/2)(width/2)(depth/2). End point criteria were melanomas ≥20 mm in diameter, ulcerating melanomas, cachexia, significant weight loss, or weakness and inactivity. Upon meeting these criteria, mice were euthanized.</p> </div> <div class="immunohistochemistry" title="sec"> <div class="title" tagx="title" title="title">
Immunohistochemistry</div> <p id="Par41">Tissues were removed from the mice directly after euthanasia. For Trp2 (tyrosinase-related protein 2) staining in hair follicles, excised skin was embedded in OCT compound (Sakura Finetek USA, Inc., Torrence, CA), frozen with dry ice, and stored at −80 °C. Frozen tissues were cut into 8 μm sections, which were then fixed in 4% paraformaldehyde for 10 min and blocked with 5% normal donkey serum in PBS for 30 min. The tissue sections were then incubated with anti-Trp2 antibody (sc-10451, Santa Cruz Biotechnology, Inc., Dallas, TX) diluted with 5% bovine serum albumin in PBS overnight at 4 °C. After washing with PBS, sections were incubated with donkey anti-goat IgG-FITC (sc-2024, Santa Cruz Biotechnology) for 1 h and counterstained with 4′,6-diamidine-2′-phenylindole dihydrochloride (DAPI, Vector Laboratories, Burlingame, CA). Immunofluorescence was observed and photographed on a TCS SP5 confocal microscope (Leica Microsystems, Inc., Buffalo Grove, IL).</p> <p id="Par42">S100 proteins are expressed in many types of human cancers<sup><a href="#CR67">67</a></sup>, among them, S100A4, S100A9 and S100B are markers with excellent sensitivity for melanoma<sup><a href="#CR68">68</a>–<a href="#CR71">71</a></sup>. Mouse monoclonal anti-S100 antibody (4C4.9) was generated with purified bovine brain S100 protein and stained S100 in melanoma and its metastasis<sup><a href="#CR72">72</a>,<a href="#CR73">73</a></sup>. For S100 staining in tumors and lymph nodes, tissues were fixed in formalin and embedded in paraffin. Five micrometer-thick sections were cut, de-paraffinized with xylene, and autoclaved for 20 min in sodium citrate buffer (10 mM, pH 6) for antigen retrieval. Staining was performed with reagents from Vector Laboratories. Endogenous alkaline phosphatase was blocked with BLOXALL blocking solution for 10 min. Avidin/Biotin blocking was performed with an Avidin/Biotin blocking kit. M.O.M mouse Ig blocking reagent was used for 1 hr to block endogenous mouse immunoglobulins. After washing with PBS and incubating with M.O.M diluent for 5 min, slides were incubated with anti-S100 antibodies (4C4.9, Thermo Fisher Scientific) in M.O.M diluent for 30 min at room temperature. After washing with PBS, M.O.M biotinylated anti-mouse IgG reagent was applied to the samples for 10 min. Slides were washed with PBS, then incubated with VECTASTAIN ABC-AP reagent for 30 min. After washing with PBS, alkaline phosphatase was localized with ImmPACT Vector Red reagent for 20 min, and slides were counterstained with Vector Hematoxylin QS.</p> <p id="Par43">For Myo10 staining in human specimens, formalin fixed and paraffin embedded tissues were cut into 5 μm sections, de-paraffinized, subjected to antigen retrieval, then blocked as described above. After incubation with 2.5% normal horse serum for 30 min, slides were incubated with anti-Myo10 antibody (Sigma HPA024223) diluted with 5% bovine serum albumin in PBS overnight at 4 °C. Slides were then washed with PBS and incubated with ImmPRESS-AP polymer anti-rabbit IgG reagent for 30 min, and counterstained as described above. Standard microscope images were acquired using a Nikon E800 microscope with SPOT camera and software version 4.6 (Diagnostic Instruments Inc., Sterling Heights, MI).</p> <p id="Par44">For tissue microarray analysis, a tissue microarray consisting of melanoma and nevi cohorts was obtained from US Biomax, Inc. (ME1004e). The TMA slide was de-paraffinized and antigen was unmasked as described above. After incubation with blocking buffer (10% goat serum, 5% BSA in PBS) for 30 min, anti-Myo10 and anti-S100 antibodies diluted in blocking buffer were applied to the slide overnight at 4 °C. After washing with PBS, slides were incubated with Alexa Fluor 488 goat anti-rabbit IgG and Alexa Fluor 568 goat anti-mouse IgG (Thermo Fisher Scientific) in 5% BSA in PBS for 1 h. Alexa Fluor 488 and 568 fluorescence were captured on the TCS SP5 confocal microscope using sequential acquisition to give separate image files for each. A scan speed of 400 Hz and 4-frame averaging were used, and gain and offset were adjusted to give sub-saturating fluorescence intensity with optimal signal-to-noise ratio. Once the settings were determined, they were used to capture all the cores of the array. Image analysis was performed with ImageJ software. TMA cores with less than 5% tumor area per spot were not included in quantitative analysis. Using ImageJ, binary image masks of Myo10 and S100 were created by setting the lower threshold level at 35 and the upper level at 255. S100-positive staining was used to define regions of interest (ROI) for analysis. The ROI was overlaid onto the Myo10 image, the integrated density in this region was measured, and the total integrated density was divided by the total area of the ROI. This value was considered as the mean intensity of Myo10 in S100-positive tumors.</p> </div> <div class="quantificationoflymphnodemetastasis" title="sec"> <div class="title" tagx="title" title="title">
Quantification of lymph node metastasis</div> <p id="Par45">Metastasis from a primary tumor to the lymph nodes was observed when the volume of the primary tumor reached ~1,000 mm<sup>3</sup>, restricting drainage of the lymph nodes (left inguinal). Fixed and embedded lymph nodes were sliced at the middle of the tissue. All the sections were stained with S100 antibody as described above and scanned manually at a magnification of 20× using the Nikon E800 microscope with SPOT camera and software version 4.6 (Diagnostic Instruments Inc.). S100-positive cells were counted in six random fields in each of three slices per mouse.</p> </div> <div class="cellculture,transfection,andcaptureoflive-cellimages" title="sec"> <div class="title" tagx="title" title="title">
Cell culture, transfection, and capture of live-cell images</div> <p id="Par46">Melb-a melanoblasts were kindly provided by Dr. Dorothy C. Bennett (St. George’s University of London, London, UK)<sup><a href="#CR31">31</a></sup>, and grown in RPMI supplemented with 10% fetal bovine serum (FBS) (Atlanta Biologicals, Flowery Branch, GA), 2 mM glutamine, 100 international units/ml penicillin, 100 μg/ml streptomycin (=complete medium; all components from Life Technologies, Thermo Fisher Scientific, Waltham, MA), 20 ng/ml mouse stem cell factor (R&amp;amp;D Systems, Minneapolis, MN) and 40 pM basic fibroblast growth factor (Novus Biologicals, Littleton, CO). B16F1 cells were purchased from American Type Culture Collection (ATCC #CRL-6323; Manassas, VA) and grown in complete Dulbecco’s Modified Eagle’s Medium (DMEM, Invitrogen, Thermo Fisher Scientific). Expression vectors were transfected with X-tremeGENE HP (Sigma-Aldrich, St. Louis, MO) according to the manufacturer’s instructions.</p> <p id="Par47">Collagen gel solutions (2 mg/ml) were prepared by mixing rat type I collagen (Corning, Bedford, MA) with 10× minimal essential medium (MEM, final concentration of ×1, Gibco, Thermo Fisher Scientific), L-glutamine (2 mM final concentration; Gibco), sodium bicarbonate (3.7 g/L final concentration; Gibco), and FBS (10% final concentration). To generate thick fibrillar collagen substrates, 200 μl of collagen solution (pH adjusted to 7.4 with 1 M NaOH) was applied to 35-mm glass bottom culture dishes (MatTek Corp., Ashland, MA) and allowed to polymerize for 1 hr at room temperature. Melb-a cells suspended in culture medium were added to the substrates and cultured overnight for next day live-cell morphological studies. For 3D migration assays, cells were resuspended in complete medium and mixed with the prepared collagen mixture; the pH was adjusted to 7.4 with 1 M NaOH. The collagen and cell mixture was spread onto 35 mm MatTek dishes and allowed to polymerize for 1 hr at 37 °C for 30 min. Complete medium was added to the gels, and the cells were assayed for motility the next day.</p> <p id="Par48">Phase contrast live images of Melb-a cells cultured on thick collagen substrates and B16F1 cells cultured in the 3D collagen matrices were captured with the Eclipse TS100 Inverted Microscope with a 20×/0.40 Ph1 ADL objective lens (Nikon).</p> </div> <div class="expressionplasmids,viralconstructs,andinfection" title="sec"> <div class="title" tagx="title" title="title">
Expression plasmids, viral constructs, and infection</div> <p id="Par49">mCherry-Lifeact was purchased from Addgene (#54491; Cambridge, MA), and pEGFP-Myo10 was previously described<sup><a href="#CR5">5</a></sup>. To knockdown mouse <i>MYO10</i> expression, we used the pLKO.1-TRC lentiviral vector (MISSION shRNA, Sigma-Aldrich) containing shRNA against mouse <i>MYO10</i> (shRNA-1;TRCN0000110606, shRNA-2;TRCN0000110608, and shRNA-3;TRCN0000379126). A non-target shRNA vector (SHC016; Sigma-Aldrich) was used as a control for the shRNA experiments. For lentiviral packaging, psPAX2 and pMD2.G vectors (Addgene plasmids #12260, #12259) were co-transfected with the pLKO.1 into HEK293T cells with X-tremeGENE HP (Sigma-Aldrich) according to the manufacturer’s instructions. Virus-containing medium was harvested 72 hrs after transfection and filtered with 0.45 μm filters (EMD Millipore, Billerica, MA). Melb-a or B16F1 cells were infected with culture supernatants from HEK293T cells at a 1:6 dilution in complete culture medium. Cells were passed after 24 hrs and selected with 4 μg/ml puromycin for 14 days.</p> </div> <div class="immunoblotting" title="sec"> <div class="title" tagx="title" title="title">
Immunoblotting</div> <p id="Par50">For kidney and brain, small portions excised from one-month-old mice were weighed and added to pre-heated Laemmli sample buffer (50 mM Tris, pH 7.4, 2% SDS, 6% glycerol, 0.1 M DTT and 0.25% bromophenol blue) at 500 μL/20 mg wet tissue weight and boiled for 15 min. Samples were homogenized with a Potter-Elehjem tissue grinder and passed through a 25 G needle with syringe 10 times. For induced dorsal skin melanomas, tumors were excised from euthanized mice and quickly frozen in liquid nitrogen. Samples were ground with a mortar and pestle in the presence of liquid nitrogen to create a powder. Pre-heated Laemmli sample buffer at 500 μL/20 mg wet tissue weight was added to the powdered tissue, and the sample was boiled for 15 min then homogenized with a 25 G needle and syringe. For cultured cells, cells grown in 6-well dishes were washed with PBS, collected with 100 μL Laemmli sample buffer, boiled for 15 min, and homogenized with a 25 G needle and syringe. All the lysates were centrifuged at 14,000 × g for 30 min at room temperature, and the supernatant was applied to SDS gels for separation followed by immunoblotting. To adjust the loading volume, serial dilutions of each lysate were subjected to SDS-PAGE using 4–12% Bis-Tris gels and then the gels were stained with Coomassie Brilliant Blue (CBB). After gel electrophoresis, lysates were transferred to nitrocellulose membranes at 4 °C in 20 mM Tris, 200 mM glycine, and 20% methanol using a Mini Trans-Blot cell (Bio-Rad) set to 100 V for 70 min, followed by blocking the nitrocellulose for 30 min in blocking buffer (PBS containing 0.1% Tween 20 with 5% dried milk). Membranes were then incubated overnight with 1 μg/mL anti-Myo10 antibody (Sigma #HPA024223) in blocking buffer. After washing three times for 5 min in washing buffer (PBS containing 0.1% Tween 20) membranes were incubated with goat anti-rabbit IgG (H + L)-HRP conjugate (Bio-Rad) diluted (1:15,000) in blocking buffer for 30 min. Following five 5-min washes, membranes were incubated with Amersham ECL Prime Western Blotting Detection Reagent (GE Healthcare) and imaged on Amersham Hyperfilm (GE Healthcare). For the loading control, HRP was inactivated by incubating with 30% (w/v) hydrogen peroxide at 37 °C for 15 min. Then, after re-blocking the membranes were re-probed with mouse monoclonal anti-α-tubulin antibodies (Sigma, T5168) and developed with ECL reagent as above. In the case of the tissue lysates, probed membranes were stained with CBB to control for the amount loaded and transfer efficiency. Images on films and CBB-stained membranes were captured with a photo scanner (Canon, Pixma) and converted to gray scale by Photoshop Elements 13 (Adobe Systems).</p> </div> <div class="immunocytochemistryand3dlive-cellimaging" title="sec"> <div class="title" tagx="title" title="title">
Immunocytochemistry and 3D live-cell imaging</div> <p id="Par51">B16F1 cells cultured in 3D collagen matrices were fixed with 4% PFA for 15 min and permeabilized in PBS containing 0.1% Triton X for 10 min. After blocking with blocking buffer (5% BSA in PBS) for 30 min, anti-Myo10 antibody or anti-VASP rabbit monoclonal antibody (9A2, Cell Signaling Technology) diluted in blocking buffer was applied to 3D cultures overnight at 4 °C. After washing with PBS, cells were incubated with Alexa Fluor 488 goat anti-rabbit IgG (Thermo Fisher Scientific) and Alexa Fluor 594 phalloidin (Invitrogen) in blocking buffer for 1 h. Cells were then washed three times with PBS and mounted in Vectashield mounting medium with DAPI (Vector Laboratories, Burlingame, CA). Fluorescence images were captured on the TCS SP5 confocal microscope with an HCX PL APO 40×/1.25 objective lens (Leica) using sequential acquisition to give separate image files for each. A scan speed of 400 Hz and 4-frame averaging were used, and gain and offset were adjusted to give sub-saturating fluorescence intensity with optimal signal-to-noise ratio.</p> <p id="Par52">3D live-cell imaging was performed using a Zeiss LSM 710 confocal microscope with an EC Plan-Neofluor 40×/1.30 objective lens (Carl Zeiss, Jena, Germany). Image capture and processing were performed using the ZEN 2012 (Zeiss) software package. B16F1 cells co-transfected with GFP-Myo10 and mCherry-Lifeact were cultured in 3D collagen matrices 16 hrs before the experiments and maintained at 5% CO<sub>2</sub> and 37 °C in a temperature-controlled chamber. Time-lapse images were obtained with a 1,024 × 1,024 (xy) pixel scan field (pixel size: 0.16 μm) by sequential epifluorescent illumination with 1.58 μsec pixel dwell and 5-min intervals over 30 min. Captured images were exported to ImageJ and reconstructed to Z-stack images from an 8 μm Z-range with 17 slices (0.5 μm intervals) using maximum intensity projection.</p> </div> <div class="transwellmigrationassay" title="sec"> <div class="title" tagx="title" title="title">
Transwell migration assay</div> <p id="Par53">A total of 5 × 10<sup>4</sup> Melb-a melanoblasts suspended in 0.5% FBS with complete medium was seeded into the upper chamber inserts (8.0 μm, 24 well) of Transwells (Corning, Bedford, MA). The top chamber inserts were placed in the bottom wells, which were filled with 5% FBS with complete medium, overnight. The non-migrating cells present on the upper surface of the insert were removed with a cotton swab. The cells that migrated onto the lower surface of the filter were fixed and stained with 0.1% crystal violet. The number of migrated cells in 4 different random fields on each membrane was counted using ImageJ software. Each experiment was repeated at least three times.</p> </div> <div class="experimentalmetastasis" title="sec"> <div class="title" tagx="title" title="title">
Experimental metastasis</div> <p id="Par54">Experimental metastasis was performed as described previously with slight modification<sup><a href="#CR74">74</a></sup>. Briefly, B16F1 cell lines were trypsinized and suspended in PBS. Cells (2 × 10<sup>5</sup>) were then injected intravenously into the tail veins of 5–7 week old female C57BL6 mice. Two weeks after injection, the mice were euthanized, the lungs were removed, and the black spherical colonies were counted. In total, 10 mice were used for control shRNA cells, and 10 mice were used for Myo10 shRNA cells.</p> </div> <div class="insilicodataanalysis" title="sec"> <div class="title" tagx="title" title="title"><i>In silico</i> data analysis</div> <p id="Par55">cDNA microarray data sets of malignant melanoma and normal tissue samples were analyzed using the Oncomine database (<a href="http://www.oncomine.org">www.oncomine.org</a>). The effect of Myo10 expression in malignant melanoma on patients’ overall survival was analyzed by using the online survival analysis tool PROGgeneV2 (<a href="http://watson.compbio.iupui.edu/chirayu/proggene/database/">http://watson.compbio.iupui.edu/chirayu/proggene/database/</a>). The cohort was divided into two groups based on median expression of Myo10 to analyze the correlation between Myo10 expression and overall survival. The Kaplan-Meier plots, logrank p value, hazard ratio, lower and upper confidence intervals of hazard ratio displayed in the survival plot legend were also generated using PROGgeneV2. Using the cBioPortal database (<a href="http://www.cbioportal.org">www.cbioportal.org</a>), the cohort from TCGA-SKCM was divided two groups with (<i>n</i> = 55) or without (<i>n</i> = 405) <i>TP53</i> mutation in melanoma patients’ samples. The data of <i>MYO10</i> mRNA expression were downloaded and analyzed.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par56">All statistical analyses (except Oncomine and PROGgeneV2 data analyses) were performed with Prism 6 (GraphPad Software, San Diego, CA). Unpaired t-test was used for comparing data between two groups. <i>P</i> &amp;lt; 0.05 was considered significant.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2018_28717_MOESM1_ESM.pdf">LINK</a> <p>Supplementary Information</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2018_28717_MOESM2_ESM.mov">LINK</a> <p>Supplementary Movie S1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2018_28717_MOESM3_ESM.mov">LINK</a> <p>Supplementary Movie S2</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Electronic supplementary material</b></p> <p><b>Supplementary information</b> accompanies this paper at 10.1038/s41598-018-28717-y.</p> </div> <div class="fn-type-" title=""> <p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank the Wellcome Trust Sanger Research Institute Mouse Genetics Project (Sanger MGP) and its funders for providing the mutant mouse line, <i>Myo10</i><sup><i>tm2(KOMP)Wtsi</i></sup>. Funding and associated primary phenotypic information may be found at <a href="http://www.sanger.ac.uk/mouseportal">www.sanger.ac.uk/mouseportal</a>. We also thank Dr. Dorothy C. Bennett (St. George’s University of London, London, UK) for Melb-a melanoblasts. Technical assistance of Salma Goummih is gratefully acknowledged. Funding was provided by NIH R01 GM111615 to L.M.C. and by Boston University, Clinical and Translational Science Institute (1UL1TR001430) to H.T.</p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>H.T. performed the experiments and analyzed the results. J.B. contributed to the immunohistochemistry of patients’ samples. H.T. and L.M.C. conceived the experiments and wrote the manuscript.</p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p id="Par57">The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Divito, M. M. &amp;amp; Cheney, R. E. <i>Myosins: A superfamily of molecular motors. Chapter 14 MYOSIN X</i>, (Springer, Dordrecht, The Netherlands, 2008).</span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Derfler</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pennisi</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corey</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X, a novel myosin with pleckstrin homology domains, associates with regions of dynamic actin</span><span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">113</span><span class="issue" tagx="issue" title="issue">Pt 19</span><span class="fpage" tagx="fpage" title="fpage">3439</span><span class="lpage" tagx="lpage" title="lpage">51</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10984435">10984435</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mattila</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lappalainen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Filopodia: molecular architecture and cellular functions</span><span class="source" tagx="source" title="source">Nat Rev Mol Cell Biol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">446</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm2406">10.1038/nrm2406</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18464790">18464790</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Umeki</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Phospholipid-dependent regulation of the motor activity of myosin X</span><span class="source" tagx="source" title="source">Nat Struct Mol Biol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">783</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nsmb.2065">10.1038/nsmb.2065</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21666676">21666676</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokuo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikebe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin X transports Mena/VASP to the tip of filopodia</span><span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">319</span><span class="fpage" tagx="fpage" title="fpage">214</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbrc.2004.04.167">10.1016/j.bbrc.2004.04.167</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15158464">15158464</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurley</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raines</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin X and its motorless isoform differentially modulate dendritic spine development by regulating trafficking and retention of vasodilator-stimulated phosphoprotein</span><span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">126</span><span class="fpage" tagx="fpage" title="fpage">4756</span><span class="lpage" tagx="lpage" title="lpage">68</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.132969">10.1242/jcs.132969</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23943878">23943878</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X provides a motor-based link between integrins and the cytoskeleton</span><span class="source" tagx="source" title="source">Nat Cell Biol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">523</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncb1136">10.1038/ncb1136</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15156152">15156152</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">XJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin X regulates netrin receptors and functions in axonal path-finding</span><span class="source" tagx="source" title="source">Nat Cell Biol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">184</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncb1535">10.1038/ncb1535</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237772">17237772</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almagro</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The motor protein myosin-X transports VE-cadherin along filopodia to allow the formation of early endothelial cell-cell contacts</span><span class="source" tagx="source" title="source">Mol Cell Biol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1703</span><span class="lpage" tagx="lpage" title="lpage">17</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MCB.01226-09">10.1128/MCB.01226-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20123970">20123970</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plantard</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">PtdIns(3,4,5)P(3) is a regulator of myosin-X localization and filopodia formation</span><span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">3525</span><span class="lpage" tagx="lpage" title="lpage">34</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.069609">10.1242/jcs.069609</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20930142">20930142</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yonezawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mouse myosin X: molecular architecture and tissue expression as revealed by northern blot and <i>in situ</i> hybridization analyses</span><span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">271</span><span class="fpage" tagx="fpage" title="fpage">526</span><span class="lpage" tagx="lpage" title="lpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/bbrc.2000.2669">10.1006/bbrc.2000.2669</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10799329">10799329</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yonezawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Possible involvement of myosin-X in intercellular adhesion: importance of serial pleckstrin homology regions for intracellular localization</span><span class="source" tagx="source" title="source">Dev Growth Differ</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">175</span><span class="lpage" tagx="lpage" title="lpage">85</span><span class="pub-id"><a href="https://dx.doi.org/10.1034/j.1600-0854.2004.00688.x">10.1034/j.1600-0854.2004.00688.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12752505">12752505</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin X is a downstream effector of PI(3)K during phagocytosis</span><span class="source" tagx="source" title="source">Nat Cell Biol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">469</span><span class="lpage" tagx="lpage" title="lpage">77</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12055636">12055636</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokac</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bement</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span></span><span class="mixed-article-title" title="mixed-article-title">A microtubule-binding myosin required for nuclear anchoring and spindle assembly</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">431</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature02834">10.1038/nature02834</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15372037">15372037</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toyoshima</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishida</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Integrin-mediated adhesion orients the spindle parallel to the substratum in an EB1- and myosin X-dependent manner</span><span class="source" tagx="source" title="source">Embo J</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">1487</span><span class="lpage" tagx="lpage" title="lpage">98</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.emboj.7601599">10.1038/sj.emboj.7601599</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17318179">17318179</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pi</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sequential roles for myosin-X in BMP6-dependent filopodial extension, migration, and activation of BMP receptors</span><span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">179</span><span class="fpage" tagx="fpage" title="fpage">1569</span><span class="lpage" tagx="lpage" title="lpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200704010">10.1083/jcb.200704010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18158328">18158328</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kee</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bronner-Fraser</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X is critical for migratory ability of Xenopus cranial neural crest cells</span><span class="source" tagx="source" title="source">Dev Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">335</span><span class="fpage" tagx="fpage" title="fpage">132</span><span class="lpage" tagx="lpage" title="lpage">42</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ydbio.2009.08.018">10.1016/j.ydbio.2009.08.018</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19712673">19712673</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwang</span><span class="given-names" tagx="given-names" title="given-names">YS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sargent</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X is required for cranial neural crest cell migration in Xenopus laevis</span><span class="source" tagx="source" title="source">Dev Dyn</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">238</span><span class="fpage" tagx="fpage" title="fpage">2522</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/dvdy.22077">10.1002/dvdy.22077</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19718754">19718754</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMichael</span><span class="given-names" tagx="given-names" title="given-names">BK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">BS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin X regulates sealing zone patterning in osteoclasts through linkage of podosomes and microtubules</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">285</span><span class="fpage" tagx="fpage" title="fpage">9506</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M109.017269">10.1074/jbc.M109.017269</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20081229">20081229</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tasca</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of Osteoclast Differentiation by Myosin X</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">7603</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-07855-9">10.1038/s41598-017-07855-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28790434">28790434</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacobs</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunn</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fanning</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X functions in polarized epithelial cells</span><span class="source" tagx="source" title="source">Mol Biol Cell</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1675</span><span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.e11-04-0358">10.1091/mbc.e11-04-0358</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22419816">22419816</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raines</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagdas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerber</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Headless Myo10 is a negative regulator of full-length Myo10 and inhibits axon outgrowth in cortical neurons</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">287</span><span class="fpage" tagx="fpage" title="fpage">24873</span><span class="lpage" tagx="lpage" title="lpage">83</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M112.369173">10.1074/jbc.M112.369173</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22661706">22661706</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bishai</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X facilitates Shigella-induced membrane protrusions and cell-to-cell spread</span><span class="source" tagx="source" title="source">Cell Microbiol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">353</span><span class="lpage" tagx="lpage" title="lpage">67</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/cmi.12051">10.1111/cmi.12051</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23083060">23083060</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kroon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inflammation-Sensitive Myosin-X Functionally Supports Leukocyte Extravasation by Cdc42-Mediated ICAM-1-Rich Endothelial Filopodia Formation</span><span class="source" tagx="source" title="source">J Immunol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">200</span><span class="fpage" tagx="fpage" title="fpage">1790</span><span class="lpage" tagx="lpage" title="lpage">1801</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29386254">29386254</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horsthemke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple roles of filopodial dynamics in particle capture and phagocytosis and phenotypes of Cdc42 and Myo10 deletion</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">292</span><span class="fpage" tagx="fpage" title="fpage">7258</span><span class="lpage" tagx="lpage" title="lpage">7273</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M116.766923">10.1074/jbc.M116.766923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28289096">28289096</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heimsath</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X knockout is semi-lethal and demonstrates that myosin-X functions in neural tube closure, pigmentation, hyaloid vasculature regression, and filopodia formation</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">17354</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-17638-x">10.1038/s41598-017-17638-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29229982">29229982</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Elevated expression of myosin X in tumours contributes to breast cancer aggressiveness and metastasis</span><span class="source" tagx="source" title="source">Br J Cancer</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">111</span><span class="fpage" tagx="fpage" title="fpage">539</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/bjc.2014.298">10.1038/bjc.2014.298</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24921915">24921915</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arjonen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mutant p53-associated myosin-X upregulation promotes breast cancer invasion and metastasis</span><span class="source" tagx="source" title="source">J Clin Invest</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">124</span><span class="fpage" tagx="fpage" title="fpage">1069</span><span class="lpage" tagx="lpage" title="lpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI67280">10.1172/JCI67280</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24487586">24487586</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bidkhori</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Reconstruction of an integrated genome-scale co-expression network reveals key modules involved in lung adenocarcinoma</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e67552</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0067552">10.1371/journal.pone.0067552</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23874428">23874428</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ai</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">NF-kappaB-mediated miR-124 suppresses metastasis of non-small-cell lung cancer by targeting MYO10</span><span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">8244</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25749519">25749519</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">MiR-340 suppresses cell migration and invasion by targeting MYO10 in breast cancer</span><span class="source" tagx="source" title="source">Oncol Rep</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">709</span><span class="lpage" tagx="lpage" title="lpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.3892/or.2015.4411">10.3892/or.2015.4411</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26573744">26573744</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowska</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wells</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peckham</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Specific Myosins Control Actin Organization, Cell Morphology, and Migration in Prostate Cancer Cells</span><span class="source" tagx="source" title="source">Cell Rep</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">2118</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2015.11.012">10.1016/j.celrep.2015.11.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26670045">26670045</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">HO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levorse</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shin</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span></span><span class="mixed-article-title" title="mixed-article-title">The endothelin receptor-B is required for the migration of neural crest-derived melanocyte and enteric neuron precursors</span><span class="source" tagx="source" title="source">Dev Biol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">259</span><span class="fpage" tagx="fpage" title="fpage">162</span><span class="lpage" tagx="lpage" title="lpage">75</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0012-1606(03)00160-X">10.1016/S0012-1606(03)00160-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12812796">12812796</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reid</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple roles for endothelin in melanocyte development: regulation of progenitor number and stimulation of differentiation</span><span class="source" tagx="source" title="source">Development</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">3911</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9012511">9012511</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goding</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mitf from neural crest to melanoma: signal transduction and transcription in the melanocyte lineage</span><span class="source" tagx="source" title="source">Genes Dev</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">1712</span><span class="lpage" tagx="lpage" title="lpage">28</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10898786">10898786</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Martin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SLUG (SNAI2) deletions in patients with Waardenburg disease</span><span class="source" tagx="source" title="source">Hum Mol Genet</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">3231</span><span class="lpage" tagx="lpage" title="lpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/hmg/11.25.3231">10.1093/hmg/11.25.3231</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12444107">12444107</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engleka</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Insertion of Cre into the Pax3 locus creates a new allele of Splotch and identifies unexpected Pax3 derivatives</span><span class="source" tagx="source" title="source">Dev Biol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">280</span><span class="fpage" tagx="fpage" title="fpage">396</span><span class="lpage" tagx="lpage" title="lpage">406</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ydbio.2005.02.002">10.1016/j.ydbio.2005.02.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15882581">15882581</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A dual role for SOX10 in the maintenance of the postnatal melanocyte lineage and the differentiation of melanocyte stem cell progenitors</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e1003644</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1003644">10.1371/journal.pgen.1003644</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23935512">23935512</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The melanocyte differentiation program predisposes to metastasis after neoplastic transformation</span><span class="source" tagx="source" title="source">Nat Genet</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">1047</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng1634">10.1038/ng1634</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16142232">16142232</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uong</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zon</span><span class="given-names" tagx="given-names" title="given-names">LI</span></span></span><span class="mixed-article-title" title="mixed-article-title">Melanocytes in development and cancer</span><span class="source" tagx="source" title="source">J Cell Physiol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">222</span><span class="fpage" tagx="fpage" title="fpage">38</span><span class="lpage" tagx="lpage" title="lpage">41</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcp.21935">10.1002/jcp.21935</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19795394">19795394</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strizzi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardy</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirsammer</span><span class="given-names" tagx="given-names" title="given-names">GT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerami</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendrix</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Embryonic signaling in melanoma: potential for diagnosis and therapy</span><span class="source" tagx="source" title="source">Lab Invest</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">819</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/labinvest.2011.63">10.1038/labinvest.2011.63</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21464823">21464823</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sousa</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berg</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meeker</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myo10 in brain: developmental regulation, identification of a headless isoform and dynamics in neurons</span><span class="source" tagx="source" title="source">J Cell Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">184</span><span class="lpage" tagx="lpage" title="lpage">94</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.02726">10.1242/jcs.02726</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16371656">16371656</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Rac1 drives melanoblast organization during mouse development by orchestrating pseudopod- driven motility and cell-cycle progression</span><span class="source" tagx="source" title="source">Dev Cell</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">722</span><span class="lpage" tagx="lpage" title="lpage">34</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2011.07.008">10.1016/j.devcel.2011.07.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21924960">21924960</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sviderskaya</span><span class="given-names" tagx="given-names" title="given-names">EV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakeling</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennett</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span></span><span class="mixed-article-title" title="mixed-article-title">A cloned, immortal line of murine melanoblasts inducible to differentiate to melanocytes</span><span class="source" tagx="source" title="source">Development</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">121</span><span class="fpage" tagx="fpage" title="fpage">1547</span><span class="lpage" tagx="lpage" title="lpage">57</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7540532">7540532</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davies</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mutations of the BRAF gene in human cancer</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">417</span><span class="fpage" tagx="fpage" title="fpage">949</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature00766">10.1038/nature00766</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12068308">12068308</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wellbrock</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">V599EB-RAF is an oncogene in melanocytes</span><span class="source" tagx="source" title="source">Cancer Res</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">2338</span><span class="lpage" tagx="lpage" title="lpage">42</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-03-3433">10.1158/0008-5472.CAN-03-3433</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15059882">15059882</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garraway</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Malignant melanoma: genetics and therapeutics in the genomic era</span><span class="source" tagx="source" title="source">Genes Dev</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">2149</span><span class="lpage" tagx="lpage" title="lpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gad.1437206">10.1101/gad.1437206</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16912270">16912270</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Modeling genomic diversity and tumor dependency in malignant melanoma</span><span class="source" tagx="source" title="source">Cancer Res</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">664</span><span class="lpage" tagx="lpage" title="lpage">73</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-07-2615">10.1158/0008-5472.CAN-07-2615</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18245465">18245465</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goel</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haluska</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic interaction between NRAS and BRAF mutations and PTEN/MMAC1 inactivation in melanoma</span><span class="source" tagx="source" title="source">J Invest Dermatol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">337</span><span class="lpage" tagx="lpage" title="lpage">41</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.0022-202X.2004.22243.x">10.1046/j.0022-202X.2004.22243.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15009714">15009714</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dankort</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Braf(V600E) cooperates with Pten loss to induce metastatic melanoma</span><span class="source" tagx="source" title="source">Nat Genet</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">544</span><span class="lpage" tagx="lpage" title="lpage">52</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.356">10.1038/ng.356</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19282848">19282848</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Starke</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maaser</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehrle-Haller</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedl</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanotransduction of mesenchymal melanoma cell invasion into 3D collagen lattices: filopod-mediated extension-relaxation cycles and force anisotropy</span><span class="source" tagx="source" title="source">Exp Cell Res</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">319</span><span class="fpage" tagx="fpage" title="fpage">2424</span><span class="lpage" tagx="lpage" title="lpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.yexcr.2013.04.003">10.1016/j.yexcr.2013.04.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23830878">23830878</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barzik</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Ena/VASP proteins enhance actin polymerization in the presence of barbed end capping proteins</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">280</span><span class="fpage" tagx="fpage" title="fpage">28653</span><span class="lpage" tagx="lpage" title="lpage">62</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M503957200">10.1074/jbc.M503957200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15939738">15939738</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Antagonism between Ena/VASP proteins and actin filament capping regulates fibroblast motility</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">509</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(02)00731-6">10.1016/S0092-8674(02)00731-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12086607">12086607</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goswami</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakshatri</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">PROGgeneV2: enhancements on the existing database</span><span class="source" tagx="source" title="source">BMC Cancer</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">970</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2407-14-970">10.1186/1471-2407-14-970</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25518851">25518851</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindsay</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">P-Rex1 is required for efficient melanoblast migration and melanoma metastasis</span><span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">555</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms1560">10.1038/ncomms1560</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22109529">22109529</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodham</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Coordination by Cdc42 of Actin, Contractility, and Adhesion for Melanoblast Movement in Mouse Skin</span><span class="source" tagx="source" title="source">Curr Biol</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">624</span><span class="lpage" tagx="lpage" title="lpage">637</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2017.01.033">10.1016/j.cub.2017.01.033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28238662">28238662</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokuo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mabuchi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikebe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The motor activity of myosin-X promotes actin fiber convergence at the cell periphery to initiate filopodia formation</span><span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">179</span><span class="fpage" tagx="fpage" title="fpage">229</span><span class="lpage" tagx="lpage" title="lpage">38</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200703178">10.1083/jcb.200703178</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17954606">17954606</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luciani</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biological and mathematical modeling of melanocyte development</span><span class="source" tagx="source" title="source">Development</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">138</span><span class="fpage" tagx="fpage" title="fpage">3943</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/dev.067447">10.1242/dev.067447</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21862558">21862558</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shibue</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinberg</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="mixed-article-title" title="mixed-article-title">An integrin-linked machinery of cytoskeletal regulation that enables experimental tumor initiation and metastatic colonization</span><span class="source" tagx="source" title="source">Cancer Cell</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">481</span><span class="lpage" tagx="lpage" title="lpage">98</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ccr.2013.08.012">10.1016/j.ccr.2013.08.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24035453">24035453</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mistriotis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Konstantopoulos</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cancer cell motility: lessons from migration in confined spaces</span><span class="source" tagx="source" title="source">Nat Rev Cancer</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="lpage" tagx="lpage" title="lpage">140</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrc.2016.123">10.1038/nrc.2016.123</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27909339">27909339</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinner</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahai</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Imaging amoeboid cancer cell motility <i>in vivo</i></span><span class="source" tagx="source" title="source">J Microsc</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">231</span><span class="fpage" tagx="fpage" title="fpage">441</span><span class="lpage" tagx="lpage" title="lpage">5</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2818.2008.02056.x">10.1111/j.1365-2818.2008.02056.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18754999">18754999</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doyle</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvajal</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsumoto</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Local 3D matrix microenvironment regulates cell migration through spatiotemporal dynamics of contractility-dependent adhesions</span><span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">8720</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms9720">10.1038/ncomms9720</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26548801">26548801</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">alpha5beta1 integrin recycling promotes Arp2/3-independent cancer cell invasion via the formin FHOD3</span><span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">210</span><span class="fpage" tagx="fpage" title="fpage">1013</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201502040">10.1083/jcb.201502040</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26370503">26370503</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bohil</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheney</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X is a molecular motor that functions in filopodia formation</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">12411</span><span class="lpage" tagx="lpage" title="lpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0602443103">10.1073/pnas.0602443103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16894163">16894163</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokuo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonda</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higuchi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikebe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin-X induces filopodia by multiple elongation mechanism</span><span class="source" tagx="source" title="source">J Biol Chem</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">285</span><span class="fpage" tagx="fpage" title="fpage">19605</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M109.093864">10.1074/jbc.M109.093864</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20392702">20392702</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsukasaki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikebe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myosin X is recruited to nascent focal adhesions at the leading edge and induces multi-cycle filopodial elongation</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">13685</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-06147-6">10.1038/s41598-017-06147-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29057977">29057977</a></span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bresnick</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmer</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span></span><span class="mixed-article-title" title="mixed-article-title">S100 proteins in cancer</span><span class="source" tagx="source" title="source">Nat Rev Cancer</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">96</span><span class="lpage" tagx="lpage" title="lpage">109</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrc3893">10.1038/nrc3893</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25614008">25614008</a></span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Therapeutic targeting of tumor growth and angiogenesis with a novel anti-S100A4 monoclonal antibody</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e72480</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0072480">10.1371/journal.pone.0072480</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24023743">24023743</a></span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hibino</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">S100A9 is a novel ligand of EMMPRIN that promotes melanoma metastasis</span><span class="source" tagx="source" title="source">Cancer Res</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">73</span><span class="fpage" tagx="fpage" title="fpage">172</span><span class="lpage" tagx="lpage" title="lpage">83</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-11-3843">10.1158/0008-5472.CAN-11-3843</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23135911">23135911</a></span></span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmer</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapidus</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>In vivo</i> screening of S100B inhibitors for melanoma therapy</span><span class="source" tagx="source" title="source">Methods Mol Biol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">963</span><span class="fpage" tagx="fpage" title="fpage">303</span><span class="lpage" tagx="lpage" title="lpage">17</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-62703-230-8_18">10.1007/978-1-62703-230-8_18</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23296618">23296618</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Simultaneous inhibition of key growth pathways in melanoma cells and tumor regression by a designed bidentate constrained helical peptide</span><span class="source" tagx="source" title="source">Biopolymers</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">344</span><span class="lpage" tagx="lpage" title="lpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/bip.22505">10.1002/bip.22505</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24839139">24839139</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso-Curbelo</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">RAB7 counteracts PI3K-driven macropinocytosis activated at early stages of melanoma development</span><span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">11848</span><span class="lpage" tagx="lpage" title="lpage">62</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26008978">26008978</a></span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh Durban</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deuker</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosenberg</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMahon</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Differential AKT dependency displayed by mouse models of BRAFV600E-initiated melanoma</span><span class="source" tagx="source" title="source">J Clin Invest</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">5104</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI69619">10.1172/JCI69619</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24200692">24200692</a></span></span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshimura</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishikawa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ihara</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taniguchi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taniguchi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">Suppression of lung metastasis of B16 mouse melanoma by N-acetylglucosaminyltransferase III gene transfection</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">8754</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.92.19.8754">10.1073/pnas.92.19.8754</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7568011">7568011</a></span></span></li> </ul> </div> </div>  </body></html>