<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Cell Biol</span><span class="iso-abbrev" title="iso-abbrev">J. Cell Biol</span><span class="publisher-id" title="publisher-id">jcb</span><span class="hwp" title="hwp">jcb</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">The Journal of Cell Biology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0021-9525</span><span class="issn-epub" tagx="issn" title="issn-epub">1540-8140</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Rockefeller University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6891092">6891092</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31594807">31594807</a></span><span class="publisher-id" title="publisher-id">201902101</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1083/jcb.201902101">10.1083/jcb.201902101</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research Articles</span>: <span class="subject" title="subject">Article</span></div>: <span class="subject" title="subject">30</span><span class="subject" title="subject">37</span><span class="subject" title="subject">36</span><span class="subject" title="subject">10</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Mechanosensing during directed cell migration requires dynamic actin polymerization at focal adhesions</div> <div class="alt-title" title="alt-title">
FA actin polymerization regulates mechanosensing</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-4515-4252</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puleo</span><span class="given-names" tagx="given-names" title="given-names">Julieann I.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-3670-6147</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">Sara S.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roman</span><span class="given-names" tagx="given-names" title="given-names">Mackenzie R.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-8871-9479</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watson</span><span class="given-names" tagx="given-names" title="given-names">Adam W.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eliato</span><span class="given-names" tagx="given-names" title="given-names">Kiarash Rahmani</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">Leilei</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-2784-1670</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saboda</span><span class="given-names" tagx="given-names" title="given-names">Kathylynn</span></span><a href="#aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roe</span><span class="given-names" tagx="given-names" title="given-names">Denise J.</span></span><a href="#aff4">4</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-9271-4091</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ros</span><span class="given-names" tagx="given-names" title="given-names">Robert</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">Frank B.</span></span><a href="#aff5">5</a><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-8103-4701</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouneimne</span><span class="given-names" tagx="given-names" title="given-names">Ghassan</span></span><a href="#aff1">1</a><a href="#cor7" /></span></div><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Department of Cellular and Molecular Medicine, University of Arizona Cancer Center, University of Arizona, Tucson, AZ</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Department of Physics, Center for Biological Physics, and Biodesign Institute, Arizona State University, Tempe, AZ</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">College of Optical Sciences, University of Arizona, Tucson, AZ</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">University of Arizona Cancer Center and Mel and Enid Zuckerman College of Public Health, University of Arizona, Tucson, AZ</span></span><span class="citation_author_institution" id="aff5">[5], <span class="institution" title="institution">The Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA</span></span><span class="citation_author_institution" id="aff6">[6], <span class="institution" title="institution">Department of Biology, Massachusetts Institute of Technology, Cambridge, MA</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Correspondence to Ghassan Mouneimne: <span class="email" tagx="email" title="email">gmouneimne@email.arizona.edu</span></div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-12-12</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-10-10</span></span><span class="volume" tagx="volume" title="volume">218</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">4215</span><span class="lpage" tagx="lpage" title="lpage">4235</span><span class="history" title="history"><span class="received" title="received">received: 2019-2-16</span><span class="rev-recd" title="rev-recd">rev-recd: 2019-6-16</span><span class="accepted" title="accepted">accepted: 2019-9-06</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license">[license_ref] http://www.rupress.org/terms/[license_ref] https://creativecommons.org/licenses/by-nc-sa/4.0/<span class="license-p" title="license-p">This article is distributed under the terms of an Attribution–Noncommercial–Share Alike–No Mirror Sites license for the first six months after the publication date (see <a href="http://www.rupress.org/terms/">http://www.rupress.org/terms/</a>). After six months it is available under a Creative Commons License (Attribution–Noncommercial–Share Alike 4.0 International license, as described at <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">https://creativecommons.org/licenses/by-nc-sa/4.0/</a>).</span></span></div><a class="self-uri" href="JCB_201902101_thumb.gif" title="self-uri">JCB_201902101_thumb.gif</a><div class="abstract" title="abstract"> <p>Puleo et al. identify the Ena/VASP protein EVL as a crucial actin polymerization factor at focal adhesions, which regulates focal adhesion maturation and mechanosensing. Interestingly, they show that EVL promotes mechanically directed motility and durotactic invasion yet suppresses chemotactic response.</p> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The mechanical properties of a cell’s microenvironment influence many aspects of cellular behavior, including cell migration. Durotaxis, the migration toward increasing matrix stiffness, has been implicated in processes ranging from development to cancer. During durotaxis, mechanical stimulation by matrix rigidity leads to directed migration. Studies suggest that cells sense mechanical stimuli, or mechanosense, through the acto-myosin cytoskeleton at focal adhesions (FAs); however, FA actin cytoskeletal remodeling and its role in mechanosensing are not fully understood. Here, we show that the Ena/VASP family member, Ena/VASP-like (EVL), polymerizes actin at FAs, which promotes cell-matrix adhesion and mechanosensing. Importantly, we show that EVL regulates mechanically directed motility, and that suppression of <i>EVL</i> expression impedes 3D durotactic invasion. We propose a model in which EVL-mediated actin polymerization at FAs promotes mechanosensing and durotaxis by maturing, and thus reinforcing, FAs. These findings establish dynamic FA actin polymerization as a central aspect of mechanosensing and identify EVL as a crucial regulator of this process.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Cancer Institute</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/100000054</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01 CA196885-01</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Cancer Institute</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/100000054</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01 CA196885-01</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">National Cancer Institute</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/100000054</span></div> </div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">University of Arizona Cancer Center</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/10.13039/100007345</span></div> </div><span class="award-id" tagx="award-id" title="award-id">P30CA023074</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Massachusetts Institute of Technology</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100006536</span></div> </div> </div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The physical microenvironment regulates many cellular functions, including cell migration (<a href="#bib56">van Helvert et al., 2018</a>). It is established that cell migration can be directed by the rigidity of the microenvironment, in a process known as durotaxis (<a href="#bib29">Lo et al., 2000</a>). Durotaxis has been implicated in physiological and pathological processes ranging from development (<a href="#bib12">Flanagan et al., 2002</a>; <a href="#bib51">Sundararaghavan et al., 2009</a>) to cancer progression (<a href="#bib3">Butcher et al., 2009</a>; <a href="#bib28">Levental et al., 2009</a>; <a href="#bib54">Ulrich et al., 2009</a>; <a href="#bib24">Lachowski et al., 2017</a>). Durotaxis requires cells to be adept at sensing mechanical stimuli (mechanosensing) and responding to anisotropic mechanical stimulation with directed motility. Although these processes are crucial aspects of durotaxis, the molecular mechanisms that regulate them remain largely unknown.</p> <p>Previous studies demonstrated that cells respond to the mechanical demands of the local microenvironment by dynamically altering their actin cytoskeleton at focal adhesions (FAs; <a href="#bib10">Choquet et al., 1997</a>; <a href="#bib3">Butcher et al., 2009</a>). In agreement with these findings, mathematical and experimental modeling suggested that the acto-myosin cytoskeleton at FAs mediates an oscillating traction force required for mechanically directed motility, the directional movement toward a mechanical stimulus (<a href="#bib42">Plotnikov et al., 2012</a>; <a href="#bib62">Wu et al., 2017</a>). However, the mechanisms that regulate these FA cytoskeletal dynamics and the distinctive role they play in mechanosensing, mechanically directed motility, and durotaxis have yet to be elucidated.</p> <p>Here, we identified the Ena/VASP family member, Ena/VASP-like (EVL), as a novel regulator of actin polymerization at FAs and found that EVL-mediated actin polymerization regulates cell-matrix adhesion and mechanosensing. We found that EVL plays a crucial role in regulating the mechanically directed motility of normal and cancer cells and, interestingly, that suppression of myosin contractility does not impede this process. Importantly, we found that suppression of <i>EVL</i> expression compromises 3D durotactic invasion of cancer cells. Furthermore, we show that response to chemotactic (biochemical) stimulation is enhanced in cells with reduced <i>EVL</i> expression, suggesting that EVL uniquely promotes response to mechanical cues. We propose a model in which EVL-mediated FA actin polymerization reinforces FAs during mechanical stimulation, thereby promoting mechanosensing, mechanically directed motility, and durotaxis.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="suppressionofmyosincontractilitydoesnotimpedemechanicallydirectedmotility" title="sec"> <div class="title" tagx="title" title="title">
Suppression of myosin contractility does not impede mechanically directed motility</div> <p>To examine mechanically directed motility, we determined the direction of motility during anisotropic mechanical stimulation of cells at nonleading edges (<a href="#bib29">Lo et al., 2000</a>; <a href="#bib42">Plotnikov et al., 2012</a>). We measured two directional motility parameters (<a href="#fig1">Fig. 1 a</a>): "sensing index" (cosine θ), a measurement of the direction of translocation with reference to the stimulation source and starting position; and "turning angles," a measurement of the change in direction over the course of the stimulation. Control breast cancer MCF7 cells rapidly directed their motility toward the mechanical stimulus, as revealed by positive sensing indices and acute turning angles (<a href="#fig1">Fig. 1, b–e</a>). Surprisingly, suppression of myosin contractility, a major component of FA cytoskeletal dynamics (<a href="#bib38">Parsons et al., 2010</a>; <a href="#bib1">Aguilar-Cuenca et al., 2014</a>), using Y-27632 did not impede mechanically directed motility on 35-kPa hydrogels, compared with control (<a href="#fig1">Fig. 1, b–e</a>; and Video 1). These data were validated using another myosin inhibitor, Blebbistatin (Fig. S1, a–d; and Video 1). Inhibition of myosin contractility was validated by loss of actin bundles and decrease in myosin light chain phosphorylation (Fig. S1 e). To examine whether higher microenvironmental forces required more myosin-mediated contractility, we examined mechanically directed motility on stiffer, 64-kPa hydrogels. Interestingly, on 64-kPa hydrogels, Y-27632 treatment did not impede mechanically directed motility, suggesting that even at a higher stiffness, myosin suppression does not impede this process (<a href="#fig1">Fig. 1, f–i</a>; and Video 1). These results suggest that MCF7 cells preserve their capacity to sense mechanical stimulation under myosin suppression.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p><b>Mechanically directed motility occurs under myosin suppression. (a)</b> Illustration depicting mechanically directed motility assays and sensing index and turning angle analyses. Crosshairs denote micropipette positions. <b>(b–e)</b> Control (no drug) and Y-27632 (25 µM)–treated MCF7 cells, plated on 35-kPa hydrogels, were mechanically stimulated. <b>(b)</b> Still images from representative time-lapse videos of control and Y-27632-treated cells (Video 1). Scale bars are 10 µm. <b>(c)</b> Corresponding cell traces at 0, 5, 10, 15, 20, 25, and 30 min, with starting positions in tan and final positions in blue. Crosshairs denote micropipette positions. <b>(d)</b> Rose plots show cumulative turning angles for control and Y-27632–treated cells. Black sectors denote turns in the direction of the mechanical stimulus, and gray sectors denote turns away from the mechanical stimulus. Data are collected from five independent experiments (<i>n</i> = 15 per condition). <b>(e)</b> Sensing indices of control and Y-27632–treated cells over time. Two-way ANOVA shows no significant difference in sensing index (P = 0.1170). Data are collected from five independent experiments; all data points are shown (<i>n</i> = 15 per condition; violin plot shows median and quartiles of sensing indices). <b>(f–i)</b> Control (no drug) and Y-27632 (25 µM)–treated MCF7 cells, plated on 64-kPa hydrogels, were mechanically stimulated. <b>(f)</b> Still images from representative time-lapse videos of control and Y-27632–treated cells (Video 1). Scale bars are 10 µm. <b>(g)</b> Corresponding cell traces at 0, 5, 10, 15, 20, 25, and 30 min, with starting positions in tan and final positions in blue. Crosshairs denote micropipette positions. <b>(h)</b> Rose plots show cumulative turning angles for control and Y-27632–treated cells. Black sectors denote turns in the direction of the mechanical stimulus, and gray sectors denote turns away from the mechanical stimulus. Data are collected from six independent experiments (<i>n</i> = 15 per condition). <b>(i)</b> Sensing indices of control and Y-27632–treated cells over time. Two-way ANOVA shows no significant difference in sensing index (P = 0.0917). Data are collected from six independent experiments; all data points are shown (<i>n</i> = 15 per condition; violin plot shows median and quartiles of sensing indices).</p>   </div> </div> <div class="evlpolymerizesactinatfas,whichregulatescell-matrixadhesionandpromotesfamaturation" title="sec"> <div class="title" tagx="title" title="title">
EVL polymerizes actin at FAs, which regulates cell-matrix adhesion and promotes FA maturation</div> <p>In addition to myosin contractility, actin polymerization is a crucial component of FA cytoskeletal dynamics and has been proposed to regulate force sensing (<a href="#bib14">Geiger et al., 2009</a>; <a href="#bib22">Iskratsch et al., 2014</a>; <a href="#bib6">Case and Waterman, 2015</a>; <a href="#bib62">Wu et al., 2017</a>). Ena/VASP are a family of actin polymerizing proteins that robustly localize to FAs (<a href="#fig2">Fig. 2 a</a>) and have been implicated in regulating FA actin remodeling (<a href="#bib13">Furman et al., 2007</a>; <a href="#bib20">Hirata et al., 2014</a>). To determine endogenous expression levels of Ena/VASP proteins, we performed quantitative PCR (qPCR) for MENA, VASP, and EVL in control MCF7 cells. Relative expression data suggest that while VASP and EVL are expressed to similar degrees in MCF7 cells, MENA is expressed at levels significantly lower than both EVL and VASP (Fig. S2 a). We examined the effect of altering the expression of <i>MENA</i>, <i>VASP</i>, and <i>EVL</i> (Fig. S2, b–d) on cell-matrix adhesion, by quantifying total FA area. <i>EVL</i> knockdown (KD) in MCF7 cells led to a dramatic decrease in FA area compared with controls, more significantly than KD of <i>MENA</i> or <i>VASP</i> (<a href="#fig2">Fig. 2, b and c</a>). These results were consistent, using two different sets of shRNA (Fig. S2, e and f). Additionally, <i>EVL KD</i> did not affect expression of MENA or VASP (Fig. S2 g). Furthermore, expression of a shRNA-resistant GFP-<i>EVL</i> (denoted shResist-EVL), but not of GFP-<i>MENA</i> or GFP-<i>VASP</i>, reversed the decrease in FA area observed in <i>EVL</i> KD cells (<a href="#fig2">Fig. 2, d and e</a>). Collectively, these results suggest that the role of EVL in cell-matrix adhesion is not fully redundant with that of MENA and VASP.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p><b>EVL is required for cell-matrix adhesion and FA maturation. (a)</b> Left: Immunofluorescent staining of MENA, EVL, and VASP in MCF7 cells. Middle: Immunofluorescent staining of paxillin shown with Ena/VASP. Right: Paxillin and Ena/VASP shown with phalloidin staining. Scale bars are 5 µm. <b>(b)</b> Representative inverted TIRF images of paxillin staining in control (LKO vector), <i>MENA</i> KD, <i>VASP</i> KD, and <i>EVL</i> KD MCF7 cells. Scale bars are 10 µm. <b>(c)</b> Dot plot shows quantification of FA area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 118 per condition; P values were determined using regression analysis; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(d)</b> Representative inverted TIRF images of paxillin staining from control (LKO vector) + GFP, <i>EVL</i> KD + GFP, <i>EVL</i> KD + GFP-VASP, <i>EVL</i> KD + GFP-MENA, and <i>EVL</i> KD + GFP-shResistEVL MCF7 cells. Boxed insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(e)</b> Dot plot shows quantification of FA area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 35 per condition; P values were determined using regression analysis; *, P ≤ 0.05; **, P ≤ 0.01; ***, P ≤ 0.001; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(f)</b> Left: Representative inverted TIRF images of paxillin staining in control (LKO vector) and <i>EVL</i> KD cells used in FA length quantification. Scale bars are 10 µm. Right: Magnified views of boxed areas from paxillin images. Scale bars are 5 µm. <b>(g)</b> Scatter plot shows individual FA length measurements from control and <i>EVL</i> KD cells. The lengths of &amp;gt;6,000 FAs from 15 cells per condition were measured; all data points are shown. <b>(h)</b> Volcano plot shows distribution of FA lengths in individual cells; all data points are shown (<i>n</i> ≥ 6,014 per condition; P values were determined using regression analysis; ****, P ≤ 0.0001; n.s., not significant; volcano plot shows median and quartiles of FA lengths; exact P values are found in Table S2). <b>(i)</b> Bubble plot shows mean FA length and number of FAs in individual control and <i>EVL</i> KD cells. Bubble area is scaled to total FA area. More than 6,000 FAs from 15 cells per condition were quantified; all cells are shown. <b>(j)</b> Bubble plot shows mean FA length and number of FAs in individual control and <i>EVL</i> KD U2OS cells. Bubble area is scaled to total FA area. More than 1,600 FAs from six cells per condition were quantified; all cells are shown. <b>(k)</b> Representative inverted TIRF images of paxillin staining in control (LKO vector) and <i>EVL KD</i> U2OS cells with corresponding quantification. Scale bars are 10 µm. Dot plots show quantification of FA area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 52 per condition; P values were determined using regression analysis; ****, P ≤ 0.0001; n.s., not significant; exact P values are found in Table S2; mean ± SEM). <b>(l)</b> Representative inverted TIRF images of paxillin staining in a panel of control (LKO vector) and <i>EVL</i> KD cells with corresponding quantification. Scale bars are 10 µm. Dot plots show quantification of FA areas from cell panel. Data are collected from three independent experiments per cell line; all data points are shown (<i>n</i> ≥ 76 per condition; P values were determined using regression analysis; ****, P ≤ 0.0001; n.s., not significant; exact P values are found in Table S2; mean ± SEM).</p>   </div> <p>To determine the effect of EVL on cell spreading, a functional parameter of adhesion, we measured total cell area after 24 h of plating on fibronectin, type I collagen, or laminin. Compared with control, <i>EVL</i> KD cells had significantly reduced cell area on all three substrates, while <i>MENA</i> KD and <i>VASP</i> KD cells did not (Fig. S2, h–j). These results suggest that the reduced cell-matrix adhesion in <i>EVL</i> KD cells is associated with compromised cell spreading. Some studies suggest that cell spreading, when restricted, could reciprocally suppress FA maturation (<a href="#bib8">Chen et al., 2003</a>). Therefore, we sought to determine the effect of <i>EVL</i> KD on adhesion while taking into consideration cell spreading as a factor. We examined FA area using regression analysis with cell area as a controlled variable (Fig. S3 a). After controlling for cell area, FA area in <i>EVL</i> KD cells was still significantly smaller than in control cells (Fig. S3 b). Additionally, we binned FA measurements by cell area to compare cells of similar spreading area as opposed to a pooled population. In bins of equivalent cell area, FA area in <i>EVL</i> KD cells was also significantly smaller than control (Fig. S3, c and d). These analyses suggest that regardless of cell spreading defects, <i>EVL</i> KD dramatically suppressed FA area. Moreover, we examined the maturation of individual FAs by measuring their length. These data revealed a significant decrease in the length of individual FAs in <i>EVL</i> KD cells compared with control, suggesting that EVL regulates adhesion at the level of single FAs (<a href="#fig2">Fig. 2, f–i</a>). In addition to the decrease in FA length, EVL KD had fewer FAs compared with control (<a href="#fig2">Fig. 2 i</a>). The effect of EVL KD on total FA area, number, and length was validated in human bone osteosarcoma (U2OS) cells (<a href="#fig2">Fig. 2, j and k</a>; and Fig. S4, a–c). Collectively, these results suggest that EVL directly regulates cell-matrix adhesion via FA maturation.</p> <p>The role of EVL in promoting cell-matrix adhesion was validated in additional cancer and normal cells, including mouse fibroblast cells (NIH-3T3), triple-negative human breast cancer cells (SUM159), human prostate cancer cells (PC3), human mammary epithelial cells (MCF10A), human colorectal cancer cells (Caco-2), and hormone receptor–positive human breast cancer cells (T47D). In these cell lines, <i>EVL</i> KD was associated with a significant decrease in FA area (<a href="#fig2">Figs. 2 l</a> and S4 d).</p> <p>To assess whether EVL could be regulating FAs via FA-localized actin polymerization, we performed in situ actin polymerization assays in MCF7 cells expressing GFP-EVL. In these assays, purified fluorescently labeled actin is provided to cells and incorporated into actively polymerizing filaments (<a href="#bib7">Chan et al., 1998</a>; <a href="#bib19">Hirata et al., 2008</a>), allowing for visualization of newly polymerized actin compared with the overall actin cytoskeleton. 3D N-STORM superresolution imaging reveals that EVL localizes to FA plaques (as delineated by paxillin immunofluorescence) and that actin is polymerized de novo within these structures (<a href="#fig3">Figs. 3 a</a> and S4 i). To mechanistically examine the role of EVL in actin polymerization at FAs, we quantified the rate of FA-localized actin polymerization in MCF7 cells expressing wild-type EVL or EVL domain deletion mutants (<a href="#fig3">Figs. 3 b</a> and S4 e). In situ actin polymerization assays followed by quantification of de novo actin fluorescence intensity within FAs revealed that <i>EVL</i> expression is sufficient to dramatically enhance actin polymerization at FAs, compared with controls (<a href="#fig3">Fig. 3, c and d</a>). Additionally, expression of shResist-<i>EVL</i> in <i>EVL</i> KD cells, which exhibit significantly less incorporation of labeled actin at FAs compared with control cells (<a href="#fig3">Fig. 3, c and d</a>), significantly increased FA actin polymerization (<a href="#fig3">Fig. 3, c and d</a>). In contrast, expression of an actin polymerization–deficient shResist-<i>EVL</i> mutant, ΔGF-PFN-<i>EVL</i>, which lacks the G-actin, F-actin, and profilin binding domains, in <i>EVL</i> KD cells did not alter FA actin polymerization significantly (<a href="#fig3">Fig. 3, b–d</a>). To determine whether EVL’s actin polymerization activity is involved in promoting cell-matrix adhesion, we examined the ability of EVL mutants to restore FA area in <i>EVL</i> KD MCF7 cells. Importantly, expression of ΔGF-PFN-<i>EVL</i>, or a second mutant with only the G-actin and F-actin deletions, ΔGF-<i>EVL</i>, in <i>EVL</i> KD cells was not sufficient to reverse the decrease in FA area seen in <i>EVL</i> KD cells (<a href="#fig3">Fig. 3, b, e, and f</a>). Furthermore, expression of an shResist-<i>EVL</i> mutant lacking the EVH1 domain, ΔEVH1-<i>EVL</i>, which, consistent with previous work (<a href="#bib2">Bear et al., 2000</a>; <a href="#bib23">Krause et al., 2003</a>), fails to localize to FAs, did not significantly alter FA actin polymerization in <i>EVL</i> KD cells or increase FA area in <i>EVL</i> KD cells compared with control (<a href="#fig3">Fig. 3, b–d, g, and h</a>). Collectively, these data suggest that EVL directly promotes actin polymerization at FAs, which is required for promoting cell-matrix adhesion.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p><b>EVL polymerizes actin at FAs, which regulates cell-matrix adhesion and promotes FA maturation. (a)</b> Representative N-STORM image of in situ actin polymerization in MCF7 cells expressing GFP-EVL. Left: TIRF images of exogenously introduced actin, GFP-EVL, GFP-EVL and exogenously introduced actin, and GFP-EVL and paxillin staining. Scale bars are 10 µm. White boxes denote region of interest. Right: N-STORM image of GFP-EVL, exogenous actin, and paxillin within region of interest taken from white boxes on TIRF images. Scale bar is 10 µm. Magnified cube is 3D view of N-STORM imaging of GFP-EVL, exogenous actin, and paxillin from boxed inset. Cube is 1 µm<sup>3</sup>. <b>(b)</b> Illustration depicting domains for full-length EVL and EVL domain deletion mutants (see Table S1 for sequences). <b>(c)</b> Representative TIRF images of barbed end labeled control (LKO vector) + GFP-EVL, control + GFP, <i>EVL</i> KD + GFP, <i>EVL</i> KD + GFP-shResist EVL, <i>EVL</i> KD + GFP-ΔEVH1 EVL, and <i>EVL</i> KD + GFP-ΔGF-PFN EVL MCF7 cells after in situ actin polymerization assays. Top: Exogenously introduced actin with paxillin staining. Middle: Phalloidin staining with paxillin. Bottom: Inverted single-channel images of exogenous actin incorporation shown for clarity. Boxed insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(d)</b> Dot plot shows quantification of exogenous actin fluorescence intensity within paxillin regions. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 82 per condition; P values were determined using regression analysis; *, P ≤ 0.05; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(e)</b> Representative inverted TIRF images of paxillin staining in control (LKO vector) + GFP, <i>EVL</i> KD + GFP, <i>EVL</i> KD + GFP-ΔGF EVL, <i>EVL</i> KD + GFP-ΔGF-PFN EVL, and <i>EVL</i> KD + GFP-shResist EVL in MCF7 cells. Boxed insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(f)</b> Dot plot shows quantification of FA area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 37 per condition; P values were determined using regression analysis; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(g)</b> Representative inverted TIRF images of paxillin staining in control (LKO vector) + GFP, <i>EVL</i> KD + GFP, <i>EVL</i> KD + GFP-ΔEVH1 EVL, and <i>EVL</i> KD + GFP-shResist EVL in MCF7 cells. Boxed insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(h)</b> Dot plot shows quantification of FA area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 44 per condition; P values were determined using regression analysis; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(i)</b> Illustration depicting domains for full-length EVL (EVH1<sup>EVL</sup>EVL) and EVH1 chimeric mutants, EVH1<sup>MENA</sup>EVL and EVH1<sup>VASP</sup>EVL (see Table S1 for sequences). <b>(j)</b> Representative inverted TIRF images of paxillin staining in control (LKO vector) + GFP, <i>EVL</i> KD + GFP, <i>EVL</i> KD + GFP-EVH1<sup>EVL</sup>EVL, <i>EVL</i> KD + GFP-EVH1<sup>MENA</sup>EVL, and <i>EVL</i> KD + GFP-EVH1<sup>VASP</sup>EVL in MCF7 cells. Scale bars are 10 µm. Inverted insets are of expression constructs, and overlay insets show expression construct with paxillin. Scale bars are 5 µm in overlay insets. <b>(k)</b> Dot plot shows quantification of FA area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 62 per condition; P values were determined using regression analysis; *, P ≤ 0.05; ***, P ≤ 0.001; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM).</p>   </div> <p>To further examine the mechanism of the unique function of EVL among Ena/VASP proteins at FAs, we generated chimeric mutants of EVL, in which the EVH1 domain of shResist-EVL was replaced with the EVH1 domain of MENA or VASP (<a href="#fig3">Figs. 3 i</a> and S4 f). Importantly, both the MENA<sup>EVH1</sup>EVL and VASP<sup>EVH1</sup>EVL chimeras appropriately localized to FAs (<a href="#fig3">Fig. 3 j</a>). However, quantification of FA area in EVL KD cells expressing either the MENA<sup>EVH1</sup>EVL or VASP<sup>EVH1</sup>EVL chimera demonstrate that neither chimera was sufficient to significantly increase the FA area of EVL KD cells, as opposed to expression of shResist-EVL (denoted EVL<sup>EVH1</sup>EVL; <a href="#fig3">Fig. 3 k</a>). These data suggest that the EVH1 domain of EVL is uniquely required for EVL’s function at FAs.</p> </div> <div class="evl-mediatedactinpolymerizationisnecessaryformechanosensing" title="sec"> <div class="title" tagx="title" title="title">
EVL-mediated actin polymerization is necessary for mechanosensing</div> <p>Mechanosensing describes the ability of cells to sense changes in the mechanical properties of the local microenvironment and modulate behaviors accordingly. We investigated the role of EVL-mediated FA actin polymerization in mechanosensing, as it is a prerequisite for mechanically directed motility since it allows cells to detect mechanical stimuli. Substrate rigidity, in particular, has been shown to regulate cell spreading (<a href="#bib41">Pelham and Wang, 1997</a>; <a href="#bib23">Krause et al., 2003</a>; <a href="#bib40">Pasapera et al., 2015</a>). Therefore, we performed comparative measurements of cell spreading on substrates of increasing rigidity to examine the sensitivity of cells to changes in substrate rigidity, as previously described (<a href="#bib41">Pelham and Wang, 1997</a>; <a href="#bib40">Pasapera et al., 2015</a>; <a href="#bib53">Swaminathan et al., 2016</a>). Analysis of cell area on 8-, 35-, and 64-kPa hydrogels revealed that <i>EVL</i> KD cells did not differentially spread on the three stiffnesses, whereas control cells exhibited a significant increase in spreading area among all stiffnesses (<a href="#fig4">Fig. 4 a</a>). Additionally, expression of ΔGF-PFN-<i>EVL</i> in <i>EVL</i> KD cells did not reverse the differential spreading deficiency of <i>EVL</i> KD cells, while expression of shResist-<i>EVL</i> displayed a significant increase in spreading on 35- and 64-kPa, compared with 8-kPa, gels (<a href="#fig4">Fig. 4 a</a>). These results suggest that EVL-mediated actin polymerization is required for the differential spreading of cells in response to changes in substrate rigidity. However, considering the spreading phenotypes of <i>EVL</i> KD cells seen on glass (Fig. S2, h–j), it is possible that the differential spreading capacity of these cells is masked by a general spreading deficit.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p><b>EVL-mediated actin polymerization is necessary for mechanosensing, and EVL is sufficient to promote mechanosensory signaling under myosin suppression. (a)</b> Dot plot shows quantification of total cell area of control (LKO vector) + GFP, <i>EVL</i> KD + GFP, <i>EVL</i> KD + GFP-ΔGF-PFN EVL, and <i>EVL</i> KD + GFP-shResist EVL MCF7 cells plated on 8-, 35-, or 64-kPa hydrogels. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 75 per condition; P values were determined using regression analysis; *, P ≤ 0.05; ***, P ≤ 0.001; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(b)</b> Representative inverted TIRF images of p-FAK staining in control (LKO vector) + GFP, <i>EVL</i> KD + GFP, <i>EVL</i> KD + GFP-ΔGF-PFN EVL, and <i>EVL</i> KD + GFP-shResist EVL MCF7 cells. Boxed insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(c)</b> Dot plot shows quantification of p-FAK area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 43 per condition; P values were determined using regression analysis; *, P ≤ 0.05; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(d)</b> Representative inverted TIRF images of paxillin staining in untreated and Y-27632–treated control (GFP) and GFP-<i>EVL</i>–overexpressing MCF7 cells. Insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(e)</b> Dot plots show quantification of paxillin area. Data are collected from four independent experiments; all data points are shown (<i>n</i> ≥ 51 per condition; P values were determined using regression analysis; **, P ≤ 0.01; ****, P ≤ 0.0001; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(f)</b> Representative inverted TIRF images of p-pax staining in untreated and Y-27632–treated control (GFP) and GFP-<i>EVL</i>–overexpressing MCF7 cells. Insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(g)</b> Dot plots show quantification of p-pax area. Data are collected from four independent experiments; all data points are shown (<i>n</i> ≥ 51 per condition; P values were determined using regression analysis; *, P ≤ 0.05; **, P ≤ 0.01; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM). <b>(h)</b> Representative inverted TIRF images of p-FAK staining in untreated and Y-27632–treated control (GFP) and GFP-<i>EVL</i>–overexpressing MCF7 cells. Insets are inverted images of expression constructs. Scale bars are 10 µm. <b>(i)</b> Dot plots show quantification of p-FAK area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 43 per condition; P values were determined using regression analysis; *, P ≤ 0.05; n.s., not significant; exact P values for all two-way comparisons are found in Table S3; mean ± SEM).</p>   </div> <p>To further examine mechanosensing, we quantified established mechanosensory signaling events at FAs, including phosphorylation of FAK at Tyrosine-397 (<a href="#bib57">Wang et al., 2001</a>; <a href="#bib50">Stutchbury et al., 2017</a>) and of paxillin at Tyrosine-118 (referred to herein as p-FAK and p-pax; <a href="#bib42">Plotnikov et al., 2012</a>; <a href="#bib50">Stutchbury et al., 2017</a>). Compared with control, <i>EVL</i> KD cells exhibited a significant decrease in p-FAK area, which was reversed by expression of shResist-<i>EVL</i> but not ΔGF-PFN-<i>EVL</i> (<a href="#fig4">Fig. 4, b and c</a>). In addition, overexpression of <i>EVL</i> significantly increased p-pax and total FA area, compared with control cells (<a href="#fig4">Fig. 4, d–g</a>). To examine if EVL is sufficient to promote mechanosensory signaling during suppression of myosin activity, we also overexpressed <i>EVL</i> in cells treated with myosin inhibitor. <i>EVL</i> overexpression significantly increased FA area in myosin-inhibited cells (<a href="#fig4">Fig. 4, d and e</a>), and importantly, significantly increased the area of p-pax and p-FAK (<a href="#fig4">Fig. 4, f–i</a>). To examine mechanosensory signaling independently of FA area, we also performed immunoblotting of p-FAK and p-pax in control and <i>EVL</i> KD MCF7 cells. Compared with control, <i>EVL</i> KD cells exhibited a significant decrease in p-FAK and p-pax levels, normalized to total FAK or paxillin, respectively (Fig. S4, g and h). Collectively, these data suggest that EVL promotes mechanosensory signaling at FAs and that EVL is sufficient to promote this signaling under baseline and myosin suppression conditions.</p> </div> <div class="evlregulatesmechanicallydirectedmotility" title="sec"> <div class="title" tagx="title" title="title">
EVL regulates mechanically directed motility</div> <p>To investigate the role of EVL in mechanically directed motility, we examined the effect of <i>EVL</i> KD on the response to anisotropic mechanical stimulation. In contrast to control cells, which exhibited directed motility toward the force stimulus, MCF7 and U2OS <i>EVL</i> KD cells translocated away from the force stimulus (<a href="#fig5">Fig. 5, a–d</a>; Fig. S5, g–j; and Videos 2 and 3). These results were consistent using a second EVL shRNA (Fig. S5, k–n; and Video 4). Importantly, the cell area in these experiments was similar between <i>EVL</i> KD and control cells, suggesting that the differences in mechanically directed motility are not necessarily due to an indirect effect of global alteration in cell spreading (Fig. S5, a–c).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p><b>EVL-mediated actin polymerization is required for mechanically directed motility. (a–d)</b> Control (LKO vector) and <i>EVL KD</i> MCF7 cells, plated on 35-kPa hydrogels, were mechanically stimulated. <b>(a)</b> Still images from representative time-lapse videos of control and <i>EVL</i> KD cells (Video 2). Scale bars are 10 µm. <b>(b)</b> Corresponding cell traces at 0, 5, 10, 15, 20, 25, and 30 min, with starting positions in tan and final positions in blue. Crosshairs denote micropipette positions. <b>(c)</b> Rose plots show cumulative turning angles for control and <i>EVL</i> KD cells. Black sectors denote turns in the direction of the force gradient, and gray sectors denote turns away from the force gradient. Data are collected from six independent experiments (<i>n</i> = 15 per condition). <b>(d)</b> Sensing indices of control and <i>EVL</i> KD cells over time. Two-way ANOVA shows a significant difference in sensing index between control and <i>EVL</i> KD cells (P &amp;lt; 0.0001). Data are collected from six independent experiments; all data points are shown (<i>n</i> = 15 per condition; violin plot shows median and quartiles of sensing indices). <b>(e)</b> Illustration depicting PDAI analysis. <b>(f)</b> Dot plot showing quantification of average PDAI per cell over time. Data are collected from six independent experiments; all data points are shown (<i>n</i> = 15 per condition; P values were determined using two-sample <i>t</i> test; ***, P ≤ 0.001; n.s., not significant; exact P values are found in Table S2; mean ± SEM). <b>(g)</b> Dot plot showing quantification of the number of de-adhesion events occurring within the proximal half of the cell. Data are collected from six independent experiments; all data points are shown (<i>n</i> = 15 per condition; P values were determined using Mann-Whitney test; ****, P ≤ 0.0001; n.s., not significant; exact P values are found in Table S2; mean ± SD). <b>(h–k)</b> Control (LKO vector) + GFP, <i>EVL</i> KD + GFP and <i>EVL</i> KD + GFP-ΔGF-PFN EVL MCF7 cells, plated on 35-kPa hydrogels, were mechanically stimulated. <b>(h)</b> Still images from representative time-lapse videos of control + GFP, <i>EVL</i> KD + GFP and <i>EVL</i> KD + GFP-ΔGF-PFN EVL cells (Video 5). Scale bars are 10 µm. <b>(i)</b> Corresponding cell traces at 0, 5, 10, 15, 20, 25 and 30 min, with starting positions in tan and final positions in blue. Crosshairs denote micropipette positions. <b>(j)</b> Rose plots show cumulative turning angles for control + GFP, <i>EVL</i> KD + GFP and <i>EVL KD</i> + GFP-ΔGF-PFN EVL cells. Black sectors denote turns in the direction of the force gradient and gray sectors denote turns away from the force gradient. Data are collected from three independent experiments (<i>n</i> = 15 per condition). <b>(k)</b> Sensing indices of control + GFP, <i>EVL</i> KD + GFP and <i>EVL</i> KD + GFP-ΔGF-PFN EVL cells over time. Two-way ANOVA shows a significant difference in sensing index between control, <i>EVL</i> KD, and <i>EVL</i> KD + GFP-ΔGF-PFN EVL cells (P &amp;lt; 0.0001). Data are collected from three independent experiments; all data points are shown (<i>n</i> = 15 per condition; violin plot shows median and quartiles of sensing indices).</p>   </div> <p>Interestingly, the directional translocation away from the force stimulus in <i>EVL</i> KD cells was accompanied by de-adhesion of the cell edge facing the stimulus (Video 2). To characterize this behavior, we established a proximal de-adhesion index (PDAI) that takes into account two factors: (a) %<i>P</i>, the percent change in cell area at the cell side proximal to the stimulus (this factor reflects the extent of de-adhesion and the direction of change, where negative values indicate de-adhesion at the proximal edge); and (b) |%<i>P</i> + %<i>D</i>|, the migration factor, which is the absolute value of the sum of the percent change in area at the proximal and distal sides, i.e., sides facing and opposite to the stimulus, respectively (this factor accounts for de-adhesion due to mere cell translocation during cell motility, during which de-adhesion on one side is equivalent to adhesion of the other side, leading to a migration factor close to zero; <a href="#fig5">Fig. 5 e</a>). Control MCF7 cells exhibited a PDAI ≥0, reflecting a migratory behavior toward the stimulus and proportional rates of adhesion and de-adhesion; <i>EVL</i> KD cells, on the other hand, exhibited a PDAI much lower than 0, suggesting that KD cells, in addition to their repulsive-migration phenotype, are disproportionately de-adhering at their proximal side (facing the stimulus; <a href="#fig5">Fig. 5, f and g</a>). These results are consistent with a diminished FA phenotype in the <i>EVL</i> KD cells and are suggestive of a destabilization of the FAs proximal to the force stimulus.</p> <p>Since actin cytoskeletal connections have been proposed to strengthen FAs (<a href="#bib62">Wu et al., 2017</a>), we investigated if EVL-mediated actin polymerization is specifically required for mechanically directed motility. We compared the directional response of control, <i>EVL</i> KD, and <i>EVL</i> KD MCF7 cells expressing the actin polymerization–deficient mutant, ΔGF-PFN-<i>EVL</i>, to mechanical stimulation. These data revealed that while control cells exhibited directed motility toward the force stimulus, <i>EVL</i> KD cells and <i>EVL</i> KD cells expressing ΔGF-PFN-<i>EVL</i> translocated away from the force stimulus (<a href="#fig5">Fig. 5, h–k</a>; Fig. S5 d; and Video 5). Furthermore, to investigate the specificity of EVL-mediated actin, we examined the involvement of formins, another family of actin-polymerizing proteins that have been implicated in regulating FA dynamics (<a href="#bib17">Gupton et al., 2007</a>; <a href="#bib34">Oakes et al., 2012</a>; <a href="#bib21">Iskratsch et al., 2013</a>; <a href="#bib62">Wu et al., 2017</a>), in mediating mechanically directed motility. We quantified the response of cells to mechanical stimulation after using the pan-formin inhibitor, SMIFH2. Treatment with SMIFH2 did not impede mechanically directed motility compared with control (Fig. S6, a–d; and Video 6). Together, these results suggest that EVL-mediated actin is specifically required for mechanically directed motility.</p> <p>Collectively, these results led us to propose a model in which EVL-mediated actin promotes mechanically directed motility by dynamically reinforcing FAs, thus preventing force-mediated FA destabilization (<a href="#fig6">Fig. 6 a</a>). In this model, since FA reinforcement functions to prevent FA destabilization during mechanical stimulation, it predicts that the extent of reinforcement required for FA stability would be dependent on the force demands of the microenvironment. To test this prediction, we examined the effect of <i>EVL</i> KD on the response to anisotropic mechanical stimulation on softer (8-kPa) hydrogels. Interestingly, on softer hydrogels, <i>EVL</i> KD cells responded directionally to the mechanical stimulus, while control cells demonstrated a reduced response (<a href="#fig6">Fig. 6, b–e</a>; Fig. S5 e; and Video 7). While the response of EVL KD cells on soft hydrogels is supportive of the model predictions, it is important to note that the directional responses of cells on soft and stiff hydrogels are not directly comparable, as gel stiffness affects the magnitude of mechanical stimulation as well as baseline FA biology.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p><b>Working model suggesting that FA actin polymerization is proportionally required to the force demands of the microenvironment. (a)</b> EVL-mediated FA actin polymerization is required for FA reinforcement in response to increasing environmental forces. Left (gray line): Low mechanical forces are below a cell’s mechanical detection range if cells have preexisting FA actin above the threshold required for FA stability. In these cases, no directional response is elicited and FAs remain at steady state. Left (red line): Low mechanical forces may be within the sensing range if cells do not have sufficient preexisting FA actin for FA stability upon stimulation. In these cases, mechanical stimulation promotes an increase in FA actin polymerization and induces mechanically directed motility. Middle (gray line): Moderate mechanical forces may be within the sensing range of cells with higher baseline FA actin if they do not have sufficient preexisting FA actin for FA stability upon stimulation. In these cases, mechanical stimulation promotes an increase in FA actin polymerization and induces mechanically directed motility. Middle (red line): Moderate mechanical forces may be above the tolerance range of cells with low capacity to dynamically polymerize sufficient FA actin, which leads to force-mediated de-adhesion and directional repulsion. Right (gray and red lines): High mechanical forces that exceed the capacity of cells to dynamically polymerize sufficient FA actin leads to force-mediated de-adhesion and repulsion. <b>(b–e)</b> Control (LKO vector) and <i>EVL</i> KD MCF7 cells, plated on 8-kPa hydrogels, were mechanically stimulated. <b>(b)</b> Still images from representative time-lapse videos of control and <i>EVL</i> KD cells (Video 7). Scale bars are 10 µm. <b>(c)</b> Corresponding cell traces at 0, 5, 10, 15, 20, 25, and 30 min, with starting positions in tan and final positions in blue. Crosshairs denote micropipette positions. <b>(d)</b> Rose plots show cumulative turning angles for control and <i>EVL</i> KD cells. Black sectors denote turns in the direction of the force gradient, and gray sectors denote turns away from the force gradient. Data are collected from four independent experiments (<i>n</i> = 15 per condition). <b>(e)</b> Sensing indices of control and <i>EVL</i> KD cells over time. Two-way ANOVA shows a significant difference in sensing index between control and <i>EVL</i> KD cells (P &amp;lt; 0.0001). Data are collected from four independent experiments; all data points are shown (<i>n</i> = 15 per condition; violin plot shows median and quartiles of sensing indices).</p>   </div> <p>Importantly, migration speed, measured on 35-kPa hydrogels in the absence of directional mechanical stimulation, is not significantly affected by <i>EVL</i> KD; yet the directionality of random migration is lower in <i>EVL</i> KD cells, compared with control (Fig. S6, e–h). Collectively, these findings suggest that EVL does not particularly enhance cell migration but rather promotes the ability to respond directionally toward a mechanical stimulus at matrices of higher stiffness.</p> <p>Since Ena/VASP proteins have been shown to heterotetramerize in vivo (<a href="#bib44">Riquelme et al., 2015</a>), we examined the sufficiency of EVL for promoting mechanically directed motility independently of MENA and VASP. We exogenously expressed <i>EVL</i> in genetically engineered mouse fibroblast (MV<sup>D7</sup>) cells, which lack expression of all three Ena/VASP family members (<a href="#bib2">Bear et al., 2000</a>). Consistent with the data from MCF7 cells, triple-null MV<sup>D7</sup> cells translocated away from the mechanical stimulus on 35-kPa hydrogels (<a href="#fig7">Fig. 7, a–d</a>; Fig. S5 f; and Video 8). However, expression of <i>EVL</i> in MV<sup>D7</sup> cells was sufficient to reverse this phenotype and promote mechanically directed motility (<a href="#fig7">Fig. 7, a-d and g</a>; and Video 8). Importantly, <i>EVL</i>-expressing cells exhibited a significantly larger FA area compared with triple-null cells (<a href="#fig7">Fig. 7, e–g</a>). These results suggest that EVL is sufficient to promote cell-matrix adhesion and mechanically directed motility in the absence of MENA and VASP.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p><b>Expression of <i>EVL</i> is sufficient to promote mechanically directed motility and cell-matrix adhesion in MV<sup>D7</sup> cells. (a–d)</b> GFP and GFP-EVL MV<sup>D7</sup> cells, plated on 35-kPa hydrogels, were mechanically stimulated. <b>(a)</b> Still images from representative time-lapse videos of GFP and GFP-EVL cells (Video 8). Scale bars are 10 µm. Boxed insets are images of expression constructs. <b>(b)</b> Corresponding cell traces at 0, 5, 10, 15, 20, 25, and 30 min, with starting positions in tan and final positions in blue. Crosshairs denote micropipette positions. <b>(c)</b> Rose plots show cumulative turning angles for GFP and GFP-EVL cells. Black sectors denote turns in the direction of the force gradient, and gray sectors denote turns away from the force gradient. Data are collected from five independent experiments (<i>n</i> ≥ 15 per condition). <b>(d)</b> Sensing indices of GFP and GFP-EVL cells over time. Two-way ANOVA shows a significant difference in sensing index between GFP and GFP-EVL cells (P &amp;lt; 0.0001). Data are collected from five independent experiments; all data points are shown (<i>n</i> ≥ 15 per condition; violin plot shows median and quartiles of sensing indices). <b>(e)</b> Top: Representative TIRF images of paxillin staining with GFP or GFP-EVL in MV<sup>D7</sup> cells. Boxed insets are inverted images of expression constructs. Bottom: Inverted single channel images of paxillin staining shown for clarity. Scale bars are 10 µm. <b>(f)</b> Quantification of FA area. Data are collected from three independent experiments; all data points are shown (<i>n</i> ≥ 121 per condition; P values were determined using regression analysis; ***, P ≤ 0.001; n.s., not significant; exact P values are found in Table S2; mean ± SEM). <b>(g)</b> Immunoblot of MV<sup>D7</sup> cells infected with control GFP vector or GFP-EVL. Top crop was probed with an antibody against EVL, and bottom crop was probed with an antibody against GAPDH.</p>   </div> </div> <div class="evlpromotesdurotacticinvasionwithin3dmatrix" title="sec"> <div class="title" tagx="title" title="title">
EVL promotes durotactic invasion within 3D matrix</div> <p>Mechanosensing and mechanically directed motility are central aspects of durotaxis (<a href="#bib42">Plotnikov et al., 2012</a>; <a href="#bib59">Wong et al., 2014</a>; <a href="#bib56">van Helvert et al., 2018</a>). During durotaxis, cells sense changes in matrix rigidity (mechanosense), direct themselves toward areas of higher rigidity (mechanically directed motility), and migrate in the direction of the stiffness gradient. To determine if EVL regulates durotaxis, we altered the expression of <i>EVL</i> and examined cellular response within a 3D durotactic invasion assay. To generate a durotactic stimulus, cells are embedded in a soft, central collagen matrix, which is surrounded by an outer collagen matrix rendered stiffer by cross-linking (<a href="#fig8">Fig. 8 a</a>). Importantly, the collagen concentration is held constant between the inner and outer matrices. The rigidity of both the inner and outer collagen matrices, as measured by atomic force microscopy–based nano-indentation, was 0.2 and 0.4 kPa, respectively (Fig. S6, i and j). <i>EVL</i> KD in MCF7 and U2OS cells significantly suppressed durotactic invasion into the stiffer matrix, compared with control (<a href="#fig8">Fig. 8, b–d</a>; and Video 9). In contrast, when 0.2-kPa collagen is used for both the inner and outer matrices, thus eliminating the durotactic stimulus, <i>EVL</i> KD led to a significant increase in invasion into the soft matrix (<a href="#fig8">Fig. 8, e and f</a>). These results suggest that, without the challenge of a stiffness gradient, <i>EVL</i> KD cells are more invasive, which is consistent with previous work (<a href="#bib32">Mouneimne et al., 2012</a>; <a href="#bib37">Padilla-Rodriguez et al., 2018</a>) and highlights the specific involvement of EVL in durotactic invasion. Importantly, using gelatin zymography to measure the ability to degrade collagen, we found no significant difference in proteolytic activity between control and <i>EVL</i> KD cells; this suggests that the suppression of durotactic invasion in <i>EVL</i> KD cells is not caused by inhibition of matrix degradation (Fig. S6, k–m). Together, these results suggest that EVL distinctively promotes durotactic invasion.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p><b>EVL promotes durotactic invasion within 3D matrix. (a)</b> Illustration depicting 3D invasion assays. <b>(b and c)</b> Control (LKO vector) and <i>EVL</i> KD MCF7 cells were plated within a durotactic invasion assay. <b>(b)</b> Large-stitch images of control and <i>EVL</i> KD invasion assays, with red dots denoting invaded cells. Scale bars are 500 µm; 0.2 and 0.4 kPa correspond to inner and outer matrices, respectively. <b>(c)</b> Quantification of cells invaded from 0.2–0.4-kPa matrix. Data are expressed as fold change and are collected from four independent experiments; all experiments are shown (<i>n</i> = 4 per condition; P values were determined using a one-sample <i>t</i> test with a hypothetical value of 1; *, P ≤ 0.05; n.s., not significant; exact P values are found in Table S2; mean ± SEM). <b>(d)</b> Control (LKO vector) and <i>EVL</i> KD U2OS cells were plated within a durotactic invasion assay. Left: Still images from time-lapse videos of invading control and <i>EVL</i> KD U2OS cells, with red lines denoting boundaries between inner and outer matrices and red dots denoting invaded cells; images are taken from final time point of Video 9. Scale bars are 50 µm. Right: Quantification of cells invaded from 0.2–0.4-kPa matrix. Data are expressed as fold change and are collected from three independent experiments; all experiments are shown (<i>n</i> = 3 per condition; P values were determined using a one-sample <i>t</i> test with a hypothetical value of 1; **, P ≤ 0.01; n.s., not significant; exact P values are found in Table S2; mean ± SEM). <b>(e and f)</b> Control (LKO vector) and <i>EVL</i> KD MCF7 cells were plated within a soft invasion assay. <b>(e)</b> Large-stitch images of control and <i>EVL</i> KD invasion assays, with red dots denoting invaded cells. Scale bars are 500 µm; 0.2 and 0.2 kPa correspond to inner and outer matrices, respectively. <b>(f)</b> Quantification of cells invaded from 0.2–0.2-kPa matrix. Data are expressed as fold change and are collected from four independent experiments; all experiments are shown (<i>n</i> = 4 per condition; P values were determined using a one-sample <i>t</i> test with a hypothetical value of 1; *, P ≤ 0.05; n.s., not significant; exact P values are found in Table S2; mean ± SEM).</p>   </div> </div> <div class="evlinhibitschemotacticsensing" title="sec"> <div class="title" tagx="title" title="title">
EVL inhibits chemotactic sensing</div> <p>Notwithstanding its role in promoting durotactic invasion, suggested herein, EVL has previously been shown to suppress invasive behavior (<a href="#bib32">Mouneimne et al., 2012</a>; <a href="#bib37">Padilla-Rodriguez et al., 2018</a>); this suggests that the regulation of invasion by EVL may be context dependent. In many physiological and pathological contexts, two main types of stimuli guide cell migration: biomechanical and biochemical. Therefore, we investigated the role of EVL in chemo-directed motility, the directional response toward biochemical gradients. Since MCF7 cells express insulin-like growth factor (IGF) receptor (<a href="#bib49">Stephen et al., 2001</a>), we introduced a focal gradient of IGF to nonleading edges of single cells (<a href="#bib31">Mouneimne et al., 2006</a>) to induce chemo-directed motility. Like mechanically directed motility, chemo-directed motility was quantified using the sensing index and turning angles of stimulated cells. Interestingly, suppression of <i>EVL</i> expression enhanced the chemotactic response of cells compared with control (<a href="#fig9">Fig. 9a-d</a> and Video 10). These results suggest that EVL plays a dichotomous role in regulating the response of cells to directional cues: it inhibits chemo-directed motility and promotes mechanically directed migration.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9.</span></a></div>   <p><b>EVL suppresses chemotactic sensing. (a–d)</b> Control (LKO vector) and <i>EVL</i> KD MCF7 cells were stimulated with an IGF gradient. <b>(a)</b> Still images from representative time-lapse videos of control and <i>EVL</i> KD cells (Video 10). Scale bars are 10 µm. Boxed insets are images of IGF gradients, provided to cells for 2 min. <b>(b)</b> Corresponding cell traces at 0, 5, 10, 15, 20, 25, and 30 min, with starting positions in tan and final positions in blue. Crosshairs denote micropipette positions. <b>(c)</b> Rose plots show cumulative turning angles for control and <i>EVL</i> KD cells. Black sectors denote turns in the direction of the IGF gradient, and gray sectors denote turns away from the IGF gradient. Data are collected from five independent experiments (<i>n</i> = 15 per condition). <b>(d)</b> Sensing indices of control and <i>EVL</i> KD cells over time. Groups are plotted separately for clarity. Two-way ANOVA shows a significant difference in sensing index between control and <i>EVL</i> KD cells (P &amp;lt; 0.0001). Data are collected from five independent experiments; all data points are shown (<i>n</i> = 15 per condition; violin plot shows median and quartiles of sensing indices).</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Durotaxis, migration toward increasing stiffness, has been implicated in various biological and pathological processes, including cancer progression (<a href="#bib3">Butcher et al., 2009</a>; <a href="#bib28">Levental et al., 2009</a>; <a href="#bib54">Ulrich et al., 2009</a>; <a href="#bib24">Lachowski et al., 2017</a>). Durotaxis requires cells to be capable of (1) sensing a mechanical stimulus (mechanosensing), (2) directing their motility toward the stimulus (mechanically directed motility), and (3) migrating in the direction of the stimulus. Although essential to durotaxis, a mechanistic understanding of these processes has been unclear. Here, we show that EVL promotes actin polymerization at FAs and that EVL-mediated FA actin polymerization regulates cell-matrix adhesion and mechanosensing. We found that EVL is a distinct regulator of mechanically directed motility, and that EVL promotes durotactic invasion. We propose a model in which EVL-mediated FA actin polymerization reinforces FAs and promotes the response to mechanical stimulation.</p> <div class="evl-mediatedactinpolymerizationatfasiscrucialforcell-matrixadhesionandmechanosensing" title="sec"> <div class="title" tagx="title" title="title">
EVL-mediated actin polymerization at FAs is crucial for cell-matrix adhesion and mechanosensing</div> <p>Previous reports have suggested that actin is polymerized at FAs; however, these studies did not identify an actin polymerization factor responsible for this process that localizes to FAs throughout their lifespan (<a href="#bib17">Gupton et al., 2007</a>; <a href="#bib9">Choi et al., 2008</a>; <a href="#bib19">Hirata et al., 2008</a>; <a href="#bib21">Iskratsch et al., 2013</a>). Our data suggest that EVL is responsible for promoting actin polymerization at FAs. Importantly, we show that EVL is specifically required for promoting cell-matrix adhesion in a panel of cancer and normal cell lines. Our results are consistent with previous work showing that Ena/VASP proteins localize to FAs (<a href="#bib13">Furman et al., 2007</a>) and regulate actin (<a href="#bib15">Gertler et al., 1996</a>; <a href="#bib26">Lambrechts et al., 2000</a>) and FA dynamics (<a href="#bib60">Worth et al., 2010</a>; <a href="#bib18">Gupton et al., 2012</a>). Furthermore, we found that expression of MENA or VASP did not restore the reduced-adhesion phenotypes observed in <i>EVL</i> KD cells, and through use of chimeric mutants, we found that the EVH1 domain of EVL is unique among Ena/VASP proteins and is specifically required for EVL’s function at FAs. These data corroborate mounting evidence that Ena/VASP proteins, once thought to be functionally interchangeable (<a href="#bib27">Laurent et al., 1999</a>), have unique and specific roles in regulating FAs. For example, MENA uniquely binds α5 integrin and modulates adhesion signaling through an actin-independent mechanism (<a href="#bib18">Gupton et al., 2012</a>), and VASP cooperates with RIAM (rap-1 interacing molecule) and zyxin to regulate β1 integrin dynamics (<a href="#bib60">Worth et al., 2010</a>) and stress fiber integrity (<a href="#bib47">Smith et al., 2010</a>), respectively. Therefore, while our data establish EVL as the primary Ena/VASP protein responsible for actin polymerization–mediated cell-matrix adhesion, MENA and VASP provide distinct yet indispensable contributions to FAs.</p> <p>In addition, our results suggest that EVL-mediated actin polymerization at FAs is required for mechanosensing. The use of EVL domain deletion mutants corroborates that the ability of EVL to mediate mechanosensing is dependent on its actin polymerization activity. Furthermore, our data suggest that expression of <i>EVL</i> is sufficient to promote cell-matrix adhesion and mechanosensory signaling under baseline and myosin suppression conditions. These results are in agreement with previous work suggesting that FA maturation in response to extracellular force is more dependent on actin polymerization and filament integrity than on myosin contractility (<a href="#bib45">Riveline et al., 2001</a>), and that an actin template is sufficient for sustaining FA signaling events under myosin inhibition (<a href="#bib34">Oakes et al., 2012</a>). Collectively, these data suggest that while myosin is a regulator of FA maturation and mechanosensing (<a href="#bib39">Pasapera et al., 2010</a>, <a href="#bib40">2015</a>; <a href="#bib1">Aguilar-Cuenca et al., 2014</a>), EVL is sufficient to promote these processes, at least in part, in the presence of low myosin contractility.</p> </div> <div class="evlregulatesmechanicallydirectedmotility,whilesuppressionofmyosincontractilitydoesnotimpedethisprocess" title="sec"> <div class="title" tagx="title" title="title">
EVL regulates mechanically directed motility, while suppression of myosin contractility does not impede this process</div> <p>Our data suggest that EVL is a crucial regulator of mechanically directed motility. We demonstrate that <i>EVL</i> KD cells display significantly reduced directed motility toward mechanical stimuli, as well as disproportional de-adhesion upon anisotropic mechanical stimulation. Importantly, we show that expression of the actin-deficient mutant ΔGF-PFN-<i>EVL</i> was not sufficient to restore mechanically directed motility in <i>EVL</i> KD cells. Considering previous works showing that FAs and their associated actin are dynamically remodeled in response to changes in environmental forces (<a href="#bib10">Choquet et al., 1997</a>; <a href="#bib11">Elosegui-Artola et al., 2014</a>; <a href="#bib58">Weng et al., 2016</a>), our results suggest a model in which EVL, via FA actin polymerization, is an integral component of this response. In this model, the requirements for FA actin polymerization shift in relation to mechanical demands, and this polymerization is, at least in part, regulated by <i>EVL</i> expression (<a href="#fig6">Fig. 6 a</a>). In support of this model, we found that <i>EVL</i> KD cells are capable of exhibiting mechanically directed motility on softer, 8-kPa hydrogels. However, it is important to note that mechanical stimulation of cells plated on hydrogels of different stiffness are not directly comparable, as hydrogel stiffness alters both the magnitude of force exerted during stimulation and the baseline FA biology of plated cells. Therefore, future work is necessary to fully characterize the specific relationships between absolute microenvironmental stiffness, force magnitude during mechanical stimulation, and actin polymerization requirements during mechanical response.</p> <p>Importantly, Ena/VASP proteins are not the only actin-polymerizing factors regulating FA dynamics in cells; formins are another family of actin-polymerizing proteins shown to affect FA dynamics (<a href="#bib17">Gupton et al., 2007</a>; <a href="#bib34">Oakes et al., 2012</a>; <a href="#bib21">Iskratsch et al., 2013</a>). Although our formin-inhibition data suggest that suppression of formin activity does not significantly impede mechanically directed motility, it is possible that remaining levels of formin activity in these cells are sufficient to support their motility. More directed studies are needed to determine if formins collaborate with Ena/VASP, for example, by mediating actin nucleation (<a href="#bib5">Campellone and Welch, 2010</a>), to contribute to this process.</p> <p>Surprisingly, we found that suppression of myosin activity does not impede mechanically directed motility. Myosin contractility has been implicated in single-cell (<a href="#bib43">Raab et al., 2012</a>) and collective cell durotaxis (<a href="#bib52">Sunyer et al., 2016</a>), and in cell migration in general (<a href="#bib55">Vallenius, 2013</a>). Nonetheless, a distinct role for myosin in the process of sensing directional mechanical stimuli has not been reported. Although it informs long-range migration, the sensing of directional cues can be regulated by distinct molecular processes (<a href="#bib30">Mouneimne et al., 2004</a>, <a href="#bib31">2006</a>). Importantly, suppression of myosin contractility did perturb retraction of the cell rear during mechanically directed motility, which is consistent with previous reports (<a href="#bib52">Sunyer et al., 2016</a>). Therefore, myosin may be dispensable in directing cells toward a mechanical stimulus (mechanically directed motility), while still being crucial for long-range migration toward increasing stiffness (durotaxis). Due to these complexities, future work is needed to fully characterize the specific role of myosin during each of these diverse mechanically regulated functions.</p> </div> <div class="evlspecificallypromotesdurotacticinvasion" title="sec"> <div class="title" tagx="title" title="title">
EVL specifically promotes durotactic invasion</div> <p>Our data suggest that EVL promotes durotactic invasion into 3D matrix; this is consistent with its essential role in mechanosensing and mechanically directed motility and highlights the significance of these processes for durotactic invasion. Importantly, we show that suppression of <i>EVL</i> expression enhances invasion into uniformly soft 3D matrix, which is consistent with our previous work demonstrating that EVL plays a suppressive role in breast cancer cell invasion (<a href="#bib32">Mouneimne et al., 2012</a>; <a href="#bib37">Padilla-Rodriguez et al., 2018</a>). These data emphasize a role for EVL in specifically mediating a response to mechanical stimulation, as opposed to promoting migration and invasion in general. Additionally, microenvironmental factors that guide cell migration are not limited to mechanical cues; cell migration can also be guided by biochemical gradients as in chemotaxis (<a href="#bib46">Roussos et al., 2011</a>). Here, we show that, unlike mechanically directed motility, EVL is suppressive to chemo-directed motility. Together, these data suggest that migration directed by mechanical and biochemical cues is regulated through distinct and possibly antagonistic mechanisms, and that EVL is selectively promoting mechanically directed modes of migration.</p> <p>Importantly, previous studies have shown that MENA plays crucial roles in promoting chemotaxis and haptotaxis (<a href="#bib16">Goswami et al., 2009</a>; <a href="#bib35">Oudin et al., 2016a</a>,<a href="#bib36">b</a>), modes of directed cell migration reliant on soluble and immobilized ligand gradients, respectively. Combined with our findings, these studies suggest that Ena/VASP proteins play divergent roles in directed cell migration and invasion. Importantly, these proteins could potentially play a significant role in integrating biochemical and mechanical signals from the cell microenvironment to guide migration in physiological and pathological contexts. While future studies will be required to address the complexity of chemical and mechanical signal integration during migration, mechanistic studies on the role of Ena/VASP proteins in signal sensing provide a valuable foundation to this emerging field.</p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="cellculture" title="sec"> <div class="title" tagx="title" title="title">
Cell culture</div> <p>MCF7, U2OS, NIH-3T3, and HEK293T cells were grown in high-glucose DMEM (Corning), supplemented with 2 mM <div class="sc_UNKNOWN">
l</div>-glutamine (Corning), 10% FBS (Gibco), and 100 U/ml penicillin with 100 µg/ml streptomycin (Corning). Caco-2 cells were grown in high glucose DMEM, supplemented with 2 mM <div class="sc_UNKNOWN">
l</div>-glutamine, 20% FBS, and 100 U/ml penicillin with 100 µg/ml streptomycin. T47D cells were grown in RPMI 1640 medium (Corning), supplemented with 2 mM <div class="sc_UNKNOWN">
l</div>-glutamine, 10% FBS, 5 µg/ml insulin (Roche), and 100 U/ml penicillin with 100 µg/ml streptomycin. PC3 cells were grown in high-glucose DMEM, supplemented with 10% FBS and 100 U/ml penicillin with 100 µg/ml streptomycin. MCF10A cells were grown in DMEM/Nutrient Mixture F-12 (DMEM/F-12; Corning), supplemented with 5% horse serum (Gibco), 20 ng/ml epidermal growth factor (Life Technologies), 0.5 µg/ml hydrocortisone (Sigma-Aldrich), 100 ng/ml cholera toxin (Sigma-Aldrich), 10 µg/ml insulin, and 100 U/ml penicillin with 100 µg/ml streptomycin. SUM159 cells were grown in Ham’s F12 medium (Corning), supplemented with 5% FBS, 1 µg/ml hydrocortisone, 5 µg/ml insulin, 10 mM Hepes (Corning), and 100 U/ml penicillin with 100 µg/ml streptomycin. MV<sup>D7</sup> cells were grown in high-glucose DMEM, supplemented with 15% FBS, 2 mM <div class="sc_UNKNOWN">
l</div>-glutamine, 100 U/ml penicillin with 100 µg/ml streptomycin, and 50 U/ml mouse interferon-γ (Millipore). MCF7, T47D, SUM159, Caco-2, PC3, U2OS, MCF10A, NIH-3T3, and HEK293T cells were maintained in a 37°C humidified incubator under 5% CO<sub>2</sub>. MV<sup>D7</sup> cells were maintained in a 32°C humidified incubator under 5% CO<sub>2</sub>, as previously described (<a href="#bib2">Bear et al., 2000</a>).</p> </div> <div class="plasmids,antibodies,andreagents" title="sec"> <div class="title" tagx="title" title="title">
Plasmids, antibodies, and reagents</div> <p>All Ena/VASP constructs were cloned into a modified pCIG3 lentiviral expression vector (Addgene plasmid #78264; a gift from Felicia Goodrum, University of Arizona, Tucson, AZ), in which GFP was replaced with a blasticidin resistance cassette to create pCIB (CMVie-IRES-BlastR). pCIB and pCIB alternate MCS have been deposited to Addgene as #119863 and #120862, respectively. <i>Mus musculus</i> EVL cDNA and <i>M. musculus</i> VASP cDNA were derived from MSCV-GFP-EVL and MSCV-GFP-VASP, respectively (gifts from Frank Gertler, Massachusetts Institute of Technology, Cambridge, MA). <i>M. musculus</i> MENA cDNA was derived from pDONR223-MENA (a gift from Lisa Gallegos, Harvard Medical School, Boston, MA). pLKO.1-TRC cloning vector was a gift from David Root (Broad Institute, Cambrige, MA; Addgene plasmid #10878). Multiple Ena/VASP shRNAs were tested and those used for experimentation included: pLKO-EVL shRNAs: shRNA#01, GE Dharmacon TRCN0000063869 (antisense 5′-TAC​TAG​GAT​CTT​CCA​TTT​GGC-3′), and shRNA#02, TRCN0000063871 (antisense 5′-TTA​CTT​GCA​AAG​TTT​AAG​CCG-3′); pLKO-VASP shRNAs: shRNA#01, GE Dharmacon TRCN0000117147 (antisense 5′-AAA​GTG​AAT​CTA​GAA​CAA​GGG-3′), and shRNA#02, TRCN0000117148 (antisense 5′-AGA​AGA​CGA​CTT​CAT​CCT​TGG-3′); and pLKO-MENA shRNAs: shRNA#01, GE Dharmacon TRCN0000061824 (antisense 5′-TTA​GAG​GAG​TCT​CAA​CAG​AGG-3′), and shRNA#02, TRCN0000061827 (antisense 5′-TTT​GAC​TTG​CTC​AGT​TCC​TGC-3′). shRNA-resistant EVL was generated via PCR using inverse PCR site-directed mutagenesis of <i>M. musculus</i> EVL cDNA derived from MSCV-GFP-EVL (TRCN0000063869 shRNA recognition site converted to 5′-GTC​AGA​CCG​AAG​ACC​CAA​GC-3′). EVL domain deletion mutants were generated using inverse PCR site-directed mutagenesis of shRNA-resistant EVL and chimeras were generated by PCR of the EVH1 domain from MSCV-GFP-VASP or pDONR223-MENA and subcloning into shRNA-resistant EVL to replace the native EVL EVH1 sequence (sequences found in Table S1). The sequences of all constructed plasmids were confirmed. Antibodies used were as follows: mouse anti-Paxillin (clone 349; BD Biosciences, 612405), rabbit anti-Tyr118-phospho-paxillin (CST, 2541S), rabbit anti-Tyr397-phospho-FAK (clone 141-9; Invitrogen, 44-625G), mouse anti-Ser19-phospho-MLC (CST, 3675), rabbit anti-VASP (clone 9A2; CST, 3132S), rabbit anti-MENA (Sigma-Aldrich, HPA028696), rabbit anti-EVL (Sigma-Aldrich, HPA018849), mouse and rabbit anti-EVL (kindly provided by Frank Gertler), mouse anti-actin (clone C4; Abcam, AB3280; and ProteinTech, 66009-1), and rabbit anti-GAPDH (clone 14C10; CST, 2118S). Drugs used were as follows: Blebbistatin (Sigma-Aldrich), Y-27632 (Tocris), and SMIFH2 (EMD Millipore).</p> </div> <div class="lentiviralproductionandinfections" title="sec"> <div class="title" tagx="title" title="title">
Lentiviral production and infections</div> <p>HEK293T cells were transfected at 60% confluence with transfer plasmid and second-generation lentiviral packaging system (psPAX2 and pMD2.G, Addgene #12260 and #12259; gifts from Didier Trono, École polytechnique fédérale de Lausanne, Lausanne, Switzerland), using FuGENE HD (Promega) in OptiMEM (Thermo Fisher Scientific). Virus was collected 24 and 48 h after transfection and clarified by 0.45-µm filters. Recipient cells were plated to reach 60% confluence after 24 h, and virus was added to the cells with media containing Polybrene (10 µg/ml). Puromycin selection (2 µg/ml) was started 24 h after infection.</p> </div> <div class="immunoblotting" title="sec"> <div class="title" tagx="title" title="title">
Immunoblotting</div> <div class="mechanosensingexperiments" title="sec"> <div class="title" tagx="title" title="title">
Mechanosensing experiments</div> <p>Cells were lysed 8 h after plating on fibronectin (10 µg/ml; Corning)-coated dishes in buffer containing 140 mM NaCl, 10 mM Tris, pH 8, 1 mM EDTA, 0.5 mM EGTA, 1% Triton X-100, 0.1% sodium deoxycholate, and 0.1% SDS with protease and phosphatase inhibitors (Boston Bio Products). Equal amounts of protein from each sample were resolved by SDS-PAGE and transferred onto nitrocellulose membranes. Membranes were blocked in Odyssey Blocking Buffer (LI-COR) for 1 h and incubated with primary antibodies at 4°C overnight. After washing, membranes were incubated with secondary antibodies conjugated to either Alexa Fluor 680 or 790 (Thermo Fisher Scientific) for 1 h. Immunoblots were scanned using Odyssey CLx imager (LI-COR).</p> </div> <div class="allotherimmunoblots" title="sec"> <div class="title" tagx="title" title="title">
All other immunoblots</div> <p>Cells were lysed in buffer containing 10% glycerol, 1% NP-40, 50 mM Tris, pH 7.5, 200 mM NaCl, and 2 mM MgCl<sub>2</sub> with protease and phosphatase inhibitors (Boston Bio Products). Total protein concentrations were determined using a BSA standard curve in a Bradford assay (Thermo Fisher Scientific). Equal amounts of protein from each sample were resolved by SDS-PAGE and transferred onto polyvinylidene difluoride membranes. Membranes were blocked in 5% BSA for 1 h and incubated with a primary antibody at 4°C overnight. After washing, membranes were incubated with a secondary antibody conjugated to HRP (Invitrogen) for 1 h. Immunoblots were developed with Clarity Western ECL Substrate (Bio-Rad). To probe with a second antibody, membranes were stripped with 0.5 M sodium hydroxide for 15 min at room temperature, washed, and blocked. Membranes were then incubated with a primary antibody of a different species from the first antibody used, and procedures were performed as above.</p> </div> </div> <div class="zymography" title="sec"> <div class="title" tagx="title" title="title">
Zymography</div> <p>Cells were lysed in buffer containing 10% glycerol, 1% NP-40, 50 mM Tris, pH 7.5, 200 mM NaCl, and 2 mM MgCl<sub>2</sub>. Lysates were cleared via centrifugation at 14,000 rpm at 4°C. Total protein concentrations were determined using a BSA standard curve in a Bradford assay (Thermo Fisher Scientific). Equal amounts of protein from each sample were resolved by electrophoresis on gelatin zymogram gels (Thermo Fisher Scientific) under nonreducing conditions. Gels were washed in renaturing and developing buffers (Thermo Fisher Scientific) and incubated in developing buffer at 37°C overnight. Gels were stained with Coomassie blue (Bio-Rad). Band intensities were quantitated using ImageJ (National Institutes of Health).</p> </div> <div class="real-timeqpcr" title="sec"> <div class="title" tagx="title" title="title">
Real-time qPCR</div> <p>Total RNA was isolated using Isolate II RNA kit (Bioline), and cDNA was synthesized from 1 µg of RNA using XLA script cDNA kit (Worldwide Medical Products). SYBR green PCR mix (Bioline) was used for RT-qPCR on the ABI Fast 7500 system using default fast cycling parameters. Samples were run in triplicate in each experiment and relative cDNA levels for <i>Homo sapiens VASP</i> (forward: 5′-CTG​GGA​GAA​GAA​CAG​CAC​AAC​C-3′; reverse: 5′-AGG​TCC​GAG​TAA​TCA​CTG​GAG​C-3′), <i>MENA</i> (forward: 5′-GGT​CGT​GAT​AAA​CTG​TGC​CAT-3′; reverse: 5′-ACC​ATA​CAC​CTG​TCT​AGC​ATC​T-3′), and <i>EVL</i> (forward: 5′-CTT​CCG​TGA​TGG​TCT​ACG​ATG-3′; reverse: 5′-TGC​AAC​TTG​ACT​CCA​ACG​ACT-3′) or <i>M. musculus EVL</i> (forward: 5′-TGA​GAG​CCA​AAC​GGA​AGA​CC-3′; reverse: 5′-TTC​TGG​ACA​GCA​ACG​AGG​AC-3′) were normalized to the <i>Homo sapiens EEF1A1</i> housekeeping gene (forward: 5′-TCG​GGC​AAG​TCC​ACC​ACT​AC-3′; reverse: 5′-CCA​AGA​CCC​AGG​CAT​ACT​TGA-3′) and the <i>M. musculus EEF1A1</i> housekeeping gene (forward: 5′-CAA​CAT​CGT​CGT​AAT​CGG​ACA-3′; reverse: 5′-GTC​TAA​GAC​CCA​GGC​GTA​CTT-3′). Fold change comparisons were made using ΔΔC<sub>T</sub> analysis, and relative copy numbers were expressed as 2<sup>−ΔCT</sup> relative to 1 million copies of <i>EEF1A1</i>. Primer pairs were confirmed to have 90–105% efficiency based on the slope of the standard curve from a dilution series of pooled cDNAs from human or mouse cell lines.</p> </div> <div class="immunofluorescence" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence</div> <p>Cells were fixed in either 4% PFA (Electron Microscopy Services) or 4% PFA with 0.075 mg/ml saponin (Alfa Aesar) at 37°C for 10 min. PFA was quenched with 1 mg/ml sodium borohydride for 15 min at room temperature or 100 mM glycine for 10 min at room temperature. Cells fixed in PFA alone were permeabilized in 0.1% Triton X-100 (Thermo Fisher Scientific) for 10 min at room temperature. Cells were blocked in 1% BSA and 1% FBS for 1 h at room temperature or at 4°C overnight. For actin staining, fluorescently labeled phalloidin (ATTO-Tec and Invitrogen) was added to blocking buffer. Primary antibodies were used for 1.5–2.5 h at room temperature, and secondary antibodies were used for 1 h at room temperature.</p> </div> <div class="faareaandlengthquantification" title="sec"> <div class="title" tagx="title" title="title">
FA area and length quantification</div> <p>FA area and length were analyzed using paxillin immunofluorescence to denote FAs. MCF7, T47D, Caco-2, PC3, U2OS, and MCF10A cells were fixed and stained 24 h after plating, and SUM159, NIH-3T3, and MV<sup>D7</sup> cells were fixed 30 min after plating. Immunofluorescence was imaged with total internal reflection fluorescence (TIRF) illumination on a Ti-E inverted microscope (Nikon), with a 100× Apo TIRF 1.49 NA objective (Nikon), an ORCA-Flash 4.0 V2 complementary metal-oxide semiconductor (CMOS) camera (Hamamatsu), and a motorized stage. Total FA area per cell was quantified by measuring the paxillin area of nonclustered cells. Area measurements were done in Elements software (Nikon) by equally thresholding all images within an experiment using an average background fluorescence intensity and generating a binary mask. Individual FA length measurements were done in representative cells from the same experiments, selected for having total FA area closest to the quantified mean FA area in each experimental condition. FA lengths were quantified by thresholding all images within an experiment to an average background fluorescence intensity and using the length measurement tool in Elements software (Nikon). Automated area measurements of binary masks were taken, and the paxillin-positive area of all cells was plotted. In addition, total cell area measurements were used to create bins, and FA area (paxillin-positive area) was examined within each bin. Additionally, in a separate analysis, the FA area from the entire dataset was analyzed using a multiple regression model to control for differences in cell area between experimental groups.</p> </div> <div class="quantificationofp-fakandp-paxarea(mechanosensing)" title="sec"> <div class="title" tagx="title" title="title">
Quantification of p-FAK and p-pax area (mechanosensing)</div> <p>Mechanosensory signaling was analyzed using immunofluorescence of p-FAK at Tyrosine-397 and p-paxillin at Tyrosine-118. MCF7 cells were fixed and stained 24 h after plating on glass-bottom dishes (1.5; MatTek). For myosin suppression studies, cells were treated with 25 µM Y-27632 (Tocris) for 1 h before fixation. Immunofluorescence was imaged with TIRF illumination on a Ti-E inverted microscope, with a 100× Apo TIRF 1.49 NA objective, an ORCA-Flash 4.0 V2 CMOS camera, and a motorized stage. Area measurements were done in Elements software by thresholding all images within an experiment to an average background fluorescence intensity and generating a binary mask. Automated area measurements of binary masks were taken, and areas of all cells were plotted.</p> </div> <div class="insituactinpolymerizationassay" title="sec"> <div class="title" tagx="title" title="title">
In situ actin polymerization assay</div> <p>In situ actin polymerization assays (also referred to as barbed end labeling) were modified from <a href="#bib7">Chan et al. (1998)</a>. Briefly, Rhodamine-labeled actin (Cytoskeleton) was diluted to 0.2 µg/µl in buffer (pH 7.5) containing 1 mM Hepes, 0.2 mM MgCl<sub>2</sub>, and 0.2 mM ATP (Sigma-Aldrich). Actin solution was sonicated on ice before being centrifuged at 55,000 rpm for 30 min at 4°C in an Optima TLX ultracentrifuge (Beckman) and TLA 120.1 rotor (Beckman). The actin solution was then incubated on ice for an additional 30 min. Just before cell exposure to actin, glass-bottom dishes (1.5; MatTek) with cells were placed in a 37°C water bath, and the actin solution was further diluted to 0.02 µg/µl in 37°C buffer (pH 7.5) composed of 20 mM Hepes, 138 mM KCl, 4 mM MgCl<sub>2</sub>, 3 mM EGTA, 1% BSA, 1 mM ATP, and 0.2 mg/ml saponin. The actin solution was added to dishes for 1 min before the reaction was stopped with buffer (pH 7.5) composed of 20 mM Hepes, 138 mM KCl, 4 mM MgCl<sub>2</sub>, and 3 mM EGTA. Providing labeled actin for 1 min allows for visualization of actin polymerization dynamics within a defined period of time. Fixation and immunofluorescent staining immediately proceeded. Cells were imaged using TIRF illumination on a Ti-E inverted microscope, with a 100× Apo TIRF 1.49 NA objective, an ORCA-Flash 4.0 V2 CMOS camera, and a motorized stage. FA actin polymerization was quantified in Elements software by generating binary masks of paxillin staining within nonclustered cells (see FA area quantification) and measuring the fluorescence intensity of exogenously introduced actin within binary masks.</p> </div> <div class="3dn-stormsuperresolutionimaging" title="sec"> <div class="title" tagx="title" title="title">
3D N-STORM superresolution Imaging</div> <p>In situ actin polymerization assays were imaged using 3D N-STORM 5.0 system with astigmatic lens (Nikon), with a CFI HP Apochromat 100XAC TIRF 1.49 NA objective (Nikon), LU-NV 405 nm (20 mW), 488 nm (70 mW), 561 nm (70 mW), and 640 nm (125 mW) laser with power measured at the fiber tip (Nikon), Flash 4.0 V3 camera (Hamamatsu), and a motorized stage. Samples were imaged in imaging buffer containing 50 mM Tris, pH 8.0, 10 mM NaCl, 10% glucose, 14.3 mM 2-mercaptoethanol, 560 µg/ml glucose oxidase, and 34 µg/ml catalase. In situ actin polymerization assays used for superresolution imaging were performed as described above with the exception of biotin-labeled actin (Cytoskeleton) and streptavidin conjugated to Alexa Fluor 647 (Invitrogen) being used for visualization of labeled barbed ends (in lieu of Rhodamine-labeled actin). Assays were performed in MCF7 cells expressing GFP-<i>EVL</i> and were completed with paxillin immunofluorescence to identify FA plaques. For 3D image calibration, 100 nm TetraSpeck Microspheres (Thermo Fisher Scientific) were imaged every 10 nm in the z plane to generate a 1-µm z stack, which was used to measure the axial ratio of stretch generated by the astigmatic lens in x and y. For image acquisition, cells were located within the TIRF module, and a 75,000 frame imaging cycle was used for STORM imaging in 488-, 561-, and 640-nm wavelengths.</p> </div> <div class="quantificationofcellarea" title="sec"> <div class="title" tagx="title" title="title">
Quantification of cell area</div> <p>Glass-bottom dishes (1.5; MatTek) were coated with fibronectin (10 µg/ml; Corning), laminin (derived from HaCaT cells; a kind gift from Anne Cress, University of Arizona, Tucson, AZ), or collagen type I (rat tail; 50 µg/ml in 0.1% acetic acid; Corning). For plating, cells were lifted using Cellstripper (Corning) for 20 min at 37°C. Cells were imaged 24 h after plating using a Ti-E inverted microscope, with a 20× Plan Apo 0.75 NA objective (Nikon) and a CoolSNAP MYO charge-coupled device camera (Photometrics). Single cells were analyzed, and cell area was quantified by manual tracing in Elements software.</p> </div> <div class="preparationofdeformablehydrogels" title="sec"> <div class="title" tagx="title" title="title">
Preparation of deformable hydrogels</div> <p>Preparation of deformable polyacrylamide hydrogels was modified from <a href="#bib33">Ng et al. (2012)</a>. Briefly, glass-bottom dishes (1.0; MatTek) were treated with a 2% solution of 3-aminopropyltrimethoxy silane (Sigma-Aldrich) in isopropanol, for 10 min. After washing, plates were dried at 37°C for several hours. Dishes were then treated with 0.25% glutaraldehyde for 30 min, washed, and dried. Dishes were stored in a dust-free environment for up to 1 mo. Final ratios of 5/0.3%, 10/0.2%, and 10/0.3% acrylamide/bis-acrylamide were used for 8-, 35-, and 64-kPa gels, respectively. Acrylamide and bis-acrylamide were diluted in 50 mM Hepes (pH 8.5), with 0.1% APS and 0.2% tetramethylethylenediamine added to the gel solution. To create flat-cell adhesion surfaces, unpolymerized polyacrylamide gel was sandwiched between a pretreated MatTek dish and coverslip and allowed to polymerize at room temperature. Following polymerization, coverslips were removed, and gels were stored at 4°C in PBS until prepared for matrix coating. For matrix coating, gels were treated with 2 mg/ml Sulfo-SANPAH (Life Technologies) and placed under UV light for 20 min. Gels were then washed with PBS and coated with 10 µg/ml fibronectin (Corning) for a minimum of 1.5 h at 37°C. Gels were washed with PBS before cell plating.</p> </div> <div class="quantificationofcellspreadingonsubstratesofincreasingrigidity(mechanosensing)" title="sec"> <div class="title" tagx="title" title="title">
Quantification of cell spreading on substrates of increasing rigidity (mechanosensing)</div> <p>Deformable 8-, 35-, and 64-kPa hydrogels were prepared and coated with 10 µg/ml fibronectin (see Preparation of deformable hydrogels). For plating, cells were lifted using Cellstripper (Corning) for 20 min at 37°C. Cells were imaged 2 h after plating using a Ti-E inverted microscope, with a 20× Plan Apo 0.75 NA objective and an ORCA-Flash 4.0 V2 CMOS camera. Single cells were analyzed, and cell area was quantified by manual tracing in Elements software.</p> </div> <div class="mechanicallydirectedmotilityassay" title="sec"> <div class="title" tagx="title" title="title">
Mechanically directed motility assay</div> <p>Mechanically directed motility assays were performed as previously described (<a href="#bib29">Lo et al., 2000</a>; <a href="#bib42">Plotnikov et al., 2012</a>). Briefly, cells were plated on 8-, 35-, or 64-kPa polyacrylamide gels coated with 10 µg/ml fibronectin (see Preparation of deformable hydrogels) and allowed to adhere for ∼48 h. Before imaging, imaging buffer was added to cells (full medium supplemented with 10 mM Hepes). A micromanipulator (InjectMan NI2; Eppendorf) was fitted with a glass micropipette (Femtotip; Eppendorf), and a force gradient was generated by micropipette insertion ∼10 µm away from nonleading edges of single cells and pulling micropipettes a further 10 µm from the cell. This methodology allows for reproducible and consistent stimulation with respect to the position, duration, and magnitude of force. Experiments were imaged by differential interference contrast (DIC) microscopy on a Ti-E inverted microscope, with a 60× Plan Apo 1.40 NA objective and a CoolSNAP MYO charge-coupled device camera (Photometrics) or an ORCA-Flash 4.0 V2 CMOS camera (Hamamatsu), a motorized stage, and an environmental chamber. Imaging was done at 37°C with ambient 5% CO<sub>2</sub>. Stimulated cells were imaged at a 1-s frame rate for 30 min. To suppress myosin contractility, either 25 µM Y-27632 (Tocris) or 10 µM Blebbistatin (Sigma-Aldrich) was added to imaging buffer, and cells were treated for 1 h before experiment start time and maintained in drug throughout the experiment. To suppress formin activity, 15 µM SMIFH2 (EMD Millipore) was added to imaging buffer, and cells were treated for 4 h before experiment start time and maintained in drug throughout the experiment. Mechanically directed motility was quantified using the sensing index and turning angles of manual traces done in ImageJ (see Sensing index and Turning angles). Additionally, an accompanying analysis of cell area at the initial position was performed for all the experiments in which we observed a difference in mechanically directed motility.</p> </div> <div class="pdai" title="sec"> <div class="title" tagx="title" title="title">
PDAI</div> <p>PDAI = %<i>P</i>(|%<i>P</i> + %<i>D</i>|) accounts for two factors: (1) %P, which is the percentage change in area of the side of the cell proximal to the stimulus source, and (2) |%<i>P</i> + %<i>D</i>|, the migration factor, which is the absolute value of the sum of the percentage change in area at the proximal and distal sides. These factors reflect the extent of de-adhesion with respect to the direction of change and de-adhesion due to mere cell translocation during cell motility, respectively. Proximal and distal cell areas are quantified every 5 min by drawing a line from the pipette to the starting centroid of the cell and then drawing a second, perpendicular line running through the starting centroid position of the cell. The second line dictates the "proximal" and "distal" halves of the cell with respect to the stimulus source, and cell area changes are measured at starting and ending time points according to this line. Both lines are redrawn for every starting time point for all time interval comparisons to account for cell movement. Proximal and distal percentage changes in area were calculated by subtracting the respective cell area of the ending position (II<i><sub>P</sub></i>) from the respective cell area of the starting position (I<i><sub>P</sub></i>), dividing by the respective cell area of the starting position, and multiplying by 100: %<i>P</i> = 100 × (II<i><sub>P</sub></i> − I<i><sub>P</sub></i>)/I<i><sub>P</sub></i> or %D = 100 × (II<i><sub>D</sub></i> − I<i><sub>D</sub></i>)/I<i><sub>D</sub></i>.</p> </div> <div class="randommigrationassay" title="sec"> <div class="title" tagx="title" title="title">
Random migration assay</div> <p>Cells were plated on 35-kPa polyacrylamide gels coated with 10 µg/ml fibronectin (see Preparation of deformable hydrogels) and allowed to adhere for ∼24 h before imaging. Cells were imaged using a Ti-E inverted microscope, with a 20× Plan Apo 0.75 NA objective and an ORCA-Flash 4.0 V2 CMOS camera, a motorized stage, and an environmental chamber. Imaging was done at 37°C with 5% CO<sub>2</sub>. Single cells were analyzed, and cell tracking analysis of cell migration during 2-h time periods was performed in Elements software (Nikon). Speed and directionality are quantified as [path length/time] and [net path/total path], respectively. Measurements are averages from 10-, 20-, 30-, 40-, 50-, 60-, 70-, 80-, 90-, 100-, 110-, and 120-min intervals.</p> </div> <div class="chemotacticsensingassay" title="sec"> <div class="title" tagx="title" title="title">
Chemotactic sensing assay</div> <p>Chemotactic sensing assays were performed as previously described (<a href="#bib31">Mouneimne et al., 2006</a>). Briefly, cells were plated on glass-bottom dishes (1.5; MatTek) coated with 10 µg/ml fibronectin (Corning) and allowed to adhere for ∼24 h. Before imaging, imaging buffer was added to cells (full medium supplemented with 10 mM Hepes). A micromanipulator (InjectMan NI2; Eppendorf) was fitted with a glass micropipette (Femtotip; Eppendorf), which was loaded with 10 nM IGF with 3-kD fluorescently labeled Dextran clarified by 0.2-µm filters. A chemotactic gradient was generated using a pressure-regulated microinjection system at 25pi with continuous flow (FemtoJet; Eppendorf). Micropipettes were placed ∼25 µm away from nonleading edges of single cells, and IGF was released from the micropipette for 2 min. Experiments were imaged on a Ti-E inverted microscope, with a 60× Plan Apo 1.40 NA objective, and an ORCA-Flash 4.0 V2 CMOS camera, a motorized stage, and an environmental chamber. Imaging was done at 37°C with ambient 5% CO<sub>2</sub>. Stimulated cells were imaged by DIC microscopy at a 1-s frame rate for 30 min. Wide-field fluorescence, captured at a 1-min frame rate, was used to visualize the chemotactic gradient. Chemotactic sensing was quantified using the sensing index and turning angles of manual traces done in ImageJ (see Sensing index and turning angles).</p> </div> <div class="sensingindexandturningangles" title="sec"> <div class="title" tagx="title" title="title">
Sensing index and turning angles</div> <div class="sensingindex" title="sec"> <div class="title" tagx="title" title="title">
Sensing index</div> <p>Cosine θ was quantified as previously described (<a href="#bib31">Mouneimne et al., 2006</a>). Briefly, θ was measured as the angle between the pipette, the first centroid position of the cell, and each subsequent centroid position of the cell at 5, 10, 15, 20, 25, and 30 min. Sensing indices range from +1 to −1, with +1 representing the highest level of directionality toward the stimulus and −1 representing the highest level of directionality away from the stimulus.</p> </div> <div class="turningangles" title="sec"> <div class="title" tagx="title" title="title">
Turning angles</div> <p>Turning angles were measured between the pipette, the current centroid position of the cell, and subsequent centroid positions of the cell at 5-min intervals. Angles were then adjusted for plotting on a normalized polar graph by converting the cell centroid position at time 0, to <i>x</i> = 0, <i>y</i> = 0, and transforming all cell centroid positions with reference to cell centroid position at time 0. Centroid positions were then rotationally transformed so that pipette positions were set along the same axis. For plotting, rose plots were divided into 30° sectors, and turning angles from all time points were plotted as a population percentage, with each step within sectors representing 1% of all turns.</p> </div> </div> <div class="3dinvasionassays" title="sec"> <div class="title" tagx="title" title="title">
3D invasion assays</div> <p>Invasion assays were modified from <a href="#bib37">Padilla-Rodriguez et al. (2018)</a>. Chambered 1.5 coverglass (Lab-Tek) was treated with silane and glutaraldehyde (see Preparation of deformable hydrogels) and sterilized under UV light for 15 min. Cells were suspended in 1 mg/ml type I collagen (Corning) diluted in DMEM (Corning), containing 0.0625 mg/ml ethylene glycol-bis (succinic acid <i>N</i>-hydroxysuccinimide ester; MP Biomedicals), buffered to a neutral pH with sodium hydroxide. The cell/collagen solution was then plated as a 3D dome in the center of the treated coverglass and allowed to polymerize at 37°C. For durotactic invasion assays, the outer collagen matrix was prepared by combining 1 mg/ml type I collagen in DMEM with 0.625 mg/ml ethylene glycol-bis and buffering the solution to a neutral pH with sodium hydroxide. The outer collagen for soft invasion assays was prepared by combining 1 mg/ml type I collagen diluted in DMEM containing 0.0625 mg/ml ethylene glycol-bis and 20 nM IGF (Sigma-Aldrich), buffered to a neutral pH with sodium hydroxide. After cell/collagen polymerization, the outer collagen matrix solution was added onto the cell/collagen matrix and allowed to polymerize at 37°C. After polymerization, collagen gels were covered with medium. MCF7 invasion assays were performed for 4 d with medium changes every 24 h, and U2OS invasion assays were performed for 18 h with imaging beginning immediately after plating. MCF7 invasion assays were fixed and stained with Hoechst 33342 (Thermo Fisher Scientific) for 24 h at 4°C, and large stitch imaging of wide-field fluorescence and DIC microscopy was used for imaging the entire cell/matrix boundary. U2OS invasion assays were imaged using multipoint DIC microscopy for 18 h at a 10-min frame rate, with a 5-µm step z-series. All invasion assays were imaged on a Ti-E inverted microscope, with a 20× Plan Apo 0.75 NA objective, an ORCA-Flash 4.0 V2 CMOS camera, and a motorized stage. Invasion assays were quantified by counting the number of cells invaded into the outer matrix in Elements software; cells were counted as invaded once their nucleus crossed the inner/outer matrix boundary.</p> </div> <div class="atomicforcemicroscope–basednano-indentation" title="sec"> <div class="title" tagx="title" title="title">
Atomic force microscope–based nano-indentation</div> <p>A commercial atomic force microscope (MFP-3D-BIO AFM, Asylum Research) was used for the determination of the elastic moduli (i.e., stiffness) as previously described (<a href="#bib48">Staunton et al., 2016</a>; <a href="#bib61">Wu et al., 2018</a>). Briefly, spheroconical probes (LRCH, Team Nanotech) with nominal spring constants of 0.2 N ⋅ m<sup>−1</sup>, a half cone angle of 18.8°, tip length of &amp;gt;10 µm, and a spherical radius of 700 nm were used to collect force–indentation curves in four to seven 4 × 5 grids of 90 × 90-µm areas at 37°C in PBS buffer with an indenter vertical speed of 2 µm ⋅ s<sup>−1</sup>. Trigger forces of 20–30 nN resulted in indentation depths ≥10 µm. The elastic moduli were obtained by fitting the initial 10 µm of indentation of each force–indentation curve to a nonadhesive elastic contact model for spheroconical probes (<a href="#bib48">Staunton et al., 2016</a>). Collagen was assumed to be incompressible, with a Poisson ratio of 0.5 (<a href="#bib25">Lacroix et al., 2018</a>). The spring constants of the cantilevers were determined by the thermal noise method before the experiment (<a href="#bib4">Butt and Manfred, 1995</a>).</p> </div> <div class="statisticsandreproducibility" title="sec"> <div class="title" tagx="title" title="title">
Statistics and reproducibility</div> <p>Statistical analyses were performed in GraphPad Prism 8 and Stata 15. For the analysis of FA area in the different cell area bins, Mann–Whitney <i>U</i> test was used. For the multiple regression analysis of FA area as a dependent variable, experiment and cell area were used as independent variables and controlled for; data were log-transformed to accomplish normality for this analysis. For analysis of mechanically directed motility, statistical differences were determined by a two-way ANOVA, using the raw angle data after square root transformation; P values are specified in figures and figure legends. For analysis of 3D invasion, data were transformed to denote fold-change from the condition with highest values, and one-sample <i>t</i> tests were performed with a hypothetical value of 1. For de-adhesion event data, statistical differences were determined using a Mann–Whitney <i>U</i> test. For qPCR and zymogram data, statistical differences were determined using a two-tailed Student’s <i>t</i> test. For all other data, statistical differences were determined using regression analysis to control for variability between experiments, and non–normally distributed data were transformed using log or square root to achieve normality (untransformed data are graphed for clarity). Exact P values between all comparisons are plotted in Tables S2 and S3. Sample sizes and number of repeated experiments are specified in the legends. Data are presented as violin plots with median, dot plots with mean ± SEM, or dot plots with mean ± SD, as denoted in figure legends. P &amp;lt; 0.05 is considered significant; P value ranges are noted in figure legends, and exact P values are listed in Tables S2 and S3.</p> </div> <div class="onlinesupplementalmaterial" title="sec"> <div class="title" tagx="title" title="title">
Online supplemental material</div> <p>Fig. S1 shows that mechanically directed motility occurs under myosin suppression. Fig. S2 shows that EVL is required for cell-matrix adhesion and cell spreading. Fig. S3 shows that EVL KD reduces FA area independently of cell spreading. Fig. S4 shows that EVL KD reduces FA maturation and mechanosensory signaling, and that EVL localizes with sites of actin polymerization at FAs. Fig. S5 shows measurements of cell area in mechanically directed motility assays and the suppression of mechanically directed motility by reduced EVL expression. Fig. S6 shows that EVL KD uniquely impairs directionality, but not random migration speed of cells or MMP activity. Additionally, Fig. S6 shows atomic force microscopy measurement of the invasion assay inner and outer gels. Table S1 shows the confirmed sequences of shResist EVL, shResist EVL domain deletion mutants, and shResist EVL chimeric mutants. Table S2 shows exact P values from the indicated figures. Table S3 shows exact P values from all two-way comparisons done in the indicated figures. Video 1 shows a time-lapse of mechano-stimulated control and Y-27632–treated MCF7 cells on 35-kPa hydrogel (top), Blebbistatin-treated MCF7 cells on 35-kPa hydrogel (middle), and Y-27632–treated MCF7 cells on 64-kPa hydrogel (bottom). Video 2 shows a time-lapse of mechano-stimulated control and EVL KD MCF7 cells on 35-kPa hydrogel. Video 3 shows a time-lapse of mechano-stimulated control and EVL KD U2OS cells on 35-kPa hydrogel. Video 4 shows a time-lapse of mechano-stimulated control and EVL KD (shRNA #02) MCF7 cells on 35-kPa hydrogel. Video 5 shows a time-lapse of mechano-stimulated control + GFP, EVL KD + GFP, and EVL KD + GFP-ΔGF-PFN EVL MCF7 cells on 35-kPa hydrogel. Video 6 shows a time-lapse of mechano-stimulated control and SMIFH2-treated MCF7 cells on 35-kPa hydrogel. Video 7 shows a time-lapse of mechano-stimulated control and EVL KD MCF7 cells on 8-kPa hydrogel. Video 8 shows a time-lapse of mechano-stimulated GFP and GFP-EVL–expressing MVD7 cells on 35-kPa hydrogel. Video 9 shows a time-lapse of control and EVL KD U2OS cells during durotactic invasion. Video 10 shows a time-lapse of chemo-stimulated control and EVL KD MCF7 cells.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Materials (PDF)</div>  <div class="media" title="media"><a href="JCB_201902101_sm.pdf">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Tables S1-S3 (ZIP)</div>  <div class="media" title="media"><a href="JCB_201902101_TablesS1-S3.zip">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 1</div>  <div class="media" title="media"><a href="JCB_201902101_V1.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 2</div>  <div class="media" title="media"><a href="JCB_201902101_V2.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 3</div>  <div class="media" title="media"><a href="JCB_201902101_V3.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 4</div>  <div class="media" title="media"><a href="JCB_201902101_V4.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 5</div>  <div class="media" title="media"><a href="JCB_201902101_V5.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 6</div>  <div class="media" title="media"><a href="JCB_201902101_V6.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 7</div>  <div class="media" title="media"><a href="JCB_201902101_V7.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 8</div>  <div class="media" title="media"><a href="JCB_201902101_V8.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 9</div>  <div class="media" title="media"><a href="JCB_201902101_V9.mp4">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Video 10</div>  <div class="media" title="media"><a href="JCB_201902101_V10.mp4">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors acknowledge the Gregorio laboratory and the Cress laboratory at the University of Arizona for technical assistance.</p> <p>This research was supported by National Cancer Institute grant R01 CA196885-01 (G. Mouneimne), National Cancer Institute diversity supplemental grant R01 CA196885-01 (J.I. Puleo), National Cancer Institute University of Arizona Cancer Center support grant P30CA023074, and funds from the Ludwig Center at Massachusetts Institute of Technology (F.B. Gertler).</p> <p>The authors declare no competing financial interests.</p> <p>Author contributions: J.I. Puleo and G. Mouneimne designed the research. J.I. Puleo, S.S. Parker, M.R. Roman, A.W. Watson, and K.R. Eliato performed research. L. Peng, R. Ros, and F.B. Gertler provided new reagents or analytic tools and contributed to the study design. J.I. Puleo, S.S. Parker, M.R. Roman, K.R. Eliato, K. Saboda, D.J. Roe, and G. Mouneimne analyzed data. J.I. Puleo and G. Mouneimne wrote the paper.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="bib1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilar-Cuenca</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juanes-García</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicente-Manzanares</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Myosin II in mechanotransduction: master and commander of cell migration, morphogenesis, and cancer</span>. <span class="source" tagx="source" title="source">Cell. Mol. Life Sci.</span><span class="volume" tagx="volume" title="volume">71</span>:<span class="fpage" tagx="fpage" title="fpage">479</span>–<span class="lpage" tagx="lpage" title="lpage">492</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00018-013-1439-5">10.1007/s00018-013-1439-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23934154">23934154</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loureiro</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fässler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehland</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Negative regulation of fibroblast motility by Ena/VASP proteins</span>. <span class="source" tagx="source" title="source">Cell.</span><span class="volume" tagx="volume" title="volume">101</span>:<span class="fpage" tagx="fpage" title="fpage">717</span>–<span class="lpage" tagx="lpage" title="lpage">728</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(00)80884-3">10.1016/S0092-8674(00)80884-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10892743">10892743</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butcher</span><span class="given-names" tagx="given-names" title="given-names">D.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alliston</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weaver</span><span class="given-names" tagx="given-names" title="given-names">V.M.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">A tense situation: forcing tumour progression</span>. <span class="source" tagx="source" title="source">Nat. Rev. Cancer.</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">108</span>–<span class="lpage" tagx="lpage" title="lpage">122</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrc2544">10.1038/nrc2544</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19165226">19165226</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butt</span><span class="given-names" tagx="given-names" title="given-names">H.-J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manfred</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Calculation of thermal noise in atomic force microscopy</span>. <span class="source" tagx="source" title="source">Nanotechnology.</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>.</span></li> <li tag="ref"><a name="bib5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campellone</span><span class="given-names" tagx="given-names" title="given-names">K.G.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">A nucleator arms race: cellular control of actin assembly</span>. <span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">237</span>–<span class="lpage" tagx="lpage" title="lpage">251</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm2867">10.1038/nrm2867</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20237478">20237478</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">L.B.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Integration of actin dynamics and cell adhesion by a three-dimensional, mechanosensitive molecular clutch</span>. <span class="source" tagx="source" title="source">Nat. Cell Biol.</span><span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">955</span>–<span class="lpage" tagx="lpage" title="lpage">963</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ncb3191">10.1038/ncb3191</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26121555">26121555</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">A.Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raft</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailly</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyckoff</span><span class="given-names" tagx="given-names" title="given-names">J.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segall</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condeelis</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">EGF stimulates an increase in actin nucleation and filament number at the leading edge of the lamellipod in mammary adenocarcinoma cells</span>. <span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="volume" tagx="volume" title="volume">111</span>:<span class="fpage" tagx="fpage" title="fpage">199</span>–<span class="lpage" tagx="lpage" title="lpage">211</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9405304">9405304</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostuni</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitesides</span><span class="given-names" tagx="given-names" title="given-names">G.M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ingber</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Cell shape provides global control of focal adhesion assembly</span>. <span class="source" tagx="source" title="source">Biochem. Biophys. Res. Commun.</span><span class="volume" tagx="volume" title="volume">307</span>:<span class="fpage" tagx="fpage" title="fpage">355</span>–<span class="lpage" tagx="lpage" title="lpage">361</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0006-291X(03)01165-3">10.1016/S0006-291X(03)01165-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12859964">12859964</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">C.K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicente-Manzanares</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zareno</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitmore</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mogilner</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horwitz</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Actin and alpha-actinin orchestrate the assembly and maturation of nascent adhesions in a myosin II motor-independent manner</span>. <span class="source" tagx="source" title="source">Nat. Cell Biol.</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">1039</span>–<span class="lpage" tagx="lpage" title="lpage">1050</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ncb1763">10.1038/ncb1763</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19160484">19160484</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choquet</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felsenfeld</span><span class="given-names" tagx="given-names" title="given-names">D.P.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheetz</span><span class="given-names" tagx="given-names" title="given-names">M.P.</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Extracellular matrix rigidity causes strengthening of integrin-cytoskeleton linkages</span>. <span class="source" tagx="source" title="source">Cell.</span><span class="volume" tagx="volume" title="volume">88</span>:<span class="fpage" tagx="fpage" title="fpage">39</span>–<span class="lpage" tagx="lpage" title="lpage">48</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(00)81856-5">10.1016/S0092-8674(00)81856-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9019403">9019403</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elosegui-Artola</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bazellières</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andreu</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oria</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunyer</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomm</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshall</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trepat</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roca-Cusachs</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Rigidity sensing and adaptation through regulation of integrin types</span>. <span class="source" tagx="source" title="source">Nat. Mater.</span><span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">631</span>–<span class="lpage" tagx="lpage" title="lpage">637</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmat3960">10.1038/nmat3960</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24793358">24793358</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flanagan</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ju</span><span class="given-names" tagx="given-names" title="given-names">Y.-E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marg</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterfield</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janmey</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Neurite branching on deformable substrates</span>. <span class="source" tagx="source" title="source">Neuroreport.</span><span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">2411</span>–<span class="lpage" tagx="lpage" title="lpage">2415</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1097/00001756-200212200-00007">10.1097/00001756-200212200-00007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12499839">12499839</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furman</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sieminski</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwiatkowski</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubinson</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasile</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bronson</span><span class="given-names" tagx="given-names" title="given-names">R.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fässler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Ena/VASP is required for endothelial barrier function in vivo</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">179</span>:<span class="fpage" tagx="fpage" title="fpage">761</span>–<span class="lpage" tagx="lpage" title="lpage">775</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200705002">10.1083/jcb.200705002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17998398">17998398</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geiger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spatz</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bershadsky</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Environmental sensing through focal adhesions</span>. <span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">21</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm2593">10.1038/nrm2593</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19197329">19197329</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niebuhr</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reinhard</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehland</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soriano</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Mena, a relative of VASP and Drosophila Enabled, is implicated in the control of microfilament dynamics</span>. <span class="source" tagx="source" title="source">Cell.</span><span class="volume" tagx="volume" title="volume">87</span>:<span class="fpage" tagx="fpage" title="fpage">227</span>–<span class="lpage" tagx="lpage" title="lpage">239</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0092-8674(00)81341-0">10.1016/S0092-8674(00)81341-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8861907">8861907</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goswami</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Philippar</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patsialou</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avraham</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Modugno</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nistico</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condeelis</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Identification of invasion specific splice variants of the cytoskeletal protein Mena present in mammary tumor cells during invasion in vivo</span>. <span class="source" tagx="source" title="source">Clin. Exp. Metastasis.</span><span class="volume" tagx="volume" title="volume">26</span>:<span class="fpage" tagx="fpage" title="fpage">153</span>–<span class="lpage" tagx="lpage" title="lpage">159</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10585-008-9225-8">10.1007/s10585-008-9225-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18985426">18985426</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupton</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenmann</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alberts</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman-Storer</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">mDia2 regulates actin and focal adhesion dynamics and organization in the lamella for efficient epithelial cell migration</span>. <span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="volume" tagx="volume" title="volume">120</span>:<span class="fpage" tagx="fpage" title="fpage">3475</span>–<span class="lpage" tagx="lpage" title="lpage">3487</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.006049">10.1242/jcs.006049</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17855386">17855386</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupton</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riquelme</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes-Alford</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tadros</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rudina</span><span class="given-names" tagx="given-names" title="given-names">S.S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hynes</span><span class="given-names" tagx="given-names" title="given-names">R.O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauffenburger</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Mena binds α5 integrin directly and modulates α5β1 function</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">198</span>:<span class="fpage" tagx="fpage" title="fpage">657</span>–<span class="lpage" tagx="lpage" title="lpage">676</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201202079">10.1083/jcb.201202079</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22908313">22908313</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirata</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatsumi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokabe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Mechanical forces facilitate actin polymerization at focal adhesions in a zyxin-dependent manner</span>. <span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="volume" tagx="volume" title="volume">121</span>:<span class="fpage" tagx="fpage" title="fpage">2795</span>–<span class="lpage" tagx="lpage" title="lpage">2804</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.030320">10.1242/jcs.030320</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18682496">18682496</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirata</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokabe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">C.T.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Molecular mechanisms underlying the force-dependent regulation of actin-to-ECM linkage at the focal adhesions</span>. <span class="source" tagx="source" title="source">Prog. Mol. Biol. Transl. Sci.</span><span class="volume" tagx="volume" title="volume">126</span>:<span class="fpage" tagx="fpage" title="fpage">135</span>–<span class="lpage" tagx="lpage" title="lpage">154</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/B978-0-12-394624-9.00006-3">10.1016/B978-0-12-394624-9.00006-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25081617">25081617</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iskratsch</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">C.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathur</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stévenin</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwyer</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hone</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehler</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheetz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">FHOD1 is needed for directed forces and adhesion maturation during cell spreading and migration</span>. <span class="source" tagx="source" title="source">Dev. Cell.</span><span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">545</span>–<span class="lpage" tagx="lpage" title="lpage">559</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2013.11.003">10.1016/j.devcel.2013.11.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24331927">24331927</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iskratsch</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfenson</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheetz</span><span class="given-names" tagx="given-names" title="given-names">M.P.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Appreciating force and shape—the rise of mechanotransduction in cell biology</span>. <span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">825</span>–<span class="lpage" tagx="lpage" title="lpage">833</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm3903">10.1038/nrm3903</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25355507">25355507</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dent</span><span class="given-names" tagx="given-names" title="given-names">E.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loureiro</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Ena/VASP proteins: regulators of the actin cytoskeleton and cell migration</span>. <span class="source" tagx="source" title="source">Annu. Rev. Cell Dev. Biol.</span><span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">541</span>–<span class="lpage" tagx="lpage" title="lpage">564</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.cellbio.19.050103.103356">10.1146/annurev.cellbio.19.050103.103356</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14570581">14570581</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lachowski</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortes</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pink</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chronopoulos</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karim</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">P Morton</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Río Hernández</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Substrate Rigidity Controls Activation and Durotaxis in Pancreatic Stellate Cells</span>. <span class="source" tagx="source" title="source">Sci. Rep.</span><span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">2506</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-02689-x">10.1038/s41598-017-02689-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28566691">28566691</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lacroix</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunelli</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perrault</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldit</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shariatzadeh</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campos-Marin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barreto</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span>, editors. <span class="year" tagx="year" title="year">2018</span><span class="source" tagx="source" title="source">Multiscale Mechanobiology in Tissue Engineering.</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>. 203 pp.</span></li> <li tag="ref"><a name="bib26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambrechts</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwiatkowski</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanier</span><span class="given-names" tagx="given-names" title="given-names">L.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandekerckhove</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ampe</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">cAMP-dependent protein kinase phosphorylation of EVL, a Mena/VASP relative, regulates its interaction with actin and SH3 domains</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">275</span>:<span class="fpage" tagx="fpage" title="fpage">36143</span>–<span class="lpage" tagx="lpage" title="lpage">36151</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M006274200">10.1074/jbc.M006274200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10945997">10945997</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laurent</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loisel</span><span class="given-names" tagx="given-names" title="given-names">T.P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harbeck</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehman</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gröbe</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jockusch</span><span class="given-names" tagx="given-names" title="given-names">B.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehland</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlier</span><span class="given-names" tagx="given-names" title="given-names">M.F.</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Role of proteins of the Ena/VASP family in actin-based motility of Listeria monocytogenes</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">144</span>:<span class="fpage" tagx="fpage" title="fpage">1245</span>–<span class="lpage" tagx="lpage" title="lpage">1258</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.144.6.1245">10.1083/jcb.144.6.1245</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10087267">10087267</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levental</span><span class="given-names" tagx="given-names" title="given-names">K.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kass</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lakins</span><span class="given-names" tagx="given-names" title="given-names">J.N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egeblad</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erler</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fong</span><span class="given-names" tagx="given-names" title="given-names">S.F.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Csiszar</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giaccia</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weninger</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Matrix crosslinking forces tumor progression by enhancing integrin signaling</span>. <span class="source" tagx="source" title="source">Cell.</span><span class="volume" tagx="volume" title="volume">139</span>:<span class="fpage" tagx="fpage" title="fpage">891</span>–<span class="lpage" tagx="lpage" title="lpage">906</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2009.10.027">10.1016/j.cell.2009.10.027</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19931152">19931152</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dembo</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.L.</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Cell movement is guided by the rigidity of the substrate</span>. <span class="source" tagx="source" title="source">Biophys. J.</span><span class="volume" tagx="volume" title="volume">79</span>:<span class="fpage" tagx="fpage" title="fpage">144</span>–<span class="lpage" tagx="lpage" title="lpage">152</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0006-3495(00)76279-5">10.1016/S0006-3495(00)76279-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10866943">10866943</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouneimne</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soon</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DesMarais</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidani</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yip</span><span class="given-names" tagx="given-names" title="given-names">S.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Backer</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condeelis</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Phospholipase C and cofilin are required for carcinoma cell directionality in response to EGF stimulation</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">166</span>:<span class="fpage" tagx="fpage" title="fpage">697</span>–<span class="lpage" tagx="lpage" title="lpage">708</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200405156">10.1083/jcb.200405156</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15337778">15337778</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouneimne</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DesMarais</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidani</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scemes</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condeelis</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Spatial and temporal control of cofilin activity is required for directional sensing during chemotaxis</span>. <span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">16</span>:<span class="fpage" tagx="fpage" title="fpage">2193</span>–<span class="lpage" tagx="lpage" title="lpage">2205</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2006.09.016">10.1016/j.cub.2006.09.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17113383">17113383</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouneimne</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">S.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selfors</span><span class="given-names" tagx="given-names" title="given-names">L.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrak</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hickey</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallegos</span><span class="given-names" tagx="given-names" title="given-names">L.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">K.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartwig</span><span class="given-names" tagx="given-names" title="given-names">J.H.</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Differential remodeling of actin cytoskeleton architecture by profilin isoforms leads to distinct effects on cell migration and invasion</span>. <span class="source" tagx="source" title="source">Cancer Cell.</span><span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">615</span>–<span class="lpage" tagx="lpage" title="lpage">630</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ccr.2012.09.027">10.1016/j.ccr.2012.09.027</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23153535">23153535</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">M.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Besser</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danuser</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brugge</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Substrate stiffness regulates cadherin-dependent collective migration through myosin-II contractility</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">199</span>:<span class="fpage" tagx="fpage" title="fpage">545</span>–<span class="lpage" tagx="lpage" title="lpage">563</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201207148">10.1083/jcb.201207148</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23091067">23091067</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oakes</span><span class="given-names" tagx="given-names" title="given-names">P.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beckham</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stricker</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardel</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Tension is required but not sufficient for focal adhesion maturation without a stress fiber template</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">196</span>:<span class="fpage" tagx="fpage" title="fpage">363</span>–<span class="lpage" tagx="lpage" title="lpage">374</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201107042">10.1083/jcb.201107042</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22291038">22291038</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oudin</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonas</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosciuk</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broye</span><span class="given-names" tagx="given-names" title="given-names">L.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guido</span><span class="given-names" tagx="given-names" title="given-names">B.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyckoff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riquelme</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamar</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asokan</span><span class="given-names" tagx="given-names" title="given-names">S.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittaker</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2016</span><span class="comment">a</span><span class="mixed-article-title" title="mixed-article-title">Tumor Cell-Driven Extracellular Matrix Remodeling Drives Haptotaxis during Metastatic Progression</span>. <span class="source" tagx="source" title="source">Cancer Discov.</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">516</span>–<span class="lpage" tagx="lpage" title="lpage">531</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1158/2159-8290.CD-15-1183">10.1158/2159-8290.CD-15-1183</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26811325">26811325</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oudin</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klazen</span><span class="given-names" tagx="given-names" title="given-names">J.A.Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosciuk</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lussiez</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tadros</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauffenburger</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="comment">b</span><span class="mixed-article-title" title="mixed-article-title">MenaINV mediates synergistic cross-talk between signaling pathways driving chemotaxis and haptotaxis</span>. <span class="source" tagx="source" title="source">Mol. Biol. Cell.</span><span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">3085</span>–<span class="lpage" tagx="lpage" title="lpage">3094</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.e16-04-0212">10.1091/mbc.e16-04-0212</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27559126">27559126</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padilla-Rodriguez</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">S.S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">D.G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westerling</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puleo</span><span class="given-names" tagx="given-names" title="given-names">J.I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watson</span><span class="given-names" tagx="given-names" title="given-names">A.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noon</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaudin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aaron</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">The actin cytoskeletal architecture of estrogen receptor positive breast cancer cells suppresses invasion</span>. <span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">2980</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-05367-2">10.1038/s41467-018-05367-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30061623">30061623</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parsons</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horwitz</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwartz</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Cell adhesion: integrating cytoskeletal dynamics and cellular tension</span>. <span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">633</span>–<span class="lpage" tagx="lpage" title="lpage">643</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm2957">10.1038/nrm2957</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20729930">20729930</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasapera</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">I.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rericha</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlaepfer</span><span class="given-names" tagx="given-names" title="given-names">D.D.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Myosin II activity regulates vinculin recruitment to focal adhesions through FAK-mediated paxillin phosphorylation</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">188</span>:<span class="fpage" tagx="fpage" title="fpage">877</span>–<span class="lpage" tagx="lpage" title="lpage">890</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200906012">10.1083/jcb.200906012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20308429">20308429</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasapera</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plotnikov</span><span class="given-names" tagx="given-names" title="given-names">S.V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">L.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egelhoff</span><span class="given-names" tagx="given-names" title="given-names">T.T.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Rac1-dependent phosphorylation and focal adhesion recruitment of myosin IIA regulates migration and mechanosensing</span>. <span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">175</span>–<span class="lpage" tagx="lpage" title="lpage">186</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2014.11.043">10.1016/j.cub.2014.11.043</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25544611">25544611</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelham</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span>., and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Cell locomotion and focal adhesions are regulated by substrate flexibility</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA.</span><span class="volume" tagx="volume" title="volume">94</span>:<span class="fpage" tagx="fpage" title="fpage">13661</span>–<span class="lpage" tagx="lpage" title="lpage">13665</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.94.25.13661">10.1073/pnas.94.25.13661</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9391082">9391082</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plotnikov</span><span class="given-names" tagx="given-names" title="given-names">S.V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasapera</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabass</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Force fluctuations within focal adhesions mediate ECM-rigidity sensing to guide directed cell migration</span>. <span class="source" tagx="source" title="source">Cell.</span><span class="volume" tagx="volume" title="volume">151</span>:<span class="fpage" tagx="fpage" title="fpage">1513</span>–<span class="lpage" tagx="lpage" title="lpage">1527</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2012.11.034">10.1016/j.cell.2012.11.034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23260139">23260139</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raab</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swift</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dingal</span><span class="given-names" tagx="given-names" title="given-names">P.C.D.P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shin</span><span class="given-names" tagx="given-names" title="given-names">J.-W.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Discher</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Crawling from soft to stiff matrix polarizes the cytoskeleton and phosphoregulates myosin-II heavy chain</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">199</span>:<span class="fpage" tagx="fpage" title="fpage">669</span>–<span class="lpage" tagx="lpage" title="lpage">683</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201205056">10.1083/jcb.201205056</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23128239">23128239</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riquelme</span><span class="given-names" tagx="given-names" title="given-names">D.N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barzik</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keating</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Selectivity in subunit composition of Ena/VASP tetramers</span>. <span class="source" tagx="source" title="source">Biosci. Rep.</span><span class="volume" tagx="volume" title="volume">35</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e00246</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/BSR20150149">10.1042/BSR20150149</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26221026">26221026</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riveline</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zamir</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balaban</span><span class="given-names" tagx="given-names" title="given-names">N.Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwarz</span><span class="given-names" tagx="given-names" title="given-names">U.S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishizaki</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narumiya</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kam</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geiger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bershadsky</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Focal contacts as mechanosensors: externally applied local mechanical force induces growth of focal contacts by an mDia1-dependent and ROCK-independent mechanism</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">153</span>:<span class="fpage" tagx="fpage" title="fpage">1175</span>–<span class="lpage" tagx="lpage" title="lpage">1186</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.153.6.1175">10.1083/jcb.153.6.1175</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11402062">11402062</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roussos</span><span class="given-names" tagx="given-names" title="given-names">E.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condeelis</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patsialou</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Chemotaxis in cancer</span>. <span class="source" tagx="source" title="source">Nat. Rev. Cancer.</span><span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">573</span>–<span class="lpage" tagx="lpage" title="lpage">587</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrc3078">10.1038/nrc3078</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21779009">21779009</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blankman</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardel</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luettjohann</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beckerle</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">A zyxin-mediated mechanism for actin stress fiber maintenance and repair</span>. <span class="source" tagx="source" title="source">Dev. Cell.</span><span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">365</span>–<span class="lpage" tagx="lpage" title="lpage">376</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.devcel.2010.08.008">10.1016/j.devcel.2010.08.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20833360">20833360</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Staunton</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doss</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindsay</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ros</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Correlating confocal microscopy and atomic force indentation reveals metastatic cancer cells stiffen during invasion into collagen I matrices</span>. <span class="source" tagx="source" title="source">Sci. Rep.</span><span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">19686</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep19686">10.1038/srep19686</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26813872">26813872</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephen</span><span class="given-names" tagx="given-names" title="given-names">R.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">L.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corcoran</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darbre</span><span class="given-names" tagx="given-names" title="given-names">P.D.</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Insulin-like growth factor receptor levels are regulated by cell density and by long term estrogen deprivation in MCF7 human breast cancer cells</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">276</span>:<span class="fpage" tagx="fpage" title="fpage">40080</span>–<span class="lpage" tagx="lpage" title="lpage">40086</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M105892200">10.1074/jbc.M105892200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11457860">11457860</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stutchbury</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atherton</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D.-Y.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ballestrem</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Distinct focal adhesion protein modules control different aspects of mechanotransduction</span>. <span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="volume" tagx="volume" title="volume">130</span>:<span class="fpage" tagx="fpage" title="fpage">1612</span>–<span class="lpage" tagx="lpage" title="lpage">1624</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.195362">10.1242/jcs.195362</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28302906">28302906</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sundararaghavan</span><span class="given-names" tagx="given-names" title="given-names">H.G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monteiro</span><span class="given-names" tagx="given-names" title="given-names">G.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Firestein</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shreiber</span><span class="given-names" tagx="given-names" title="given-names">D.I.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Neurite growth in 3D collagen gels with gradients of mechanical properties</span>. <span class="source" tagx="source" title="source">Biotechnol. Bioeng.</span><span class="volume" tagx="volume" title="volume">102</span>:<span class="fpage" tagx="fpage" title="fpage">632</span>–<span class="lpage" tagx="lpage" title="lpage">643</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/bit.22074">10.1002/bit.22074</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18767187">18767187</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunyer</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conte</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escribano</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elosegui-Artola</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Labernadie</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valon</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Navajas</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Aznar</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muñoz</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roca-Cusachs</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trepat</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Collective cell durotaxis emerges from long-range intercellular force transmission</span>. <span class="source" tagx="source" title="source">Science.</span><span class="volume" tagx="volume" title="volume">353</span>:<span class="fpage" tagx="fpage" title="fpage">1157</span>–<span class="lpage" tagx="lpage" title="lpage">1161</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aaf7119">10.1126/science.aaf7119</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27609894">27609894</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swaminathan</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The FAK-Arp2/3 interaction promotes leading edge advance and haptosensing by coupling nascent adhesions to lamellipodia actin</span>. <span class="source" tagx="source" title="source">Mol. Biol. Cell.</span><span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">1085</span>–<span class="lpage" tagx="lpage" title="lpage">1100</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.E15-08-0590">10.1091/mbc.E15-08-0590</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26842895">26842895</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ulrich</span><span class="given-names" tagx="given-names" title="given-names">T.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Juan Pardo</span><span class="given-names" tagx="given-names" title="given-names">E.M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The mechanical rigidity of the extracellular matrix regulates the structure, motility, and proliferation of glioma cells</span>. <span class="source" tagx="source" title="source">Cancer Res.</span><span class="volume" tagx="volume" title="volume">69</span>:<span class="fpage" tagx="fpage" title="fpage">4167</span>–<span class="lpage" tagx="lpage" title="lpage">4174</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-08-4859">10.1158/0008-5472.CAN-08-4859</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19435897">19435897</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vallenius</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Actin stress fibre subtypes in mesenchymal-migrating cells</span>. <span class="source" tagx="source" title="source">Open Biol.</span><span class="volume" tagx="volume" title="volume">3</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">130001</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsob.130001">10.1098/rsob.130001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23782578">23782578</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Helvert</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storm</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedl</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Mechanoreciprocity in cell migration</span>. <span class="source" tagx="source" title="source">Nat. Cell Biol.</span><span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">8</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41556-017-0012-0">10.1038/s41556-017-0012-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29269951">29269951</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dembo</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanks</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Focal adhesion kinase is involved in mechanosensing during fibroblast migration</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA.</span><span class="volume" tagx="volume" title="volume">98</span>:<span class="fpage" tagx="fpage" title="fpage">11295</span>–<span class="lpage" tagx="lpage" title="lpage">11300</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.201201198">10.1073/pnas.201201198</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11572981">11572981</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Mechanosensitive subcellular rheostasis drives emergent single-cell mechanical homeostasis</span>. <span class="source" tagx="source" title="source">Nat. Mater.</span><span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">961</span>–<span class="lpage" tagx="lpage" title="lpage">967</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmat4654">10.1038/nmat4654</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27240108">27240108</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">W.-H.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.-L.</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Fibroblasts probe substrate rigidity with filopodia extensions before occupying an area</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA.</span><span class="volume" tagx="volume" title="volume">111</span>:<span class="fpage" tagx="fpage" title="fpage">17176</span>–<span class="lpage" tagx="lpage" title="lpage">17181</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1412285111">10.1073/pnas.1412285111</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25404288">25404288</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worth</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodivala-Dilke</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">S.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morton</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Humphries</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parsons</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Alpha v beta3 integrin spatially regulates VASP and RIAM to control adhesion dynamics and migration</span>. <span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">189</span>:<span class="fpage" tagx="fpage" title="fpage">369</span>–<span class="lpage" tagx="lpage" title="lpage">383</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200912014">10.1083/jcb.200912014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20404115">20404115</a></span></span></li> <li tag="ref"><a name="bib61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">P.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aroush</span><span class="given-names" tagx="given-names" title="given-names">D.R.-B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asnacios</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dokukin</span><span class="given-names" tagx="given-names" title="given-names">M.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doss</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durand-Smet</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ekpenyong</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guck</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guz</span><span class="given-names" tagx="given-names" title="given-names">N.V.</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">A comparison of methods to assess cell mechanical properties</span>. <span class="source" tagx="source" title="source">Nat. Methods.</span><span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">491</span>–<span class="lpage" tagx="lpage" title="lpage">498</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41592-018-0015-1">10.1038/s41592-018-0015-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29915189">29915189</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plotnikov</span><span class="given-names" tagx="given-names" title="given-names">S.V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moalim</span><span class="given-names" tagx="given-names" title="given-names">A.Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterman</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span>, and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Two Distinct Actin Networks Mediate Traction Oscillations to Confer Focal Adhesion Mechanosensing</span>. <span class="source" tagx="source" title="source">Biophys. J.</span><span class="volume" tagx="volume" title="volume">112</span>:<span class="fpage" tagx="fpage" title="fpage">780</span>–<span class="lpage" tagx="lpage" title="lpage">794</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bpj.2016.12.035">10.1016/j.bpj.2016.12.035</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28256237">28256237</a></span></span></li> </ul> </div> </div>  </body></html>