<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Onco Targets Ther</span><span class="iso-abbrev" title="iso-abbrev">Onco Targets Ther</span><span class="publisher-id" title="publisher-id">OTT</span><span class="pmc" title="pmc">ott</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">OncoTargets and therapy</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1178-6930</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Dove</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7182457">7182457</a></span><span class="publisher-id" title="publisher-id">246031</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.2147/OTT.S246031">10.2147/OTT.S246031</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Research</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Chrysin Induced Cell Apoptosis and Inhibited Invasion Through Regulation of TET1 Expression in Gastric Cancer Cells</div> <div class="alt-title" title="alt-title">
Zhong et al</div> <div class="alt-title" title="alt-title">
Zhong et al</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-8334-7872</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">Xiaowei</span></span><a href="#AFF0001">1</a><a href="#FT0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Dianfeng</span></span><a href="#AFF0001">1</a><a href="#FT0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Ziping</span></span><a href="#AFF0002">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Chengshun</span></span><a href="#AFF0001">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Lin</span></span><a href="#AFF0001">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Yidan</span></span><a href="#AFF0002">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Da</span></span><a href="#AFF0003">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3586-1911</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">Qunyan</span></span><a href="#AN0001" /><a href="#AFF0004">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Dongxu</span></span><a href="#AFF0001">1</a></span><span class="citation_author_institution" id="AFF0001">[1], <span class="institution" title="institution">Laboratory Animal Center, College of Animal Science, Jilin University</span><span class="addr-line" title="addr-line">Changchun</span><span class="country" tagx="country" title="country">People’s Republic of China</span></span><span class="citation_author_institution" id="AFF0002">[2], <span class="institution" title="institution">Department of Hand Surgery, The First Hospital of Jilin University</span><span class="addr-line" title="addr-line">Changchun</span><span class="country" tagx="country" title="country">People’s Republic of China</span></span><span class="citation_author_institution" id="AFF0003">[3], <span class="institution" title="institution">Department of Pharmacy, Changchun University of Chinese Medicine</span><span class="addr-line" title="addr-line">Changchun</span><span class="country" tagx="country" title="country">People’s Republic of China</span></span><span class="citation_author_institution" id="AFF0004">[4], <span class="institution" title="institution">Department of Gastroenterology and Hepatology, Zhongshan Hospital, Fudan University</span><span class="addr-line" title="addr-line">Shanghai</span><span class="country" tagx="country" title="country">People’s Republic of China</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Correspondence: Qunyan Yao; Dongxu Wang Email yao.qunyan@zs-hospital.sh.cn; wang_dong_xu@jlu.edu.cn</div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p>These authors contributed equally to this work</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">3277</span><span class="lpage" tagx="lpage" title="lpage">3287</span><span class="history" title="history"><span class="received" title="received">received: 2020-1-15</span><span class="accepted" title="accepted">accepted: 2020-4-02</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Zhong et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This work is published and licensed by Dove Medical Press Limited. The full terms of this license are available at <a href="https://www.dovepress.com/terms.php">https://www.dovepress.com/terms.php</a> and incorporate the Creative Commons Attribution – Non Commercial (unported, v3.0) License (<a href="http://creativecommons.org/licenses/by-nc/3.0/">http://creativecommons.org/licenses/by-nc/3.0/</a>). By accessing the work you hereby accept the Terms. Non-commercial uses of the work are permitted without any further permission from Dove Medical Press Limited, provided the work is properly attributed. For permission for commercial use of this work, please see paragraphs 4.2 and 5 of our Terms (<a href="https://www.dovepress.com/terms.php">https://www.dovepress.com/terms.php</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="objective" title="sec"> <div class="title" tagx="title" title="title">
Objective</div> <p>Ten-eleven translocation (TET) enzymes that oxidize a 5-methylcytosine (5mC) to yield 5-hydroxymethylcytosine (5hmC) have been responsible for fine-tuning methylation patterns and exhibit role in epigenetic modifications. Chrysin, a natural flavone frequently present in honey, has been recognized to exhibit anti-tumor properties. In this study, we investigated the effects of Chrysin in the expression pattern of TET proteins in gastric cancer (GC) cells.</p> </div> <div class="materials and methods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <p>Using qRT-PCR and Western blot analysis, we analyzed the expression of TET1 in GC cells in vitro following treatment with Chrysin. Immunofluorescence staining detected the expression levels of 5mC and 5hmC. Flow cytometry, wound healing, and Matrigel invasion assays were performed to determine cell proliferation, cell cycle, apoptosis, and migration and invasion of GC cells following treatment with Chrysin, si-TET1, and TET1-KO. Furthermore, a xenograft model was developed to analyze the expression pattern of TET1 on tumor development in vivo.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>qRT-PCR and Western blot assays indicated that treatment with Chrysin significantly promoted the expression of TET1 in GC cells. Immunofluorescence study further confirmed that TET1 and 5hmC levels were significantly enhanced following treatment with Chrysin in MKN45 cells. Moreover, our results suggested that Chrysin could noticeably induce cell apoptosis and inhibit cell migration and invasion. Further, knockdown and overexpression of TET1 were conducted to investigate whether TET1 expression affected cell apoptosis, and cell migration and invasion in MKN45 cells. The results indicated that overexpression of TET1 markedly promoted cell apoptosis and inhibited cell migration and invasion. Furthermore, the TET1 gene knocked out was generated using the CRISPR/Cas9 system. Our data suggested that TET1 expression was associated with GC tumor growth in vivo.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>This study indicated that Chrysin exerted anti-tumor effects through the regulation of TET1 expression in GC and presented TET1 as a novel promising therapeutic target for GC therapy.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Chrysin</div> <div class="kwd" title="kwd">
TET1</div> <div class="kwd" title="kwd">
5hmC</div> <div class="kwd" title="kwd">
cell apoptosis</div> <div class="kwd" title="kwd">
cell invasion</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This work was supported by the National Natural Science Foundation of China under Grant 81803680, the China Postdoctoral Science Foundation under Grant 2018T110250 and 2016M601384, the Fundamental Research Funds for the Central Universities under Grant 2019JCKT-70, the Natural Science Foundation under Grant 2018SCZWSZX-045, the Jilin Education Department Program under Grant JJKH20200950KJ, and the Jilin Scientific and Technological Development Program under Grant 20190103071JH, 20180101254JC, and 0170623093-TC.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Globally, gastric cancer (GC) remains the third leading cause of cancer-associated mortality, with an estimated 783,000 deaths in 2018.<a href="#CIT0001">1</a> Etiologically, GC is considered a classic example of gene-environment interactions with <i>Helicobacter pylori</i> (<i>H. pylori</i>) as the most prevalent causes of gastric pathogenesis; besides, various genetic and epigenetic alterations have been associated with its carcinogenesis. Accumulating evidence indicated that aberrant epigenetic modifications, including DNA methylation, was observed in gastric cancer cells.<a href="#CIT0002">2</a> Ten-eleven translocation (TET) enzymes, which catalyze the oxidation of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), have emerged as major drivers of global epigenetic modification.<a href="#CIT0003">3</a> Previous studies have revealed that TET1-mediated demethylation was involved in the acquisition of aggressive behavior in gastric adenocarcinoma with enteroblastic differentiation.<a href="#CIT0004">4</a> Moreover, TET2 affects malignant cell activity through RASSF1A methylation in GC.<a href="#CIT0005">5</a> More recently, the down-regulation of TET3 expression and loss of 5hmC levels have also been demonstrated in GC.<a href="#CIT0006">6</a> These pieces of evidence indicated that TET proteins which regulated 5hmC level were closely associated with the development of GC.</p> <p>Chrysin (5,7-dihydroxyflavone), a natural and biologically active flavone, is extracted from chamomile, pleurotusostreatus, and honeycomb. It is well documented that Chrysin exhibits potent anti-inflammatory and antioxidant properties and cancer chemo-preventive activity through the induction of cell death by perturbing cell cycle progression across diverse cancer lines, including GC cells.<a href="#CIT0007">7</a> Moreover, Chrysin regulates MMP-9 expression through inhibition of AP-1 activity via suppression of the JNK1/2 and ERK1/2 signaling pathways in gastric cancer AGS cells.<a href="#CIT0008">8</a> Recent studies also suggested that Chrysin exhibits anti-cancer effects by suppressing the expression of RON through blocking Egr-1 and NF-κB in gastric cancer AGS cells.<a href="#CIT0009">9</a> Considering these studies, Chrysin may have a potential anti-cancer effect in GC.</p> <p>The high efficiency, specificity, and versatility of the CRISPR/Cas9 site-specific nuclease system have led to its extensive use as a genome-editing tool. The system utilizes an RNA-guide Cas9 protein combined with a short RNA (sgRNA) to induce the Cas9 mediated double-strand breaks in the target genomic DNA.<a href="#CIT0010">10</a> Besides, the CRISPR/Cas9 system can selectively modulate epigenetic factors, including DNA methylation. Therefore, the present study was initiated to investigate the gene expression profile of TET1 in MKN45 cells. Furthermore, the effects of TET1 expression on cell apoptosis, cell cycle, and cell migration and invasion were analyzed through overexpression or knockdown of TET1. The findings of this study suggested that Chrysin exerts anti-tumor activities through the regulation of TET1 expression in GC cells.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="cellcultureanddrugtreatment" title="sec"> <div class="title" tagx="title" title="title">
Cell Culture and Drug Treatment</div> <p>The human gastric epithelial cell line GES-1 and human GC cell lines MKN-45 were obtained from Procell (Wuhan, China). Cells were cultured in high glucose Dulbecco’s modified Eagle’s medium (DMEM; Gibco, Thermo Fisher Scientific, Inc., Waltham, MA, USA) supplemented with 10% fetal bovine serum (FBS; Gibco) and 1% penicillin-streptomycin mixture (Invitrogen; Thermo Fisher Scientific, Inc.) at 37°C in a humidified environment containing 5% CO<sub>2</sub>. 2 × 10<sup>5</sup> cells/mL cells at 80% confluence were cultured and pre-treated with Chrysin (Yuanye Bio-Technology, Shanghai) for 48 h.</p> </div> <div class="knockdownandoverexpressionoftet1" title="sec"> <div class="title" tagx="title" title="title">
Knockdown and Overexpression of <i>TET1</i></div> <p>Synthetic RNA oligonucleotides targeting <i>TET1</i> were obtained from RiboBio (Guangzhou, China). The siRNA targeting <i>TET1</i> sequence was GCACGCATGAATTTGGATA. Flag-HA-TET1 (ID 49792; FH-TET1-pEF) was procured from Addgene.</p> <p>The CRISPR/Cas9 plasmids were obtained from Addgene (px458). The sgRNA design and the procedures for the in vitro transcription have been described previously.<a href="#CIT0010">10</a> The sgRNA-oligo sequences used in this study are listed in <u><a href="https://www.dovepress.com/get_supplementary_file.php?f=246031.docx">Supplementary Table 1</a></u>.</p> <p>MKN45 cells were transfected with siTET1, TET1-KO, and FH-TET1-pEF for 48 h using Lipofectamine 2000 (ThermoFisher Scientific), respectively. Control cells were transfected with non-specific and scrambled siRNA.</p> </div> <div class="geneexpressionanalysis" title="sec"> <div class="title" tagx="title" title="title">
Gene Expression Analysis</div> <p>Total RNA was isolated from GES-1 and MKN45 cells using the TRNzol reagent (TIANGEN, Beijing, China) following the manufacturer’s instructions. cDNA was synthesized using the BioRT cDNA first-strand synthesis kit (Bioer Technology, Hangzhou, China) following treatment with DNase I (FermenTSA). Quantitative real-time PCR (qRT-PCR) was performed to determine gene expression of TET1 using the BioEasy SYBR Green I Real-Time PCR Kit (Bioer Technology, Hangzhou, China) on BIO-RAD iQ5 Multicolor Real-Time PCR Detection System (Bioer Tech. China). The primer sequences used in this study were summarized in <u><a href="https://www.dovepress.com/get_supplementary_file.php?f=246031.docx">Supplementary Table 2</a></u>. qRT-PCR was performed. PCR was performed by initial denaturation at 95°C for 3 min, followed by 40 cycles of denaturation at 95°C for 10 s, annealing at 60°C for 15 s, and extension at 72°C for 30 s. The 2<sup>−ΔΔCT</sup> method was used to determine relative gene expression, which was normalized to the amount of GAPDH mRNA. All experiments were performed at least in triplicate for each gene. Data are expressed as the mean ± SEM.</p> </div> <div class="westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Western Blot Analysis</div> <p>For Western blot analysis, total proteins were extracted from cell lines (1×10<sup>6</sup> cells/well) supplemented with protease inhibitors cocktail with protein extraction buffer (Novagen, Madison, WI, USA) with 2 × SDS lysis buffer. Protein concentrations were quantified using the BCA protein assay kit (TIANGEN, Beijing, China). Proteins were separated using 10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and then transferred onto a polyvinylidene difluoride (PVDF) membrane. Subsequently, membranes were blocked with 5% non-fat milk powder dry milk in Tris-buffered saline with Tween-20 (TBS-T; 0.1% Tween-20 in TBS) and incubated with primary antibodies including rabbit anti-TET1 (Abcam), anti-Bax (Abcam), anti-Bcl2 (Abcam) and mouse anti-GAPDH (Abcam), respectively each at a dilution of 1:2000 in 5% blocking buffer overnight at 4ºC. Then, the membranes were washed twice with TBS-T, and membranes were incubated with HRP-conjugated secondary antibodies (anti-mouse or anti-rabbit, Invitrogen) for 1 h at room temperature (RT). The target bands were visualized using the Chemiluminescence Kit, and the protein bands were quantified using ECL Super Signal (Pierce, USA).</p> </div> <div class="cellcountingkit-8assay" title="sec"> <div class="title" tagx="title" title="title">
Cell Counting Kit-8 Assay</div> <p>Cell viability was measured using the Cell Counting Kit-8 (CCK-8) assay kit (Dojindo, Kumamoto, Japan) was described previously.<a href="#CIT0011">11</a> Briefly, cells at a density of 4 × 10<sup>3</sup> cells/well were seeded in 96-well plates and incubated for 48 h (37°C, 5% CO<sub>2</sub>). Following incubation, 10 μL of CCK-8 solution was added to each well of the 96-well plates and incubated at 37°C for 2.5 h. Absorbance was measured at 450 nm using an automated microplate reader (Infinite M200, TECAN).</p> </div> <div class="cellcycleandapoptosisanalysis" title="sec"> <div class="title" tagx="title" title="title">
Cell Cycle and Apoptosis Analysis</div> <p>The cell cycle profile was determined using Propidium Iodide (PI) staining. In brief, MKN45 cells (1 × 10<sup>6</sup> cells/mL) were treated with Chrysin, siRNA, or FH-TET1-pEF for 48 h. Following incubation, the cells were washed with PBS and then fixed in 70% ethanol for 2 h at 4°C. These cells were incubated with PI and RNase A for 30 min and fluorescence (FL2) was measured using an AccuriTM C6 flow cytometer was used for the analysis of the cell cycle.</p> <p>Apoptotic cell death was measured using a fluorescein isothiocyanate (FITC)-conjugated Annexin V/PI assay, as described previously.<a href="#CIT0012">12</a> Briefly, MKN45 cells (5 × 10<sup>5</sup> cells/mL) were washed with ice-cold PBS, resuspended in 100 μL binding buffer, and stained with Annexin V (10 mg/mL) and -FITC/PI (50 mg/mL) after treatment for 48 h. Following incubation, the cells were washed with PBS twice and collected at a concentration of 1×10<sup>6</sup> cells/mL. For each treated cell sample, Annexin V-FITC and PI were added according to the manufacturer’s instructions. These cells were incubated for 30 min and then analyzed with an AccuriTM C6 flow cytometer (BD Biosciences, Franklin Lakes, NJ, USA). Cells stained with only annexin V were evaluated as being in early apoptosis; late apoptotic and necrotic cells were stained with both annexin V and propidium iodide.</p> </div> <div class="cellmigrationandinvasion" title="sec"> <div class="title" tagx="title" title="title">
Cell Migration and Invasion</div> <p>The cell migration assay was carried out using the in vitro scratch wound-healing assay. Cells were seeded in six-well plates. Briefly, at 48 h post-transfection, 5 × 10<sup>5</sup> cells were seeded into 6-wells plates and were starved for 12 h in DMEM containing 0.1% FBS. When the cell density reached about 90% confluence, a scratch line wound was created with a 200 µL sterile pipette tip perpendicular to the bottom of the plate. The location of the wound in the six-well plates was marked, and images were captured by a microscope at the indicated treatment time points 0 h, 12 h, 24 h, and 48 h at 37°C, The cells were quantitatively measured in the wound area and were photographed using an inverted microscope.</p> <p>For invasion assay, the cell invasion assay was carried out using Matrigel (BD Biosciences, NJ) coated basement membrane matrix (BD Biosciences, SA) with 10% FBS containing DMEM as the chemoattractant in the lower chamber. Subsequently, 3 × 10<sup>4</sup> of cells were added to the upper chamber with PMA in the presence of Chrysin, control or si-TET1, and allowed to invade the Matrigel for 24 h. After 24 h, the non-invading cells were removed through washes with PBS. The invading cells were fixed in 4% paraformaldehyde and stained with 0.1% crystal violet (Solarbio, China). The stained cells were counted using an inverted microscope.</p> </div> <div class="immunofluorescencestaining" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence Staining</div> <p>Briefly, the cells were plated onto coverslips and fixed with 4% paraformaldehyde solution for 15 min. After fixation, the cells were washed with PBS and permeabilized with PBS containing 0.5% Triton X-100 for 30 min. Subsequently, to denature the DNA, the cells were incubated with 4 M HCl for 15 min, rinsed with distilled water, and placed in 100 mM Tris-HCl (pH 8.5) for 10 min. After washing with PBS, non-specific binding sites were blocked with blocking buffer (10% FCS, 0.1% Tween-20 in PBS) for 1 h. Next, the cells were probed with different primary antibodies as follows: TET1 (1:100, Abcam), 5mC (1:100, Cell Signaling Technology) or 5hmC (1:100, Cell Signaling Technology) at 4°C overnight. The cells were washed with PBS three times for 10 min each followed by incubation with Alexa Fluor 488- or 594-conjugated secondary (anti-mouse or anti-rabbit) antibodies for 1 h at RT. Then, cells were stained with Hoechst 33342 (Solarbio, China) for 10 min. The cells were washed thrice with PBS for 10 min each, air-dried, and mounted on a coverslip and a glass slide using an antifade mounting medium 4,6-diamidino-2-phenylindole (DAPI; BOSTER, China). The cells were imaged using a fluorescence microscope. The average fluorescence intensities were analyzed using ImageJ analysis software.<a href="#CIT0013">13</a></p> </div> <div class="nudemicexenograftmodel" title="sec"> <div class="title" tagx="title" title="title">
Nude Mice Xenograft Model</div> <p>The animals were cared for in accordance with the Guide for the care and use of laboratory animals in China. All experimental procedures were approved by the Animal Care and Use Committee of Jilin University, Changchun, China (Grant No. SY201907008). The mice were housed in laboratory cages under controlled laboratory conditions at 24°C with a relative humidity of 50–60%, under a 12-hour light/dark cycles. Animals were provided <i>ad libitum</i> access to standard rodent food and tap water. All the mice were healthy and had no infection during the experimental period. All surgical procedures were carried out under aseptic conditions.</p> <p>Thirty female nude mice (6 weeks old) were procured from the Laboratory Animal Center, Jilin University. The Control, FH-TET1-pEF, and TET1-KO cells (3 × 10<sup>5</sup> cells) were injected subcutaneously into the left flank areas of nude mice. The tumors were observed after eight days. The mice were used for experiments when their tumor volumes reached approximately 40–60 mm<sup>3</sup>. The mice were randomly divided into six groups (N = 5). The chrysin group was administered with 20 mg/kg by gavage for 14 days. The Con group was administered with an equal volume of saline. The size of the tumors was measured every day. The length (L) and width (W) were recorded and the tumor volumes were calculated as (L × W<sup>2</sup>/2).</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analysis</div> <p>All data were expressed as mean ± standard deviations (SD). Differences between groups were assessed using the unpaired Student’s <i>t</i>-test. Analysis of variance (ANOVA) with post-hoc analysis was used for multiple comparisons. All statistical analyses were performed using GraphPad Prism 5.0 (GraphPad Software, Inc., San Diego, CA). A p-value of &amp;lt; 0.05 was considered statistically significant.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="chrysininducedcellapoptosisandinhibitedinvasion" title="sec"> <div class="title" tagx="title" title="title">
Chrysin Induced Cell Apoptosis and Inhibited Invasion</div> <p>To evaluate TET1 expression, qRT-PCR was performed. The results indicated that the expression of <i>TET1</i> was reduced in MKN45 cells (<a href="#F0001">Figure 1A</a>). To determine if Chrysin affected the growth of MKN45 cells, the CCK8 assay was carried out. Our results indicated that 80 and 160 μM of Chrysin exhibited a toxic effect on MKN45 cells. Therefore, 40 μM of Chrysin was used for qPCR and Western blot analysis (<a href="#F0001">Figure 1B</a>). The qPCR results indicated <i>TET1</i> expression was significantly increased in MKN45 cells after treatment with Chrysin (<a href="#F0001">Figure 1C</a>). The findings of Western blot analysis were consistent with qRT-PCR data (<a href="#F0001">Figure 1D</a>). To further confirm the TET1 protein expression pattern, we analyzed TET1, 5mC, and 5hmC levels. As anticipated, increased TET1 (<a href="#F0001">Figure 1E</a> and <a href="#F0001">F</a>) and 5hmC (<a href="#F0001">Figure 1G</a> and <a href="#F0001">H</a>) levels were observed after treatment with Chrysin. These results indicated that Chrysin noticeably regulated the expression of TET1, which significantly enhanced 5hmC levels in MKN45 cells. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Analysis of TET1 expression pattern after treatment with Chrysin. Relative expression of <i>TET1</i> between GES-1 and MKN45 cells (<b>A</b>). The cell growth was analyzed by CCK8 assay (<b>B</b>). Relative expression of <i>TETs</i> was analyzed by qRT-PCR after treatment with Chrysin in MKN45 cells (<b>C</b>). The expression of TET1 was analyzed by Western blot assay (<b>D</b>). Immunofluorescence localization of TET1 after treatment with Chrysin (<b>E</b>). Green, indicates TET1. Blue, indicates Hoechst. Statistical analyses of TET1 fluorescence intensity (<b>F</b>). Immunofluorescence localization of 5mC and 5hmC (<b>G</b>). Green, indicates 5mC. Red, indicates 5hmC. Blue, indicates Hoechst. Statistical analyses of 5mC and 5hmC+ fluorescence intensity (<b>H</b>). The bar represents 50 μm. * (<i>p</i> &amp;lt; 0.05), ** (<i>p</i> &amp;lt; 0.01) and *** (<i>p</i> &amp;lt; 0.005) indicate statistically significant differences.</p>   </div> <p /> <p>To further confirm the effect of Chrysin, cell apoptosis, cell cycle, migration, and invasion were analyzed in MKN45 cells. Cell apoptosis result indicated that apoptosis was significantly higher in MKN45 cells treated with Chrysin as compared to the control (<a href="#F0002">Figure 2A</a> and <a href="#F0002">B</a>). The expression of BAX and BCL2 further confirmed cell apoptosis findings (<u><a href="https://www.dovepress.com/get_supplementary_file.php?f=246031.docx">Supplementary Figure 1</a></u>). Analysis of the cell cycle suggested that the cells were noticeably arrested in G0/G1 phase, while reduced number of the Chrysin-treated cells was observed in the S phase (<a href="#F0002">Figure 2C</a> and <a href="#F0002">D</a>). In addition, Chrysin treatment markedly inhibited cell migration at 48 h (<a href="#F0002">Figure 2E</a> and <a href="#F0002">F</a>). Moreover, a significantly reduced invasion of cells was showed after Chrysin treatment in MKN45 cells (<a href="#F0002">Figure 2G</a> and <a href="#F0002">H</a>). Together, these results indicated that treatment with Chrysin notably altered the cell apoptosis, cell cycle, and cell migration and invasion capabilities of MKN45 cells. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Analysis of cell apoptosis, cell cycle, and cell migration and invasion in GC cells. Cell apoptosis was analyzed after treatment with Chrysin (<b>A</b>). Statistical analysis of the percentage of cell apoptosis (<b>B</b>). The cell cycle was analyzed after treatment with Chrysin (<b>C</b>). Statistical analysis of the percentage of cells in the major phases of the cell cycle (<b>D</b>). Cell migration was analyzed after treatment with Chrysin (<b>E</b>). Statistical analysis of cell migration with wound closure assay (<b>F</b>). The cell invasion was analyzed after treatment with Chrysin (<b>G</b>). Statistical analysis of the cell invasion (<b>H</b>). * (<i>p</i> &amp;lt; 0.05), ** (<i>p</i> &amp;lt; 0.01) and *** (<i>p</i> &amp;lt; 0.005) indicate statistically significant differences.</p>   </div> <p /> </div> <div class="effectoftet1expressionpatternoncellapoptosis,cellcycle,migration,andinvasion" title="sec"> <div class="title" tagx="title" title="title">
Effect of TET1 Expression Pattern on Cell Apoptosis, Cell Cycle, Migration, and Invasion</div> <p>To further analyze the effects of <i>TET1</i> expression pattern on cell apoptosis, cell cycle, cell migration, and invasion, knockdown and overexpression of <i>TET1</i> were performed. qRT-PCR results indicated a reduced expression of <i>TET1</i> in the si-TET1 group, while overexpression of <i>TET1</i> was observed in the FH-TET1-pEF group (<a href="#F0003">Figure 3A</a>). Western blot assay further confirmed the findings of qPCR data (<a href="#F0003">Figure 3B</a>). CCK8 assay also showed that overexpression of <i>TET1</i> markedly inhibited cell growth (<a href="#F0003">Figure 3C</a>). IF result also revealed increased expression levels of TET1 in the FH-TET1-pEF group (<a href="#F0003">Figure 3D</a> and <a href="#F0003">E</a>). Moreover, the global DNA methylation level indicated that 5hmC was up-regulated through the overexpression of TET1 (<a href="#F0003">Figure 3F</a> and <a href="#F0003">G</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Analysis of TET1 expression pattern in MKN45 cells. Relative expression of <i>TET1</i> in Nc, FH-TET1-pEF, and si-TET1 groups (<b>A</b>) using qPCR. The expression of TET1 using Western blot assay (<b>B</b>). The cell growth was analyzed by CCK8 assay (<b>C</b>). Immunofluorescence localization of TET1 (<b>D</b>). Green, indicates TET1. Blue, indicates Hoechst. Statistical analyses of TET1 fluorescence intensity (<b>E</b>). Immunofluorescence localization of 5mC and 5hmC (<b>F</b>). Green, indicates 5mC. Red, indicates 5hmC. Blue, indicates Hoechst. Statistical analyses of 5mC and 5hmC fluoressence intensity (<b>G</b>). The bar represents 50 μm. * (<i>p</i>&amp;lt; 0.05) and ** (<i>p</i> &amp;lt; 0.01) indicate statistically significant differences.</p>   </div> <p /> <p>The findings of cell apoptosis showed that overexpression of <i>TET1</i> significantly induced cell death (<a href="#F0004">Figure 4A</a> and <a href="#F0004">B</a>). However, up-regulation or down-regulation of TET1 expression did not alter the cell cycle (<a href="#F0004">Figure 4C</a> and <a href="#F0004">D</a>). Also, overexpression of TET1 markedly inhibited cell migration at 48 h (<a href="#F0005">Figure 5A</a> and <a href="#F0005">B</a>). Compared to the control group, reduced cell invasion was observed in the FH-TET1-pEF group (<a href="#F0005">Figure 5C</a> and <a href="#F0005">D</a>). Furthermore, the effect of Chrysin on cell migration, invasion, and growth was analyzed following transfection with si-TET1 in GC cells. The results indicated that cell migration, invasion, and growth were markedly rescued after treatment with Chrysin in si-TET1 group (<a href="#F0002">Supplementary Figure 2</a>). Taken together, these results suggested that the overexpression of <i>TET1</i> plays a crucial role in cell apoptosis and migration and invasion. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Analysis of cell apoptosis and cell cycle after knockdown and overexpression of TET1. The cell apoptosis was analyzed in Nc, FH-TET1-pEF, and si-TET1 groups (<b>A</b>). Statistical analysis of the percentage of cell apoptosis (<b>B</b>). The cell cycle was analyzed in Nc, FH-TET1-pEF, and si-TET1 groups (<b>C</b>). Statistical analysis of the percentage of cell apoptosis (<b>D</b>). The data represents the mean ± SD of three independent experiments. ** (<i>p</i> &amp;lt; 0.01) indicate statistically significant differences.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Analysis of cell migration and invasion in GC cells with after knockdown and overexpression of TET1. The cell migration was analyzed in Nc, FH-TET1-pEF and si-TET1 groups (<b>A</b>). Statistical analysis of cell migration with wound closure assay (<b>B</b>). The cell invasion was analyzed in Nc, FH-TET1-pEFs and si-TET1 groups (<b>C</b>). Statistical analysis of the percentage of cell invasion (<b>D</b>). The data represents the mean ± SD of three independent experiments.* (<i>p</i>&amp;lt; 0.05), ** (<i>p</i> &amp;lt; 0.01) and *** (<i>p</i> &amp;lt; 0.005) indicate statistically significant differences.</p>   </div> <p /> </div> <div class="chrysininhibitedtumorgrowthinvivo" title="sec"> <div class="title" tagx="title" title="title">
Chrysin Inhibited Tumor Growth in vivo</div> <p>To further confirm the chrysin-meditated TET1 expression, MKN45 cells were injected into nude mice. After eight days, the mice were treated with Chrysin (20 mg/kg) or an equal volume of saline for 14 days. The results demonstrated that Chrysin treatment substantially inhibited the growth of the tumors (<a href="#F0006">Figure 6A</a> and <a href="#F0006">B</a>). In addition, the expression pattern of TET1 was also analyzed in vivo. The qRT-PCR and Western blot results indicated that Chrysin promoted the expression of TET1 (<a href="#F0006">Figure 6C</a> and <a href="#F0006">D</a>). Collectively, these results suggested that Chrysin significantly inhibited tumor growth through overexpression of TET1 in vivo. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Analysis of the tumor volume and TET1 expression in vivo. Morphological alterations in mouse tumor tissue after treatment with chrysin (20 mg/kg) (<b>A</b>). Analysis of tumor volume (<b>B</b>). Relative expression of <i>TET1</i> using qRT-PCR in the tumor of mice (<b>C</b>). The expression of TET1 by Western blot assay (<b>D</b>). * (<i>p</i> &amp;lt; 0.05) and ** (<i>p</i> &amp;lt; 0.01) indicate statistically significant differences.</p>   </div> <p /> </div> <div class="crispr/cas9mediatedtet1expression" title="sec"> <div class="title" tagx="title" title="title">
CRISPR/Cas9 Mediated TET1 Expression</div> <p>In order to knock out (KO) TET1 in MKN45 cells, two sgRNAs targeting the exon2 of TET1 were designed (<a href="#F0007">Figure 7A</a>). The qRT-PCR and Western blot results suggested that the expression of TET1 was reduced in TET1 KO cells (<a href="#F0007">Figure 7B</a> and <a href="#F0007">C</a>). To investigate the expression pattern of TET1 in vivo, the TET1 KO cells were injected into nude mice. The results demonstrated that the under-expression of TET1 markedly enhanced tumor growth. However, treatment with Chrysin could inhibit tumor growth in the TET1 KO group (<a href="#F0007">Figure 7D</a> and <a href="#F0007">E</a>). Together, these findings suggested that TET1 expression was closely associated with tumor growth in GC. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>CRISPR/Cas9-mediated gene targeting of TET1. Schematic representation of sgRNA targeting the TET1 gene loci (<b>A</b>). The expression of TET1 using qRT-PCR (<b>B</b>) and Western blot (<b>C</b>). The tumor morphology (<b>D</b>) and volume (<b>E</b>). E indicated Exon. ** (<i>p</i> &amp;lt; 0.01), *** (<i>p</i> &amp;lt; 0.005) and **** (<i>p</i> &amp;lt; 0.001) indicate statistically significant differences.</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Chrysin is a natural and biologically active flavonoid presented in many plant extracts, including blue passionflower (<i>Passiflora caerulea</i>), honey, and propolis. Chrysin has been known to exert potent anti-inflammatory, antioxidant, and anti-cancer properties across diverse cancers.<a href="#CIT0014">14</a> In this study, we have investigated the anti-cancer effects of Chrysin against GC cells both in vitro and in vivo and our data suggested that Chrysin exerted significant anti-tumor capabilities including enhanced cell apoptosis, cell cycle arrest, increased cell migration, and invasion in GC cells which further supported our previous findings in melanoma<a href="#CIT0015">15</a> cells and osteoarthritis chondrocytes.<a href="#CIT0016">16</a> Recently, studies have indicated that Chrysin exhibited the ability to regulate gene expression of hTERT in breast cancer.<a href="#CIT0017">17</a> Moreover, Chrysin could regulate MMP-9 expression in GC.<a href="#CIT0008">8</a> However, there is a paucity of studies on the effects of Chrysin in altering the expression of TET1 in GC. Previous studies have demonstrated that TET1 plays a crucial role in the development of cancer.<a href="#CIT0018">18</a>,<a href="#CIT0019">19</a> Moreover, loss of 5hmC was also found to be associated with GC.<a href="#CIT0020">20</a> Thus, the present study investigated the effects of treatment with Chrysin on the gene expression pattern of TET1 in GC cells.</p> <p>Furthermore, to validate the role of TET1 in cell apoptosis, cell cycle, cell migration, and invasion, knockdown and overexpression of <i>TET1</i> were conducted in MKN45 cells. The results indicated that overexpression of <i>TET1</i> significantly induced cell apoptosis, which was consistent with previous studies.<a href="#CIT0021">21</a> More recently, a study also indicated that <i>TET1</i> overexpression was considerably associated with cell proliferation in colon cancer cells.<a href="#CIT0022">22</a> Correspondingly, our results also indicated that overexpression of <i>TET1</i> inhibited cell growth of GC cells. Besides, overexpression of <i>TET1</i> also inhibited cell migration and invasive capabilities of GC cells, consistent with previous results in hepatocellular carcinoma.<a href="#CIT0023">23</a> Collectively, these results suggested that <i>TET1</i> is crucially involved in biological processes in GC cells.</p> <p>Furthermore, to clarify the effect of TET1 proteins on gastric cancer in vivo, we established a xenograft model using nude mice. Chrysin treatment significantly promoted TET1 expression in vivo, which further confirmed our data in MKN45 cells. Furthermore, Chrysin inhibited tumor growth in mice, which was in accordance with previous data in melanoma.<a href="#CIT0024">24</a> Accumulating studies suggested that the RNA-guided nucleases from CRISPR/Cas9 system as the most reliable tools for gene editing in GC cells.<a href="#CIT0025">25</a> Through CRISPR/Cas9 system-mediated knock out of the TET1 gene, this study revealed that tumor growth was markedly enhanced upon TET1 knockdown in GC cells. Overall, these results indicated that Chrysin-mediated anti-tumor activities in GC cells via overexpression of TET1 expression in vivo. Thus, the ability of Chrysin to induce tumor cell apoptosis through regulation of TET1 expression is an important strategy for the development of an anti-cancer agent in GC therapy.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In conclusion, the present study demonstrated that Chrysin regulates TET1 expression in GC cells. Overexpression of <i>TET1</i> significantly enhanced 5hmC levels, promoted cell apoptosis, and inhibited cell migration and invasion. Although further studies are warranted to elucidate the underlying molecular mechanism of Chrysin-mediated cell apoptosis, migration, and invasion in GC cells, this study indicated that Chrysin exerted anti-tumor effects in GC and presented TET1 as a novel promising biomarker therapeutic target for GC therapy.</p> </div> </div> <div class="back" title="back"> <div class="ethicsandconsentstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics and Consent Statement</div> <p>The experiments involving mice were carried out in accordance with the Guidelines on Animal Care and Use of Animals in Research approved by the Animal Care and Use Committee of Jilin University, Changchun, China (Grant No. SY201907008). Thirty female nude mice (6 weeks old) were obtained from the Laboratory Animal Center, Jilin University. The mice were housed at 24 °C with 50–60% humidity and 12-hour light/dark cycles. All operations were carried out under aseptic conditions.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>All authors made substantial contributions to conception and design, acquisition of data, or analysis and interpretation of data; took part in drafting the article or revising it critically for important intellectual content; gave final approval of the version to be published; and agree to be accountable for all aspects of the work.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure</div> <p>The authors declare that they have no competing interests in this work.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bray</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ferlay</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Soerjomataram</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Siegel</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Torre</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jemal</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span>. <span class="source" tagx="source" title="source"><i>CA Cancer J Clin</i></span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">68</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">394</span>–<span class="lpage" tagx="lpage" title="lpage">424</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.3322/caac.21492">10.3322/caac.21492</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30207593">30207593</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">DQ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Downregulation of microRNA-27b-3p via aberrant DNA methylation contributes to malignant behavior of gastric cancer cells by targeting GSPT1</span>. <span class="source" tagx="source" title="source"><i>Biomed Pharmacother</i></span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">119</span>:<span class="fpage" tagx="fpage" title="fpage">109417</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biopha.2019.109417">10.1016/j.biopha.2019.109417</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31539861">31539861</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">YF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">BZ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Tet-mediated formation of 5-carboxylcytosine and its excision by TDG in mammalian DNA</span>. <span class="source" tagx="source" title="source"><i>Science</i></span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">333</span>(<span class="issue" tagx="issue" title="issue">6047</span>):<span class="fpage" tagx="fpage" title="fpage">1303</span>–<span class="lpage" tagx="lpage" title="lpage">1307</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1210944">10.1126/science.1210944</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21817016">21817016</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yatagai</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Akazawa</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">TP53 inactivation and expression of methylation-associated proteins in gastric adenocarcinoma with enteroblastic differentiation</span>. <span class="source" tagx="source" title="source"><i>Virchows Arch Int J Pathol</i></span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">474</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">315</span>–<span class="lpage" tagx="lpage" title="lpage">324</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1007/s00428-018-2508-9">10.1007/s00428-018-2508-9</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">The FENDRR/miR-214-3P/TET2 axis affects cell malignant activity via RASSF1A methylation in gastric cancer</span>. <span class="source" tagx="source" title="source"><i>Am J Transl Res</i></span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">3211</span>–<span class="lpage" tagx="lpage" title="lpage">3223</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30416662">30416662</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kurabe</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Matsushima</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Robust quantitative assessments of cytosine modifications and changes in the expressions of related enzymes in gastric cancer</span>. <span class="source" tagx="source" title="source"><i>Gastric Cancer</i></span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">516</span>–<span class="lpage" tagx="lpage" title="lpage">525</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1007/s10120-014-0409-4">10.1007/s10120-014-0409-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25098926">25098926</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mohammadian</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pilehvar-Soltanahmadi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alipour</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dadashpour</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zarghami</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chrysin alters microRNAs expression levels in gastric cancer cells: possible molecular mechanism</span>. <span class="source" tagx="source" title="source"><i>Drug Res</i></span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">67</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">509</span>–<span class="lpage" tagx="lpage" title="lpage">514</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1055/s-0042-119647">10.1055/s-0042-119647</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Khoi</span><span class="given-names" tagx="given-names" title="given-names">PN</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Chrysin inhibits tumor promoter-induced MMP-9 expression by blocking AP-1 via suppression of ERK and JNK pathways in gastric cancer cells</span>. <span class="source" tagx="source" title="source"><i>PLoS One</i></span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">e0124007</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0124007">10.1371/journal.pone.0124007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25875631">25875631</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Khoi</span><span class="given-names" tagx="given-names" title="given-names">PN</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Chrysin inhibits cell invasion by inhibition of Recepteur d’origine Nantais via suppressing early growth response-1 and NF-kappa B transcription factor activities in gastric cancer cells</span>. <span class="source" tagx="source" title="source"><i>Int J Oncol</i></span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">46</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">1835</span>–<span class="lpage" tagx="lpage" title="lpage">1843</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.3892/ijo.2015.2847">10.3892/ijo.2015.2847</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25625479">25625479</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ran</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Multiplex genome engineering using CRISPR/Cas systems</span>. <span class="source" tagx="source" title="source"><i>Science</i></span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">339</span>(<span class="issue" tagx="issue" title="issue">6121</span>):<span class="fpage" tagx="fpage" title="fpage">819</span>–<span class="lpage" tagx="lpage" title="lpage">823</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1231143">10.1126/science.1231143</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23287718">23287718</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="mixed-article-title" title="mixed-article-title">Combined application of anti-VEGF and anti-EGFR attenuates the growth and angiogenesis of colorectal cancer mainly through suppressing AKT and ERK signaling in mice model</span>. <span class="source" tagx="source" title="source"><i>BMC Cancer</i></span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">791</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1186/s12885-016-2834-8">10.1186/s12885-016-2834-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27729020">27729020</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">William-Faltaos</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rouillard</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lechat</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bastian</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cell cycle arrest and apoptosis induced by oxaliplatin (L-OHP) on four human cancer cell lines</span>. <span class="source" tagx="source" title="source"><i>Anticancer Res</i></span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">3A</span>):<span class="fpage" tagx="fpage" title="fpage">2093</span>–<span class="lpage" tagx="lpage" title="lpage">2099</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16827150">16827150</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaitseva</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaitsev</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alenina</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bader</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Krivokharchenko</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dynamics of DNA-demethylation in early mouse and rat embryos developed in vivo and in vitro</span>. <span class="source" tagx="source" title="source"><i>Mol Reprod Dev</i></span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">74</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">1255</span>–<span class="lpage" tagx="lpage" title="lpage">1261</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1002/mrd.20704">10.1002/mrd.20704</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17290422">17290422</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="label" tagx="label" title="label">14.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Salimi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roudkenar</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Seydi</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Chrysin as an anti-cancer agent exerts selective toxicity by directly inhibiting mitochondrial complex II and V in CLL B-lymphocytes</span>. <span class="source" tagx="source" title="source"><i>Cancer Invest</i></span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">35</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">174</span>–<span class="lpage" tagx="lpage" title="lpage">186</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1080/07357907.2016.1276187">10.1080/07357907.2016.1276187</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28301251">28301251</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">YW</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kuo</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Chrysin inhibit human melanoma A375.S2 cell migration and invasion via affecting MAPK signaling and NF-kappaB signaling pathway in vitro</span>. <span class="source" tagx="source" title="source"><i>Environ Toxicol</i></span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">34</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">434</span>–<span class="lpage" tagx="lpage" title="lpage">442</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1002/tox.22697">10.1002/tox.22697</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30578657">30578657</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Chrysin protects human osteoarthritis chondrocytes by inhibiting inflammatory mediator expression via HMGB1 suppression</span>. <span class="source" tagx="source" title="source"><i>Mol Med Rep</i></span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">1222</span>–<span class="lpage" tagx="lpage" title="lpage">1229</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.3892/mmr.2018.9724">10.3892/mmr.2018.9724</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30535473">30535473</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rasouli</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zarghami</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>. <span class="mixed-article-title" title="mixed-article-title">Synergistic growth inhibitory effects of chrysin and metformin combination on breast cancer cells through hTERT and cyclin D1 suppression</span>. <span class="source" tagx="source" title="source"><i>Asian Pac J Cancer Prev</i></span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">977</span>–<span class="lpage" tagx="lpage" title="lpage">982</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.22034/APJCP.2018.19.4.977">10.22034/APJCP.2018.19.4.977</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29693804">29693804</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Connections between TET proteins and aberrant DNA modification in cancer</span>. <span class="source" tagx="source" title="source"><i>Trends Genet</i></span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">464</span>–<span class="lpage" tagx="lpage" title="lpage">474</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tig.2014.07.005">10.1016/j.tig.2014.07.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25132561">25132561</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cimmino</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dolgalev</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Restoration of TET2 function blocks aberrant self-renewal and Leukemia progression</span>. <span class="source" tagx="source" title="source"><i>Cell</i></span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">170</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1079</span>–<span class="lpage" tagx="lpage" title="lpage">1095</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2017.07.032">10.1016/j.cell.2017.07.032</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28823558">28823558</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Ten-eleven translocation 1 dysfunction reduces 5-hydroxymethylcytosine expression levels in gastric cancer cells</span>. <span class="source" tagx="source" title="source"><i>Oncol Lett</i></span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">278</span>–<span class="lpage" tagx="lpage" title="lpage">284</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.3892/ol.2017.7264">10.3892/ol.2017.7264</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29285192">29285192</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Epigenetic co-deregulation of EZH2/TET1 is a senescence-countering, actionable vulnerability in triple-negative breast cancer</span>. <span class="source" tagx="source" title="source"><i>Theranostics</i></span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">761</span>–<span class="lpage" tagx="lpage" title="lpage">777</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.7150/thno.29520">10.7150/thno.29520</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30809307">30809307</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">TET1 suppresses colon cancer proliferation by impairing beta-catenin signal pathway</span>. <span class="source" tagx="source" title="source"><i>J Cell Biochem</i></span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">120</span>:<span class="fpage" tagx="fpage" title="fpage">12559</span>–<span class="lpage" tagx="lpage" title="lpage">12565</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30825236">30825236</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Circular RNA circTRIM33-12 acts as the sponge of MicroRNA-191 to suppress hepatocellular carcinoma progression</span>. <span class="source" tagx="source" title="source"><i>Mol Cancer</i></span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">1</span>). doi:<span class="pub-id"><a href="https://dx.doi.org/10.1186/s12943-019-1031-1">10.1186/s12943-019-1031-1</a></span>.</span></li> <li tag="ref"><a name="CIT0024" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sassi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Maatouk</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">El Gueder</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Chrysin, a natural and biologically active flavonoid suppresses tumor growth of mouse B16F10 melanoma cells: in vitro and in vivo study</span>. <span class="source" tagx="source" title="source"><i>Chem Biol Interact</i></span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">283</span>:<span class="fpage" tagx="fpage" title="fpage">10</span>–<span class="lpage" tagx="lpage" title="lpage">19</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cbi.2017.11.022">10.1016/j.cbi.2017.11.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29352974">29352974</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>. et al. <span class="mixed-article-title" title="mixed-article-title">METTL3-mediated m6 A modification of HDGF mRNA promotes gastric cancer progression and has prognostic significance</span>. <span class="source" tagx="source" title="source"><i>Gut</i></span>. <span class="year" tagx="year" title="year">2019</span>:<span class="page-range" tagx="page-range" title="page-range">gutjnl-2019–319639</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1136/gutjnl-2019-319639">10.1136/gutjnl-2019-319639</a></span></span></li> </ul> </div> </div>  </body></html>