<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Enzyme Inhib Med Chem</span><span class="iso-abbrev" title="iso-abbrev">J Enzyme Inhib Med Chem</span><span class="publisher-id" title="publisher-id">IENZ</span><span class="publisher-id" title="publisher-id">ienz20</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Enzyme Inhibition and Medicinal Chemistry</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1475-6366</span><span class="issn-epub" tagx="issn" title="issn-epub">1475-6374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844378">6844378</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31690133">31690133</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/14756366.2019.1684911">10.1080/14756366.2019.1684911</a></span><span class="publisher-id" title="publisher-id">1684911</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Paper</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Structure activity relationship studies on Amb639752: toward the identification of a common pharmacophoric structure for DGKα inhibitors</div> <div class="alt-title" title="alt-title">
S. Velnati et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velnati</span><span class="given-names" tagx="given-names" title="given-names">Suresh</span></span><a href="#AF0001"><sup>a</sup></a><a href="#AF0002"><sup>b</sup></a><a href="#FN0001"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massarotti</span><span class="given-names" tagx="given-names" title="given-names">Alberto</span></span><a href="#AF0003"><sup>c</sup></a><a href="#FN0001"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antona</span><span class="given-names" tagx="given-names" title="given-names">Annamaria</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talmon</span><span class="given-names" tagx="given-names" title="given-names">Maria</span></span><a href="#AF0004"><sup>d</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fresu</span><span class="given-names" tagx="given-names" title="given-names">Luigia Grazia</span></span><a href="#AF0004"><sup>d</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galetto</span><span class="given-names" tagx="given-names" title="given-names">Alessandra Silvia</span></span><a href="#AF0001"><sup>a</sup></a><a href="#AF0005"><sup>e</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capello</span><span class="given-names" tagx="given-names" title="given-names">Daniela</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertoni</span><span class="given-names" tagx="given-names" title="given-names">Alessandra</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mercalli</span><span class="given-names" tagx="given-names" title="given-names">Valentina</span></span><a href="#AF0003"><sup>c</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graziani</span><span class="given-names" tagx="given-names" title="given-names">Andrea</span></span><a href="#AF0006"><sup>f</sup></a><a href="#AF0007"><sup>g</sup></a><a href="#AN0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tron</span><span class="given-names" tagx="given-names" title="given-names">Gian Cesare</span></span><a href="#AF0003"><sup>c</sup></a><a href="#FN0002" /><a href="#AN0002" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldanzi</span><span class="given-names" tagx="given-names" title="given-names">Gianluca</span></span><a href="#AF0001"><sup>a</sup></a><a href="#AF0002"><sup>b</sup></a><a href="#FN0002" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Department of Translational Medicine, University of Piemonte Orientale</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Institute for Research and Cure of Autoimmune Diseases, CAAD, University of Piemonte Orientale</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">Department of Pharmaceutical Sciences, University of Piemonte Orientale</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0004">[d], <span class="institution" title="institution">Department of Health Sciences, School of Medicine, University of Piemonte Orientale</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0005">[e], <span class="institution" title="institution">Palliative Care Division, A.S.L.</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0006">[f], <span class="institution" title="institution">Università Vita-Salute San Raffaele</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="AF0007">[g], <span class="institution" title="institution">Department of Molecular Biotechnology and Health Sciences, Molecular Biotechnology Center, University of Torino</span><span class="country" tagx="country" title="country">Italy</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p>These authors contributed equally to this work.</p> </div> <div class="fn-type-" title=""> <p>†These authors shared senior authorship.</p> </div> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed <a href="https://doi.org/10.1080/14756366.2019.1684911"><u>here</u></a>.</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Andrea Graziani <span class="email" tagx="email" title="email">andrea.graziani@hsr.it</span><span class="institution" title="institution">Università Vita-Salute San Raffaele</span>, Milan 20132, <span class="country" tagx="country" title="country">Italy</span>; </div> <div class="corresp" title="corresp">
Gian Cesare Tron <span class="email" tagx="email" title="email">giancesare.tron@uniupo.it</span><span class="institution" title="institution">Department of Pharmaceutical Science, University of Piemonte Orientale</span>, Novara 28100, <span class="country" tagx="country" title="country">Italy</span></div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-11-11</span></span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">96</span><span class="lpage" tagx="lpage" title="lpage">108</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-08</span><span class="rev-recd" title="rev-recd">rev-recd: 2019-10-21</span><span class="accepted" title="accepted">accepted: 2019-10-21</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="IENZ_35_1684911.pdf" title="self-uri">IENZ_35_1684911.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>A series of analogues of Amb639752, a novel diacylglycerol kinase (DGK) inhibitor recently discovered by us via virtual screening, have been tested. The compounds were evaluated as DGK inhibitors on α, θ, and ζ isoforms, and as antagonists on serotonin receptors. From these assays emerged two novel compounds, namely <b>11</b> and <b>20</b>, which with an IC<sub>50</sub> respectively of 1.6 and 1.8 µM are the most potent inhibitors of DGKα discovered to date. Both compounds demonstrated the ability to restore apoptosis in a cellular model of X-linked lymphoproliferative disease as well as the capacity to reduce the migration of cancer cells, suggesting their potential utility in preventing metastasis. Finally, relying on experimental biological data, molecular modelling studies allow us to set a three-point pharmacophore model for DGK inhibitors.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Diacylglycerol kinase</div> <div class="kwd" title="kwd">
kinase inhibitors</div> <div class="kwd" title="kwd">
structure–activity relationship</div> <div class="kwd" title="kwd">
enzyme assays</div> <div class="kwd" title="kwd">
molecular modelling</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Università del Piemonte Orientale</span><span class="named-content" title="named-content">10.13039/501100005699</span></div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Telethon Foundation</span></div><span class="award-id" tagx="award-id" title="award-id">GGP16252</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">National Ministry of University and Research PRIN 2017</span></div><span class="award-id" tagx="award-id" title="award-id">201799WCRH</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Consorzio Interuniversitario di Biotecnologie (CIB) bando "Network-CIB: Catalisi dell’Innovazione nelle biotecnologie"</span></div> </div> <div class="funding-statement" title="funding-statement">
This work was supported by Università del Piemonte Orientale, Telethon Foundation [Grant GGP16252 to AG and GB], National Ministry of University and Research PRIN 2017 [Grant 201799WCRH to GB], Consorzio Interuniversitario di Biotecnologie (CIB) bando "Network-CIB: Catalisi dell’Innovazione nelle biotecnologie" to GB.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>Diacylglycerol kinases (DGKs) are a large family of enzymes that share a common catalytic activity: the phosphorylation of diacylglycerol (DAG) to phosphatidic acid (PA). Remarkably, both the substrate (DAG) and the product (PA) of the DGK-catalysed reaction, are bioactive lipids that can act as second messengers<a href="#CIT0001"><sup>1</sup></a>. DGK activity consequently serves as a switch to simultaneously dampen DAG-mediated signals and boost PA-mediated signals<a href="#CIT0002"><sup>2</sup></a>. Ten mammalian DGK isoforms (α, β, γ, δ, ε, ζ, η, θ, ι, and к) have been identified and divided into five groups (type I–V) according to their structural features<a href="#CIT0003"><sup>3</sup></a><sup>,</sup><a href="#CIT0004"><sup>4</sup></a>. The expression of these isoforms varies depending on the cell type. Among the 10 isoforms, the α isoform is among the most studied and characterised. This kinase is highly expressed in the brain, spleen, and thymus and, along with θ isoform, in the bone marrow. This enzyme is also highly expressed in T-lymphocytes, where it acts together with DGKζ as negative regulator of the T-cell receptor (TCR) response, and a mediator of IL-2 mediated proliferation<a href="#CIT0003"><sup>3</sup></a><sup>,</sup><a href="#CIT0005"><sup>5</sup></a>. The biological relevance of DGKα is best demonstrated in patients with X-linked lymphoproliferative disease (XLP-1), who experience life-threatening, uncontrolled accumulation of CD8<sup>+</sup> T cells in response to the Epstein–Barr virus (EBV) infection<a href="#CIT0006"><sup>6</sup></a>. In those patients, germline mutations of the adaptor protein SAP (SH2D1A) perturb TCR signalling and render DGKα constitutively active<a href="#CIT0007"><sup>7</sup></a>. Deregulated DGKα activity renders patient-derived lymphocytes resistant to reactivation-induced cell death (RICD). Thus, antigen-activated lymphocytes accumulate in lymphonodes and liver, resulting in severe immunopathology<a href="#CIT0008"><sup>8</sup></a>. Importantly, DGKα inhibitors restore RICD sensitivity <i>in vitro</i> and <i>in vivo</i>, thus avoiding immunopathology and suggesting a putative therapeutic use of those molecules in XLP-1<a href="#CIT0009"><sup>9</sup></a>.</p> <p>Apart from T-cell regulation, DGKα also plays a role in cancer, mediating numerous aspects of cancer cell progression including survival<a href="#CIT0010"><sup>10</sup></a><sup>,</sup><a href="#CIT0011"><sup>11</sup></a>, migration and invasion of cancer cells<a href="#CIT0012"><sup>12–14</sup></a>. In particular, it has been reported that DGKα is over expressed in hepatocellular carcinoma<a href="#CIT0015"><sup>15</sup></a>, and melanoma cells<a href="#CIT0011"><sup>11</sup></a> while other reports suggested that the growth of colon and breast cancer cell lines was significantly inhibited by DGKα-siRNA<a href="#CIT0016"><sup>16</sup></a> and DGKα/atypical PKC/β1 integrin signalling pathway was crucial for matrix invasion of breast carcinoma cells<a href="#CIT0017"><sup>17</sup></a>. In addition, expression is also higher in lymphonodal metastasis than in breast and gastric original tumour<a href="#CIT0018"><sup>18</sup></a><sup>,</sup><a href="#CIT0019"><sup>19</sup></a>. Finally, knock down of DGKα impairs glioblastoma tumorigenesis<a href="#CIT0020"><sup>20</sup></a><sup>,</sup><a href="#CIT0021"><sup>21</sup></a>.</p> <p>For all these reasons, the identification of strong and selective DGKα inhibitors, it is an important field of research. To date, only a handful of two-digit micromolar inhibitors of DGKα have been identified, but only three were the most characterised, namely, R59949, R59022, and ritanserin (<a href="#F0001">Figure 1)</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Three of the most studied DGKα inhibitors.</p>   </div> <p>In our assay system, R59949 and R59022 have an IC<sub>50</sub> of 11 and 20 µM, respectively<a href="#CIT0009"><sup>9</sup></a>. Their efficacy has been evaluated <i>in vivo</i> studies on mice, and is limited by their rapid clearance (<i>t</i><sub>1/2</sub>=∼2 h)<a href="#CIT0022"><sup>22</sup></a>. Furthermore, these two inhibitors are also able to target different isoforms of DGK, in particular R59022 acts on type III and V (ε e θ), while R59949 on type I and II (γ, δ e κ)<a href="#CIT0023"><sup>23</sup></a><sup>,</sup><a href="#CIT0024"><sup>24</sup></a> and a study conducted by Boroda et al. recently demonstrated their strong antagonistic activity on 5-HT<sub>2</sub> receptors (R59022 IC<sub>50</sub> 5HT<sub>2A</sub>=2.2 nM; R5994 IC<sub>50</sub> 5HT<sub>2A</sub>=9.2 nM)<a href="#CIT0025"><sup>25</sup></a>.</p> <p>A search on ChEMBL database<a href="#CIT0026"><sup>26</sup></a> (<a href="https://www.ebi.ac.uk/chembl/">https://www.ebi.ac.uk/chembl/</a>) show how these two molecules have activity at the same range of concentration with other biological targets, behaving like a sort of promiscuous ligands. Ritanserin, a well-known serotoninergic antagonist, is structurally similar to R59022, differing for an H-F isosteric substitution on a phenyl ring. Despite this small modification, Boroda et al. showed that ritanserin was a DGKα inhibitor (IC<sub>50</sub>=15 µM) more potent than R59022 and R59949 and with a better pharmacokinetic profile (<i>t</i><sub>1/2</sub>=40 h in human)<a href="#CIT0025"><sup>25</sup></a>. However, the comparison of ritanserin IC<sub>50</sub> as serotonin antagonist and as DGKα inhibitor, 0.9 nM and 15,000 nM, respectively, reveal that ritanserin is much a powerful serotonin antagonist than a DGK inhibitor. In addition, ritanserin is also a potent inhibitor on dopaminergic receptors with an IC<sub>50</sub> of 69 nM<a href="#CIT0027"><sup>27</sup></a>.</p> <p>Due to these drawbacks, at the beginning, in order to eliminate the strong serotoninergic activity of R59949, we reasoned to replace its protonable nitrogen atom, which at physiologically pH mimics the amino group of serotonin, with a carbon atom. We decided therefore to synthesise compound <b>1</b> (<a href="#F0002">Figure 2</a>) (see supporting information for its synthesis and a complete characterisation) and to test it as DGKα inhibitor.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Structures of the deaza analogue of R59949 and Amb639752.</p>   </div> <p>Interestingly, the compound was totally devoid of inhibitory activity on the enzyme, showing the importance of the basic nitrogen atom not only for the anti-serotoninergic activity, but also for the interaction with the kinase. With this in mind, we recently used an in-silico approach based on chemical homology with the two commercially available DGKα inhibitors R59022 and R59949 using the programmes ROCS<a href="#CIT0028"><sup>28</sup></a> and EON<a href="#CIT0029"><sup>29</sup></a>. From this study, we identified a compound, Amb639752 (<a href="#F0002">Figure 2</a>), featuring a lower IC<sub>50</sub> for DGKα than ritanserin (IC<sub>50</sub>=17 µM), a better selectivity for the α-isoform and devoid of anti-serotoninergic activity. Along with CU-3, which features an IC<sub>50</sub> of 0.6 µM on DGKα<a href="#CIT0030"><sup>30</sup></a> but contains a reactive Michael acceptor<a href="#CIT0031"><sup>31</sup></a>, Amb639752 is the most effective pharmacological tool available to study DGKα<a href="#CIT0009"><sup>9</sup></a>. In this manuscript, we report the structure–activity studies on Amb639752 and, in combination with data on ritanserin, the generation of a pharmacophore model for this class of compounds, which could be useful for the identification of other potential DGKα inhibitors.</p> </div> <div class="methods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Methods</div> <div class="chemistryprocedures" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Chemistry procedures</div> <p>Commercially available reagents and solvents were used without further purification. Toluene were distilled immediately before use from Na/benzophenone under a slight positive atmosphere of dry nitrogen. Dichloromethane was dried by distillation from P<sub>2</sub>O<sub>5</sub> and stored on activated molecular sieves (4 Å). When needed the reactions were performed in flame- or oven-dried glassware under a positive pressure of dry nitrogen. Melting points were determined in open glass capillary with a Stuart scientific SMP3 apparatus and are uncorrected. All compounds were checked by IR (FT-IR THERMO-NICOLET AVATAR), <sup>1</sup>H and 13C APT (JEOL ECP 300 MHz spectrometer), and mass spectrometry (Thermo Finningan LCQ-deca XP-plus, San Jose, CA) equipped with an ESI source and an ion trap detector. Chemical shifts are reported in parts per million (ppm). Flash column chromatography was performed on silica gel (Merck Kieselgel 60, 230–400 mesh ASTM, Kenilworth, NJ). Thin layer chromatography (TLC) was carried out on 5 × 20 cm plates with a layer thickness of 0.25 mm (Merck Silica gel 60 F<sub>254</sub>, Kenilworth, NJ). When necessary they were developed with KMnO<sub>4</sub> reagent. Purity of tested compounds was established by elemental analysis. Elemental analysis (C, H, N) of the target compounds is within ±0.4% of the calculated values, confirming ≥95% purity.</p> <div class="preparationof2-chloro-1-(2,6-dimethyl-1h-indol-3-yl)ethan-1-one(5)" title="sec"><span class="label" tagx="label" title="label">2.1.1.</span><div class="title" tagx="title" title="title">
Preparation of 2-chloro-1-(2,6-dimethyl-1H-indol-3-yl)ethan-1-one (5)</div> <p>In a Schlenk tube, under nitrogen, 2,6-dimethyl-1<i>H</i>-indole (<b>3</b>) (0.20 g, 1.38 mmol, 1 eq) was dissolved in 4 mL of dichloroethane dry and 0.25 mL of DBU (1.66 mmol, 1.2 eq) were added. The resulting solution was heated at 90 °C. When reached this temperature, chloroacetyl chloride (<b>4</b>) (0.12 mL, 1.52 mmol, 1.1 eq) was added. The reaction was stirred for 30 min, then solvent was evaporated and the crude purified by column chromatography using PE/EtOAc 7:3 and PE/EtOAc 5:5 as eluants to give 270 mg of product as violet solid: yield 90%; m.p. 243.7–244.2 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.88 (br s, NH), 7.85 (d, <i>J</i> = 7.9 Hz, 1H), 7.18 (s, 1H), 6.99 (d, <i>J</i> = 8.0 Hz, 1H), 4.89 (s, 2H), 2.68 (s, 3H), 2.39 (s, 3H). MS (ESI) <i>m/z</i>: 222 [M + H]<sup>+</sup>.</p> </div> <div class="preparationoftert-butyl4-(2-(2,6-dimethyl-1h-indol-3-yl)-2-oxoethyl)piperazine-1-carboxylate(6)" title="sec"><span class="label" tagx="label" title="label">2.1.2.</span><div class="title" tagx="title" title="title">
Preparation of tert-butyl 4-(2-(2,6-dimethyl-1H-indol-3-yl)-2-oxoethyl)piperazine-1-carboxylate (6)</div> <p>Under nitrogen, 200 mg of <b>5</b> (0.90 mmol, 1 eq) was dissolved in toluene dry, then <i>N</i>-Boc-piperazine (0.17 g, 0.90 mmol, 1 eq), K<sub>2</sub>CO<sub>3</sub> (0.32 g, 2.25 mmol, 2.5 eq), and KI (0.015 g, 0.09 mmol, 0.1 eq) were added. The reaction was heated at 80 °C overnight. Solvent was evaporated and the crude was purified by column chromatography using PE/EtOAc 4:6 and PE/EtOAc 2:8 as eluants to give 215 mg of product as yellow amorphous solid: yield 63%; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) <i>δ</i> 8.52 (br s, NH), 7.77 (d, <i>J</i> = 8.2 Hz, 1H), 7.12 (s, 1H), 7.08 (d, <i>J</i> = 8.0 Hz, 1H), 3.79 (s, 2H), 3.47–3.43 (m, 6H), 2.75 (s, 3H), 2.64 (br s, 2H), 2.44 (s, 3H), 1.47 (s, 9H). IR (KBr): 3190, 2964, 1698, 1413, 1417, 1364, 1126, 806 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>. MS (ESI) <i>m/z</i>: 372 [M + H]<sup>+</sup>.</p> </div> <div class="preparationof1-(2,6-dimethyl-1h-indol-3-yl)-2-(piperazin-1-yl)ethan-1-one(7)" title="sec"><span class="label" tagx="label" title="label">2.1.3.</span><div class="title" tagx="title" title="title">
Preparation of 1-(2,6-dimethyl-1H-indol-3-yl)-2-(piperazin-1-yl)ethan-1-one (7)</div> <p>Two hundred and fifteen milligrams of <b>6</b> (0.58 mmol, 1 eq) were dissolved in dichloromethane dry. The resulting solution was cooled at 0 °C and 0.69 mL of trifluoroacetic acid (9.28 mmol, 16 eq) was added. After 3 h, the reaction was worked up adding NaOH 2 M solution until pH = 12. Then, NaCl solid was added and the solution was extracted with THF (x2). The combined organic phases were dried on sodium sulphate. After evaporation of the solvent, the crude was purified by column chromatography using EtOAc/MeOH 9:1 and MeCN/NH<sub>3</sub> 9:1 as eluants to give 103 mg of the product as yellowish solid: yield: 65%; m.p.: 232.9–233.6 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.79 (br s, NH), 7.85 (d, <i>J</i> = 7.9 Hz, 1H), 7.15 (s, 1H), 6.95 (d, <i>J</i> = 8.2 Hz, 1H), 3.83(br s, 4H), 3.62 (s, 2H), 2.86 (br s, 4H), 2.67 (s, 3H), 2.38 (s, 3H); IR (KBr): 3446, 3181, 2813, 1634, 1455, 1330, 821 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 272 [M + H]<sup>+</sup>.</p> </div> <div class="generalprocedureforthesynthesisoffinalcompounds8,11–22" title="sec"><span class="label" tagx="label" title="label">2.1.4.</span><div class="title" tagx="title" title="title">
General procedure for the synthesis of final compounds 8, 11–22</div> <p>1-(2,6-Dimethyl-1<i>H</i>-indol-3-yl)-2-(piperazin-1-yl)ethan-1-one (<b>7</b>) (1 eq) was dissolved in dichloromethane dry. To the resulting solution EDCI (1 eq), TEA (2 eq), DMAP (0.1 eq) and the appropriate carboxylic acid (1 eq) were sequentially added. The reaction was stirred under nitrogen at room temperature overnight. Evaporation of the solvent gave a crude which was directly purified by column chromatography.</p> </div> <div class="2-(4-benzoylpiperazin-1-yl)-1-(2,6-dimethyl-1h-indol-3-yl)ethan-1-one(8)" title="sec"><span class="label" tagx="label" title="label">2.1.5.</span><div class="title" tagx="title" title="title">
2-(4-Benzoylpiperazin-1-yl)-1-(2,6-dimethyl-1H-indol-3-yl)ethan-1-one (8)</div> <p>Yellow solid; yield 29%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 213.8–214.3 °C. <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.72 (br s, NH), 7.85 (d, <i>J</i> = 8.2 Hz, 1H), 7.46–7.40 (m, 5H), 7.15 (s, 1H), 6.96 (d, <i>J</i> = 7.9 Hz, 1H), 3.67 (br s, 4H), 3.18 (br s, 2H) 2.67 (s, 3H), 2.61–2.56 (m, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 192.8, 169.5, 144.7, 136.5, 135.7, 131.5, 130.0, 129.0, 127.5, 125.1, 123.4, 121.2, 112.9, 111.6, 66.9, 53.2, 53.1, 21.7, 15.7; IR (KBr) 3189, 2990, 2828, 1609, 1446, 1282, 807 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>. MS (ESI) <i>m/z</i>: 374 [M–H]<sup>+</sup>. Anal. Calcd. for C<sub>23</sub>H<sub>25</sub>N<sub>3</sub>O<sub>2</sub>: C, 73.57; H, 6.71; N, 11.19; found C, 73.76; H, 6.94; N, 10.85.</p> </div> <div class="2-(4-(4-chlorobenzoyl)piperazin-1-yl)-1-(2,6-dimethyl-1h-indol-3-yl)ethan-1-one(11)" title="sec"><span class="label" tagx="label" title="label">2.1.6.</span><div class="title" tagx="title" title="title">
2-(4-(4-Chlorobenzoyl)piperazin-1-yl)-1-(2,6-dimethyl-1H-indol-3-yl)ethan-1-one (11)</div> <p>Yellow solid; yield 33%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 240.7–241.3 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.70 (br s, NH), 7.85 (d, <i>J</i> = 7.9 Hz, 1H), 7.50 (br d, AA’XX’, 2H), 7.42 (br d, AA’XX’, 2H), 7.14 (s, 1H), 6.95 (d, <i>J</i> = 8.2 Hz, 1H), 3.67 (br s, 4H), 3.33 (br s), 2.67 (s, 3H), 2.61–2.56 (m, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 193.17, 168.4, 144.8, 135.7, 135.2, 134.8, 131.5, 129.5, 129.1, 125.1, 123.5, 121.2, 112.7, 111.6, 53.14, 53.14, 52.9, 21.7, 15.7. IR (KBr): 3225, 2793, 2358, 1609, 1442, 1261, 864 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>. MS (ESI) <i>m/z</i>: 410 [M + H]<sup>+</sup>. Anal. Calcd. for C<sub>23</sub>H<sub>24</sub> ClN<sub>3</sub>O<sub>2</sub>: C, 67.39; H, 5.90; N, 10.25; found C, 67.11; H, 6.12; N, 10.54.</p> </div> <div class="1-(2,6-dimethyl-1h-indol-3-yl)-2-(4-(4-methoxybenzoyl)piperazin-1-yl)ethan-1-one(12)" title="sec"><span class="label" tagx="label" title="label">2.1.7.</span><div class="title" tagx="title" title="title">
1-(2,6-Dimethyl-1H-indol-3-yl)-2-(4-(4-methoxybenzoyl)piperazin-1-yl)ethan-1-one (12)</div> <p>Yellow solid; yield 53%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 219.9–220.8 °C.; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.70 (br s, NH), 7.85 (d, <i>J</i> = 8.2 Hz, 1H), 7.38 (br d, AA’XX’, 2H), 7.14 (s, 1H), 6.99–6.94 (m, 3H), 3.79 (br d, 3H), 3.67 (br s, 2H), 3.51 (br s, 4H) 2.67 (s, 3H), 2.58 (br s, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 192.8, 169.5, 144.6, 136.5, 135.7, 131.5, 130.0, 129.0, 127.5, 125.1, 123.4, 121.2, 112.8, 111.6, 66.9, 55.8, 53.2, 53.1, 21.7, 15.7; IR (KBr): 3235, 3003, 2807, 1613, 1463, 1253, 977 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 406 [M + H]<sup>+</sup>. Anal. Calcd. for C<sub>24</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>: C, 71.09; H, 6.71; N, 10.36; found C, 71.10; H, 6.75; N, 10.32.</p> </div> <div class="4-(4-(2-(2,6-dimethyl-1h-indol-3-yl)-2-oxoethyl)piperazine-1-carbonyl)benzonitrile(13)" title="sec"><span class="label" tagx="label" title="label">2.1.8.</span><div class="title" tagx="title" title="title">
4-(4-(2-(2,6-Dimethyl-1H-indol-3-yl)-2-oxoethyl)piperazine-1-carbonyl)benzonitrile (13)</div> <p>Yellow solid; yield 23%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 243.9–244.8 °C. <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.70 (br s, NH), 7.92–7.84 (m, 3H), 7.58 (br d, AA’XX’, 2H), 7.15 (s, 1H), 6.96 (br d, 1H), 3.68 (br s, 4H), 3.29 (br s, 2H), 2.67 (s, 3H), 2.51 (br s, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 192.5, 166.3, 143.2, 139.4, 134.1, 131.6, 129.9, 126.8, 123.5, 121.9, 119.6, 117.4, 111.2, 111.1, 110.1, 65.1, 51.7, 51.2, 20.1, 14.1; IR (KBr): 3410, 3254, 2816, 2790, 2233, 1609, 1454, 1291, 979 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 401[M + H]<sup>+</sup>. Anal. Calcd. for C<sub>24</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub>: C, 71.98; H, 6.04; N, 13.99; found C, 72.13; H, 6.23; N, 14.08.</p> </div> <div class="1-(2,6-dimethyl-1h-indol-3-yl)-2-(4-(thiophene-2-carbonyl)piperazin-1-yl)ethan-1-one(14)" title="sec"><span class="label" tagx="label" title="label">2.1.9.</span><div class="title" tagx="title" title="title">
1-(2,6-Dimethyl-1H-indol-3-yl)-2-(4-(thiophene-2-carbonyl)piperazin-1-yl)ethan-1-one (14)</div> <p>Yellow solid; yield 29%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 200.3–201.2 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.74 (br s, NH), 7.86 (d, <i>J</i> = 8.2 Hz, 1H), 7.76 (br d, 1H), 7.41–7.40 (m, 1H), 7.15–7.10 (m, 2H), 6.96 (d, <i>J</i> = 8.2 Hz, 1H), 3.75–3.67 (m, 4H), 3.37 (br s, 2H) 2.68 (s, 3H), 2.62 (s, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 192.8, 167.8, 144.7, 137.8, 135.7, 131.5, 130.0, 129.6, 127.7, 125.1, 124.1, 123.4, 121.2, 112.8, 111.6, 66.7, 53.3, 21.7, 15.7; IR (KBr): 3270, 2927, 2793, 1642, 1454, 1261, 809 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 382[M + H]<sup>+</sup>; Anal. Calcd. for C<sub>21</sub>H<sub>23</sub>N<sub>3</sub>O<sub>2</sub>S: C, 66.12; H, 6.08; N, 11.01; found C, 66.23; H, 6.26; N, 10.93.</p> </div> <div class="1-(2,6-dimethyl-1h-indol-3-yl)-2-(4-nicotinoylpiperazin-1-yl)ethan-1-one(15)" title="sec"><span class="label" tagx="label" title="label">2.1.10.</span><div class="title" tagx="title" title="title">
1-(2,6-Dimethyl-1H-indol-3-yl)-2-(4-nicotinoylpiperazin-1-yl)ethan-1-one (15)</div> <p>Yellow solid; yield 39%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 216.2–216.8 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.70 (br s, NH), 8.65–8.61 (m, 2H), 7.87–7.82 (m, 2H), 7.50–7.45 (m, 1H), 7.15 (s, 1H), 6.95 (br d, 1H), 3.68 (br s, 4H), 3.37 (br s, 2H), 2.67 (s, 3H), 2.56 (br s, 4H), 2.37 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 192.6, 167.2, 151.0, 148.1, 144.7, 135.7, 135.4, 132.3, 131.5, 125.1, 124.1, 123.5, 121.2, 116.2, 111.6, 66.7, 53.4, 52.9, 21.7, 15.7; IR (KBr): 3414, 3213, 2828, 1621, 1454, 1267, 1301, 817 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 377 [M + H]<sup>+</sup>; Anal. Calcd. for C<sub>22</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub>: C, 70.19; H, 6.43; N, 14.88; found C, 70.21; H, 6.44; N, 14.73.</p> </div> <div class="1-(2,6-dimethyl-1h-indol-3-yl)-2-(4-(2-methylbenzoyl)piperazin-1-yl)ethan-1-one(16)" title="sec"><span class="label" tagx="label" title="label">2.1.11.</span><div class="title" tagx="title" title="title">
1-(2,6-Dimethyl-1H-indol-3-yl)-2-(4-(2-methylbenzoyl)piperazin-1-yl)ethan-1-one (16)</div> <p>Yellow solid; yield 41%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 200.5–201.6 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.70 (br s, NH), 7.85 (d, <i>J</i> = 8.2 Hz, 1H), 7.31–7.27 (m, 3H), 7.26–7.24 (m, 2H), 7.15 (d, 1H), 3.69 (br s, 2H), 3.34 (br s, 2H), 3.15 (br s, 2H), 2.67 (s, 3H), 2.51 (br s, 4H), 2.44 (s, 3H), 2.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 192.6, 169.0, 144.7, 135.7, 134.2, 131.8, 131.5, 130.7, 129.2, 126.4, 126.2, 123.4, 121.2, 112.8, 66.8, 53.5, 53.0, 46.8, 41.4, 21.7, 19.2, 15.7; IR (KBr): 3431, 3221, 2919, 2797, 1615, 1454, 1257, 748 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 390 [M + H]<sup>+</sup>. Anal. Calcd. for C<sub>24</sub>H<sub>27</sub>N<sub>3</sub>O<sub>2</sub>: C, 74.01; H, 6.99; N, 10.79; found C, 74.01; H, 7.01; N, 10.63.</p> </div> <div class="1-(2,6-dimethyl-1h-indol-3-yl)-2-(4-(3-methoxybenzoyl)piperazin-1-yl)ethan-1-one(17)" title="sec"><span class="label" tagx="label" title="label">2.1.12.</span><div class="title" tagx="title" title="title">
1-(2,6-Dimethyl-1H-indol-3-yl)-2-(4-(3-methoxybenzoyl)piperazin-1-yl)ethan-1-one (17)</div> <p>Yellow solid; yield 32%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 209.8–210.4 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.71 (br s, NH), 7.85 (d, <i>J</i> = 7.9 Hz, 1H), 7.34 (t, 1H), 7.15 (br s, 1H), 7.02–6.92 (m, 4H), 3.78 (s, 3H), 3.68 (br s, 3H), 3.35 (br s, 3H), 2.67 (br s, 7H), 2.38 (s, 3H); <sup>13</sup>C NMR (75 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 192.7, 169.1, 159.7, 144.7, 137.9, 135.7, 131.5, 130.2, 125.1, 123.4, 121.2, 119.4, 115.7, 112.7, 111.6, 66.8, 55.8, 53.3, 53.11, 47.4, 21.7, 15.7; IR (KBr): 3131, 3049, 2944, 1651, 1455, 1292, 1130, 968 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 406 [M + H]<sup>+</sup>. Anal. Calcd. for C<sub>24</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>: C, 71.09; H, 6.71; N, 10.36; found C, 70.85; H, 6.45; N, 10.76.</p> </div> <div class="2-(4-(3,4-difluorobenzoyl)piperazin-1-yl)-1-(2,6-dimethyl-1h-indol-3-yl)ethan-1-one(18)" title="sec"><span class="label" tagx="label" title="label">2.1.13.</span><div class="title" tagx="title" title="title">
2-(4-(3,4-Difluorobenzoyl)piperazin-1-yl)-1-(2,6-dimethyl-1H-indol-3-yl)ethan-1-one (18)</div> <p>Yellow solid; yield 27%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 233.8–235.0 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.71 (br s, NH), 7.85 (d, <i>J</i> = 7.9 Hz, 1H), 7.56–7.47 (m, 2H), 7.28 (br s, 1H), 7.15 (s, 1H), 6.96 (d, <i>J</i> = 7.9 Hz, 1H), 3.68 (br s, 2H), 3.33 (br s, 2H), 2.67 (s, 3H), 2.51 (br s, 6H), 2.38 (s, 3H); MS (ESI) <i>m/z</i>: 412 [M + H]<sup>+</sup>; IR (KBr): 3252, 2919, 2795, 1618, 1469, 1286, 1046, 980 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) 412 [M + H]<sup>+</sup>. Anal. Calcd. for C<sub>23</sub>H<sub>23</sub>F<sub>2</sub>N<sub>3</sub>O<sub>2</sub>: C, 67.14; H, 5.63; N, 10.21; found C, 67.43; H, 5.79; N, 10.59.</p> </div> <div class="2-(4-(3-chlorobenzoyl)piperazin-1-yl)-1-(2,6-dimethyl-1h-indol-3-yl)ethan-1-one(19)" title="sec"><span class="label" tagx="label" title="label">2.1.14.</span><div class="title" tagx="title" title="title">
2-(4-(3-Chlorobenzoyl)piperazin-1-yl)-1-(2,6-dimethyl-1H-indol-3-yl)ethan-1-one (19)</div> <p>Yellow solid; yield 32%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 222.3–223.5 °C; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) <i>δ</i> 9.42 (br s, NH), 7.71 (d, <i>J</i> = 7.9 Hz, 1H), 7.40–7.25 (m, 4H), 7.04 (d, <i>J</i> = 8.2 Hz, 2H), 3.85 (br s, 4H), 3.47 (br s, 2H), 2.80 (br s, 1H), 2.68 (br s, 6H), 2.40 (s, 3H); <sup>13</sup>C NMR (75 MHz CDCl<sub>3</sub>,) <i>δ</i> 192.5, 169.0, 144.7, 137.5, 135.3, 134.7, 132.3, 130.0, 127.3, 125.2, 124.0, 123.7, 120.6, 112.8, 111.4, 67.0, 53.7, 53.7, 29.6, 21.5, 15.7; IR (KBr): 3264, 2916, 2795, 1646, 1454, 1256, 978, 809 <i>ν</i><sub>max</sub>/cm<sup>–1</sup>; MS (ESI) <i>m/z</i>: 410 [M + H]<sup>+</sup>. Anal. Calcd. for C<sub>23</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>2</sub>: C, 67.39; H, 5.90; N, 10.25; found C, 67.38; H, 5.90; N, 10.24.</p> </div> <div class="1-(2,6-dimethyl-1h-indol-3-yl)-2-(4-(4-methylbenzoyl)piperazin-1-yl)ethan-1-one(20)" title="sec"><span class="label" tagx="label" title="label">2.1.15.</span><div class="title" tagx="title" title="title">
1-(2,6-Dimethyl-1H-indol-3-yl)-2-(4-(4-methylbenzoyl)piperazin-1-yl)ethan-1-one (20)</div> <p>Yellow solid; yield 48%; column eluants: EtOAc, EtOAc/MeOH 9:1; m.p.: 230.9–231.2 °C; <sup>1</sup>H NMR (300 MHz, DMSO-d<sub>6</sub>) <i>δ</i> 11.71 (br s, NH), 7.86–7.82 (m, 1H), 7.31–7.23 (m, 4H), 7.15 (s, 1H), 6.96 (d, <i>J</i> = 7.9 Hz, 1H), 3.69 (br s, 2H), 3.35 (br s, 4H), 2.67 (s, 3H), 2.51. IR (KBr): 3228, 2915, 2792, 1607, 1454, 1260, 979 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 390 [M + H]<sup>+</sup>; Anal. Calcd. for C<sub>24</sub>H<sub>27</sub>N<sub>3</sub>O<sub>2</sub>: C, 74.01; H, 6.99; N, 10.79; found C, 74.12; H, 7.02; N, 10.79.</p> </div> <div class="2-(4-(cyclopentanecarbonyl)piperazin-1-yl)-1-(2,6-dimethyl-1h-indol-3-yl)ethan-1-one(21)" title="sec"><span class="label" tagx="label" title="label">2.1.16.</span><div class="title" tagx="title" title="title">
2-(4-(Cyclopentanecarbonyl)piperazin-1-yl)-1-(2,6-dimethyl-1H-indol-3-yl)ethan-1-one (21)</div> <p>Brown oil; yield 61%; column eluants: EtOAc, EtOAc/MeOH 9:1; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) <i>δ</i> 9.00 (br s, NH), 7.73 (d, <i>J</i> = 8.2 Hz, 1H), 7.26 (br d, 1H), 7.12 (s, 1H), 7.05 (d, <i>J</i> = 8.2 Hz, 1H), 3.83 (s, 2H), 3.75 (br s, 2H), 3.63 (br s, 2H), 2.80–2.71 (m, 7H), 2.45 (s, 3H), 1.92–1.46 (m, 7H); IR (KBr): 3253, 2944, 2862, 1650, 1620, 1455, 1234, 957 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 368 [M + H]<sup>+</sup>; Anal. Calcd. for C<sub>22</sub>H<sub>29</sub>N<sub>3</sub>O<sub>2</sub>: C, 71.90; H, 7.95; N, 11.43; found C, 72.23; H, 8.31; N, 11.32.</p> </div> <div class="1-(4-(2-(2,6-dimethyl-1h-indol-3-yl)-2-oxoethyl)piperazin-1-yl)heptan-1-one(22)" title="sec"><span class="label" tagx="label" title="label">2.1.17.</span><div class="title" tagx="title" title="title">
1-(4-(2-(2,6-Dimethyl-1H-indol-3-yl)-2-oxoethyl)piperazin-1-yl)heptan-1-one (22)</div> <p>Brown oil; yield 76%; column eluants: EtOAc, EtOAc/MeOH 9:1; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) <i>δ</i> 10.25 (br s, NH), 7.68 (d, <i>J</i> = 8.2 Hz, 1H), 7.08 (s, 1H), 7.00 (d, <i>J</i> = 7.9 Hz, 1H), 3.83 (s, 2H), 3.73 (br s, 2H), 3.56 (br s, 2H), 2.77–2.71 (m, 3H), 2.62 (s, 3H), 2.38 (s, 3H), 2.32–2.27 (m, 5H), 1.59–1.54 (m, 2H), 1.26 (br s, 4H), 0.85 (br t, 3H); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) <i>δ</i> 191.7, 172.4, 145.4, 135.5, 132.1, 124.0, 123.6, 120.4, 112.4, 111.6, 66.2, 53.3, 53.1, 45.3, 41.2, 34.5, 31.6, 28.9, 25.0, 22.5, 21.5, 15.6, 14.1; IR (KBr): 2927, 2857, 1731, 1645, 1455, 1434, 1234, 668 <i>ν</i><sub>max</sub>/cm<sup>−1</sup>; MS (ESI) <i>m/z</i>: 384 [M + H]<sup>+</sup>; Anal. Calcd. for C<sub>23</sub>H<sub>33</sub>N<sub>3</sub>O<sub>2</sub>: C, 72.03; H, 8.67; N, 10.96; found C, 72.03; H, 8.73; N, 11.21.</p> </div> </div> <div class="celllines" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
Cell lines</div> <p>Madin-Darby canine kidney (MDCK) cells stably expressing One Strep Tag DGKα (OST-DGKα) were prepared by infecting MDCK cells with a vector expressing an inducible OST tagged DGKα constructs<a href="#CIT0017"><sup>17</sup></a>. MDCK cells infected with empty vector were used as controls. MDCK cells were cultured in MEM (minimal essential medium) with 5% FBS (foetal bovine serum) and 1% antibiotic–antimycotic solution. Routinely, cells were splitted every 3–4 days with trypsin–EDTA 0.25% in standard 100 mm dishes.</p> <p>Human embryonic kidney 293T cells (10 cm<sup>2</sup> plates) were cultured in RPMI with 10% FBS and 1% penicillin/streptomycin and cultures were maintained by splitting them for every 2–3 days using trypsin–EDTA 0.25%.</p> <p>Michigan Cancer Foundation 7 (MCF7) cells were cultured in DMEM with 10% FBS + 1% penicillin/streptomycin and cultures were maintained by splitting them for every 2–3 days using trypsin–EDTA 0.25%.</p> </div> <div class="primarycells" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Primary cells</div> <p>PBL were isolated from healthy anonymous human donors by Ficoll-Paque PLUS (GE Healthcare, Chicago, IL) density gradient centrifugation, washed, and resuspended at 2 × 10<sup>6</sup> cell/mL in RPMI-GlutaMAX containing 10% heat inactivated FCS, 2 mM glutamine, and 100 U/mL of penicillin and streptomycin. T cells were activated with 1 µg/mL anti-CD3 (UCHT1) and anti-CD28 (clone CD28.2) antibodies. After three days, activated T cells were washed and cultured in medium additionated of 100 IU/mL rhIL-2 (Peprotech, Rocky Hill, NJ) at 1.2 × 10<sup>6</sup> cells/mL for ≥7 days by changing media for every 2–3 days.</p> <p>Human monocytes were isolated from healthy anonymous human buffy coats (provided by the Transfusion Service of Ospedale Maggiore della Carità, Novara, Italy) by the standard technique of dextran sedimentation and Histopaque (density = 1.077 g cm<sup>3</sup>, Sigma-Aldrich, Milano, Italy) gradient centrifugation (400×<i>g</i>, 30 min, room temperature) and recovered by fine suction at the interface, as described previously<a href="#CIT0032"><sup>32</sup></a>. Purified monocytes populations were obtained by adhesion (90 min, 37 °C, 5% CO<sub>2</sub>) in serum-free RPMI 1640 medium (Sigma-Aldrich, Milano, Italy) supplemented with 2 mM glutamine and antibiotics. Cell viability (trypan blue dye exclusion) was usually &amp;gt;98%.</p> </div> <div class="preparationofdgkαenrichedhomogenates" title="sec"><span class="label" tagx="label" title="label">2.4.</span><div class="title" tagx="title" title="title">
Preparation of DGKα enriched homogenates</div> <p>Large cultures of MDCK cells for enzyme preparation were done by plating 5 × 10<sup>6</sup> cells in 245 mm<sup>2</sup> dishes. Once they reached nearly 70% confluence, cells were treated with doxycycline (1 µg/mL, two days). After two days of treatment, each plate was washed in cold PBS and cells homogenised in 5 mL of homogenate buffer (25 mM Hepes (pH 8), 20% glycerol, 135 mM NaCl, 5 mM ethylenediaminetetraacetic acid (EDTA), 1 mM ethylene glycol-bis(beta-aminoethyl ether)-N,N,N′,N′-tetraacetic acid (EGTA), 1 mM sodium orthovanadate, and protease inhibitor cocktail) for each dish. Cells were collected with a rubber scraper, homogenised by passing them through a 29 G-needle syringe 20 times and stored in aliquots at –80 °C. Presence of OST-DGKα was confirmed by western blotting and enzyme assay, transduced DGKα has an activity &amp;gt;100 folds the endogenous DGK.</p> </div> <div class="preparationofdgkζanddgkθenrichedhomogenates" title="sec"><span class="label" tagx="label" title="label">2.5.</span><div class="title" tagx="title" title="title">
Preparation of DGKζ and DGKθ enriched homogenates</div> <p>293T cells were transiently transfected with indicated DGK isoform plasmid DNA using Lipofectamine 3000, Invitrogen (Carlsbad, CA). Forty eight hours after transfection, cells were harvested and homogenised with a 29 G-needle using 500 µL of homogenate buffer for each dish and immediately stored in aliquots at –80 °C. Cells transfected with empty vector were used as controls, overexpressed DGK has an activity &amp;gt;50 folds the endogenous.</p> </div> <div class="dgkassay" title="sec"><span class="label" tagx="label" title="label">2.6.</span><div class="title" tagx="title" title="title">
DGK assay</div> <p>Essentionally, the same procedure was followed as reported previously in Velnati et al.<a href="#CIT0009"><sup>9</sup></a> In brief, DGK activity was assayed by measuring initial velocities (5 min at 27 °C) in presence of saturating substrate concentrations. Reaction conditions: 0.9 µg/µL 1,2-dioleoyl-sn-glycerol, 5 mM ATP, 0.01 µCi/µL [γ<sup>32</sup>P]-ATP, 1 mM sodium orthovanadate, 10 mM MgCl<sub>2</sub>, 1.2 mM EGTA in 7.5 mM Hepes pH 8<a href="#CIT0012"><sup>12</sup></a>. Reaction mixture is assembled mixing enzyme (24.5 µL of homogenate), 100× inhibitor or DMSO (0.5 µL), 5× ATP solution (10 µL of 25 mM ATP, 0.05 µCi/µL [γ<sup>32</sup>P]-ATP (Perkin-Elmer, Milan, Italy), 5 mM sodium orthovanadate, 50 mM MgCl<sub>2</sub>), and 3.3× DAG solution (15 µL of 3 µg/µL 1,2-dioleoyl-sn-glycerol resuspended by sonication in 4 mM EGTA in 25 mM Hepes pH 8). The reaction was stopped after 5 min by adding 200 µL of freshly prepared 1 M HCl and lipid was extracted by adding 200 µL of CH<sub>3</sub>OH:CHCl<sub>3</sub> 1:1 solution and vortexing for 1 min. The two phases were separated by centrifugation (12,000 RCF for 2 min). Twenty-five microlitres of the lower organic phase was spotted in small drops on silica TLC plates. TLC was run 10 cm and dried before radioactive signals were detected by GS-250 molecular imager and was quantified by quantity one (Bio-Rad, Hercules, CA) software assuring the absence of saturated spots.</p> <p>Percentage residual activity was calculated as follows: (OST-DGKα homogenate with inhibitor – vector homogenate)/(OST-DGKα homogenate with DMSO – vector homogenate)×100.</p> </div> <div class="superoxideanion(o2−)production" title="sec"><span class="label" tagx="label" title="label">2.7.</span><div class="title" tagx="title" title="title">
Superoxide anion (O<sup>2−</sup>) production</div> <p>All the experiments were performed in triplicate using cells isolated from each single donor.</p> <p>Monocytes (250,000 cells/well) were treated for 1 h with the indicated drugs (10 µM) with or without serotonin (1 µM). Then, cells were stimulated with phorbol 12-myristate 13-acetate (PMA; Sigma-Aldrich, Milano, Italy) 1 µM for 30 min. PMA is a well-known stimulus that induces a strong and significant respiratory burst via PKC activation<a href="#CIT0033"><sup>33</sup></a>. Superoxide anion production was then evaluated by the superoxide dismutase (SOD)-sensitive cytochrome C (CytC) reduction assay and expressed as nmoles CytC reduced/10<sup>6</sup> cells/30 min, using an extinction coefficient of 21.1 mM. To avoid interference with spectrophotometrical recordings, cells were incubated with RPMI 1640 without phenol red, antibiotics, and FBS.</p> </div> <div class="ricdassayinsapsilencedtcells" title="sec"><span class="label" tagx="label" title="label">2.8.</span><div class="title" tagx="title" title="title">
RICD assay in SAP silenced T cells</div> <p>Activated human PBLs were transfected with 200 pmol of siRNA oligonucleotides specific for the target protein (Stealth Select siRNA; Life Technologies, Carlsbad, CA) or a non-specific control oligo (Life Technologies, Carlsbad, CA). Transient transfections were performed using Amaxa nucleofector kits for human T cells (Lonza, Basel, Switzerland) and the Amaxa Nucleofector II or 4D systems (programmes T-20 or EI-115). Cells were cultured in IL-2 (100 IU/mL) for four days to allow target gene knockdown. Knockdown efficiency was periodically evaluated by Western blotting.</p> <p>Non-specific Stealth RNAi Negative Control Duplexes (12935-300, Life Technologies, Carlsbad, CA) were used as a negative control.</p> <p>siRNA SAP: sense strand UGUACUGCCUAUGUGUGCUGUAUCA, antisense strand UGAUACAGCAGACAUAGGCAGUACA.</p> <p>To test restimulation induced cell death, activated T cells (10<sup>5</sup> cells/well) were plated in triplicate in 96-well round-bottom plate and treated with anti-CD3 (clone OKT3) (10 ng/mL) in RPMI-GlutaMAX supplemented with 100 IU/mL rhIL-2 for 24 h. In these assays, inhibitors (10 µM) were added 30 min before the restimulation with OKT3. 24 h after treatment, cells were stained with 20 ng/mL propidium iodide and collected for a constant time of 30 s per sample on Attune Nxt Flow Cytometer (Thermo Fisher Scientific, Waltham, MA). Cell death is expressed as % cell loss and calculated as: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" position="float" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
%</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
cell</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
loss</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
number</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
of</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
viable</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
cells</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
in</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
sample</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
number</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
of</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
viable</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
cells</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
in</div> <div class="mi_UNKNOWN"> </div> <div class="mtext_UNKNOWN">
control</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
100</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> </div> <p /> <p>Results were expressed as mean ± standard error of the mean (SEM). We always compared controls and SAP silenced lymphocytes from the same donors as there is a large individual variability in RICD sensitivity.</p> </div> <div class="migrationassays" title="sec"><span class="label" tagx="label" title="label">2.9.</span><div class="title" tagx="title" title="title">
Migration assays</div> <p>Cell migration assays were performed using the Culture-Insert 2 well in µ-Dish (ibidi GmbH, Martinsried, Germany).</p> <p>Briefly, 25,000 MCF7 cells were plated in each well and cultured for 24 h. The day after, the culture insert was removed and the cells were washed with PBS before treating them with respective DGKα inhibitors (10 µM) or DMSO for 15 h in complete medium (DMEM 10% FBS + 1% penicillin/streptomycin), while medium without FBS was used as a negative control for migration.</p> <p>Phase-contrast pictures were taken immediately after treatment (0 h) and after 15 h under 5× magnification.</p> <p>Finally, wound areas were determined using ImageJ software (NIH, , Bethesda, MD). Wound reduction was calculated by using the following formula: (wound area at 15 h/wound at 0 h)×100, the values obtained were expressed as the percentage of wound area compared to the initial area.</p> </div> <div class="quantificationandstatisticalanalysis" title="sec"><span class="label" tagx="label" title="label">2.10.</span><div class="title" tagx="title" title="title">
Quantification and statistical analysis</div> <p>Data for the screen on OST-DGKα homogenates are the mean of duplicates. The compounds showing inhibitory activity in this assay were tested &amp;gt;4 times and the mean ± SEM is reported.</p> <p>To calculate IC<sub>50</sub> values of active inhibitors, the inhibitor activity was measured at least three times at 0.1, 1.0, 10.0, and 100.0 µM concentration. Data were analysed using [inhibitor] vs. normalised response parameters with least square [ordinary] curve fitting method in GraphPad PRISM 8.0 software (GraphPad Software, La Jolla, CA) mentioning 95% confidence interval and IC<sub>50</sub> values always greater than 0.0. Graph shows the mean ± SEM of inhibitor activity at the indicated concentration. In all the experiments, the data were normalised with the controls.</p> <p>Evaluation of <i>in vitro</i> assays across multiple treatments (RICD), SOD-sensitive CytC reduction assay, migration assays were analysed by using one-way ANOVA with multiple comparisons correction using GraphPad PRISM 8.0 software (GraphPad Software, La Jolla, CA). Error bars are described in figure legends as ± SEM or ± SD where appropriate. A single, double, triple and four asterisk denotes significance of a <i>p</i> value ≤0.05, ≤0.01, ≤0.001, and ≤0.0001 respectively in all experiments.</p> </div> <div class="pharmacophoricmodel" title="sec"><span class="label" tagx="label" title="label">2.11.</span><div class="title" tagx="title" title="title">
Pharmacophoric model</div> <p>A representative 3D structure of each compound was generated using OMEGA2 software<a href="#CIT0034"><sup>34–36</sup></a>. The generated file was used to generate a pharmacophore model with the Pharmagist web server (<a href="http://bioinfo3d.cs.tau.ac.il/PharmaGist/">bioinfo3d.cs.tau.ac.il/PharmaGist/</a>)<a href="#CIT0037"><sup>37</sup></a>.</p> </div> </div> <div class="results" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results</div> <div class="chemistry" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Chemistry</div> <p>At the beginning, we purchased 14 analogues of Amb639752 by vendors (<a href="#F0003">Figure 3</a>), while one analogue (<b>2</b>), being not commercially available, was synthesised (see <a href="https://doi.org/10.1080/14756366.2019.1684911">Supplementary material</a>). All the compounds were evaluated for their inhibitory activity on DGKα at a concentration of 100 µM (<a href="#t0001">Table 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>First set of compounds tested for their inhibitory activity on DGKα.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Inhibitory activity on DGKα (I).</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" width="1*" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" width="1*" /></div>  <thead> <tr> <th align="left">Compound</th> <th align="center">Residue activity at 100 µM</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">R59022</td> <td align="char" char=".">27</td> </tr> <tr> <td align="left">R59949</td> <td align="char" char=".">28</td> </tr> <tr> <td align="left">Amb639752</td> <td align="char" char=".">4</td> </tr> <tr> <td align="left">Amb758897</td> <td align="char" char=".">67</td> </tr> <tr> <td align="left">Amb37496</td> <td align="char" char=".">114</td> </tr> <tr> <td align="left">Amb626577</td> <td align="char" char=".">127</td> </tr> <tr> <td align="left">Amb1926062</td> <td align="char" char=".">88</td> </tr> <tr> <td align="left">Amb94966</td> <td align="char" char=".">196</td> </tr> <tr> <td align="left">Amb730692</td> <td align="char" char=".">45</td> </tr> <tr> <td align="left">Amb98433</td> <td align="char" char=".">81</td> </tr> <tr> <td align="left">Amb18718</td> <td align="char" char=".">135</td> </tr> <tr> <td align="left">Amb726054</td> <td align="char" char=".">126</td> </tr> <tr> <td align="left">Amb731111</td> <td align="char" char=".">130</td> </tr> <tr> <td align="left">Amb22018852</td> <td align="char" char=".">160</td> </tr> <tr> <td align="left">Amb687392</td> <td align="char" char=".">115</td> </tr> <tr> <td align="left">Amb758592</td> <td align="char" char=".">150</td> </tr> <tr> <td align="left">Amb629700</td> <td align="char" char=".">101</td> </tr> <tr> <td align="left"><b>2</b></td> <td align="char" char=".">135</td> </tr> </tbody> </table> </div> <p>Each inhibitor was tested in duplicate at least once, and DGKα activity was expressed as percentage of residual DGKα activity compared to DMSO control in the same assay. Assay uses OST-DGKα overexpressing cell lysates in presence of saturating exogenous DAG and ATP. We considered R59022 (commercially available) and the lead compound Amb63975230 as our reference molecules. As expected, our reference inhibitors R59022 and Amb639752 featured 73% and 96% inhibition respectively, confirming the quality of data obtained.</p> <p>This first screening showed us how Amb639752 exhibits a rigid structure activity relationship. Indeed, both the methyl groups on the 2,6 position of indole are mandatory, the NH indole cannot be alkylated as well as ramifications on the alkyl chain are detrimental. We then focussed our attention on furan ring knowing its intrinsic toxicity via metabolic activation<a href="#CIT0038"><sup>38</sup></a>. Unfortunately, there were no analogues available by vendors. Our first goal was to replace the furan moiety with the phenyl ring, investigating two different synthetic pathways.</p> <p>In the first one, the commercially available 2,6-dimethyl-1<i>H</i>-indole <b>3</b> was acylated with 2-chloroacetyl chloride <b>4</b>, in the presence of DBU in dichloroethane<a href="#CIT0039"><sup>39</sup></a> to give the derivative <b>5</b> 90% yield. Then, the acylated compound <b>5</b> was reacted with <i>N</i>-Boc piperazine in the presence of potassium carbonate and potassium iodide to afford the piperazinic derivative <b>6</b> in 63% yield. Boc deprotection with trifluoroacetic acid, followed by coupling with benzoic acid using the condensing agent EDCI afforded the final compound <b>8</b> (<a href="#SCH0001">Scheme 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Scheme 1.</span></a></div>   <p>The first synthetic route for the compound <b>8</b>.</p>   </div> <p>In the second synthetic strategy, we initially coupled the benzoic acid with <i>N</i>-Boc piperazine in the presence of EDCI to give piperazinic derivative <b>9</b> in 49% yield. Boc deprotection gave in quantitative yield the compound <b>10</b>. Due to its high aqueous solubility, solvent was evaporated and the crude as trifluoroacetate salt was directly used for the next step, where it was reacted with the acylated indole <b>5</b> to give the final compound <b>8</b> in 25% yield (<a href="#SCH0002">Scheme 2</a>) (see <a href="https://doi.org/10.1080/14756366.2019.1684911">Supplementary material</a> for full synthetic details).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Scheme 2.</span></a></div>   <p>The second synthetic route for the compound <b>8</b>.</p>   </div> <p>Overall yield calculation was 11% for both strategies, but with the first route it was possible to use a common synthetic intermediate <b>7</b> which can be coupled with different carboxylic acids. Furthermore, the second route requires more purification steps. For this reason, we applied the first route and coupled the advanced intermediate <b>7</b> with 12 different carboxylic acids (<a href="#F0004">Figure 4</a>) to afford compounds <b>11</b>–<b>22</b> (<a href="#F0005">Figure 5</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Carboxylic acids used.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Putative DGKα inhibitors synthesised. In brackets the yield of the coupling reaction with the common intermediate <b>7</b>.</p>   </div> <p>All those molecules were dissolved in DMSO and tested at a concentration of 100 µM for the ability to inhibit DGKα using equal amounts of DMSO as control. We identified eight compounds capable of reducing OST-DGKα activity similar or superior to R59022 (<a href="#t0002">Table 2</a>).</p> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Inhibitory activity on DGKα (II).</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" width="1*" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" width="1*" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" width="1*" /></div>  <thead> <tr> <th align="left">Compound</th> <th align="center">Residue activity at 100 µM</th> <th align="center">IC<sub>50</sub> (µM)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left"><b>8</b></td> <td align="char" char=".">5</td> <td align="char" char=".">3.2</td> </tr> <tr> <td align="left"><b>11</b></td> <td align="char" char=".">6</td> <td align="char" char=".">1.6</td> </tr> <tr> <td align="left"><b>12</b></td> <td align="char" char=".">6</td> <td align="char" char=".">3.6</td> </tr> <tr> <td align="left"><b>13</b></td> <td align="char" char=".">14</td> <td align="char" char=".">6.9</td> </tr> <tr> <td align="left"><b>14</b></td> <td align="char" char=".">6</td> <td align="char" char=".">3.0</td> </tr> <tr> <td align="left"><b>15</b></td> <td align="char" char=".">89</td> <td align="center">–</td> </tr> <tr> <td align="left"><b>16</b></td> <td align="char" char=".">27</td> <td align="char" char=".">32.8</td> </tr> <tr> <td align="left"><b>17</b></td> <td align="char" char=".">41</td> <td align="center">–</td> </tr> <tr> <td align="left"><b>18</b></td> <td align="char" char=".">47</td> <td align="center">–</td> </tr> <tr> <td align="left"><b>19</b></td> <td align="char" char=".">26</td> <td align="char" char=".">49.7</td> </tr> <tr> <td align="left"><b>20</b></td> <td align="char" char=".">3</td> <td align="char" char=".">1.8</td> </tr> <tr> <td align="left"><b>21</b></td> <td align="char" char=".">39</td> <td align="center">–</td> </tr> <tr> <td align="left"><b>22</b></td> <td align="char" char=".">48</td> <td align="center">–</td> </tr> </tbody> </table> </div> </div> <div class="potencyandisoformspecificityofactivemolecules" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
Potency and isoform specificity of active molecules</div> <p>To measure the inhibitor potency, we determined the IC<sub>50</sub> values for the compounds that resulted active when tested at 100 µM by measuring the residual OST-DGKα activity over a dose range of inhibitor concentrations (0.1 µM, 1.0 µM, 10.0 µM, and 100.0 µM).</p> <p>For R59022 and Amb639752, we measured IC<sub>50</sub> values of 15.2 ± 5.8 µM and 6.9 ± 3.0 µM respectively which were comparable to previous reports using similar assay conditions<a href="#CIT0009"><sup>9</sup></a>. Considering those two as reference/template compounds, we measured the IC<sub>50</sub> values of <b>8</b>, <b>11</b>, <b>12</b>, <b>13</b>, <b>14</b>, <b>16</b>, <b>19</b>, and <b>20</b> as 3.2 ± 1.0, 1.6 ± 0.4, 3.6 ± 1.2, 6.9 ± 2.3, 3.0 ± 1.0, 32.8 ± 11.5, 49.7 ± 31.7, and 1.8 ± 0.4 µM, respectively, signifying that their activity is equal or superior to the template compounds (<a href="#F0006">Figure 6)</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>Dose–response curves for novel DGKα inhibitors. Dose–response of the most active compounds along with their IC<sub>50</sub> values. Data from at least three independent experiments performed in triplicate.</p>   </div> <p>Due to their higher IC<sub>50</sub> values, we thus decided to exclude <b>16</b> and <b>19</b> for further experiments. In summary, we recognised six compounds with equal or superior inhibitory activity compared to commercially available DGKα inhibitors.</p> <p>To check the isoform specificity of those active molecules, we tested them, along with Amb639752, for their ability to inhibit DGKα, DGKζ (the other major DGK isoform expressed in lymphocytes), and the more distantly related and widely expressed DGKθ. At the highest concentration of 100 µM, all those molecules resulted in highly specific against DGKα as like their parent molecule, Amb639752 by completely inhibit DGKα whereas they do not have significant effects on DGKζ and DGKθ apart from <b>20</b> which, at the contrary, acts as an activator of DGKθ (<a href="#F0007">Figure 7)</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>Isoform specificity of novel DGKα inhibitors. 293T cells were transfected with different DGK isoforms (A – DGKα, B – DGKζ, C – DGKθ, respectively) or empty vectors and homogenised. All the molecules were tested at 100 μM for their capacity to inhibit the DGK activity of the different isoform homogenates. Data are means ± SEM of at least three independent experiments performed in triplicate.</p>   </div> </div> <div class="activityofcompoundsonserotoninreceptors" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
Activity of compounds on serotonin receptors</div> <p>R59022, R59949, and ritanserin feature a dual activity as DGKα inhibitors and serotonin receptor antagonists<a href="#CIT0025"><sup>25</sup></a>. Conversely, Amb639752 was reported as a selective DGKα inhibitor which has no effects on serotonin activity<a href="#CIT0009"><sup>9</sup></a>. Thus, we investigated whether the active molecules identified affect serotonin signalling.</p> <p>To this purpose, we measured the effect of serotonin on PMA-induced oxidative burst in human monocytes. As previously shown, 1 µM serotonin reverses the oxidative burst to control values<a href="#CIT0033"><sup>33</sup></a>. Known serotonin receptor antagonists ritanserin and ketanserin (10 µM) impaired serotonin action, while pure DGKα inhibitors such as Amb639752 had no effect (<a href="#F0008">Figure 8)</a>. These data indicate that this assay is sensitive to perturbations in serotonin signalling independently of activity against DGKα. Interestingly, as like Amb639752, all the newly synthesised active molecules did not affect serotonin action (<a href="#F0008">Figure 8</a>), indicating that all those molecules are not serotonin receptor antagonists.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Novel DGKα inhibitors do not affect serotonin signalling. Human monocytes were pre-incubated for 1 h with the indicated drugs in absence or presence of serotonin and then stimulated with PMA 1 µM for 30 min (■ control unstimulated cells, <span class="inline-graphic" title="inline-graphic" /> PMA stimulated cells, ■ PMA and serotonin stimulated cells). Results are expressed as n moles of reduced cytochrome C/10<sup>6</sup> cells. Data are means ± SEM of 10 independent experiments performed in triplicate.</p>   </div> </div> <div class="dgkαinhibitorsrestorericdinsapdeficienttcells" title="sec"><span class="label" tagx="label" title="label">3.4.</span><div class="title" tagx="title" title="title">
DGKα inhibitors restore RICD in SAP deficient T cells</div> <p>Ruffo et al. demonstrated that the defective RICD observed in T cells from XLP-1 patients was rescued by silencing DGKα expression or by pre-treatment with DGKα inhibitors R59949 or R59022<a href="#CIT0008"><sup>8</sup></a>. Interestingly, R59022 also showed beneficial effects in an <i>in vivo</i> model of XLP-1, but due to its poor pharmacological proprieties, its use in human patients results unlikely. We therefore tested the effect all those active molecules along with Amb639752 on RICD sensitivity of SAP-deficient T cells. As additional controls, we also included ritanserin and ketanserin to evaluate the contribution of serotonin antagonism to the effects observed.</p> <p>To evaluate inhibitor efficacy in physiological context, we modelled XLP-1 by silencing SAP in primary peripheral blood T lymphocytes (PBLs) and restimulating them with anti-CD3 antibody (OKT3 10 ng/mL, 24 h). We pre-treated the cells with the indicated inhibitors for 30 min at a concentration of 10 µM<a href="#CIT0008"><sup>8</sup></a>. In control siRNA-transfected cells, DGKα inhibitors poorly affect RICD, with Amb639752, <b>11</b> and <b>14</b> slightly reducing it (<a href="#F0009">Figure 9)</a>. Conversely, DGKα inhibitors significantly rescued the apoptotic defect of SAP-deficient T cells although not reaching control levels. At 10 µM, all the new molecules showed an efficacy comparable to Amb637952 and ritanserin used as positive reference molecules. Conversely, the serotonin antagonist ketanserin is inactive, excluding the involvement of serotonin receptors in rescuing the RICD in SAP-deficient T cells (<a href="#F0009">Figure 9)</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9.</span></a></div>   <p>DGKα inhibitors partially restores apoptosis in SAP deficient lymphocytes. Lymphocytes from normal subjects were transfected with control or SAP specific siRNA (■ control siRNA, ■ SAP siRNA). After 4 days, the cells were restimulated with CD3 agonist OKT3 (10 ng/mL) in presence of respective inhibitor. Vehicle (DMSO) was used as control. Twenty-four hours later, the % of cell loss was evaluated by PI staining. Data are the mean ± SEM of nine independent experiments performed in triplicate.</p>   </div> <p>In summary, these data confirm that the newly identified DGKα inhibitors can rescue RICD susceptibility in T cell models of XLP-1 suggesting a putative use for XLP-1 therapy.</p> </div> <div class="dgkαinhibitorsreducemigrationofthecancercells(mcf7)" title="sec"><span class="label" tagx="label" title="label">3.5.</span><div class="title" tagx="title" title="title">
DGKα inhibitors reduce migration of the cancer cells (MCF7)</div> <p>Previous studies conducted in our laboratory demonstrated that the inhibition of DGK activity decreases chemotaxis, proliferation, migration, and invasion of many cancer cell lines<a href="#CIT0012"><sup>12–14</sup></a>. To evaluate if our newly synthetised DGKα inhibitors were effective in impairing cancer cell migration, we measured serum induced wound healing in MCF7 breast cancer cells in presence of 10 µM inhibitor. In presence of serum, none of the inhibitors is toxic for MCF7 cells even after prolonged treatment (data not shown). After 15 h of treatment, all the newly synthesised active molecules equally reduced cell migration when compared to the vehicle (DMSO) delaying wound closure (<a href="#F0010">Figure 10)</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 10.</span></a></div>   <p>Novel DGKα inhibitors slow tumour cell migration. MCF7 monolayer was wounded and treated for 15 h with serum in presence of our new DGKα inhibitors (10 μM) or vehicle (DMSO). Results are expressed as the percentage of wound area compared to the initial area. Data are the mean ± SEM of nine independent experiments.</p>   </div> <p>Besides being in good agreement with the notion that DGKα is required for cancer cells migration, this observation indicates that our new DGKα inhibitors reduce cancer cell motility, suggesting a potential utility in a metastasis setting.</p> </div> <div class="generationofapharmacophorehypothesis" title="sec"><span class="label" tagx="label" title="label">3.6.</span><div class="title" tagx="title" title="title">
Generation of a pharmacophore hypothesis</div> <p>From the data obtained, it is possible to identify some key pharmacophoric points crucial for the biological activity of the Amb compounds on DGKs namely: (i) a basic nitrogen; (ii) the methyl groups at the 2 and 6 position of the indole nucleus, and (iii) a (hetero)aromatic ring. This information allows us to build a four-point pharmacophoric model represented in <a href="#F0011">Figure 11</a> superimposed with the minimised structure of compound <b>11</b>. Although, we are not able to evaluate the importance of the two carbonyl groups, it represents the first attempts in order to identify the minimum structural request to interact with DGK catalytic site considering the molecular structure of the four most active inhibitors discovered to date (Amb639752, ritanserin, R59022, and R59949). We feel that this model might be useful to identify novel compounds active on DGKα through more targeted virtual screening campaigns, overcoming the current scaffolds.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 11.</span></a></div>   <p>Proposed pharmacophoric model.</p>   </div> </div> </div> <div class="discussion" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Discussion</div> <p>As a key component of several signal transduction pathways, DGKα represent an emerging pharmacological target. We have demonstrated the efficacy of DGKα inhibitors for XLP-1 treatment<a href="#CIT0008"><sup>8</sup></a>, while others have proposed them for cancer treatment<a href="#CIT0022"><sup>22</sup></a> and to remove immune-checkpoints promoting immune vigilance against cancer<a href="#CIT0040"><sup>40</sup></a>. Commercially available DGKα inhibitors are limited by poor specificity<a href="#CIT0025"><sup>25</sup></a><sup>,</sup><a href="#CIT0041"><sup>41</sup></a> and pharmacokinetic<a href="#CIT0020"><sup>20</sup></a>. The CU-3 molecule described by other features a noteworthy activity and specificity but its reactive chemical structure make unlikely an <i>in vivo</i> use<a href="#CIT0030"><sup>30</sup></a>. With intent of developing molecules suitable for therapeutic use we selected Amb639752 as a novel inhibitor with remarkable DGKα activity. Amb639752 also features improved selectivity for DGKα as it does not affect serotonin signalling<a href="#CIT0009"><sup>9</sup></a>. Despite numerous efforts a structure of mammalian DGKs is still missing, thus we decided to explore the structure–activity relationship of this molecule to improve its activity and pave the way for further developments. Our efforts allowed us to build a pharmacophoric model for DGKα inhibitors characterised by three required features. We also characterised a set of novel compounds with improved IC<sub>50</sub> in the low µM range and identified the most profitable synthetic route for them. The mode of DGKα inhibition by those molecules is still unknown apart for ritanserin, which binds at the same time the DGKα catalytic accessory domain and the C1 domain putatively promoting a close inactive conformation<a href="#CIT0041"><sup>41</sup></a>.</p> <p>The second-generation inhibitors we described in this work maintain the specificity of Amb639752 as they not affect DGKζ, the predominant isoform of lymphocytes<a href="#CIT0042"><sup>42</sup></a> and the broadly expressed DGKθ<a href="#CIT0043"><sup>43</sup></a>. Those DGKα inhibitors are active in a lymphocyte based XLP-1 assay and in a cancer cell migration assay, holding the promise for a potential therapeutic application. However, their efficacy is still to be determined in <i>in vivo</i> models of disease where some of the parental compounds showed efficacy but poor pharmacokinetic<a href="#CIT0008"><sup>8</sup></a><sup>,</sup><a href="#CIT0020"><sup>20</sup></a>.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="IENZ_A_1684911_SM5976.pdf">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="disclosurestatement" title="sec"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>The authors declare no competing financial interest.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abramovici</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gee</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Topham</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinases as sources of phosphatidic acid</span>. <span class="source" tagx="source" title="source">Biochim Biophys Acta</span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">1791</span>:<span class="fpage" tagx="fpage" title="fpage">942</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19264149">19264149</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishna</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of diacylglycerol kinases in T cell development and function</span>. <span class="source" tagx="source" title="source">Crit Rev Immunol</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">33</span>:<span class="fpage" tagx="fpage" title="fpage">97</span>–<span class="lpage" tagx="lpage" title="lpage">118</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23582058">23582058</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merida</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avila-Flores</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merino</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinases: at the hub of cell signalling</span>. <span class="source" tagx="source" title="source">Biochem J</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">409</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18062770">18062770</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakane</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kai</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinases: why so many of them?</span><span class="source" tagx="source" title="source">Biochim Biophys Acta</span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">1771</span>:<span class="fpage" tagx="fpage" title="fpage">793</span>–<span class="lpage" tagx="lpage" title="lpage">806</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17512245">17512245</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldanzi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bettio</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malacarne</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graziani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinases: shaping diacylglycerol and phosphatidic acid gradients to control cell polarity</span>. <span class="source" tagx="source" title="source">Front Cell Dev Biol</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">140</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27965956">27965956</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tangye</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span></span><span class="mixed-article-title" title="mixed-article-title">XLP: clinical features and molecular etiology due to mutations in sh2d1a encoding sap</span>. <span class="source" tagx="source" title="source">J Clin Immunol</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">772</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25085526">25085526</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldanzi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pighini</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bettio</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sap-mediated inhibition of diacylglycerol kinase alpha regulates TCR-induced diacylglycerol signaling</span>. <span class="source" tagx="source" title="source">J Immunol</span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">187</span>:<span class="fpage" tagx="fpage" title="fpage">5941</span>–<span class="lpage" tagx="lpage" title="lpage">51</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22048771">22048771</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruffo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malacarne</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of diacylglycerol kinase α restores restimulation-induced cell death and reduces immunopathology in XLP-1</span>. <span class="source" tagx="source" title="source">Sci Transl Med</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">321</span>–<span class="lpage" tagx="lpage" title="lpage">327</span>.</span></li> <li tag="ref"><a name="CIT0009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velnati</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruffo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massarotti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a novel DGKalpha inhibitor for XLP-1 therapy by virtual screening</span>. <span class="source" tagx="source" title="source">Eur J Med Chem</span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">164</span>:<span class="fpage" tagx="fpage" title="fpage">378</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30611057">30611057</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bacchiocchi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldanzi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carbonari</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Activation of alpha-diacylglycerol kinase is critical for the mitogenic properties of anaplastic lymphoma kinase</span>. <span class="source" tagx="source" title="source">Blood</span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">106</span>:<span class="fpage" tagx="fpage" title="fpage">2175</span>–<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15928040">15928040</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yanagisawa</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yasuda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kai</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinase alpha suppresses tumor necrosis factor-alpha-induced apoptosis of human melanoma cells through NF-kappaB activation</span>. <span class="source" tagx="source" title="source">Biochim Biophys Acta</span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">1771</span>:<span class="fpage" tagx="fpage" title="fpage">462</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17276726">17276726</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldanzi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cutrupi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chianale</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinase-alpha phosphorylation by Src on Y335 is required for activation, membrane recruitment and Hgf-induced cell motility</span>. <span class="source" tagx="source" title="source">Oncogene</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">942</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17700527">17700527</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Filigheddu</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cutrupi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Porporato</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinase is required for Hgf-induced invasiveness and anchorage-independent growth of mda-mb-231 breast cancer cells</span>. <span class="source" tagx="source" title="source">Anticancer Res</span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">1489</span>–<span class="lpage" tagx="lpage" title="lpage">92</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17595766">17595766</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rainero</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caswell</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinase alpha controls RCP-dependent integrin trafficking to promote invasive migration</span>. <span class="source" tagx="source" title="source">J Cell Biol</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">196</span>:<span class="fpage" tagx="fpage" title="fpage">277</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22270919">22270919</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeishi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taketomi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirabe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinase alpha enhances hepatocellular carcinoma progression by activation of Ras–Raf–MEK–ERK pathway</span>. <span class="source" tagx="source" title="source">J Hepatol</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">57</span>:<span class="fpage" tagx="fpage" title="fpage">77</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22425622">22425622</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres-Ayuso</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daza-Martin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin-Perez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinase alpha promotes 3D cancer cell growth and limits drug sensitivity through functional interaction with Src</span>. <span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">9710</span>–<span class="lpage" tagx="lpage" title="lpage">26</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25339152">25339152</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rainero</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cianflone</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Porporato</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The diacylglycerol kinase alpha/atypical PKC/beta1 integrin pathway in SDF-1alpha mammary carcinoma invasiveness</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">e97144</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24887021">24887021</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Differential gene and protein expression in primary breast malignancies and their lymph node metastases as revealed by combined cDNA microarray and tissue microarray analysis</span>. <span class="source" tagx="source" title="source">Cancer</span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">100</span>:<span class="fpage" tagx="fpage" title="fpage">1110</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15022276">15022276</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marchet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mocellin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belluco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Gene expression profile of primary gastric cancer: towards the prediction of lymph node status</span>. <span class="source" tagx="source" title="source">Ann Surg Oncol</span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">1058</span>–<span class="lpage" tagx="lpage" title="lpage">64</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17106627">17106627</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dominguez</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Floyd</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinase α is a critical signaling node and novel therapeutic target in glioblastoma and other cancers</span>. <span class="source" tagx="source" title="source">Cancer Discov</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">782</span>–<span class="lpage" tagx="lpage" title="lpage">97</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23558954">23558954</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kefas</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Floyd</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comeau</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A miR-297/hypoxia/DGK-α axis regulating glioblastoma survival</span>. <span class="source" tagx="source" title="source">Neuro Oncol</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">1652</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24158111">24158111</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purow</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular pathways: targeting diacylglycerol kinase alpha in cancer</span>. <span class="source" tagx="source" title="source">Clin Cancer Res</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">21</span>:<span class="fpage" tagx="fpage" title="fpage">5008</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26420856">26420856</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakane</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanoh</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Selectivity of the diacylglycerol kinase inhibitor 3-[2-(4-[bis-(4-fluorophenyl)methylene]-1-piperidinyl)ethyl]-2, 3-dihydro-2-thioxo-4(1h)quinazolinone (r59949) among diacylglycerol kinase subtypes</span>. <span class="source" tagx="source" title="source">Biochem Pharmacol</span><span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">59</span>:<span class="fpage" tagx="fpage" title="fpage">763</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10718334">10718334</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasaki</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluations of the selectivities of the diacylglycerol kinase inhibitors r59022 and r59949 among diacylglycerol kinase isozymes using a new non-radioactive assay method</span>. <span class="source" tagx="source" title="source">Pharmacology</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">92</span>:<span class="fpage" tagx="fpage" title="fpage">99</span>–<span class="lpage" tagx="lpage" title="lpage">107</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23949095">23949095</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boroda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niccum</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raje</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Dual activities of ritanserin and r59022 as DGKα inhibitors and serotonin receptor antagonists</span>. <span class="source" tagx="source" title="source">Biochem Pharmacol</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">123</span>:<span class="fpage" tagx="fpage" title="fpage">29</span>–<span class="lpage" tagx="lpage" title="lpage">39</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27974147">27974147</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaulton</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hersey</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowotka</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The ChEMBL database in 2017</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">45</span>:<span class="fpage" tagx="fpage" title="fpage">D945</span>–<span class="lpage" tagx="lpage" title="lpage">D954</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899562">27899562</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leysen</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gommeren</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Gompel</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor-binding properties in vitro and in vivo of ritanserin: a very potent and long acting serotonin-S2 antagonist</span>. <span class="source" tagx="source" title="source">Mol Pharmacol</span><span class="year" tagx="year" title="year">1985</span>;<span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">600</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2860558">2860558</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawkins</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skillman</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholls</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of shape-matching and docking as virtual screening tools</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">50</span>:<span class="fpage" tagx="fpage" title="fpage">74</span>–<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17201411">17201411</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muchmore</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Souers</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akritopoulou-Zanze</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">The use of three-dimensional shape and electrostatic similarity searching in the identification of a melanin-concentrating hormone receptor 1 antagonist</span>. <span class="source" tagx="source" title="source">Chem Biol Drug Des</span><span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">67</span>:<span class="fpage" tagx="fpage" title="fpage">174</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16492165">16492165</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunii</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakuma</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A novel diacylglycerol kinase α-selective inhibitor, CU-3, induces cancer cell apoptosis and enhances immune response</span>. <span class="source" tagx="source" title="source">J Lipid Res</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">57</span>:<span class="fpage" tagx="fpage" title="fpage">368</span>–<span class="lpage" tagx="lpage" title="lpage">79</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26768655">26768655</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baell</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holloway</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span></span><span class="mixed-article-title" title="mixed-article-title">New substructure filters for removal of pan assay interference compounds (pains) from screening libraries and for their exclusion in bioassays</span>. <span class="source" tagx="source" title="source">J Med Chem</span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">53</span>:<span class="fpage" tagx="fpage" title="fpage">2719</span>–<span class="lpage" tagx="lpage" title="lpage">40</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20131845">20131845</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavagno</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunella</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bardelli</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Anti-inflammatory drugs and tumor necrosis factor-alpha production from monocytes: role of transcription factor NF-kappa B and implication for rheumatoid arthritis therapy</span>. <span class="source" tagx="source" title="source">Eur J Pharmacol</span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">501</span>:<span class="fpage" tagx="fpage" title="fpage">199</span>–<span class="lpage" tagx="lpage" title="lpage">208</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15464079">15464079</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talmon</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pastore</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Vortioxetine exerts anti-inflammatory and immunomodulatory effects on human monocytes/macrophages</span>. <span class="source" tagx="source" title="source">Br J Pharmacol</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">175</span>:<span class="fpage" tagx="fpage" title="fpage">113</span>–<span class="lpage" tagx="lpage" title="lpage">24</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29057467">29057467</a></span></span></li> <li tag="ref"><a name="CIT0034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Omega</span></span>, version 2.4.6; OpenEye Scientific Software: Santa Fe, NM. Available from: <a href="http://www.eyesopen.com">http://www.eyesopen.com</a></span></li> <li tag="ref"><a name="CIT0035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawkins</span><span class="given-names" tagx="given-names" title="given-names">PCD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skillman</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warren</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Conformer generation with omega: algorithm and validation using high quality structures from the Protein Databank and Cambridge Structural Database</span>. <span class="source" tagx="source" title="source">J Chem Inf Model</span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">50</span>:<span class="fpage" tagx="fpage" title="fpage">572</span>–<span class="lpage" tagx="lpage" title="lpage">84</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20235588">20235588</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawkins</span><span class="given-names" tagx="given-names" title="given-names">PCD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholls</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Conformer generation with omega: learning from the data set and the analysis of failures</span>. <span class="source" tagx="source" title="source">J Chem Inf Model</span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">52</span>:<span class="fpage" tagx="fpage" title="fpage">2919</span>–<span class="lpage" tagx="lpage" title="lpage">36</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23082786">23082786</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneidman-Duhovny</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dror</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inbar</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pharmagist: a webserver for ligand-based pharmacophore detection</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">36</span>:<span class="fpage" tagx="fpage" title="fpage">W223</span>–<span class="lpage" tagx="lpage" title="lpage">228</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18424800">18424800</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wermuth</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aldous</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raboisson</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rognan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="source" tagx="source" title="source">The practice of medicinal chemistry</span>. <span class="edition" tagx="edition" title="edition">4th ed</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span>; <span class="year" tagx="year" title="year">2015</span>.</span></li> <li tag="ref"><a name="CIT0039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urruticoechea</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sejer Pedersen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">A scalable method for regioselective 3-acylation of 2-substituted indoles under basic conditions</span>. <span class="source" tagx="source" title="source">J Org Chem</span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">80</span>:<span class="fpage" tagx="fpage" title="fpage">471</span>–<span class="lpage" tagx="lpage" title="lpage">81</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25469854">25469854</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakane</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mizuno</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Komenoi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diacylglycerol kinases as emerging potential drug targets for a variety of diseases: an update</span>. <span class="source" tagx="source" title="source">Front Cell Dev Biol</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27583247">27583247</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franks</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purow</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The ligand binding landscape of diacylglycerol kinases</span>. <span class="source" tagx="source" title="source">Cell Chem Biol</span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">24</span>:<span class="fpage" tagx="fpage" title="fpage">870</span>–<span class="lpage" tagx="lpage" title="lpage">80.e5</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28712745">28712745</a></span></span></li> <li tag="ref"><a name="CIT0042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joshi</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The ζ isoform of diacylglycerol kinase plays a predominant role in regulatory T cell development and TCR-mediated ras signaling</span>. <span class="source" tagx="source" title="source">Sci Signal</span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">ra102</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24280043">24280043</a></span></span></li> <li tag="ref"><a name="CIT0043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tu-Sekine</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldschmidt</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raben</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span><span class="mixed-article-title" title="mixed-article-title">DGK-θ: structure, enzymology, and physiological roles</span>. <span class="source" tagx="source" title="source">Front Cell Dev Biol</span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">101</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27683659">27683659</a></span></span></li> </ul> </div> </div>  </body></html>