<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Cell Death Dis</span><span class="iso-abbrev" title="iso-abbrev">Cell Death Dis</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Cell Death &amp;amp; Disease</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2041-4889</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7181758">7181758</a></span><span class="publisher-id" title="publisher-id">2434</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41419-020-2434-z">10.1038/s41419-020-2434-z</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Reprogramming of lipid metabolism in cancer-associated fibroblasts potentiates migration of colorectal cancer cells</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">Jin</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9277-3048</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Yiyun</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Huaqin</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Chunqi</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Zhong</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Xiaowei</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Jiamei</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">Yuanyuan</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sang</span><span class="given-names" tagx="given-names" title="given-names">Na</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">Xinying</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-4988-2190</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Liang</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9760-5238</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cen</span><span class="given-names" tagx="given-names" title="given-names">Xiaobo</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">Xiao</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">Duxiao_home@163.com</span></span><a href="#Aff2">2</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Yinglan</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">zhaoyinglan@scu.edu.cn</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], </span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0807 1581</span><span class="institution-id" title="institution-id">grid.13291.38</span><span class="institution" title="institution">Department of Gastrointestinal Surgery, West China Hospital, </span><span class="institution" title="institution">Sichuan University, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0807 1581</span><span class="institution-id" title="institution-id">grid.13291.38</span><span class="institution" title="institution">National Chengdu Center for Safety Evaluation of Drugs, State Key Laboratory of Biotherapy, West China Hospital, </span><span class="institution" title="institution">Sichuan University, and Collaborative Innovation Center for Biotherapy, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0632 4559</span><span class="institution-id" title="institution-id">grid.411634.5</span><span class="institution" title="institution">Department of General Surgery, </span><span class="institution" title="institution">Ya an People’s Hospital, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-4</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">267</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-30</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-3-23</span><span class="accepted" title="accepted">accepted: 2020-3-23</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Metabolic interaction between cancer-associated fibroblasts (CAFs) and colorectal cancer (CRC) cells plays a major role in CRC progression. However, little is known about lipid alternations in CAFs and how these metabolic reprogramming affect CRC cells metastasis. Here, we uncover CAFs conditioned medium (CM) promote the migration of CRC cells compared with normal fibroblasts CM. CAFs undergo a lipidomic reprogramming, and accumulate more fatty acids and phospholipids. CAFs CM after protein deprivation still increase the CRC cells migration, which suggests small molecular metabolites in CAFs CM are responsible for CRC cells migration. Then, we confirm that CRC cells take up the lipids metabolites that are secreted from CAFs. Fatty acids synthase (FASN), a crucial enzyme in fatty acids synthesis, is significantly increased in CAFs. CAF-induced CRC cell migration is abolished by knockdown of FASN by siRNA or reducing the uptake of fatty acids by CRC cells by sulfo-N-succinimidyloleate sodium in vitro and CD36 monoclonal antibody in vivo. To conclude, our results provide a new insight into the mechanism of CRC metastasis and suggest FASN of CAFs or CD36 of CRC cells may be potential targets for anti-metastasis treatment in the future.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Cancer metabolism</div> <div class="kwd" title="kwd">
Cancer microenvironment</div> <div class="kwd" title="kwd">
Cancer microenvironment</div> <div class="kwd" title="kwd">
Cancer metabolism</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National keypoint research and invention program of china Ministry of Science and Technologyh(MOST-2016YFC1303200) National S&amp;amp;T Major project(2018ZX09201018)</span></div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Colorectal cancer (CRC), the third most commonly diagnosed malignant cancer, ranks third in cancer mortality worldwide<sup><a href="#CR1">1</a></sup>. Metastasis is the major cause for the death of terminal stage patients of CRC. The majority of patients with metastatic CRC only have a median survival of 2 years that is far from satisfaction. Recent studies illustrate that tumor metastasis is closely depended on the interaction of tumor cells with tumor microenvironment (TME), which is a dynamic, changing network.</p> <p id="Par3">There are many non-tumor stromal cells in the TME and the functional changes of these stromal cells affect the malignant potential of tumor cells. Cancer-associated fibroblasts (CAFs) are one of the main components of stromal cells surrounding cancer cells. CAFs are active fibroblasts that proliferate faster, and secrete more cytokines, matrix proteins, and immunomodulatory factors compared with normal fibroblasts (NFs)<sup><a href="#CR2">2</a>–<a href="#CR4">4</a></sup>. It is now established that CAFs play an indispensable role in tumor initiation, progression, and metastasis mainly by remodeling of the extracellular matrix (ECM), induction of angiogenesis, secretion growth factors and cytokines, and promoting epithelial mesenchymal transition (EMT)<sup><a href="#CR5">5</a></sup>. For instance, CAFs promote cancer cells invasion by making passageways through collagen I/Matrigel gels. CAFs also lead tumor growth and metastasis via cytokine and microRNA interactions<sup><a href="#CR5">5</a></sup>. Moreover, recent works reveal the impact of metabolic interaction between CAFs and tumor cells on tumor metastasis.</p> <p id="Par4">Metabolic reprogramming is a hallmark of tumor cells. Cancer cells frequently have a dominating glycolysis metabolism, the "Warburg effect", to generate the building block for synthesis and a large amount of ATP for their fast proliferation in nutrient shorted microenvironment. The metabolic adaptation of cancer cells not only permits the development and establishment of a tumor in a certain microenvironment, but also conditions the response to the therapy. Metabolic adaptation cannot be assessed exclusively from the cancer cell point of view, but should also consider the contribution of normal (noncancerous) cells in the same tissue or organ. In cancer, the surrounding cells (fibroblasts, endothelial cells, immune cells, and adipocytes) influence the TME, the tumor cell biology, and metabolism<sup><a href="#CR6">6</a></sup>. The emerging view becomes focusing on metabolic alternation of CAFs. In order to adapt to the deficiency of nutrients and oxygen in TEM, and the increased requirements of energy and building blocks crucial for maintaining high proliferation, CAFs undergo metabolic reprogramming. Current studies on CAFs metabolic reprogramming mainly focus on glycol metabolism and amino acid metabolism reprogramming<sup><a href="#CR7">7</a></sup>. Some studies show that compared with NFs, CAFs have an enhanced aerobic glycolysis and decreased oxidative phosphorylation<sup><a href="#CR8">8</a></sup>. As a consequence of glycolysis, CAFs could produce large amounts of lactate. The excess lactate is then released to the ECM and results in acidic microenvironment that is thought to favor the metastatic potential of cancer cells<sup><a href="#CR9">9</a></sup>. Studies on the role of amino acid metabolism in CAFs have demonstrated that CAFs produce a variety of metabolic substrates for cancer cells, including glutamine. Glutamine increases the autophagy of fibroblasts that may be the fundamental energy to promote the activity of mitochondria in cancer cells<sup><a href="#CR10">10</a></sup>. Glutamate ammonia ligase, the key enzyme in glutamine synthesis, is upregulated in CAFs. The glutamine importers (e.g., SLC6 A14) of cancer cells are upregulated in a glutamine concentration-dependent pattern in the presence of fibroblasts<sup><a href="#CR10">10</a></sup>. Besides well-known glycolysis and glutamine metabolism, there is increasing evidence that lipids play critical roles in tumorigenesis and progression.</p> <p id="Par5">Changes in lipid metabolism affect many cancer cells process, including cell growth, proliferation, motility, autophagy<sup><a href="#CR11">11</a></sup>, and apoptosis<sup><a href="#CR12">12</a></sup>. Moreover, studies have demonstrated that lipids can directly regulate non-tumor stromal cells in the TME. For example, activated CD8+ T cells reprogram the cholesterol metabolism and synthesize more free cholesterol to support rapid cell proliferation. Pancreatic stellate cells (PSCs) derived CAFs secrete abundant lyso-phosphatidylcholines (LPCs) in the activated, fibroblastic state. Lipid released by PSCs promotes multiple hallmarks of aggressive pancreatic ductal adenocarcinoma (PDAC) progression, including PDAC cell proliferation and migration<sup><a href="#CR13">13</a></sup>. BM adipocytes support acute monocytic leukemia (AMoL) cell survival by regulating their metabolic energy balance, and that the disruption of FAO in BM adipocytes may be an alternative, novel therapeutic strategy for AMoL therapy<sup><a href="#CR14">14</a></sup>. Therefore, lipid modification in CAFs may have impact on tumor progression. However, little is known about lipid alternation in CAFs and how these metabolic reprogramming affect CRC cells metastasis.</p> <p id="Par6">In this study, we comprehensively analyze the lipidomic modification of CAFs by using an Acquity UPLC-quadrupole time of flight mass spectrometry (UPLC-Q-TOF/MS). Our studies reveal CAFs from CRC tissues undergo a lipidomic reprogramming, and enhance the CRC cells migration by these lipid metabolites crosstalk, which are from CAFs activated fatty acid de novo synthesis and reduced catabolism. Taken together, our study provides a new insight to explain how CAFs stimulate tumor metastasis and present a potential target for tumor metastasis therapy.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="characterizationofprimarynfsandcafs" title="sec"> <div class="title" tagx="title" title="title">
Characterization of primary NFs and CAFs</div> <p id="Par7">The abundances of fibroblasts in TME were confirmed by immunohistochemistry (IHC) staining of α-SMA, and the results showed that high density of α-SMA expression were observed in CRC tissues than that in normal adjacent tissues (NATs), which suggest that fibroblasts is more abundance in CRC tissues (Fig. <a href="#Fig1">1a</a>). The primary NFs and CAFs were isolated from fresh CRC tissues and NATs, and they showed a spindle shape (Fig. <a href="#Fig1">1b</a>). Immunofluorescence results showed that Vimentin and α-SMA expression was positive, whereas cytokeratin expression was negative in NFs and CAFs (Fig. <a href="#Fig1">1c</a>), which suggesting pure fibroblasts were obtained. Moreover, α-SMA were overexpressed in CAFs compared with in NFs, which indicating that CAFs were more active than NFs. The expression of fibroblast biomarkers was further confirmed by qRT-PCR and western blot. The results showed that both mRNA and protein expression levels of α-SMA in CAFs were significantly higher than that of NFs (Fig. <a href="#Fig1">1d, e</a>). Altogether, these data indicated that we successfully isolated high-purity NFs and CAFs from CRC samples. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <div class="title" tagx="title" title="title">
Characterization of primary NFs and CAFs.</div> <p><b>a</b> The expression of α-SMA in CRC tissues were assayed by immunohistochemistry staining, indicating that CAFs were abundant in CRC tumor stroma (up panels scale bars, 100 µm; down panels scale bars, 50 µm). <b>b</b> The morphological images of NFs and CAFs (up panels scale bars, 200 µm; down panels scale bars, 100 µm). <b>c</b> Representative immunofluorescence images of CAFs and NFs that was immunostained for nucleus (blue), Vimentin (red, the first row), Cytokeratin AE1–AE3 (red, the second row), and α-SMA (green, the third row). Scale bars, 50 µm. <b>d</b> The mRNA level of α-SMA in NFs and CAFs by qRT-PCR; <i>n</i> = 3 (replicating from three patients); ***<i>p</i> &amp;lt; 0.001; **<i>p</i> &amp;lt; 0.01. <b>e</b> The protein expression of α-SMA in NFs and CAFs derived from three patients were analyzed by western blot analysis.</p>   </div> <p /> </div> <div class="cafspromotesthemigrationofcrccells" title="sec"> <div class="title" tagx="title" title="title">
CAFs promotes the migration of CRC cells</div> <p id="Par8">In order to investigate the effect of CAFs on CRC cells, we firstly examined the effect of CAFs on CRC cells proliferation by using MTT assay. The result showed that both CAFs conditioned medium (CM) and NFs CM promoted the proliferation of CRC compared with CRC CM. However, there was no significant difference between CAFs CM and NFs CM (Supplementary Fig. <a href="#MOESM4">1a</a>). Then, we explored the effects of CAFs on CRC cells migration. We firstly screened the metastatic potential of CRC cells by detecting the ratio of E-cadherin relative to Vimentin, since E-cadherin and Vimentin are important EMT marker. The results showed DLD1 cells had higher value of E-cadherin/Vimentin, which indicated DLD1 cells had the least metastasis inclination (Supplementary Fig. <a href="#MOESM4">1b</a>). Thus, we chose DLD1 to detect the impact of CAFs on cell migration. The transwell assay and wound-healing assay showed CAFs CM significantly enhanced the migration of DLD1 cells compared with NFs CM and DLD1 CM (Supplementary Fig. <a href="#MOESM5">2a–c</a>). Immunofluorescence results showed that the expression Vimentin increased, while the expression of E-cadherin decreased in DLD1 after CAFs CM treatment (Supplementary Fig. <a href="#MOESM5">2d</a>). We then examined the mRNA expression of metastasis-related gene in DLD1 cells after CAFs CM treatment. The result indicated that CAFs CM increased the mRNA expression of Vimentin, MMP2, MMP9, Snail, whereas NFs CM not (Supplementary Fig. <a href="#MOESM5">2e</a>). Meanwhile, western blot analysis showed CAFs CM enhanced Vimentin protein expression and reduced E-cadherin expression in DLD1, whereas NFs CM not (Supplementary Fig. <a href="#MOESM5">2f</a>). Taken together, these data illustrated that CAFs promoted the DLD1 migration.</p> </div> <div class="cafsundergoalipidomicreprogramming" title="sec"> <div class="title" tagx="title" title="title">
CAFs undergo a lipidomic reprogramming</div> <p id="Par9">Previous studies have reported enhanced glycolysis in CAFs of CRC. However, there is little research refer to the lipidomic profile of CAFs. To explore the alternation of lipid metabolism of CAFs and whether it promotes migration of CRC cells, we detected lipid profiling of CAFs and NFs and their CM by using UPLC-Q-TOF/MS based on lipidomic assay. The lipidomic profile of CAFs CM are significant different from NFs CM. The heat map displayed that most of phospholipids, fatty acyls, and cholesteryl ester (CE) was increased in CAFs CM (Fig. <a href="#Fig2">2a, b</a>). And we presented the relative level of all lipid derivatives in CAFs and NFs (Supplementary Fig. <a href="#MOESM6">3a</a>). Lipids class statistic showed among 92 changed lipids, the fatty acyls accounted for the most proportion of those modified lipids, which is 44% (Fig. <a href="#Fig2">2c</a>). Deeply analysis of fatty acyls profile showed most of altered fatty acyls are lipids with long, polyunsaturated carbon chain (Fig. <a href="#Fig2">2e–h</a>). Further analysis of lipid metabolites revealed that fatty acids, diglycerides (DGs), phosphatidic acid (PA), phosphatidylinositol (PI), LPC, and phosphatidylethanolamines (PE) significantly increased (Fig. <a href="#Fig2">2d</a>). Meanwhile, the lipidomic profile of CAFs and NFs showed that fatty acyls in CAFs was still higher than NFs. However, lipid in cells was significantly lower than that in CM (Supplementary Fig. <a href="#MOESM6">3b</a>). Taken together, these data indicated that CAFs underwent a lipidomic reprogramming, and excreted more fatty acyls and phospholipids than NFs. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <div class="title" tagx="title" title="title">
CAFs undergo a lipid reprogramming.</div> <p><b>a</b>, <b>b</b> Heat map of the modified lipids between CAFs CM and NFs CM. The values on heat map is normalized by the mean value of control group (NFs CM group), and the color bars represent the log<sub>10</sub> value of the ratio of each lipid species. <b>d</b> The normalized abundance of lipids in CAFs CM and NFs CM. <b>c</b> The ratio of four main lipid classes in total altered metabolites, including fatty acyls (FA), GL, GP, and SP. <b>e</b> Proportions of changed FA with long or medium carbon chain. <b>f</b> Overall difference of FA with long or medium carbon chain. Medium carbon chain (C ≤ 12), long carbon chain (C ≥ 14). <b>g</b> Proportions of different FA (sorted by unsaturation). <b>h</b> Overall difference of FA (sorted by unsaturation). Fatty AH fatty aldehyde, FE fatty ester, GL glycerides, GP glycerophospholipids, SP sphingolipids, CE cholesterol ester, ST steroid, LPC lyso-phosphatidylcholine, PC phosphatidylcholine, TG triglyceride, PE phosphatidylethanolamine, PA phosphatidic acid, SM sphingomyelin.</p>   </div> <p /> </div> <div class="smallmolecularmetabolitesincafscminducecrccellsmigration" title="sec"> <div class="title" tagx="title" title="title">
Small molecular metabolites in CAFs CM induce CRC cells migration</div> <p id="Par10">While extracellular cytokine can enhance the migration ability of tumor cells<sup><a href="#CR15">15</a>,<a href="#CR16">16</a></sup>, lipid uptake could modify the plasma membrane that is closely related to cell motility<sup><a href="#CR17">17</a>,<a href="#CR18">18</a></sup>. Thus, we hypothesized that the CAFs may enhance CRC cells migration by supporting lipids metabolites. In order to demonstrate our speculation, we deprived the large molecules protein in CM by using three freeze-thaw cycles to damage the protein followed by filtration with 3-kD cutoff filter. We observed that CAFs CM after large molecules protein deprivation still had the ability to increase the DLD1 migration in both wound-healing assay (Fig. <a href="#Fig3">3a, c</a>) and transwell migration assay (Fig. <a href="#Fig3">3b, c</a>), whereas NFs CM not. Immunofluorescence results showed that the expression Vimentin increased, while the expression of E-cadherin decreased in DLD1 after protein-filtered CAFs CM treatment (Fig. <a href="#Fig3">3d</a>). Moreover, CAFs CM after large molecules protein deprivation enhanced Vimentin expression and reduced E-cadherin expression in both mRNA and protein level (Fig. <a href="#Fig3">3e, f</a>). These results indicated that the factors inducing DLD1 migration were small size and lacked tertiary structure, suggesting that the factors could be metabolites. Therefore, CAFs CM after large molecules protein deprivation was used in following experiments. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <div class="title" tagx="title" title="title">
Metabolites in CAF CM enhance DLD1 migration ability.</div> <p><b>a</b>, <b>b</b> Migration of DLD1 cells were analyzed by Wound-Healing assay <b>a</b> (scale bars, 200 µm) and transwell assay <b>b</b> (scale bars, 100 µm), respectively, after protein-filtered CM treatment. In transwell assay, 1# and 3# used the same image of DLD1 CM. <b>c</b> Cells migration ability in wound-healing assay and transwell assay was displayed as cell migrated rate (%) and relative permeability, respectively. The values are expressed as the mean ± SD of the average cells number of five randomly selected fields relative to the control group. The control group is DLD1 CM group. <b>d</b> Representative immunofluorescence images of Vimentin (Green) and E-cadherin (Red) in DLD1 after protein-filtered CM treatment (scale bars, 16 µm). <b>e</b> mRNA level of metastasis-related genes after co-culturing with protein-filtered CM; the values are expressed as the mean ± SD (<i>n</i> = 3); <i>*p</i> &amp;lt; 0.05; <i>**p</i> &amp;lt; 0.01; <i>***p</i> &amp;lt; 0.001. <b>f</b> Protein expression of Vimentin and E-cadherin in DLD1 after protein-filtered CM treatment.</p>   </div> <p /> </div> <div class="lipidsmetabolitesfromcafsaretakenupbycrccells" title="sec"> <div class="title" tagx="title" title="title">
Lipids metabolites from CAFs are taken up by CRC cells</div> <p id="Par11">In order to further verify our hypothesis, we next performed a series of metabolomic studies to verify whether DLD1 cells take up the metabolites secreted by CAFs (Fig. <a href="#Fig4">4a</a>). Specifically, we sought molecules that were overrepresented in CAFs CM (and therefore secreted by CAFs); underrepresented in the CAFs CM after contact with DLD1 cells (removed by DLD1 cells); and overrepresented in DLD1 cells co-cultured with the CAFs CM (taken up by DLD1 cells). Of ~200 metabolites analyzed, we found 19 lipids followed this pattern, including five fatty acyls, five glycerolipids (GL), eight glycerophospholipids (GP), and one CE (Fig. <a href="#Fig4">4b</a>). We next analyzed the lipids metabolites in DLD1 cells co-cultured with CAFs CM comparing with DLD1 cells co-cultured with DLD1 CM. The results showed lipids, including PC, PE, phosphatidylserine, PI, phosphatidic acid (PA), DG, and triglyceride (TG) are increased significantly after CAFs CM treatment (Fig. <a href="#Fig5">5a</a>). The statistic results indicated that lipids metabolites in GP and GL pathway (both are downstream metabolites of fatty acids) took account ~69% of the total modified lipids in DLD1 cells. However, the fatty acyls only took account ~12% (Fig. <a href="#Fig5">5b</a>). We then deeply analyzed the length and unsaturation of carbon chain in these modified GPs and GLs. Interestingly, large part of these GPs and GLs have long and polyunsaturated carbon chain (Fig. <a href="#Fig5">5c–f</a>), which were in accordance with the pattern of fatty acyls increased in CAF CM (Fig. <a href="#Fig2">2d–g</a>). These results suggested that DLD1 might use fatty acyls from CAFs as resource of lipids synthesis. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <div class="title" tagx="title" title="title">
Lipids secreted by CAFs are taken in by DLD1 cells.</div> <p><b>a</b> Schematic of paracrine tracing experimental procedure. <b>b</b> Relative metabolite level in CAF CM, cultured CM, and DLD1. Each bar represents the average (mean ± SD, <i>n</i> = 5); <i>*p</i> &amp;lt; 0.05; <i>**p</i> &amp;lt; 0.01; <i>***p</i> &amp;lt; 0.001.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <div class="title" tagx="title" title="title">
Lipids modification in DLD1 after co-culturing with CAFs.</div> <p><b>a</b> The normalized abundance of lipid modification profile in DLD1 co-culturing with CAFs CM and DLD1 CM. <b>b</b> The ratio of four main lipid classes in total altered metabolites. <b>c</b> Proportions of changed lipids with two fatty acyls chains (including DG, GPs, and SPs; sorted by length of carbon chain). <b>d</b> Overall difference of changed lipids with two fatty acyls chains (sorted by length of carbon chain). <b>e</b> Proportions of changed lipids with two fatty acyls chains (sorted by unsaturation). <b>f</b> Overall difference of changed lipids with two fatty acyls chains.</p>   </div> <p /> </div> <div class="fasnknockdownincafsorinhibitionfauptakereducecrccellsmigration" title="sec"> <div class="title" tagx="title" title="title">
FASN knockdown in CAFs or inhibition FA uptake reduce CRC cells migration</div> <p id="Par12">According to the difference of lipidomic profile in CM between CAFs and NFs (Fig. <a href="#Fig2">2a, b</a>), we detected the relative mRNA level of metabolic enzymes in de novo synthesis of fatty acids pathways, fatty acid catabolism pathway, and phospholipid synthesis pathways (Fig. <a href="#Fig6">6a</a>), showed fatty acid synthase (FASN) was significantly increased and most degrading enzymes were significantly decreased in CAFs (Fig. <a href="#Fig6">6a, b</a>), which suggested that CAFs accumulated more fatty acids. These results were consistent with the trend of increasing fatty acids in CM (Fig. <a href="#Fig2">2a, b</a>). FASN, the key enzyme of fatty acids synthesis, catalyzes the synthesis of fatty acids from acetyl CoA and malonyl-CoA. We then examined the expression of interstitial FASN in CRC tissues and NATs. The immunofluorescence results showed that the level of interstitial FASN in CRC tissues is higher than that in NATs (Fig. <a href="#Fig6">6c</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <div class="title" tagx="title" title="title">
Lipids biosynthesis and degradation in cells.</div> <p><b>a</b> Lipids biosynthesis and degradation schematic in cells. <b>b</b> The relative mRNA level of metabolic enzymes in de novo synthesis of fatty acids pathways, fatty acid catabolism pathway, and phospholipid synthesis pathways. The values are expressed as the mean ± SD (<i>n</i> = 3); <i>*p</i> &amp;lt; 0.05; <i>**p</i> &amp;lt; 0.01; <i>***p</i> &amp;lt; 0.001. <b>c</b> Representative immunofluorescence images of CRC tissues and NATs that was immunostained for nucleus (blue), FASN (red), and α-SMA (green). Scale bars, 50 µm.</p>   </div> <p /> <p id="Par13">Thus, we knockdown the expression of FASN in CAFs by siRNA (Fig. <a href="#Fig7">7e</a>). FASN knockdown reduced and production of fatty acids in CAFs, and then decrease the lipid metabolites in the downstream of fatty acids, which were taken up by DLD1 cells. Wound-healing assay (Fig. <a href="#Fig7">7a, c</a>) and transwell assay (Fig. <a href="#Fig7">7b, c</a>) showed CM from siFASN CAF reduced DLD1 cells migration compared with the CM from siNC CAFs. In addition, E-cadherin decrease and Vimentin increase in DLD1 cells induced by CAF CM was restored after FASN knockdown in CAFs (Fig. <a href="#Fig7">7d, e</a>). And we also found FASN knockdown reduced the level of α-SMA in CAFs (Fig. <a href="#Fig7">7e</a>). These results confirmed that siFASN in CAFs reduced the migration ability of DLD1 cells and indicated that FASN was a crucial enzyme in CAFs. In addition, we knockdown the expression of FASN in NFs by siRNA (Supplementary Fig. <a href="#MOESM7">4a</a>). Wound-healing assay and transwell assay (Supplementary Fig. <a href="#MOESM7">4a</a>) showed CM from siFASN NFs reduced DLD1 cells migration compared with the CM from siNC NFs. But compared with CAF, the reduction is not obvious. It may be due to the lower expression of FASN in NFs. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <div class="title" tagx="title" title="title">
FASN knockdown in CAFs or inhibition FA uptake reduce CRC cells migration.</div> <p><b>a</b>, <b>b</b> Migration of DLD1 cells were analyzed by wound-healing assay <b>a</b> (scale bars, 200 µm) and transwell assay <b>b</b> (scale bars, 100 µm), respectively, after CAFs CM treatment. <b>c</b> Cells migration ability in wound-healing assay and transwell assay was displayed as cell migrated rate (%) and relative permeability, respectively. The values are expressed as the mean ± SD of the average cells number of five randomly selected fields relative to the control group. <b>d</b> Relative mRNA level of metastasis-related genes after co-culturing with CM. The values are expressed as the mean ± SD (<i>n</i> = 3); *<i>p</i> &amp;lt; 0.05; **<i>p</i> &amp;lt; 0.01; ***<i>p</i> &amp;lt; 0.001. <b>e</b> Protein expression of FASN and α-SMA in CAFs after siRNA knockdown; protein expression of Vimentin and E-cadherin in DLD1 after CM treatment. <b>f</b>, <b>g</b> Cells migration ability in transwell assay was displayed as relative permeability. The values shown are expressed as the mean ± SD of the average cells number of five randomly selected fields relative to the control group; <i>*p</i> &amp;lt; 0.05; <i>**p</i> &amp;lt; 0.01; <i>***p</i> &amp;lt; 0.001.</p>   </div> <p /> <p id="Par14">To further confirm the results that fatty acids secreted by CAFs promoted the migration of CRC cells, we used sulfo-N-succinimidyloleate (SSO), an inhibitor of CD36, which is the most important fatty acid transporter on the cell surface<sup><a href="#CR19">19</a>–<a href="#CR21">21</a></sup>. CRC cells were treated with 50 μM SSO for 4 h (ref. <sup><a href="#CR22">22</a></sup>). Transwell assay (Fig. <a href="#Fig7">7f, g</a>) showed SSO reduced the CAF CM induced migration of DLD1 and HCT-15 cells. In addition, considering the influence of lipids in fetal bovine serum (FBS), we changed FBS to dialyzed FBS, which filtered out molecules &amp;lt;10,000 D. We repeated all transwell studies using 1# primary fibroblasts. These results were consistent with our previous research results (Supplementary Fig. <a href="#MOESM7">4b</a>). These data suggested that fatty acids secreted from CAFs contributed to CRC cells metastasis in vitro.</p> <p id="Par15">To evaluate the pro-metastatic effect of CAFs in vivo, CRC cancer DLD1 cells (1 × 10<sup>6</sup>) were mixed or not mixed with an equal number of CAFs or NFs, and injected into the spleen of 7-week-old female nude mice. For the mice injected the mixture of DLD1 and CAFs, intraperitoneal injection of PBS or CD36 monoclonal antibody (JC63.1), which reported to block fatty acid uptake in vivo. As expected, the mice injected with the mixture of DLD1 and CAFs had more metastatic nodules in the liver than other mice did. The loss of CD36 reduced the metastasis in mouse liver (Fig. <a href="#Fig8">8a</a>). Hematoxylin and eosin (H&amp;amp;E) staining further showed that the mice injected with the mixture of DLD1 and CAFs had more metastasis in liver in comparison with that of other group, and the loss of CD36 reduced the metastasis (Fig. <a href="#Fig8">8b</a>). These data suggested that fatty acids secreted from CAFs contributed to CRC cells metastasis in vivo. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <div class="title" tagx="title" title="title">
Inhibition FA uptake reduce CRC cells migration in vivo.</div> <p><b>a</b> The tumor metastasis node in liver of each group. <b>b</b> H&amp;amp;E staining of liver sections; scale bar, 100 μm. <b>c</b> Schematic representation of the contribution of lipids derived from CAFs to EMT in CRC.</p>   </div> <p /> <p id="Par16">Collectively, our data provided evidence that CAFs secreted fatty acid that is took up by CRC cells for the synthesis of other lipids, then promoted migration of CRC cells (Fig. <a href="#Fig8">8c</a>).</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par17">CRC is one of the leading causes of cancer death worldwide<sup><a href="#CR23">23</a></sup>. Although the development of CRC has been widely studied, the exact mechanism of CRC metastasis has not been fully clarified. Metabolic interaction between CAFs and CRC cells plays a major role in CRC progression. In this study, we find that lipid metabolism reprogramming occurs in CAFs, mainly in the increase of fatty acids, phospholipids, and glycerides. The fatty acids and phospholipids that are secreted by CAFs are uptake by CRC cells, and then promote CRC cells migration. By knockdown FASN in CAFs or inhibiting CRC cells uptake of fatty acids by SSO in vitro or CD36 monoclonal antibody in vivo, migration promotion effect of CAFs on CRC cells is partially reversed. These findings reveal a novel lipidomic interaction between CAFs and CRC cells, and link this lipids crosstalk with CRC metastasis. This finding highlights a previously unappreciated lipidomic network within CRC, provides a new insight into the mechanism of CRC metastasis.</p> <p id="Par18">In recent years, more and more studies have shown that CAFs communicate with tumor cells via secreting growth factors and producing ECM-degrading proteases<sup><a href="#CR24">24</a></sup>, other studies have shown that metabolic reprogramming of CAFs are important in tumor progress. For example, enhanced glycolysis in CAFs has been discovered in various kinds of cancers, including lung cancer, prostate cancer, melanoma, as well as CRC (refs. <sup><a href="#CR8">8</a>,<a href="#CR25">25</a>–<a href="#CR27">27</a></sup>). Moreover, CAFs can secrete metabolites to fuel the proliferation of tumor cells. Lactate, the product of Warburg effect secreted by CAFs could be absorbed by prostate cancer cells and promote their growth<sup><a href="#CR28">28</a></sup>. PSCs (fibroblasts in pancreatic tumor) support pancreatic cancer metabolism by secreting alanine<sup><a href="#CR29">29</a></sup>. Although numerous efforts have been put on metabolic interaction between CAFs and tumor cells, most of them are considered glucose and glutamine metabolism, few considered lipid metabolism. In this study, we compare the lipidomic profile of CM from three paired NFs and CAFs from patients with CRC. To the best of our knowledge, this study is the first to analyze the lipidomic profile of fibroblasts in CRC. Our results show that the greatest number of lipid changes in the exocrine secretion of CAFs are fatty acyl, and that the number of long-chain and unsaturated fatty acids change most, and the content of phospholipid pathway-related metabolites in CAFs is higher than that in NFs. We speculate that the exocrine lipids of CAFs may affect the metastasis of CRC cells in a direct or indirect way. On the one hand, these elevated lipids may directly promote the growth and metastasis of tumors. We observed an increase in prostaglandins, such as PGE2 (FA 23:3) and it has been reported that PEG2 can promote the formation of CRC (ref. <sup><a href="#CR30">30</a></sup>). Exogenous lipid intake can support the rapid proliferation of tumors as an essential ability of cancer cells<sup><a href="#CR31">31</a></sup>. Fatty acid uptake-related genes, such as caveolin-1 and CD36, are abundantly present in metastatic tumors and have been associated with EMT in multiple cancer<sup><a href="#CR32">32</a></sup>. On the other hand, the elevation of these exogenous lipids may indirectly increase malignant transformation of tumors by creating an immunosuppressive environment. We find that unsaturated fatty acyl substances increase significantly, which may be related to the creation of an immunosuppressive environment. Unsaturated fatty acids have been reported to inhibit the antitumor activity of T cells through CD36/PPAR pathway<sup><a href="#CR33">33</a></sup>.</p> <p id="Par19">We next confirm that small molecule metabolites in CM of CAFs can promote the migration of CRC cells. We find that CRC cells uptake the metabolites secreted by CAFs, including fatty acids, phospholipids, DGs, and cholesterol esters, through co-culture metabolism experiments. They alter tumor metabolism and promote the activation of downstream pathways, such as phospholipids, sphingolipids, and glycerides. Lipid metabolism reprogramming of tumors can affect their own characteristics<sup><a href="#CR34">34</a></sup>. Rapid proliferative tumors need to synthesize large amount of lipids to support the synthesis of organelle biofilms, and also is required to compensate for the lack of energy caused by glucose deficiency through lipid decomposition. In addition, lipids can also promote tumor migration<sup><a href="#CR35">35</a></sup>, such as sphingosine-1-phosphate and fatty acids can promote tumor metastasis ability<sup><a href="#CR36">36</a>,<a href="#CR37">37</a></sup>. In the present study, we find that long-chain and unsaturated lipids change most in CRC cells after co-culturing with CAFs CM. Junjie Li et al. reported that lipid desaturation could promote CSC phenotypes in breast cancer cells, and CSCs are closely related to tumor recurrence and metastasis<sup><a href="#CR38">38</a></sup>. The expression of fatty acid translocase CD36 elevates intracellular fatty acid levels and promotes EMT of hepatocellular carcinoma cells<sup><a href="#CR32">32</a></sup>. These results can provide some support for CAFs to promote tumor development through lipid metabolites.</p> <p id="Par20">Thereafter, we interfere with the expression of CAFs FASN by siRNA or reduce the uptake of fatty acids by CRC cells by SSO in vitro or CD36 monoclonal antibody in vivo, and the migration ability of CRC cells are significantly downregulated. There are studies have demonstrated the important role of fatty acid uptake and CD36 in cancer progress. For example, CD36 deletion restricts fatty acid uptake from the TME, reduces cancer-mediated lipid biosynthesis from fatty acid precursors and the generation of oncogenic lipid signaling pathways, and attenuates cancer growth, exogenous fatty acids are of equal importance as glucose for energy provision and contribute to the production of complex lipids in human prostate cancer. And we also found the protein expression levels of α-SMA in siNC CAFs and siFASN CAFs, showed that FASN knockdown reduced the level of α-SMA in CAFs. These results suggested that a high-fatty acid metabolism and interaction prompted metastasis and FASN of CAFs or CD36 of tumor cells may be two potential targets for anti-metastasis in the future and FASN was a crucial enzyme in CAFs. Previously, FASN, a key regulator of the de novo synthesis of fatty acids, has been widely studied as a target of anticancer metabolic therapy<sup><a href="#CR39">39</a>,<a href="#CR40">40</a></sup>. Overexpression of FASN is associated with poor prognosis and increased multidrug resistance. Efthymia et al. reported FASN inhibition using orlistat is an effective adjunctive treatment for ovarian cancers that have become platinum resistant using a cisplatin-resistant ovarian tumor xenograft model in mice<sup><a href="#CR41">41</a></sup>. Gong et al. reported inhibition of FASN can suppress migration, invasion, and healing of hepatoma carcinoma cells by decreasing HIF-1α and IGFBP1 (ref. <sup><a href="#CR42">42</a></sup>). However, our results suggest that it may not be enough to inhibit the lipid synthesis of tumor cells. The upregulation of FASN in CAFs provides the central precursor of lipid synthesis-fatty acid for cancer cells to promote the synthesis of downstream lipids, such as phospholipids and sphingolipids. Therefore, in the future, the intervention of lipid metabolism in tumor cells may not only focus on the tumor cells, but also consider the fibroblasts.</p> <p id="Par21">Taken together, our results demonstrate novel lipidomic interaction between CAFs and CRCs, and link this lipids crosstalk with CRC metastasis. This finding highlights a previously unappreciated lipidomic network within CRC, which provides new research direction and potential target for clinical anti-metastasis therapy. The mechanism how CRCs use the lipids secreted by CAFs to increase their metastasis ability is worth further study.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="cellculture" title="sec"> <div class="title" tagx="title" title="title">
Cell culture</div> <p id="Par22">Human CRC cell lines DLD1 and HCT-15 were purchased from American Type Culture Collection (ATCC, Rockville, Md.). Cells were recently authenticated by STR profiling. Cells were cultured in DMEM medium (Hyclone, SH30243.01) containing 10% FBS (WISENT, 086–150) at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub>.</p> </div> <div class="isolationandcultureofcafsandnfs" title="sec"> <div class="title" tagx="title" title="title">
Isolation and culture of CAFs and NFs</div> <p id="Par23">To isolate fibroblasts, human CRC tissues and matched NATs were obtained from patients in the West China Hospital, Sichuan University, China (Chengdu, China). The experiments were approved by the Ethics Committee of West China Hospital, and written informed consent was obtained from all patients before surgery. After cleaning and disinfection, the tissues were cut into ~1 mm<sup>3</sup>, then digested with 1 mg/ml collagenase type IV (Sigma Aldrich, C5138–100MG) for 1 h at 37 °C in 5% CO<sub>2</sub>. The mixture was vortexes 30 s for fully digestion every 10 min. After filtration with 70 μm filter, cells were harvested by 1400 r.p.m. centrifugation for 3 min. Red blood cells were lysed by Red Blood Cell Lysis Buffer (Solarbio, R1010) on ice for 3 min, then cells precipitation was collected and seeded into 25-cm<sup>2</sup> culture flask. After 4 days, the medium was replaced with fresh medium to remove non-adherent cells. The doubling time of fibroblasts is ~2–3 days. When passaging, cells were seed with 0% FBS DME/F-12 medium followed by removing non-adherent cells (most are tumor cells) after 30 min. After three passages, pure fibroblasts were obtained and cultured in DMEM medium containing 10% FBS at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub>. The fibroblasts isolated from CRC tissues were defined as CAFs and the fibroblasts from NATs as NFs.</p> </div> <div class="immunofluorescentandihcstaining" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescent and IHC staining</div> <p id="Par24">The purity of fibroblasts was confirmed by immunofluorescence. Briefly, fibroblasts were plated in 24-well plate pre-paved coverslips. After fixation, permeabilization, and blocking, antibodies against human Vimentin (Abcam, ab8978), alpha smooth muscle Actin (α-SMA, Abcam, ab5694), pan Cytokeratin [AE1 + AE3] (Abcam, ab961) were used to incubate overnight. After the secondary antibody was combined, cells were observed under an upright microscope. Cells with strong α-SMA and Vimentin, and no cytokeratin AE1–AE3 fluorescence were considered the pure fibroblasts.</p> <p id="Par25">Human CRC tissues and matched NATs were fixed with 10% formalin. Paraffin-embedded specimens were sectioned at 4 μm thickness. Though dewaxing, 3% hydrogen peroxide for 10 min to exhaust endogenous peroxidase activity, antigen retrieval, and 5% BSA closed, then the sections were incubated with primary antibody α-SMA (Abcam, ab5694), FASN (CST, C20G5) for overnight at 4 °C. Then, the sections were sequentially incubated with the secondary antibody for 40 min, SAB compound, and stained with diaminobenizidine.</p> </div> <div class="collectionofthecm" title="sec"> <div class="title" tagx="title" title="title">
Collection of the CM</div> <p id="Par26">CM was generated by adding fresh 10% FBS DMEM medium to cells grown at ~80% confluence. Medium was harvested 48 h later, then centrifuged at 2000r.p.m at 4 °C for 10 min. The supernatant was collected and passed through 0.22 μm filters. For large molecule cutoff experiments, CM was freezed-thawed in three consecutive cycles of 15 min at −80 °C, and 15 min at 37 °C, and passed through 0.22 μm filters to remove precipitate, then filtered through 3-kDa cutoff columns (EMD Millipore, UFC900308). CM was kept at −80 °C until use. The CMs from CAFs, NFs, and DLD1 were defined as CAFs CM, NFs CM, and DLD1 CM, respectively.</p> </div> <div class="qrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
qRT-PCR</div> <p id="Par27">Total RNA was extracted from CRC cells or fibroblasts using AxyPrep Multisource Total RNA Miniprep Kit (Axygen, P-MN-MS-RNA-50) following the manufacturer’s instructions. Reverse transcription was performed with PrimeScript RT reagent Kit with gDNA Eraser (Takara, RR047A) following the manufacturer’s instructions. qRT-PCR was conducted using iQ SYBR Green Supermix (Bio-rad, 170–8880) according to the manufacturer’s instructions. The PCR primers used in our study were synthesized by Sangon (Shanghai, China) and shown in Supplementary Table <a href="#MOESM2">1</a>.</p> </div> <div class="westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Western blot analysis</div> <p id="Par28">Protein expression was examined by western blot analysis. Briefly, cells receiving indicated treatments was washed twice with ice-cold PBS. Protein was harvested by RIPA lysis and extraction buffer (Thermo Scientific, 89900) added with protease inhibitor cocktail (Thermo Scientific, 78438). Equal amount of proteins was fractionated on 10% SDS–PAGE gel and transferred to polyvinylidence difluoride (PVD) membranes. The PVD membranes were incubated with the indicated primary and secondary antibodies. Proteins were ultimately visualized by enhanced chemiluminescence (Thermo Scientific, 32106) and autoradiography. Antibodies against β-actin, E-cadherin, Vimentin, and FASN were purchased from Abcam (ab8226, ab99359, ab8978, and ab1416).</p> </div> <div class="lipidextraction" title="sec"> <div class="title" tagx="title" title="title">
Lipid extraction</div> <p id="Par29">For cell lipid extraction, cells were washed twice with cold PBS and scrape from flask with PBS followed by centrifugation at 1400 r.p.m. for 7 min. Supernatant were removed and chloroform (300 μL) and methanol (150 μL) were added to the precipitate followed by vigorously mix for 5 min. Waters (135 μL) was added to the mixture. After fully mixing and centrifugation at 14,000 × <i>g</i> for 10 min, the lower layer (organic phase) was collected and dried in a vacuum centrifuge. The samples were stored at −80 °C awaiting lipidomic analysis.</p> <p id="Par30">For CM lipid extraction, CM (600 μL) was added to 2.25 mL methanol/chloroform (v/v, 2:1). After fully mixing, samples were stored at −80 °C for 30 min to improve protein precipitation, then added 0.75 mL chloroform and 0.75 mL water. Samples were mixed by 3 min vortex and then centrifuged at 14,000 × <i>g</i> for 10 min. The lower layer (organic phase) was collected and dried in a vacuum centrifuge. The samples were stored at −80 °C awaiting lipidomic analysis.</p> </div> <div class="lipidomicanalysis" title="sec"> <div class="title" tagx="title" title="title">
Lipidomic analysis</div> <p id="Par31">Lipidomic analysis was performed by using an UPLC-Q-TOF/MS system (Waters Ltd.). The dried samples were redissolved in acetonitrile/isopropanol (v/v, 7:3). The injection volume was fixed at 5 μl, and an ACQUITY UPLC HSS T3 column C<sub>18</sub> CSH column (100 mm × 2.1 mm, 1.7 μm; Waters) was used for separation at 55 °C. Flow rate was 400 μL/min. The mobile phase A consists of acetonitrile/water (v/v, 6:4) mixed with 2 mM ammonium formate and 0.1% formic acid, and mobile phase B isopropanol/ acetonitrile (v/v, 9:1) mixed with 2 mM ammonium formate and 0.1% formic acid. A linear gradient was used as follows: 40–70% B at 0–3 min, 70–95% B at 3–14 min, and 95% B at 14–15.5 min. The column was reequilibrated for 3.5 min, giving a total run 19 min time.</p> <p id="Par32">The MS was operated in the positive and negative modes, respectively. In positive ion mode capillary voltage was set at 3.0 kV and the cone voltage 40 V. In the negative ion mode, the capillary and cone voltage was 2.5 kV and −40V, respectively. The desolvation gas was set to 600 L/h at a temperature of 300 °C; the cone gas was set to 50 L/h and the source temperature was 120 °C.</p> </div> <div class="dataprocessing" title="sec"> <div class="title" tagx="title" title="title">
Data processing</div> <p id="Par33">LC–MS data was processed by the Progenesis QI software (Newcastle, UK). The alignment, peak picking, and identification of lipids were performed. Metabolite annotation was made by searching <i>m</i>/<i>z</i> ratios on two online databases, including Lipid Maps Database (<a href="http://www.lipidmaps.org">www.lipidmaps.org</a>) and the Human Metabolome Database (<a href="http://www.hmdb.ca/">http://www.hmdb.ca/</a>). The data were processed by unsupervised principal component analysis and supervised orthogonal partial least-squares discriminant analysis methods to obtain group clusters. Besides, unpaired Student’s <i>t</i>-test (<i>p</i> &amp;lt; 0.05) to the chemical shifts was also used to assess the significance of each metabolite and <i>p</i> &amp;lt; 0.05 were identified as significant difference.</p> </div> <div class="wound-healingassay" title="sec"> <div class="title" tagx="title" title="title">
Wound-healing assay</div> <p id="Par34">DLD1 cells (1 × 10<sup>6</sup>/well) were seeded into six-well plate in 2 mL 10% FBS DMEM medium. After 24 h, cells were grown to a confluent layer. Then a scratch was made in each well using a 200 μL pipette tip. The scratched cells were removed by gently washing with non-FBS DMEM. Then specific CMs were added to the respective wells and cultured for 48 h. Images were acquired at time point 0 and 48 h.</p> </div> <div class="transwellassay" title="sec"> <div class="title" tagx="title" title="title">
Transwell assay</div> <p id="Par35">DLD1 Cells (1 × 10<sup>5</sup>/well) or HCT-15 Cells (4 × 10<sup>5</sup>/well) were plated on the top of the filter membrane in a transwell insert in 200 μL specific CM. DMEM containing 10% FBS was added to the lower chamber. After 24 h, the non-migrating cells were scraped and the migrating cells were fixed using methanol, stained with 0.1% crystal violet, and photographed under microscope.</p> </div> <div class="assaysfortumorlivermetastasisinvivo" title="sec"> <div class="title" tagx="title" title="title">
Assays for tumor liver metastasis in vivo</div> <p id="Par36">All animals were fed according to the guidelines of the Animal Care and Use Committee of Sichuan University (Chengdu, Sichuan, China). Female BALB/c nude mice were purchased from HFK Biotechnology Company (Beijing, China), and all mice had Animal Quarantine Conformity Certificates. Twelve female BALB/c nude mice (6–8-week-old mice) were randomized in four groups (<i>N</i> = 3). CRC cancer DLD1 cells (1 × 10<sup>6</sup>) were mixed or not mixed with an equal number of CAFs or NFs in 100 μL PBS and injected into the spleen of 7-week-old female nude mice. For the mice injected the mixture of DLD1 and CAFs, intraperitoneal injection of PBS or CD36 monoclonal antibody (JC63.1) at 10 μg was performed thrice weekly for 4 weeks. The animals were euthanized on the 35th day after introduction of xenografts, and the liver of mice was harvested for further research. Paraffin section (4 μm) of the livers were subjected to H&amp;amp;E staining for histological assessment. The investigator who measured tumor metastasis node and H&amp;amp;E staining was unaware of group allocation.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par37">Data are presented as the mean ± SD. Differences between two means were assessed by unpaired and two-tailed Student’s <i>t</i>-test. <i>P</i> values &amp;lt; 0.05 were considered statistically significant.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2020_2434_MOESM1_ESM.docx">LINK</a> <p>Supplementary Figure Legends</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2020_2434_MOESM2_ESM.docx">LINK</a> <p>Supplementary table1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2020_2434_MOESM3_ESM.docx">LINK</a> <p>Supplementary table2</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2020_2434_MOESM4_ESM.tif">LINK</a> <p>Supplementary Fig. 1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2020_2434_MOESM5_ESM.tif">LINK</a> <p>Supplementary Fig. 2</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2020_2434_MOESM6_ESM.tif">LINK</a> <p>Supplementary Fig. 3</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41419_2020_2434_MOESM7_ESM.tif">LINK</a> <p>Supplementary Fig. 4</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>Edited by J.-E. Ricci</p> </div> <div class="fn-type-" title=""> <p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>These authors contributed equally: Jin Gong, Yiyun Lin</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p><b>Supplementary Information</b> accompanies this paper at (10.1038/s41419-020-2434-z).</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This work was supported by Project of the National keypoint research and invention program of China Ministry of Science and Technology (MOST-2016YFC1303200) and National S&amp;amp;T Major project (2018ZX09201018), National Natural Science Foundation of China (81773198).</p> </div> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par38">The authors declare that they have no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siegel</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jemal</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cancer statistics, 2019</span><span class="source" tagx="source" title="source">CA Cancer J. Clin.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="lpage" tagx="lpage" title="lpage">34</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30620402">30620402</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biological heterogeneity and versatility of cancer-associated fibroblasts in the tumor microenvironment</span><span class="source" tagx="source" title="source">Oncogene</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">4887</span><span class="lpage" tagx="lpage" title="lpage">4901</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41388-019-0765-y">10.1038/s41388-019-0765-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30816343">30816343</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franco</span><span class="given-names" tagx="given-names" title="given-names">OE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strand</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayward</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cancer associated fibroblasts in cancer pathogenesis</span><span class="source" tagx="source" title="source">Semin Cell Dev. Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">39</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.semcdb.2009.10.010">10.1016/j.semcdb.2009.10.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19896548">19896548</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shiga</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cancer-associated fibroblasts: their characteristics and their roles in tumor growth</span><span class="source" tagx="source" title="source">Cancers (Basel)</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">2443</span><span class="lpage" tagx="lpage" title="lpage">2458</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/cancers7040902">10.3390/cancers7040902</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26690480">26690480</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cancer associated fibroblasts: an essential role in the tumor microenvironment</span><span class="source" tagx="source" title="source">Oncol. Lett.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">2611</span><span class="lpage" tagx="lpage" title="lpage">2620</span><span class="pub-id"><a href="https://dx.doi.org/10.3892/ol.2017.6497">10.3892/ol.2017.6497</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28927027">28927027</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopes-Coelho</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouveia-Fernandes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serpa</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic cooperation between cancer and non-cancerous stromal cells is pivotal in cancer progression</span><span class="source" tagx="source" title="source">Tumor Biol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">1010428318756203</span><span class="pub-id"><a href="https://dx.doi.org/10.1177/1010428318756203">10.1177/1010428318756203</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhuo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic reprogramming of carcinoma-associated fibroblasts and its impact on metabolic heterogeneity of tumors</span><span class="source" tagx="source" title="source">Semin Cell Dev. Biol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">125</span><span class="lpage" tagx="lpage" title="lpage">131</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.semcdb.2016.11.003">10.1016/j.semcdb.2016.11.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27833036">27833036</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic reprogramming of cancer-associated fibroblasts by IDH3alpha downregulation</span><span class="source" tagx="source" title="source">Cell Rep.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1335</span><span class="lpage" tagx="lpage" title="lpage">1348</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2015.02.006">10.1016/j.celrep.2015.02.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25732824">25732824</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fiaschi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Carbonic anhydrase IX from cancer-associated fibroblasts drives epithelial-mesenchymal transition in prostate carcinoma cells</span><span class="source" tagx="source" title="source">Cell Cycle</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1791</span><span class="lpage" tagx="lpage" title="lpage">1801</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/cc.24902">10.4161/cc.24902</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23656776">23656776</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ko</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Glutamine fuels a vicious cycle of autophagy in the tumor stroma and oxidative mitochondrial metabolism in epithelial cancer cells: implications for preventing chemotherapy resistance</span><span class="source" tagx="source" title="source">Cancer Biol. Ther.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1085</span><span class="lpage" tagx="lpage" title="lpage">1097</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/cbt.12.12.18671">10.4161/cbt.12.12.18671</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22236876">22236876</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabinowitz</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Autophagy and metabolism</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">330</span><span class="fpage" tagx="fpage" title="fpage">1344</span><span class="lpage" tagx="lpage" title="lpage">1348</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1193497">10.1126/science.1193497</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21127245">21127245</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freter</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipid metabolism, apoptosis and cancer therapy</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">924</span><span class="lpage" tagx="lpage" title="lpage">949</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms16010924">10.3390/ijms16010924</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25561239">25561239</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auciello</span><span class="given-names" tagx="given-names" title="given-names">FR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A stromal lysolipid-autotaxin signaling axis promotes pancreatic tumor progression</span><span class="source" tagx="source" title="source">Cancer Discov.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">617</span><span class="lpage" tagx="lpage" title="lpage">627</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/2159-8290.CD-18-1212">10.1158/2159-8290.CD-18-1212</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30837243">30837243</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tabe</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bone marrow adipocytes facilitate fatty acid oxidation activating AMPK and a transcriptional network supporting survival of acute monocytic leukemia cells</span><span class="source" tagx="source" title="source">Cancer Res.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">1453</span><span class="lpage" tagx="lpage" title="lpage">1464</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-16-1645">10.1158/0008-5472.CAN-16-1645</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28108519">28108519</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Targeting galectin-1 in carcinoma-associated fibroblasts inhibits oral squamous cell carcinoma metastasis by downregulating MCP-1/CCL2 expression</span><span class="source" tagx="source" title="source">Clin. Cancer Res.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">1306</span><span class="lpage" tagx="lpage" title="lpage">1316</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/1078-0432.CCR-10-1824">10.1158/1078-0432.CCR-10-1824</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21385934">21385934</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Todaro</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">CD44v6 is a marker of constitutive and reprogrammed cancer stem cells driving colon cancer metastasis</span><span class="source" tagx="source" title="source">Cell Stem Cell</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">342</span><span class="lpage" tagx="lpage" title="lpage">356</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.stem.2014.01.009">10.1016/j.stem.2014.01.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24607406">24607406</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Turnover and flow of the cell membrane for cell migration</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">12970</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-13438-5">10.1038/s41598-017-13438-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29021607">29021607</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keren</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell motility: the integrating role of the plasma membrane</span><span class="source" tagx="source" title="source">Eur. Biophys. J.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">1013</span><span class="lpage" tagx="lpage" title="lpage">1027</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00249-011-0741-0">10.1007/s00249-011-0741-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21833780">21833780</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Abumrad, N. A., EI-Maghrabi, M. R., Amril, E. Z., Lopea E., Grimaldi, P. A. Cloning of a rat adipocyte membrane protein implicated in binding or transport of long-chain fattyacids that is induced during preadipocyte differentiation. <i>J. Biol. Chem.</i><b>268</b>, 17665–17668 (1993).</span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nicholls, H. T. et al. Hematopoietic cell–restricted deletion of CD36 reduces high-fat diet–induced macrophage infiltration and improves insulin signaling in adipose tissue. <i>Diabetes</i><b>60</b>, 1100–1110 (2011).</span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuda</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sulfo-N-succinimidyl oleate (SSO) inhibits fatty acid uptake and signaling for intracellular calcium via binding CD36 lysine 164: SSO also inhibits oxidized low density lipoprotein uptake by macrophages</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">288</span><span class="fpage" tagx="fpage" title="fpage">15547</span><span class="lpage" tagx="lpage" title="lpage">15555</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M113.473298">10.1074/jbc.M113.473298</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23603908">23603908</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hua</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">CD36 mediated fatty acid-induced podocyte apoptosis via oxidative stress</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0127507</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0127507">10.1371/journal.pone.0127507</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26000608">26000608</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Increased mtDNA copy number promotes cancer progression by enhancing mitochondrial oxidative phosphorylation in microsatellite-stable colorectal cancer</span><span class="source" tagx="source" title="source">Signal Transduct. Target Ther.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41392-018-0011-z">10.1038/s41392-018-0011-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29610678">29610678</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuelten</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Breast cancer cells induce stromal fibroblasts to express MMP-9 via secretion of TNF-alpha and TGF-beta</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">118</span><span class="fpage" tagx="fpage" title="fpage">2143</span><span class="lpage" tagx="lpage" title="lpage">2153</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.02334">10.1242/jcs.02334</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15855236">15855236</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavlides</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The reverse Warburg effect: aerobic glycolysis in cancer associated fibroblasts and the tumor stroma</span><span class="source" tagx="source" title="source">Cell Cycle</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">3984</span><span class="lpage" tagx="lpage" title="lpage">4001</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/cc.8.23.10238">10.4161/cc.8.23.10238</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19923890">19923890</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guido</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic reprogramming of cancer-associated fibroblasts by TGF-beta drives tumor growth: connecting TGF-beta signaling with "Warburg-like" cancer metabolism and L-lactate production</span><span class="source" tagx="source" title="source">Cell Cycle</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">3019</span><span class="lpage" tagx="lpage" title="lpage">3035</span><span class="pub-id"><a href="https://dx.doi.org/10.4161/cc.21384">10.4161/cc.21384</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22874531">22874531</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaudhri</span><span class="given-names" tagx="given-names" title="given-names">VK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metabolic alterations in lung cancer-associated fibroblasts correlated with increased glycolytic metabolism of the tumor</span><span class="source" tagx="source" title="source">Mol. Cancer Res.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">579</span><span class="lpage" tagx="lpage" title="lpage">592</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/1541-7786.MCR-12-0437-T">10.1158/1541-7786.MCR-12-0437-T</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23475953">23475953</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fiaschi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Reciprocal metabolic reprogramming through lactate shuttle coordinately influences tumor-stroma interplay</span><span class="source" tagx="source" title="source">Cancer Res.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">5130</span><span class="lpage" tagx="lpage" title="lpage">5140</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-12-1949">10.1158/0008-5472.CAN-12-1949</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22850421">22850421</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sousa</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pancreatic stellate cells support tumour metabolism through autophagic alanine secretion</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">536</span><span class="fpage" tagx="fpage" title="fpage">479</span><span class="lpage" tagx="lpage" title="lpage">483</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature19084">10.1038/nature19084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27509858">27509858</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenhough</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The COX-2/PGE2 pathway: key roles in the hallmarks of cancer and adaptation to the tumour microenvironment</span><span class="source" tagx="source" title="source">Carcinogenesis</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">377</span><span class="lpage" tagx="lpage" title="lpage">386</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/carcin/bgp014">10.1093/carcin/bgp014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19136477">19136477</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nieman</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Adipocytes promote ovarian cancer metastasis and provide energy for rapid tumor growth</span><span class="source" tagx="source" title="source">Nat. Med.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">1498</span><span class="lpage" tagx="lpage" title="lpage">1503</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nm.2492">10.1038/nm.2492</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22037646">22037646</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nath</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic alterations in fatty acid transport and metabolism genes are associated with metastatic progression and poor prognosis of human cancers</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">18669</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep18669">10.1038/srep18669</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26725848">26725848</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Stulnig, T. M. et al. Polyunsaturated fatty acids inhibit T cell signal transduction by modification of detergent-insoluble membrane domains. <i>J. Cell Biol</i>. <b>143</b>, 637–644 (1998).</span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beloribi-Djefaflia</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasseur</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillaumond</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipid metabolic reprogramming in cancer cells</span><span class="source" tagx="source" title="source">Oncogenesis</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e189</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/oncsis.2015.49">10.1038/oncsis.2015.49</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26807644">26807644</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Emerging roles of lipid metabolism in cancer metastasis</span><span class="source" tagx="source" title="source">Mol. Cancer</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">76</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12943-017-0646-3">10.1186/s12943-017-0646-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28399876">28399876</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visentin</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Validation of an anti-sphingosine-1-phosphate antibody as a potential therapeutic in reducing growth, invasion, and angiogenesis in multiple tumor lineages</span><span class="source" tagx="source" title="source">Cancer Cell</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">225</span><span class="lpage" tagx="lpage" title="lpage">238</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ccr.2006.02.023">10.1016/j.ccr.2006.02.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16530706">16530706</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pascual</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Targeting metastasis-initiating cells through the fatty acid receptor CD36</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">541</span><span class="fpage" tagx="fpage" title="fpage">41</span><span class="lpage" tagx="lpage" title="lpage">45</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature20791">10.1038/nature20791</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27974793">27974793</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Lipid desaturation is a metabolic marker and therapeutic target of ovarian cancer stem cells</span><span class="source" tagx="source" title="source">Cell Stem Cell</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">303</span><span class="lpage" tagx="lpage" title="lpage">314 e305</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.stem.2016.11.004">10.1016/j.stem.2016.11.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28041894">28041894</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alwarawrah</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Fasnall, a selective FASN inhibitor, shows potent anti-tumor activity in the MMTV-Neu model of HER2(+) breast cancer</span><span class="source" tagx="source" title="source">Cell. Chem. Biol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">678</span><span class="lpage" tagx="lpage" title="lpage">688</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chembiol.2016.04.011">10.1016/j.chembiol.2016.04.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27265747">27265747</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">LEE, J. S., LEE, M. S., OH, W. K., SUL, J. Y. Fatty acid synthase inhibition by amentoflavone induces apoptosis and antiproliferation in human breast cancer cells. <i>Biol. Pharm. Bull.</i><b>32</b>, 1427–1432 (2009).</span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papaevangelou</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almeida</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Box</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">deSouza</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span></span><span class="mixed-article-title" title="mixed-article-title">The effect of FASN inhibition on the growth and metabolism of a cisplatin-resistant ovarian carcinoma model</span><span class="source" tagx="source" title="source">Int. J. Cancer</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">143</span><span class="fpage" tagx="fpage" title="fpage">992</span><span class="lpage" tagx="lpage" title="lpage">1002</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/ijc.31392">10.1002/ijc.31392</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29569717">29569717</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of FASN suppresses migration, invasion and growth in hepatoma carcinoma cells by deregulating the HIF-1alpha/IGFBP1 pathway</span><span class="source" tagx="source" title="source">Int. J. Oncol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">883</span><span class="lpage" tagx="lpage" title="lpage">892</span><span class="pub-id"><a href="https://dx.doi.org/10.3892/ijo.2017.3867">10.3892/ijo.2017.3867</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28197637">28197637</a></span></span></li> </ul> </div> </div>  </body></html>