<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Mol Cell</span><span class="iso-abbrev" title="iso-abbrev">Mol. Cell</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Molecular Cell</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1097-2765</span><span class="issn-epub" tagx="issn" title="issn-epub">1097-4164</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cell Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6375727">6375727</a></span><span class="publisher-id" title="publisher-id">S1097-2765(18)30983-3</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.molcel.2018.11.013">10.1016/j.molcel.2018.11.013</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Noncanonical Modulation of the eIF2 Pathway Controls an Increase in Local Translation during Neural Wiring</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagnetta</span><span class="given-names" tagx="given-names" title="given-names">Roberta</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">Hovy Ho-Wai</span></span><a href="#aff1">1</a><a href="#fn1">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frese</span><span class="given-names" tagx="given-names" title="given-names">Christian K.</span></span><a href="#aff2">2</a><a href="#aff3">3</a><a href="#aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallucci</span><span class="given-names" tagx="given-names" title="given-names">Giovanna R.</span></span><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krijgsveld</span><span class="given-names" tagx="given-names" title="given-names">Jeroen</span></span><a href="#aff2">2</a><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">Christine E.</span></span><span class="email" tagx="email" title="email">ceh33@cam.ac.uk</span><a href="#aff1">1</a><a href="#fn2">7</a><a href="#cor1">∗</a></span></div><span class="citation_author_institution" id="aff1">[1], </span><span class="citation_author_institution" id="aff2">[2], </span><span class="citation_author_institution" id="aff3">[3], </span><span class="citation_author_institution" id="aff4">[4], </span><span class="citation_author_institution" id="aff5">[5], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">∗</span>Corresponding author <span class="email" tagx="email" title="email">ceh33@cam.ac.uk</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">6</span><p id="ntpara0010">Present address: Centre of Research in Neuroscience, The Research Institute of the McGill University Health Centre, Montreal General Hospital, Montréal, QC H3A 2B4, Canada</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">7</span><p id="ntpara0015">Lead Contact</p> </div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-2-2</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-2-2</span></span><span class="volume" tagx="volume" title="volume">73</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">474</span><span class="lpage" tagx="lpage" title="lpage">489.e5</span><span class="history" title="history"><span class="received" title="received">received: 2018-4-20</span><span class="rev-recd" title="rev-recd">rev-recd: 2018-9-18</span><span class="accepted" title="accepted">accepted: 2018-11-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/4.0/).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Summary</div> <p>Local translation is rapidly regulated by extrinsic signals during neural wiring, but its control mechanisms remain elusive. Here we show that the extracellular cue Sema3A induces an initial burst in local translation that precisely controls phosphorylation of the translation initiation factor eIF2α via the unfolded protein response (UPR) kinase PERK. Strikingly, in contrast to canonical UPR signaling, Sema3A-induced eIF2α phosphorylation bypasses global translational repression and underlies an increase in local translation through differential activity of eIF2B mediated by protein phosphatase 1. Ultrasensitive proteomics analysis of axons reveals 75 proteins translationally controlled via the Sema3A-p-eIF2α pathway. These include proteostasis- and actin cytoskeleton-related proteins but not canonical stress markers. Finally, we show that PERK signaling is needed for directional axon migration and visual pathway development <i>in vivo</i>. Thus, our findings reveal a noncanonical eIF2 signaling pathway that controls selective changes in axon translation and is required for neural wiring.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Graphical Abstract</div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Highlights</div> <p> </p><ul> <li><span class="label" tagx="label" title="label">•</span><p id="p0010">eIF2α phosphorylation underlies Sema3A-induced upregulation of global translation</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0015">pSILAC-SP3 reveals 75 nascent proteins regulated by the Sema3A-p-eIF2α pathway</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0020">eIF2B modulation is the key node between Sema3A and canonical stress responses</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0025">PERK signaling is required for axon guidance and neural connectivity</p> </li> </ul> <p /> </div> <div class="abstract" title="abstract"> <p>Cagnetta et al. find that Sema3A triggers phosphorylation of eIF2α in axons through an initial wave of local protein synthesis. eIF2α phosphorylation upregulates global translation via differential eIF2B activity and is required for neural wiring. Proteomics analysis identifies the axonal translational changes regulated by the Sema3A-p-eIF2α pathway.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
unfolded protein response</div> <div class="kwd" title="kwd">
Sema3A</div> <div class="kwd" title="kwd">
PERK</div> <div class="kwd" title="kwd">
eIF2α</div> <div class="kwd" title="kwd">
eIF2B</div> <div class="kwd" title="kwd">
local translation</div> <div class="kwd" title="kwd">
axon</div> <div class="kwd" title="kwd">
retinal ganglion cell</div> <div class="kwd" title="kwd">
axon guidance</div> <div class="kwd" title="kwd">
axon branching</div> </div> </div> <p id="misc0010">Published: December 27, 2018</p> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="p0030">Precise connectivity between neurons is needed to generate operative nervous systems. The initial assembly of neural circuits is mediated by the growth cone, a specialized structure at the tip of a growing axon that senses extracellular cues along the pathway and transduces them into directional changes, thus navigating to its synaptic partner (<a href="#bib49">Stoeckli, 2018</a>, <a href="#bib23">Jung et al., 2012</a>). When at the target, axons elaborate highly branched arbors and form synapses. The growth cone is often far from the soma, and local mRNA translation mediates its rapid responses to extracellular cues (<a href="#bib4">Campbell and Holt, 2001</a>, <a href="#bib23">Jung et al., 2012</a>). Extrinsic cues, such as Semaphorin-3A (Sema3A), specifically remodel the nascent axonal proteome within just 5 min and orchestrate further dynamic changes over the next 25 min (<a href="#bib3">Cagnetta et al., 2018</a>). However, the translational mechanisms that control the cue-induced local nascent proteome remain elusive.</p> <p id="p0035">One way to control translation is to modulate one or more components of the translational machinery. For instance, the α subunit of eukaryotic initiation factor 2 (eIF2α) mediates translational regulation in response to stress. eIF2α, guanosine triphosphate (GTP), and the methionyl initiator tRNA constitute the ternary complex (eIF2·GTP·tRNA<sub>i</sub><sup>Met</sup>) that is delivered to the ribosome. As GTP is hydrolyzed during each round of translation initiation, eIF2 recharges via the guanine nucleotide exchange factor (GEF) eIF2B (<a href="#bib59">Webb and Proud, 1997</a>). Stress stimuli elicit phosphorylation of eIF2α at Ser51 via four possible kinases, including the PKR-like endoplasmic reticulum kinase (PERK) (<a href="#bib21">Holcik and Sonenberg, 2005</a>). Upon phosphorylation, p-eIF2α binds to and inhibits its own GEF, eIF2B, whose concentration is much lower than that of eIF2. Therefore, eIF2B can no longer return p-eIF2 to its active GTP-bound state (<a href="#bib59">Webb and Proud, 1997</a>). This causes a reduction of the ternary complex available to reinitiate translation, which represses the translation of most mRNAs and selectively upregulates a small subset of mRNAs (∼2.5% of total mRNAs; <a href="#bib13">Dang Do et al., 2009</a>). This mechanism allows the cell to conserve resources and to translate transcripts involved in the cytoprotective response or, when the stress is prolonged, in apoptosis (<a href="#bib21">Holcik and Sonenberg, 2005</a>). For instance, endoplasmic reticulum (ER) stress phosphorylates eIF2α via PERK to turn on the unfolded protein response (UPR), maintaining the homeostasis of the protein folding environment within the ER (<a href="#bib38">Pavitt and Ron, 2012</a>). Interestingly, Semaphorin signaling has been shown to govern epidermal morphogenesis via eIF2α dephosphorylation in <i>C. elegans</i> (<a href="#bib36">Nukazuka et al., 2008</a>), raising the possibility that Sema3A similarly employs the eIF2 pathway to control local translation-dependent axon guidance in vertebrate neurons.</p> <p id="p0040">Here we investigate the role of eIF2α in regulating the nascent proteome in the axonal compartment of retinal ganglion cells (RGCs) in response to Sema3A. Our findings reveal a noncanonical PERK-p-eIF2α signaling pathway that underlies the Sema3A-induced increase in local protein synthesis and is required for neural wiring. Further, our results identify eIF2B modulation as a pivotal switch between the responses to stress and Sema3A.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="sema3ainduceseif2αphosphorylationinaxons" title="sec"> <div class="title" tagx="title" title="title">
Sema3A Induces eIF2α Phosphorylation in Axons</div> <p id="p0045">The extracellular cue Sema3A induces protein synthesis-dependent chemotropic responses in axonal growth cones, peaking 10 min after stimulation (<a href="#bib4">Campbell and Holt, 2001</a>, <a href="#bib6">Campbell et al., 2001</a>). Sema governs epidermal morphogenesis via eIF2α dephosphorylation in <i>C. elegans</i> (<a href="#bib36">Nukazuka et al., 2008</a>), prompting us to ask whether Sema3A similarly modulates eIF2α phosphorylation in axons. Quantitative immunofluorescence (qIF) revealed that Sema3A induces a significant increase in the p-eIF2α signal, but not in total-eIF2α, in retinal growth cones following 10 min stimulation (<a href="#fig1">Figures 1</a>A and 1B). The direction of the Sema-induced change in p-eIF2α was unexpectedly opposite to that seen in epidermal cells (<a href="#bib36">Nukazuka et al., 2008</a>) and was reminiscent of the p-eIF2α increase seen in the stress response. As a positive control, we compared the p-eIF2α signal in growth cones after stimulation with Sema3A versus treatment with the ER stress-inducing agent thapsigargin (Tg), an inhibitor of the sarco-endoplasmic reticulum Ca<sup>2+</sup> ATPase (<a href="#bib58">Vuppalanchi et al., 2012</a>). Consistent with data from fibroblasts (<a href="#bib43">Sadighi Akha et al., 2011</a>), a 15 min treatment with Tg induced an increase in p-eIF2α, but not total-eIF2α, in axons (<a href="#fig1">Figures 1</a>A and 1B). Interestingly, in contrast to increased p-eIF2α levels that persist for hours in UPR signaling (<a href="#bib43">Sadighi Akha et al., 2011</a>), the increase with Sema3A treatment was rapid and transient, lasting minutes (<a href="#mmc1">Figure S1</a>A). These data reveal that the physiological extracellular cue Sema3A triggers rapid and transient phosphorylation of eIF2α in axons. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>eIF2α Phosphorylation Underlies Sema3A-Induced Upregulation of Axonal Protein Synthesis</p> <p>(A and B) IF representative images (A) and quantification (B) for total-eIF2α and p-eIF2α in growth cones treated with Tg (15 min) or Sema3A (10 min) (unpaired t test).</p> <p>(C and D) IF representative images (C) and quantification (D) for puromycin in growth cones incubated with puromycin and co-treated with Tg (15 min) or Sema3A (10 min) and ISRIB (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>Error bars indicate SEM. Scale bars, 5 μm. See also <a href="#mmc1">Figure S1</a>.</p>   </div> <p /> </div> <div class="eif2αphosphorylationdifferentiallyregulatestranslationinastimulus-specificmanner" title="sec"> <div class="title" tagx="title" title="title">
eIF2α Phosphorylation Differentially Regulates Translation in a Stimulus-Specific Manner</div> <p id="p0050">Sema3A increases global translation locally in retinal axons (<a href="#bib4">Campbell and Holt, 2001</a>, <a href="#bib65">Yoon et al., 2012</a>). However, paradoxically, Sema3A stimulation results in increased p-eIF2α, which is known to repress global translation (<a href="#bib21">Holcik and Sonenberg, 2005</a>). Therefore, we next explored the role of p-eIF2α on Sema3A-induced global translation in growth cones. To this end, newly synthesized proteins (NSPs) were tagged by puromycin pulse labeling (<a href="#bib44">Schmidt et al., 2009</a>). We stimulated with either Sema3A or the ER stressors Tg and DTT and co-treated with the pharmacological reagent integrated stress response inhibitor (ISRIB). ISRIB stabilizes eIF2B, making eIF2B’s GEF activity resistant to the effects of p-eIF2α without directly affecting eIF2α phosphorylation (<a href="#bib47">Sidrauski et al., 2013</a>, <a href="#bib48">Sidrauski et al., 2015</a>, <a href="#bib53">Tsai et al., 2018</a>). The released truncated puromycilated proteins were then quantified by IF using an anti-puromycin antibody. In accord with previous findings in whole cells (<a href="#bib47">Sidrauski et al., 2013</a>), Tg and DTT induced a decrease in the puromycin signal, signifying a decrease in global translation in the growth cone, which was blocked by ISRIB (<a href="#fig1">Figures 1</a>C, 1D, <a href="#mmc1">S1</a>B, and S1C). Surprisingly, ISRIB completely abolished the Sema3A-induced increase in global translation, indicating that eIF2α phosphorylation also underlies the Sema3A-induced increase in protein synthesis in axons (<a href="#fig1">Figures 1</a>C and 1D). These results confirm that the stress response is conserved in axons (<a href="#bib58">Vuppalanchi et al., 2012</a>) and validate the mechanism of action of ISRIB in our system. Remarkably, they reveal that eIF2α phosphorylation can differentially regulate translation in a stimulus-specific manner.</p> </div> <div class="sema3aregulatesaspecificsubsetofaxonalnspsviaeif2αphosphorylation" title="sec"> <div class="title" tagx="title" title="title">
Sema3A Regulates a Specific Subset of Axonal NSPs via eIF2α Phosphorylation</div> <p id="p0055">We next investigated the proteins translationally regulated in the axon compartment of a single neuronal type (RGC) via the Sema3A-p-eIF2α pathway by employing pulsed stable isotope labelling of amino acids in cell culture together with single-pot solid phase-enhanced sample preparation (pSILAC-SP3; <a href="#bib22">Hughes et al., 2014</a>, <a href="#bib67">Hughes et al., 2018</a>, <a href="#bib3">Cagnetta et al., 2018</a>). RGC axons grown on transwell filters were incubated for 1 hr in depletion medium depleted of lysine and arginine and then severed from their cell bodies. Somaless axons were incubated for 15 min with Sema3A and "heavy" isotope-coded amino acids (Lys8 and Arg10) or with Sema3A, ISRIB, and "medium" isotope-coded amino acids (Lys4 and Arg6) (<a href="#fig2">Figure 2</a>A). ISRIB makes eIF2B insensitive to p-eIF2α, focusing the window of sensitivity of pSILAC-SP3 on the axonal NSPs regulated by the Sema3A-p-eIF2α pathway. pSILAC-SP3 revealed 75 significant NSP changes mediated by Sema3A-p-eIF2α signaling (<a href="#fig2">Figure 2</a>B; <a href="#mmc2">Table S1</a>). Intriguingly, Atf4 mRNA is resident in axons (<a href="#bib66">Zivraj et al., 2010</a>), and the upstream open reading frames (uORFs) previously detected in the mouse are conserved in its 5′ UTR of <i>Xenopus laevis</i> and of mouse axons (<a href="#mmc1">Figures S2</a>A and S2B). This leads to the prediction that Atf4 is upregulated when eIF2α is phosphorylated, and the level of ternary complex available to reinitiate translation decreases (<a href="#bib55">Vattem and Wek, 2004</a>). However, no upregulation of this classical stress marker was identified downstream of Sema3A-p-eIF2α signaling (<a href="#fig2">Figure 2</a>B; <a href="#mmc2">Table S1</a>). This result was also confirmed by puromycilation of NSPs and proximity ligation assay (puro-PLA; <a href="#bib51">tom Dieck et al., 2015</a>) in the presence of Sema3A, whereas Tg treatment increased the Atf4 puro-PLA signal (<a href="#mmc1">Figures S2</a>C and S2D), suggesting that the Sema3A-p-eIF2α pathway generates a level of ternary complex higher than the canonical stress response. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>pSILAC-SP3 Reveals 75 Nascent Proteins Regulated by the Sema3A-p-eIF2α Pathway</p> <p>(A) Schematic of the pSILAC-SP3 methodology applied to somaless retinal axons.</p> <p>(B) Subset of NSPs regulated in response to Sema3A by p-eIF2α. Only significant NSP changes are shown (blue, downregulation; red, upregulation; p &amp;lt; 0.10).</p> <p>(C) KEGG pathway analysis (red, upregulated pathway; blue, downregulated pathway; cutoff ≥ 2 proteins per pathway).</p> <p>(D) Network-based cluster analysis of the enriched NSP changes induced by Sema3A-p-eIF2α signaling and their associated functional classes (blue nodes, downregulated NSPs; red nodes, upregulated NSPs; light blue edges, interactions known from databases; purple edges, interactions experimentally determined; green stars, NSPs belonging to the "response to stress" category; false discovery rate [FDR] &amp;lt; 0.05).</p> <p>(E and F) IF representative images (E) and quantification (F) for Gsn, Actb, and RpL7a in growth cones pre-incubated with ISRIB for 30 min and co-stimulated with Sema3A for 15 min (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(G) Correlation analysis of pSILAC- and qIF-derived detection of protein changes (r = 0.92).</p> <p>Error bars indicate SEM. Scale bar, 5 μm. See also <a href="#mmc1">Figure S2</a> and <a href="#mmc2">Table S1</a>.</p>   </div> <p /> <p id="p0060">Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway and functional gene ontology (GO) enrichment analyses revealed that the axonal NSPs regulated by Sema3A-p-eIF2α signaling encompass several functions (<a href="#fig2">Figures 2</a>C, 2D, and <a href="#mmc1">S2</a>E). eIF2α phosphorylation upregulates the KEGG pathway "protein processing in ER," including Canx and Vcp, which are involved in protein folding and quality control (<a href="#fig2">Figure 2</a>C). NSPs belonging to "metabolic pathways," including proteins for the biosynthesis of amino acids, showed enhanced local translation (<a href="#fig2">Figure 2</a>C). Furthermore, eIF2α phosphorylation emerged to control the translation of some proteasomal subunits and ribosomal proteins (<a href="#fig2">Figures 2</a>C and <a href="#mmc1">S2</a>E), possibly to remodel pre-existing proteasomes and ribosomes and confer substrate-specific functions (<a href="#bib45">Shi et al., 2017</a>, <a href="#bib37">Padmanabhan et al., 2016</a>). The Sema3A-p-eIF2α pathway also upregulates the "barbed-end actin filament" GO term (<a href="#mmc1">Figure S2</a>E), including the actin-binding protein Gelsolin, which has been shown previously to mediate filopodium retraction (<a href="#bib26">Lu et al., 1997</a>). The translational upregulation of Gelsolin and of the ribosomal protein RpL7a concur with their mRNA presence in embryonic axons (<a href="#mmc2">Table S1</a>; <a href="#bib18">Gumy et al., 2011</a>) and were validated by qIF (<a href="#fig2">Figures 2</a>E and 2F). Network-based functional enrichment analysis revealed that some NSP changes constitute functionally coherent sets of mRNAs undergoing differential translation regulation (<a href="#fig2">Figure 2</a>D). Interestingly, two upregulated NSPs, Hspd1 and Vcp, belong to the "response to misfolded proteins" GO term, and 31 NSPs are members of the "response to stress" GO term (<a href="#fig2">Figure 2</a>D). The Sema3A-p-eIF2α pathway also regulates several NSPs involved in "organization of the actin cytoskeleton," including β-actin (<a href="#fig2">Figure 2</a>D), whose mRNA is present in retinal axons (<a href="#mmc2">Table S1</a>; <a href="#bib66">Zivraj et al., 2010</a>) and whose downregulation was validated by qIF (<a href="#fig2">Figures 2</a>E and 2F). Collectively, comparison of the protein changes detected by pSILAC-SP3 versus the ones detected by qIF showed a strong positive correlation (r = 0.92; <a href="#fig2">Figure 2</a>G).</p> <p id="p0065">Next we tested whether the NSPs controlled downstream of the Sema3A-p-eIF2α pathway are predicted to be regulated by other <i>trans</i>-acting translational regulators. We performed an upstream regulator analysis based on previous datasets identifying targets of different translational regulators, including two canonical stress responses characterized in mouse liver downstream of the kinases PERK and GCN2. Only a very small fraction of targets was shared between Sema3A-p-eIF2α signaling and 4 of 8 translational regulators (PERK, GCN2, mTOR, and Apc) and more than 85% of the NSPs were predicted to be specifically regulated by the Sema3A-p-eIF2α pathway (<a href="#mmc1">Figure S2</a>F). Alternatively, differences between experimental systems (e.g., transcriptome specificity) might contribute to this limited overlap. These results identify the subset of axonal NSPs selectively regulated by the Sema3A-p-eIF2α pathway.</p> </div> <div class="sema3a-inducedinitialwaveoflocaltranslationtriggerseif2αphosphorylationviaperk" title="sec"> <div class="title" tagx="title" title="title">
Sema3A-Induced Initial Wave of Local Translation Triggers eIF2α Phosphorylation via PERK</div> <p id="p0070">Four kinases are known to phosphorylate eIF2α (PERK, PKR, HRI, and GCN2) (<a href="#bib21">Holcik and Sonenberg, 2005</a>), prompting us to ask whether any of them are resident in retinal axons. PERK, PKR, and HRI are annotated in <i>X. laevis</i>, and IF indicated their presence in retinal axons (<a href="#fig3">Figure 3</a>A). Next, we asked which kinase is involved and how it is activated downstream of Sema3A. Within just 5 min (i.e., before eIF2α is phosphorylated; <a href="#mmc1">Figure S1</a>A), Sema3A upregulates more than 60 significant axonal NSP changes without apparent links to proteostasis (<a href="#bib3">Cagnetta et al., 2018</a>). By contrast, at 15 min (i.e., after eIF2α is phosphorylated; <a href="#mmc1">Figure S1</a>A), the Sema3A-p-eIF2α pathway upregulates NSPs linked to the biosynthesis of amino acids (e.g., Pkm), ER and mitochondrion protein quality control (e.g., the transitional ER ATPase Vcp), and chaperones (e.g., the ER chaperone Canx) (<a href="#fig2">Figures 2</a>B and 2D). This suggests a sequence of events in which an initial p-eIF2α-independent wave of translation places a burden on the ER, which, in turn, activates the kinase PERK to trigger a counteracting translational control program to preserve proteostasis. To test this hypothesis, we first blocked the Sema3A-induced burst in translation with the protein synthesis inhibitor cycloheximide (CHX) and immunostained for total-eIF2α and p-eIF2α. Remarkably, CHX completely blocked the Sema3A-induced phosphorylation of eIF2α without affecting the level of total-eIF2α (<a href="#fig3">Figures 3</a>B and 3C), indicating that upregulation of local translation is required for eIF2α phosphorylation. In particular, we tested whether the initial wave of local translation mediated by mTOR and ERK-1/2 (extracellular signal-regulated kinase, also known as mitogen-activated protein kinases p42 and p44), already active at 5 min of Sema3A stimulation (<a href="#bib4">Campbell and Holt, 2001</a>, <a href="#bib5">Campbell and Holt, 2003</a>), underlies the phosphorylation of eIF2α. Co-treatment of axons with Sema3A and the mTOR inhibitor PP242 or the ERK-1/2 inhibitor U0126 completely inhibited phosphorylation of eIF2α (<a href="#fig3">Figures 3</a>B and 3C), supporting the hypothesis. Each inhibitor on its own was sufficient to block eIF2α phosphorylation, suggesting that there is crosstalk between the ERK-1/2 and the mTOR pathways, in line with previous observations (<a href="#bib33">Mendoza et al., 2011</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Sema3A-Induced Initial Wave of Local Protein Synthesis Elicits eIF2α Phosphorylation via PERK</p> <p>(A) Retinal axons were immunostained for PERK, HRI, and PKR.</p> <p>(B and C) IF representative images (B) and quantification (C) for total-eIF2α and p-eIF2α in growth cones co-treated with Sema3A and CHX, PP242, or U0126 for 10 min (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(D and E) IF representative images (D) and quantification (E) for p-eIF2α in growth cones co-treated with Sema3A and GSK2606414 (GSK) for 10 min (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(F and G) IF representative images (F) and quantification (G) for puromycin in growth cones co-incubated with puromycin, Sema3A and GSK for 10 min (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(H and I) IF representative images (H) and quantification (I) for puromycin in growth cones of embryos injected with CoMO or PERK MO co-incubated with puromycin and Sema3A for 10 min (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>Error bars indicate SEM. Scale bars, 5 μm. See also <a href="#mmc1">Figure S3</a>.</p>   </div> <p /> <p id="p0075">Finally, we stimulated axons with Sema3A in the presence of the PERK inhibitor GSK2606414 (GSK) and performed qIF for p-eIF2α. GSK blocked eIF2α phosphorylation (<a href="#fig3">Figures 3</a>D and 3E), revealing that PERK is activated by Sema3A. Consistent with our previous findings (<a href="#fig1">Figures 1</a>C and 1D), puromycilating the NSPs and co-treating with Sema3A and GSK phenocopied the effect of ISRIB by abolishing the Sema3A-induced increase in global translation (<a href="#fig3">Figures 3</a>F and 3G). To further verify these results, we knocked down PERK in embryos with a morpholino (MO), which resulted in ∼60% knockdown (KD) (<a href="#mmc1">Figure S3</a>), and we assayed global translation in response to Sema3A stimulation. The Sema3A-induced increase in global translation was completely inhibited in PERK morphants (<a href="#fig3">Figures 3</a>H and 3I). Thus, PERK can be activated under physiological conditions following an initial wave of local translation and is a crucial component of the Sema3A pathway to upregulate local protein synthesis.</p> </div> <div class="localtranslationanddephosphorylationofeif2bεdistinguishthesema3a-inducedresponsefromthecanonicalstressresponse" title="sec"> <div class="title" tagx="title" title="title">
Local Translation and Dephosphorylation of eIF2Bε Distinguish the Sema3A-Induced Response from the Canonical Stress Response</div> <p id="p0080">Our findings revealed that the Sema3A response and the canonical UPR are both mediated by phosphorylation of eIF2α. Therefore, we investigated the mechanism underlying the differential translational control downstream of these two stimuli. p-eIF2α induced by Sema3A signaling, unlike the canonical UPR, does not induce global translational repression (<a href="#fig1">Figures 1</a>C and 1D) nor the translation of classical stress markers such as Atf4 (<a href="#fig2">Figures 2</a>B and <a href="#mmc1">S2</a>A–S2D), both of which are triggered by low levels of ternary complex. eIF2 recharges with GTP by eIF2B, which constitutes a rate-limiting factor for ternary complex availability (<a href="#bib59">Webb and Proud, 1997</a>). Therefore, we hypothesized that, while inducing phosphorylation of eIF2α, Sema3A may concomitantly modulate the GEF activity of eIF2B to alter the probability of generating the ternary complex. We reasoned that the modulation of eIF2B activity could be achieved in two non-mutually exclusive ways: by increasing the total amount of eIF2B available and by adjusting the phosphorylation level of a conserved Ser residue on the subunit ε of eIF2B, wherein phosphorylation decreases eIF2B activity, whereas dephosphorylation increases eIF2B activity (<a href="#bib60">Welsh et al., 1998</a>, <a href="#bib40">Quevedo et al., 2003</a>). We first examined the level of total eIF2Bε in the growth cone following Sema3A versus Tg treatment. qIF showed a more than 35% increase in total eIF2Bε in response to 10 min Sema3A but not Tg (<a href="#fig4">Figures 4</a>A and 4B). This rapid increase could result from local translation because eIF2Bε mRNA resides in retinal axons (<a href="#fig4">Figure 4</a>C). CHX blocked the Sema3A-induced increase in eIF2Bε (<a href="#fig4">Figures 4</a>A and 4B), indicating that eIF2Bε, unlike eIF2α (<a href="#fig3">Figures 3</a>B and 3C), is locally translated in response to Sema3A. This finding is in accord with the detection of eIF2Bε mRNA translation in mouse retinal axons in the tectum <i>in vivo</i>, where Sema3A is expressed (<a href="#bib46">Shigeoka et al., 2016</a>). Like reticulocytes, where the eIF2:eIF2B ratio is 7:1 (<a href="#bib59">Webb and Proud, 1997</a>), eIF2Bε is much less abundant than eIF2α in axons, as indicated by the inability of the pSILAC-SP3 approach to detect either nascent or pre-existing eIF2Bε protein, whereas eIF2α protein is readily detected (<a href="#bib3">Cagnetta et al., 2018</a>). Because Sema3A is known to activate axonal mTOR and ERK-1/2 within 5 min stimulation (<a href="#bib4">Campbell and Holt, 2001</a>, <a href="#bib5">Campbell and Holt, 2003</a>), we asked whether these translational regulators control eIF2Bε rapid local translation. The results showed that eIF2Bε increases within just 5 min of Sema3A stimulation, but this is not mediated by mTOR or ERK-1/2 (<a href="#mmc1">Figures S4</a>A and S4B). The negative control showed that co-treatment with ISRIB does not affect the eIF2Bε increase (<a href="#mmc1">Figures S4</a>A and S4B), consistent with the lack of eIF2α phosphorylation at 5 min stimulation (<a href="#mmc1">Figure S1</a>A). Upstream regulator analysis based on previous datasets identifying the targets of several translational regulators predicted that Apc, Mena, Fmrp, Tdp43, Fus, or mTOR do not control eIF2Bε translation. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Sema3A and Canonical UPR Signaling Differentially Control Translation by Distinct Modulation of eIF2B</p> <p>(A and B) IF representative images (A) and quantification (B) for eIF2Bε in growth cones co-treated with Sema3A and CHX (10 min) or Tg (15 min) (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(C) RT-PCR for Actb (positive control; <a href="#bib54">Turner-Bridger et al., 2018</a>), Brn3a (negative control; <a href="#bib65">Yoon et al., 2012</a>), and eIF2Bε mRNAs.</p> <p>(D and E) IF representative images (D) and quantification (E) for p-eIF2Bε (Ser539) in growth cones co-treated with Sema3A and tautomycin (TM) or U0126 (10 min) or treated with Tg (15 min) (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(F and G) IF representative images (F) and quantification (G) for puromycin in growth cones co-incubated with puromycin, Sema3A and TM for 10 min (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(H and I) Immunoblot (H) and quantification (I) of puromycin signal intensity in lysates of intact brains of stage 35-36 embryos incubated with TM for 30 min and puromycilated over the last 15 min of the treatment (unpaired t test).</p> <p>Error bars indicate SEM. Scale bars, 5 μm. See also <a href="#mmc1">Figure S4</a>.</p>   </div> <p /> <p id="p0085">We next examined the level of phosphorylation of eIF2Bε, which regulates eIF2B activity. We stimulated with Sema3A for 10 min and immunostained growth cones for p-eIF2Bε (Ser539). qIF showed an ∼35% decrease in p-eIF2Bε in response to Sema3A but not Tg (<a href="#fig4">Figures 4</a>D and 4E). Previous work in rat cortical neurons has shown that dephosphorylation of p-eIF2Bε can be mediated by protein phosphatase 1 (PP1) activation following its interaction with ERK-1/2 (<a href="#bib40">Quevedo et al., 2003</a>), which constitutes a major component of the Sema3A intracellular pathway (<a href="#bib5">Campbell and Holt, 2003</a>). Therefore, we tested whether PP1 is responsible for eIF2Bε dephosphorylation by co-treating with Sema3A and tautomycin (TM), which preferentially inhibits PP1 (<a href="#bib29">MacKintosh and MacKintosh, 1994</a>). TM abolished the Sema3A-induced dephosphorylation of eIF2Bε (<a href="#fig4">Figures 4</a>D and 4E), revealing that Sema3A activates PP1 to mediate eIF2Bε dephosphorylation. In contrast, PP1 does not regulate the eIF2α phosphorylation level (<a href="#mmc1">Figures S4</a>C and S4D), indicating a lack of involvement of the PP1-eIF2α-recruiting scaffold protein GADD34 (growth arrest and DNA damage-inducible protein) at 10 min Sema3A stimulation (<a href="#bib7">Choy et al., 2015</a>). Finally, we tested whether ERK-1/2 is upstream of eIF2Bε dephosphorylation by co-treating with Sema3A and U0126 and immunostaining for p-eIF2Bε. Interestingly, the results revealed that the dephosphorylation of eIF2Bε switches to phosphorylation (<a href="#fig4">Figures 4</a>D and 4E). This, together with previous studies in cortical neurons, suggests that ERK-1/2 activates PP1 and simultaneously suppresses the activity of GSK-3β (<a href="#bib40">Quevedo et al., 2003</a>, <a href="#bib19">Hetman et al., 2002</a>), which is known to phosphorylate eIF2Bε and to be repressed at low Sema3A concentrations in a dose-dependent manner (<a href="#bib60">Welsh et al., 1998</a>, <a href="#bib30">Manns et al., 2012</a>).</p> <p id="p0090">To further test whether eIF2B activity represents a key node between the UPR and Sema3A differential translation control, we asked whether it is possible to switch the Sema3A-induced global translation upregulation to repression by only modulating eIF2B activity. To this end, we puromycilated the NSPs and stimulated with Sema3A in the presence of TM, which inhibits eIF2Bε dephosphorylation (<a href="#fig4">Figures 4</a>D and 4E) and blocks its increase in activity. Strikingly, the Sema3A-induced rapid increase in global translation switched to repression (<a href="#fig4">Figures 4</a>F and 4G), mimicking the effects of the Tg- and DTT-induced stress response (<a href="#fig1">Figures 1</a>C and 1D; <a href="#mmc1">Figures S1</a>B and S1C). This result indicates that the Sema3A-induced local increase in total-eIF2Bε (<a href="#fig4">Figures 4</a>A and 4B) on its own is not sufficient to increase eIF2B overall GEF activity and that Sema3A-induced phosphorylation of eIF2α underlies the increase in global translation by engaging eIF2Bε dephosphorylation. Finally, we asked whether this translational control mechanism is also detectable in the developing nervous system <i>in vivo</i>. We incubated brains of embryos at a stage when Sema3A is known to act on retinal axons (stage 35/36; <a href="#bib6">Campbell et al., 2001</a>) with TM for 30 min, puromycilated NSPs over the last 15 min of the treatment, and carried out western blotting to probe for puromycin. The results showed a decrease in global translation equal to ∼25% (<a href="#fig4">Figures 4</a>H and 4I), indicating that translation mediated by p-eIF2α-eIF2Bε signaling occurs in the developing brain <i>in vivo</i>. Altogether, the findings indicate that, during neurodevelopment, Sema3A-induced phosphorylation of axonal eIF2α underlies the noncanonical increase in global translation by enhancing eIF2B activity primarily through dephosphorylation of its ε subunit.</p> </div> <div class="sema3a-inducedpolarizedphosphorylationofeif2αisrequiredfordirectionalmigration" title="sec"> <div class="title" tagx="title" title="title">
Sema3A-Induced Polarized Phosphorylation of eIF2α Is Required for Directional Migration</div> <p id="p0095">We next explored the functional significance of eIF2α phosphorylation downstream of Sema3A. pSILAC-SP3 had revealed that Sema3A-p-eIF2α signaling controls the translation of several proteins involved in the regulation of the actin cytoskeleton and in axon guidance (<a href="#fig2">Figure 2</a>D). For example, the Sema3A-p-eIF2α pathway upregulates Gelsolin (<a href="#fig2">Figures 2</a>B, 2E, and 2F), an actin-binding protein required for filopodium retraction (<a href="#bib26">Lu et al., 1997</a>), and L1cam (<a href="#fig2">Figure 2</a>B), a cell adhesion molecule necessary for topographic mapping of retinal axons (<a href="#bib15">Demyanenko and Maness, 2003</a>). Therefore, we investigated whether phosphorylation of eIF2α plays a role in Sema3A-induced chemorepulsion. We carried out growth cone turning assays with a polarized gradient of Sema3A and bath-applied ISRIB or GSK. Both treatments blocked Sema3A-induced repulsive turning (<a href="#fig5">Figures 5</a>A, 5B, and <a href="#mmc1">S5</a>A). In somaless axons, repulsive turning was also inhibited by ISRIB (<a href="#fig5">Figures 5</a>C, 5D, and <a href="#mmc1">S5</a>B), indicating that local phosphorylation of eIF2α mediates Sema3A-induced chemorepulsion. Because the Sema3A-p-eIF2α-mediated increase in translation is dictated by eIF2Bε dephosphorylation (<a href="#fig4">Figures 4</a>D and 4G), we tested whether blocking PP1 also affects Sema3A-induced repulsive turning. Bath application of TM inhibited chemorepulsion (<a href="#fig5">Figures 5</a>A, 5B, and <a href="#mmc1">S5</a>A). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Spatially Polarized Phosphorylation of eIF2α Mediates Sema3A-Induced Chemorepulsion</p> <p>(A) Turning assay. Arrows indicate the position of the pipette.</p> <p>(B) Cumulative distribution of turning assay outcome. A polarized gradient of Sema3A was generated, and ISRIB, GSK, or TM were bath-applied. Positive values indicate attraction, and negative values indicate repulsion (unpaired t test).</p> <p>(C) Turning assay with somaless axons. Arrows indicate the position of the pipette. Eye explants were removed immediately prior to the experiment.</p> <p>(D) Cumulative distribution of turning assay outcome. A polarized gradient of Sema3A was generated, and ISRIB was bath-applied. Positive values indicate attraction, and negative values indicate repulsion (unpaired t test).</p> <p>(E) Growth cone immunostained for p-eIF2α with a line dividing the near and far sides. Arrowheads indicate the 90° polarized gradient of Sema3A.</p> <p>(F) Cumulative distribution assessing the asymmetric increase of p-eIF2α with the near:far ratio method (unpaired t test).</p> <p>(G) Asymmetric increase of p-eIF2α, assessed by the center of mass method (unpaired t test).</p> <p>(H) Sema3A-induced repulsive growth cone model. p-eIF2α increases on the near-stimulus side, controlling the β-actin polarized decrease (<a href="#bib3">Cagnetta et al., 2018</a>), thus helping with asymmetric cytoskeleton deconstruction and filopodium collapse.</p> <p>Error bars indicate SEM. Scale bar, 5 μm. See also <a href="#mmc1">Figure S5</a>.</p>   </div> <p /> <p id="p0100">In light of these results, we reasoned that, during the chemotropic response, a directional stimulus of Sema3A might produce a polarized asymmetrical phosphorylation of eIF2α, generating an internal gradient of proteomic change across the growth cone. To test this possibility, a gradient of Sema3A was applied for 10 min at a 90° angle to the growth cone as an assay to achieve a steep difference between the "near" and the "far" sides of the growth cone (<a href="#fig5">Figure 5</a>E). qIF indicated that a Sema3A gradient significantly increases eIF2α phosphorylation on the near-stimulus side (<a href="#fig5">Figures 5</a>E, 5F, and <a href="#mmc1">S5</a>C). Further confirmation of the asymmetrical phosphorylation was obtained by comparing the center-of-mass value of the p-eIF2α IF signal between the control and Sema3A gradient conditions, which revealed an equivalent shift toward the Sema3A source (<a href="#fig5">Figures 5</a>G and <a href="#mmc1">S5</a>D). These results, together with the finding that Sema3A downregulates β-actin via p-eIF2α (<a href="#fig2">Figures 2</a>B, 2E, and 2F), are consistent with previous work showing that β-actin decreases on the near-stimulus side in response to a polarized gradient of Sema3A (<a href="#bib3">Cagnetta et al., 2018</a>) and support a growth cone-repulsive model where p-eIF2α increases on the side close to the source of Sema3A to mediate cytoskeleton disassembly and filopodium collapse (<a href="#fig5">Figure 5</a>H). Collectively, the data show that polarized phosphorylation of eIF2α within the growth cone is required for Sema3A-induced directional migration.</p> </div> <div class="perksignalingisinvolvedinretinotectalaxonnavigationin vivo" title="sec"> <div class="title" tagx="title" title="title">
PERK Signaling Is Involved in Retinotectal Axon Navigation <i>In Vivo</i></div> <p id="p0105">We next investigated whether eIF2α phosphorylation is involved in the navigation of retinal axons <i>in vivo</i>. The PERK MO was injected into only one of the two dorsal blastomeres, leading to embryos in which one half of the CNS is depleted of PERK and the other half is wild-type (<a href="#fig6">Figure 6</a>A; <a href="#bib42">Roque et al., 2016</a>). During development, RGC axons cross the midline at the optic chiasm and project contralaterally; therefore, 1,1′-Dioctadecyl-3,3,3′,3′-tetramethylindocarbocyanine perchlorate (DiI) injection in the ipsilateral eye enabled us to test the contribution of axonal PERK in RGCs only without affecting the optic tract substrate (<a href="#fig6">Figures 6</a>A and 6B). Embryos injected with PERK MO overall appeared to develop normal projections as in the control MO (CoMO) (<a href="#fig6">Figures 6</a>C and 6D), showing no difference in optic tract width (<a href="#mmc1">Figures S6</a>A and S6B). PERK morphants exhibited a slight decrease (not statistically significant) in the mid-diencephalic turn (MDT) angle (<a href="#fig6">Figures 6</a>B, 6H, and <a href="#mmc1">S6</a>C) and an increase in the proportion of embryos with an MDT angle smaller than that in the wild-type (i.e., MDT &amp;lt; 45°) (<a href="#fig6">Figure 6</a>I). Measurement of the tectal projection angle (TPA) (<a href="#fig6">Figure 6</a>B) revealed an increased tendency of axons to turn away from the posterior tectal border (not statistically significant; <a href="#fig6">Figures 6</a>J, 6K, and <a href="#mmc1">S6</a>D). Collectively, this result is in line with previous studies showing, in the same <i>Xenopus</i> visual system, that no gross abnormalities were observed in axon navigation after either Sema3A KD or acute inhibition of protein synthesis (<a href="#bib1">Atkinson-Leadbeater et al., 2010</a>, <a href="#bib61">Wong et al., 2017</a>). Similarly, the retinotectal projection did not exhibit evident defects after genetic deletion or pharmacological inhibition of mTOR in zebrafish (<a href="#bib9">Cioni et al., 2018</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>PERK Signaling Is Required for Visual Pathway Development <i>In Vivo</i></p> <p>(A) Experimental outline to investigate the contribution of axonal PERK in RGCs only and Slit1 in the optic tract pathway substrate. Unilateral MO injection leads to targeted KD in half of the nervous system.</p> <p>(B) Schematic of axons navigating the optic tract and reaching the tectum. TPB, tectal posterior boundary; TAB, tectal anterior boundary; TPA tectal projection angle; MDT, mid-diencephalic turn; A, anterior; P, posterior; OC, optic chiasm; Tec, tectum; Di, diencephalon; Hy, hypothalamus; Tel, telencephalon.</p> <p>(C–G) Representative images of DiI-filled stage 41 retinotectal projections in Control MO (C), unilateral KD of PERK in the axons (D), unilateral KD of Slit1 in the optic tract substrate (E), or both (F and G) (Ax, axon; Br, brain).</p> <p>(H) Cumulative distribution of MDT angle measurements in unilateral KD of PERK in the axons or Slit1 in the optic tract substrate or both (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(I) Penetrance for MDT angles of less than 45° (Fisher’s exact test).</p> <p>(J) Cumulative distribution of TPA measurements in unilateral KD of PERK in the axons or Slit1 in the optic tract substrate or both. Positive values indicate angles pointing toward the TPB, and negative values indicate angles pointing toward the TAB (one-way ANOVA with Bonferroni’s multiple comparisons test).</p> <p>(K) Penetrance of posterior tectum avoidance, measured as TPA &amp;lt; mean TPA in CoMO (i.e., −8.6°) (Fisher’s exact test).</p> <p>(L) Single RGC axons in the tectum and line drawings of the corresponding trajectories shown by color-coded branch order: white, axon shaft; branches: red, primary; blue, secondary; yellow, tertiary.</p> <p>(M) Number of axon branches in the various orders and total number of branches in the PERK morphants (two-way ANOVA).</p> <p>(N) Length of axon branches in the PERK morphants (unpaired t test).</p> <p>(O) Formulation of axon complexity index (ACI). Color indicates the branch order as in (L).</p> <p>(P) ACI in the PERK morphants (Fisher’s exact test).</p> <p>Error bars indicate SEM. Scale bars, 100 μm (C–G) and 20 μm (L). See also <a href="#mmc1">Figure S6</a>.</p>   </div> <p /> <p id="p0110">Previous work has shown that Sema3A and Slit1 transcripts are both present at the mid-diencephalic turn and in the tectum and that these two cues cooperate to guide the turning of axons caudally (<a href="#bib6">Campbell et al., 2001</a>, <a href="#bib20">Hocking et al., 2010</a>, <a href="#bib1">Atkinson-Leadbeater et al., 2010</a>). Furthermore, a recent study of mouse dorsal root ganglion growth cones found that Sema3A and Slit1 induce chemorepulsion through distinct mechanisms (<a href="#bib32">McConnell et al., 2016</a>). Therefore, we reasoned that Slit1 might act via a PERK-independent route and compensate PERK KD downstream of Sema3A. We first tested whether Slit1 affects eIF2α phosphorylation. Interestingly, qIF showed no change in growth cone basal p-eIF2α level after Slit1 stimulation (Figures S6E and S6F), indicating that PERK is selectively activated downstream of Sema3A. Next, we tested Slit1 MO in our system (∼55% KD) (<a href="#mmc1">Figure S6</a>G) and knocked down Slit1 in the optic tract substrate <i>in vivo</i> (<a href="#fig6">Figure 6</a>A). Consistent with previous results (<a href="#bib1">Atkinson-Leadbeater et al., 2010</a>), Slit1 KD did not interfere with optic tract width (<a href="#fig6">Figures 6</a>C, 6E, and <a href="#mmc1">S5</a>B) or with navigation (<a href="#fig6">Figures 6</a>C, 6E, 6H–6K, <a href="#mmc1">S6</a>C, and S6D). We then simultaneously knocked down PERK in the axon and Slit1 in the optic tract substrate (<a href="#fig6">Figure 6</a>A). Remarkably, DiI axon labeling revealed that, although the optic tract width remained unaffected (<a href="#mmc1">Figure S6</a>B), the whole axonal bundle failed to turn caudally (<a href="#fig6">Figures 6</a>F–6I and <a href="#mmc1">S6</a>C) and did not correctly enter the tectum in the midbrain (<a href="#fig6">Figures 6</a>F, 6G, 6J, 6K, and <a href="#mmc1">S6</a>D).</p> <p id="p0115">Further, we exposed the intact brain to ISRIB treatment by removal of the overlying epidermis during the period of optic pathway formation (<a href="#mmc1">Figure S6</a>H). In line with the PERK morphants, the results showed no significant difference in optic tract width, MDT angle, and tectal entry (<a href="#mmc1">Figures S6</a>I, S6J, S6L, and S6M). When ISRIB treatment was combined with Slit1 KD in the optic tract substrate (<a href="#mmc1">Figure S6</a>H), the brains exhibited axon guidance defects that phenocopied those seen with PERK-Slit1 KD (<a href="#mmc1">Figure S6</a>K, S6M, and S6N) without affecting optic tract width (<a href="#mmc1">Figure S6</a>L). Collectively, the data indicate that PERK-p-eIF2α signaling cooperates with other p-eIF2α-independent pathways in guidance cue integration during retinotectal axon navigation <i>in vivo</i>.</p> </div> <div class="perksignalingisrequiredforaxonterminalbranchingin vivo" title="sec"> <div class="title" tagx="title" title="title">
PERK Signaling Is Required for Axon Terminal Branching <i>In Vivo</i></div> <p id="p0120">Upon reaching the tectum, where Sema3A is expressed (<a href="#bib6">Campbell et al., 2001</a>), RGC axons elaborate terminal branches and form synapses. Sema3A has been shown to elicit branching of retinal and GABAergic axons (<a href="#bib6">Campbell et al., 2001</a>, <a href="#bib8">Cioni et al., 2013</a>), and axon branching is dependent on local protein synthesis <i>in vivo</i> (<a href="#bib61">Wong et al., 2017</a>), leading us to ask whether eIF2α phosphorylation is required for branching <i>in vivo</i>. The PERK MO and a membrane-targeted GFP (mGFP) reporter were co-electroporated into the eye at stage 28, and single axon arbors were imaged at stage 45 (<a href="#bib61">Wong et al., 2017</a>). Although CoMO-electroporated axons exhibited complex arbors, PERK MO axons exhibited a much simpler arbor architecture (<a href="#fig6">Figure 6</a>L). Quantitative analysis revealed that the branch numbers decreased across different branch orders, leading to an overall drop of 56% (<a href="#fig6">Figure 6</a>M). Furthermore, a 55% reduction of the total branch length was observed (<a href="#fig6">Figure 6</a>N). The axon complexity index (ACI) (<a href="#fig6">Figure 6</a>O; <a href="#bib31">Marshak et al., 2007</a>) showed a marked decrease in the PERK morphants (<a href="#fig6">Figure 6</a>P). These data reveal that PERK signaling is required for developing axon arbor complexity <i>in vivo</i>.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="p0125">Extracellular stimuli can rapidly remodel the local nascent proteome in axons, and here we investigated the underlying translational control mechanisms and mRNA targets. We used nascent proteome analysis combined with <i>in vitro</i> and <i>in vivo</i> models to demonstrate that a Sema3A-induced initial wave of local translation triggers a noncanonical eIF2 signaling pathway. Subsequently, this pathway upregulates local translation and orchestrates a set of proteomic changes required for axon guidance and neural connectivity.</p> <p id="p0130">A canonical role for eIF2α phosphorylation under physiological conditions, rather than in response to stress or in pathology, has already emerged from recent studies (<a href="#bib16">Di Prisco et al., 2014</a>, <a href="#bib12">Dalton et al., 2013</a>, <a href="#bib52">Trinh et al., 2014</a>, <a href="#bib62">Woo et al., 2012</a>). Our work differs by showing, for the first time, that eIF2α phosphorylation can underlie an increase in translation as opposed to the decrease characterizing the canonical stress model. Notably, our results show that eIF2α phosphorylation (10 min) is dependent on the Sema3A-induced initial wave (≥5 min) of protein synthesis mediated by mTOR and ERK-1/2 (<a href="#fig7">Figure 7</a>). These findings support a model where the ER, known to reside throughout the axon (<a href="#bib27">Luarte et al., 2018</a>), becomes overloaded with new unfolded (i.e., yet to be folded) proteins following Sema3A stimulation (∼30% increase in global translation in only 10 min). This may cause physiological stress, which activates the ER stress sensor PERK and leads to eIF2α phosphorylation. This model is further supported by the downstream selective translation of NSPs involved in protein folding, ER and mitochondria protein quality control, and biosynthesis of amino acids, possibly to sustain the burst in global translation. Interestingly, ERK-1/2 also controls eIF2Bε, likely by activating PP1 and suppressing GSK-3β (<a href="#fig7">Figure 7</a>; <a href="#bib40">Quevedo et al., 2003</a>, <a href="#bib19">Hetman et al., 2002</a>). Therefore, the findings reveal a dependency between the pathways triggered downstream of Sema3A, wherein p-eIF2α-eIF2Bε signaling can be activated at the second stage of a cascade, and account for the dynamic and changing nature of the nascent axonal proteome during the 30 min post-stimulation (<a href="#bib3">Cagnetta et al., 2018</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>eIF2Bε Constitutes a Pivotal Node between the Responses to Canonical Stress and Sema3A</p> <p>Sema3A induces an initial (5 min) wave of local translation independent of the eIF2 pathway mediated by ERK-1/2 and mTOR. Simultaneously, eIF2Bε is locally translated in an ERK-1/2-mTOR-independent manner. The rapid increase in local protein synthesis triggers eIF2α phosphorylation via PERK at 10 min stimulation. Within this time course, ERK-1/2 represses GSK-3β and activates PP1, dephosphorylating eIF2Bε and increasing eIF2B activity. The engagement of p-eIF2α and increased eIF2B GEF activity generates a specific level of ternary complex higher than in the canonical stress response, resulting in uORF-independent selective translation of 75 NSPs, upregulating global translation. +p, phosphorylation, −p, dephosphorylation, ↑ and <b>↓</b>, axonal translation upregulation and downregulation; dashed lines, indirect activation following a rise in local protein synthesis; dotted lines, interaction; dashed circles, timing.</p>   </div> <p /> <p id="p0135">The phosphorylation status of eIF2Bε, and, hence, the GEF activity of eIF2B (<a href="#bib40">Quevedo et al., 2003</a>), dictates the outcome of the global translation levels distinguishing the response to Sema3A from the canonical UPR. Specifically, we found that Sema3A, unlike Tg, induces local translation and dephosphorylation of eIF2Bε via PP1 (<a href="#fig7">Figure 7</a>). The absence of Atf4 upregulation suggests that Sema3A-induced regulation of eIF2B and eIF2α phosphorylation precisely influences the rate of generation of the ternary complex, promoting higher exchange of guanosine diphosphate (GDP) for GTP on eIF2 than in the canonical stress response, which is instead triggered by low levels of ternary complex (<a href="#bib55">Vattem and Wek, 2004</a>). One possibility is that increased eIF2B activity exchanges GDP for GTP with higher efficiency than in the canonical UPR on the subpopulation of eIF2 that escapes phosphorylation on its α subunit. A further possibility is that dephosphorylation of eIF2Bε may stabilize the eIF2B dimer, decreasing its affinity for p-eIF2α and permitting higher GDP-GTP exchange than in the canonical UPR, similar to the mechanism of action of ISRIB (<a href="#bib53">Tsai et al., 2018</a>, <a href="#bib48">Sidrauski et al., 2015</a>). This bypasses the global translational repression and the uORF-mediated upregulation of canonical stress markers and regulates the translational efficiency of a subset of mRNAs possibly sensitive to such levels of ternary complex (<a href="#fig7">Figure 7</a>). It is interesting to speculate that this translational control mechanism may also be employed by other biological processes to tackle large increases in protein synthesis (<a href="#bib2">Baleriola et al., 2014</a>) and consequent ER overload, bypassing the translation of pro-apoptotic factors (e.g., Chop; <a href="#bib62">Woo et al., 2012</a>) and regulating the translation of a specific subset of mRNAs (e.g., for proteostasis). Furthermore, this noncanonical way to control translation may suggest new therapeutic targets for disorders involving the detrimental expression of UPR markers and pathogenic translation repression (<a href="#bib34">Moreno et al., 2012</a>, <a href="#bib28">Ma et al., 2013</a>).</p> <p id="p0140">Our study also revealed that a physiological extracellular stimulus can control the phosphorylation of eIF2α in axons with spatio-temporal precision. The phosphorylation is polarized to the near-stimulus compartment of the growth cone, indicating that its translational control mechanism can be further spatially compartmentalized at the subcellular level (<a href="#fig5">Figure 5</a>H). Of particular interest is that the Sema3A-induced phosphorylation of eIF2α is transient, peaking 10 min post-stimulation and declining thereafter, which is in contrast to the canonical stress response, where eIF2α phosphorylation typically peaks at 30 min and endures for hours (<a href="#bib43">Sadighi Akha et al., 2011</a>). The phosphatase PP2Cα topped the list of selectively upregulated nascent proteins in response to Sema3A via p-eIF2α. PP2Cα dephosphorylates and, thereby, inhibits the 5′ adenosine monophosphate-activated protein kinase (AMPK) (<a href="#bib24">Lammers and Lavi, 2007</a>), which has been reported to be required for PERK activation in a specific UPR pathway but not in response to Tg (<a href="#bib64">Yang et al., 2013</a>). This raises the possibility that Sema3A turns off the PERK-p-eIF2α pathway with a built-in negative feedback loop by triggering the local translation of PP2Cα, thus accounting for the eIF2α transient phosphorylation. This fast mechanism could accommodate the rapid cue-induced response locally, likely required <i>in vivo</i> for the growing axons to make timely navigational and connectivity decisions.</p> <p id="p0145">PERK signaling is involved in axon retinotectal navigation <i>in vivo</i> by working at the mid-diencephalic turn and in the tectum together with Slit1, whose downstream signaling pathway is p-eIF2α-independent. This mechanism may have evolved to build a more robust system and increase axon navigation accuracy. PERK signaling is also required for axon arbor formation in the tectum, in line with recent <i>in vivo</i> evidence showing that acute inhibition of protein synthesis impairs axon terminal branching (<a href="#bib61">Wong et al., 2017</a>). Importantly, given that the axons are exposed to various guidance cues in the tectum, we do not exclude the interesting possibility that PERK may act downstream of further extracellular stimuli that, like Sema3A, induce a strong global increase in local translation (<a href="#bib65">Yoon et al., 2012</a>).</p> <p id="p0150">Last, some of the NSPs regulated via Sema3A-p-eIF2α signaling are neurological disease-associated (<a href="#mmc1">Table S2</a>), suggesting links between defective axonal translational control in neural wiring and disease. In addition, Sema3A is known to inhibit axon regeneration following injury in the adult nervous system (<a href="#bib17">Giger et al., 2010</a>); hence, the eIF2 pathway may represent a therapeutic target for neural repair. In conclusion, the noncanonical signaling reported could open new avenues of investigation in translational control and lead to a better understanding of neural wiring and potentially help with the development of new therapeutic approaches.</p> </div> <div class="star★methods" title="sec"> <div class="title" tagx="title" title="title">
STAR★Methods</div> <div class="keyresourcestable" title="sec"> <div class="title" tagx="title" title="title">
Key Resources Table</div> <p id="p0155"> <div class="table-wrap_UNKNOWN" position="float" id="undtbl1"> </div></p><table frame="hsides" rules="groups"> <thead> <tr> <th>REAGENT or RESOURCE</th> <th>SOURCE</th> <th>IDENTIFIER</th> </tr> </thead> <tbody> <tr> <td colspan="3"><b>Antibodies</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>anti-eIF2α</td> <td>Abcam</td> <td>Cat#ab137626; RRID: <a href="nif-antibody:AB_2736873">AB_2736873</a></td> </tr> <tr> <td>anti-p-eIF2α</td> <td>Abcam</td> <td>Cat#ab32157; RRID: <a href="nif-antibody:AB_732117">AB_732117</a></td> </tr> <tr> <td>anti-puromycin Alexa Fluor 488 conjugate</td> <td>Millipore</td> <td>Cat#MABE343-AF488; RRID: <a href="nif-antibody:AB_2736875">AB_2736875</a></td> </tr> <tr> <td>anti-puromycin Alexa Fluor 647 conjugate</td> <td>Millipore</td> <td>Cat#MABE343-AF647; RRID:<a href="nif-antibody:AB_2736876">AB_2736876</a></td> </tr> <tr> <td>anti-puromycin</td> <td>Millipore</td> <td>Cat#MABE343; RRID: <a href="nif-antibody:AB_2566826">AB_2566826</a></td> </tr> <tr> <td>anti-Actb</td> <td>Abcam</td> <td>Cat#ab6277; RRID: <a href="nif-antibody:AB_305394">AB_305394</a></td> </tr> <tr> <td>anti-Gsn</td> <td>Proteintech</td> <td>Cat#11644-2-AP; RRID: <a href="nif-antibody:AB_2295090">AB_2295090</a></td> </tr> <tr> <td>anti-RpL7a</td> <td>Abcam</td> <td>Cat#ab155147; RRID: <a href="nif-antibody:AB_2736874">AB_2736874</a></td> </tr> <tr> <td>anti-Atf4</td> <td>Abcam</td> <td>Cat# ab85049; RRID: <a href="nif-antibody:AB_1861369">AB_1861369</a></td> </tr> <tr> <td>anti-eIF2Bε</td> <td>Abcam</td> <td>Cat#ab32713; RRID: <a href="nif-antibody:AB_2230901">AB_2230901</a></td> </tr> <tr> <td>anti-p-eIF2Bε (Ser539)</td> <td>Proteintech</td> <td>N/A (customized)</td> </tr> <tr> <td>anti-PERK</td> <td>Cell signaling</td> <td>Cat#3192; RRID: <a href="nif-antibody:AB_2095847">AB_2095847</a></td> </tr> <tr> <td>anti-Tuba</td> <td>Sigma</td> <td>Cat#T6074; RRID: <a href="nif-antibody:AB_477582">AB_477582</a></td> </tr> <tr> <td>anti-Slit1</td> <td>Abcam</td> <td>Cat# ab115892; RRID: <a href="nif-antibody:AB_10903854">AB_10903854</a></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>Sema3A</td> <td>R&amp;amp;D Systems</td> <td>Cat#1250-S3</td> </tr> <tr> <td>Slit1</td> <td>R&amp;amp;D Systems</td> <td>Cat#6514-SL</td> </tr> <tr> <td>ISRIB</td> <td>Sigma</td> <td>Cat#SML0843</td> </tr> <tr> <td>Thapsigargin</td> <td>Sigma</td> <td>Cat#T9033</td> </tr> <tr> <td>Dithiothreitol</td> <td>Sigma</td> <td>Cat#D0632</td> </tr> <tr> <td>GSK2606414</td> <td>Calbiochem</td> <td>Cat#516535</td> </tr> <tr> <td>Cycloheximide</td> <td>Sigma</td> <td>Cat#C4859</td> </tr> <tr> <td>PP242</td> <td>Tocris</td> <td>Cat#4257</td> </tr> <tr> <td>U0126</td> <td>Tocris</td> <td>Cat#1144</td> </tr> <tr> <td>Tautomycin</td> <td>Calbiochem</td> <td>Cat#580551</td> </tr> <tr> <td>Poly-L-lysine</td> <td>Sigma</td> <td>Cat#P1274</td> </tr> <tr> <td>Laminin</td> <td>Sigma</td> <td>Cat#L2020</td> </tr> <tr> <td>Leibovitz L-15 medium –Lys -Arg</td> <td>GIBCO Life Technologies</td> <td>N/A (customized)</td> </tr> <tr> <td>Stable isotope-coded amino acids Lys4</td> <td>Silantes GmbH</td> <td>Cat#211103913</td> </tr> <tr> <td>Stable isotope-coded amino acids Lys8</td> <td>Silantes GmbH</td> <td>Cat#211603902</td> </tr> <tr> <td>Stable isotope-coded amino acids Arg6</td> <td>Silantes GmbH</td> <td>Cat#201203902</td> </tr> <tr> <td>Stable isotope-coded amino acids Arg10</td> <td>Silantes GmbH</td> <td>Cat#201603902</td> </tr> <tr> <td>Puromycin</td> <td>Sigma</td> <td>Cat#P8833</td> </tr> <tr> <td>Sera-Mag Speed Beads A</td> <td>GE Healthcare</td> <td>Cat#24152105050250</td> </tr> <tr> <td>Sera-Mag Speed Beads B</td> <td>GE Healthcare</td> <td>Cat#44152105050250</td> </tr> <tr> <td>Trypsin/LysC</td> <td>Promega</td> <td>Cat#V5071</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Critical Commercial Assays</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>RNAqueous-Micro Total RNA Isolation Kit</td> <td>Invitrogen</td> <td>Cat#AM1931</td> </tr> <tr> <td>OneStep RT-PCR kit</td> <td>QIAGEN</td> <td>Cat#210210</td> </tr> <tr> <td>Duolink <i>in situ</i> PLA kit</td> <td>Sigma</td> <td>Cat#DUO92014</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Deposited Data</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>Proteomics data</td> <td>This paper</td> <td>PRIDE: <a href="pride:PXD009250">PXD009250</a></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Experimental Models: Organisms/Strains</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td><i>X. laevis</i></td> <td>Nasco</td> <td><a href="https://www.enasco.com/p/9+-cm-Mature-Female-Xenopus-laevis-Frogs,-Live-Specimen+LM00535?fbclid=IwAR3sBev2jD7ZPRLqRfaecQb0P29WkPCI_Aov1xtHkNhmNbaIAHxRHGLnX9Y">https://www.enasco.com/p/9+-cm-Mature-Female-Xenopus-laevis-Frogs,-Live-Specimen+LM00535?fbclid=IwAR3sBev2jD7ZPRLqRfaecQb0P29WkPCI_Aov1xtHkNhmNbaIAHxRHGLnX9Y</a></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Oligonucleotides</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>Primer: β-actin</td> <td rowspan="3">Sigma</td> <td rowspan="3">N/A</td> </tr> <tr> <td><i>for</i> 5′ CCTGTGCAGGAAGATCACAT 3′</td> </tr> <tr> <td><i>rev</i> 5′ TGTTAAAGAGAATGAGCCCC 3′</td> </tr> <tr> <td>Primer: Brn3a</td> <td>Sigma</td> <td>N/A</td> </tr> <tr> <td><i>for</i> 5′ TGAGCGATTCAAGCAGAGGAGG 3′</td> <td>Sigma</td> <td>N/A</td> </tr> <tr> <td><i>rev</i> 5′ TGCGACAGGGTGAGGGATTCAAAC 3′</td> <td>Gene Tools</td> <td>N/A</td> </tr> <tr> <td>Primer: eIF2Bε</td> <td>Gene Tools</td> <td>N/A</td> </tr> <tr> <td><i>for</i> 5′ tgatgatgcaggcgctggaa 3′</td> <td>Gene Tools</td> <td>N/A</td> </tr> <tr> <td><i>rev 5′</i> caggtgaagcagggtggctttctg 3′</td> <td /> <td>N/A</td> </tr> <tr> <td>Morpholino: Control MO (CoMO)</td> <td /> <td>N/A</td> </tr> <tr> <td>5′ CCTCTTACCTCAGTTACAATTTATA 3′</td> <td /> <td>N/A</td> </tr> <tr> <td>Morpholino: PERK MO</td> <td /> <td>N/A</td> </tr> <tr> <td>5′ CGAACACTTTCACCTCATAACACTT 3′</td> <td /> <td>N/A</td> </tr> <tr> <td>Morpholino: Slit1 MO</td> <td /> <td>N/A</td> </tr> <tr> <td>5′ AGTAGTCTCAATGACACAATGACCA 3′</td> <td /> <td>N/A</td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td colspan="3"><b>Software and Algorithms</b></td> </tr> <tr> <td colspan="3"> <hr /> </td> </tr> <tr> <td>Volocity</td> <td>v.6.3.1</td> <td>RRID: <a href="rridsoftware:SCR_002668">SCR_002668</a></td> </tr> <tr> <td>GraphPad PRISM</td> <td>v.5.0c</td> <td>RRID: <a href="rridsoftware:SCR_002798">SCR_002798</a></td> </tr> <tr> <td>ImageJ</td> <td>v.149</td> <td>RRID: <a href="rridsoftware:SCR_003070">SCR_003070</a></td> </tr> <tr> <td>DAVID</td> <td>v.6.8</td> <td>RRID: <a href="rridsoftware:SCR_001881">SCR_001881</a></td> </tr> <tr> <td>KEGG</td> <td>N/A</td> <td>RRID: <a href="rridsoftware:SCR_012773">SCR_012773</a></td> </tr> <tr> <td>String</td> <td>v10.5</td> <td>RRID: <a href="rridsoftware:SCR_005223">SCR_005223</a></td> </tr> <tr> <td>Maxquant</td> <td>v1.4.1.2</td> <td>RRID: <a href="rridsoftware:SCR_014485">SCR_014485</a></td> </tr> <tr> <td>Perseus</td> <td>v1.5.1.6 or 1.5.5.3</td> <td>RRID: <a href="rridsoftware:SCR_015753">SCR_015753</a></td> </tr> </tbody> </table>  <p /> </div> <div class="contactforreagentandresourcesharing" title="sec"> <div class="title" tagx="title" title="title">
Contact for Reagent and Resource Sharing</div> <p id="p0160">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Christine E. Holt (<a href="mailto:ceh33@cam.ac.uk">ceh33@cam.ac.uk</a>).</p> </div> <div class="experimentalmodelandsubjectdetails" title="sec"> <div class="title" tagx="title" title="title">
Experimental Model and Subject Details</div> <div class="xenopuslaevisembryosmaintenance" title="sec"> <div class="title" tagx="title" title="title"><i>Xenopus laevis</i> embryos maintenance</div> <p id="p0165"><i>Xenopus laevis</i> embryos of either sex were obtained by <i>in vitro</i> fertilization as previously described (<a href="#bib4">Campbell and Holt, 2001</a>), raised in 0.1x modified Barth’s saline (MBS; 8.8 mM NaCl, 0.1 mM KCl, 0.24 mM NaHCO<sub>3</sub>, 0.1 mM HEPES, 82 μM MgSO<sub>4</sub>, 33 μM Ca(NO<sub>3</sub>)<sub>2</sub>, 41 μM CaCl<sub>2</sub>) at 14–22°C and staged according to <a href="#bib35">Nieuwkoop and Faber (1994)</a>. This research has been regulated under the Animals (Scientific Procedures) Act 1986 Amendment Regulations 2012 following ethical review by the University of Cambridge Animal Welfare and Ethical Review Body (AWERB).</p> </div> </div> <div class="methoddetails" title="sec"> <div class="title" tagx="title" title="title">
Method Details</div> <div class="retinalexplantcultureandaxotomyassayontranswellfilter" title="sec"> <div class="title" tagx="title" title="title">
Retinal explant culture and axotomy assay on transwell filter</div> <p id="p0170">Whole eyes of anesthetized stage 35/36 embryos were dissected out and cultured at 20°C for 24h in 60% L15 minimal medium (Invitrogen), 1x Penicillin Streptomycin Fungizone on glass bottom dishes (MatTek) or on the top compartment of 6-well hanging inserts (Boyden chambers) with 1 μm membrane pores (Falcon), coated on both sides of the membrane with poly-L-lysine (10 μg/ml, Sigma) and only on the bottom side with laminin (10 μg/ml, Sigma).</p> <p id="p0175">For the pSILAC experiment 100 eye explants were cultured per condition, as detailed in <a href="#bib3">Cagnetta et al. (2018)</a>. After 24 h, eye explants were pre-incubated with ISRIB for 30 min. Subsequently, eye explants were removed, scraped and washed off 7 times from the top compartment of the filter, leaving the somaless axons (∼2 μg protein typical yield) at the bottom. Sema3A or Sema3A and ISRIB were added, together with respective stable isotope-coded amino acids, to the somaless axons for 15 min. After stimulation the membrane was cut away, rinsed with ice cold PBS and lysed for protein extraction.</p> </div> <div class="pharmacologicaltreatments" title="sec"> <div class="title" tagx="title" title="title">
Pharmacological treatments</div> <p id="p0180">Stimulations were carried out using the following concentrations: Sema3A (150 ng/ml), Slit1 (200 ng/ml), ISRIB (200 nM), Thapsigargin (500 nM), Dithiothreitol (1 mM), GSK2606414 (300 nM), Cycloheximide (50 μM), PP242 (2.5 μM), U0126 (10 μM), Tautomycin (4nM).</p> </div> <div class="pulsedstableisotopelabelingbyaminoacidsincellculture" title="sec"> <div class="title" tagx="title" title="title">
Pulsed Stable Isotope Labeling by Amino acids in Cell culture</div> <p id="p0185">Experiments were performed in three independent biological replicates. Retinal explants were cultured in SILAC <i>light</i> medium (Lys0, Arg0) for 24 h and incubated in depletion medium (-Lys, -Arg) for 60 min prior pulse labeling. Subsequently, cell bodies were removed and somaless axons were incubated for 15 min with <i>medium</i> (M) (Lys4, Arg6) or <i>heavy</i> (H) isotope-coded amino acids (Lys8, Arg10). At 15 min samples were lysed, immediately pooled and processed by SP3.</p> </div> <div class="single-potsolid-phase-enhancedsamplepreparation" title="sec"> <div class="title" tagx="title" title="title">
Single-Pot Solid-Phase-enhanced Sample Preparation</div> <p id="p0190">Axons were harvested by the addition of lysis buffer (1% SDC, 0.1% SDS, 100mM TrisHCl ph 8.5, 10mM DTT, 1x protease inhibitor EDTA free). Samples were supplemented with 25 units Benzonase nuclease (Merck), and lysed in a Bioruptor (Diagenode) for 5 minutes (cycle 30/30, 4°C). Alkylation was performed by addition of 30 mM Chloroacetamide followed by incubation in the dark for 30 min. Protein clean-up, digestion and peptide clean-up were performed using a modified version of the ultrasensitive sample preparation protocol SP3 (<a href="#bib22">Hughes et al., 2014</a>). In brief, 5 μL of beads (1:1 mixture of hydrophilic and hydrophobic SeraMag Carboxylate-Modified beads, GE Life Sciences) were added to each sample. Acidified acetonitrile was added to achieve a final fraction of organic solvent of 50%. Beads were incubated for 10 min to allow complete binding of proteins to the beads. Protein clean-up was performed by subsequent wash with 70% Ethanol and once with Acetonitrile. For digestion, 0.1 μg sequencing grade Trypsin/LysC (Promega) was added and digestion was performed at 37°C for 16 h. Peptides were eluted with 9 μL 5% DMSO. 1 μL 10% formic acid was added and samples were stored at −20°C prior to MS analysis.</p> </div> <div class="massspectrometry" title="sec"> <div class="title" tagx="title" title="title">
Mass Spectrometry</div> <p id="p0195">Samples were analyzed on a Orbitrap Velos Pro mass spectrometer (Thermo Scientific) using default settings. The mass spectrometer was coupled to a UPLC systems (Waters nanoAcquity UPLC). Peptides were loaded onto trap columns (Waters nanoAcquity Symmetry C<sub>18</sub>, 5 μm, 180 μm × 20 mm) with Buffer A (0.1% formic acid in water) and separated over a 25 cm analytical column (Acclaim PepMap RSLC, 75 μm × 2 μm) using 240 minute linear gradients from 3%–40% Buffer B (0.1% formic acid in Acetonitrile). MS2 Fragmentation was set to CID, and MSMS scans were acquired in the ion trap.</p> </div> <div class="proteomicsdataprocessing" title="sec"> <div class="title" tagx="title" title="title">
Proteomics data processing</div> <p id="p0200">Raw data were processed with Maxquant (version 1.4.1.2) (<a href="#bib11">Cox and Mann, 2008</a>) using default settings. MSMS spectra were searched against the <i>Xenopus laevis</i> Uniprot database (v20140925) concatenated to a database containing protein sequences of common contaminants. Raw data from <a href="#bib3">Cagnetta et al. (2018)</a> was used as a library to increase depth of identifications using the match-between-runs option, which was enabled in Maxquant. Enzyme specificity was set to trypsin/P, allowing a maximum of two missed cleavages. Cysteine carbamidomethylation was set as fixed modification, and methionine oxidation and protein N-terminal acetylation were used as variable modifications. The minimal peptide length was set to six amino acids. The mass tolerances were set to 20 ppm for the first search, and 4.5 ppm for the main search. Global false discovery rates for peptide and protein identification were set to 1%. The match-between-runs and re-quantify options were enabled.</p> </div> <div class="immunocytochemistry" title="sec"> <div class="title" tagx="title" title="title">
Immunocytochemistry</div> <p id="p0205">Retinal cultures were fixed by paraformaldehyde except for anti-β-actin (AC-15 FITC) and anti-Gsn where methanol fixation was carried out. Secondary antibodies were species-specific dye-conjugated (Alexa Fluor, Invitrogen).</p> </div> <div class="puromycilationofnsps" title="sec"> <div class="title" tagx="title" title="title">
Puromycilation of NSPs</div> <p id="p0210">Retinal cultures were incubated with puromycin (2 ng/μl) for the condition and time (up to 15 min) of interest, fixed and incubated with anti-puromycin Alexa Fluor conjugate antibody.</p> <p id="p0215">Intact brains were incubated with puromycin (5 ng/μl) for 15 min in the condition of interest (Control or Tautomycin (20nM)), rinsed in culture medium, lysed and western blot anti-puromycin was carried out.</p> </div> <div class="puromycilationofnspsandproximityligationassay" title="sec"> <div class="title" tagx="title" title="title">
Puromycilation of NSPs and Proximity Ligation Assay</div> <p id="p0220">Retinal cultures were incubated with puromycin (2 ng/μl) for 10 min in the condition of interest, fixed and incubated with anti-puromycin and anti-Atf4 antibodies. Subsequently, Proximity Ligation Assay (PLA) was carried out using species-specific probes (<a href="#bib51">tom Dieck et al., 2015</a>).</p> </div> <div class="reversetranscriptionpolymerasechainreaction" title="sec"> <div class="title" tagx="title" title="title">
Reverse Transcription Polymerase Chain Reaction</div> <p id="p0225">RNA was extracted from using RNAqueous-Micro Total RNA Isolation Kit. Primers were designed using <i>Primer3Plus</i> software. The annealing temperature used was 58°C for β-actin and Brn3a, 67°C for eIF2Bε.</p> </div> <div class="growthconeturningassay" title="sec"> <div class="title" tagx="title" title="title">
Growth cone turning assay</div> <p id="p0230">Retinal explants from stage 35/36 embryos were cultured for 14-18 h on coverslips coated with poly-L-lysine (10 μg/ml) and laminin (10 μg/ml). Gradients of Sema3A (9 μg/ml) or control were generated by pulsatile ejection as described previously (<a href="#bib25">Lohof et al., 1992</a>, <a href="#bib4">Campbell and Holt, 2001</a>) for 60 min placing the micropipette at a starting distance equal to 100 μm and at an angle of 45<b>°</b> relative to the initial direction of the axon shaft. ISRIB (200 nM), GSK2606414 (300 nM), or Tautomycin (4nM) were bath-applied immediately prior to the start of the gradient assay.</p> <p id="p0235">For growth cone gradient assay the gradient was generated for 10 min placing the micropipette at 70 μm distance and at an angle of 90<b>°</b> relative to the growth cone and the initial direction of the axon shaft (<a href="#bib3">Cagnetta et al., 2018</a>). Subsequently samples were immediately fixed and immunostained for p-eIF2α.</p> </div> <div class="blastomereinjection" title="sec"> <div class="title" tagx="title" title="title">
Blastomere injection</div> <p id="p0240"><i>Xenopus</i> embryos were injected at the 4-cell stage in the dorsal animal blastomeres as previously described (<a href="#bib42">Roque et al., 2016</a>). 18 ng of PERK/Slit1/Control MO were injected into the blastomere of interest (<a href="#fig6">Figure 6</a>A).</p> </div> <div class="diianterogradeaxonlabeling" title="sec"> <div class="title" tagx="title" title="title">
DiI anterograde axon labeling</div> <p id="p0245">Stage 41 embryos were fixed in 4% formaldehyde in PBS at 4<b>°</b>C overnight. DiI solution was prepared by dissolving DiI powder (Thermo Scientific) in ethanol and injected into the eye cavity until completely filled. The embryos were incubated at room temperature for 48h to ensure complete dye diffusion. The brain was dissected and mounted in 1xPBS. The contralateral brain hemisphere was imaged.</p> </div> <div class="exposedbrains" title="sec"> <div class="title" tagx="title" title="title">
Exposed brains</div> <p id="p0250">Stage 33/34 embryo brains were exposed by removing the overlying eye and epidermis (<a href="#bib61">Wong et al., 2017</a>) to ISRIB (2 μM) treatment at 22<b>°</b>C overnight, fixed and DiI injection was carried out.</p> </div> <div class="electroporation" title="sec"> <div class="title" tagx="title" title="title">
Electroporation</div> <p id="p0255">Target eye electroporation was performed as previously described (<a href="#bib68">Wong and Holt, 2018</a>). The eye primordia of embryos stage 28 were injected with electroporation mixture (1 μg/μl pCS2+mGFP and 0.5 mM Control MO/PERK MO), followed by electric pulses of 50 ms duration at 1000 ms intervals, delivered at 18 V. The embryos were raised in 0.1x MBS until stage 45.</p> </div> <div class="westernblot" title="sec"> <div class="title" tagx="title" title="title">
Western blot</div> <p id="p0260">Puromycilation assay of brains of stage 35/36 embryos and MOs specificity test on brain and eye tissue were carried out by western blot. Pierce BCA Protein Assay kit (Thermo Fisher Scientific) and spectrophotometry were used to determine the sample concentration. Bovine albumin serum (BSA, Invitrogen) was used to create a standard curve for protein concentration and for normalizing the concentration among samples. The antibody of interest was incubated at 4<b>°</b>C overnight in 5% BSA solution for the anti-Slit1 antibody, or 5% milk solution for the anti-puromycin and anti-PERK antibodies. The blots were then incubated with HRP-conjugated secondary antibodies (Abcam) at room temperature for 45 min, followed by ECL-based detection (Invitrogen).</p> </div> </div> <div class="quantificationandstatisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Quantification and Statistical Analysis</div> <div class="statistics" title="sec"> <div class="title" tagx="title" title="title">
Statistics</div> <p id="p0265">Data were analyzed with PRISM 5 (GraphPad). Data are presented as mean and error bars represent s.e.m. Experiments were performed in at least three independent biological replicates. Details of statistical tests are presented in the figure legends.</p> <p id="p0270"><sup>∗</sup>p ≤ 0.05, <sup>∗∗</sup>p ≤ 0.01, <sup>∗∗∗</sup>p ≤ 0.001, <sup>∗∗∗∗</sup>p ≤ 0.0001, ns: non-significant.</p> </div> <div class="bioinformaticdataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatic data analysis</div> <p id="p0275">For protein quantification a minimum ratio count of 2 was set. The iBAQ was calculated to determine relative abundance levels of the pre-existing light-labeled proteins. Protein ratios were log2-transformed using the Perseus computational framework, and H/M ratios of NSPs were normalized to the median to center the distribution of ratios at 0 on the log2 scale, i.e., comparable numbers of proteins are upregulated and downregulated, respectively. To test whether the log2 ratio of each protein was significantly different from zero, p values were computed by a moderated t test implemented in the R/Bioconductor package limma (<a href="#bib41">Ritchie et al., 2015</a>). p values were corrected for multiple testing by controlling the false discovery rate with the method of Benjamini-Hochberg. Enrichment of categorical annotations (Gene Ontology) was determined using DAVID. Pathway and disease analyses were carried out using KEGG. Interaction network analysis was obtained by employing String v10.5 database. Each node represents a NSP change and each edge shows protein-protein interaction, disconnected nodes are not shown for simplicity. Upstream regulator analyses were carried out based on previous datasets identifying the targets of the following translational regulators: Apc, Mena, Fmrp, Tdp43, Fus, mTOR, PERK and GCN2 (<a href="#bib39">Preitner et al., 2014</a>, <a href="#bib56">Vidaki et al., 2017</a>, <a href="#bib14">Darnell et al., 2011</a>, <a href="#bib10">Colombrita et al., 2012</a>, <a href="#bib50">Thoreen et al., 2012</a>, <a href="#bib13">Dang Do et al., 2009</a>).</p> </div> <div class="quantificationofimmunofluorescence" title="sec"> <div class="title" tagx="title" title="title">
Quantification of Immunofluorescence</div> <p id="p0280">For the quantification of fluorescence intensity, isolated growth cones were selected randomly with phase optics. Low exposure was set up to avoid pixel saturation and the same gain and exposure settings were used for digital capture of images for each experiment which was performed in the same day, except for the growth cone gradient assay where the IF of the near and far sides within the same growth cone were compared. The outline of each single growth cone was traced using the phase image to define the region of interest (ROI) and the mean pixel intensity per unit area was measured in each channel using <i>Volocity</i> software. The background fluorescence was measured in a ROI as close as possible to the growth cone selected and subtracted to the mean fluorescence value of the growth cone. In the figures brightness/contrast settings were adjusted equally across images collected in the same experiment for presentation clarity.</p> <p id="p0285">For the growth cone gradient assay IF ratio analysis, the growth cone was bisected into two areas by a line drawn through the axon shaft and the background fluorescence level was subtracted. For the center of mass analysis, measurement was calculated as the average of all pixel locations weighted as intensity by using <i>ImageJ</i> software. The center of mass of the bright field was subtracted from the center of mass of the fluorescence signal.</p> </div> <div class="turningassaymeasurement" title="sec"> <div class="title" tagx="title" title="title">
Turning assay measurement</div> <p id="p0290">Turning angles were measured on growth cone images taken at 0 and 60 min using <i>ImageJ</i> software.</p> </div> <div class="diiquantification" title="sec"> <div class="title" tagx="title" title="title">
DiI quantification</div> <p id="p0295">For optic tract width quantification ten equally spaced concentric circles (C1-C10) were overlaid on the tract images with the center of the circles overlying the optic chiasm (OC) and C10 overlaying the Tectal Posterior Boundary (TPB) (<a href="#mmc1">Figure S6</a>A). The widths of C2-4 and C5-8, corresponding respectively to pre- and post-caudal turn, were averaged. Lastly, the pre- and post-turn widths were normalized to the brain size, defined by the distance between OC and TPB. Mid-diencephalic turn (MDT) was measured as the angle between the pre-turn axon bundle (drawing a line from the optic chiasm and the ventral side of the MDT) and the post-turn axon bundle (drawing a line from the ventral side of the MDT and the tip of the most pioneer axon) (<a href="#fig6">Figure 6</a>B). The tectal projection angle (TPA) was measured as the angle between the post-turn tract and the most anteriorly projecting axon (<a href="#fig6">Figure 6</a>B). TPA was considered positive if pointing toward the posterior tectum, negative if pointing toward the anterior side of the tectum.</p> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data and Software Availability</div> <p id="p0300">The accession number for the mass spectrometry proteomics data reported in this paper is ProteomeXchange Consortium via the PRIDE (<a href="#bib57">Vizcaíno et al., 2016</a>): <a href="pride:PXD009250">PXD009250</a>.</p> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkinson-Leadbeater</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertolesi</span><span class="given-names" tagx="given-names" title="given-names">G.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hehr</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webber</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cechmanek</span><span class="given-names" tagx="given-names" title="given-names">P.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McFarlane</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dynamic expression of axon guidance cues required for optic tract development is controlled by fibroblast growth factor signaling</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="volume" tagx="volume" title="volume">30</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">685</span><span class="lpage" tagx="lpage" title="lpage">693</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20071533">20071533</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baleriola</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jean</span><span class="given-names" tagx="given-names" title="given-names">Y.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crary</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Troy</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hengst</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Axonally synthesized ATF4 transmits a neurodegenerative signal across brain regions</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">158</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1159</span><span class="lpage" tagx="lpage" title="lpage">1172</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25171414">25171414</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagnetta</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frese</span><span class="given-names" tagx="given-names" title="given-names">C.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shigeoka</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krijgsveld</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid cue-specific remodeling of the nascent axonal proteome</span><span class="source" tagx="source" title="source">Neuron</span><span class="volume" tagx="volume" title="volume">99</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">46.e4</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30008298">30008298</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Chemotropic responses of retinal growth cones mediated by rapid local protein synthesis and degradation</span><span class="source" tagx="source" title="source">Neuron</span><span class="volume" tagx="volume" title="volume">32</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">1013</span><span class="lpage" tagx="lpage" title="lpage">1026</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11754834">11754834</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Apoptotic pathway and MAPKs differentially regulate chemotropic responses of retinal growth cones</span><span class="source" tagx="source" title="source">Neuron</span><span class="volume" tagx="volume" title="volume">37</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">939</span><span class="lpage" tagx="lpage" title="lpage">952</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12670423">12670423</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regan</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tannahill</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">W.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Semaphorin 3A elicits stage-dependent collapse, turning, and branching in Xenopus retinal growth cones</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="volume" tagx="volume" title="volume">21</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">8538</span><span class="lpage" tagx="lpage" title="lpage">8547</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11606642">11606642</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choy</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusoff</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">I.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newton</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goh</span><span class="given-names" tagx="given-names" title="given-names">C.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Page</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shenolikar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peti</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural and Functional Analysis of the GADD34:PP1 eIF2α Phosphatase</span><span class="source" tagx="source" title="source">Cell Rep.</span><span class="volume" tagx="volume" title="volume">11</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">1885</span><span class="lpage" tagx="lpage" title="lpage">1891</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26095357">26095357</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cioni</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Telley</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saywell</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cadilhac</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jourdan</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">J.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jahannault-Talignani</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ango</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">SEMA3A signaling controls layer-specific interneuron branching in the cerebellum</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="volume" tagx="volume" title="volume">23</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">850</span><span class="lpage" tagx="lpage" title="lpage">861</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23602477">23602477</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cioni</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bressan</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kodama</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">W.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Axon-axon interactions regulate topographic optic tract sorting via CYFIP2-dependent WAVE complex function</span><span class="source" tagx="source" title="source">Neuron</span><span class="volume" tagx="volume" title="volume">97</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">1078</span><span class="lpage" tagx="lpage" title="lpage">1093.e6</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29518358">29518358</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colombrita</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onesto</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megiorni</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pizzuti</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baralle</span><span class="given-names" tagx="given-names" title="given-names">F.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buratti</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silani</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratti</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">TDP-43 and FUS RNA-binding proteins bind distinct sets of cytoplasmic messenger RNAs and differently regulate their post-transcriptional fate in motoneuron-like cells</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">287</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">15635</span><span class="lpage" tagx="lpage" title="lpage">15647</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22427648">22427648</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</span><span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="volume" tagx="volume" title="volume">26</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">1367</span><span class="lpage" tagx="lpage" title="lpage">1372</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029910">19029910</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalton</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyons</span><span class="given-names" tagx="given-names" title="given-names">D.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lomvardas</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Co-opting the unfolded protein response to elicit olfactory receptor feedback</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">155</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">321</span><span class="lpage" tagx="lpage" title="lpage">332</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24120133">24120133</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dang Do</span><span class="given-names" tagx="given-names" title="given-names">A.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kimball</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavener</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jefferson</span><span class="given-names" tagx="given-names" title="given-names">L.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">eIF2alpha kinases GCN2 and PERK modulate transcription and translation of distinct sets of mRNAs in mouse liver</span><span class="source" tagx="source" title="source">Physiol. Genomics</span><span class="volume" tagx="volume" title="volume">38</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">328</span><span class="lpage" tagx="lpage" title="lpage">341</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19509078">19509078</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darnell</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Driesche</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hung</span><span class="given-names" tagx="given-names" title="given-names">K.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mele</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">E.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fak</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chi</span><span class="given-names" tagx="given-names" title="given-names">S.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">146</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">247</span><span class="lpage" tagx="lpage" title="lpage">261</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21784246">21784246</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demyanenko</span><span class="given-names" tagx="given-names" title="given-names">G.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maness</span><span class="given-names" tagx="given-names" title="given-names">P.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The L1 cell adhesion molecule is essential for topographic mapping of retinal axons</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="volume" tagx="volume" title="volume">23</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">530</span><span class="lpage" tagx="lpage" title="lpage">538</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12533613">12533613</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Prisco</span><span class="given-names" tagx="given-names" title="given-names">G.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buffington</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonnen</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Placzek</span><span class="given-names" tagx="given-names" title="given-names">A.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidrauski</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krnjević</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaufman</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walter</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa-Mattioli</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Translational control of mGluR-dependent long-term depression and object-place learning by eIF2α</span><span class="source" tagx="source" title="source">Nat. Neurosci.</span><span class="volume" tagx="volume" title="volume">17</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1073</span><span class="lpage" tagx="lpage" title="lpage">1082</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24974795">24974795</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giger</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hollis</span><span class="given-names" tagx="given-names" title="given-names">E.R.</span><span class="suffix" tagx="suffix" title="suffix">2nd</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuszynski</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Guidance molecules in axon regeneration</span><span class="source" tagx="source" title="source">Cold Spring Harb. Perspect. Biol.</span><span class="volume" tagx="volume" title="volume">2</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">a001867</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20519341">20519341</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gumy</span><span class="given-names" tagx="given-names" title="given-names">L.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeo</span><span class="given-names" tagx="given-names" title="given-names">G.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tung</span><span class="given-names" tagx="given-names" title="given-names">Y.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zivraj</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willis</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coppola</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">B.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twiss</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fawcett</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis of embryonic and adult sensory axons reveals changes in mRNA repertoire localization</span><span class="source" tagx="source" title="source">RNA</span><span class="volume" tagx="volume" title="volume">17</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">98</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21098654">21098654</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hetman</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsuan</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Habas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">ERK1/2 antagonizes glycogen synthase kinase-3beta-induced apoptosis in cortical neurons</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">277</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">49577</span><span class="lpage" tagx="lpage" title="lpage">49584</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12393899">12393899</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hocking</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hehr</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertolesi</span><span class="given-names" tagx="given-names" title="given-names">G.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McFarlane</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Distinct roles for Robo2 in the regulation of axon and dendrite growth by retinal ganglion cells</span><span class="source" tagx="source" title="source">Mech. Dev.</span><span class="volume" tagx="volume" title="volume">127</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">36</span><span class="lpage" tagx="lpage" title="lpage">48</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19961927">19961927</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holcik</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonenberg</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Translational control in stress and apoptosis</span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">318</span><span class="lpage" tagx="lpage" title="lpage">327</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15803138">15803138</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foehr</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garfield</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furlong</span><span class="given-names" tagx="given-names" title="given-names">E.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinmetz</span><span class="given-names" tagx="given-names" title="given-names">L.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krijgsveld</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ultrasensitive proteome analysis using paramagnetic bead technology</span><span class="source" tagx="source" title="source">Mol. Syst. Biol.</span><span class="volume" tagx="volume" title="volume">10</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">757</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25358341">25358341</a></span></span></li> <li tag="ref"><a name="bib67" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moggridge</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sorensen</span><span class="given-names" tagx="given-names" title="given-names">P.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morin</span><span class="given-names" tagx="given-names" title="given-names">G.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krijgsveld</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Single-pot, solid-phase-enhanced sample preparation for proteomics experiments</span><span class="source" tagx="source" title="source">Nat Protoc.</span><span class="year" tagx="year" title="year">2018</span><span class="comment">Published online November 21, 2018</span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">B.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Axonal mRNA localization and local protein synthesis in nervous system assembly, maintenance and repair</span><span class="source" tagx="source" title="source">Nat. Rev. Neurosci.</span><span class="volume" tagx="volume" title="volume">13</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">308</span><span class="lpage" tagx="lpage" title="lpage">324</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22498899">22498899</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lammers</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of type 2C protein phosphatases in growth regulation and in cellular stress signaling</span><span class="source" tagx="source" title="source">Crit. Rev. Biochem. Mol. Biol.</span><span class="volume" tagx="volume" title="volume">42</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">437</span><span class="lpage" tagx="lpage" title="lpage">461</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18066953">18066953</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohof</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quillan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poo</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Asymmetric modulation of cytosolic cAMP activity induces growth cone turning</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="volume" tagx="volume" title="volume">12</span><span class="year" tagx="year" title="year">1992</span><span class="fpage" tagx="fpage" title="fpage">1253</span><span class="lpage" tagx="lpage" title="lpage">1261</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1372932">1372932</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Witke</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwiatkowski</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosik</span><span class="given-names" tagx="given-names" title="given-names">K.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Delayed retraction of filopodia in gelsolin null mice</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">138</span><span class="year" tagx="year" title="year">1997</span><span class="fpage" tagx="fpage" title="fpage">1279</span><span class="lpage" tagx="lpage" title="lpage">1287</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9298983">9298983</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luarte</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornejo</span><span class="given-names" tagx="given-names" title="given-names">V.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertin</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallardo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Couve</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The axonal endoplasmic reticulum: One organelle-many functions in development, maintenance, and plasticity</span><span class="source" tagx="source" title="source">Dev. Neurobiol.</span><span class="volume" tagx="volume" title="volume">78</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">208</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29134778">29134778</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trinh</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wexler</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourbon</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gatti</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pierre</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavener</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klann</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Suppression of eIF2α kinases alleviates Alzheimer’s disease-related plasticity and memory deficits</span><span class="source" tagx="source" title="source">Nat. Neurosci.</span><span class="volume" tagx="volume" title="volume">16</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">1299</span><span class="lpage" tagx="lpage" title="lpage">1305</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23933749">23933749</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacKintosh</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacKintosh</span><span class="given-names" tagx="given-names" title="given-names">R.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibitors of protein kinases and phosphatases</span><span class="source" tagx="source" title="source">Trends Biochem. Sci.</span><span class="volume" tagx="volume" title="volume">19</span><span class="year" tagx="year" title="year">1994</span><span class="fpage" tagx="fpage" title="fpage">444</span><span class="lpage" tagx="lpage" title="lpage">448</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7855884">7855884</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manns</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cook</span><span class="given-names" tagx="given-names" title="given-names">G.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keynes</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Differing semaphorin 3A concentrations trigger distinct signaling mechanisms in growth cone collapse</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="volume" tagx="volume" title="volume">32</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">8554</span><span class="lpage" tagx="lpage" title="lpage">8559</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22723695">22723695</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshak</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikolakopoulou</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dirks</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martens</span><span class="given-names" tagx="given-names" title="given-names">G.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen-Cory</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell-autonomous TrkB signaling in presynaptic retinal ganglion cells mediates axon arbor growth and synapse maturation during the establishment of retinotectal synaptic connectivity</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="volume" tagx="volume" title="volume">27</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">2444</span><span class="lpage" tagx="lpage" title="lpage">2456</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17344382">17344382</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McConnell</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edward van Veen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vidaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwiatkowski</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A requirement for filopodia extension toward Slit during Robo-mediated axon repulsion</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="volume" tagx="volume" title="volume">213</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">261</span><span class="lpage" tagx="lpage" title="lpage">274</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27091449">27091449</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendoza</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Er</span><span class="given-names" tagx="given-names" title="given-names">E.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blenis</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Ras-ERK and PI3K-mTOR pathways: cross-talk and compensation</span><span class="source" tagx="source" title="source">Trends Biochem. Sci.</span><span class="volume" tagx="volume" title="volume">36</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">320</span><span class="lpage" tagx="lpage" title="lpage">328</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21531565">21531565</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreno</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radford</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peretti</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinert</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verity</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halliday</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinsdale</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortori</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sustained translational repression by eIF2α-P mediates prion neurodegeneration</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">485</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">507</span><span class="lpage" tagx="lpage" title="lpage">511</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22622579">22622579</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nieuwkoop</span><span class="given-names" tagx="given-names" title="given-names">P.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faber</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Normal table of Xenopus laevis (Daudin): A Systematical and Chronological Survey of the Development from the Fertilized Egg till the End of Metamorphosis</span><span class="year" tagx="year" title="year">1994</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Garland</span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nukazuka</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujisawa</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inada</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oda</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takagi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Semaphorin controls epidermal morphogenesis by stimulating mRNA translation via eIF2alpha in Caenorhabditis elegans</span><span class="source" tagx="source" title="source">Genes Dev.</span><span class="volume" tagx="volume" title="volume">22</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">1025</span><span class="lpage" tagx="lpage" title="lpage">1036</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18413715">18413715</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Padmanabhan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuong</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hochstrasser</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assembly of an Evolutionarily Conserved Alternative Proteasome Isoform in Human Cells</span><span class="source" tagx="source" title="source">Cell Rep.</span><span class="volume" tagx="volume" title="volume">14</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">2962</span><span class="lpage" tagx="lpage" title="lpage">2974</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26997268">26997268</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavitt</span><span class="given-names" tagx="given-names" title="given-names">G.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ron</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">New insights into translational regulation in the endoplasmic reticulum unfolded protein response</span><span class="source" tagx="source" title="source">Cold Spring Harb. Perspect. Biol.</span><span class="volume" tagx="volume" title="volume">4</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">a012278</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22535228">22535228</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Preitner</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowakowski</span><span class="given-names" tagx="given-names" title="given-names">D.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hancock</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tcherkezian</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young-Pearse</span><span class="given-names" tagx="given-names" title="given-names">T.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flanagan</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">APC is an RNA-binding protein, and its interactome provides a link to neural development and microtubule assembly</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">158</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">368</span><span class="lpage" tagx="lpage" title="lpage">382</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25036633">25036633</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quevedo</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salinas</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcázar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Initiation factor 2B activity is regulated by protein phosphatase 1, which is activated by the mitogen-activated protein kinase-dependent pathway in insulin-like growth factor 1-stimulated neuronal cells</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">278</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">16579</span><span class="lpage" tagx="lpage" title="lpage">16586</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12624094">12624094</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ritchie</span><span class="given-names" tagx="given-names" title="given-names">M.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phipson</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Law</span><span class="given-names" tagx="given-names" title="given-names">C.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smyth</span><span class="given-names" tagx="given-names" title="given-names">G.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">limma powers differential expression analyses for RNA-sequencing and microarray studies</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">43</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">e47</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25605792">25605792</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roque</span><span class="given-names" tagx="given-names" title="given-names">C.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">J.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tumor protein Tctp regulates axon development in the embryonic visual system</span><span class="source" tagx="source" title="source">Development</span><span class="volume" tagx="volume" title="volume">143</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">1134</span><span class="lpage" tagx="lpage" title="lpage">1148</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26903505">26903505</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadighi Akha</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harper</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salmon</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schroeder</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyra</span><span class="given-names" tagx="given-names" title="given-names">H.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rutkowski</span><span class="given-names" tagx="given-names" title="given-names">D.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Heightened induction of proapoptotic signals in response to endoplasmic reticulum stress in primary fibroblasts from a mouse model of longevity</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">286</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">30344</span><span class="lpage" tagx="lpage" title="lpage">30351</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21757703">21757703</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">E.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clavarino</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ceppi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pierre</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">SUnSET, a nonradioactive method to monitor protein synthesis</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">275</span><span class="lpage" tagx="lpage" title="lpage">277</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19305406">19305406</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujii</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovary</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Genuth</span><span class="given-names" tagx="given-names" title="given-names">N.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Röst</span><span class="given-names" tagx="given-names" title="given-names">H.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teruel</span><span class="given-names" tagx="given-names" title="given-names">M.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barna</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Heterogeneous Ribosomes Preferentially Translate Distinct Subpools of mRNAs Genome-wide</span><span class="source" tagx="source" title="source">Mol. Cell</span><span class="volume" tagx="volume" title="volume">67</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">71</span><span class="lpage" tagx="lpage" title="lpage">83.e7</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28625553">28625553</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shigeoka</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner-Bridger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohk</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">J.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amieux</span><span class="given-names" tagx="given-names" title="given-names">P.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dynamic Axonal Translation in Developing and Mature Visual Circuits</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">166</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">192</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27321671">27321671</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidrauski</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Acosta-Alvear</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khoutorsky</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vedantham</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hearn</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gamache</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallagher</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ang</span><span class="given-names" tagx="given-names" title="given-names">K.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pharmacological brake-release of mRNA translation enhances cognitive memory</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">2</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">e00498</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23741617">23741617</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidrauski</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kampmann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hearn</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vedantham</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaishankar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokabe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendez</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newton</span><span class="given-names" tagx="given-names" title="given-names">B.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">E.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pharmacological dimerization and activation of the exchange factor eIF2B antagonizes the integrated stress response</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">4</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">e07314</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25875391">25875391</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoeckli</span><span class="given-names" tagx="given-names" title="given-names">E.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Understanding axon guidance: are we nearly there yet?</span><span class="source" tagx="source" title="source">Development</span><span class="volume" tagx="volume" title="volume">145</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">dev151415</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29759980">29759980</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thoreen</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chantranupong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keys</span><span class="given-names" tagx="given-names" title="given-names">H.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">N.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabatini</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A unifying model for mTORC1-mediated regulation of mRNA translation</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">485</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">109</span><span class="lpage" tagx="lpage" title="lpage">113</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22552098">22552098</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">tom Dieck</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kochen</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanus</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heumüller</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartnik</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nassim-Assir</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merk</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garg</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bunse</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Direct visualization of newly synthesized target proteins in situ</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="volume" tagx="volume" title="volume">12</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">411</span><span class="lpage" tagx="lpage" title="lpage">414</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25775042">25775042</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trinh</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaphzan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharya</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antion</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavener</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoeffer</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klann</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The eIF2α kinase PERK limits the expression of hippocampal metabotropic glutamate receptor-dependent long-term depression</span><span class="source" tagx="source" title="source">Learn. Mem.</span><span class="volume" tagx="volume" title="volume">21</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">298</span><span class="lpage" tagx="lpage" title="lpage">304</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24741110">24741110</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller-Vedam</span><span class="given-names" tagx="given-names" title="given-names">L.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anand</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaishankar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">H.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renslo</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walter</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of the nucleotide exchange factor eIF2B reveals mechanism of memory-enhancing molecule</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">359</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">eaaq0939</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29599213">29599213</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner-Bridger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jakobs</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muresan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franze</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">W.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Single-molecule analysis of endogenous β-actin mRNA trafficking reveals a mechanism for compartmentalized mRNA localization in axons</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">115</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">E9697</span><span class="lpage" tagx="lpage" title="lpage">E9706</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30254174">30254174</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vattem</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wek</span><span class="given-names" tagx="given-names" title="given-names">R.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reinitiation involving upstream ORFs regulates ATF4 mRNA translation in mammalian cells</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">101</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">11269</span><span class="lpage" tagx="lpage" title="lpage">11274</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15277680">15277680</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vidaki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drees</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saxena</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanslots</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taliaferro</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatarakis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burge</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">E.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertler</span><span class="given-names" tagx="given-names" title="given-names">F.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A requirement for Mena, an actin regulator, in local mRNA translation in developing neurons</span><span class="source" tagx="source" title="source">Neuron</span><span class="volume" tagx="volume" title="volume">95</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">608</span><span class="lpage" tagx="lpage" title="lpage">622.e5</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28735747">28735747</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vizcaíno</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Csordas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">del-Toro</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dianes</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griss</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavidas</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayer</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perez-Riverol</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reisinger</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ternent</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">2016 update of the PRIDE database and its related tools</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">44</span><span class="issue" tagx="issue" title="issue">D1</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">D447</span><span class="lpage" tagx="lpage" title="lpage">D456</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26527722">26527722</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuppalanchi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merianda</span><span class="given-names" tagx="given-names" title="given-names">T.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donnelly</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pacheco</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratan</span><span class="given-names" tagx="given-names" title="given-names">R.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willis</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twiss</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lysophosphatidic acid differentially regulates axonal mRNA translation through 5'UTR elements</span><span class="source" tagx="source" title="source">Mol. Cell. Neurosci.</span><span class="volume" tagx="volume" title="volume">50</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">136</span><span class="lpage" tagx="lpage" title="lpage">146</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22522146">22522146</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proud</span><span class="given-names" tagx="given-names" title="given-names">C.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Eukaryotic initiation factor 2B (eIF2B)</span><span class="source" tagx="source" title="source">Int. J. Biochem. Cell Biol.</span><span class="volume" tagx="volume" title="volume">29</span><span class="year" tagx="year" title="year">1997</span><span class="fpage" tagx="fpage" title="fpage">1127</span><span class="lpage" tagx="lpage" title="lpage">1131</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9438375">9438375</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welsh</span><span class="given-names" tagx="given-names" title="given-names">G.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loughlin</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">N.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proud</span><span class="given-names" tagx="given-names" title="given-names">C.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of eukaryotic initiation factor eIF2B: glycogen synthase kinase-3 phosphorylates a conserved serine which undergoes dephosphorylation in response to insulin</span><span class="source" tagx="source" title="source">FEBS Lett.</span><span class="volume" tagx="volume" title="volume">421</span><span class="year" tagx="year" title="year">1998</span><span class="fpage" tagx="fpage" title="fpage">125</span><span class="lpage" tagx="lpage" title="lpage">130</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9468292">9468292</a></span></span></li> <li tag="ref"><a name="bib61" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">J.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ströhl</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roque</span><span class="given-names" tagx="given-names" title="given-names">C.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cioni</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagnetta</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner-Bridger</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laine</span><span class="given-names" tagx="given-names" title="given-names">R.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">W.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaminski</span><span class="given-names" tagx="given-names" title="given-names">C.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RNA docking and local translation regulate site-specific axon remodeling in vivo</span><span class="source" tagx="source" title="source">Neuron</span><span class="volume" tagx="volume" title="volume">95</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">852</span><span class="lpage" tagx="lpage" title="lpage">868.e8</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28781168">28781168</a></span></span></li> <li tag="ref"><a name="bib68" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Targeted Electroporation in the CNS in Xenopus Embryos</span><span class="source" tagx="source" title="source">Methods Mol Biol.</span><span class="volume" tagx="volume" title="volume">1865</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="lpage" tagx="lpage" title="lpage">131</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30151763">30151763</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">C.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kutzler</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kimball</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tabas</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Toll-like receptor activation suppresses ER stress factor CHOP and translation inhibition through activation of eIF2B</span><span class="source" tagx="source" title="source">Nat. Cell Biol.</span><span class="volume" tagx="volume" title="volume">14</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">192</span><span class="lpage" tagx="lpage" title="lpage">200</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22231169">22231169</a></span></span></li> <li tag="ref"><a name="bib64" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sha</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davisson</span><span class="given-names" tagx="given-names" title="given-names">R.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phenformin activates the unfolded protein response in an AMP-activated protein kinase (AMPK)-dependent manner</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="volume" tagx="volume" title="volume">288</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">13631</span><span class="lpage" tagx="lpage" title="lpage">13638</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23548904">23548904</a></span></span></li> <li tag="ref"><a name="bib65" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">B.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwivedy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Hare</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zivraj</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Local translation of extranuclear lamin B promotes axon maintenance</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">148</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">752</span><span class="lpage" tagx="lpage" title="lpage">764</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22341447">22341447</a></span></span></li> <li tag="ref"><a name="bib66" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zivraj</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tung</span><span class="given-names" tagx="given-names" title="given-names">Y.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piper</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gumy</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fawcett</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeo</span><span class="given-names" tagx="given-names" title="given-names">G.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Subcellular profiling reveals distinct and developmentally regulated repertoire of growth cone mRNAs</span><span class="source" tagx="source" title="source">J. Neurosci.</span><span class="volume" tagx="volume" title="volume">30</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">15464</span><span class="lpage" tagx="lpage" title="lpage">15478</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21084603">21084603</a></span></span></li> </ul> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplemental Information</div> <p id="p0330"> </p><div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Document S1. Figures S1–S6 and Table S2</div>  <div class="media" title="media"><a href="mmc1.pdf">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S1. Axonal NSP Changes Induced by the Sema3A-p-eIF2α Pathway, Related to Figure 2, Figure S2, and Table S2</div>  <div class="media" title="media"><a href="mmc2.xlsx">LINK</a></div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Document S2. Article plus Supplemental Information</div>  <div class="media" title="media"><a href="mmc3.pdf">LINK</a></div> </div> <p /> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We thank D. Ron, S.J. Marciniak, and J.E. Chambers for initial input and sharing reagents; M. Halliday, C.G. Roque, and W.A. Harris for valuable discussions; T. Shigeoka for the heatmap diagram; and K. Mooslehner, A. Dwivedy, A. Bellon, and J.Q. Lin for technical assistance. This work was supported by </p><div class="funding-source" title="funding-source">
Wellcome Trust</div> grants (085314/Z/08/Z and 203249/Z/16/Z) and <div class="funding-source" title="funding-source">
ERC</div> Advanced Investigator Grant (322817) (to C.E.H.), <div class="funding-source" title="funding-source">
BBSRC</div> (RG68461) (to R.C.), <div class="funding-source" title="funding-source">
EMBO</div> postdoctoral fellowship (LTF1006-2013) (to C.K.F.), <div class="funding-source" title="funding-source">
Excellence Cluster CellNetworks</div> (to J.K.), <div class="funding-source" title="funding-source">
ERC</div> <div class="funding-source" title="funding-source">
Consolidator Award UPR Neuro</div> (647479) and <div class="funding-source" title="funding-source">
UK DRI</div> award (UKDRI-2001) (to G.R.M.), and the <div class="funding-source" title="funding-source">
Cambridge Trust</div>, <div class="funding-source" title="funding-source">
Croucher Foundation</div>, and <div class="funding-source" title="funding-source">
Sir Edward Youde Memorial Fund</div> (to H.H.-W.W.).<p /> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p id="p0315">R.C. and C.E.H. conceived the project and wrote the manuscript. R.C. designed the experimental strategy. R.C. performed the puromycilation assay, qIF, RT-PCR, pSILAC, bioinformatic analyses, puro-PLA, growth cone gradient assay, western blot, blastomere injection, and DiI. H.H.-W.W. performed the electroporation and branching experiment. C.K.F. carried out SP3-MS and proteomics data processing. All authors commented on the manuscript. C.E.H., J.K., and G.R.M. supervised the project.</p> </div> <div class="declarationofinterests" title="sec"> <div class="title" tagx="title" title="title">
Declaration of Interests</div> <p id="p0320">The authors declare no competing interests.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-supplementary-material" title="supplementary-material"> <p id="p0325">Supplemental Information includes six figures and two tables and can be found with this article online at <a href="10.1016/j.molcel.2018.11.013">https://doi.org/10.1016/j.molcel.2018.11.013</a>.</p> </div> </div> </div>  </body></html>