<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Cell Dev Biol</span><span class="iso-abbrev" title="iso-abbrev">Front Cell Dev Biol</span><span class="publisher-id" title="publisher-id">Front. Cell Dev. Biol.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Cell and Developmental Biology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2296-634X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7249963">7249963</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fcell.2020.00278">10.3389/fcell.2020.00278</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell and Developmental Biology</span>: <span class="subject" title="subject">Original Research</span></div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Circular RNA hsa_circ_0000848 Promotes Trophoblast Cell Migration and Invasion and Inhibits Cell Apoptosis by Sponging hsa-miR-6768-5p</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Hui</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Jianming</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Zhiyong</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Jingxin</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Yong</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Yang</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Bohong</span></span><a href="#aff2"><sup>2</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/850758/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">Jiansheng</span></span><a href="#aff2"><sup>2</sup></a><a href="#c001"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/826495/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Jing</span></span><a href="#aff3"><sup>3</sup></a><a href="#c002"><sup>*</sup></a></span></div><span class="citation_author_institution" id="aff1">[], <sup>1</sup><span class="institution" title="institution">Department of Obstetrics and Gynecology, Shenzhen Maternity and Child Healthcare Hospital, Southern Medical University</span><span class="addr-line" title="addr-line">Shenzhen</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="aff2">[], <sup>2</sup><span class="institution" title="institution">Medical Genetic Center, Shenzhen Maternity and Child Healthcare Hospital, Southern Medical University</span><span class="addr-line" title="addr-line">Shenzhen</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="aff3">[], <sup>3</sup><span class="institution" title="institution">Department of Obstetrics and Gynecology, Nanfang Hospital, Southern Medical University</span><span class="addr-line" title="addr-line">Guangzhou</span><span class="country" tagx="country" title="country">China</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Louis Lefebvre, University of British Columbia, Canada</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Michael Cowley, North Carolina State University, United States; Meghan Riddell, University of Alberta, Canada; Ursula Nosi, University of Toronto, Canada</p> </div> <div class="corresp" title="corresp">
*Correspondence: Jiansheng Xie, <span class="email" tagx="email" title="email">jsxieszmch@aliyun.com</span></div> <div class="corresp" title="corresp">
Jing Li, <span class="email" tagx="email" title="email">lijing090422@126.com</span></div> <div class="fn-type-other" title="other"> <p>This article was submitted to Developmental Epigenetics, a section of the journal Frontiers in Cell and Developmental Biology</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-5-5</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">278</span><span class="history" title="history"><span class="received" title="received">received: 2019-10-12</span><span class="accepted" title="accepted">accepted: 2020-3-31</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Wang, Zhang, Xu, Yang, Xu, Liu, Li, Xie and Li</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Fetal growth restriction (FGR) is a worldwide problem, and a major cause of perinatal morbidity. The precise molecular mechanisms involved in placental development and function during FGR remain poorly understood. Circular RNAs (circRNAs) are important biological molecules associated with disease pathogenesis. However, the role of circRNAs in FGR has not been well studied.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>circRNA expression profiles in placental tissues with and without FGR were identified by circRNA microarray. circRNA expression was verified by quantitative reverse-transcription PCR (RT-qPCR) assay. The effect of hsa_circ_0000848 and hsa-miR-6768-5p on HTR-8 cell apoptosis, migration, and invasion was evaluated. The association between hsa_circ_0000848 and hsa-miR-6768-5p was confirmed by dual luciferase activity and anti-AGO2 RNA immunoprecipitation (RIP) assays. Protein levels were examined via western blotting.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>RT-qPCR results showed that hsa_circ_0000848 expression was significantly down-regulated in FGR placenta. Hsa_circ_0000848 overexpression and hsa-miR-6768-5p inhibitor suppressed apoptosis, and promoted cell migration and invasion. In addition, hsa_circ_0000848 overexpression and hsa-miR-6768-5p inhibitor increased the protein abundance of BCL2, MMP2 and MMP9, and decreased the protein abundance of cleaved caspase-3, cleaved caspase-9, and BAX, whereas hsa_circ_0000848 knockdown caused the opposite effect. Moreover, a significant increase in hsa-miR-6768-5p expression and a negative correlation between hsa_circ_0000848 and hsa-miR-6768-5p were identified in the FGR tissues. Luciferase reporter and RIP assay results revealed binding of hsa-miR-6768-5p to hsa_circ_0000848. Furthermore, hsa-miR-6768-5p overexpression eliminated the effect of hsa_circ_0000848 overexpression in HTR-8 cells.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>hsa_circ_0000848 expression is significantly down-regulated in the FGR placenta. hsa_circ_0000848 promotes trophoblast cell migration and invasion, and inhibits cell apoptosis via the sponging of hsa-miR-6768-5p. Our study provided a novel insight into mechanisms underlying the pathogenesis of FGR, as well as into new strategies for the treatment of FGR.</p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
fetal growth restriction</div> <div class="kwd" title="kwd">
circ_0000848</div> <div class="kwd" title="kwd">
hsa-miR-6768-5p</div> <div class="kwd" title="kwd">
ceRNA</div> <div class="kwd" title="kwd">
human extravillous trophoblast cell line</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Fetal growth restriction (FGR), also known as intrauterine growth restriction (IUGR), is defined as a fetal birth weight less than 2,500 g after 37 weeks gestation, a fetal weight less than two standard deviations below the mean weight for the same gestational age, or a fetal weight below the 10th percentile of the normal weight for the same gestational age (<a href="#B25">Sharma et al., 2016a</a>; <a href="#B7">Figueras and Gratacos, 2017</a>; <a href="#B18">Nardozza et al., 2017</a>). FGR is a worldwide problem, and the major cause of multifarious mortality and morbidity in the neonatal and fetal population (<a href="#B26">Sharma et al., 2016b</a>; <a href="#B18">Nardozza et al., 2017</a>). FGR affects 5–10% of pregnancies, and the stillbirth rate of FGR is approximately 30% (<a href="#B8">Gynecologists, 2013</a>; <a href="#B18">Nardozza et al., 2017</a>). The etiology and pathogenesis of FGR is complex; maternal, fetal, and uteroplacental factors are involved (<a href="#B8">Gynecologists, 2013</a>; <a href="#B18">Nardozza et al., 2017</a>). Increases in inflammatory cytokine levels in maternal plasma can cause FGR (<a href="#B22">Robb et al., 2017</a>). In addition, placental factors are widely responsible for FGR, and poor placental perfusion is the main cause of FGR (<a href="#B18">Nardozza et al., 2017</a>). Inappropriate invasion of trophoblast cells may lead to poor placental perfusion (<a href="#B16">Mifsud and Sebire, 2014</a>; <a href="#B6">Deng et al., 2015</a>). During normal pregnancy, trophoblast cells invade maternal decidua, rapidly proliferate and differentiate, and intrude into the endometrium matrix in a controlled manner. Excessive apoptosis of trophoblast cells is one of the factors leading to deficient trophoblast invasion (<a href="#B21">Reister et al., 2001</a>; <a href="#B11">Huppertz et al., 2005</a>). At present, the precise molecular mechanisms regulating trophoblast cells in FGR are poorly understood. Therefore, identification of potential molecular FGR markers in the placenta, and investigating their roles in both the regulation of inappropriate invasion and the excessive apoptosis of trophoblast cells, is essential to fully understand the etiology and pathogenesis of FGR.</p> <p>Circular RNAs (circRNAs) are ubiquitously present in mammalian cells, and are involved in gene expression regulation at the transcriptional and post-transcriptional levels (<a href="#B24">Salzman, 2016</a>; <a href="#B10">Hsiao et al., 2017</a>). CircRNAs can be formed by circularization of one or more exons, both exons and intron sequences, or intronic sequences by a process known as "back-splicing" (<a href="#B13">Lasda and Parker, 2014</a>). circRNAs have a closed covalent ring structure, so they are resistant to RNA exonuclease and are more stable than linear RNAs and microRNAs (miRNAs) (<a href="#B20">Qu et al., 2015</a>). Many studies have shown that circRNAs can bind to miRNAs and function as "miRNA sponges" to regulate gene transcription (<a href="#B9">Hansen et al., 2013</a>; <a href="#B14">Liu et al., 2019</a>). Accumulating evidence supports that circRNAs are important biological molecules associated with disease pathogenesis (<a href="#B12">Kristensen et al., 2019</a>; <a href="#B32">Yu et al., 2019</a>). <a href="#B27">Shen et al. (2018)</a> identified 44 differentially expressed circRNAs in pig livers using adult IUGR and normal pigs as models, via RNA sequencing. They identified ATF4/miR-214/circRNA7964 and TCF7/miR-22-3p/circRNA16347 as two competing endogenous networks by bioinformatic analysis (<a href="#B27">Shen et al., 2018</a>). In addition, circPAPPA and circTRNC18 play regulatory roles in trophoblast cells (<a href="#B28">Shen et al., 2019</a>; <a href="#B34">Zhou et al., 2019</a>) and a subset of placental cells, thus performing critical functions in fetal growth and survival. These results indicate that circRNAs may play key regulatory roles in FGR. However, the role of circRNAs in FGR has not been well studied.</p> <p>In this study, we first performed microarray analysis to investigate the expression profiles of circRNAs in FGR tissues. Furthermore, we investigated the function and regulatory mechanism of hsa_circ_0000848 in HTR-8 cells, a human extravillous trophoblast (EVT) cell line, to study trophoblast functions including cell fusion, migration, and invasion. This study provided novel insights into the mechanism of the pathogenesis of FGR, as well as into new strategies for FGR treatment.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="tissuesamples" title="sec"> <div class="title" tagx="title" title="title">
Tissue Samples</div> <p>Forty pregnant women with FGR were included in this study, and their detailed characteristics are shown in <a href="#TS1">Supplementary Table S1</a>. FGR was defined as an estimated fetal weight of below the 10th percentile for gestational age, calculated using the Hadlock equation, and confirmed at birth by a neonatal weight below the 10th percentile. Forty gestation-matched placentas from healthy pregnant woman were used as controls, and their detailed characteristics are shown in <a href="#TS1">Supplementary Table S1</a>. All participants had normal platelet counts, normal functioning of the livers and kidneys, normal blood pressure, and normal blood sugar concentration. The pregnant women of both groups were single pregnancy, and there were no other accompanying complications during pregnancy. The collected placenta was divided into four quadrants, and samples from each quadrant were combined for circRNA microarray analysis and RT-qPCR assay. Placental tissues were placed in RNA-later solution (Sangon Biotech, Shanghai, China) immediately after delivery, then frozen in liquid nitrogen. This study was approved by the Ethics Committee of Shenzhen Maternity and Child Health Hospital, Shenzhen, China. All participants provided written informed consent.</p> </div> <div class="circrnamicroarrayanalysis" title="sec"> <div class="title" tagx="title" title="title">
CircRNA Microarray Analysis</div> <p>Six samples (three samples each from FGR tissues and healthy controls) from the first individuals enrolled in this study, were used in the microarray screening. Sample preparation and microarray hybridization were performed according to the standard protocols of Arraystar Circular RNA Microarray Version 2.0 (Arraystar, Rockville, MD, United States). After washing, arrays were scanned with the Agilent G2505C Scanner (Agilent Technologies, Inc., Santa Clara, CA, United States). Agilent Feature Extraction software (version 11.0.1.1) was used to extract raw data. Quantile normalization was conducted using customized R software package. circRNAs having fold changes &amp;gt;1.5 and <i>p</i>-values &amp;lt;0.05 were determined as being significantly differentially expressed. CircRNA/microRNA interactions were predicted using Arraystar’s home-made miRNA target prediction software. Microarray analysis was performed by the KangChen Bio-tech company (Shanghai, China).</p> </div> <div class="cellcultureandtransfection" title="sec"> <div class="title" tagx="title" title="title">
Cell Culture and Transfection</div> <p>HTR-8/SVneo cells were obtained from the American Type Culture Collection (ATCC, Manassas, VA, United States), and cultured in RPMI 1640 medium (Gibco, Carlsbad, CA, United States), supplemented with 10% fetal bovine serum (Gibco), 1% penicillin/streptomycin, and 1% <div class="sc_UNKNOWN">
L</div>-glutamine at 37°C, in humidified air with 5% CO<sub>2</sub>.</p> <p>Full length hsa_circ_0000848 (position: chr18:46470601-46470865; spliced length: 264 nt) was cloned into the pLCD5H-ciR plasmid between <i>Eco</i>RI and <i>Bam</i>HI sites (Guangzhou Geneseed Biotech Co., Ltd., China), by DNA synthesis <i>in vitro</i>, to construct the hsa_circ_0000848 overexpression vector (ov-circ_0000848). Empty pLCD5H-ciR plasmid was used as a negative control (NC). Small interfering RNA (siRNA) targeting hsa_circ_0000848 (si-circ_0000848; sense sequence: AAUGUGGGGGUGGAGGCUUTT), negative control siRNA (si-NC; sense sequence: UUCUCCG AACGUGUCACGUTT), hsa-miR-6768-5p mimic (CACAC AGGAAAAGCGGGGCCCUG), hsa-miR-6768-5p inhibitor (CA GGGCCCCGCUUUUCCUGUGUG), negative control miRNA inhibitor (miR-NC inhibitor; UCUACUCUU UCUAGGAGGUUGUGA), and negative control miRNA (miR-NC; UCACAACCUCCUAGAAAGAGUAGA) were obtained from GenePharma Co., Ltd. (Shanghai, China).</p> <p>HTR-8 cells (2 × 10<sup>5</sup> cells/well) were seeded in six-well plates and grown overnight. On the following day, cells were washed, placed in serum-free medium, then transfected with the above vector (1 μg/per well), siRNAs (final concentration, 50 nM), and miRNAs (final concentration, 50 nM) according to the manufacturer’s protocol using Lipofectamine<sup>TM</sup> 2000 Reagent (Invitrogen, Carlsbad, CA, United States). After 6 h, the medium was replaced with complete medium, and cells were cultured at 37°C, in humidified air with 5% CO<sub>2</sub>. After transfection for 48 h, cells were harvested for the following assays, except for the cell migration and invasion assays.</p> </div> <div class="rnaisolationandquantitativereverse-transcriptionpcr(rt-qpcr)assay" title="sec"> <div class="title" tagx="title" title="title">
RNA Isolation and Quantitative Reverse-Transcription PCR (RT-qPCR) Assay</div> <p>Total RNA was extracted from tissue samples or cells using TRIzol reagent (Invitrogen). Total RNAs were quantified using a NanoDrop ND-1000 spectrophotometer. After incubation for 15 min at 37°C with 3 U/μg of RNase R (Epicentre Biotechnologies, Madison, WI, United States), the RNA was used to obtain cDNA using the ImProm-IITM Reverse Transcription System (Promega, Madison, WI, United States). Random primers were used for reverse transcription to detect circRNAs. To detect miRNA expression, the primers used for reverse transcription and PCR detection were designed based on the stem-loop principle. Quantitative RT-PCR (RT-qPCR) analysis was performed using SYBR GREEN qPCR Super Mix (Promega, United States). The primers used for RT-qPCR are shown in <a href="#TS2">Supplementary Table S2</a>. GAPDH was used as an internal control for circRNAs and mRNAs. U6 was used as an internal control for miRNAs. The RT-qPCR assay was performed using three independent replicates. Relative expression level of RNA was calculated using the 2<sup>–</sup><sup>Δ</sup><sup>Δ</sup><sup><i>C</i><i>t</i></sup> method, in which one sample from the FGR or normal group was randomly chosen and set to 1.</p> </div> <div class="bioinformaticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatic Analysis</div> <p>The detailed information of circRNAs was obtained from circBase. Target miRNAs of hsa_circ_0000848 were predicted using the TargetScan and miRanda databases. The target mRNAs of hsa-miR-6768-5p were predicted using MirAncesTar, miRDB, Mirza-G, and RNAhybrid. The signaling pathways and biological processes of hsa-miR-6768-5p targets were analyzed using the KEGG (Kyoto Encyclopedia of Genes and Genomes) or GO (Gene Ontology) databases.</p> </div> <div class="dualluciferaseactivityassay" title="sec"> <div class="title" tagx="title" title="title">
Dual Luciferase Activity Assay</div> <p>For the construction of the recombinant luciferase reporter plasmid, fragments of wild-type hsa_circ_0000848 containing the binding sequences of hsa-miR-6768-5p were amplified and cloned into the psi-CHECK-2 vector (Promega) and termed ‘wt-circ_0000848’. The binding sequences of hsa-miR-6768-5p in the recombinant plasmids were mutated via site-directed mutagenesis using one-step overlap extension PCR and termed "mut-circ_0000848." HTR-8 cells were plated on 24-well plates, and co-transfected with 100 ng recombinant plasmid and 50 nM of hsa-miR-6768-5p mimic or miR-NC. Forty-eight hours after transfection, Firefly and Renilla luciferase activities were measured using the Dual-Luciferase Reporter Assay System (Promega), according to the manufacturer’s instructions. Relative luciferase activity was calculated using the ratio of Renilla and Firefly luciferase activities (R/F). Three independent experiments were performed.</p> </div> <div class="anti-ago2rnaimmunoprecipitation(rip)assay" title="sec"> <div class="title" tagx="title" title="title">
Anti-AGO2 RNA Immunoprecipitation (RIP) Assay</div> <p>The RIP assay was carried out according to the Magna RIP RNA-Binding Protein Immunoprecipitation Kit instructions (Millipore, Bedford, MA, United States). Briefly, approximately 1 × 10<sup>7</sup> cells, transfected with hsa-miR-6768-5p mimic or miR-NC, were lysed in polysome lysis buffer including protease inhibitors. Before immunoprecipitation, partial cell lysate was collected for use as a positive control for the RIP assay, termed "input." Subsequently, 100 μl of cell lysate was incubated with magnetic bead-antibody (IgG or Ago2) complex at 4°C, overnight, to pulldown the complex that binds to hsa-miR-6768-5p mimic or miR-NC. RNA in the cell lysate of the positive control and RIP products was extracted and purified. The levels of hsa_circ_0000848 in purified RNA were detected by RT-qPCR. All assays were performed using three independent replicates.</p> </div> <div class="cellmigrationandinvasionassay" title="sec"> <div class="title" tagx="title" title="title">
Cell Migration and Invasion Assay</div> <p>After transfection for 24 h, HTR-8 cells were starved for 24 h and subsequently suspended in serum-free medium. Transwell chambers (Corning, Steuben County, NY, United States) with 24 wells and 8 μm pores were coated with fibronectin for the migration assay, or with Matrigel (BD Biosciences, Bedford, MA, United States) for the invasion assay. HTR-8 cells were seeded in the upper chambers. After 24 h of incubation at 37°C, cells that had successfully migrated onto the lower membrane surface, or had successfully invaded through the membrane, were fixed in 4% formaldehyde and stained with 0.5% Crystal Violet. The number of migrating or invading cells was counted using a light microscope (Nikon company, Tokyo, Japan). All assays were performed using three independent replicates.</p> </div> <div class="cellapoptosisexperiment" title="sec"> <div class="title" tagx="title" title="title">
Cell Apoptosis Experiment</div> <p>The apoptosis assay was performed using the Annexin V-FITC/PI Apoptosis Detection Kit (KeyGEN Biotech, Nanjing, China). Briefly, 2 × 10<sup>5</sup> cells/well were seeded into six-well plates. Forty-eight hours after transfection, cells were harvested and washed with cold PBS. Then, cells were incubated with 5 μl Annexin V-FITC and 5 μl propidium iodide (PI) on ice for 5 min in the dark. Apoptosis of HTR-8 cells was analyzed using a BD FACSCalibur flow cytometer following the manufacturer’s instructions. Cell apoptosis data were interpreted using the FlowJo software version 8 (FlowJo, Ashland, OR, United States). The percentages of early (Annexin V-FITC<sup>+</sup>/PI<sup>–</sup>) and late (Annexin V-FITC<sup>+</sup>/PI<sup>+</sup>) apoptotic cells (as calculated by the number of early or late cells over the total number of all apoptotic cells) were analyzed to evaluate the effect on apoptosis. All assays were performed using three independent replicates.</p> </div> <div class="westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Western Blot Analysis</div> <p>Total proteins were extracted using the RIP assay strong lysis buffer (Beyotime, Shanghai, China). Protein concentrations were quantified using the Bio-Rad Protein Assay Kit (Bio-Rad Laboratories, Hercules, CA, United States). Sodium dodecyl sulfate-polyacrylamide gel electrophoresis was then performed to separate the proteins, which were then transferred onto a polyvinylidene fluoride membrane. Then, the membranes were incubated with primary antibodies (BLC2, 1:1,000, ab32124; MMP2, 1:2,000, ab215986; MMP9, 1:1,000, ab219372; caspase-3, 1:1,500, ab32351; caspase-9, 1:500, ab2324; and BAX, 1:1,000, ab199677; all purchased from Abcam, Cambridge, MA, United States) overnight at 4°C. Afterward, the membranes were washed and incubated with a secondary antibody (anti-rabbit IgG, HRP-linked Antibody, #7074, 1:5,000; Cell Signaling Technology, Beverly, MA, United States) at room temperature for 2 h. Finally, the bands on the membranes were visualized using enhanced chemiluminescence (Pierce, Rockford, IL, United States), and signals were exposed to X-ray. All films were scanned and densitometric analysis was performed three times using Image Pro-Plus 6.0 software (Media Cybernetics, Silver Spring, MD, United States). The relative protein abundance was expressed as the ratio of the target protein densitometric value to the GAPDH densitometric value.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analysis</div> <p>Statistical analyses were performed using SPSS version 18.0 (IBM Corp, Armonk, NY, United States) and GraphPad Prism version 6.0 (GraphPad Software, San Diego, CA, United States). Data were expressed as mean ± standard deviation (SD) based on three separate experiments. Differences between the two experimental groups were analyzed using unpaired <i>t</i>-tests when the data was normally distributed, or non-parametric tests when the data was not normally distributed. The correlation between hsa_circ_0000848 and hsa-miR-6768-5p in the FGR placental tissues was analyzed by linear regression analysis using GraphPad Prism version 7.0 (GraphPad Software). <i>P</i> &amp;lt; 0.05 was considered statistically significant.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="characteristicsoftheparticipants" title="sec"> <div class="title" tagx="title" title="title">
Characteristics of the Participants</div> <p>As shown in <a href="#TS1">Supplementary Table S1</a>, the average maternal ages in the FGR and normal groups were 30.58 ± 5.41 years, and 30.53 ± 4.62 years, respectively, and were not significantly different between the groups (<i>P</i> = 0.96). The average gestation times in the FGR and normal groups were 37.77 ± 1.6 weeks, and 38.46 ± 0.88 weeks, respectively, which were significantly different (<i>P</i> = 0.0156). The birth weights in the FGR and normal groups were 2,231 ± 352 g, and 3,385.75 ± 429 g, respectively, and were significantly differenct (<i>P</i> &amp;lt; 0.0001).</p> </div> <div class="identificationofdifferentiallyexpressedcircrnasinfgrplacenta" title="sec"> <div class="title" tagx="title" title="title">
Identification of Differentially Expressed circRNAs in FGR Placenta</div> <p>To study the expression pattern of circRNAs in FGR and normal placental tissues, we used a circRNA microarray to identify differentially expressed circRNAs. Hierarchical clustering distinguished circRNA expression levels in the associated heat map (<a href="#F1">Figure 1A</a>). The red points in the Volcano plots represent the statistically significantly differentially expressed circRNAs between the FGR and normal groups based on statistical criteria of fold-change &amp;gt;1.5 and <i>P</i> &amp;lt; 0.05 (<a href="#F1">Figure 1B</a>). Based on our statistical criteria, 244 circRNAs were found to be differentially expressed between the placental tissues of the FGR and control groups. For high stringency, the definition for a differentially regulated gene also requires FDR &amp;lt; 0.05. FDR, fully named as False Discovery Rate, is a compromising point of view regarding the error in multiple comparisons suggested by Benjamini and Hochberg (<a href="#B4">Benjamini et al., 2001</a>). In our circRNA microarray data, the FDRs of the differentially expressed circRNAs are &amp;gt;0.05. Nevertheless, the results of our microarray analysis were used for further characterization of differentially expressed circRNA based on our initial statistical criteria.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Bioinformatic analysis of the results of circRNA microarray analysis. <b>(A)</b> Hierarchical clustering of differentially expressed circRNAs in FGR vs Normal samples (N). The results of hierarchical clustering show a distinguishable circRNA expression profile among samples. Black: no change; red: upregulation; green: down-regulation. <b>(B)</b> Volcano plots of differentially expressed circRNAs in FGR vs Normal samples (N). Volcano plots are useful tools for visualizing differential expression between two different conditions. The red point in the plot represents the statistically significant differentially expressed circRNAs.</p>   </div> </div> <div class="hsa_circ_0000848expressionwassignificantlydown-regulatedinfgrplacenta" title="sec"> <div class="title" tagx="title" title="title">
Hsa_circ_0000848 Expression Was Significantly Down-Regulated in FGR Placenta</div> <p>To verify circRNA expression in the FGR placenta, we chose hsa_circ_0007738, hsa_circ_0071271, and hsa_circ_0000848, which had the greatest expression differences between the FGR and normal groups (fold-change &amp;gt;1.5 and <i>P</i> &amp;lt; 0.05), for further analysis. Hsa_circ_0000848 expression level was significantly lower in 12 FGR placental tissues than in 12 normal placenta tissues (<i>P</i> = 0.0257), whereas hsa_circ_0007738 (<i>P</i> = 0.5512) and hsa_circ_0071271 (<i>P</i> = 0.6297) expression levels did not change significantly (<a href="#F2">Figures 2A–C</a>). Subsequently, hsa_circ_0000848 expression was further verified in FGR and normal placenta tissues (<i>n</i> = 37 each) and was found to be significantly lower in FGR tissues than in normal placenta tissues (<i>P</i> = 0.0081) (<a href="#F2">Figure 2D</a>). Detailed information regarding hsa_circ_0000848, found in circBase, is shown in <a href="#F2">Figure 2E</a>. The position of the splice junctions shown in circBase can be verified by our sequence analysis of the hsa_circ_0000848 fragment in RT-qPCR (<a href="#F2">Figure 2F</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Hsa_circ_0000848 expression was significantly down-regulated in FGR placenta. <b>(A–C)</b> Expression levels of hsa_circ_0007738, hsa_circ_0071271, and hsa_circ_0000848 in FGR and normal placental tissues (<i>n</i> = 12). <b>(D)</b> Expression levels of hsa_circ_0000848 in FGR and normal placental tissues (37). <b>(E)</b> Detailed information of hsa_circ_0000848 from circBase. <b>(F)</b> Sequencing analysis around the splice junction of hsa_circ_0000848 fragment in RT-qPCR. FGR vs normal using a non-parametric test, n = 37.</p>   </div> </div> <div class="hsa_circ_0000848inhibitedapoptosisandpromotedmigrationandinvasionofhtr-8cells" title="sec"> <div class="title" tagx="title" title="title">
Hsa_circ_0000848 Inhibited Apoptosis and Promoted Migration and Invasion of HTR-8 Cells</div> <p>To evaluate the functional role of hsa_circ_0000848 in FGR, the HTR-8 cell line was studied. As shown in <a href="#F3">Figure 3A</a>, hsa_circ_0000848 expression levels in HTR-8 cells of the ov-circ_0000848 group were 44-fold higher than those of the NC group, indicating that hsa_circ_0000848 was successfully overexpressed in HTR-8 cells. As shown in <a href="#F3">Figure 3B</a>, hsa_circ_0000848 expression levels in HTR-8 cells of the si-circ_0000848 group were 77% lower than those of the si-NC group, indicating that hsa_circ_0000848 was successfully knocked down in HTR-8 cells.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Effect of hsa_circ_0000848 overexpression or knockdown on apoptosis of HTR-8 cells. <b>(A)</b> hsa_circ_0000848 is overexpressed in HTR-8 cells after transfection with hsa_circ_0000848 containing plasmid (ov-circ_0000848) compared to negative control plasmid (NC). *<i>P</i> &amp;lt; 0.05, when compared to NC using unpaired <i>t</i>-tests. <b>(B)</b> hsa_circ_0000848 expression is lower in HTR-8 cells after transfection with siRNA targeting hsa_circ_0000848 (si-circ_0000848) compared to that in NC siRNA (si-NC). *<i>P</i> &amp;lt; 0.05, when compared to si-NC using unpaired <i>t</i>-tests. <b>(C,D)</b> Effect of hsa_circ_0000848 overexpression or knockdown on the apoptosis of HTR-8 cells. Panel <b>C</b> shows quantification of the percentage of apoptotic cells. Panel <b>D</b> shows representative graphs of the apoptosis detected using a flow cytometer. <b>(E)</b> Effect of hsa_circ_0000848 overexpression or knockdown on protein abundance of MMP-2 and MMP-9 in HTR-8 cells. The left panel includes representative images of western blots. The right panel graphs represent the quantification of relative protein abundance. *<i>P</i> &amp;lt; 0.05, when compared to NC using unpaired <i>t</i>-tests. <sup>#</sup><i>P</i> &amp;lt; 0.05, when compared to si-NC using unpaired <i>t</i>-tests.</p>   </div> <p>We subsequently characterized the role of hsa_circ_0000848 in HTR-8 cell apoptosis. As shown in <a href="#F3">Figures 3C,D</a>, the percentage of apoptotic cells in the ov-circ_0000848 group was clearly lower than that in the NC group, and the percentage of apoptotic cells in the si-circ_0000848 group was clearly more than that in the si-NC group. In addition, the abundance of the apoptosis-related proteins cleaved caspase-3, cleaved caspase-9, BCL2, and BAX was also determined. The abundance of BCL2 was higher in the ov-circ_0000848 group than in the NC group, and was lower in the si-circ_0000848 group than in the si-NC group (<a href="#F3">Figure 3E</a>). Moreover, the abundance of cleaved caspase-3, cleaved caspase-9, and BAX was lower in the ov-circ_0000848 group than in the NC group, and was higher in the si-circ_0000848 group than in the si-NC group (<a href="#F3">Figure 3E</a>).</p> <p>Next, we investigated the role of hsa_circ_0000848 in the migration and invasion of HTR-8 cells by performing a Transwell assay. The results showed that the number of cells that migrated to, or invaded, the bottom chamber was higher in the ov-circ_0000848 group than in the NC group, and was lower in the si-circ_0000848 group than in the si-NC group (<a href="#F4">Figure 4A</a>). In addition, the abundance of metastasis-related proteins MMP-2 and MMP-9 was also determined. The abundance of MMP-2 and MMP-9 was higher in the ov-circ_0000848 group than in the NC group, and was lower in the si-circ_0000848 group than in the si-NC group (<a href="#F4">Figure 4B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Effect of hsa_circ_0000848 overexpression or knockdown on migration and invasion of HTR-8 cells. <b>(A)</b> The results of transwell assay. The left panel includes representative images of migration and invasion by Transwell assay. The right panel includes bar graphs representing the quantification of migrated or invaded cell numbers. <b>(B)</b> Effect of hsa_circ_0000848 overexpression or knockdown on protein abundance of cleaved caspase-3, cleaved caspase-9, BCL2, and BAX in HTR-8 cells. The left panel includes representative images of western blots. The right panel graphs denote the quantification of relative protein expression levels. *<i>P</i> &amp;lt; 0.05, when compared to NC using unpaired <i>t</i>-tests. <sup>#</sup><i>P</i> &amp;lt; 0.05, when compared to si-NC using unpaired <i>t</i>-tests.</p>   </div> </div> <div class="hsa_circ_0000848directlytargetedhsa-mir-6768-5p" title="sec"> <div class="title" tagx="title" title="title">
Hsa_circ_0000848 Directly Targeted hsa-miR-6768-5p</div> <p>We identified 23 microRNA response elements (MREs) within the sequence of hsa_circ_0000848 using the TargetScan, miRanda, and RNAhybid tools, as shown in <a href="#FS1">Supplementary Figure S1</a>. Total context<sup>++</sup> score (<a href="#B1">Agarwal et al., 2015</a>) was used for prioritizing the potential targets of hsa_circ_0000848. Among these miRNAs shown in <a href="#FS1">Supplementary Figure S1</a>, the context<sup>++</sup> scores of hsa-miR-609, hsa-miR-514b-5p, hsa-miR-6768-5p, hsa-miR-6873-5p, hsa-miR-4330, hsa-miR-4267, and hsa-miR-6764-5p were higher than those of the other miRNAs. Therefore, the seven miRNAs above were chosen to further identify the direct target.</p> <p>We first investigated the levels of the seven miRNAs in si-circ_0000848 and si-NC HTR-8 cells. We found that only hsa-miR-6768-5p expression was significantly higher in the si-circ_0000848 group than in the si-NC group (<i>P</i> = 0.0116) (<a href="#F5">Figure 5A</a>), indicating that hsa_circ_0000848 knockdown increased hsa-miR-6768-5p levels. Then, hsa-miR-6768-5p expression was further verified in 37 FGR and 37 normal placental tissues. The results indicated that hsa-miR-6768-5p expression was significantly higher in the FGR placental tissues (<i>n</i> = 37) than in the normal placental tissues (<i>n</i> = 37) (<a href="#F5">Figure 5B</a>). Linear regression analysis results also yielded a significant negative correlation between hsa-miR-6768-5p and hsa_circ_0000848 levels (<i>r</i> = −0.3285, <i>P</i> = 0.0471) (<a href="#F5">Figure 5C</a>). Moreover, a dual luciferase activity assay was carried out to confirm the binding of hsa-miR-6768-5p to linear hsa_circ_0000848 cloned into psi-CHECK2. The binding site of hsa-miR-6768-5p on hsa_circ_0000848 and the effect of the mutant hsa_circ_0000848 on the dual luciferase activity assay are shown in <a href="#F6">Figure 6A</a>. The dual luciferase activity assay results showed that relative luciferase activity was lower after hsa-miR-6768-5p mimic transfection than after NC transfection, while relative luciferase activity was higher after hsa-miR-6768-5p inhibitor transfection than after miR-NC inhibitor transfection (<a href="#F6">Figure 6B</a>), indicating that hsa-miR-6768-5p mimic bound to the sequence of hsa_circ_0000848 cloned into psi-CHECK2. Finally, an anti-AGO2 RIP assay was carried out to further investigate whether hsa-miR-6768-5p binds to hsa_circ_0000848. Since AGO2 binds to all miRNAs, the hsa-miR-6768-5p mimic was transfected into HTR-8 cells to increase the percentage of the hsa-miR-6768-5p-AGO2 complex. RIP assay results showed that hsa_circ_0000848 levels were higher in the RIP product of miR-6768-5p-transfected cells than those in the miR-NC-transfected cells (<a href="#F6">Figure 6C</a>), indicating that hsa-miR-6768-5p bound to endogenous hsa_circ_0000848 in HTR-8 cells.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 5</span></a></div>   <p>hsa-miR-6768-5p expression is up-regulated in FGR placenta and is negatively correlated with hsa_circ_0000848. <b>(A)</b> Expression of hsa-miR-609, hsa-miR-514b-5p, hsa-miR-6768-5p, hsa-miR-6873-5p, hsa-miR-4330, hsa-miR-4267, and hsa-miR-6764-5p in HTR-8 cells transfected with siRNA targeting hsa_circ_0000848 (si-circ_0000848) or negative control siRNA (si-NC). *<i>P</i> &amp;lt; 0.05 using unpaired <i>t</i>-tests. <b>(B)</b> hsa-miR-6768-5p expression in FGR or normal placental tissues (<i>n</i> = 37). Differences between FGR and normal groups were analyzed using non-parametric tests. <b>(C)</b> Linear regression analysis to analyze the correlation of hsa_circ_0000848 and hsa-miR-6768-5p in FGR placental tissues (<i>n</i> = 37).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 6</span></a></div>   <p>hsa-miR-6768-5p can bind to hsa_circ_0000848 in HTR-8 cells. <b>(A)</b> The binding site of hsa-miR-6768-5p on hsa_circ_0000848 and the mutant information for hsa_circ_0000848 in the dual luciferase activity assay. <b>(B)</b> Results of the dual luciferase activity assay. <b>(C)</b> Results of the anti-AGO2 immunoprecipitation (RIP) assay. hsa_circ_0000848 levels in purified RNA collected from products of anti-AGO2 or anti-IgG RIP assay and cell lysate before immunoprecipitation (input) was detected by RT-qPCR. *<i>P</i> &amp;lt; 0.05 using unpaired <i>t</i>-tests. wt, wild type; mut, mutant.</p>   </div> </div> <div class="hsa-mir-6768-5poverexpressioneliminatedtheeffectofhsa_circ_0000848overexpressioninhtr-8cells" title="sec"> <div class="title" tagx="title" title="title">
Hsa-miR-6768-5p Overexpression Eliminated the Effect of hsa_circ_0000848 Overexpression in HTR-8 Cells</div> <p>To further confirm whether hsa_circ_0000848 acts by sponging hsa-miR-6768-5p, hsa_circ_0000848 overexpression plasmid and hsa-miR-6768-5p mimics were co-transfected into HTR-8 cells. Cells transfected with NC plus miR-NC or ov-circ_0000848 + miR-NC were used as controls. As shown in <a href="#F7">Figure 7A</a>, the number of cells that migrated to, or invaded, the bottom chamber was lower in the ov-circ_0000848 + miR-6768-5p group than in the ov-circ_0000848 + miR-NC group. As shown in <a href="#F7">Figure 7B</a>, the percentage of apoptotic cells was clearly higher in the ov-circ_0000848 + miR-6768-5p group than in the ov-circ_0000848 + miR-NC group. Moreover, the protein abundance of MMP-2, MMP-9, cleaved caspase-3, cleaved caspase-9, BCL2, and BAX was determined. The results showed that the abundance of BCL2, MMP-2, and MMP-9 was lower in the ov-circ_0000848 + miR-6768-5p group than in the ov-circ_0000848 + miR-NC group (<a href="#F7">Figure 7C</a>). The protein abundance of cleaved caspase-3, cleaved caspase-9, and BAX was higher in the ov-circ_0000848 + miR-6768-5p group than in the ov-circ_0000848 + miR-NC group (<a href="#F7">Figure 7C</a>). Furthermore, we found that the number of migrated or invaded cells, the percentage of apoptotic cells, and the protein abundance of MMP-2, MMP-9, cleaved caspase-3, cleaved caspase-9, BCL2, and BAX were not significantly different between the ov-circ_0000848 + miR-6768-5p group and NC + miR-NC group (<a href="#F7">Figure 7</a>), indicating that overexpressing miR-6768-5p and hsa_circ_0000848 simultaneously did not induce obvious phenotypic changes in HTR-8 cells. All these results indicate that hsa-miR-6768-5p overexpression eliminated the effect of hsa_circ_0000848 overexpression in HTR-8 cells.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 7</span></a></div>   <p>hsa-miR-6768-5p overexpression eliminated the effect of hsa_circ_0000848 overexpression in HTR-8 cells. To confirm whether hsa_circ_0000848 acts by sponging hsa-miR-6768-5p, hsa_circ_0000848 overexpressing plasmid and hsa-miR-6768-5p mimics were co-transfected into HTR-8 cells to carry out rescue assays. Cells transfected with NC plus miR-NC or ov-circ_0000848 + miR-NC were used as controls. The results showed that hsa-miR-6768-5p overexpression eliminated the effect of hsa_circ_0000848 overexpression on migration, invasion <b>(A)</b>, apoptosis <b>(B)</b> and protein abundance of MMP-2, MMP-9, cleaved caspase-3, cleaved caspase-9, BCL2 and BAX <b>(C)</b> in HTR-8 cells. *<i>P</i> &amp;lt; 0.05 using unpaired <i>t</i>-tests.</p>   </div> </div> <div class="hsa-mir-6768-5pinhibitorsuppressedapoptosisandpromotedmigrationandinvasionofhtr-8cells" title="sec"> <div class="title" tagx="title" title="title">
Hsa-miR-6768-5p Inhibitor Suppressed Apoptosis and Promoted Migration and Invasion of HTR-8 Cells</div> <p>To evaluate the functional role of hsa-miR-6768-5p in FGR, miR-NC inhibitor and miR-6768-5p inhibitor were transfected into HTR-8 cells. We investigated the role of hsa-miR-6768-5p in the migration and invasion of HTR-8 cells by performing a Transwell assay. The results showed that the number of cells that migrated to, or invaded, the bottom chamber was higher in the miR-6768-5p inhibitor group than in the miR-NC inhibitor group (<a href="#F8">Figure 8A</a>). We subsequently characterized the role of hsa-miR-6768-5p in HTR-8 cell apoptosis. As shown in <a href="#F8">Figure 8B</a>, the percentage of apoptotic miR-6768-5p inhibitor group cells was clearly lower than in the miR-NC inhibitor group. The protein abundance of MMP-2, MMP-9, cleaved caspase-3, cleaved caspase-9, BCL2, and BAX was also determined. The abundance of BCL2, MMP-2, and MMP-9 was higher in the miR-6768-5p inhibitor group than in the miR-NC inhibitor group (<a href="#F8">Figure 8C</a>). Furthermore, the protein abundance of cleaved caspase-3, cleaved caspase-9, and BAX was lower in the miR-6768-5p inhibitor group than in the miR-NC inhibitor group (<a href="#F8">Figure 8C</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 8</span></a></div>   <p>Effect of hsa-miR-6768-5p knockdown on apoptosis, migration and invasion of HTR-8 cells. miR-6768-5p inhibitor was transfected into HTR-8 cells to knockdown hsa-miR-6768-5p. miR-NC inhibitor was transfected as a control. <b>(A)</b> The results of the Transwell assay. The left panel includes representative images of migration and invasion by the Transwell assay. The right panel includes bar graphs of the quantification of migrated or invaded cell numbers. <b>(B)</b> Effect of hsa-miR-6768-5p knockdown on apoptosis. The left panel shows quantification of the percentage of apoptotic cells. The right panel shows representative graphs of the apoptosis detected using a flow cytometer. <b>(C)</b> Effect of hsa-miR-6768-5p knockdown on protein abundance of MMP-2, MMP-9, cleaved caspase-3, cleaved caspase-9, BCL2, and BAX in HTR-8 cells. The left panel includes representative images of western blots. The right panel graphs represent the quantification of relative protein expression levels. *<i>P</i> &amp;lt; 0.05 unpaired <i>t</i>-tests.</p>   </div> </div> <div class="predictedtargets,signalingpathwaysandbiologicalprocessesofhsa-mir-6768-5p" title="sec"> <div class="title" tagx="title" title="title">
Predicted Targets, Signaling Pathways and Biological Processes of hsa-miR-6768-5p</div> <p>To preliminarily explore the mechanisms of hsa-miR-6768-5p, we aimed to predict the target mRNAs. As shown in <a href="#TS3">Supplementary Table S3</a>, 8,137 target mRNAs were predicted. Base on the results of GO and KEGG analysis, we found that 482 targets of hsa-miR-6768-5p were implicated in pathways involved in proliferation, apoptosis and metastasis (<a href="#TS4">Supplementary Table S4</a>). Importantly, we identified SMAD7 as one potential target of hsa-miR-6768-5p. According to the information in circBase, hsa_circ_0000848 is located in an intron of the SMAD7 gene.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>FGR is a major cause of perinatal morbidity (<a href="#B18">Nardozza et al., 2017</a>), although the mechanisms are not fully understood. Thus, a better understanding of the underlying molecular mechanisms of FGR progression is needed. CircRNAs are highly conserved and widely expressed in mammalian cells (<a href="#B23">Rybak-Wolf et al., 2015</a>). Unlike miRNAs and long-non-coding RNAs, circRNAs are more stable and less degradable with a covalently closed loop structure (<a href="#B20">Qu et al., 2015</a>). In view of these advantages, circRNAs have great potential as ideal biomarkers for the diagnosis of diseases. Nevertheless, little is known about the role of circRNAs in FGR placenta. Therefore, it is important to obtain a comprehensive view of the circRNA expression profile and function to understand the etiology and pathogenesis of FGR.</p> <p>The placenta plays a decisive role in fetal growth by connecting the mother with the fetus. Therefore, we first examined circRNA expression profiles in three placentas of FGR fetuses. After validating the expression of three candidate circRNAs in additional samples, hsa_circ_0000848 was confirmed to be significantly down-regulated in FGR tissues. The functional role of hsa_circ_0000848 in FGR was evaluated by investigating its role in migration, invasion, and apoptosis of HTR-8 cells. Our results showed that hsa_circ_0000848 overexpression inhibited apoptosis and promoted cell migration and invasion. In addition, we found that hsa_circ_0000848 overexpression increased the abundance of the anti-apoptotic protein BCL2 (<a href="#B30">Wu et al., 2018</a>) and decreased the abundance of the pro-apoptotic proteins (<a href="#B30">Wu et al., 2018</a>) cleaved caspase-3, cleaved caspase-9, and BAX in HTR-8 cells. Furthermore, hsa_circ_0000848 overexpression increased the expression levels of MMP2 and MMP9, which promote cell migration and invasion processes (<a href="#B33">Zhang et al., 2017</a>). Knockdown of hsa_circ_0000848 had the opposite effect to that of hsa_circ_0000848 overexpression. These results indicated that hsa_circ_0000848 may promote survival and metastasis of HTR-8 cells. HTR-8 is a cell line derived from EVT, a subset of placental cells. EVT plays a critical role in fetal growth and survival. EVT cells migrate to, and invade, the uterine wall leading to remodeling of the maternal vasculature (<a href="#B15">Lyall et al., 2013</a>; <a href="#B16">Mifsud and Sebire, 2014</a>; <a href="#B5">Burton and Jauniaux, 2018</a>). Inadequate invasion and migration of EVT cells is one of the triggers of FGR (<a href="#B16">Mifsud and Sebire, 2014</a>; <a href="#B35">Zong et al., 2017</a>). Therefore, we hypothesize that hsa_circ_0000848 plays a role in the etiology and pathogenesis of FGR and may be a potential therapeutic target for FGR. However, our hypothesis needs to be confirmed by further <i>in vivo</i> evidence.</p> <p>Increasing evidence shows that circRNAs are involved in the regulation of gene expression by sequestering miRNAs (<a href="#B19">Patop and Kadener, 2018</a>; <a href="#B14">Liu et al., 2019</a>). CircRNAs bind to miRNAs and function as "miRNA sponges." The inhibitory effect of miRNAs on the target genes then disappears, and the expression of the target gene increases. This process is referred to as the competitive endogenous RNA (ceRNA) mechanism (<a href="#B9">Hansen et al., 2013</a>; <a href="#B14">Liu et al., 2019</a>). We predicted potential targets of hsa_circ_0000848 via bioinformatic analysis to explore the mechanism of its function. We identified hsa-miR-6768-5p as a potential target of hsa_circ_0000848. Further, we identified a significant increase in hsa-miR-6768-5p expression and a negative correlation between hsa_circ_0000848 and hsa-miR-6768-5p expression levels in FGR placental tissues. Moreover, hsa_circ_0000848 knockdown increased hsa-miR-6768-5p expression in HTR-8 cells. These results indicate a close association between hsa_circ_0000848 and hsa-miR-6768-5p. The association between hsa_circ_0000848 and hsa-miR-6768-5p was further verified by luciferase reporter and AGO2 RIP assays. Luciferase reporter assay results revealed that hsa-miR-6768-5p bound to hsa_circ_0000848. Additionally, the RIP assay indicated that hsa-miR-6768-5p interacted with circ_0000848. Taken together, these results suggest that hsa_circ_0000848 acted as a sponge to hsa-miR-6768-5p and that hsa-miR-6768-5p was a direct target of hsa_circ_0000848. Moreover, we found that hsa-miR-6768-5p overexpression eliminated the effect of hsa_circ_0000848 overexpression in HTR-8 cells. This result indicated that hsa-miR-6768-5p was also involved in the regulation of apoptosis, migration, and invasion of HTR-8 cells, and hsa_circ_0000848 played its role, at least partially, by sponging hsa-miR-6768-5p.</p> <p>MiRNAs inhibit target mRNA translation or participate in the degradation of target mRNA (<a href="#B17">Mohr and Mott, 2015</a>; <a href="#B29">Wehbe et al., 2019</a>). We found that there are several potential targets of the hsa_circ_0000848/hsa-miR-6768-5p axis, and complex pathways are involved in apoptosis and metastasis. Therefore, the identification of the hsa_circ_0000848/hsa-miR-6768-5p axis targets is likely a complicated task. However, we found that SMAD7 was one potential target of hsa-miR-6768-5p, meanwhile hsa_circ_0000848 is located in an intron of the SMAD7 gene. SMAD7 has been reported to antagonize TGF-beta signaling in the placenta and plays a role in trophoblast growth and differentiation (<a href="#B31">Xu et al., 2016</a>). It is reported that inhibition of TGF-β signaling may be potentially therapeutic for FGR (<a href="#B3">Alejandre Alcazar et al., 2011</a>; <a href="#B2">Alcazar et al., 2014</a>). Therefore, we intend to systematically investigate the hsa_circ_0000848/hsa-miR-6768-5p/SMAD7 axis in a future study.</p> <p>Our study has some limitations. First, the function of hsa_circ_0000848 was evaluated in the EVT cell line, while its differential expression was identified in samples from term placenta after birth, which is devoid of EVT cells. Therefore, there is a logical gap in examining the function of hsa_circ_0000848 in EVT cells. The experiment was designed this way because there is no way to ethically obtain a developing placenta with trophoblast cells for research. Second, we did not carry out in-depth exploration of other mechanisms that might regulate hsa-miR-6768-5p and hsa_circ_0000848 besides the ceRNA mechanism. Our luciferase reporter assay results and RIP assay did prove that hsa_circ_0000848 could sponge hsa-miR-6768-5p, but RT-qPCR also demonstrated a reduction in hsa -miR-6768-5p after hsa_circ_0000848 overexpression, and upregulation of hsa-miR-6768-5p after hsa_circ_0000848 knockdown. Therefore, we hypothesized that there are other mechanisms that regulate hsa-miR-6768-5p by hsa_circ_0000848. Finally, a small number of human samples were used in the microarray analysis, leading to results that the FDRs of the differentially expressed circRNAs are &amp;gt;0.05.</p> <p>In conclusion, hsa_circ_0000848 expression was significantly down-regulated in FGR placental tissues. Further, hsa_circ_0000848 also promoted trophoblast cell migration and invasion, and inhibited cell apoptosis via sponging hsa-miR-6768-5p. Based on the importance of the role of EVT in FGR (<a href="#B16">Mifsud and Sebire, 2014</a>; <a href="#B35">Zong et al., 2017</a>), we hypothesize that hsa_circ_0000848 plays a role in the etiology and pathogenesis of FGR. Our results provide new avenues for targeted FGR therapy.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>The datasets GENERATED for this study can be found in the NCBI GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147721">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147721</a>.</p> </div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics Statement</div> <p>The studies involving human participants were reviewed and approved by Ethics Committee of Shenzhen Maternity and Child Health Hospital, Shenzhen, China. The patients/participants provided their written informed consent to participate in this study.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>HW, JX, and JL contributed to the conception and design of the study. HW wrote the first version of the manuscript. HW, JZ, and ZX performed the experiments. JY, YX, YL, and BL constructed the figures. JY and YX were involved in the clinical data analysis. All authors contributed to revising the manuscript, and read and approved the submitted version.</p> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This study was supported by grants from the Shenzhen Science and Technology Innovation Committee (JCYJ20170413092818116), and the Shenzhen Maternal and Child Health Hospital In-scientific Research Project (FYB2017025).</p> </div> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fcell.2020.00278/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fcell.2020.00278/full#supplementary-material</a></p> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">FIGURE S1</span> <p>The list of miRNAs that may be potential targets of hsa_circ_0000848.</p>  <div class="media" title="media"><a href="Image_1.TIF">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">TABLE S1</span> <p>Detailed characteristics of participants.</p>  <div class="media" title="media"><a href="Table_1.XLSX">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">TABLE S2</span> <p>Sequences of primers used in this study.</p>  <div class="media" title="media"><a href="Table_2.DOCX">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">TABLE S3</span> <p>Predicted targets of hsa-miR-6768-5p.</p>  <div class="media" title="media"><a href="Table_3.XLSX">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">TABLE S4</span> <p>Predicted targets of hsa-miR-6768-5p implicated in pathways involved in proliferation, apoptosis and metastasis.</p>  <div class="media" title="media"><a href="Table_4.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agarwal</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">G. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nam</span><span class="given-names" tagx="given-names" title="given-names">J. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartel</span><span class="given-names" tagx="given-names" title="given-names">D. P.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Predicting effective microRNA target sites in mammalian mRNAs.</span><span class="source" tagx="source" title="source"><i>eLife</i></span><span class="volume" tagx="volume" title="volume">4</span>:<span class="issue" tagx="issue" title="issue">e5005</span>. <span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.05005">10.7554/eLife.05005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26267216">26267216</a></span></span></li> <li tag="ref"><a name="B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcazar</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinger</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rother</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostreicher</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vohlen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plank</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Prevention of early postnatal hyperalimentation protects against activation of transforming growth factor-beta/bone morphogenetic protein and interleukin-6 signaling in rat lungs after intrauterine growth restriction.</span><span class="source" tagx="source" title="source"><i>J. Nutr.</i></span><span class="volume" tagx="volume" title="volume">144</span><span class="fpage" tagx="fpage" title="fpage">1943</span>–<span class="lpage" tagx="lpage" title="lpage">1951</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3945/jn.114.197657">10.3945/jn.114.197657</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25411031">25411031</a></span></span></li> <li tag="ref"><a name="B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alejandre Alcazar</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morty</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lendzian</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vohlen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oestreicher</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plank</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Inhibition of TGF-beta signaling and decreased apoptosis in IUGR-associated lung disease in rats.</span><span class="source" tagx="source" title="source"><i>PLoS One</i></span><span class="volume" tagx="volume" title="volume">6</span>:<span class="issue" tagx="issue" title="issue">e26371</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0026371">10.1371/journal.pone.0026371</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22028866">22028866</a></span></span></li> <li tag="ref"><a name="B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benjamini</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drai</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmer</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kafkafi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golani</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Controlling the false discovery rate in behavior genetics research.</span><span class="source" tagx="source" title="source"><i>Behav. Brain Res.</i></span><span class="volume" tagx="volume" title="volume">125</span><span class="fpage" tagx="fpage" title="fpage">279</span>–<span class="lpage" tagx="lpage" title="lpage">284</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/s0166-4328(01)00297-2">10.1016/s0166-4328(01)00297-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11682119">11682119</a></span></span></li> <li tag="ref"><a name="B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burton</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jauniaux</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Pathophysiology of placental-derived fetal growth restriction.</span><span class="source" tagx="source" title="source"><i>Am. J. Obstet. Gynecol.</i></span><span class="volume" tagx="volume" title="volume">218</span><span class="fpage" tagx="fpage" title="fpage">S745</span>–<span class="lpage" tagx="lpage" title="lpage">S761</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajog.2017.11.577">10.1016/j.ajog.2017.11.577</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29422210">29422210</a></span></span></li> <li tag="ref"><a name="B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shan</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">N-acetylglucosaminyltransferase V inhibits the invasion of trophoblast cells by attenuating MMP2/9 activity in early human pregnancy.</span><span class="source" tagx="source" title="source"><i>Placenta</i></span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">1291</span>–<span class="lpage" tagx="lpage" title="lpage">1299</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.placenta.2015.08.014">10.1016/j.placenta.2015.08.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26349781">26349781</a></span></span></li> <li tag="ref"><a name="B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Figueras</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gratacos</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">An integrated approach to fetal growth restriction.</span><span class="source" tagx="source" title="source"><i>Best Pract. Res. Clin. Obstet. Gynaecol.</i></span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">48</span>–<span class="lpage" tagx="lpage" title="lpage">58</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bpobgyn.2016.10.006">10.1016/j.bpobgyn.2016.10.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27940123">27940123</a></span></span></li> <li tag="ref"><a name="B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gynecologists</span><span class="given-names" tagx="given-names" title="given-names">A. C. O. O. A.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">ACOG Practice bulletin no. 134: fetal growth restriction.</span><span class="source" tagx="source" title="source"><i>Obstet. Gynecol.</i></span><span class="volume" tagx="volume" title="volume">121</span><span class="fpage" tagx="fpage" title="fpage">1122</span>–<span class="lpage" tagx="lpage" title="lpage">1133</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1097/01.AOG.0000429658.85846.f9">10.1097/01.AOG.0000429658.85846.f9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23635765">23635765</a></span></span></li> <li tag="ref"><a name="B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">T. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">T. I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clausen</span><span class="given-names" tagx="given-names" title="given-names">B. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bramsen</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finsen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Damgaard</span><span class="given-names" tagx="given-names" title="given-names">C. K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Natural RNA circles function as efficient microRNA sponges.</span><span class="source" tagx="source" title="source"><i>Nature</i></span><span class="volume" tagx="volume" title="volume">495</span><span class="fpage" tagx="fpage" title="fpage">384</span>–<span class="lpage" tagx="lpage" title="lpage">388</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature11993">10.1038/nature11993</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23446346">23446346</a></span></span></li> <li tag="ref"><a name="B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsiao</span><span class="given-names" tagx="given-names" title="given-names">K. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">H. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Circular RNA - New member of noncoding RNA with novel functions.</span><span class="source" tagx="source" title="source"><i>Exp. Biol. Med.</i></span><span class="volume" tagx="volume" title="volume">242</span><span class="fpage" tagx="fpage" title="fpage">1136</span>–<span class="lpage" tagx="lpage" title="lpage">1141</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1177/1535370217708978">10.1177/1535370217708978</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28485684">28485684</a></span></span></li> <li tag="ref"><a name="B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huppertz</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hemmings</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renaud</span><span class="given-names" tagx="given-names" title="given-names">S. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bulmer</span><span class="given-names" tagx="given-names" title="given-names">J. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dash</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chamley</span><span class="given-names" tagx="given-names" title="given-names">L. W.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Extravillous trophoblast apoptosis–a workshop report.</span><span class="source" tagx="source" title="source"><i>Placenta</i></span><span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">Suppl. A</span>), <span class="fpage" tagx="fpage" title="fpage">S46</span>–<span class="lpage" tagx="lpage" title="lpage">S48</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15837067">15837067</a></span></span></li> <li tag="ref"><a name="B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kristensen</span><span class="given-names" tagx="given-names" title="given-names">L. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stagsted</span><span class="given-names" tagx="given-names" title="given-names">L. V. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebbesen</span><span class="given-names" tagx="given-names" title="given-names">K. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">T. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kjems</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">The biogenesis, biology and characterization of circular RNAs.</span><span class="source" tagx="source" title="source"><i>Nat. Rev. Genet.</i></span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">675</span>–<span class="lpage" tagx="lpage" title="lpage">691</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41576-019-0158-7">10.1038/s41576-019-0158-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31395983">31395983</a></span></span></li> <li tag="ref"><a name="B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lasda</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Circular RNAs: diversity of form and function.</span><span class="source" tagx="source" title="source"><i>RNA</i></span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">1829</span>–<span class="lpage" tagx="lpage" title="lpage">1842</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1261/rna.047126.114">10.1261/rna.047126.114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25404635">25404635</a></span></span></li> <li tag="ref"><a name="B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">N. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Circ_0015756 promotes proliferation, invasion and migration by microRNA-7-dependent inhibition of FAK in hepatocellular carcinoma.</span><span class="source" tagx="source" title="source"><i>Cell Cycle</i></span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">2939</span>–<span class="lpage" tagx="lpage" title="lpage">2953</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/15384101.2019.1664223">10.1080/15384101.2019.1664223</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31522588">31522588</a></span></span></li> <li tag="ref"><a name="B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyall</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robson</span><span class="given-names" tagx="given-names" title="given-names">S. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bulmer</span><span class="given-names" tagx="given-names" title="given-names">J. N.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Spiral artery remodeling and trophoblast invasion in preeclampsia and fetal growth restriction: relationship to clinical outcome.</span><span class="source" tagx="source" title="source"><i>Hypertension</i></span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">1046</span>–<span class="lpage" tagx="lpage" title="lpage">1054</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1161/HYPERTENSIONAHA.113.01892">10.1161/HYPERTENSIONAHA.113.01892</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24060885">24060885</a></span></span></li> <li tag="ref"><a name="B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mifsud</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sebire</span><span class="given-names" tagx="given-names" title="given-names">N. J.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Placental pathology in early-onset and late-onset fetal growth restriction.</span><span class="source" tagx="source" title="source"><i>Fetal Diagn. Ther.</i></span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">117</span>–<span class="lpage" tagx="lpage" title="lpage">128</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1159/000359969">10.1159/000359969</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24577279">24577279</a></span></span></li> <li tag="ref"><a name="B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohr</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mott</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Overview of microRNA biology.</span><span class="source" tagx="source" title="source"><i>Semin. Liver Dis.</i></span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">3</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1055/s-0034-1397344">10.1055/s-0034-1397344</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25632930">25632930</a></span></span></li> <li tag="ref"><a name="B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nardozza</span><span class="given-names" tagx="given-names" title="given-names">L. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caetano</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zamarian</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazzola</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva</span><span class="given-names" tagx="given-names" title="given-names">C. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcal</span><span class="given-names" tagx="given-names" title="given-names">V. M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Fetal growth restriction: current knowledge.</span><span class="source" tagx="source" title="source"><i>Arch. Gynecol. Obstet.</i></span><span class="volume" tagx="volume" title="volume">295</span><span class="fpage" tagx="fpage" title="fpage">1061</span>–<span class="lpage" tagx="lpage" title="lpage">1077</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00404-017-4341-9">10.1007/s00404-017-4341-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28285426">28285426</a></span></span></li> <li tag="ref"><a name="B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patop</span><span class="given-names" tagx="given-names" title="given-names">I. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadener</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">circRNAs in cancer.</span><span class="source" tagx="source" title="source"><i>Curr. Opin. Genet. Dev.</i></span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">121</span>–<span class="lpage" tagx="lpage" title="lpage">127</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gde.2017.11.007">10.1016/j.gde.2017.11.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29245064">29245064</a></span></span></li> <li tag="ref"><a name="B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Circular RNA: a new star of noncoding RNAs.</span><span class="source" tagx="source" title="source"><i>Cancer Lett.</i></span><span class="volume" tagx="volume" title="volume">365</span><span class="fpage" tagx="fpage" title="fpage">141</span>–<span class="lpage" tagx="lpage" title="lpage">148</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.canlet.2015.06.003">10.1016/j.canlet.2015.06.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26052092">26052092</a></span></span></li> <li tag="ref"><a name="B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reister</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frank</span><span class="given-names" tagx="given-names" title="given-names">H. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingdom</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heyl</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaufmann</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rath</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Macrophage-induced apoptosis limits endovascular trophoblast invasion in the uterine wall of preeclamptic women.</span><span class="source" tagx="source" title="source"><i>Lab. Invest.</i></span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">1143</span>–<span class="lpage" tagx="lpage" title="lpage">1152</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/labinvest.3780326">10.1038/labinvest.3780326</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11502865">11502865</a></span></span></li> <li tag="ref"><a name="B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robb</span><span class="given-names" tagx="given-names" title="given-names">K. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotechini</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allaire</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sperou</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">C. H.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Inflammation-induced fetal growth restriction in rats is associated with increased placental HIF-1alpha accumulation.</span><span class="source" tagx="source" title="source"><i>PLoS One</i></span><span class="volume" tagx="volume" title="volume">12</span>:<span class="issue" tagx="issue" title="issue">e0175805</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0175805">10.1371/journal.pone.0175805</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28423052">28423052</a></span></span></li> <li tag="ref"><a name="B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rybak-Wolf</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stottmeister</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glazar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jens</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pino</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giusti</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Circular RNAs in the mammalian brain are highly abundant. conserved, and dynamically expressed.</span><span class="source" tagx="source" title="source"><i>Mol. Cell</i></span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">870</span>–<span class="lpage" tagx="lpage" title="lpage">885</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molcel.2015.03.027">10.1016/j.molcel.2015.03.027</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25921068">25921068</a></span></span></li> <li tag="ref"><a name="B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Circular RNA expression: its potential regulation and function.</span><span class="source" tagx="source" title="source"><i>Trends Genet.</i></span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">309</span>–<span class="lpage" tagx="lpage" title="lpage">316</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tig.2016.03.002">10.1016/j.tig.2016.03.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27050930">27050930</a></span></span></li> <li tag="ref"><a name="B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farahbakhsh</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shastri</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2016a</span>). <span class="mixed-article-title" title="mixed-article-title">Intrauterine growth restriction - part 2.</span><span class="source" tagx="source" title="source"><i>J. Matern. Fetal. Neonatal. Med.</i></span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">4037</span>–<span class="lpage" tagx="lpage" title="lpage">4048</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26979578">26979578</a></span></span></li> <li tag="ref"><a name="B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shastri</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farahbakhsh</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2016b</span>). <span class="mixed-article-title" title="mixed-article-title">Intrauterine growth restriction - part 1.</span><span class="source" tagx="source" title="source"><i>J. Matern. Fetal. Neonatal. Med.</i></span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">3977</span>–<span class="lpage" tagx="lpage" title="lpage">3987</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26856409">26856409</a></span></span></li> <li tag="ref"><a name="B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">The landscape of non-coding RNA in an adult pig model of intrauterine growth restriction.</span><span class="source" tagx="source" title="source"><i>Cell Physiol. Biochem.</i></span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">1764</span>–<span class="lpage" tagx="lpage" title="lpage">1778</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1159/000494794">10.1159/000494794</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30384377">30384377</a></span></span></li> <li tag="ref"><a name="B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">X. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">L. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">H. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">Y. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">CircTRNC18 inhibits trophoblast cell migration and epithelial-mesenchymal transition by regulating miR-762/Grhl2 pathway in pre-eclampsia.</span><span class="source" tagx="source" title="source"><i>RNA Biol.</i></span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1565</span>–<span class="lpage" tagx="lpage" title="lpage">1573</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/15476286.2019.1644591">10.1080/15476286.2019.1644591</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31354028">31354028</a></span></span></li> <li tag="ref"><a name="B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wehbe</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nasser</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pintus</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Badran</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eid</span><span class="given-names" tagx="given-names" title="given-names">A. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baydoun</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">MicroRNAs in cardiac hypertrophy.</span><span class="source" tagx="source" title="source"><i>Int. J. Mol. Sci.</i></span><span class="volume" tagx="volume" title="volume">20</span>:<span class="issue" tagx="issue" title="issue">4714</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms20194714">10.3390/ijms20194714</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31547607">31547607</a></span></span></li> <li tag="ref"><a name="B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medeiros</span><span class="given-names" tagx="given-names" title="given-names">L. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">K. H.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Apoptosis signaling and BCL-2 pathways provide opportunities for novel targeted therapeutic strategies in hematologic malignances.</span><span class="source" tagx="source" title="source"><i>Blood Rev.</i></span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">8</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.blre.2017.08.004">10.1016/j.blre.2017.08.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28802908">28802908</a></span></span></li> <li tag="ref"><a name="B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivasubramaniyam</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yinon</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tagliaferro</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nevo</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Aberrant TGFbeta Signaling Contributes to Altered Trophoblast Differentiation in Preeclampsia.</span><span class="source" tagx="source" title="source"><i>Endocrinology</i></span><span class="volume" tagx="volume" title="volume">157</span><span class="fpage" tagx="fpage" title="fpage">883</span>–<span class="lpage" tagx="lpage" title="lpage">899</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1210/en.2015-1696">10.1210/en.2015-1696</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26653761">26653761</a></span></span></li> <li tag="ref"><a name="B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">CircRNAs in cancer metabolism: a review.</span><span class="source" tagx="source" title="source"><i>J. Hematol. Oncol.</i></span><span class="volume" tagx="volume" title="volume">12</span>:<span class="issue" tagx="issue" title="issue">90</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13045-019-0776-8">10.1186/s13045-019-0776-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31484561">31484561</a></span></span></li> <li tag="ref"><a name="B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Knockdown of Diaph1 expression inhibits migration and decreases the expression of MMP2 and MMP9 in human glioma cells.</span><span class="source" tagx="source" title="source"><i>Biomed. Pharmacother.</i></span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">596</span>–<span class="lpage" tagx="lpage" title="lpage">602</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biopha.2017.10.031">10.1016/j.biopha.2017.10.031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29035824">29035824</a></span></span></li> <li tag="ref"><a name="B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Down-regulated circPAPPA suppresses the proliferation and invasion of trophoblast cells via the miR-384/STAT3 pathway.</span><span class="source" tagx="source" title="source"><i>Biosci. Rep.</i></span><span class="volume" tagx="volume" title="volume">39</span>:<span class="issue" tagx="issue" title="issue">BSR20191965</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1042/BSR20191965">10.1042/BSR20191965</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31427481">31427481</a></span></span></li> <li tag="ref"><a name="B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gou</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Zinc improves learning and memory abilities of fetal growth restriction rats and promotes trophoblast cell invasion and migration via enhancing STAT3-MMP-2/9 axis activity.</span><span class="source" tagx="source" title="source"><i>Oncotarget</i></span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">115190</span>–<span class="lpage" tagx="lpage" title="lpage">115201</span>. <span class="pub-id"><a href="https://dx.doi.org/10.18632/oncotarget.23122">10.18632/oncotarget.23122</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29383152">29383152</a></span></span></li> </ul> </div> </div>  </body></html>