<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Microbiologyopen</span><span class="iso-abbrev" title="iso-abbrev">Microbiologyopen</span><span class="doi" title="doi">10.1002/(ISSN)2045-8827</span><span class="publisher-id" title="publisher-id">MBO3</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">MicrobiologyOpen</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-8827</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7066466">7066466</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/mbo3.983">10.1002/mbo3.983</a></span><span class="publisher-id" title="publisher-id">MBO3983</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span>: <span class="subject" title="subject">Original Articles</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Molecular ecological network analysis of the response of soil microbial communities to depth gradients in farmland soils</div> <div class="alt-title" title="alt-title">
YU et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Hang</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1894-446X</span><a href="#mbo3983-aff-0001"><sup>1</sup></a><a href="#mbo3983-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Dongmei</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-6697-2078</span><a href="#mbo3983-aff-0001"><sup>1</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">xuedongmei@tjnu.edu.cn</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Yidong</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5399-2803</span><a href="#mbo3983-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Wei</span></span><a href="#mbo3983-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Guilong</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-9705-6646</span><a href="#mbo3983-aff-0003"><sup>3</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">zhangguilong@caas.cn</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Zhong‐Liang</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-3740-4677</span><a href="#mbo3983-aff-0001"><sup>1</sup></a></span></div><span class="citation_author_institution" id="mbo3983-aff-0001">[1], <span class="named-content" title="named-content">Tianjin Key Laboratory of Water Resources and Environment</span><span class="institution" title="institution">Tianjin Normal University</span><div class="unknown" title="city" tag="city">
Tianjin</div><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="mbo3983-aff-0002">[2], <span class="named-content" title="named-content">Tianjin Key Laboratory of Environmental Change and Ecological Restoration</span><span class="named-content" title="named-content">School of Geographic and Environmental Sciences</span><span class="institution" title="institution">Tianjin Normal University</span><div class="unknown" title="city" tag="city">
Tianjin</div><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="mbo3983-aff-0003">[3], <span class="institution" title="institution">Agro‐Environmental Protection Institute</span><span class="institution" title="institution">Ministry of Agriculture</span><div class="unknown" title="city" tag="city">
Tianjin</div><span class="country" tagx="country" title="country">China</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b><br /><br /> Dongmei Xue, Tianjin Key Laboratory of Water Resources and Environment, Tianjin Normal University, Tianjin 300387, China.<br /><br /> Email: <span class="email" tagx="email" title="email">xuedongmei@tjnu.edu.cn</span><br /><br /> Guilong Zhang, Agro‐Environmental Protection Institute, Ministry of Agriculture, Tianjin, 300191, China.<br /><br /> Email: <span class="email" tagx="email" title="email">zhangguilong@caas.cn</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-3</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">3</span><span class="issue-id" tagx="issue-id" title="issue-id">10.1002/mbo3.v9.3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e983</span><span class="history" title="history"><span class="received" title="received">received: 2019-3-04</span><span class="rev-recd" title="rev-recd">rev-recd: 2019-12-04</span><span class="accepted" title="accepted">accepted: 2019-12-04</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</span></span></div><a class="self-uri" href="file:MBO3-9-e983.pdf" title="self-uri">file:MBO3-9-e983.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Soil microorganisms are considered to be important indicators of soil fertility and soil quality. Most previous studies have focused solely on surface soil, but there were numerous active cells in deeper soil layers. However, studies regarding microbial communities in deeper soil layers were not comprehensive and sufficient. In this study, phylogenetic molecular ecological networks (pMENs) based on the 16S rRNA Miseq sequencing technique were applied to study the response of soil microbial communities to depth gradients and the changes of key genera along 3 meter depth gradients (0–0.2 m, 0.2–0.4 m 0.4–0.6 m, 0.6–0.8 m, 0.8–1.0 m, 1.0–1.3 m, 1.3–1.6 m, 1.6–2.0 m, 2.0–2.5 m, and 2.5–3.0 m). The results showed that the modularity of microbial communities was consistently high in all soil layers and each layer was similar, which indicated that microbial communities were more resistant to depth changes. The pMENs further demonstrated that microbial community interactions were stable as the depth increased and they cooperated well to adapt to changes in different soil gradients. This was evidenced by similar positive links, average degree, and average clustering coefficient. In addition, key genera were obtained by analyzing module hubs in the pMENs. There may be at least one dominant genus in each layer that adapted to and resisted changes in the soil environment. It seems microbial communities demonstrate a stable and strong adaptability to depth gradients in farmland soils.</p> </div> <div class="abstract" title="abstract"> <p>In this study, phylogenetic molecular ecological networks (pMENs) based on pyrosequencing data of the 16S rRNA gene were applied to study the response of soil microbial communities to depth gradients and the changes of key genus in each depth layer. It seems that microbial communities demonstrate stable and strong adaptability to depth gradients. </p><div class="boxed-text" tagx="boxed-text" title="boxed-text"> <div class="graphic_UNKNOWN" position="anchor" id="nlm-graphic-1" orientation="portrait" /> </div> <p /> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
farmland soil</div> <div class="kwd" title="kwd">
microbial community</div> <div class="kwd" title="kwd">
MiSeq sequencing</div> <div class="kwd" title="kwd">
molecular ecological network</div> <div class="kwd" title="kwd">
soil depth</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Ministry of Science and Technology of the People’s Republic of China</div><span class="award-id" tagx="award-id" title="award-id">2018YFD0800400</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">The Natural Science Foundation of Tianjin </span><span class="institution-id" title="institution-id">10.13039/501100006606</span></div> </div><span class="award-id" tagx="award-id" title="award-id">19JCZDJC40700</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
The Innovation Team Training Plan of the Tianjin Education Committee</div><span class="award-id" tagx="award-id" title="award-id">TD12‐5037</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">The National Natural Science Foundation of China </span><span class="institution-id" title="institution-id">10.13039/501100001809</span></div> </div><span class="award-id" tagx="award-id" title="award-id">41973017</span></div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="cover-date" value="March 2020" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.7.7 mode:remove_FC converted:12.03.2020" /> </div> </div> <p content-type="self-citation"><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z‐L</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular ecological network analysis of the response of soil microbial communities to depth gradients in farmland soils</span>. <span class="source" tagx="source" title="source">MicrobiologyOpen</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">9</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e983</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/mbo3.983">10.1002/mbo3.983</a></span></span></p> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Most natural and artificial habitats of terrestrial ecosystems are covered by soils (Bai, Wang, Deng, and He (<a href="#mbo3983-bib-0002">2017</a>)). Since precipitation, illumination, and temperature within an area are similar, high spatial heterogeneity and physiochemical properties are responsible for the diverse range of microorganisms found in soil ecosystems (Schimel &amp;amp; Schaeffer, <a href="#mbo3983-bib-0041">2012</a>). Soil microorganisms are considered to be important indicators of soil fertility and quality (Paula, Rodrigues, Zhou, &amp;amp; Wu, <a href="#mbo3983-bib-0037">2014</a>). Recent studies based on culture‐independent techniques have shown that large‐scale soil microbial diversity and community composition were predominantly driven by soil pH (Griffiths, Thomson, James, &amp;amp; Bell, <a href="#mbo3983-bib-0021">2011</a>) and other soil properties, such as organic matter and salinity (Campbell &amp;amp; Kirchman, <a href="#mbo3983-bib-0005">2013</a>; Tripathi, Kim, Tateno, &amp;amp; Kim, <a href="#mbo3983-bib-0049">2015</a>; Zheng, Xue, Li, &amp;amp; Deng, <a href="#mbo3983-bib-0058">2017</a>). It was known that biogeochemical processes were mainly driven by microorganisms throughout the soil profile; however, previous studies on the structure and diversity of soil microbial communities had solely focused on the top 15 cm of the soil column (Eilers, Debenport, Anderson, &amp;amp; Fierer, <a href="#mbo3983-bib-0012">2012</a>). Numerous studies have suggested that microorganisms living in deeper soil layers were generally considered to be unimportant due to their low biomass density and low activity levels (Blume, Bischoff, Reichert, &amp;amp; Moorman, <a href="#mbo3983-bib-0003">2002</a>; Fierer, Schimel, &amp;amp; Holden, <a href="#mbo3983-bib-0016">2003</a>; Hartmann, Lee, Hallam, &amp;amp; Mohn, <a href="#mbo3983-bib-0023">2009</a>). However, Fierer et al. (Fierer, Schimel, et al., <a href="#mbo3983-bib-0016">2003</a>) noted that not only was a large number of microorganisms present in the underground soil but also that the potential activity of these underground microorganisms might be higher than that of surface microorganisms. Although microbial biomass generally exponentially decreased with depth (Blume et al., <a href="#mbo3983-bib-0003">2002</a>; Fierer, Schimel, et al., <a href="#mbo3983-bib-0016">2003</a>; Hartmann et al., <a href="#mbo3983-bib-0023">2009</a>), in deeper soil layers, there were numerous active cells (Buss, Bruns, Schultz, &amp;amp; Moore, <a href="#mbo3983-bib-0004">2005</a>; Richter &amp;amp; Markewitz, <a href="#mbo3983-bib-0040">1995</a>), which may have a greater impact on the soil formation process than their surface counterparts because of their proximity to parent material (Buss et al., <a href="#mbo3983-bib-0004">2005</a>). Fierer, Allen, Schimel, and Holden (<a href="#mbo3983-bib-0015">2003</a>) also demonstrated that deep microbial communities differed in composition from those in the surface layer. On a depth‐weighted basis, deep microbial communities have also been found to be diverse and abundant (Li, Yan, Tang, &amp;amp; Jia, <a href="#mbo3983-bib-0027">2014</a>; Will, Thürmer, Wollherr, &amp;amp; Nacke, <a href="#mbo3983-bib-0055">2010</a>). Previous studies have reported that in forest soil, soil depth affected the diversity and composition of the archaea community (Thoms, Gattinger, &amp;amp; Jacob, <a href="#mbo3983-bib-0047">2010</a>; Too, Keller, Sickel, &amp;amp; Lee, <a href="#mbo3983-bib-0048">2018</a>). Specific bacterial communities in deep wetland soils have been detected, and the interaction between these communities was more noticeable than that of the surface layer communities (Steinmuller, Dittmer, White, &amp;amp; Chambers, <a href="#mbo3983-bib-0045">2019</a>). However, how a microbial community adapts to soil depth gradients through community‐level adjustment of compositions and interactions has not yet been completely elucidated.</p> <p>Many network analysis approaches have been developed, such as the equation‐based network (Gardner, Di, Lorenz, &amp;amp; Collins, <a href="#mbo3983-bib-0018">2003</a>; Yeung, Tegner, &amp;amp; Collins, <a href="#mbo3983-bib-0056">2002</a>), the related/co‐expression network (Horvath, Zhang, Carlson, &amp;amp; Lu, <a href="#mbo3983-bib-0025">2006</a>; Oldham, Horvath, &amp;amp; Geschwind, <a href="#mbo3983-bib-0035">2006</a>), and Bayesian network (Gerstung, Baudis, Moch, &amp;amp; Beerenwinkel, <a href="#mbo3983-bib-0020">2009</a>). One frequently studied and widely used method is the phylogenetic molecular ecological networks (pMENs) proposed by Deng, Jiang, Yang, and He (<a href="#mbo3983-bib-0008">2012</a>), who were the first to apply metagenomic techniques (such as sequencing and microarrays) (Deng, Zhang, Qin, &amp;amp; Tu, <a href="#mbo3983-bib-0009">2016</a>; Faust &amp;amp; Raes, <a href="#mbo3983-bib-0014">2012</a>). In the entire network, various parameters, such as topology, node modules, topological roles, and network composition, better reflect the relationship between microbial communities and their associated niche functions (Hahn, Konwar, Louca, &amp;amp; Hanson, <a href="#mbo3983-bib-0022">2016</a>; Zhou, Deng, Luo, &amp;amp; He, <a href="#mbo3983-bib-0059">2010</a>, <a href="#mbo3983-bib-0060">2011</a>). Thus, in the present study, the pMENs were applied to investigate the responses of soil microbial communities to soil depth gradients and the underlying microbial interactions, including the following: (a) phylogenetic diversity and structure of microbial community shift; and (b) changes of key genera in microbial network interactions along soil depth gradients.</p> </div> <div class="materials-and-methods" title="materials-and-methods"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
MATERIAL AND METHODS</div> <div class="studysitesandsoilsamplecollection" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Study sites and soil sample collection</div> <p>The study sites were located in Tianjin, a municipality close to Bohai Bay, which is influenced by a continental monsoon climate. Both hydrology and climate played the significant roles on the physical and chemical properties of soil. In November 2013, soil samples were collected from eight farmland sites in five regions of Tianjin, China (S1: 39°36′54.49″N, 116°58′04.20″E; S2: 39°32′09.31″N, 116°59′32.31″E; S3: 39°40′37.50″N, 117°21′03.07″E; S4: 39°40′32.66″N, 117°20′39.53″E; S5: 40°04′14.87″N, 117°20′01.90″E; S6: 40°05′25.91″N, 117°38′10.21″E; S7: 38°43′09.78″N, 117°26′44.00″E; S8:38°43′06.33″N, 117°26′20.41″E; Figure <a href="#mbo3983-fig-0001">1</a>). Soil samples were continuously collected using a 5.5‐cm‐diameter hollow‐stem hand auger in a vertical profile of 10 layers (from the ground surface to a depth of 3 m), including layer A: 0–0.2 m, layer B: 0.2–0.4 m, layer C: 0.4–0.6 m, layer D: 0.6–0.8 m, layer E: 0.8–1.0 m, layer F: 1.0–1.3 m, layer G: 1.3–1.6 m, layer H: 1.6–2.0 m, layer I: 2.0–2.5 m, and layer J: 2.5–3.0 m (the soil profile was over 3 m, as the shallow groundwater table is close to this depth). The 80 collected samples were stored in polyethylene bags. The physicochemical parameters of soil water extracts were determined by adding 200 ml Milli‐Q water to 100 g soil sample, followed by shaking (45 min at room temperature), centrifuging (2,057 <i>g</i>), and filtering. The pH and salinity were determined by a portable analyzer (Orion Star A329, Thermo, USA). The cation concentrations of K<sup>+</sup>, Na<sup>+</sup>, Ca<sup>2+</sup>, and Mg<sup>2+</sup> were determined by an inductively coupled plasma atomic emission spectrometer (Optima 8300, PE, USA), and anion concentrations of <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
SO</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div> </div></div> and Cl<sup>−</sup> were determined by ion chromatography (ICS‐2100, Dionex, USA). <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
-</div> </div></div>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div></div>, and <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NH</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
+</div> </div></div> were determined by a continuous flow analyzer (Auto Analyzer 3, Seal, Germany). Part of the soil samples were freeze‐dried and stored at −20°C for genomic DNA extraction.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Study area sampling point diagram</p>   </div> </div> <div class="dnaextraction,illuminamiseqsequencingandanalysis" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
DNA extraction, Illumina MiSeq sequencing and analysis</div> <p>DNA of the microorganisms in 80 soil samples was extracted by biotechnological methods using the Ezup Genomic DNA Extraction Kit (Sangon Biotech, China, Cat# SK8264). After extraction, a NanoDrop Spectrophotometer was used to detect the concentration and mass of DNA. The DNA extraction was diluted to 10 ng/μl.</p> <p>The operational taxonomic unit (OTU) data used to construct the pMENs were generated by the 16S rRNA gene. Miseq sequencing of the V4 hypervariable region of the 16S rRNA gene was performed using the universal primers 515F (5'‐GTGCCAGCMGCCGCGGTAA‐3') and 806R (5'‐GGACTACHVGGGTWTCTAAT‐3'): Amplification was then measured using a MiSeq sequencer (Caporaso, Lauber, &amp;amp; Walters, <a href="#mbo3983-bib-0006">2011</a>; Lin, De, Li, &amp;amp; Li, <a href="#mbo3983-bib-0029">2016</a>; Zheng et al., <a href="#mbo3983-bib-0058">2017</a>). Ten nanograms of soil genomic DNA, 0.4 mM of each deoxynucleotide triphosphate, 1.0 μM of each primer, 0.5 U of Ex Taq (TaKaRa, Dalian), 1× PCR buffer, and 1.5 mM MgCl<sub>2</sub> were completely mixed in a 25 μl mixture for performing PCR (Zheng et al., <a href="#mbo3983-bib-0058">2017</a>). The PCR amplification was conducted as described by Li, Rui, Mao, and Yannarel (<a href="#mbo3983-bib-0028">2014</a>). Under the reading conditions with the unique identification bar code of the sample, sequencing was performed on the Illumina Miseq platform of the Environmental Genome Platform of the Chengdu Institute of Biology. Eighty samples were sequenced using the Reagent Kit v2 2 × 250 bp. The results were processed by QIIME pipeline version 1.7.0 (<a href="http://qiime.org/">http://qiime.org/</a>). The original data were first screened, and an average base quality score of 20 was the lowest; when the low‐quality scoring sequence was removed, the sequence length was at least 300 bp and without an ambiguous base "N." These sequences were used for downstream analysis (Zheng et al., <a href="#mbo3983-bib-0058">2017</a>). Next, the chimeric sequence was removed using the UCHIME algorithm (Edgar, Haas, Clemente, &amp;amp; Quince, <a href="#mbo3983-bib-0011">2011</a>). The processed sequence was clustered by the complete‐linkage clustering method of the Ribosomal Database project pyro pipeline. A 97% nucleotide sequence similarity cutoff was used to classify OTUs. The RDP classifier was used to assign taxonomy (Wang, Garrity, Tiedje, &amp;amp; Cole, <a href="#mbo3983-bib-0052">2007</a>). Re‐sampling was performed to a total of 2,590 reads of the same sequence depth. Chao1 estimator of richness and Shannon index were calculated using the RDP classifier. The original sequence data were deposited at the European Nucleotide Archive by accession PRJEB21751 (<a href="http://www.ebi.ac.uk/ena/data/view/PRJEB21751">http://www.ebi.ac.uk/ena/data/view/PRJEB21751</a>) (Zheng et al., <a href="#mbo3983-bib-0058">2017</a>).</p> </div> <div class="statisticalanalysis" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Statistical analysis</div> <p>Microbial community diversity was compared using IBM SPSS 20 with a univariate ANOVA based on Chao1 richness and Shannon diversity indices. Correlation analysis between microbial communities and environmental variables was performed by the Mantel test, conducted by PCORD 5.0. In addition, based on the study of Fukuyama (Fukuyama, Mcmurdie, Dethlefsen, Relman, &amp;amp; Holmes, <a href="#mbo3983-bib-0017">2012</a>), double principal coordinate analysis (DPCoA) was performed on the relative abundance data of the OTUs.</p> </div> <div class="establishmentandanalysisofpmens" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
Establishment and analysis of pMENs</div> <p>Construction of pMENs is based on the random matrix theory (RMT). The network was built using the online pipeline provided by the Institute of Environmental Genomics at the University of Oklahoma (<a href="http://ieg2.ou.edu/MENA">http://ieg2.ou.edu/MENA</a>) (Deng et al., <a href="#mbo3983-bib-0008">2012</a>; Zhou, Deng, Luo, &amp;amp; He, <a href="#mbo3983-bib-0059">2010</a>, <a href="#mbo3983-bib-0060">2011</a>). The establishment of pMENs was divided into nine key steps, namely collection of metagenomic sequences, data standardization, Pearson's correlation estimation, adjacency matrix determination by an RMT‐based approach, network characterization, module detection, eigengene network analysis, network comparison, and correlation based on the RMT method (Zhou, Deng, Luo, &amp;amp; He, <a href="#mbo3983-bib-0060">2011</a>). High‐throughput metagenomic data were collected and transformed to construct Pearson's correlation matrix (Zhou, Deng, Luo, &amp;amp; He, <a href="#mbo3983-bib-0059">2010</a>), which was further converted into a similarity matrix. The method of RMT is to automatically select the optimal threshold of the network according to the data to build an optimal network. The automatically determined threshold was used to derive the adjacency matrix from the similarity matrix. Using the adjacency matrix to encode the strength of the connection between each pair of nodes, module analysis and network characterization were performed to generate different network topology attributes. The procedure was detailed in a previous MENA study (Deng et al., <a href="#mbo3983-bib-0008">2012</a>). Different topological roles were described by the following two parameters: the within‐module connectivity (Zi) and among‐module connectivity (Pi). The Zi reflected the degree to which nodes were connected to other nodes within their own module, while the Pi reflected the degree to which nodes were connected to different modules. Since we were primarily interested in the impact of soil depth on network interactions, pMENs were constructed based on the relative abundance of samples obtained from each layer of the 8 sampling sites.</p> </div> </div> <div class="results" title="results"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
RESULTS</div> <p>A total of 834,485 high‐quality 16S rRNA gene sequences were obtained for 80 samples. They were resampled to 2,590 sequences per sample, which were clustered into 7,403 OTUs. The rarefaction curves showed that the number of samples was reasonable (Figure <a href="#mbo3983-fig-0005">A1</a>). Both Shannon and Chao1 indices showed no significant differences between the depth layers (Table <a href="#mbo3983-tbl-0005">A1</a>). To understand the effects of soil depth on the linkages between microorganisms, OTU data obtained from MiSeq sequencing was used to construct the microbial pMENs based on 10 layers of the 8 sampling sites (Table <a href="#mbo3983-tbl-0001">1</a>). One node represented an OTU, with each link representing the correlation between two connected nodes. The results presented in Table <a href="#mbo3983-tbl-0001">1</a> showed that microbial connectivity had similar thresholds (0.930–0.960). Modularity acted as an indicator of system resistance (Carpenter, Arrow, Barrett, &amp;amp; Biggs, <a href="#mbo3983-bib-0007">2012</a>). In our study, the modularity values were similar (0.771–0.810) and higher than 0.4, which indicated the modular structure of the network (Newman, <a href="#mbo3983-bib-0034">2006</a>) and the strong resistance to environmental changes (Bai et al., <a href="#mbo3983-bib-0002">2017</a>). The numbers of module hubs varied from 1 to 5, which indicated that the degree of soil generalization differed from that in the 10 layers, with smaller values representing lower degrees of generalization (Wang, Zhang, Zheng, &amp;amp; Deng, <a href="#mbo3983-bib-0053">2016</a>). The average clustering coefficient (avgCC) described the relationship between a particular node and its adjacent node (Shen, Huang, Zeng, &amp;amp; Yu, <a href="#mbo3983-bib-0042">2016</a>). The avgCC values varied over a similar range from 0.240 to 0.365 in the 10 layers, showing connections similar to that of the neighbors (Deng et al., <a href="#mbo3983-bib-0008">2012</a>). The avgKK values varied from 3.973 to 5.559, which indicated that each layer of the network had different degrees of complexity. The R‐squared values were all greater than 0.700, implying that the RMT‐based ecological network should be scale‐free (Zhou et al., <a href="#mbo3983-bib-0059">2010</a>). Scale‐free implied that most nodes in the network had limited connections with other nodes, while a few nodes had many connections (Sun, Wang, Lin, &amp;amp; Zhou, <a href="#mbo3983-bib-0046">2015</a>). As shown in Table <a href="#mbo3983-tbl-0001">1</a>, the percentages of positive links (70%–86%) were much higher than those of the negative links (14%–30%), which indicated that the relationship between microbial communities was based more on mutual cooperation than competition (Newman, <a href="#mbo3983-bib-0034">2006</a>). Alternatively, positive links have also been found to be related to mutualism among genes during coevolution processes (Faust &amp;amp; Raes, <a href="#mbo3983-bib-0014">2012</a>).</p> <div class="table-wrap_UNKNOWN" id="mbo3983-tbl-0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Topological properties of the phylogenetic molecular ecological networks of microbial communities in ten layers (A to J)</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Depth</th> <th align="left" valign="top" rowspan="1" colspan="1">No. of original OTUs</th> <th align="left" valign="top" rowspan="1" colspan="1">Similarity threshold</th> <th align="left" valign="top" rowspan="1" colspan="1">Modularity</th> <th align="left" valign="top" rowspan="1" colspan="1">Module</th> <th align="left" valign="top" rowspan="1" colspan="1">Numbers of module hubs</th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>Average clustering coefficient</p> <p>(avgCC)</p> </th> <th align="left" valign="top" rowspan="1" colspan="1">Average degree (avgKK)</th> <th align="left" valign="top" rowspan="1" colspan="1">Total nodes</th> <th align="left" valign="top" rowspan="1" colspan="1">Total links</th> <th align="left" valign="top" rowspan="1" colspan="1">R square of power‐low</th> <th align="left" valign="top" rowspan="1" colspan="1">Percentage of positive links</th> <th align="left" valign="top" rowspan="1" colspan="1">Percentage of negative links</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.93</td> <td align="left" rowspan="1" colspan="1">0.796</td> <td align="left" rowspan="1" colspan="1">39</td> <td align="left" rowspan="1" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">0.240</td> <td align="left" rowspan="1" colspan="1">3.973</td> <td align="left" rowspan="1" colspan="1">375</td> <td align="left" rowspan="1" colspan="1">745</td> <td align="left" rowspan="1" colspan="1">0.857</td> <td align="left" rowspan="1" colspan="1">70</td> <td align="left" rowspan="1" colspan="1">30</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">B</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.93</td> <td align="left" rowspan="1" colspan="1">0.771</td> <td align="left" rowspan="1" colspan="1">40</td> <td align="left" rowspan="1" colspan="1">4</td> <td align="left" rowspan="1" colspan="1">0.244</td> <td align="left" rowspan="1" colspan="1">4.105</td> <td align="left" rowspan="1" colspan="1">362</td> <td align="left" rowspan="1" colspan="1">743</td> <td align="left" rowspan="1" colspan="1">0.860</td> <td align="left" rowspan="1" colspan="1">75</td> <td align="left" rowspan="1" colspan="1">25</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">C</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.95</td> <td align="left" rowspan="1" colspan="1">0.781</td> <td align="left" rowspan="1" colspan="1">38</td> <td align="left" rowspan="1" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">0.279</td> <td align="left" rowspan="1" colspan="1">5.088</td> <td align="left" rowspan="1" colspan="1">351</td> <td align="left" rowspan="1" colspan="1">893</td> <td align="left" rowspan="1" colspan="1">0.791</td> <td align="left" rowspan="1" colspan="1">77</td> <td align="left" rowspan="1" colspan="1">23</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">D</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.93</td> <td align="left" rowspan="1" colspan="1">0.810</td> <td align="left" rowspan="1" colspan="1">33</td> <td align="left" rowspan="1" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">0.328</td> <td align="left" rowspan="1" colspan="1">5.559</td> <td align="left" rowspan="1" colspan="1">354</td> <td align="left" rowspan="1" colspan="1">984</td> <td align="left" rowspan="1" colspan="1">0.841</td> <td align="left" rowspan="1" colspan="1">86</td> <td align="left" rowspan="1" colspan="1">14</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">E</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.95</td> <td align="left" rowspan="1" colspan="1">0.790</td> <td align="left" rowspan="1" colspan="1">26</td> <td align="left" rowspan="1" colspan="1">4</td> <td align="left" rowspan="1" colspan="1">0.343</td> <td align="left" rowspan="1" colspan="1">5.190</td> <td align="left" rowspan="1" colspan="1">294</td> <td align="left" rowspan="1" colspan="1">763</td> <td align="left" rowspan="1" colspan="1">0.856</td> <td align="left" rowspan="1" colspan="1">79</td> <td align="left" rowspan="1" colspan="1">21</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">F</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.94</td> <td align="left" rowspan="1" colspan="1">0.782</td> <td align="left" rowspan="1" colspan="1">38</td> <td align="left" rowspan="1" colspan="1">4</td> <td align="left" rowspan="1" colspan="1">0.302</td> <td align="left" rowspan="1" colspan="1">4.361</td> <td align="left" rowspan="1" colspan="1">366</td> <td align="left" rowspan="1" colspan="1">798</td> <td align="left" rowspan="1" colspan="1">0.808</td> <td align="left" rowspan="1" colspan="1">76</td> <td align="left" rowspan="1" colspan="1">24</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">G</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.94</td> <td align="left" rowspan="1" colspan="1">0.795</td> <td align="left" rowspan="1" colspan="1">35</td> <td align="left" rowspan="1" colspan="1">5</td> <td align="left" rowspan="1" colspan="1">0.267</td> <td align="left" rowspan="1" colspan="1">4.133</td> <td align="left" rowspan="1" colspan="1">331</td> <td align="left" rowspan="1" colspan="1">684</td> <td align="left" rowspan="1" colspan="1">0.902</td> <td align="left" rowspan="1" colspan="1">79</td> <td align="left" rowspan="1" colspan="1">21</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">H</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.96</td> <td align="left" rowspan="1" colspan="1">0.796</td> <td align="left" rowspan="1" colspan="1">26</td> <td align="left" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1">0.347</td> <td align="left" rowspan="1" colspan="1">4.070</td> <td align="left" rowspan="1" colspan="1">257</td> <td align="left" rowspan="1" colspan="1">523</td> <td align="left" rowspan="1" colspan="1">0.822</td> <td align="left" rowspan="1" colspan="1">82</td> <td align="left" rowspan="1" colspan="1">18</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.94</td> <td align="left" rowspan="1" colspan="1">0.789</td> <td align="left" rowspan="1" colspan="1">29</td> <td align="left" rowspan="1" colspan="1">5</td> <td align="left" rowspan="1" colspan="1">0.365</td> <td align="left" rowspan="1" colspan="1">4.476</td> <td align="left" rowspan="1" colspan="1">319</td> <td align="left" rowspan="1" colspan="1">714</td> <td align="left" rowspan="1" colspan="1">0.856</td> <td align="left" rowspan="1" colspan="1">78</td> <td align="left" rowspan="1" colspan="1">22</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">J</td> <td align="left" rowspan="1" colspan="1">7,188</td> <td align="left" rowspan="1" colspan="1">0.93</td> <td align="left" rowspan="1" colspan="1">0.778</td> <td align="left" rowspan="1" colspan="1">31</td> <td align="left" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1">0.341</td> <td align="left" rowspan="1" colspan="1">5.288</td> <td align="left" rowspan="1" colspan="1">351</td> <td align="left" rowspan="1" colspan="1">928</td> <td align="left" rowspan="1" colspan="1">0.795</td> <td align="left" rowspan="1" colspan="1">82</td> <td align="left" rowspan="1" colspan="1">18</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="mbo3983-ntgp-0001"> <div class="fn-type-" title=""> <p>Abbreviation: OTU, operational taxonomic unit.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <p>Results of the Pi and Zi analysis, as depicted in the Z‐P diagram (Figure <a href="#mbo3983-fig-0002">2</a>), revealed that different nodes played different roles in the pMENs. From an ecological point of view, each module in the pMENs indicated one niche (Wang et al., <a href="#mbo3983-bib-0052">2007</a>). Peripherals (Zi &amp;lt; 2.5 and Pi &amp;lt; 0.625) referred to specialists, while module hubs (Zi &amp;gt; 2.5 and Pi &amp;lt; 0.625) and connectors (Zi &amp;lt; 2.5 and Pi &amp;gt; 0.625) showed more generalists, and network hubs (Zi &amp;gt; 2.5 and Pi &amp;gt; 0.625) embodied supergeneralists (Deng et al., <a href="#mbo3983-bib-0008">2012</a>; Lewinsohn, Prado, Jordano, &amp;amp; Bascompte, <a href="#mbo3983-bib-0026">2006</a>). No network hubs were found in the Z‐P diagram in all ten layers. The vast majority of OTUs in each layer occurred in peripherals, which represented most of their links inside their own modules (Deng et al., <a href="#mbo3983-bib-0008">2012</a>). The module hubs and connectors, which occurred in each layer of the community, as shown in Figure <a href="#mbo3983-fig-0002">2</a>, indicated that the microbial communities in all ten layers were stable. According to the module hubs in the Z‐P diagram, corresponding OTUs can be used to determine the key genus of each layer (see Table <a href="#mbo3983-tbl-0002">2</a>). It can be observed that the key genus in each layer was basically different and will be further discussed in the discussion section.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>The network of the pMENs of ten layers. The Z‐P plot showing the topological roles of each OTU based on Zi (within‐module connectivity) and Pi (among‐module connectivity) of microbial communities in ten soil layers. According to values of Zi (2.5) and Pi (0.625), the roles of nodes were classified into four categories</p>   </div> <div class="table-wrap_UNKNOWN" id="mbo3983-tbl-0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Operational taxonomic units (OTUs) and corresponding keystone genus in ten layers (A to J)</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Depth</th> <th align="left" valign="top" rowspan="1" colspan="1">Numbers of module hubs</th> <th align="left" valign="top" rowspan="1" colspan="1">OTU</th> <th align="left" valign="top" rowspan="1" colspan="1">Phylum</th> <th align="left" valign="top" rowspan="1" colspan="1">Class</th> <th align="left" valign="top" rowspan="1" colspan="1">Genera</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="3" colspan="1">A</td> <td align="left" rowspan="3" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">OTU832</td> <td align="left" rowspan="1" colspan="1"><i>Gemmatimonadetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Gemmatimonadetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Gemmatimonas</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU2510</td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria_Gp25</i></td> <td align="left" rowspan="1" colspan="1"><i>Gp25</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU4954</td> <td align="left" rowspan="1" colspan="1"><i>Bacteroidetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Bacteroidetes_incertae_sedis</i></td> <td align="left" rowspan="1" colspan="1"><i>Ohtaekwangia</i></td> </tr> <tr> <td align="left" rowspan="4" colspan="1">B</td> <td align="left" rowspan="4" colspan="1">4</td> <td align="left" rowspan="1" colspan="1">OTU71</td> <td align="left" rowspan="1" colspan="1"><i>Spirochetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Spirochaetia</i></td> <td align="left" rowspan="1" colspan="1"><i>Treponema</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU498</td> <td align="left" rowspan="1" colspan="1"><i>Thaumarchaeota</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> <td align="left" rowspan="1" colspan="1"><i>Nitrosopumilus</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU774</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Deltaproteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1532</td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="3" colspan="1">C</td> <td align="left" rowspan="3" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">OTU650</td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1466</td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Gp4</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1457</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Cystobacter</i></td> </tr> <tr> <td align="left" rowspan="2" colspan="1">D</td> <td align="left" rowspan="2" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">OTU723</td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU967</td> <td align="left" rowspan="1" colspan="1"><i>Chloroplast</i></td> <td align="left" rowspan="1" colspan="1"><i>Chloroplast</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="4" colspan="1">E</td> <td align="left" rowspan="4" colspan="1">4</td> <td align="left" rowspan="1" colspan="1">OTU631</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Gammaproteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1510</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Betaproteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1853</td> <td align="left" rowspan="1" colspan="1"><i>Verrucomicrobia</i></td> <td align="left" rowspan="1" colspan="1"><i>Spartobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Spartobacteria_genera_incertae_sedis</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU4767</td> <td align="left" rowspan="1" colspan="1"><i>Chloroflexi</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="4" colspan="1">F</td> <td align="left" rowspan="4" colspan="1">4</td> <td align="left" rowspan="1" colspan="1">OTU396</td> <td align="left" rowspan="1" colspan="1"><i>Spirochetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Spirochaetia</i></td> <td align="left" rowspan="1" colspan="1"><i>Treponema</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1362</td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria_Gp7</i></td> <td align="left" rowspan="1" colspan="1"><i>Gp7</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1366</td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Gaiella</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU2089</td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Gaiella</i></td> </tr> <tr> <td align="left" rowspan="5" colspan="1">G</td> <td align="left" rowspan="5" colspan="1">5</td> <td align="left" rowspan="1" colspan="1">OTU316</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Gammaproteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Steroidobacter</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1664</td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Actinobacteria</i></td> <td align="left" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU2980</td> <td align="left" rowspan="1" colspan="1"><i>Gemmatimonadetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Gemmatimonadetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Gemmatimonas</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU1853</td> <td align="left" rowspan="1" colspan="1"><i>Verrucomicrobia</i></td> <td align="left" rowspan="1" colspan="1"><i>Spartobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Spartobacteria_genera_incertae_sedis</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU2885</td> <td align="left" rowspan="1" colspan="1"><i>Firmicutes</i></td> <td align="left" rowspan="1" colspan="1"><i>Clostridia</i></td> <td align="left" rowspan="1" colspan="1"><i>Ruminococcus2</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">H</td> <td align="left" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1">OTU679</td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="5" colspan="1">I</td> <td align="left" rowspan="5" colspan="1">5</td> <td align="left" rowspan="1" colspan="1">OTU15</td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU156</td> <td align="left" rowspan="1" colspan="1"><i>Bacteroidetes</i></td> <td align="left" rowspan="1" colspan="1"><i>Flavobacteriia</i></td> <td align="left" rowspan="1" colspan="1"><i>Croceibacter</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU570</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Betaproteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU408</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Betaproteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>–</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">OTU2365</td> <td align="left" rowspan="1" colspan="1"><i>Proteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Alphaproteobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Sphingomonas</i></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">J</td> <td align="left" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1">OTU2930</td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria</i></td> <td align="left" rowspan="1" colspan="1"><i>Acidobacteria_Gp18</i></td> <td align="left" rowspan="1" colspan="1"><i>Gp18</i></td> </tr> </tbody> </table> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <p>The Mantel test was applied to determine the relationship between microbial communities and environmental factors (K<sup>+</sup>, Ca<sup>2+</sup>, Na<sup>+</sup>, Mg<sup>2+</sup>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NH</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
+</div> </div></div>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div></div>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
-</div> </div></div>, Cl<sup>−</sup>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
SO</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div> </div></div>, total organic carbon (TOC), total nitrogen (TN), pH and salinity). As shown in Table <a href="#mbo3983-tbl-0003">3</a>, most of the environmental factors did not have significant effects on microbial communities in each layer, excluding Mg<sup>2+</sup> in layer A, Ca<sup>2+</sup> in layer C, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div></div> in layer F, TOC and TN in layer B, and pH in layers E and I. It was obvious that the microbial communities in all ten layers were adapted to depth gradients and demonstrated stable resistance to environmental changes in oil profiles (Table <a href="#mbo3983-tbl-0003">3</a>). In addition, DPCoA was performed on the microbial communities in each layer (Figure <a href="#mbo3983-fig-0003">3</a>). The DPCoA results revealed an independent relationship among the ten layers of microbial communities.</p> <div class="table-wrap_UNKNOWN" id="mbo3983-tbl-0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3</span> <p>Correlations between environmental variables and the microbial community composition by the Mantel test</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Environmental variable</th> <th align="left" valign="top" rowspan="1" colspan="1">A</th> <th align="left" valign="top" rowspan="1" colspan="1">B</th> <th align="left" valign="top" rowspan="1" colspan="1">C</th> <th align="left" valign="top" rowspan="1" colspan="1">D</th> <th align="left" valign="top" rowspan="1" colspan="1">E</th> <th align="left" valign="top" rowspan="1" colspan="1">F</th> <th align="left" valign="top" rowspan="1" colspan="1">G</th> <th align="left" valign="top" rowspan="1" colspan="1">H</th> <th align="left" valign="top" rowspan="1" colspan="1">I</th> <th align="left" valign="top" rowspan="1" colspan="1">J</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">K<sup>+</sup></td> <td align="char" char="." rowspan="1" colspan="1">−0.2526</td> <td align="char" char="." rowspan="1" colspan="1">0.2784</td> <td align="char" char="." rowspan="1" colspan="1">0.0069</td> <td align="char" char="." rowspan="1" colspan="1">0.2629</td> <td align="char" char="." rowspan="1" colspan="1">0.0323</td> <td align="char" char="." rowspan="1" colspan="1">0.1102</td> <td align="char" char="." rowspan="1" colspan="1">0.2133</td> <td align="char" char="." rowspan="1" colspan="1">−0.1142</td> <td align="char" char="." rowspan="1" colspan="1">−0.2386</td> <td align="char" char="." rowspan="1" colspan="1">−0.1356</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Ca<sup>2+</sup></td> <td align="char" char="." rowspan="1" colspan="1">0.1232</td> <td align="char" char="." rowspan="1" colspan="1">0.1828</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.3250</b><a href="#mbo3983-note-0002">*</a></td> <td align="char" char="." rowspan="1" colspan="1">−0.0835</td> <td align="char" char="." rowspan="1" colspan="1">0.4148</td> <td align="char" char="." rowspan="1" colspan="1">0.3644</td> <td align="char" char="." rowspan="1" colspan="1">0.3378</td> <td align="char" char="." rowspan="1" colspan="1">0.0620</td> <td align="char" char="." rowspan="1" colspan="1">0.2396</td> <td align="char" char="." rowspan="1" colspan="1">0.2697</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Na<sup>+</sup></td> <td align="char" char="." rowspan="1" colspan="1">0.1031</td> <td align="char" char="." rowspan="1" colspan="1">0.0392</td> <td align="char" char="." rowspan="1" colspan="1">0.1531</td> <td align="char" char="." rowspan="1" colspan="1">−0.0123</td> <td align="char" char="." rowspan="1" colspan="1">0.1002</td> <td align="char" char="." rowspan="1" colspan="1">0.1490</td> <td align="char" char="." rowspan="1" colspan="1">0.2646</td> <td align="char" char="." rowspan="1" colspan="1">0.6197</td> <td align="char" char="." rowspan="1" colspan="1">0.0087</td> <td align="char" char="." rowspan="1" colspan="1">0.0022</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Mg<sup>2+</sup></td> <td align="char" char="." rowspan="1" colspan="1"><b>0.4541</b><a href="#mbo3983-note-0003">**</a></td> <td align="char" char="." rowspan="1" colspan="1">−0.0086</td> <td align="char" char="." rowspan="1" colspan="1">0.0099</td> <td align="char" char="." rowspan="1" colspan="1">−0.0211</td> <td align="char" char="." rowspan="1" colspan="1">0.0160</td> <td align="char" char="." rowspan="1" colspan="1">0.1873</td> <td align="char" char="." rowspan="1" colspan="1">0.0447</td> <td align="char" char="." rowspan="1" colspan="1">−0.0481</td> <td align="char" char="." rowspan="1" colspan="1">0.0403</td> <td align="char" char="." rowspan="1" colspan="1">0.0567</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NH</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
+</div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.1450</td> <td align="char" char="." rowspan="1" colspan="1">−0.0292</td> <td align="char" char="." rowspan="1" colspan="1">−0.1802</td> <td align="char" char="." rowspan="1" colspan="1">−0.2909</td> <td align="char" char="." rowspan="1" colspan="1">0.0031</td> <td align="char" char="." rowspan="1" colspan="1">−0.2291</td> <td align="char" char="." rowspan="1" colspan="1">−0.1221</td> <td align="char" char="." rowspan="1" colspan="1">–</td> <td align="char" char="." rowspan="1" colspan="1">−0.2189</td> <td align="char" char="." rowspan="1" colspan="1">0.0495</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.0023</td> <td align="char" char="." rowspan="1" colspan="1">−0.2641</td> <td align="char" char="." rowspan="1" colspan="1">−0.2089</td> <td align="char" char="." rowspan="1" colspan="1">−0.1498</td> <td align="char" char="." rowspan="1" colspan="1">−0.3799</td> <td align="char" char="." rowspan="1" colspan="1"><b>0.0259</b><a href="#mbo3983-note-0003">**</a></td> <td align="char" char="." rowspan="1" colspan="1">−0.0240</td> <td align="char" char="." rowspan="1" colspan="1">0.0773</td> <td align="char" char="." rowspan="1" colspan="1">0.3007</td> <td align="char" char="." rowspan="1" colspan="1">0.3038</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
-</div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.0503</td> <td align="char" char="." rowspan="1" colspan="1">0.0296</td> <td align="char" char="." rowspan="1" colspan="1">–</td> <td align="char" char="." rowspan="1" colspan="1">0.0295</td> <td align="char" char="." rowspan="1" colspan="1">−0.0436</td> <td align="char" char="." rowspan="1" colspan="1">0.0573</td> <td align="char" char="." rowspan="1" colspan="1">0.4301</td> <td align="char" char="." rowspan="1" colspan="1">0.0077</td> <td align="char" char="." rowspan="1" colspan="1">0.3232</td> <td align="char" char="." rowspan="1" colspan="1">0.1084</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Cl<sup>−</sup></td> <td align="char" char="." rowspan="1" colspan="1">−0.0296</td> <td align="char" char="." rowspan="1" colspan="1">−0.0215</td> <td align="char" char="." rowspan="1" colspan="1">0.0336</td> <td align="char" char="." rowspan="1" colspan="1">0.0990</td> <td align="char" char="." rowspan="1" colspan="1">0.1781</td> <td align="char" char="." rowspan="1" colspan="1">0.0962</td> <td align="char" char="." rowspan="1" colspan="1">0.2342</td> <td align="char" char="." rowspan="1" colspan="1">0.4263</td> <td align="char" char="." rowspan="1" colspan="1">−0.0937</td> <td align="char" char="." rowspan="1" colspan="1">0.1597</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
SO</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.2695</td> <td align="char" char="." rowspan="1" colspan="1">−0.1983</td> <td align="char" char="." rowspan="1" colspan="1">0.0338</td> <td align="char" char="." rowspan="1" colspan="1">0.0487</td> <td align="char" char="." rowspan="1" colspan="1">−0.1010</td> <td align="char" char="." rowspan="1" colspan="1">−0.0365</td> <td align="char" char="." rowspan="1" colspan="1">0.0196</td> <td align="char" char="." rowspan="1" colspan="1">0.0316</td> <td align="char" char="." rowspan="1" colspan="1">−0.0016</td> <td align="char" char="." rowspan="1" colspan="1">0.1273</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOC</td> <td align="char" char="." rowspan="1" colspan="1">0.2977</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.3063</b><a href="#mbo3983-note-0003">**</a></td> <td align="char" char="." rowspan="1" colspan="1">−0.1422</td> <td align="char" char="." rowspan="1" colspan="1">0.2033</td> <td align="char" char="." rowspan="1" colspan="1">−0.2898</td> <td align="char" char="." rowspan="1" colspan="1">−0.2273</td> <td align="char" char="." rowspan="1" colspan="1">−0.3077</td> <td align="char" char="." rowspan="1" colspan="1">−0.2347</td> <td align="char" char="." rowspan="1" colspan="1">−0.0410</td> <td align="char" char="." rowspan="1" colspan="1">0.2256</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TN</td> <td align="char" char="." rowspan="1" colspan="1">0.2379</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.3035</b><a href="#mbo3983-note-0002">*</a></td> <td align="char" char="." rowspan="1" colspan="1">0.0121</td> <td align="char" char="." rowspan="1" colspan="1">0.2963</td> <td align="char" char="." rowspan="1" colspan="1">−0.2296</td> <td align="char" char="." rowspan="1" colspan="1">−0.2503</td> <td align="char" char="." rowspan="1" colspan="1">–</td> <td align="char" char="." rowspan="1" colspan="1">0.0370</td> <td align="char" char="." rowspan="1" colspan="1">0.1700</td> <td align="char" char="." rowspan="1" colspan="1">0.0865</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">pH</td> <td align="char" char="." rowspan="1" colspan="1">0.2063</td> <td align="char" char="." rowspan="1" colspan="1">0.3800</td> <td align="char" char="." rowspan="1" colspan="1">−0.1289</td> <td align="char" char="." rowspan="1" colspan="1">−0.0567</td> <td align="char" char="." rowspan="1" colspan="1"><b>0.5459</b><a href="#mbo3983-note-0002">*</a></td> <td align="char" char="." rowspan="1" colspan="1">0.4617</td> <td align="char" char="." rowspan="1" colspan="1">0.0454</td> <td align="char" char="." rowspan="1" colspan="1">0.1930</td> <td align="char" char="." rowspan="1" colspan="1"><b>0.4508</b><a href="#mbo3983-note-0002">*</a></td> <td align="char" char="." rowspan="1" colspan="1">−0.0622</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">salinity</td> <td align="char" char="." rowspan="1" colspan="1">−0.3322</td> <td align="char" char="." rowspan="1" colspan="1">−0.0446</td> <td align="char" char="." rowspan="1" colspan="1">0.3782</td> <td align="char" char="." rowspan="1" colspan="1">0.1613</td> <td align="char" char="." rowspan="1" colspan="1">0.1321</td> <td align="char" char="." rowspan="1" colspan="1">0.1311</td> <td align="char" char="." rowspan="1" colspan="1">0.3876</td> <td align="char" char="." rowspan="1" colspan="1">0.1579</td> <td align="char" char="." rowspan="1" colspan="1">−0.0108</td> <td align="char" char="." rowspan="1" colspan="1">0.1887</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="mbo3983-ntgp-0002"> <div class="fn-type-" title=""> <p>Bold: <i>P</i>‐value of the correlation between physical and chemical factors and microorganisms in each layer.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p><i>p</i> &amp;lt; .05.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">**</span><p><i>p</i> &amp;lt; .01.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Double principle coordinate analysis of microbial communities based on the OTU relative abundances</p>   </div> </div> <div class="discussion" title="discussion"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
DISCUSSION</div> <p>According to the results of the pMEN analysis, the modularity values of each layer in our study were relatively high and close, reflecting habitat heterogeneity, different selection mechanisms, and phylogenetic clustering of closely related species (Lewinsohn et al., <a href="#mbo3983-bib-0026">2006</a>), which might lead to nonrandom interaction patterns and ecological network complexity (Olesen, Bascompte, Dupont, &amp;amp; Jordano, <a href="#mbo3983-bib-0036">2007</a>). This can be explained as follows: (1) modules with closely linked species may be the key units of coevolution, in which reciprocal selection leads to trait convergence of unrelated species (Olesen et al., <a href="#mbo3983-bib-0036">2007</a>); (2) species converge on a combination of related traits shaped by similar interaction patterns (Olesen et al., <a href="#mbo3983-bib-0036">2007</a>); and (3) such a process may result in an interactive heterogeneous network, with taxonomically or functionally related taxa packed into different modules (Deng et al., <a href="#mbo3983-bib-0008">2012</a>). In this study, modularity values were high, which indicated that the ecological network was complex, and the microbial community can fully adapt to the environment. Wang et al. (<a href="#mbo3983-bib-0052">2007</a>) suggested that the lack of module hubs would lead to fragmentation of the module, and essentially there was no cascading effect on other modules. Furthermore, the modular structure exists to dampen the rapid spread of disturbance in the community (Wang et al., <a href="#mbo3983-bib-0052">2007</a>). Module hubs were detected in each layer in our study, indicating cascade effects between modules and strong resistance of the microbial community to the environment. In addition, positive links can be explained by the mutualism among genes in coevolution (Zhang, Zhao, &amp;amp; Dai, <a href="#mbo3983-bib-0057">2014</a>), or to have similar niches (Zheng et al., <a href="#mbo3983-bib-0058">2017</a>) or cross‐feeding (Sun et al., <a href="#mbo3983-bib-0046">2015</a>), while negative relationships are due to competition (Faust &amp;amp; Raes, <a href="#mbo3983-bib-0014">2012</a>). Strong positive links were detected in all networks in this study, which implied that microbial communities were potentially more inclined to function together to adapt to depth‐changing environments. In contrast, some microbes that did not possess the ability to compete with other microbes were filtered out (Pointing, Chan, Lacap, &amp;amp; Lau, <a href="#mbo3983-bib-0038">2009</a>). Although modularity changes in our study were not as apparent as the changes reported in the study conducted by Bai et al. (<a href="#mbo3983-bib-0002">2017</a>) (Figure <a href="#mbo3983-fig-0004">4</a>), most molecular ecological network parameters varied over a relatively narrow range, which showed that the microbial communities in our study were more stable and adaptable to environmental changes. This was related to many reasons, such as microbial diversity, soil parent material, availability of resources along soil depth gradients, and environmental conditions, which required further research in the future.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>The phylogenetic molecular ecological network parameters change along the depth gradient. (a) Results of Bai's study (A: 0 – 0.05 m, B: 0.05 – 0.2 m, C: 0.2 – 0.4 m, D: 0.4 – 0.6 m), (b) Results of this study (A: 0 – 0.2 m, B: 0.2 – 0.4 m, C: 0.4 – 0.6 m, D: 0.6 – 0.8 m, E: 0.8 – 1.0 m, F: 1.0 – 1.3 m, G: 1.3 – 1.6 m, H: 1.6 – 2.0 m, I: 2.0 – 2.5 m, J: 2.5 – 3.0 m)</p>   </div> <p>Since module hubs represented key species in the network (Bai et al., <a href="#mbo3983-bib-0002">2017</a>), we obtained key genera that corresponded to key nodes at different soil layers (Table <a href="#mbo3983-tbl-0002">2</a>). It was evident that multiple module hubs occurred in most layers excluding those of D, H and J with only 1‐2 module hubs. A total of 16 module hubs were found from layers A to E, which was consistent with other studies that showed microbial communities were more active within one meter (Blume et al., <a href="#mbo3983-bib-0003">2002</a>; Eilers et al., <a href="#mbo3983-bib-0012">2012</a>; Fierer, Schimel, et al., <a href="#mbo3983-bib-0016">2003</a>; Hartmann et al., <a href="#mbo3983-bib-0023">2009</a>). However, a total number of 16 module hubs were obtained from layers F to J indicating potential activity of subsurface microorganisms. Approximately, 56% of the key genera belonged to the phyla <i>Acidobacteria, Actinobacteria and Proteobacteria. Acidobacteria</i> appeared in layers A, C, F, and G with the dominant species, being <i>Gp25</i>, <i>Gp4</i>, <i>Gp7</i>, and <i>Gp18</i>, respectively. These microbiota are particularly sensitive to pH variations (Pointing et al., <a href="#mbo3983-bib-0038">2009</a>). Although <i>Acidobacteria</i> was discovered from layers A to J, it was not the invariably key phylum in module hubs. The key genus in each layer was potentially related to microorganism diversity, their related network, and environmental conditions, resulting in different module hubs in each layer. The classes <i>Gammaproteobacteria, Deltaproteobacteria, Alphaproteobacteria and Betaproteobacteria</i> belong to the phylum <i>Proteobacteria</i>. <i>Gammaproteobacteria</i> was found in layers E and G and was the aerobic nitrogen‐fixing bacteria involved in soil nitrogen cycling (Tsoy, Ravcheev, &amp;amp; Cuklina, <a href="#mbo3983-bib-0050">2016</a>). <i>Deltaproteobacteria</i>, which was also present in layer B, was considered a major choline user. These results provided new insights into our understanding of the use of choline by soil microorganisms (Eleanor, Jason, &amp;amp; Helen, <a href="#mbo3983-bib-0013">2018</a>). Moreover, choline may be involved in the carbon mineralization process (Mussmann, Ishii, &amp;amp; Rabus, <a href="#mbo3983-bib-0032">2005</a>). <i>Betaproteobacteria,</i> which preferred relatively oxygen‐rich conditions (Lüdemann, Arth, &amp;amp; Liesack, <a href="#mbo3983-bib-0030">2000</a>), appeared in both layers E and I. Similarly, <i>Sphingomonas</i> was likely to occur under aerobic conditions (White, Sutton, &amp;amp; Ringelberg, <a href="#mbo3983-bib-0054">1996</a>), but was found in layer I and requires further study. The dominant genera found in layers B, C, D, F, and G were members of the phylum <i>Actinobacteria.</i> Most <i>Actinobacteria</i> appeared in the upper soil, which was related to aerobic degree (Eilers et al., <a href="#mbo3983-bib-0012">2012</a>), and its presence may accelerate the decay of animal and plant remains in soil (Stackebrandt, Rainey, &amp;amp; Ward‐Rainey, <a href="#mbo3983-bib-0044">1997</a>). <i>Firmicutes</i> and <i>Chloroflexi</i> appeared in the layers G and E, which can be explained by their adaptation to low‐nutrient environments (Holanda &amp;amp; Hedrich, <a href="#mbo3983-bib-0024">2015</a>). <i>Nitrosopumilus</i> was found in layer B and was involved in ammonia oxidation in the nitrogen cycle of the ecosystem (Nakagawa &amp;amp; Stahl, <a href="#mbo3983-bib-0033">2013</a>). The phylum <i>Verrucomicrobia</i> appeared in the layers E and G, which was consistent with previous studies reported that they were prone to occur in relatively anoxic environments (Eilers et al., <a href="#mbo3983-bib-0012">2012</a>). Therefore, there should be at least one dominant genus in each layer that must adapt to and resist changes in the soil environment. Factors such as carbon source, microbial decomposition, and species interactions also affect the distribution of microorganisms in the soil depth gradients as well (Marilley &amp;amp; Aragno, <a href="#mbo3983-bib-0031">1999</a>). Notably, module hubs were extremely high at 2 m soil depth. Only a few studies exist on deep microbial communities; thus, more attention should be given to the study of subsurface microbial communities in future.</p> <p>We performed the Mantel test on key genera obtained by module hubs and environmental factors (K<sup>+</sup>, Ca<sup>2+</sup>, Na<sup>+</sup>, Mg<sup>2+</sup>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NH</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
+</div> </div></div>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div></div>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
-</div> </div></div>, Cl<sup>−</sup>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
SO</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div> </div></div>, TOC, TN, pH, and salinity) along depth gradients (Table <a href="#mbo3983-tbl-0006">A2</a>). Most environmental factors did not have significant effects on key genera obtained by module hubs along depth gradients, excluding K<sup>+</sup> and Mg<sup>2+</sup> in layer A, K<sup>+</sup> in layer C, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
-</div> </div></div> in layers G and H, Na<sup>+</sup>, Cl<sup>−</sup>, <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
SO</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div> </div></div>, salinity in layer I, and K<sup>+</sup>, Na<sup>+</sup>, and <div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
SO</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div> </div></div> in layer J. It was evident that the key genera obtained by module hubs in all ten layers had no significant difference in pH, TOC, and TN. Soil layers dominant with <i>Acidobacteria</i> were mostly associated with potassium. Ding, Jiang, and Ma (<a href="#mbo3983-bib-0010">2016</a>) noted that inorganic fertilizers may lower the pH of soil and lead to its acidification. Members of the phylum <i>Proteobacteria</i> had a variety of metabolic types (Song, Liu, &amp;amp; Liang, <a href="#mbo3983-bib-0043">2016</a>). This may affect the sensitivity of <i>Proteobacteria</i> to environmental factors. Bacteria within the phylum <i>Bacteroidetes</i> can be distributed throughout the ecological niches (Garrity &amp;amp; Holt, <a href="#mbo3983-bib-0019">2001</a>), some of which were likely to occur in deep soil layers. During the formation and development of soil, each layer has its own physical and chemical properties due to the migration of energy and matter (Agnelli, Ascher, &amp;amp; Corti, <a href="#mbo3983-bib-0001">2004</a>). In general, depth‐related differences in the physicochemical and structural characteristics of soil profiles can include many microenvironmentally complex microbial populations that can evolve (Ranjard &amp;amp; Richaume, <a href="#mbo3983-bib-0039">2001</a>). In contrast, by aiming to determine the correlation between each physical and chemical factor index in each layer for whole microorganisms in our study, it can be concluded that the microorganisms were generally stable. Therefore, some mechanisms affected the adaptation of microbial communities to the environment. Our findings revealed that the adaptability of microbial communities to depth gradients may result in an approximate ten‐layer network analysis. Key genera play an important role in maintaining community stability. Notably, salinity has been shown to be a key driving factor for microbial communities in the same research region, as described by Zheng et al. (<a href="#mbo3983-bib-0058">2017</a>), but in our study, soil depth seems to also create important effects in the microbial communities.</p> </div> <div class="conclusions" title="conclusions"><span class="label" tagx="label" title="label">5</span><div class="title" tagx="title" title="title">
CONCLUSIONS</div> <p>In this study, based on the ecological network, we found that microbial communities along the depth gradient had a strong overall resistance. The change in the environmental conditions with soil depth represents an ecological filter; however, the microbial communities fully adapted to the depth gradient, which indicated that there are certain mechanisms that affect the adaptation of microbial communities to the environment.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
CONFLICT OF INTEREST</div> <p>None declared.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
AUTHOR CONTRIBUTIONS</div> <p>Hang Yu: Conceptualization; Formal analysis; Writing‐original draft; Writing‐review &amp;amp; editing. Dongmei Xue: Conceptualization; Funding acquisition; Writing‐review &amp;amp; editing. Yidong Wang: Funding acquisition; Writing‐review &amp;amp; editing. Wei Zheng: Conceptualization. Guilong Zhang: Funding acquisition. Zhongliang Wang: Funding acquisition. </p> </div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
ETHICS STATEMENT</div> <p>None required.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGMENTS</div> <p>We gratefully acknowledge Linzhen Guo and Mengfan Yang for soil sampling and Key Laboratory of Environmental and Applied Microbiology, Chinese Academy of Sciences for 16S rRNA Miseq sequencing analysis. This work was financially supported by Ministry of Science and Technology of the People's Republic of China (2018YFD0800400), the National Natural Science Foundation of China (41973017), the Natural Science Foundation of Tianjin (19JCZDJC40700), and the Innovation Team Training Plan of the Tianjin Education Committee (TD12‐5037).</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
DATA AVAILABILITY STATEMENT</div> <p>The datasets used and/or analyzed during the current study are available from the corresponding author upon reasonable request. The original sequence data are deposited at the European Nucleotide Archive: <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB21751">https://www.ebi.ac.uk/ena/data/view/PRJEB21751</a>.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="mbo3983-bib-0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Agnelli</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ascher</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Corti</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Distribution of microbial communities in a forest soil profile investigated by microbial biomass, soil respiration and DGGE of total and extracellular DNA</span>. <span class="source" tagx="source" title="source">Soil Biology and Biochemistry</span>, <span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">859</span>–<span class="lpage" tagx="lpage" title="lpage">868</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2004.02.004">10.1016/j.soilbio.2004.02.004</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bai</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial community and functional structure significantly varied among distinct types of paddy soils but responded differently along gradients of soil depth layers</span>. <span class="source" tagx="source" title="source">Frontiers in Microbiology</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">945</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2017.00945">10.3389/fmicb.2017.00945</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28611747">28611747</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Blume</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bischoff</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Reichert</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moorman</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Surface and subsurface microbial biomass, community structure and metabolic activity as a function of soil depth and season</span>. <span class="source" tagx="source" title="source">Applied Soil Ecology</span>, <span class="volume" tagx="volume" title="volume">592</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0929-1393(02)00025-2">10.1016/S0929-1393(02)00025-2</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Buss</span>, <span class="given-names" tagx="given-names" title="given-names">H. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bruns</span>, <span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schultz</span>, <span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moore</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">The coupling of biological iron cycling and mineral weathering during saprolite formation, Luquillo Mountains, Puerto Rico</span>. <span class="source" tagx="source" title="source">Geobiology</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">247</span>–<span class="lpage" tagx="lpage" title="lpage">260</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1472-4669.2006.00058.x">10.1111/j.1472-4669.2006.00058.x</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Campbell</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kirchman</span>, <span class="given-names" tagx="given-names" title="given-names">D. L.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Bacterial diversity, community structure and potential growth rates along an estuarine salinity gradient</span>. <span class="source" tagx="source" title="source">The ISME Journal</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">210</span>–<span class="lpage" tagx="lpage" title="lpage">220</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ismej.2012.93">10.1038/ismej.2012.93</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22895159">22895159</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Caporaso</span>, <span class="given-names" tagx="given-names" title="given-names">J. G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lauber</span>, <span class="given-names" tagx="given-names" title="given-names">C. L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Walters</span>, <span class="given-names" tagx="given-names" title="given-names">W. A.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">108</span>, <span class="fpage" tagx="fpage" title="fpage">4516</span>–<span class="lpage" tagx="lpage" title="lpage">4522</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1000080107">10.1073/pnas.1000080107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20534432">20534432</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Carpenter</span>, <span class="given-names" tagx="given-names" title="given-names">S. R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arrow</span>, <span class="given-names" tagx="given-names" title="given-names">K. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Barrett</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Biggs</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">General resilience to cope with extreme events</span>. <span class="source" tagx="source" title="source">Sustainability</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">3248</span>–<span class="lpage" tagx="lpage" title="lpage">3259</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/su4123248">10.3390/su4123248</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jiang</span>, <span class="given-names" tagx="given-names" title="given-names">Y. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yang</span>, <span class="given-names" tagx="given-names" title="given-names">Y. F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span>, <span class="given-names" tagx="given-names" title="given-names">Z. L.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular ecological network analyses</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">113</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-13-113">10.1186/1471-2105-13-113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22646978">22646978</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Qin</span>, <span class="given-names" tagx="given-names" title="given-names">Y. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tu</span>, <span class="given-names" tagx="given-names" title="given-names">Q. C.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Network succession reveals the importance of competition in response to emulsified vegetable oil amendment for uranium bioremediation</span>. <span class="source" tagx="source" title="source">Environmental Microbiology</span>, <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">205</span>–<span class="lpage" tagx="lpage" title="lpage">218</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/1462-2920.12981">10.1111/1462-2920.12981</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26177312">26177312</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ding</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jiang</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ma</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Effect of 35 years inorganic fertilizer and manure amendment on structure of bacterial and archaeal communities in black soil of northeast China</span>. <span class="source" tagx="source" title="source">Applied Soil Ecology</span>, <span class="volume" tagx="volume" title="volume">105</span>, <span class="fpage" tagx="fpage" title="fpage">187</span>–<span class="lpage" tagx="lpage" title="lpage">195</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apsoil.2016.04.010">10.1016/j.apsoil.2016.04.010</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Edgar</span>, <span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Haas</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clemente</span>, <span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quince</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">UCHIME improves sensitivity and speed of chimera detection</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">2194</span>–<span class="lpage" tagx="lpage" title="lpage">2200</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr381">10.1093/bioinformatics/btr381</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674">21700674</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Eilers</span>, <span class="given-names" tagx="given-names" title="given-names">K. G.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Debenport</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anderson</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fierer</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Digging deeper to find unique microbial communities: The strong effect of depth on the structure of bacterial and archaeal communities in soil</span>. <span class="source" tagx="source" title="source">Soil Biology &amp;amp; Biochemistry</span>, <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">58</span>–<span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2012.03.011">10.1016/j.soilbio.2012.03.011</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Eleanor</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jason</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Helen</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Deltaproteobacteria (Pelobacter) and Methanococcoides are responsible for choline‐dependent methanogenesis in a coastal saltmarsh sediment</span>. <span class="source" tagx="source" title="source">ISME Journal</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">277</span>–<span class="lpage" tagx="lpage" title="lpage">289</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30206424">30206424</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Faust</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Raes</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial interactions: From networks to models</span>. <span class="source" tagx="source" title="source">Nature Reviews Microbiology</span>, <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">538</span>–<span class="lpage" tagx="lpage" title="lpage">550</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro2832">10.1038/nrmicro2832</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22796884">22796884</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fierer</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Allen</span>, <span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schimel</span>, <span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holden</span>, <span class="given-names" tagx="given-names" title="given-names">P. A.</span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Controls on microbial CO<sub>2</sub> production: A comparison of surface and subsurface soil horizons</span>. <span class="source" tagx="source" title="source">Global Change Biology</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">1322</span>–<span class="lpage" tagx="lpage" title="lpage">1332</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2486.2003.00663.x">10.1046/j.1365-2486.2003.00663.x</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fierer</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schimel</span>, <span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holden</span>, <span class="given-names" tagx="given-names" title="given-names">P. A.</span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Variations in microbial community composition through two soil depth profiles</span>. <span class="source" tagx="source" title="source">Soil Biology &amp;amp; Biochemistry</span>, <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">167</span>–<span class="lpage" tagx="lpage" title="lpage">176</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0038-0717(02)00251-1">10.1016/S0038-0717(02)00251-1</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fukuyama</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mcmurdie</span>, <span class="given-names" tagx="given-names" title="given-names">P. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dethlefsen</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Relman</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holmes</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Comparisons of distance methods for combining covariates and abundances in microbiome studies</span>. <span class="source" tagx="source" title="source">Biocomputing</span>, <span class="fpage" tagx="fpage" title="fpage">213</span>–<span class="lpage" tagx="lpage" title="lpage">224</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174277">22174277</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gardner</span>, <span class="given-names" tagx="given-names" title="given-names">T. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Di</span>, <span class="given-names" tagx="given-names" title="given-names">B. D.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lorenz</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Collins</span>, <span class="given-names" tagx="given-names" title="given-names">J. J.</span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Inferring genetic networks and identifying compound mode of action via expression profiling</span>. <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">301</span>, <span class="fpage" tagx="fpage" title="fpage">102</span>–<span class="lpage" tagx="lpage" title="lpage">105</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1081900">10.1126/science.1081900</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12843395">12843395</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Garrity</span>, <span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holt</span>, <span class="given-names" tagx="given-names" title="given-names">J. G.</span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">The road map to the manual</span> In <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garrity</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (Ed.), <span class="source" tagx="source" title="source">Bergey's manual of systematic bacteriology</span> (Vol. <span class="volume" tagx="volume" title="volume">1</span>, <span class="edition" tagx="edition" title="edition">2nd ed</span>, pp. <span class="fpage" tagx="fpage" title="fpage">119</span>–<span class="lpage" tagx="lpage" title="lpage">166</span>). <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer‐Verlag</span>.</span></li> <li tag="ref"><a name="mbo3983-bib-0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gerstung</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baudis</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moch</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Beerenwinkel</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Quantifying cancer progression with conjunctive Bayesian networks</span>. <span class="source" tagx="source" title="source">Bioinform</span>, <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">2809</span>–<span class="lpage" tagx="lpage" title="lpage">2815</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp505">10.1093/bioinformatics/btp505</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Griffiths</span>, <span class="given-names" tagx="given-names" title="given-names">R. I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thomson</span>, <span class="given-names" tagx="given-names" title="given-names">B. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">James</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bell</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">The bacterial biogeography of British soils</span>. <span class="source" tagx="source" title="source">Environmental Microbiology</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">1642</span>–<span class="lpage" tagx="lpage" title="lpage">1654</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-2920.2011.02480.x">10.1111/j.1462-2920.2011.02480.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21507180">21507180</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hahn</span>, <span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Konwar</span>, <span class="given-names" tagx="given-names" title="given-names">K. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Louca</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hanson</span>, <span class="given-names" tagx="given-names" title="given-names">N. W.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The information science of microbial ecology</span>. <span class="source" tagx="source" title="source">Current Opinion in Microbiology</span>, <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">209</span>–<span class="lpage" tagx="lpage" title="lpage">216</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mib.2016.04.014">10.1016/j.mib.2016.04.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27183115">27183115</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hartmann</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lee</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hallam</span>, <span class="given-names" tagx="given-names" title="given-names">S. J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mohn</span>, <span class="given-names" tagx="given-names" title="given-names">W. W.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Bacterial, archaeal and eukaryal community structures throughout soil horizons of harvested and naturally disturbed forest stands</span>. <span class="source" tagx="source" title="source">Environmental Microbiology</span>, <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">3045</span>–<span class="lpage" tagx="lpage" title="lpage">3062</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-2920.2009.02008.x">10.1111/j.1462-2920.2009.02008.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19659501">19659501</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holanda</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hedrich</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Falagan C. characteristics of <i>Acidibacillus</i> Spp.: A novel genus of acidophilic iron‐oxidising firmicutes</span>. <span class="source" tagx="source" title="source">Adva Matr Res</span>, <span class="volume" tagx="volume" title="volume">1130</span>, <span class="fpage" tagx="fpage" title="fpage">36</span>–<span class="lpage" tagx="lpage" title="lpage">39</span>.</span></li> <li tag="ref"><a name="mbo3983-bib-0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Horvath</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Carlson</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lu</span>, <span class="given-names" tagx="given-names" title="given-names">K. V.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of oncogenic signaling networks in glioblastoma identifies ASPM as a molecular target</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">103</span>, <span class="fpage" tagx="fpage" title="fpage">17402</span>–<span class="lpage" tagx="lpage" title="lpage">17407</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0608396103">10.1073/pnas.0608396103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17090670">17090670</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lewinsohn</span>, <span class="given-names" tagx="given-names" title="given-names">T. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Prado</span>, <span class="given-names" tagx="given-names" title="given-names">P. I.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jordano</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bascompte</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Structure in plant‐animal interaction assemblages</span>. <span class="source" tagx="source" title="source">Oikos</span>, <span class="volume" tagx="volume" title="volume">113</span>, <span class="fpage" tagx="fpage" title="fpage">174</span>–<span class="lpage" tagx="lpage" title="lpage">184</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.0030-1299.2006.14583.x">10.1111/j.0030-1299.2006.14583.x</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">C. H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yan</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tang</span>, <span class="given-names" tagx="given-names" title="given-names">L. S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jia</span>, <span class="given-names" tagx="given-names" title="given-names">Z. J.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Change in deep soil microbial communities due to long‐term fertilization</span>. <span class="source" tagx="source" title="source">Soil Biology &amp;amp; Biochemistry</span>, <span class="volume" tagx="volume" title="volume">75</span>, <span class="fpage" tagx="fpage" title="fpage">264</span>–<span class="lpage" tagx="lpage" title="lpage">272</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2014.04.023">10.1016/j.soilbio.2014.04.023</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rui</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mao</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yannarell</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Dynamics of the bacterial community structure in the rhizosphere of a maize cultivar</span>. <span class="source" tagx="source" title="source">Soil Biology and Biochemistry</span>, <span class="volume" tagx="volume" title="volume">68</span>, <span class="fpage" tagx="fpage" title="fpage">392</span>–<span class="lpage" tagx="lpage" title="lpage">401</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2013.10.017">10.1016/j.soilbio.2013.10.017</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lin</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De</span>, <span class="given-names" tagx="given-names" title="given-names">V. J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Temperature affects microbial abundance, activity and interactions in anaerobic digestion</span>. <span class="source" tagx="source" title="source">Bioresource Technology</span>, <span class="volume" tagx="volume" title="volume">209</span>, <span class="fpage" tagx="fpage" title="fpage">228</span>–<span class="lpage" tagx="lpage" title="lpage">236</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2016.02.132">10.1016/j.biortech.2016.02.132</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26970926">26970926</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lüdemann</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Arth</span>, <span class="given-names" tagx="given-names" title="given-names">I.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liesack</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Spatial changes in the bacterial community structure along a vertical oxygen gradient in flooded paddy soil cores</span>. <span class="source" tagx="source" title="source">Applied and Environment Microbiology</span>, <span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">754</span>–<span class="lpage" tagx="lpage" title="lpage">762</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.66.2.754-762.2000">10.1128/AEM.66.2.754-762.2000</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Marilley</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Aragno</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic diversity of bacterial communities differing in degree of proximity of <i>Lolium perenne</i> and <i>Trifolium repens</i> roots</span>. <span class="source" tagx="source" title="source">Applied Soil Ecology</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">127</span>–<span class="lpage" tagx="lpage" title="lpage">136</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0929-1393(99)00028-1">10.1016/S0929-1393(99)00028-1</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mussmann</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ishii</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rabus</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Diversity and vertical distribution of cultured and uncultured Deltaproteobacteria in an intertidal mud flat of the Wadden Sea</span>. <span class="source" tagx="source" title="source">Environmental Microbiology</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">405</span>–<span class="lpage" tagx="lpage" title="lpage">418</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-2920.2005.00708.x">10.1111/j.1462-2920.2005.00708.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15683401">15683401</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nakagawa</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stahl</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Transcriptional response of the Archaeal ammonia oxidizer <i>Nitrosopumilus maritimus</i> to low and environmentally relevant ammonia concentrations</span>. <span class="source" tagx="source" title="source">Applied and Environment Microbiology</span>, <span class="volume" tagx="volume" title="volume">79</span>, <span class="fpage" tagx="fpage" title="fpage">6911</span>–<span class="lpage" tagx="lpage" title="lpage">6916</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.02028-13">10.1128/AEM.02028-13</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Newman</span>, <span class="given-names" tagx="given-names" title="given-names">M. E. J.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Modularity and community structure in networks</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">103</span>, <span class="fpage" tagx="fpage" title="fpage">8577</span>–<span class="lpage" tagx="lpage" title="lpage">8582</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0601602103">10.1073/pnas.0601602103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16723398">16723398</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Oldham</span>, <span class="given-names" tagx="given-names" title="given-names">M. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Horvath</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Geschwind</span>, <span class="given-names" tagx="given-names" title="given-names">D. H.</span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Conservation and evolution of gene coexpression networks in human and chimpanzee brains</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">103</span>, <span class="fpage" tagx="fpage" title="fpage">17973</span>–<span class="lpage" tagx="lpage" title="lpage">17978</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0605938103">10.1073/pnas.0605938103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17101986">17101986</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Olesen</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bascompte</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dupont</span>, <span class="given-names" tagx="given-names" title="given-names">Y. L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jordano</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">The modularity of pollination networks</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">104</span>, <span class="fpage" tagx="fpage" title="fpage">19891</span>–<span class="lpage" tagx="lpage" title="lpage">19896</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0706375104">10.1073/pnas.0706375104</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18056808">18056808</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paula</span>, <span class="given-names" tagx="given-names" title="given-names">F. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rodrigues</span>, <span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">J. Z.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wu</span>, <span class="given-names" tagx="given-names" title="given-names">L. Y.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Land use change alters functional gene diversity, composition and abundance in Amazon forest soil microbial communities</span>. <span class="source" tagx="source" title="source">Molecular Ecology</span>, <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">2988</span>–<span class="lpage" tagx="lpage" title="lpage">2999</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.12786">10.1111/mec.12786</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24806276">24806276</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pointing</span>, <span class="given-names" tagx="given-names" title="given-names">S. B.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chan</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lacap</span>, <span class="given-names" tagx="given-names" title="given-names">D. C.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lau</span>, <span class="given-names" tagx="given-names" title="given-names">M. C. Y.</span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Highly specialized microbial diversity in hyper‐arid polar desert</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">106</span>, <span class="fpage" tagx="fpage" title="fpage">19964</span>–<span class="lpage" tagx="lpage" title="lpage">19969</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0908274106">10.1073/pnas.0908274106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19850879">19850879</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ranjard</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Richaume</span>, <span class="given-names" tagx="given-names" title="given-names">A. S.</span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Quantitative and qualitative microscale distribution of bacteria in soil</span>. <span class="source" tagx="source" title="source">Research in Microbiology</span>, <span class="volume" tagx="volume" title="volume">152</span>, <span class="fpage" tagx="fpage" title="fpage">707</span>–<span class="lpage" tagx="lpage" title="lpage">716</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0923-2508(01)01251-7">10.1016/S0923-2508(01)01251-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11686384">11686384</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Richter</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Markewitz</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">How deep is soil?</span><span class="source" tagx="source" title="source">Biosci</span>, <span class="volume" tagx="volume" title="volume">45</span>, <span class="fpage" tagx="fpage" title="fpage">600</span>–<span class="lpage" tagx="lpage" title="lpage">609</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2307/1312764">10.2307/1312764</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schimel</span>, <span class="given-names" tagx="given-names" title="given-names">J. P.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schaeffer</span>, <span class="given-names" tagx="given-names" title="given-names">S. M.</span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial control over carbon cycling in soil</span>. <span class="source" tagx="source" title="source">Frontiers in Microbiology</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">348</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23055998">23055998</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Shen</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zeng</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yu</span>, <span class="given-names" tagx="given-names" title="given-names">Z. B.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Progress of the molecular ecology network analysis</span>. <span class="source" tagx="source" title="source">Environmental Science and Technology</span>, <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">94</span>–<span class="lpage" tagx="lpage" title="lpage">98</span>.</span></li> <li tag="ref"><a name="mbo3983-bib-0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Song</span>, <span class="given-names" tagx="given-names" title="given-names">Z. L. W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liu</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liang</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The diversities of Proteobacteria in four acidic hot springs in Yunnan</span>. <span class="source" tagx="source" title="source">Journal of Henan Agricultural University</span>, <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">376</span>–<span class="lpage" tagx="lpage" title="lpage">382</span>.</span></li> <li tag="ref"><a name="mbo3983-bib-0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stackebrandt</span>, <span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rainey</span>, <span class="given-names" tagx="given-names" title="given-names">F. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ward‐Rainey</span>, <span class="given-names" tagx="given-names" title="given-names">N. L.</span></span> (<span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Proposal for a New Hierarchic Classification System, Actinobacteria classis nov</span>. <span class="source" tagx="source" title="source">International Journal of Systematic Bacteriology</span>, <span class="volume" tagx="volume" title="volume">47</span>, <span class="fpage" tagx="fpage" title="fpage">479</span>–<span class="lpage" tagx="lpage" title="lpage">491</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/00207713-47-2-479">10.1099/00207713-47-2-479</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Steinmuller</span>, <span class="given-names" tagx="given-names" title="given-names">H. E.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dittmer</span>, <span class="given-names" tagx="given-names" title="given-names">K. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">White</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chambers</span>, <span class="given-names" tagx="given-names" title="given-names">L. G.</span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Understanding the fate of soil organic matter in submerging coastal wetland soils: A microcosm approach</span>. <span class="source" tagx="source" title="source">Geoderma</span>, <span class="volume" tagx="volume" title="volume">337</span>, <span class="fpage" tagx="fpage" title="fpage">1267</span>–<span class="lpage" tagx="lpage" title="lpage">1277</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.geoderma.2018.08.020">10.1016/j.geoderma.2018.08.020</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sun</span>, <span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lin</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular ecological network analyses revealing the effects of livestock grazing on soil microbial community in the Tibetan grassland</span>. <span class="source" tagx="source" title="source">Microbiology</span>, <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">1818</span>–<span class="lpage" tagx="lpage" title="lpage">1831</span>.</span></li> <li tag="ref"><a name="mbo3983-bib-0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thoms</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gattinger</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jacob</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thomas</span>, <span class="given-names" tagx="given-names" title="given-names">F. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gleixner</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Direct and indirect effects of tree diversity drive soil microbial diversity in temperate deciduous forest[J]</span>. <span class="source" tagx="source" title="source">Soil Biology and Biochemistry</span>, <span class="volume" tagx="volume" title="volume">42</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">1558</span>–<span class="lpage" tagx="lpage" title="lpage">1565</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2010.05.030">10.1016/j.soilbio.2010.05.030</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Too</span>, <span class="given-names" tagx="given-names" title="given-names">C. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Keller</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sickel</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lee</span>, <span class="given-names" tagx="given-names" title="given-names">S. M.</span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial community structure in a Malaysian tropical peat swamp forest: The influence of tree species and depth</span>. <span class="source" tagx="source" title="source">Frontiers in Microbiology</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">2859</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2018.02859">10.3389/fmicb.2018.02859</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30564202">30564202</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tripathi</span>, <span class="given-names" tagx="given-names" title="given-names">B. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kim</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tateno</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kim</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Soil pH and biome are both key determinants of soil archaeal community structure</span>. <span class="source" tagx="source" title="source">Soil Biology &amp;amp; Biochemistry</span>, <span class="volume" tagx="volume" title="volume">88</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.soilbio.2015.05.004">10.1016/j.soilbio.2015.05.004</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tsoy</span>, <span class="given-names" tagx="given-names" title="given-names">O. V.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ravcheev</span>, <span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cuklina</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Nitrogen fixation and molecular oxygen: Comparative genomic reconstruction of transcription regulation in Alphaproteobacteria</span>. <span class="source" tagx="source" title="source">Frontiers in Microbiology</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1343</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2016.01343">10.3389/fmicb.2016.01343</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27617010">27617010</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Garrity</span>, <span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tiedje</span>, <span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cole</span>, <span class="given-names" tagx="given-names" title="given-names">J. R.</span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Naïve Bayesian classifier for rapid assignment of rRNA sequences into the new bacteria taxonomy</span>. <span class="source" tagx="source" title="source">Applied and Environment Microbiology</span>, <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">5261</span>–<span class="lpage" tagx="lpage" title="lpage">5267</span>.</span></li> <li tag="ref"><a name="mbo3983-bib-0053" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zheng</span>, <span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Bacterioplankton community resilience to ocean acidification: Evidence from microbial network analysis</span>. <span class="source" tagx="source" title="source">ICES Journal of Marine Science</span>, <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">865</span>–<span class="lpage" tagx="lpage" title="lpage">875</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/icesjms/fsv187">10.1093/icesjms/fsv187</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">White</span>, <span class="given-names" tagx="given-names" title="given-names">D. C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sutton</span>, <span class="given-names" tagx="given-names" title="given-names">S. D.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ringelberg</span>, <span class="given-names" tagx="given-names" title="given-names">D. B.</span></span> (<span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">The genus Sphingomonas: Physiology and ecology</span>. <span class="source" tagx="source" title="source">Current Opinion in Biotechnology</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">301</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0958-1669(96)80034-6">10.1016/S0958-1669(96)80034-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8785434">8785434</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Will</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Thürmer</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wollherr</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nacke</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Horizon‐specific bacterial community composition of German grassland soils, as revealed by pyrosequencing‐based analysis of 16SrRNA genes</span>. <span class="source" tagx="source" title="source">Applied and Environment Microbiology</span>, <span class="volume" tagx="volume" title="volume">76</span>, <span class="fpage" tagx="fpage" title="fpage">6751</span>–<span class="lpage" tagx="lpage" title="lpage">6759</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.01063-10">10.1128/AEM.01063-10</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yeung</span>, <span class="given-names" tagx="given-names" title="given-names">M. K. S.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tegner</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Collins</span>, <span class="given-names" tagx="given-names" title="given-names">J. J.</span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Reverse engineering gene networks using singular value decomposition and robust regression</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">99</span>, <span class="fpage" tagx="fpage" title="fpage">6163</span>–<span class="lpage" tagx="lpage" title="lpage">6168</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.092576199">10.1073/pnas.092576199</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11983907">11983907</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0057" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhao</span>, <span class="given-names" tagx="given-names" title="given-names">Z. H.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dai</span>, <span class="given-names" tagx="given-names" title="given-names">M. H.</span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Drivers shaping the diversity and biogeography of total and active bacterial communities in the South China Sea</span>. <span class="source" tagx="source" title="source">Molecular Ecology</span>, <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">2260</span>–<span class="lpage" tagx="lpage" title="lpage">2274</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.12739">10.1111/mec.12739</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24684298">24684298</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0058" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zheng</span>, <span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xue</span>, <span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span>, <span class="given-names" tagx="given-names" title="given-names">X. Z.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">The responses and adaptations of microbial communities to salinity in farmland soils: A molecular ecological network analysis</span>. <span class="source" tagx="source" title="source">Applied Soil Ecology</span>, <span class="volume" tagx="volume" title="volume">120</span>, <span class="fpage" tagx="fpage" title="fpage">239</span>–<span class="lpage" tagx="lpage" title="lpage">246</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apsoil.2017.08.019">10.1016/j.apsoil.2017.08.019</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0059" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">J. Z.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Luo</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span>, <span class="given-names" tagx="given-names" title="given-names">Z. L.</span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Functional molecular ecological networks</span>. <span class="source" tagx="source" title="source">Mbio</span>, <span class="volume" tagx="volume" title="volume">1</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e00169‐10</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00169-10">10.1128/mBio.00169-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20941329">20941329</a></span></span></li> <li tag="ref"><a name="mbo3983-bib-0060" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Deng</span>, <span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Luo</span>, <span class="given-names" tagx="given-names" title="given-names">F.</span></span>, &amp;amp; <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic molecular ecological network of soil microbial communities in response to elevated CO<sub>2</sub></span>. <span class="source" tagx="source" title="source">MBio</span>, <span class="volume" tagx="volume" title="volume">2</span>, <span class="elocation-id" tagx="elocation-id" title="elocation-id">e00122‐11</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00122-11">10.1128/mBio.00122-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21791581">21791581</a></span></span></li> </ul> </div> <div class="app-group" title="app-group"> <div class="app" title="app"><span class="label" tagx="label" title="label">Appendix 1</span><div class="sec" title="sec"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure A1</span></a></div>   <p>Rarefaction curves in 10 layers</p>   </div> <div class="table-wrap_UNKNOWN" id="mbo3983-tbl-0005" orientation="portrait" position="anchor"><span class="label" tagx="label" title="label">Table A1</span> <p>The univariate ANOVA based on Shannon and Chao1 indices</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Depth</th> <th align="left" valign="top" rowspan="1" colspan="1">Shannon</th> <th align="left" valign="top" rowspan="1" colspan="1">Chao1</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">A</td> <td align="char" char="±" rowspan="1" colspan="1">7.98 ± 0.26</td> <td align="char" char="±" rowspan="1" colspan="1">1015.48 ± 96.92</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">B</td> <td align="char" char="±" rowspan="1" colspan="1">7.67 ± 0.78</td> <td align="char" char="±" rowspan="1" colspan="1">986.46 ± 199.05</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">C</td> <td align="char" char="±" rowspan="1" colspan="1">7.83 ± 0.79</td> <td align="char" char="±" rowspan="1" colspan="1">1034.79 ± 91.86</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">D</td> <td align="char" char="±" rowspan="1" colspan="1">7.53 ± 0.83</td> <td align="char" char="±" rowspan="1" colspan="1">1026.91 ± 109.15</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">E</td> <td align="char" char="±" rowspan="1" colspan="1">7.63 ± 0.79</td> <td align="char" char="±" rowspan="1" colspan="1">1025.48 ± 171.69</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">F</td> <td align="char" char="±" rowspan="1" colspan="1">7.60 ± 0.60</td> <td align="char" char="±" rowspan="1" colspan="1">1023.90 ± 108.78</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">G</td> <td align="char" char="±" rowspan="1" colspan="1">7.52 ± 0.89</td> <td align="char" char="±" rowspan="1" colspan="1">984.15 ± 123.27</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">H</td> <td align="char" char="±" rowspan="1" colspan="1">7.82 ± 0.93</td> <td align="char" char="±" rowspan="1" colspan="1">1040.34 ± 181.42</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">I</td> <td align="char" char="±" rowspan="1" colspan="1">7.58 ± 0.93</td> <td align="char" char="±" rowspan="1" colspan="1">1035.21 ± 171.58</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">J</td> <td align="char" char="±" rowspan="1" colspan="1">7.81 ± 0.39</td> <td align="char" char="±" rowspan="1" colspan="1">1013.16 ± 78.05</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="mbo3983-ntgp-0101"> <div class="fn-type-" title=""> <p>The values in the table were the mean ± standard deviation (<i>SD</i>) of eight replicates. No significant difference in depth gradient (p &amp;gt; .05).</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> <div class="table-wrap_UNKNOWN" id="mbo3983-tbl-0006" orientation="portrait" position="anchor"><span class="label" tagx="label" title="label">Table A2</span> <p>Correlations between environmental variables and key genus obtained by the module hubs along depth gradient by the Mantel test</p>  <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" style="border-right:solid 1px #000000" span="1" /></div> <thead valign="top"> <tr style="border-bottom:solid 1px #000000"> <th align="left" valign="top" rowspan="1" colspan="1">Environmental Variable</th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>A</p> <p>OTU832</p> <p>OTU2510</p> <p>OTU4954</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>B</p> <p>OTU71</p> <p>OTU498</p> <p>OTU774</p> <p>OTU1532</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>C</p> <p>OTU650</p> <p>OTU1466</p> <p>OTU1457</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>D</p> <p>OTU723</p> <p>OTU967</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>E</p> <p>OTU631</p> <p>OTU1510</p> <p>OTU1853</p> <p>OTU4767</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>F</p> <p>OTU396</p> <p>OTU1362</p> <p>OTU1366</p> <p>OTU2089</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>G</p> <p>OTU316</p> <p>OTU1664</p> <p>OTU2980</p> <p>OTU1853</p> <p>OTU2885</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>H</p> <p>OTU679</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>I</p> <p>OTU15</p> <p>OTU156</p> <p>OTU570</p> <p>OTU408</p> <p>OTU2365</p> </th> <th align="left" valign="top" rowspan="1" colspan="1"> <p>J</p> <p>OTU2930</p> </th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">K<sup>+</sup></td> <td align="char" char="." rowspan="1" colspan="1"><b>0.4474</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1">−0.1732</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.2505</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1">−0.2347</td> <td align="char" char="." rowspan="1" colspan="1">0.0621</td> <td align="char" char="." rowspan="1" colspan="1">−0.0162</td> <td align="char" char="." rowspan="1" colspan="1">0.1845</td> <td align="char" char="." rowspan="1" colspan="1">−0.1926</td> <td align="char" char="." rowspan="1" colspan="1">−0.2084</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.2024</b><a href="#mbo3983-note-0102"><b>*</b></a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Ca<sup>2+</sup></td> <td align="char" char="." rowspan="1" colspan="1">−0.1757</td> <td align="char" char="." rowspan="1" colspan="1">−0.1326</td> <td align="char" char="." rowspan="1" colspan="1">−0.2785</td> <td align="char" char="." rowspan="1" colspan="1">0.1966</td> <td align="char" char="." rowspan="1" colspan="1">0.3680</td> <td align="char" char="." rowspan="1" colspan="1">0.0851</td> <td align="char" char="." rowspan="1" colspan="1">0.4014</td> <td align="char" char="." rowspan="1" colspan="1">−0.0175</td> <td align="char" char="." rowspan="1" colspan="1">0.0853</td> <td align="char" char="." rowspan="1" colspan="1">0.1758</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Na<sup>+</sup></td> <td align="char" char="." rowspan="1" colspan="1">0.0859</td> <td align="char" char="." rowspan="1" colspan="1">0.1264</td> <td align="char" char="." rowspan="1" colspan="1">0.3348</td> <td align="char" char="." rowspan="1" colspan="1">−0.1997</td> <td align="char" char="." rowspan="1" colspan="1">−0.0563</td> <td align="char" char="." rowspan="1" colspan="1">0.1755</td> <td align="char" char="." rowspan="1" colspan="1">−0.0877</td> <td align="char" char="." rowspan="1" colspan="1">−0.0631</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.5248</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.1960</b><a href="#mbo3983-note-0102"><b>*</b></a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Mg<sup>2+</sup></td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.3225</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1">−0.1878</td> <td align="char" char="." rowspan="1" colspan="1">−0.0016</td> <td align="char" char="." rowspan="1" colspan="1">0.2329</td> <td align="char" char="." rowspan="1" colspan="1">0.1755</td> <td align="char" char="." rowspan="1" colspan="1">0.1049</td> <td align="char" char="." rowspan="1" colspan="1">−0.0454</td> <td align="char" char="." rowspan="1" colspan="1">0.0425</td> <td align="char" char="." rowspan="1" colspan="1">0.2017</td> <td align="char" char="." rowspan="1" colspan="1">−0.2367</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NH</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
+</div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.1156</td> <td align="char" char="." rowspan="1" colspan="1">−0.0810</td> <td align="char" char="." rowspan="1" colspan="1">−0.0673</td> <td align="char" char="." rowspan="1" colspan="1">−0.2864</td> <td align="char" char="." rowspan="1" colspan="1">−0.2065</td> <td align="char" char="." rowspan="1" colspan="1">0.0586</td> <td align="char" char="." rowspan="1" colspan="1">0.2035</td> <td align="char" char="." rowspan="1" colspan="1">−</td> <td align="char" char="." rowspan="1" colspan="1">0.0185</td> <td align="char" char="." rowspan="1" colspan="1">0.2037</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.1066</td> <td align="char" char="." rowspan="1" colspan="1">0.2282</td> <td align="char" char="." rowspan="1" colspan="1">−0.0259</td> <td align="char" char="." rowspan="1" colspan="1">0.0508</td> <td align="char" char="." rowspan="1" colspan="1">0.0905</td> <td align="char" char="." rowspan="1" colspan="1">0.1427</td> <td align="char" char="." rowspan="1" colspan="1">−</td> <td align="char" char="." rowspan="1" colspan="1">0.0182</td> <td align="char" char="." rowspan="1" colspan="1">−0.0544</td> <td align="char" char="." rowspan="1" colspan="1">−0.1956</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
NO</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
-</div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.0288</td> <td align="char" char="." rowspan="1" colspan="1">−0.1500</td> <td align="char" char="." rowspan="1" colspan="1">−</td> <td align="char" char="." rowspan="1" colspan="1">0.3382</td> <td align="char" char="." rowspan="1" colspan="1">0.1822</td> <td align="char" char="." rowspan="1" colspan="1">−0.0630</td> <td align="char" char="." rowspan="1" colspan="1"><b>0.3516</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1"><b>0.3547</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1">−0.0001</td> <td align="char" char="." rowspan="1" colspan="1">−0.1995</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Cl<sup>−</sup></td> <td align="char" char="." rowspan="1" colspan="1">0.0520</td> <td align="char" char="." rowspan="1" colspan="1">0.0577</td> <td align="char" char="." rowspan="1" colspan="1">0.3111</td> <td align="char" char="." rowspan="1" colspan="1">−0.1237</td> <td align="char" char="." rowspan="1" colspan="1">0.1017</td> <td align="char" char="." rowspan="1" colspan="1">0.0973</td> <td align="char" char="." rowspan="1" colspan="1">−0.1182</td> <td align="char" char="." rowspan="1" colspan="1">−0.1553</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.5618</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1">−0.1395</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mtext_UNKNOWN">
SO</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
-</div> </div> </div></div></td> <td align="char" char="." rowspan="1" colspan="1">−0.0939</td> <td align="char" char="." rowspan="1" colspan="1">0.1519</td> <td align="char" char="." rowspan="1" colspan="1">0.1277</td> <td align="char" char="." rowspan="1" colspan="1">0.1251</td> <td align="char" char="." rowspan="1" colspan="1">0.2083</td> <td align="char" char="." rowspan="1" colspan="1">−0.0863</td> <td align="char" char="." rowspan="1" colspan="1">−0.0776</td> <td align="char" char="." rowspan="1" colspan="1">−0.0644</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.3232</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.2408</b><a href="#mbo3983-note-0102"><b>*</b></a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TOC</td> <td align="char" char="." rowspan="1" colspan="1">0.0596</td> <td align="char" char="." rowspan="1" colspan="1">0.2642</td> <td align="char" char="." rowspan="1" colspan="1">0.0393</td> <td align="char" char="." rowspan="1" colspan="1">0.1997</td> <td align="char" char="." rowspan="1" colspan="1">0.1807</td> <td align="char" char="." rowspan="1" colspan="1">−0.0481</td> <td align="char" char="." rowspan="1" colspan="1">−0.4587</td> <td align="char" char="." rowspan="1" colspan="1">−0.2181</td> <td align="char" char="." rowspan="1" colspan="1">−0.0410</td> <td align="char" char="." rowspan="1" colspan="1">−0.1434</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">TN</td> <td align="char" char="." rowspan="1" colspan="1">0.07764</td> <td align="char" char="." rowspan="1" colspan="1">0.2453</td> <td align="char" char="." rowspan="1" colspan="1">0.1979</td> <td align="char" char="." rowspan="1" colspan="1">0.1101</td> <td align="char" char="." rowspan="1" colspan="1">0.0962</td> <td align="char" char="." rowspan="1" colspan="1">0.0132</td> <td align="char" char="." rowspan="1" colspan="1">−</td> <td align="char" char="." rowspan="1" colspan="1">−0.149500</td> <td align="char" char="." rowspan="1" colspan="1">−0.0903</td> <td align="char" char="." rowspan="1" colspan="1">−0.0952</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">pH</td> <td align="char" char="." rowspan="1" colspan="1">0.1327</td> <td align="char" char="." rowspan="1" colspan="1">−0.0847</td> <td align="char" char="." rowspan="1" colspan="1">−0.0094</td> <td align="char" char="." rowspan="1" colspan="1">−0.2744</td> <td align="char" char="." rowspan="1" colspan="1">0.1294</td> <td align="char" char="." rowspan="1" colspan="1">0.0678</td> <td align="char" char="." rowspan="1" colspan="1">−0.1846</td> <td align="char" char="." rowspan="1" colspan="1">−0.0379</td> <td align="char" char="." rowspan="1" colspan="1">−0.0903</td> <td align="char" char="." rowspan="1" colspan="1">−0.0141</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Salinity</td> <td align="char" char="." rowspan="1" colspan="1">−0.0306</td> <td align="char" char="." rowspan="1" colspan="1">−0.0493</td> <td align="char" char="." rowspan="1" colspan="1">0.1228</td> <td align="char" char="." rowspan="1" colspan="1">−0.0614</td> <td align="char" char="." rowspan="1" colspan="1">0.2043</td> <td align="char" char="." rowspan="1" colspan="1">−0.0623</td> <td align="char" char="." rowspan="1" colspan="1">0.0147</td> <td align="char" char="." rowspan="1" colspan="1">0.1850</td> <td align="char" char="." rowspan="1" colspan="1"><b>−0.5251</b><a href="#mbo3983-note-0102"><b>*</b></a></td> <td align="char" char="." rowspan="1" colspan="1">−0.1903</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN" id="mbo3983-ntgp-0102"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p><i>p</i> &amp;lt; .05</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">**</span><p><i>p</i> &amp;lt; .01.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd</span>, </span></div> </div> </div> </div> </div> </div>  </body></html>