<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Emerg Infect Dis</span><span class="iso-abbrev" title="iso-abbrev">Emerging Infect. Dis</span><span class="publisher-id" title="publisher-id">EID</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Emerging Infectious Diseases</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1080-6040</span><span class="issn-epub" tagx="issn" title="issn-epub">1080-6059</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Centers for Disease Control and Prevention</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6433011">6433011</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30730827">30730827</a></span><span class="publisher-id" title="publisher-id">18-0779</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3201/eid2504.180779">10.3201/eid2504.180779</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research</span>: <span class="subject" title="subject">Research</span>: <span class="subject" title="subject">Human-Origin Influenza A(H3N2) Reassortant Viruses in Swine, Southeast Mexico</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Human-Origin Influenza A(H3N2) Reassortant Viruses in Swine, Southeast Mexico</div> <div class="alt-title" title="alt-title">
Influenza A(H3N2) Viruses in Swine, Mexico</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">Martha I.</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Souza</span><span class="given-names" tagx="given-names" title="given-names">Carine K.</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trovão</span><span class="given-names" tagx="given-names" title="given-names">Nídia S.</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diaz</span><span class="given-names" tagx="given-names" title="given-names">Andres</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mena</span><span class="given-names" tagx="given-names" title="given-names">Ignacio</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rovira</span><span class="given-names" tagx="given-names" title="given-names">Albert</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">Amy L.</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torremorell</span><span class="given-names" tagx="given-names" title="given-names">Montserrat</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marthaler</span><span class="given-names" tagx="given-names" title="given-names">Douglas</span></span><a href="#FN1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Culhane</span><span class="given-names" tagx="given-names" title="given-names">Marie R.</span></span></span><span class="citation_author_institution" id="aff1">[], </span><span class="citation_author_institution" id="aff2">[], </span><span class="citation_author_institution" id="aff3">[], </span><span class="citation_author_institution" id="aff4">[], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Address for correspondence: Martha I. Nelson, National Institutes of Health, 16 Center Dr, Bethesda, MD 20892, USA; email: <span class="email" tagx="email" title="email">nelsonma@mail.nih.gov</span></div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-4</span></span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">691</span><span class="lpage" tagx="lpage" title="lpage">700</span><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The genetic diversity of influenza A viruses circulating in swine in Mexico complicates control efforts in animals and presents a threat to humans, as shown by influenza A(H1N1)pdm09 virus. To describe evolution of swine influenza A viruses in Mexico and evaluate strains for vaccine development, we sequenced the genomes of 59 viruses and performed antigenic cartography on strains from 5 regions. We found that genetic and antigenic diversity were particularly high in southeast Mexico because of repeated introductions of viruses from humans and swine in other regions in Mexico. We identified novel reassortant H3N2 viruses with genome segments derived from 2 different viruses that were independently introduced from humans into swine: pandemic H1N1 viruses and seasonal H3N2 viruses. The Mexico swine viruses are antigenically distinct from US swine lineages. Protection against these viruses is unlikely to be afforded by US virus vaccines and would require development of new vaccines specifically targeting these diverse strains.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords: </div> <div class="kwd" title="kwd">
influenza A virus</div> <div class="kwd" title="kwd">
influenza virus</div> <div class="kwd" title="kwd">
reassortant influenza viruses</div> <div class="kwd" title="kwd">
viruses</div> <div class="kwd" title="kwd">
influenza</div> <div class="kwd" title="kwd">
swine</div> <div class="kwd" title="kwd">
evolution</div> <div class="kwd" title="kwd">
phylogenetic analysis</div> <div class="kwd" title="kwd">
antigenic cartography</div> <div class="kwd" title="kwd">
reassortment</div> <div class="kwd" title="kwd">
phylogeography</div> <div class="kwd" title="kwd">
pandemic</div> <div class="kwd" title="kwd">
human origin</div> <div class="kwd" title="kwd">
reverse zoonosis</div> <div class="kwd" title="kwd">
zoonoses</div> <div class="kwd" title="kwd">
vaccine</div> <div class="kwd" title="kwd">
respiratory infections</div> <div class="kwd" title="kwd">
Mexico</div> </div> </div> </div> <div class="body" title="body"> <p>Genetically diverse influenza A viruses (IAV) circulate in swine globally, complicating efforts to control the virus and increasing the threat that a novel virus will emerge in pigs with the capacity to infect humans. This threat was exemplified by the influenza A(H1N1)pdm09 virus, which originated in swine in Mexico, most likely in the west or central–north regions (<a href="#R1"><i>1</i></a>). Although IAVs have been endemic in US swine herds since 1919 (<a href="#R2"><i>2</i></a>), there is no evidence that IAVs circulated in swine in Mexico before the 1990s. The emergence of Mexico as a hub of swine IAV diversity with pandemic potential is a relatively recent event.</p> <p>During 1989–2015, &amp;gt;2 million hogs raised in the United States were transported to Mexico, representing &amp;gt;87% of all US swine exports (<a href="#F1">Figure 1</a>, panel A). During the 1990s, ≈100,000 live hogs were transported annually, on average, from the United States to Mexico (<a href="#F1">Figure 1</a>, panel B), thus facilitating establishment of 2 major North American swine IAV lineages in Mexico by the end of the decade: triple reassortant swine H3N2 (TRswH3N2) viruses and classical swine H1N1 (CswH1N1) viruses (<a href="#R1"><i>1</i></a>) (referred to as lineage 1A according to recently proposed H1 nomenclature [<i>3</i>]). Avian-like Eurasian swine H1N1 (EAswH1N1, lineage 1C [<i>3</i>]) viruses were also introduced into multiple regions of Mexico from Europe. Previously, EAswH1N1 viruses had only been detected in Europe and Asia (<a href="#R4"><i>4</i></a>–<a href="#R6"><i>6</i></a>), and Mexico is the only country in the Americas where TRswH3N2, CswH1N1, and EAswH1N1 lineages co-circulate and exchange genome segments by reassortment (<a href="#R1"><i>1</i></a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Swine exportation to Mexico from the United States and eradication of CSFV in Mexico. A) Hogs exported from the United States to other countries globally during 1989–2015. Of ≈3.7 million exported, ≈3.1 million (≈87%) were exported to Mexico. B) Since 1989, the number of hogs exported from the United States to Mexico has experienced year-to-year variation. Data are available from the US International Trade Commission (<a href="https://dataweb.usitc.gov">https://dataweb.usitc.gov</a>).</p>   </div> <p>Increased surveillance of swine IAV in Mexico and other understudied countries has elucidated how international trade of live swine drives the long-distance migration of these viruses. For many decades, countries with less-established swine production have imported breeding stock from North America and Europe, facilitating long-range dissemination of swine IAV lineages to China and Southeast Asia (<a href="#R4"><i>4</i></a>,<a href="#R7"><i>7</i></a>,<a href="#R8"><i>8</i></a>). Swine movements between US regions regularly facilitate long-range swine IAV migration across the country (<a href="#R9"><i>9</i></a>), but long-distance movements of pigs between regions in Mexico are less frequent.</p> <p>The 6 largest swine-producing states in Mexico are located in the west (Jalisco, ≈2.8 million swine), northwest (Sonora, ≈1.7 million swine), east (Puebla, ≈1.6 million swine and Veracruz, ≈1.5 million swine), central–north (Guanajuato, ≈0.9 million swine), and southeast (Yucatan, ≈0.9 million swine) regions (<a href="#F2">Figure 2</a>). During 1973–2009, Mexico implemented a campaign to eradicate classical swine fever virus (CSFV), including restrictions on animal movements between CSFV-free areas and central regions experiencing outbreaks (<a href="#R10"><i>10</i></a>) (<a href="#F2">Figure 2</a>). Lifting of internal movement controls during 2009 provided new opportunities for swine IAV to migrate between regions in Mexico.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Swine exportation to Mexico from the United States and eradication of CSFV in Mexico. The 32 states of Mexico shaded according to the year when CSFV was declared to be eradicated by the Secretariat of Agriculture, Livestock, Rural Development, Fishery and Food of the government of Mexico. International borders are shaded between Mexico and the United States (red), Belize (blue), and Guatemala (violet). The 8 regions of Mexico are indicated, and their borders are shaded orange. CSFV, classical swine fever virus.</p>   </div> <p>To determine the genetic and antigenic diversity of swine IAVs circulating in Mexico, we sequenced genomes of 59 swine IAV samples collected in northwest and southeast Mexico (GenBank accession nos. MG836712–831) (<a href="#SD1">Appendix</a> Table 1). We also evaluated the relationship of Mexico swine IAVs to those used in existing vaccines to determine whether new vaccine development is necessary.</p> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="collectionofswineiavinmexico" title="sec"> <div class="title" tagx="title" title="title">
Collection of Swine IAV in Mexico</div> <p>Respiratory tract samples were collected by veterinarians and farm staff from pigs with clinical signs of respiratory disease on farms in the northwest and southeast regions of Mexico during 2010–2014. These samples were sent ad hoc to the University of Minnesota Veterinary Diagnostic Laboratory (St. Paul, MN, USA) for swine respiratory disease diagnostic investigations. This laboratory conducted matrix gene testing for swine IAV by using a real-time reverse transcription PCR (RT-PCR) (<a href="#R11"><i>11</i></a>) and virus isolation in MDCK cells (<a href="#R12"><i>12</i></a>) for swine IAV PCR-positive samples, as requested by the submitting party. Hemagglutinin (HA) and neuraminidase (NA) subtyping of RT-PCR–positive samples was completed (<a href="#R13"><i>13</i></a>). Sequencing of the HA, but not other segments, was performed previously (<a href="#R14"><i>14</i></a>).</p> <p>We obtained whole-genome sequences from virus isolates or directly from the originally submitted respiratory tract material. RNA was extracted from the original material or virus isolates as described (<a href="#R14"><i>14</i></a>). In brief, we extracted virus RNA from 50 μL of swab supernatant by using a magnetic bead procedure and obtained segment-specific PCR fragments by using the One-Step RT-PCR (QIAGEN, <a href="https://www.qiagen.com">https://www.qiagen.com</a>) and IAV-specific primers for each genome segment as described (<a href="#R15"><i>15</i></a>). We obtained 59 complete genomes, 37 from southeast Mexico and 22 from northwest Mexico. All sequences were submitted to GenBank under accessions nos. MG836712–831.</p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Analysis</div> <p>We generated nucleotide alignments for each of the 8 segments of the virus genome (HA, matrix protein [MP], NA, nucleoprotein [NP], nonstructural [NS], polymerase acidic [PA], polymerase basic 1 [PB1], and polymerase basic 2 [PB2]) by using MUSCLE version 3.8.31 (<a href="#R16"><i>16</i></a>). We inferred initial trees for each segment by using neighbor-joining methods to identify major lineages, including pandemic (p), TRswH3N2 (t), CswH1N1 (c), and EAswH1N1 (e): PB2t, PB1p, PB1t, PAp, PAt, H1c, H1p, H3t + H3h, NPp, NPt, N1c, N1e, N1p, N2t + N2h, MPe, MPp, MPt, NSp, and NSt. For simplicity, we visualized human H3N2 (huH3N2) and TRswH3N2 together on the H3 and N2 phylogenies. As additional background data, we downloaded related human and swine sequences from the Influenza Virus Resource (<a href="#R17"><i>17</i></a>) that were studied previously, including swine viruses from Mexico, which are indicated with the abbreviation AVX (e.g., A/swine/Mexico/AVX-24/2012[H1N1]) (<a href="#R1"><i>1</i></a>).</p> <p>For each pandemic H1N1 virus alignment, we inferred the phylogenetic relationships by using the maximum-likelihood method in RAxML version 7.2.6 (<a href="#R18"><i>18</i></a>) and incorporated a general time-reversible model of nucleotide substitution with a gamma-distributed rate variation among sites. To assess the robustness of each node, we performed a bootstrap resampling process (500 replicates). We used the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health (Bethesda, MD, USA) (<a href="http://biowulf.nih.gov">http://biowulf.nih.gov</a>). For TRswH3N2, EAswH1N1, CswH1N1, and huH3N2, we inferred time-scaled phylogenies by using Markov chain Monte Carlo (MCMC) methods available in the BEAST package version 1.8.4 (<a href="#R19"><i>19</i></a>) and used a relaxed uncorrelated lognormal molecular clock, a constant population demographic model, and a general time-reversible model of nucleotide substitution with a gamma-distributed rate variation among sites. We ran the MCMC chain separately 3 times for each of the datasets for <u>&amp;gt;</u>100 million iterations, with subsampling every 10,000 iterations.</p> <p>We used the BEAGLE library (<a href="#R20"><i>20</i></a>) to improve computational performance. All parameters reached convergence, as assessed visually by using Tracer version 1.6 in BEAST; statistical uncertainty was reflected by values of the 95% highest posterior density. The initial 10% of the chain was removed as burn-in, runs were combined by using LogCombiner version 1.8.4 in BEAST, and maximum clade credibility trees were summarized by using TreeAnnotator version 1.8.4 in in BEAST. We classified H1 viruses by using the H1 swine clade classification tool available at the Influenza Research Database (<a href="http://www.fludb.org">http://www.fludb.org</a>) (<a href="#R3"><i>3</i></a>) (<a href="#SD1">Appendix</a> Table 1).</p> <p>We inferred the timing of the introduction of human seasonal H3N2 viruses into swine in southeast Mexico on the basis of the time to the most recent common ancestor of the clade of viruses identified in swine and the most closely related human viruses, sampled in the mid-1990s. Inferring the precise timing of human-to-swine transmission events that occurred many decades ago is complicated by the lack of sequence data available from swine in previous decades, and the most parsimonious interpretation is that long branch lengths indicate gaps in sampling in swine, rather than humans, an approach that has been described in detail (<a href="#R13"><i>13</i></a>).</p> </div> <div class="spatialanalysis" title="sec"> <div class="title" tagx="title" title="title">
Spatial Analysis</div> <p>The 31 states of Mexico (plus the Federal District) are located within 8 defined regions (<a href="#F2">Figure 2</a>). We conducted phylogeographic analysis at a regional level to ensure deidentification for swine producers. Swine IAV sequences were available for 5 regions in Mexico (northwest, west, central–north, east, and southeast). To reduce complexity for maximum clade credibility, we categorized viruses categorized as northwest region (NW), southeast region (SE), or central-north (CN), east (E), or west (W) regions.</p> <p>We categorized global background sequences as USA/Canada, Asia, Europe, South America, and humans (globally). The location state was specified for each virus sequence, which enabled the expected number of location state transitions in the ancestral history conditional on the data observed at the 3 tips to be estimated by using Markov jump counts (<a href="#R21"><i>21</i></a>), providing a quantitative measure of asymmetry in gene flow between regions. The location of viruses in the pandemic H1N1 clade (pH1N1) was left uninformed, which enabled the reconstruction of the location state of the common ancestor of pH1N1 to be unbiased by human data. For computational efficiency we conducted the phylogeographic analysis by using an empirical distribution of 1,000 trees (<a href="#R22"><i>22</i></a>), running the MCMC chain for 25 million iterations, and sampling every 1,000 steps. We used a nonreversible diffusion model and Bayesian stochastic search variable selection to improve statistical efficiency for all datasets containing &amp;gt;4 location states. Heat maps were constructed by using the R package to summarize Markov jump counts inferred over phylogenies for all segments and swine IAV lineages (<a href="#R23"><i>23</i></a>).</p> </div> <div class="movementsofliveswine" title="sec"> <div class="title" tagx="title" title="title">
Movements of Live Swine</div> <p>We obtained the number of US live swine exports during 1989–2015 from the US International Trade Commission (<a href="https://dataweb.usitc.gov/scripts/REPORT.asp">https://dataweb.usitc.gov/scripts/REPORT.asp</a>) by using HS Code 0103. Data were cross-referenced against the United Nations Commodity Trade Statistics Database (<a href="http://comtrade.un.org">http://comtrade.un.org</a>), which provides the trade value (in US dollars) for live swine trade between countries for the years 1996–2012. We obtained the estimated live swine population size of Mexico from the Mexico Secretariat of Agriculture, Livestock, Rural Development, Fisheries and Food (<a href="https://www.siap.gob.mx">https://www.siap.gob.mx</a>). Information on movements of live swine between Mexican states is available only for the relatively small number of pigs that go through plants that pass official government inspections, known as Tipo Inspección Federal plants.</p> <p>Information on the year of CSFV eradication and opening of border restrictions in each state of Mexico was based on reports of the Secretariat of Agriculture, Livestock, Rural Development, Fisheries and Food. We corroborated these data with reports from the Animal and Plant Health Inspection Service of the US Department of Agriculture (US National Archives) (<a href="https://www.federalregister.gov">https://www.federalregister.gov</a>; document nos. 02-11897, 02-24753, and E7-10641).</p> </div> <div class="antigeniccharacterization" title="sec"> <div class="title" tagx="title" title="title">
Antigenic Characterization</div> <p>Representative H3N2 strains from the United States and Mexico and a human seasonal strain (<a href="#SD1">Appendix</a> Table 2) genetically related to the unique H3N2 swine virus introduced into Mexico were selected from sequence and motif analyses and used for hemagglutination inhibition (HI) assays. The analysis included 5 swine influenza viruses from Mexico reported by Mena et al. (<a href="#R1"><i>1</i></a>). Monovalent antiserum against swine IAV H3N2 strains was prepared in swine as described (<a href="#R24"><i>24</i></a>) and used for HI assays (<a href="#R25"><i>25</i></a>) with 5 H3N2 swine viruses from Mexico as antigens. HI data were used to determine the antigenic relationships between swine H3 from Mexico and the United States by using antigenic cartography 3-dimensional maps (<a href="#R24"><i>24</i></a>). Antigenic distances between viruses were calculated in antigenic units (AUs), in which 1 AU is equivalent to a 2-fold loss in HI cross-reactivity (<a href="#SD1">Appendix</a> Table 3). Antigenic distances generated in the 3-dimensional map between the H3N2 antigens were plotted by using GraphPad Prism Version 7.03 (<a href="https://www.graphpad.com">https://www.graphpad.com</a>).</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="geneticdiversityofswineiavinmexico" title="sec"> <div class="title" tagx="title" title="title">
Genetic Diversity of Swine IAV in Mexico</div> <p>Phylogenetic analysis of the 59 swine IAV whole-genome sequences generated for this study showed that extensive genetic diversity is circulating in the northwest and southeast regions of Mexico. This viral diversity was produced by 3 evolutionary processes: human-to-swine transmission, long-distance movements of swine, and genomic reassortment. We identified 15 genotypes, including 13 reassortant genotypes with segments from multiple IAV lineages (<a href="#F3">Figure 3</a>). Three IAV lineages were observed in swine in northwest Mexico: human-origin pH1N1 (1A.3.3.2), TRswH3, and CswH1. These 3 lineages also were identified in southeast Mexico, along with 2 additional lineages: EAswH1N1, likely introduced from pigs in Europe, and huH3N2, which appears to have been introduced from humans into swine in the 1990s. Eight reassortant genotypes were identified that were not identified in swine in Mexico in previous studies (<a href="#R1"><i>1</i></a>), including 1 triple reassortant (TRswH3N2/pH1N1/EAswH1N1, genotype 6) (<a href="#F3">Figure 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Genetic diversity of influenza A viruses (IAVs) circulating in swine in southeast and northwest Mexico. Fifteen genotypes were identified by surveillance in swine herds in Mexico during 2010–2014. Each oval represents 1 of the 8 segments of the virus genome. The surface antigens HA and NA are listed first, followed by the 6 internal gene segments. Shading of each oval corresponds to 1 of 5 major genetic lineages of swine IAV circulating in humans and swine globally. The rightmost columns indicate the number of viruses from each genotype collected from the northwest and southeast regions in Mexico. Ovals shaded with 2 colors represent mixed infections. CswH1, classical swine H1N1; EAswH1, avian-like Eurasian swine H1N1; HA, hemagglutinin; huH3N2, human H3N2; MP, matrix protein; NA, neuraminidase; NP, nucleoprotein; NS, nonstructural; NW, northwest; PA, polymerase acidic; PB, polymerase basic; pH1N1, pandemic H1N1 clade; SE, southeast; SW, southwest.TRswH3N2, triple reassortant swine H3N2.</p>   </div> <p>Seven of the 8 reassortant genotypes identified in northwest Mexico were generated by reassortment between TRswH3N2 and pH1N1 viruses. Genotype 8 is similar to the pH1N1 precursor viruses identified previously in the west and central-north regions of Mexico (<a href="#R1"><i>1</i></a>) and represents the first detection of the genotype outside of the west and central-north regions (<a href="#F3">Figure 3</a>). Approximately 35% (13/37) of the viruses identified in southeast Mexico were pH1N1/huH3N2 reassortants (<a href="#F3">Figure 3</a>), in which internal genes were replaced with internal genes of a pH1N1 virus. Seven viruses had possible evidence of co-infection, but only in the MP segment. However, this finding requires further confirmation.</p> </div> <div class="geneticdiversityofhaandnasegments" title="sec"> <div class="title" tagx="title" title="title">
Genetic Diversity of HA and NA Segments</div> <p>Overall, greater genetic diversity was observed in the HA and NA antigens than in the 6 internal gene segments, a pattern that has been observed in other countries (<a href="#R26"><i>26</i></a>). CswH1N1-H1β (1A.2), pH1N1 (1A.3.3.2), and TRswH3N2 (cluster IV) viruses were identified in northwest Mexico. Five HAs (CswH1N1-H1γ [1A.3.3.3], pH1N1 [1A.3.3.2], TRswH3N2 [cluster IV], and huH3N2) (<a href="#F3">Figures 3</a>, <a href="#F4">4</a>; <a href="#SD1">Appendix</a> Table 1) were identified in southeast Mexico. Five NA lineages (EAswH1N1, pH1N1, huH3N2, TRswH3N2/N2–1998, and TRswH3N2/N2–2002) (<a href="#F5">Figure 5</a>) were identified in southeast Mexico. Given the high genetic diversity of H3 viruses, including huH3N2 viruses that were not phylogenetically related to any known US swine lineages (<a href="#F4">Figure 4</a>), we selected 10 representative H3 viruses from Mexico on the basis of phylogenetic position (<a href="#F4">Figure 4</a>) and antigenic motif patterns (<a href="#R27"><i>27</i></a>) for antigenic characterization.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Evolutionary relationships between H3 segments of avian influenza viruses collected in humans and swine globally. Time-scaled Bayesian maximum clade credibility tree is inferred for the H3 segment. The tree includes newly generated sequences from northwest and southeast Mexico, along with background sequences from swine in Mexico, Asia, the United States, and Canada, as well as seasonal H3N2 viruses from humans. The color of each branch indicates the most probable location state. Posterior probabilities for key nodes are provided. Clades with viruses from swine in Mexico obtained for this study are highlighted in yellow. *Major virus introductions into a location, indicating direct introductions from humans. CN, central; NW, northwest; S, south; SE, southeast; W, west. The 10 H3N2 viruses selected for antigenic characterization (<a href="#F6">Figure 6</a>) are indicated by numbers 1–10 in red. A more detailed phylogeny, including tip labels and all posterior probabilities, is provided in the <a href="#SD1">Appendix</a> Figure.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Evolutionary relationships between N2 segments of avian influenza viruses collected in humans and swine globally. Time-scaled Bayesian maximum clade credibility tree is inferred for N2 segment. Labeling and shading is similar to that in <a href="#F4">Figure 4</a>, with the additional labeling of the N2–1998 and N2–2002 lineages. NW, northwest; SE, southeast; W, west; W/C, west/central. *Direct introduction from humans. A more detailed phylogeny, including tip labels and all posterior probabilities, is provided in the <a href="#SD1">Appendix</a> Figure.</p>   </div> </div> <div class="antigenicanalysisofh3viruses" title="sec"> <div class="title" tagx="title" title="title">
Antigenic Analysis of H3 Viruses</div> <p>We generated an antigenic map from HI data to visualize antigenic relationships among contemporary H3N2 swine IAV from multiple regions in Mexico and the United States (<a href="#F6">Figure 6</a>, panel A). Antigenic distances were extracted from the map to measure distance between H3N2 viruses from Mexico and swine strains from the United States that represent putative antigenic clusters contained in US commercial vaccines (<a href="#F6">Figure 6</a>, panel B). Of the 10 viruses from Mexico selected for antigenic characterization, 8 were antigenically clustered with H3N2 cluster IV strains from the United States. Two viruses from the southeast region (A/sw/Mex/985778/2013 and A/sw/Mex/84706352130/2015) were more antigenically similar to the human seasonal vaccine strain WU/95 (&amp;lt;3 AUs). The most similar swine strain is the cluster I TX/98.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Antigenic relationships between contemporary influenza A (H3N2) viruses from Mexico and representative strains from the United States. A) Antigenic map of contemporary swine H3N2 viruses from Mexico and the United States. Antigenic clusters are indicated by color as in <a href="#F4">Figures 4</a> and <a href="#F5">5</a>. CN, central; E, east; NW, northwest; SE, southeast; W, west. B) Antigenic distance between H3N2 virus from Mexico and representative swine strains from the United States and a putative ancestral human seasonal vaccine virus strain (WU/95). The US swine strains represent antigenic clusters previously defined as TX/98 (cornflower), OH/04 and MN/09 (cyan), NY/11 (red), and IA/14 (green) (<a href="#R27"><i>27</i></a>,<a href="#R31"><i>31</i></a>).</p>   </div> </div> <div class="spatialstructureofswineiavinmexico" title="sec"> <div class="title" tagx="title" title="title">
Spatial Structure of Swine IAV in Mexico</div> <p>We observed no evidence of viral migration between the northwest and southeast regions of Mexico. Although certain clades of pH1N1 viruses from northwest and southeast Mexico share a common ancestor, the low posterior probabilities supporting these nodes indicate that multiple independent human-to-swine transmission events of closely related human pH1N1 viruses are more likely. There might have been 4 independent introductions of pH1N1 virus from humans into swine in southeast Mexico and 5 introductions into the northwest region (<a href="#SD1">Appendix</a> Figure). Similarly, there is no evidence of dispersal of huH3N2 viruses from southeast Mexico to any other regions of Mexico. TRswH3N2 and CswH1N1 viruses from swine in the United States were imported into the northwest region of Mexico, which is located near the US border (<a href="#F7">Figure 7</a>, panel A), reflecting the large number of live swine transported from the southern United States into Mexico each year (<a href="#F1">Figure 1</a>). In contrast, TRswH3N2 and CswH1N1 viruses were imported into the southeast region of Mexico primarily from the west region (<a href="#F7">Figure 7</a>, panel B), an inference that was observed with high support across all phylogenies (<a href="#F4">Figures 4</a>, <a href="#F5">5</a>; <a href="#SD1">Appendix</a> Figure).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Sources of influenza A viruses circulating in swine in northwestern and southeastern Mexico. Each region is shaded according to the proportion of total Markov jump counts from that particular region (source) into A) northwest or B) southeast regions of Mexico (destination). Red indicates high proportion of jumps (major source of viruses); light yellow indicates low proportion of jumps (not a major source of viruses); black indicates destination; white indicates no jumps/no data available. Seven locations were considered in the analysis: Canada, United States, Mexico (northwest), Mexico (central-west), Mexico (central-north), Mexico (east), and Mexico (southeast). Scale bar indicates proportion of total Markov jump counts from a particular region.</p>   </div> <p>The trees indicate that 3 independent introductions of reassortant viruses, including EAswH1N1 segments (genotypes 6 and 8) (<a href="#F3">Figure 3</a>), occurred during ≈2011–2013 on the basis of estimated times to the most recent common ancestor (<a href="#F8">Figure 8</a>; <a href="#SD1">Appendix</a> Table 4). No additional reassortment events have been observed between genotype 6 or 8 viruses and huH3N2 and pH1N1 viruses also circulating in southeast Mexico.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>Introductions of influenza A viruses into swine in southeast Mexico. Introductions of these viruses from humans and swine into southeast Mexico are indicated as groups of 8 horizontal lines, and each line represents 1 of 8 segments of the virus genome. Segments are numbered 1–8, longest to shortest, according to convention: 1, polymerase basic 2; 2, polymerase basic 1; 3, polymerase acidic; 4, hemagglutinin; 5, nucleoprotein; 6, neuraminidase; 7, matrix protein; 8, nonstructural. The position of each shaded circle along the x-axis represents the estimated timing of virus introduction, estimated directly from the maximum clade credibility trees. Lines and circles are shaded according to the virus lineage, similar to that in <a href="#F2">Figure 2</a>. *Reassortment events. Introduction of genotype 8 viruses is indicated. The blue dotted vertical line represents the date of the declaration of Secretariat of Agriculture, Livestock, Rural Development, Fisheries and Food in Mexico that classical swine fever virus had been eradicated from all states in Mexico during January 2009. The red dotted vertical line represents the date of first isolation of pH1N1 virus in humans. CSFV, classical swine fever virus; CswH1, classical swine H1N1; EAswH1, avian-like Eurasian swine H1N1; huH3N2, human H3N2; pH1N1, pandemic H1N1 clade; TRswH3N2, triple reassortant swine H3N2.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The recent characterization of the origins of the influenza A(H1N1)pdm09 virus in swine in Mexico underscored the need for efforts to control the unexpectedly diverse swine IAV populations circulating in large swine herds in this country (<a href="#R1"><i>1</i></a>). Long-range animal movements between Europe, Mexico, and the United States established an exceptionally diverse population of swine IAV in the west and central-north regions of Mexico, including EAswH1N1 reassortants, which have now disseminated onward into southeast Mexico. Recognition of the genetic diversity of swine IAV in Mexico, including lineages not found in US herds, has stimulated interest from producers in Mexico in developing new vaccines customized to protect against the virus strains in this country, including huH3N2 viruses in southeast Mexico that were introduced from humans in the 1990s and are highly divergent from all known US swine virus strains. The huH3N2 viruses have recently reassorted with pH1N1 viruses, generating novel reassortant genotypes that were identified frequently in southeast Mexico. Eighty percent of H3N2 viruses in swine in southeast Mexico had the HA from the huH3N2 lineage. Our analysis indicates that the huH3N2 viruses have been circulating in swine for many decades, consistent with being fit, well-adapted viruses in pigs. However, the breadth of our sampling is insufficient to draw strong conclusions about whether these viruses are widespread in southeast Mexico, and further surveillance is critical.</p> <p>Producers invest considerable time and resources into efforts to control swine IAV through vaccination, particularly of sows (<a href="#R28"><i>28</i></a>). Influenza vaccines are licensed for swine in Europe and North America, including multivalent formulations targeting the genetically diverse virus populations found in these regions (<a href="#R29"><i>29</i></a>,<a href="#R30"><i>30</i></a>). Mexico uses some vaccines made in North America, but relies heavily on customized (autogenous) vaccines designed from strains isolated in the field. US producers also frequently use autogenous vaccines because commercial vaccine formulations cannot keep up with the rapid emergence of antigenically novel swine IAV lineages. The H3N2 antigenic maps demonstrate that swine IAV strains detected in most parts of Mexico were antigenically more similar to older US cluster IV strains that circulated during 2004–2011 (<a href="#R31"><i>31</i></a>). Two swine IAV strains from the southeast region of Mexico retained more cross-reactivity with the Wuhan/95 human seasonal H3N2 vaccine strain, compared with other human virus strains and US cluster IV swine viruses, supporting the phylogenetic inference of a separate human-to-swine introduction in the 1990s in Mexico. Although these 2 strains also demonstrated antigenic relatedness to US cluster I swine virus TX/98, phylogenetic analysis indicates that the introduction into Mexico was separate from that into the United States, albeit from an antigenically similar human seasonal H3N2 ancestor. With such long branch lengths on the phylogenetic tree, we cannot exclude the possibility that human-to-swine transmission occurred in a different country where surveillance of swine is low and where movements of pigs into Mexico could have introduced the virus (e.g., Guatemala), but at this time there is no evidence for such a scenario.</p> <p>Increased surveillance in swine in the years after the 2009 H1N1 pandemic (<a href="#R32"><i>32</i></a>) has enhanced our understanding of swine IAV evolution. Whereas movements of humans and wild birds tend to follow symmetric routes of workflows (<a href="#R33"><i>33</i></a>), flight patterns (<a href="#R22"><i>22</i></a>), and flyways (<a href="#R34"><i>34</i></a>), the movements of pigs are often asymmetric, inconsistent with standard gravity models, and driven by fluctuating economic considerations. We demonstrated how genetic diversity that evolved in regions of central Mexico has spread &amp;gt;1,000 miles to southeast Mexico and introduced new lineages, including those of Eurasian origin. Introduction of genotype 8 viruses into southeast Mexico is notable, given their role in the genesis of the 2009 H1N1 pandemic in humans and their complex evolutionary history involving multiple intercontinental migration events and interspecies transmission events (<a href="#F9">Figure 9</a>). Time-scaled phylogenies consistently indicate that the west region of Mexico is the original source of the relatively recent (≈2011–2013) incursions of reassortant viruses into the southeast region. Removal of movement restrictions in 2009 might have facilitated virus flow between these regions because the final 3 states to eradicate CSFV in 2009 (Chiapas, Oaxaca, and Tabasco) are geographically situated between the southeast and west/central-north regions of Mexico (<a href="#F2">Figure 2</a>). However, as is the case in most countries, there are no reliable data on pig movements within Mexico, and our phylogeographic methods cannot exclude the possibility of virus movement through unsampled intermediary locations.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>Evolutionary history of genotype 8 influenza viruses in southeast Mexico, showing interspecies transmission events, reassortment events, and virus migration events leading to the emergence of genotype 8 viruses in swine in Mexico in the W and CN regions and continued spread into SE Mexico in ≈2011. Within each pig, vertical lines represent the 8 segments of the influenza A virus genome, ordered longest to shortest, which are shaded according to virus lineage. Arrows represent direct transmission events between hosts. Reassortment events are indicated by multiple lines shaded in different colors. Segments donated from each lineage are indicated. Birds are depicted to reflect uncertainty about the specific avian species that transmitted influenza A viruses to swine in Europe and North America. CN, central; E, east; HA, hemagglutinin; MP, matrix protein; NA, neuraminidase, NP, nucleoprotein; NS, nonstructural; PA, polymerase acidic; PB, polymerase basic; SE, southeast; W, west.</p>   </div> <p>Trade volumes of live swine from the United States into Mexico can fluctuate by orders of magnitude (<a href="#F1">Figure 1</a>, panel B), driven by economic factors and disease-control efforts, such as during the outbreak of porcine reproductive and respiratory syndrome virus in the United States (<a href="#R35"><i>35</i></a>). However, it is difficult to explore temporal links with international virus movements given that no swine IAV sequence data for Mexico are available before 2009 (<a href="#R36"><i>36</i></a>). The increase in swine IAV surveillance in Mexico since 2009 has been critical for evaluating potential vaccine effectiveness and catalyzing development of new vaccine formulations. Earlier detection of newly emergent strains and better predictions about whether new strains will increase in prevalence will require deeper, population-based surveillance of IAV in swine. Given limited public funding available for surveillance, advances in vaccine effectiveness will require a culture of greater transparency among producers, and guarantees that data shared will not be used to target individual producers or countries.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title" /> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Appendix</div> <p>Additional information on human-origin influenza A(H3N2) reassortant viruses in swine, southeast Mexico.</p>  <div class="media" title="media"><a href="18-0779-Techapp-s1.pdf">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-citation" title="citation"> <p><i>Suggested citation for this article</i>: Nelson MI, Souza C, Trovão NS, Diaz A, Mena I, Rovira A, et al. Human-origin influenza A(H3N2) reassortant viruses in swine, southeast Mexico. Emerg Infect Dis. 2019 Apr [<i>date cited</i>]. <a href="https://doi.org/10.3201/eid2504.180779">https://doi.org/10.3201/eid2504.180779</a></p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">1</span><p>Current affiliation: Kansas State University, Manhattan, Kansas, USA.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgment</div> <p>We thank all swine producers who contributed viruses for this study.</p> <p>Whole-genome sequencing for this study was generously sponsored by Zoetis Latin America through an agreement with the University of Minnesota (SOW 140-N-3545181). Evolutionary and antigenic analyses were supported by the Center for Research on Influenza Pathogenesis, a National Institute of Allergy and Infectious Diseases–funded Center of Excellence in Influenza Research and Pathogenesis (contract #HHSN272201400008C). This study was supported in part by an in-house collaborative research network for the study of influenza virus epidemiology and evolution (MISMS) led by the Fogarty International Center, National Institutes of Health.</p> </div> <div class="bio" tagx="bio" title="bio"> <p>Dr. Nelson is a staff scientist in the Division of International Epidemiology and Population Studies, Fogarty International Center, National Institutes of Health, Bethesda, MD. Her primary research interest is the evolutionary dynamics of rapidly evolving RNA viruses at the human–animal interface.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="R1" /><span class="label" tagx="label" title="label">1. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mena</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Quezada-Monroy</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dutta</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cortes-Fernández</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lara-Puente</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Origins of the 2009 H1N1 influenza pandemic in swine in Mexico.</span><span class="source" tagx="source" title="source">eLife</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e16777</span>. <span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.16777">10.7554/eLife.16777</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27350259">27350259</a></span></span></li> <li tag="ref"><a name="R2" /><span class="label" tagx="label" title="label">2. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Koen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">A practical method for field diagnosis of swine diseases.</span><span class="source" tagx="source" title="source">Am J Vet Med</span>. <span class="year" tagx="year" title="year">1919</span>;<span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">468</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>.</span></li> <li tag="ref"><a name="R3" /><span class="label" tagx="label" title="label">3. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Macken</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scheuermann</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Van Reeth</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">IH</span></span>, <span class="etal" title="etal"><i>et al.</i></span> A phylogeny-based global nomenclature system and automated annotation tool for H1 hemagglutinin genes from swine influenza viruses. mSphere. <span class="year" tagx="year" title="year">2016</span>;1:pii: e00275–16. </span></li> <li tag="ref"><a name="R4" /><span class="label" tagx="label" title="label">4. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">LLM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Long-term evolution and transmission dynamics of swine influenza A virus.</span><span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">473</span>:<span class="fpage" tagx="fpage" title="fpage">519</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature10004">10.1038/nature10004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21614079">21614079</a></span></span></li> <li tag="ref"><a name="R5" /><span class="label" tagx="label" title="label">5. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Watson</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Langat</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Reid</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cotten</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span>; <span class="collab'">collab: ESNIP3 Consortium</span>. <span class="mixed-article-title" title="mixed-article-title">ESNIP3 Consortium. Molecular epidemiology and evolution of influenza viruses circulating within European swine between 2009 and 2013.</span><span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">89</span>:<span class="fpage" tagx="fpage" title="fpage">9920</span>–<span class="lpage" tagx="lpage" title="lpage">31</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00840-15">10.1128/JVI.00840-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26202246">26202246</a></span></span></li> <li tag="ref"><a name="R6" /><span class="label" tagx="label" title="label">6. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Reassortment events among swine influenza A viruses in China: implications for the origin of the 2009 influenza pandemic.</span><span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">85</span>:<span class="fpage" tagx="fpage" title="fpage">10279</span>–<span class="lpage" tagx="lpage" title="lpage">85</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.05262-11">10.1128/JVI.05262-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21795347">21795347</a></span></span></li> <li tag="ref"><a name="R7" /><span class="label" tagx="label" title="label">7. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Culhane</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Detmer</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Global migration of influenza A viruses in swine.</span><span class="source" tagx="source" title="source">Nat Commun</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">6696</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms7696">10.1038/ncomms7696</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25813399">25813399</a></span></span></li> <li tag="ref"><a name="R8" /><span class="label" tagx="label" title="label">8. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Takemae</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parchariyanon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Damrongwatanapokin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Uchida</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ruttanapumma</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genetic diversity of swine influenza viruses isolated from pigs during 2000 to 2005 in Thailand.</span><span class="source" tagx="source" title="source">Influenza Other Respi Viruses</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">181</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1750-2659.2008.00062.x">10.1111/j.1750-2659.2008.00062.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19453423">19453423</a></span></span></li> <li tag="ref"><a name="R9" /><span class="label" tagx="label" title="label">9. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Detmer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Spatial dynamics of human-origin H1 influenza A virus in North American swine.</span><span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">e1002077</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002077">10.1371/journal.ppat.1002077</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21695237">21695237</a></span></span></li> <li tag="ref"><a name="R10" /><span class="label" tagx="label" title="label">10. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vargas Terán</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Calcagno Ferrat</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lubroth</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Situation of classical swine fever and the epidemiologic and ecologic aspects affecting its distribution in the American continent.</span><span class="source" tagx="source" title="source">Ann N Y Acad Sci</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">1026</span>:<span class="fpage" tagx="fpage" title="fpage">54</span>–<span class="lpage" tagx="lpage" title="lpage">64</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1196/annals.1307.007">10.1196/annals.1307.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15604470">15604470</a></span></span></li> <li tag="ref"><a name="R11" /><span class="label" tagx="label" title="label">11. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Slomka</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Densham</span><span class="given-names" tagx="given-names" title="given-names">ALE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Coward</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Essen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brookes</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Irvine</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Real time reverse transcription (RRT)-polymerase chain reaction (PCR) methods for detection of pandemic (H1N1) 2009 influenza virus and European swine influenza A virus infections in pigs.</span><span class="source" tagx="source" title="source">Influenza Other Respi Viruses</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">277</span>–<span class="lpage" tagx="lpage" title="lpage">93</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1750-2659.2010.00149.x">10.1111/j.1750-2659.2010.00149.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20716157">20716157</a></span></span></li> <li tag="ref"><a name="R12" /><span class="label" tagx="label" title="label">12. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gauger</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Isolation of swine influenza virus in cell cultures and embryonated chicken eggs.</span><span class="source" tagx="source" title="source">Methods Mol Biol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">1161</span>:<span class="fpage" tagx="fpage" title="fpage">265</span>–<span class="lpage" tagx="lpage" title="lpage">76</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4939-0758-8_22">10.1007/978-1-4939-0758-8_22</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24899436">24899436</a></span></span></li> <li tag="ref"><a name="R13" /><span class="label" tagx="label" title="label">13. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Harmon</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>. <span class="mixed-article-title" title="mixed-article-title">RNA extraction from swine samples and detection of influenza A virus in swine by real-time RT-PCR.</span><span class="source" tagx="source" title="source">Methods Mol Biol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">1161</span>:<span class="fpage" tagx="fpage" title="fpage">277</span>–<span class="lpage" tagx="lpage" title="lpage">93</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4939-0758-8_23">10.1007/978-1-4939-0758-8_23</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24899437">24899437</a></span></span></li> <li tag="ref"><a name="R14" /><span class="label" tagx="label" title="label">14. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Culhane</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rovira</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Torremorell</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Guerrero</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Norambuena</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Novel human-like influenza A viruses circulate in swine in Mexico and Chile.</span><span class="source" tagx="source" title="source">PLoS Curr</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26345598">26345598</a></span></span></li> <li tag="ref"><a name="R15" /><span class="label" tagx="label" title="label">15. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stech</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Perez</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Universal primer set for the full-length amplification of all influenza A viruses.</span><span class="source" tagx="source" title="source">Arch Virol</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">146</span>:<span class="fpage" tagx="fpage" title="fpage">2275</span>–<span class="lpage" tagx="lpage" title="lpage">89</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s007050170002">10.1007/s007050170002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11811679">11811679</a></span></span></li> <li tag="ref"><a name="R16" /><span class="label" tagx="label" title="label">16. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>. <span class="mixed-article-title" title="mixed-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">32</span>:<span class="fpage" tagx="fpage" title="fpage">1792</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="R17" /><span class="label" tagx="label" title="label">17. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bolotov</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dernovoy</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kiryutin</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaslavsky</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tatusova</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The influenza virus resource at the National Center for Biotechnology Information.</span><span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">82</span>:<span class="fpage" tagx="fpage" title="fpage">596</span>–<span class="lpage" tagx="lpage" title="lpage">601</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02005-07">10.1128/JVI.02005-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17942553">17942553</a></span></span></li> <li tag="ref"><a name="R18" /><span class="label" tagx="label" title="label">18. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.</span><span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">2688</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btl446">10.1093/bioinformatics/btl446</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a></span></span></li> <li tag="ref"><a name="R19" /><span class="label" tagx="label" title="label">19. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7.</span><span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">29</span>:<span class="fpage" tagx="fpage" title="fpage">1969</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mss075">10.1093/molbev/mss075</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">22367748</a></span></span></li> <li tag="ref"><a name="R20" /><span class="label" tagx="label" title="label">20. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Many-core algorithms for statistical phylogenetics.</span><span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">1370</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp244">10.1093/bioinformatics/btp244</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19369496">19369496</a></span></span></li> <li tag="ref"><a name="R21" /><span class="label" tagx="label" title="label">21. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Minin</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Counting labeled transitions in continuous-time Markov models of evolution.</span><span class="source" tagx="source" title="source">J Math Biol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">56</span>:<span class="fpage" tagx="fpage" title="fpage">391</span>–<span class="lpage" tagx="lpage" title="lpage">412</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00285-007-0120-8">10.1007/s00285-007-0120-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17874105">17874105</a></span></span></li> <li tag="ref"><a name="R22" /><span class="label" tagx="label" title="label">22. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bielejec</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baele</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Unifying viral genetics and human transportation data to predict the global transmission dynamics of human influenza H3N2.</span><span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">e1003932</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003932">10.1371/journal.ppat.1003932</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24586153">24586153</a></span></span></li> <li tag="ref"><a name="R23" /><span class="label" tagx="label" title="label">23. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: R Core Team</span>. R: a language and environment for statistical computing. Vienna: R Foundation for Statistical Computing; <span class="year" tagx="year" title="year">2012</span> [cited 2018 Dec 19]. <a href="http://www.r-project.org/">http://www.r-project.org/</a>.</span></li> <li tag="ref"><a name="R24" /><span class="label" tagx="label" title="label">24. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kitikoon</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Substitutions near the hemagglutinin receptor-binding site determine the antigenic evolution of influenza A H3N2 viruses in U.S. swine.</span><span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>:<span class="fpage" tagx="fpage" title="fpage">4752</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03805-13">10.1128/JVI.03805-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24522915">24522915</a></span></span></li> <li tag="ref"><a name="R25" /><span class="label" tagx="label" title="label">25. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kitikoon</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gauger</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Hemagglutinin inhibition assay with swine sera.</span><span class="source" tagx="source" title="source">Methods Mol Biol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">1161</span>:<span class="fpage" tagx="fpage" title="fpage">295</span>–<span class="lpage" tagx="lpage" title="lpage">301</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4939-0758-8_24">10.1007/978-1-4939-0758-8_24</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24899438">24899438</a></span></span></li> <li tag="ref"><a name="R26" /><span class="label" tagx="label" title="label">26. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Culhane</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">LaPointe</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Introductions and evolution of human-origin seasonal influenza a viruses in multinational swine populations.</span><span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>:<span class="fpage" tagx="fpage" title="fpage">10110</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01080-14">10.1128/JVI.01080-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24965467">24965467</a></span></span></li> <li tag="ref"><a name="R27" /><span class="label" tagx="label" title="label">27. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Abente</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Santos</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gauger</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stratton</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The molecular determinants of antibody recognition and antigenic drift in the H3 hemagglutinin of swine influenza A virus.</span><span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">90</span>:<span class="fpage" tagx="fpage" title="fpage">8266</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01002-16">10.1128/JVI.01002-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27384658">27384658</a></span></span></li> <li tag="ref"><a name="R28" /><span class="label" tagx="label" title="label">28. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Beaudoin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Davies</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bender</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gramer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Characterization of influenza a outbreaks in Minnesota swine herds and measures taken to reduce the risk of zoonotic transmission.</span><span class="source" tagx="source" title="source">Zoonoses Public Health</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">59</span>:<span class="fpage" tagx="fpage" title="fpage">96</span>–<span class="lpage" tagx="lpage" title="lpage">106</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1863-2378.2011.01423.x">10.1111/j.1863-2378.2011.01423.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21824375">21824375</a></span></span></li> <li tag="ref"><a name="R29" /><span class="label" tagx="label" title="label">29. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sandbulte</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Spickler</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaabel</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Roth</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Optimal use of vaccines for control of influenza A virus in swine.</span><span class="source" tagx="source" title="source">Vaccines (Basel)</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">22</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/vaccines3010022">10.3390/vaccines3010022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26344946">26344946</a></span></span></li> <li tag="ref"><a name="R30" /><span class="label" tagx="label" title="label">30. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Perez</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rajao</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Abente</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Walia</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Influenza A virus vaccines for swine.</span><span class="source" tagx="source" title="source">Vet Microbiol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">206</span>:<span class="fpage" tagx="fpage" title="fpage">35</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vetmic.2016.11.026">10.1016/j.vetmic.2016.11.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27923501">27923501</a></span></span></li> <li tag="ref"><a name="R31" /><span class="label" tagx="label" title="label">31. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kitikoon</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Substitutions near the hemagglutinin receptor-binding site determine the antigenic evolution of influenza A H3N2 viruses in U.S. swine.</span><span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>:<span class="fpage" tagx="fpage" title="fpage">4752</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03805-13">10.1128/JVI.03805-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24522915">24522915</a></span></span></li> <li tag="ref"><a name="R32" /><span class="label" tagx="label" title="label">32. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Awada</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Claes</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dauphin</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Review of influenza A virus in swine worldwide: a call for increased surveillance and research.</span><span class="source" tagx="source" title="source">Zoonoses Public Health</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">61</span>:<span class="fpage" tagx="fpage" title="fpage">4</span>–<span class="lpage" tagx="lpage" title="lpage">17</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/zph.12049">10.1111/zph.12049</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23556412">23556412</a></span></span></li> <li tag="ref"><a name="R33" /><span class="label" tagx="label" title="label">33. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bjørnstad</span><span class="given-names" tagx="given-names" title="given-names">ON</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Synchrony, waves, and spatial hierarchies in the spread of influenza.</span><span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">312</span>:<span class="fpage" tagx="fpage" title="fpage">447</span>–<span class="lpage" tagx="lpage" title="lpage">51</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1125237">10.1126/science.1125237</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16574822">16574822</a></span></span></li> <li tag="ref"><a name="R34" /><span class="label" tagx="label" title="label">34. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT-Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ip</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Halpin</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stockwell</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Migratory flyway and geographical distance are barriers to the gene flow of influenza virus among North American birds.</span><span class="source" tagx="source" title="source">Ecol Lett</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">24</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2011.01703.x">10.1111/j.1461-0248.2011.01703.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22008513">22008513</a></span></span></li> <li tag="ref"><a name="R35" /><span class="label" tagx="label" title="label">35. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Petry</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Paarlberg</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span>. <span class="mixed-article-title" title="mixed-article-title">PRRS and the North American swine trade: a trade barrier analysis.</span><span class="source" tagx="source" title="source">Journal of Agricultural and Applied Economics.</span><span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">33</span>:<span class="fpage" tagx="fpage" title="fpage">425</span>–<span class="lpage" tagx="lpage" title="lpage">36</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1017/S1074070800008749">10.1017/S1074070800008749</a></span></span></li> <li tag="ref"><a name="R36" /><span class="label" tagx="label" title="label">36. </span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Escalera-Zamudio</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cobián-Güemes</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">de los Dolores Soto-del Río</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Isa</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sánchez-Betancourt</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Parissi-Crivelli</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Characterization of an influenza A virus in Mexican swine that is related to the A/H1N1/2009 pandemic clade.</span><span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">433</span>:<span class="fpage" tagx="fpage" title="fpage">176</span>–<span class="lpage" tagx="lpage" title="lpage">82</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2012.08.003">10.1016/j.virol.2012.08.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22921315">22921315</a></span></span></li> </ul> </div> </div>  </body></html>