<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Int J Mol Sci</span><span class="iso-abbrev" title="iso-abbrev">Int J Mol Sci</span><span class="publisher-id" title="publisher-id">ijms</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">International Journal of Molecular Sciences</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1422-0067</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7216121">7216121</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/ijms21082746">10.3390/ijms21082746</a></span><span class="publisher-id" title="publisher-id">ijms-21-02746</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Knockout of <i>ACTB</i> and <i>ACTG1</i> with CRISPR/Cas9(D10A) Technique Shows that Non-Muscle β and γ Actin Are Not Equal in Relation to Human Melanoma Cells’ Motility and Focal Adhesion Formation</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malek</span><span class="given-names" tagx="given-names" title="given-names">Natalia</span></span><a href="#fn1-ijms-21-02746">†</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1204-6108</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mrówczyńska</span><span class="given-names" tagx="given-names" title="given-names">Ewa</span></span><a href="#fn1-ijms-21-02746">†</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michrowska</span><span class="given-names" tagx="given-names" title="given-names">Aleksandra</span></span></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5418-5712</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazurkiewicz</span><span class="given-names" tagx="given-names" title="given-names">Ewa</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavlyk</span><span class="given-names" tagx="given-names" title="given-names">Iuliia</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">Antonina Joanna</span></span><a href="#c1-ijms-21-02746">*</a></span></div><span class="citation_author_institution" id="af1-ijms-21-02746">[], <span class="email" tagx="email" title="email">en.malek@gmail.com</span><span class="email" tagx="email" title="email">ewa.mrowczynska@uwr.edu.pl</span><span class="email" tagx="email" title="email">282709@uwr.edu.pl</span><span class="email" tagx="email" title="email">ewa.mazurkiewicz@uwr.edu.pl</span><span class="email" tagx="email" title="email">pavlykj@gmail.com</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">antonina.mazur@uwr.edu.pl</span>; Tel.: +48-713-756-206</div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">†</span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-4</span></span><span class="volume" tagx="volume" title="volume">21</span><span class="issue" tagx="issue" title="issue">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2746</span><span class="history" title="history"><span class="received" title="received">received: 2020-2-14</span><span class="accepted" title="accepted">accepted: 2020-4-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Non-muscle actins have been studied for many decades; however, the reason for the existence of both isoforms is still unclear. Here we show, for the first time, a successful inactivation of the <i>ACTB</i> (CRISPR clones with inactivated <i>ACTB</i>, CR-<i>ACTB</i>) and <i>ACTG1</i> (CRISPR clones with inactivated <i>ACTG1</i>, CR-<i>ACTG1</i>) genes in human melanoma cells (A375) via the RNA-guided D10A mutated Cas9 nuclease gene editing [CRISPR/Cas9(D10A)] technique. This approach allowed us to evaluate how melanoma cell motility was impacted by the lack of either β actin coded by <i>ACTB</i> or γ actin coded by <i>ACTG1</i>. First, we observed different distributions of β and γ actin in the cells, and the absence of one actin isoform was compensated for via increased expression of the other isoform. Moreover, we noted that γ actin knockout had more severe consequences on cell migration and invasion than β actin knockout. Next, we observed that the formation rate of bundled stress fibers in CR-<i>ACTG1</i> cells was increased, but lamellipodial activity in these cells was impaired, compared to controls. Finally, we discovered that the formation rate of focal adhesions (FAs) and, subsequently, FA-dependent signaling were altered in both the CR-<i>ACTB</i> and CR-<i>ACTG1</i> clones; however, a more detrimental effect was observed for γ actin-deficient cells. Our research shows that both non-muscle actins play distinctive roles in melanoma cells’ FA formation and motility.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
beta actin</div> <div class="kwd" title="kwd">
gamma actin</div> <div class="kwd" title="kwd">
actin isoforms</div> <div class="kwd" title="kwd">
melanoma</div> <div class="kwd" title="kwd">
CRISPR/Cas9(D10A) technique</div> <div class="kwd" title="kwd">
focal adhesion</div> <div class="kwd" title="kwd">
adhesion</div> <div class="kwd" title="kwd">
migration</div> <div class="kwd" title="kwd">
LPA</div> <div class="kwd" title="kwd">
PMA</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>The actin cytoskeleton is essential for the proper functioning of many cellular processes, including maintenance of the cell shape, chemotaxis, cell movement, adhesion, transport of cellular organelles, mitosis, replication, transcription, and even DNA repair [<a href="#B1-ijms-21-02746">1</a>,<a href="#B2-ijms-21-02746">2</a>]. Such a wide array of functions must be coordinated by several signaling pathways, which often intersect with each other, and a number of proteins. There are more than 200 actin-binding proteins (ABPs), which are responsible for the nucleation of actin polymerization as well as branching, crosslinking, and stabilization, but also severing of polymerized actin (filamentous actin, F-actin). Some ABPs, due to their activity in actin monomer sequestration, contribute to the maintenance of the cell’s monomeric actin pool (globular actin, G-actin) [<a href="#B3-ijms-21-02746">3</a>,<a href="#B4-ijms-21-02746">4</a>,<a href="#B5-ijms-21-02746">5</a>].</p> <p>In a cell, there are linear and branched actin filament networks. The spatial organization of a particular actin network and the type of ABPs associated with that network determine the type of F-actin-based subcellular structure. The lamellipodium, which is involved in cell movement, is a dynamic structure that is composed of a dense network of short and branched actin filaments. Stress fibers, which consist of about 10–30 microfilaments aligned in parallel, are partly responsible for cell contraction and adhesion [<a href="#B3-ijms-21-02746">3</a>]. Other structures built by F-actin in tumor cells are invadopodia, which, due to the secretion of metalloproteases, are responsible for proteolytic degradation of the extracellular matrix and, thus, participate in the adhesion and migration/invasion of neoplastic cells [<a href="#B6-ijms-21-02746">6</a>].</p> <p>In higher vertebrates, including mammals, there are four muscle isoforms and two non-muscle isoforms of actin [<a href="#B7-ijms-21-02746">7</a>,<a href="#B8-ijms-21-02746">8</a>,<a href="#B9-ijms-21-02746">9</a>,<a href="#B10-ijms-21-02746">10</a>], and although actins have been intensively studied for several decades, there are still controversies about the functional diversification of non-muscle isoactins (β and γ actin), which differ by only four amino acids within their N-termini [<a href="#B7-ijms-21-02746">7</a>,<a href="#B8-ijms-21-02746">8</a>]. Their different properties are explained, on the one hand, by protein features that manifest through dissimilar polymerization dynamics [<a href="#B11-ijms-21-02746">11</a>], divergent interactions with ABPs [<a href="#B7-ijms-21-02746">7</a>,<a href="#B12-ijms-21-02746">12</a>], and discrepancies in posttranslational modifications [<a href="#B13-ijms-21-02746">13</a>]. On the other hand, it is claimed that it is not the isoactins’ amino acid sequence but instead their nucleotide sequence that is more relevant for these differences [<a href="#B14-ijms-21-02746">14</a>], which are mirrored in mRNA trafficking [<a href="#B15-ijms-21-02746">15</a>] and the speed of isoactin biosynthesis [<a href="#B16-ijms-21-02746">16</a>]. However, the existence of two extremely similar actin isoforms has to be of high relevance since their expression patterns differ in various tissues (summarized in [<a href="#B7-ijms-21-02746">7</a>,<a href="#B8-ijms-21-02746">8</a>,<a href="#B10-ijms-21-02746">10</a>]), although their protein levels are similar in most cell types. Supporting this notion, mutations in genes encoding β and γ actins give different outcomes in, e.g., patients with Baraitser–Winter cerebrofrontofacial syndrome (BWCFF), where the cases with <i>ACTB</i> mutations are associated with severe forms of this condition, whereas cases with mutations in <i>ACTG1</i> are restricted to brain malformations [<a href="#B17-ijms-21-02746">17</a>,<a href="#B18-ijms-21-02746">18</a>]. </p> <p>Because there is no consensus regarding the purpose of the existence of two non-muscle actins, it is of high importance to study the functions of β and γ actin. Several published studies aimed to elucidate the roles of non-muscle actins by overexpression, silencing, or knockout of the coding sequences for β or γ actin (reviewed by [<a href="#B7-ijms-21-02746">7</a>]). However, in the overexpression and silencing studies, endogenous isoactins were present in the tested cells, which could have affected the results, whereas the knockout studies were conducted using normal cells, e.g., murine fibroblasts. In our studies, we focused on melanoma cells since this very aggressive tumor is characterized by its high plasticity [<a href="#B19-ijms-21-02746">19</a>] and the ability to form tumors from a single transformed cell [<a href="#B20-ijms-21-02746">20</a>]. This makes melanoma a perfect model to investigate the role of the aforementioned actin isoforms in migration. Moreover, as far as we know, there is no previous research that aimed to simultaneously estimate the role of the β and γ actins in melanoma cell migration. Here, by inactivation the genes encoding the β (<i>ACTB</i>) and γ (<i>ACTG1</i>) actins with the help of the RNA-guided D10A mutated Cas9 nuclease gene editing, CRISPR/Cas9(D10A), system in A375 cells, we report, for the first time, that the lack of γ actin had the most severe consequences on A375 cells and these non-muscle actins are unequal in their features, particularly their influence on the formation of focal adhesions (FAs) and protrusive structures and, thus, the motility of melanoma cells. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results</div> <div class="2.1.bothnon-muscleactinsarehighlyabundantina375cellsbutoccupydifferentareasofthecell" title="sec"> <div class="title" tagx="title" title="title">
2.1. Both Non-Muscle Actins Are Highly Abundant in A375 Cells but Occupy Different Areas of the Cell</div> <p>Prior to subjecting the cells to the CRISPR/Cas9(D10A) procedure, we examined the expression level of both non-muscle actins in melanoma cells. In all the melanoma cell lines we tested, we noted various levels of the transcripts that encode both of the non-muscle isoactins (<a href="#app1-ijms-21-02746">Figure S1</a>). Upon normalization of the results to A375 cells there were statistically significant differences in <i>ACTB</i> and <i>ACTG1</i> expression for other melanoma cells lines when compared to A375 cells, however the differences were not so high varying form ca. 0.5- to 2-fold. Nevertheless, given the strict regulation of expression of the actin-encoding genes, even about twice as high <i>ACTB</i> expression (WM1341D cells) or twice as low <i>ACTG1</i> (WM9 cells) may have important implications for cell biology. For this study, we chose the A375 cell line as it is a cell line that was isolated from primary melanoma localized within skin with the potential to form metastases and it is easy to transfect. First, we looked at the distribution of β and γ actin in A375 cells. The specificity of the antibodies used in this study to detect both isoactins was checked using recombinant β and γ actin (<a href="#app1-ijms-21-02746">Figure S2</a>). Two types of antibodies against β actin and another two against γ actin were proven to be specific, as declared by the manufacturers. We noted that β actin in A375 was incorporated in F-actin structures, such as stress fibers and invadopodia, whereas γ actin, apart from being present in these structures as well, was also very prominently present as the dense actin mesh, both in the perinuclear area and within lamellipodia (<a href="#ijms-21-02746-f001">Figure 1</a>A). This was particularly easy to observe with double staining, where G-actin was detected together with β or γ actin (<a href="#ijms-21-02746-f001">Figure 1</a>B). Both actins seemed to be expressed at similar levels in these cells, as revealed by 2-D PAGE (<a href="#ijms-21-02746-f001">Figure 1</a>C). Simultaneous detection of both non-muscle actins in A375 cells revealed that the distribution of fluorescently labeled actins differed: β actin was more frequently located within actin bundles and γ actin at the periphery of the cell (<a href="#ijms-21-02746-f001">Figure 1</a>D). Histograms of the fluorescence signals that represent β and γ actin, conducted from the middle of the cell towards its border, clearly show that the fluorescence signal coming from the detection of β actin did not overlap with the γ actin fluorescence signal along the whole measured distance of the cell. However, intriguingly, a colocalization between these two non-muscle actins was observed at the cell cortex (<a href="#ijms-21-02746-f001">Figure 1</a>D). These observations indicate that both β and γ actin have partially different localizations in a cell. </p> </div> <div class="2.2.successfulknockoutofgenescodingforβandγactinleadstoalterednon-muscleisoactinexpressionandaffectscells’migration" title="sec"> <div class="title" tagx="title" title="title">
2.2. Successful Knockout of Genes Coding for β and γ Actin Leads to Altered Non-Muscle Isoactin Expression and Affects Cells’ Migration</div> <p>In order to test the hypothesis that β and γ actin have different functions in A375 cells, we decided to knock out the <i>ACTB</i> and <i>ACTG1</i> genes via the CRISPR/Cas9(D10A) technique. We successfully obtained these lines, which we confirmed by genomic DNA (gDNA) (<a href="#app1-ijms-21-02746">Figure S3A</a>), Western blot (<a href="#ijms-21-02746-f002">Figure 2</a>A,B and <a href="#app1-ijms-21-02746">Figure S3B</a>), and qPCR (<a href="#ijms-21-02746-f002">Figure 2</a>C) analyses and by immunocytochemical staining of the obtained clones (<a href="#ijms-21-02746-f002">Figure 2</a>D and <a href="#app1-ijms-21-02746">Figure S4</a>). Apart from cells devoid of β and γ actin synthesis, we also obtained control cells, i.e., clones transfected with CRISPR/Cas9(D10A) plasmids that code for scrambled guide RNAs, gRNAs. The control clones had normal levels of both of the non-muscle actins (<a href="#app1-ijms-21-02746">Figure S3C</a>). Interestingly, we observed compensation after the depletion of one isoform, i.e., upregulation of the level of the other isoform. This was seen in both the Western blot (<a href="#ijms-21-02746-f002">Figure 2</a>A,B) and microscopic analyses (<a href="#ijms-21-02746-f002">Figure 2</a>D,E, <a href="#app1-ijms-21-02746">Figure S4</a>). In the Western blot analysis, we observed this occurrence only if we lysed the cells in the lysis buffer used for 2-D PAGE, which contains high levels of urea and thiourea and, thus, better extracts the proteins associated with the detergent-resistant membrane fractions of the cells. When we used cytoskeletal-bound protein extraction buffer, we did not observe any compensation (<a href="#app1-ijms-21-02746">Figure S5</a>). For all the other Western blot analyses, we used cytoskeletal-bound protein extraction buffer. A closer look at the stained cells revealed (<a href="#ijms-21-02746-f002">Figure 2</a>F) that in the case of CR-<i>ACTB</i> [clones with inactivated <i>ACTB</i> with the help of CRISPR/Cas9(D10A) technique] cells, the elevated levels of γ actin manifested more prominently in structures that are characteristic for this isoform, as shown in <a href="#ijms-21-02746-f001">Figure 1</a>A,B. A similar observation was made for the β actin distribution in A375 CR-<i>ACTG1</i> [clones with inactivated <i>ACTG1</i> with the help of CRISPR/Cas9(D10A) technique] cells. Here, the stress fibers’ formation was more pronounced than in the control cells (<a href="#ijms-21-02746-f002">Figure 2</a>F); this was reflected in the elevated filamentous (F) to globular (G) actin ratio (<a href="#ijms-21-02746-f002">Figure 2</a>G) and more pronounced staining for F-actin (<a href="#ijms-21-02746-f002">Figure 2</a>H) in CR-<i>ACTG1</i> cells. In the case of the cells without β actin, the F:G actin ratio was unchanged in comparison to the control cells (<a href="#ijms-21-02746-f002">Figure 2</a>G) and the F-actin staining patterns were similar between the CR-<i>ACTB</i> clones and control cells (<a href="#ijms-21-02746-f002">Figure 2</a>H). These results show that the CRISPR/Cas9(D10A) technique is suitable for knocking out <i>ACTB</i> and <i>ACTG1</i> and that the obtained clones have altered actin cytoskeleton architecture.</p> </div> <div class="2.3.migrationandinvasionareaffectedinthecr-actbclonesandgreatlyaffectedinthecr-actg1clones" title="sec"> <div class="title" tagx="title" title="title">
2.3. Migration and Invasion Are Affected in the CR-ACTB Clones and Greatly Affected in the CR-ACTG1 Clones</div> <p>Because there were discrepancies, in terms of actin architecture, between the studied clones, we checked the influence of depletion of β or γ actin on the cells’ ability to migrate and invade. The distance covered by the CR-<i>ACTB</i> cells (<a href="#ijms-21-02746-f003">Figure 3</a>A,B) and their movement speed (<a href="#ijms-21-02746-f003">Figure 3</a>C) was similar to the results obtained for the control cells, but, surprisingly, the directionality of the β actin-devoid cells was changed (<a href="#ijms-21-02746-f003">Figure 3</a>D). In contrast, the 2-D migration of the CR-<i>ACTG1</i> clones was severely affected. The distances covered by these cells and their velocity were statistically significantly lowered in comparison to the control cells (<a href="#ijms-21-02746-f003">Figure 3</a>A–C), but their directionality was not (<a href="#ijms-21-02746-f003">Figure 3</a>D). The increased directionality of the CR-<i>ACTB</i> cells was reflected in the higher collective migration of these cells, compared to control cells (<a href="#ijms-21-02746-f003">Figure 3</a>E, <a href="#app1-ijms-21-02746">Figure S7</a>). Again, for the CR-<i>ACTG1</i> cells, we observed less migration than for the control (CR-CTRL) clones (<a href="#ijms-21-02746-f003">Figure 3</a>E, <a href="#app1-ijms-21-02746">Figure S7</a>). Next, we wanted to check the 3-D migration abilities of the studied cells and for that we used Transwell™ filters filled with Matrigel™ gel to mimic basement membrane. For the CR-<i>ACTB</i> and CR-<i>ACTG1</i> clones, we noted that fewer cells were capable of crossing the 3-D matrix towards the medium with 20% fetal bovine serum (FBS) (<a href="#ijms-21-02746-f003">Figure 3</a>F). Intriguingly, we observed that in these cells the number of invadopodia was higher than in the control cells (<a href="#ijms-21-02746-f003">Figure 3</a>G,H). This was even more drastic in the case of CR-<i>ACTG1</i> cells since the number of invadopodia was much higher than in the CR-<i>ACTB</i> clones. Additionally, we observed that the area digested by metalloproteases in the CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells was larger than that of the control cells, although the differences were not statistically significant (<a href="#ijms-21-02746-f003">Figure 3</a>I). Next, we stained the cells for cofilin and cortactin, two proteins that are important for invadopodia formation [<a href="#B21-ijms-21-02746">21</a>,<a href="#B22-ijms-21-02746">22</a>]. This staining revealed that both cofilin and cortactin are indeed present in these structures (<a href="#ijms-21-02746-f003">Figure 3</a>J); however, cofilin, unlike cortactin, is not present to the same extent in invadopodia for the CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells as it is in the control cells. This was especially evident in the fluorescence intensity plot profiles of the presence of cortactin and cofilin at invadopodia (<a href="#ijms-21-02746-f003">Figure 3</a>J). Afterwards, we checked the levels of cofilin and its phosphorylated form (p-cofilin). We noted that the levels of total cofilin were highly increased in the CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells, when compared to control clones (<a href="#ijms-21-02746-f003">Figure 3</a>K, <a href="#app1-ijms-21-02746">Figure S8</a>); however, we only observed a statistically higher amount of p-cofilin<sup>3</sup> in the CR-<i>ACTB</i> cells, compared to the control clones (<a href="#ijms-21-02746-f003">Figure 3</a>K, <a href="#app1-ijms-21-02746">Figure S8</a>). We estimated the p-cofilin<sup>3</sup>:cofilin ratios for all the cell types and noted that the ratio statistically significantly increased only for CR-<i>ACTB</i> cells, although we observed a similar tendency in the CR-<i>ACTG1</i> cells (<a href="#ijms-21-02746-f003">Figure 3</a>L). In summary, a lack of γ actin has more severe consequences on the cells’ 2-D and 3-D migration than a lack of β actin. </p> </div> <div class="2.4.bundlingofstressfibersisalteredinclonesthatlackeitherβorγactin" title="sec"> <div class="title" tagx="title" title="title">
2.4. Bundling of Stress Fibers Is Altered in Clones that Lack Either β or γ Actin</div> <p>Subsequently, we checked the formation of F-actin-rich structures as they are directly involved in cell motility. We incubated cells that were starved for 24 h with 1 micromolar (μM) lysophosphatidic acid (LPA) for 10 min in order to challenge the cells to form stress fibers [<a href="#B23-ijms-21-02746">23</a>]. This was achieved (<a href="#ijms-21-02746-f004">Figure 4</a>A), and we calculated the number of "thick" stress fiber bundles, plotting a histogram of fluorescence over the cell length and identifying the number of fluorescent values that were higher or equal to 75% of the highest fluorescence peak. The highest number of these structures was noted in the CR-<i>ACTG1</i> clones with LPA stimulation (<a href="#ijms-21-02746-f004">Figure 4</a>B). Interestingly, in the case of control cells and CR-<i>ACTG1</i> cells, there were more thick stress fiber bundles after stimulation than in nonstimulated cells, whereas this was not true in the CR-<i>ACTB</i> cells. We then looked at activation of Rho-GTPases upon LPA stimulation, as LPA is a Rho activator [<a href="#B23-ijms-21-02746">23</a>]. After the addition of LPA, a statistically significant decrease in the level of active RhoA was observed only in the CR-<i>ACTG1</i> cells (<a href="#ijms-21-02746-f004">Figure 4</a>C). The levels of Rac1 and Cdc42 activity were similar between the clones (<a href="#ijms-21-02746-f004">Figure 4</a>C). Next, we evaluated the F:G actin ratio; this ratio was similar in all three cell types (<a href="#ijms-21-02746-f004">Figure 4</a>D), although there was a tendency toward a decrease for the CR-<i>ACTB</i> cells. Next, we looked at the distribution of the human FH1/FH2 domain-containing protein 1, FHOD1, formin (a microfilament bundling and capping protein [<a href="#B24-ijms-21-02746">24</a>]), and Myosin IIa, an actin motor protein that possesses microfilament bundling activity [<a href="#B25-ijms-21-02746">25</a>]. While we did not observe any changes in the cellular distribution of FHOD1, the Myosin IIa staining pattern was altered in CR-<i>ACTG1</i> cells (<a href="#ijms-21-02746-f004">Figure 4</a>E). The signal for Myosin IIa in these cells was not homogenously distributed, as in the control cells, but instead clear, thick bundles were seen. Although this was not reflected in the results of the analysis of averaged histograms, which represent the fluorescence signal of detected FHOD1 or Myosin IIa, it was mirrored in the plot profiles of representative cells for every condition (<a href="#ijms-21-02746-f004">Figure 4</a>F). Moreover, the FHOD1 and Myosin IIa levels were similar in the CR-<i>ACTB</i>, CR-<i>ACTG1</i>, and control cells (<a href="#ijms-21-02746-f004">Figure 4</a>G, <a href="#app1-ijms-21-02746">Figure S9</a>). We also checked the phosphorylated myosin light chain (pMLC) levels, as pMLC is an indicator of myosin ATPase activity and, thus, activation of the contractile apparatus [<a href="#B26-ijms-21-02746">26</a>]. The pMLC levels were similar among all the cell types (<a href="#ijms-21-02746-f004">Figure 4</a>G, <a href="#app1-ijms-21-02746">Figure S9</a>). Although there were no differences in the levels of Myosin IIa or pMLC between the different clones after LPA stimulation, when grown in full medium, significantly higher levels of Myosin IIa and pMLC were exhibited by the CR-<i>ACTB</i> cells and the CR-<i>ACTG1</i> clones had lower basal levels of FHOD1 (<a href="#app1-ijms-21-02746">Figure S10</a>). Finally, we stained the cells for pMLC and we noticed a strong decoration of stress fibers with pMLC in control and CR-<i>ACTG1</i> cells, but not in the CR-<i>ACTB</i> clones (<a href="#ijms-21-02746-f004">Figure 4</a>H). These observations imply that the formation rate of stress fibers is more enhanced in the cells devoid of γ actin than in control cells or cells without β actin. </p> </div> <div class="2.5.lamellipodialdynamicsarealteredincellsdevoidofeitherβorγactin" title="sec"> <div class="title" tagx="title" title="title">
2.5. Lamellipodial Dynamics Are Altered in Cells Devoid of Either β or γ Actin</div> <p>Since we checked the clones’ ability to form stress fibers, we also looked at the other F-actin protrusive structures, lamellipodia. Phorbol-12-myristate-13-acetate (PMA) stimulation was performed to induce formation of lamellipodia [<a href="#B27-ijms-21-02746">27</a>]. F-actin staining in fixed cells and analysis of micrographs revealed that cells devoid of γ actin formed narrower and thinner leading edges than did control cells (<a href="#ijms-21-02746-f005">Figure 5</a>A–C). The width and the thickness of lamellipodium were measured by plotting a histogram of fluorescence over the lamellipodium, and using a ruler in the software or reading the whole intensity of fluorescence through the lamellipodium, respectively. In the case of the CR-<i>ACTB</i> cells, there were no differences in these aspects compared to controls. Afterwards, we performed live imaging analysis of clones that were transfected with a plasmid that encoded LifeAct-TagRFP in order to track F-actin in real time upon PMA stimulation. Analysis of the kymographs (<a href="#ijms-21-02746-f005">Figure 5</a>D) showed that the dynamics of F-actin within the leading edge were more prominent in CR-<i>ACTB</i> cells than in control cells (<a href="#ijms-21-02746-f005">Figure 5</a>D–F). Although, in the CR-<i>ACTG1</i> cells, there was no difference in the lamellipodial dynamics between the PMA-stimulated and nonstimulated groups, the overall lamellipodial dynamics were lower in those cells in comparison to those present in the controls (<a href="#ijms-21-02746-f005">Figure 5</a>F). Next, we evaluated the activation of Rho-GTPases upon PMA stimulation versus nonstimulated cells. We evaluated activity of GTPases with activation assays (G-LISA<sup>®</sup>) detecting given GTPase bound with GTP in cells lysates. Increased Rac1 activity was noted in CR-<i>ACTB</i> cells (<a href="#ijms-21-02746-f005">Figure 5</a>G); however, we did not observe any changes in the RhoA or Cdc42 activity between the clones in response to PMA treatment. The F:G actin ratio was also checked and a statistically significant decrease was noted only for the CR-<i>ACTG1</i> cells (<a href="#ijms-21-02746-f005">Figure 5</a>H). Altogether, these observations imply that while the leading edge dynamics were increased in CR-<i>ACTB</i> cells, the opposite effect was observed for CR-<i>ACTG1</i> cells, when compared to control cells. </p> </div> <div class="2.6.therearedifferencesinthedistributionandlevelsofactinpolymerizationnucleatorswithintheleadingedge" title="sec"> <div class="title" tagx="title" title="title">
2.6. There Are Differences in the Distribution and Levels of Actin Polymerization Nucleators within the Leading Edge</div> <p>Because we noticed divergences between the clones in terms of their leading edge dynamics, we decided to look at proteins that trigger actin polymerization and reside in the leading edge. Upon exposure to PMA, the cells were fixed and stained for vasodilator-stimulated phosphoprotein (VASP) and actin-related protein 3 (Arp3) as these proteins are the nucleators of actin polymerization within the lamellipodium [<a href="#B28-ijms-21-02746">28</a>]. We found that both proteins were prominent at the leading edges of all the analyzed cell types; however, the presence of VASP was more pronounced in the CR-<i>ACTG1</i> clones, whereas the Arp3 level was diminished in these cells (<a href="#ijms-21-02746-f006">Figure 6</a>A,B). The presence of neural Wiskott-Aldrich syndrome protein (N-WASP) and Wiskott-Aldrich syndrome protein family member 2 (WAVE-2), two activators of Arp3, was also evaluated; while the amount of N-WASP was significantly increased at the leading edges of CR-<i>ACTB</i> cells, there were no observed differences in the amount of WAVE-2 between the tested clones (<a href="#ijms-21-02746-f006">Figure 6</a>C,D). Interestingly, we observed that VASP colocalized with Arp3 within circular dorsal ruffles (CDRs) in all the tested clones (<a href="#ijms-21-02746-f006">Figure 6</a>A). We also noticed the presence of Arp3 and WAVE-2 at the leading edges and CDRs in all the tested clones (<a href="#ijms-21-02746-f006">Figure 6</a>C). Western blot analysis showed that there was less N-WASP in the CR-<i>ACTG1</i> cells, whereas the level of Arp3 was increased in the CR-<i>ACTB</i> and CR-<i>ACTG1</i> clones, compared to controls (<a href="#ijms-21-02746-f006">Figure 6</a>E, <a href="#app1-ijms-21-02746">Figure S11</a>). We also checked the levels of those proteins under full medium conditions. The level of N-WASP was significantly statistically higher in both the CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells when compared to control clones (<a href="#app1-ijms-21-02746">Figure S12</a>). Similarly, there was an increase in the level of Arp3 in both CRISPR clone types when compared to controls. There were no notable changes in the levels of WAVE-2 between the cell types (<a href="#app1-ijms-21-02746">Figure S12</a>). These observations suggest that alterations in the leading edge dynamics could be at least partially caused by changes in the subcellular distribution and amount of the proteins studied here. </p> </div> <div class="2.7.focaladhesion(fa)formationischangedintheclonesdevoidofeitherβorγactin" title="sec"> <div class="title" tagx="title" title="title">
2.7. Focal Adhesion (FA) Formation Is Changed in the Clones Devoid of Either β or γ Actin</div> <p>Because VASP is not only found at the leading edge of a cell, but also in FAs and we have noticed differences in its presence at the leading edge between cell types, we decided to evaluate the FA formation in cells grown in full medium or under LPA or PMA stimulation conditions. For this purpose, we stimulated the cells, fixed them, and stained for two proteins present in FAs, VASP and α-parvin. Both proteins colocalized with integrin αVβ3, a known constituent of FAs (<a href="#app1-ijms-21-02746">Figure S13</a>). Closer analysis of stained cells revealed that VASP is also present in these FAs, just like α-parvin, but not only there; there is a subtle accumulation of VASP in the submembranous area of a cell, which is highlighted by arrows in <a href="#app1-ijms-21-02746">Figure S13</a>. The same was observed in nonmanipulated A375 cells stained with antibodies that recognize VASP or α-parvin and total actin (<a href="#ijms-21-02746-f007">Figure 7</a>A). In the case of α-parvin, we observed this protein solely at the tips of stress fibers. On the other hand, we noted a clear VASP localization at the tips of stress fibers, but also in submembranous aggregates that were also positive for total actin (<a href="#ijms-21-02746-f007">Figure 7</a>A). We consider these structures as nascent FAs that were most likely devoid of α-parvin at this stage of formation. Using photos of the cells with immunofluorescently detected α-parvin and VASP, we calculated and measured the number of FAs and their areas, respectively. In the case of staining with anti-VASP antibodies, some puncta were so small that we did not include measurements of their areas in order to avoid introducing any artifacts, and, therefore, the FAs’ areas were measured only for α-parvin-positive FA staining. The acquired results show that, under full medium conditions, the number of α-parvin-rich FAs was statistically significantly elevated in CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells, in comparison to control cells, but, surprisingly, their surface area was decreased only in the CR-<i>ACTG1</i> clones (<a href="#ijms-21-02746-f007">Figure 7</a>B,C). There were more VASP puncta in cells without β actin when compared to the control clones, whereas the greatest number of VASP puncta was observed in the CR-<i>ACTG1</i> cells (<a href="#ijms-21-02746-f007">Figure 7</a>B,D). LPA incubation of the CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells resulted in almost twice as many α-parvin-rich FAs in comparison to controls, whereas their surface area was increased only in the cells devoid of γ actin (<a href="#ijms-21-02746-f007">Figure 7</a>E,F). The number of VASP-rich nascent FAs and FAs was increased in both the β and γ actin knockout cells upon LPA stimulation (<a href="#ijms-21-02746-f007">Figure 7</a>E,G). We also checked the impact of PMA on the number of FAs. We noted no changes in the number of α-parvin-rich FAs in CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells in comparison to controls (<a href="#ijms-21-02746-f007">Figure 7</a>H,I). The FA surface area was bigger in the clones without β actin, but surprisingly smaller in the CR-<i>ACTG1</i> cells, compared to control cells. </p> <p>Subsequently, we compared the results for all the clones concerning the numbers and sizes of FAs upon LPA or PMA stimulation under full medium conditions by employing slope graphs. We observed the same tendency for all the tested cells, i.e., LPA and PMA incubation increased the number of α-parvin-rich FAs and PMA treatment decreased FA size (<a href="#ijms-21-02746-f008">Figure 8</a>A,C). In the case of CR-<i>ACTG1</i> cells, we noted, surprisingly, an increase in the size of α-parvin-rich FAs upon LPA stimulation when compared to controls and CR-<i>ACTB</i> cells, for which a decrease was noted (<a href="#ijms-21-02746-f008">Figure 8</a>A). Similarly, after LPA treatment, the number of VASP-rich FAs decreased in the cells devoid of γ actin and increased in the control and CR-<i>ACTB</i> cells (<a href="#ijms-21-02746-f008">Figure 8</a>B). These results indicate that FA formation differed between the clones and that CR-<i>ACTG1</i> cells had impaired FA turnover. </p> <p>Finally, we evaluated the surface area of the cells under all the tested conditions. Although there were discrepancies in the numbers and sizes of FAs between the studied clones, the cell surface areas were similar between these clones under all the tested conditions (<a href="#ijms-21-02746-f008">Figure 8</a>D). However, upon PMA stimulation, the average cell size decreased in comparison to the no-treatment and LPA conditions. </p> </div> <div class="2.8.signalingconnectedtofasisalteredintheclonesdevoidofeitherβorγactin" title="sec"> <div class="title" tagx="title" title="title">
2.8. Signaling Connected to FAs Is Altered in the Clones Devoid of Either β or γ Actin</div> <p>Consequently, to follow on the observed changes in FA formation, we decided to check the levels of some FA structural proteins that play significant roles in FA-associated signaling, using Western blotting. We estimated the levels of VASP, phosphorylated VASP (pVASP<sup>157</sup> and pVASP<sup>239</sup>, VASP forms without actin-binding activity [<a href="#B29-ijms-21-02746">29</a>]), α-parvin, and the phosphorylated forms of proto-oncogene tyrosine-protein kinase Src (Src) (pSrc<sup>530</sup>, inactive form of Src [<a href="#B30-ijms-21-02746">30</a>]) and focal adhesion kinase 1 (FAK) (pFAK<sup>397</sup>, active form of FAK [<a href="#B31-ijms-21-02746">31</a>]). In CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells, we observed elevated level of VASP under full medium, LPA, and PMA conditions (<a href="#ijms-21-02746-f009">Figure 9</a>). Although we did not observe any signal for pVASP<sup>157</sup>, we noted a lower level of pVASP<sup>239</sup> in the cells devoid of γ actin that were grown under LPA and PMA conditions (<a href="#ijms-21-02746-f009">Figure 9</a>). Interestingly, the α-parvin amount increased in the CR-<i>ACTB</i> and CR-<i>ACTG1</i> clones that grew in full medium, but decreased in the cells without γ actin upon exposure to LPA. LPA incubation upregulated the pSrc<sup>530</sup> level in β actin-deficient clones, but PMA stimulation upregulated pSrc<sup>530</sup> in the γ actin-deficient clones. Next, we looked at FAK activation and we noticed a decrease in the pFAK<sup>397</sup> level in CR-<i>ACTG1</i> cells under all the tested conditions, compared to control cells. In summary, these results clearly show that altered FA formation in CR-<i>ACTB</i> and CR-<i>ACTG1</i> clones has a direct impact on FA-associated signaling.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
3. Discussion</div> <p>There is a constantly expanding plethora of questions to be answered regarding actin isoforms’ diversification to fully understand their involvement in spatiotemporal actin cytoskeleton regulation, which is crucial for the vast majority of cellular processes [<a href="#B8-ijms-21-02746">8</a>]. In regard to melanoma cells, there is only one paper that addresses non-muscle actins’ role in these cells. That study showed, based solely on qRT-PCR analysis, that there was increased β actin expression in invasive T1C1 cells in comparison to non-invasive 1C8 cells [<a href="#B32-ijms-21-02746">32</a>]. Because there was a gap in the knowledge about non-muscle actins’ involvement in melanoma cell biology, we decided to unveil the role of both isoactins in melanoma cells’ motility. </p> <p>We noted that in genetically nonmanipulated A375 cells, β actin is predominantly present in stress fibers, whereas γ actin is found mainly as a dense mesh in the cell body but also in stress fibers. Similar non-muscle isoactin subcellular localization was also observed by Dugina and her colleagues in fibroblasts, where β actin was predominantly present at cell–cell contact sites, in stress fibers, and in circular bundles, whereas γ actin was observed as a meshwork at the cortex and lamellipodium [<a href="#B33-ijms-21-02746">33</a>]. For colon carcinoma, lung adenocarcinoma cells, and keratinocytes, Dugina et al. obtained similar results, showing that β actin is mainly found in stress fibers, whereas γ actin is localized submembranously [<a href="#B34-ijms-21-02746">34</a>]. Analogous results were obtained by another group for neuroblastoma cells [<a href="#B35-ijms-21-02746">35</a>]. On the other hand, several papers have reported β actin localization in the lamellipodium [<a href="#B36-ijms-21-02746">36</a>,<a href="#B37-ijms-21-02746">37</a>,<a href="#B38-ijms-21-02746">38</a>] and γ actin in microfilaments in the cytoplasm [<a href="#B15-ijms-21-02746">15</a>,<a href="#B37-ijms-21-02746">37</a>]. Several studies reported compensation for the loss of one non-muscle actin isoform with overexpression of another actin isoform to restore the level of total actin [<a href="#B39-ijms-21-02746">39</a>,<a href="#B40-ijms-21-02746">40</a>,<a href="#B41-ijms-21-02746">41</a>]; we observed the same phenomenon. Higher expression of β or γ actin in the CR-<i>ACTG1</i> or CR-<i>ACTB</i> clones, respectively, was corroborated by microscopic analysis. </p> <p>Intriguingly, we found out that knocking out either β or γ actin influenced the expression levels of some ABPs. The expression of several ABPs is regulated by serum response factor (SRF) [<a href="#B42-ijms-21-02746">42</a>]; however, our observations cannot be solely explained by changes in the F:G ratio leading to altered SRF signaling, resulting in up- or down-regulation of ABP expression. We justify our notion by the fact that VASP, α-parvin, and N-WASP were up- or down-regulated in CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells under various conditions, whereas the expression of these proteins is not regulated by SRF. On the other hand, we noted changed levels of cofilin and Arp3, proteins whose expression depends on SRF [<a href="#B42-ijms-21-02746">42</a>], in the cells deprived of either β or γ actin. However, in CR-<i>ACTB</i> clones, there were no changes in the F:G actin ratio. Alterations in ABPs expression levels were also noted by Latham and colleagues, who defined <i>ACTB</i>-AST, a new clinical entity manifested by microcephaly, intellectual disability, minor facial anomalies, white blood cell anomalies and thrombocytopenia caused by mutations clustered in the 3′ region of <i>ACTB</i> [<a href="#B39-ijms-21-02746">39</a>]. They noted the significantly increased expression of some ABPs: The α-actinin, myosin 2A, filamin A, and tropomyosin 4.2. Except for tropomyosin 4.2, the other proteins were recruited at a higher extent to F-actin bundles built by the mutant β actin and α-smooth muscle actin (α-SMA) in patient-derived fibroblasts, compared to control cells. Other research showed that mouse myoblasts that overexpressed β actin also exhibited elevated levels of thymosin β and actin-depolymerizing factor (ADF) [<a href="#B43-ijms-21-02746">43</a>]. Recent findings show that the cell’s different actin networks have to compete for the same restricted pool of G-actin [<a href="#B44-ijms-21-02746">44</a>,<a href="#B45-ijms-21-02746">45</a>,<a href="#B46-ijms-21-02746">46</a>]. The new, exquisite work from Antkowiak and his colleagues provides more details into this picture [<a href="#B47-ijms-21-02746">47</a>]. They showed that Arp2/3, profilin, Cap1/2, ADF/cofilin, and tropomyosin and their varying concentrations differently affected the balance between linear, formin-based, and Arp2/3 branched networks in both in vitro and in vivo models. The notion that the changes in the composition and dynamics of particular actin networks globally influence the pool of monomeric actin, and thus all F-actin structures present in the cell, was recently called "global treadmilling" by Carlier and Shekhar [<a href="#B28-ijms-21-02746">28</a>]. In light of the data presented here, it seems interesting that in the CR-<i>ACTG1</i> cells, the lamellipodial activity and Arp3 presence at the leading edge were diminished, while the formation of thick stress fiber bundles was prominently enhanced when compared to control and CR-<i>ACTB</i> cells. These observations are consistent with other studies that show that reduced activity of the Arp2/3 complex resulted in a disturbance in the formation of branched networks in favor of more extensive linear networks [<a href="#B48-ijms-21-02746">48</a>,<a href="#B49-ijms-21-02746">49</a>,<a href="#B50-ijms-21-02746">50</a>]. </p> <p>There are studies showing that β and γ actin have different affinities in terms of interaction with ABPs. For instance Müller with colleagues [<a href="#B51-ijms-21-02746">51</a>] proved that β actin more strongly activates non-muscle myosin 2C1, while γ actin preferentially activates myosin 7A. Dugina et al. [<a href="#B34-ijms-21-02746">34</a>], on the other hand, by application of co-immunoprecipitation and proximity ligation assay, showed that γ and not β actin associates with cofilin 1, PP1, WAVE2, and p34-Arc in lung adenocarcinoma A549 cells. Thus it is plausible to expect that the action of ABPs on actin cytoskeleton could be dependent on β:γ actin ratio. The issue of the capability of different isoactin monomers to copolymerize has been discussed for many years. Recently, it was shown by Bergeron and colleagues [<a href="#B11-ijms-21-02746">11</a>] that β and γ actin can copolymerize. Different β:γ actin ratio in microfilament composition could determine the filament polymerization dynamics and ABPs’ interaction kinetics and finally availability of polymerizable pool of actin monomer. In consequence, this could be mirrored in the character and dynamics of F-actin structures and finally would determine the cell behavior. The observed in this study effects of <i>ACTB</i> and <i>ACTG1</i> knock outs might be caused by either the lack of given actin isoform or the up-regulated level of the other one or by both. </p> <p>The activity of Rho-GTPases regulates the formation of F-actin-based structures (lamellipodia, filopodia, and stress fibers) and thus migration [<a href="#B52-ijms-21-02746">52</a>]. Here, however, we do not see a clear correlation between Rho-GTPase activity upon LPA and PMA treatment and the F:G ratio or the formation of lamellipodia and stress fibers in the tested clones. We believe that the observed alterations in Rac1 and RhoA activity are a downstream effect of the altered actin network dynamics in the clones devoid of β or γ actin due to two possible mechanisms (<a href="#ijms-21-02746-f010">Figure 10</a>). In the case of CR-<i>ACTB</i> cells challenged with PMA, there was more N-WASP at the leading edges of the cells, which allowed more Arp2/3 to be activated and this resulted in better lamellipodial activity. As a result, this could eventually lead to higher Rac1 activity in comparison to control and CR-<i>ACTG1</i> cells since, as discussed by Dang and Gautreau, lamellipodial activity might have a positive feedback loop on Rac1 activity [<a href="#B53-ijms-21-02746">53</a>]. The downstream localization of Rac1 in the signaling cascade is further corroborated by the fact that the presence of WAVE-2, which is normally a downstream effector of Rac1 [<a href="#B52-ijms-21-02746">52</a>], at the leading edge was unchanged under these conditions. On the other hand, in the LPA-treated CR-<i>ACTG1</i> clones, there were as many mature FAs (α-parvin-rich) as there were in the CR-<i>ACTB</i> cells, but they were bigger. Moreover, there was a smaller number of nascent FAs (VASP-rich) in CR-<i>ACTG1</i> cells than in CR-<i>ACTB</i> cells. According to Wozniak and colleagues, Mena/VASP, together with Vinculin, act at nascent FAs (initial adhesions), whereas Src and FAK are recruited to adhesion sites later during the clustering of integrins [<a href="#B54-ijms-21-02746">54</a>]. Alternatively, at nascent FAs, kindlin, together with the integrin-linked kinase (ILK):LIM and senescent cell antigen-like-containing domain protein (PINCH):parvin complex [<a href="#B55-ijms-21-02746">55</a>] and/or the migfilin:VASP complex [<a href="#B56-ijms-21-02746">56</a>], associates with integrins at first and, following FAK and Src, associates with nascent FAs [<a href="#B55-ijms-21-02746">55</a>]. Here, however, we observed α-parvin’s presence at well-developed FAs. In accordance with the literature, FA formation activates RhoA [<a href="#B57-ijms-21-02746">57</a>], but FA size may not determine the range of RhoA activation, but instead the ability to form new FAs. However, the diminished activity of RhoA in CR-<i>ACTG1</i> cells might not be solely caused by the altered FA turnover. We also assume that the extent of stress fiber formation somehow influences RhoA activity in a negative feedback loop as the highest number of thickly bundled stress fibers and lowest RhoA activity, in comparison to control cells, were noted for CR-<i>ACTG1</i> cells. We want to stress our belief that, at the core, the observed phenotypes are probably due to the different properties of actin isoforms in terms of their interactions with ABPs. Interestingly, we did not observe any changes in Cdc42 activity in response to LPA or PMA stimulation, though there is an interplay between Rho-GTPases [<a href="#B58-ijms-21-02746">58</a>]. These would be in agreement with the notion that Rac1 and RhoA are mainly affected by the changes in dynamics of integrin-based structures [<a href="#B59-ijms-21-02746">59</a>] and here we observed disturbed FA formation/turnover.</p> <p>We showed here that the studied actin isoforms have different effects on FA formation. Alterations in FA formation in cells lacking either β or γ actin were accompanied by alterations in FA-associated signaling. Coherently with our findings, others noted that in the cells isolated from <i>ACTB<sup>-/-</sup></i>-mice, there were more stress fibers and FAs than in controls [<a href="#B41-ijms-21-02746">41</a>,<a href="#B60-ijms-21-02746">60</a>]. On the other hand, it was shown for HeLa cells that down-regulation of <i>ACTB</i> caused by the knockdown of the gene that encodes the RNA-binding protein HuR was accompanied by smaller numbers of stress fibers and impaired adhesion [<a href="#B61-ijms-21-02746">61</a>]. Additionally, in contrast to our findings, Bunnell with colleagues observed that enhanced FA formation was positively correlated with impaired 2-D migration in mouse embryonic fibroblasts devoid of β actin [<a href="#B41-ijms-21-02746">41</a>]. This could, however, be explained by the use of a different cellular model. Recently, we showed that A375 cells with lowered thymosin β4 expression exhibited a higher number of FAs and this was positively correlated with impaired 2-D and 3-D migration [<a href="#B62-ijms-21-02746">62</a>]. It is definitely worthwhile to continue to study the influence of actin isoforms on FA formation and composition as the integrins, receptors of extracellular matrix (ECM) proteins and key components of FAs, are being discussed as therapeutic targets in melanoma [<a href="#B63-ijms-21-02746">63</a>] and interact with receptor tyrosine kinases [<a href="#B52-ijms-21-02746">52</a>], like epidermal growth factor receptor (EGFR) or hepatocyte growth factor receptor (c-MET), which play an important role in melanoma cells (discussed in [<a href="#B64-ijms-21-02746">64</a>]). </p> <p>Certainly, adhesion is crucial for anchorage-dependent cell motility [<a href="#B52-ijms-21-02746">52</a>]. Because of that, we examined the motility of the tested clones. We showed earlier on mesenchymally and ameboidally moving cells [<a href="#B65-ijms-21-02746">65</a>,<a href="#B66-ijms-21-02746">66</a>] that β and γ actins have the same effect on the motility of the tested cell lines; however, our previous approach was based on isoactin overexpression and the endogenous presence of both isoactins in the cell could obscure the effect of overexpression. Here, in the wound-healing experiments, which mirror collective migration, we noted that CR-<i>ACTB</i> cells moved faster than control cells, whereas the CR-<i>ACTG1</i> clones closed the wound more slowly than CR-<i>ACTB</i> and control cells. These findings were in alignment with the observed lamellipodial activity of both types of clones upon PMA exposure. It was also previously shown that lowered γ actin levels in neuroblastoma cells resulted in slower collective migration [<a href="#B35-ijms-21-02746">35</a>]. However, in terms of invasiveness, the lack of both isoactins had diminishing effects, although the clones formed an increased number of invadopodia and they were functional in terms of ECM digestion. We hypothesize that both isoactins have to be present in invadopodia, probably due to the structural features of filamentous actin or recruitment of appropriate ABPs, since our earlier studies showed that both β and γ actin are present in active invadopodia [<a href="#B67-ijms-21-02746">67</a>]. Our invasion results differ from those obtained by Dugina and colleagues [<a href="#B34-ijms-21-02746">34</a>], which showed that reduced γ actin levels were accompanied by decreased invasion potential, but reduced β actin expression increased the invasion potential. This discrepancy can again be explained by different cellular models and, thus, divergent ABP landscapes and the fact that they used short hairpin RNA (shRNA) to down-regulate the expression of the given isoactin and here we had a complete knockout of non-muscle actin genes. Nevertheless, similarly to Dugina and colleagues [<a href="#B34-ijms-21-02746">34</a>], we observed that γ actin plays a more important role than β actin, at least in melanoma cells, in motility and adhesion. </p> <p>We believe that with this study, we placed another brick in our understanding of the complex interaction between actin cytoskeleton components. We are deeply convinced that our results clearly indicate that both non-muscle actins play only partly redundant roles in melanoma cells’ adhesion and motility and that changes in the β:γ actin ratio can reprogram a cell’s fate; if the functions of β and γ actin were exactly the same in melanoma cells, the lack of one of these non-muscle actins would be compensated for by elevations in the level of the other actin and thus result in restored cellular functions. Moreover, we were able to show that the expression level of each isoactin influences the expression of different sets of proteins. Having generated stable clones that were deprived of β or γ actin, it is crucial to address several pivotal issues in future studies. One such issue is the influence of knockout of <i>ACTB</i> or <i>ACTG1</i> on the cell proliferation rate, signaling pathways involved in tumor progression, and nuclear processes regulated by actin. Alas, it is of high priority to design studies to analyze, in depth, the nature of the interactions between non-muscle actins and ABPs, such as the Arp2/3 complex, VASP, FHOD1, or Myosin IIA, as the differences between the ABPs could be at least partially responsible for the aforementioned effects. On that note, a thorough knowledge of the mechanisms that regulate melanoma cells’ motility would allow the design of new cancer therapies.</p> </div> <div class="4.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
4. Materials and Methods </div> <div class="4.1.materials" title="sec"> <div class="title" tagx="title" title="title">
4.1. Materials</div> <p>Plasmids used for inactivation of <i>ACTB</i> (catalogue number: sc-400000-NIC) and <i>ACTG1</i> (catalogue number: sc-400006-NIC) genes as well control set of plasmids (sc-437281), phorbol-12-myristate-13-acetate (PMA), lysophosphatidic acid (LPA), and cytochalasin D were bought at Santa Cruz Biotechnology Inc. (Heidelberg, Germany). Protease Inhibitors Cocktail, Phosphatase Inhibitor Cocktail 2, Phosphatase Inhibitor Cocktail 3, and Bradford reagent were from Merck (Darmstadt, Germany). Recombinant human β actin (04-CE15) was purchased from Novoprotein (Summit, NJ, USA), whereas recombinant human γ actin (TP301730) was from OriGene (Rockville, MD, USA). All other reagents were of analytical grade.</p> </div> <div class="4.2.celllinesandcellcultureconditions" title="sec"> <div class="title" tagx="title" title="title">
4.2. Cell Lines and Cell Culture Conditions</div> <p>A375 and Hs294T cell lines were obtained from the American Type Culture Collection (LGC Standards, Lomianki, Poland), WM1341D and WM9 cell lines were from Rockland Immunochemicals, Inc. (Limerick, PA, USA), whereas SK-MEL-28 were bought at CLS Cell Lines Service GmbH (Eppelheim, Germany). Cell lines were subcultured twice a week. Some experiments were conducted on PMA- and LPA-stimulated cells. Before stimulations, cells were starved for 24 h in medium without FBS (fetal bovine serum). Cells were stimulated with 1 µM LPA for 10 min or with 100 nanomolar (nM) PMA for 5 min at 37 °C in 5% CO<sub>2</sub>.</p> </div> <div class="4.3.crispr/cas9(d10a)inactivationofactbandactg1genes" title="sec"> <div class="title" tagx="title" title="title">
4.3. CRISPR/Cas9(D10A) Inactivation of ACTB and ACTG1 Genes</div> <p>A375 cells growing in a 35-mm plate were transfected with a mixture of two plasmids coding for Cas9(D10A), two specific for a gene of interest gRNAs (or scrambled gRNAs in the case of control clones), puromycin resistance, and green fluorescent protein (GFP), according to the manufacturer and with the help of Lipofectamine™ 2000 (ThermoFisher Scientific, Warsaw, Poland). The gRNAs’ sequences for <i>ACTB</i> are 5′atgtgcaaggccggcttcgc3′ and 5′gccgttgtcgacgacgagcg3′, and for <i>ACTG1</i> – 5′gcacccagtgctgctgaccg3′ and 5′cacgcgcagctcgttgtaga3′. Information about control gRNAs sequences is confidential, according to the manufacturer. Upon 24 h, the cells were trypsynized and seeded onto a 15-cm plate. On the next day, the medium was changed to selection medium containing 1 μg/mL of puromycin, which was next replenished every 2–3 days. After approximately two weeks of selection, single clones were isolated by the application of glass cylinders and silicone paste. From this moment every clone was cultivated further in the selection medium with 0.5 μg/mL puromycin. The clones were verified for the absence of β and γ actin by immunostaining, Western blot, qPCR, and gDNA analysis. We found out that upon selection of the clones there was no GFP signal. It could be caused by the fact that puromycin resistance and GFP were not coded by the same plasmid, which in turn caused most probably removal of the plasmid-encoding GFP from cells’ genome. </p> </div> <div class="4.4.invasionassay" title="sec"> <div class="title" tagx="title" title="title">
4.4. Invasion Assay</div> <p>The assay is described elsewhere [<a href="#B62-ijms-21-02746">62</a>]. Briefly, the assay was performed using Transwell™ filters (BD Biosciences, Wroclaw, Poland) coated with Matrigel™ (BD Biosciences, Wroclaw, Poland). Cells that traversed the Matrigel™ layer and attached to the lower surface of the filter were stained with Hoechst 33342 and counted under a fluorescent microscope. </p> </div> <div class="4.5.cellmigrationassays" title="sec"> <div class="title" tagx="title" title="title">
4.5. Cell Migration Assays</div> <p>For spontaneous migration assay, 1000 cells were seeded into wells of 96-well IncuCyte ImageLock plates. Migration assay was performed using IncuCyte<sup>®</sup> Live Cell Analysis Imaging System (Essen BioScience, Ltd., Royston, UK) for 72 h with images taken every 2 h. Using Manual Tracking plug-in (ImageJ, version 1.52p, F. Cordelieres, Institute Curie, Paris, France), cellular trajectory, velocity, and distance covered by a single cell were determined. End-point directionality was calculated as the ratio between straight-line displacement (distance to origin, DTO) and total path traveled by the cell (total distance, TD) [<a href="#B62-ijms-21-02746">62</a>]. For each parameters, 30 cells (10 cells per clone) per group were analyzed. </p> <p>For collective migration assay, cells were cultured in 96-well IncuCyte<sup>®</sup> ImageLock plates. Upon reaching the confluence by the cells, wounds in the cells’ monolayers were done using IncuCyte<sup>®</sup> WoundMaker. Then, the closure of the wound was followed using IncuCyte<sup>®</sup> Live Cell Analysis Imaging System. Images collected every 2 h for 48 h were used for data analysis (IncuCyte<sup>®</sup> ZOOM 2018A software, Essen BioScience, Ltd., Royston, UK), which was presented as a percent of scratch overgrown by the cells over time. </p> </div> <div class="4.6.immunocytochemistryandconfocalmicroscopy" title="sec"> <div class="title" tagx="title" title="title">
4.6. Immunocytochemistry and Confocal Microscopy</div> <p>For immunostaining assays, the cells cultured on glass coverslips were fixed with 4% formaldehyde for 20 min and then permeabilized with 0.1% Triton X-100 in phosphate-buffered saline (PBS). Nonspecific bindings were blocked by using 1% bovine serum albumin (BSA) in PBS. The coverslips were incubated overnight at 4 °C with primary antibodies diluted in blocking solution. Dilutions are shown in <a href="#app1-ijms-21-02746">Table S1</a>. Depending on the experiment, the coverslips were incubated with secondary antibodies conjugated with different Alexa-Fluor™ dyes (<a href="#app1-ijms-21-02746">Table S1</a>) and with Hoechst 33342 (ThermoFisher Scientific, Warsaw, Poland) to stain cells’ nuclei. In order to detect G-actin and F-actin, some stainings were done with the usage of Alexa Fluor™ 594-labeled DNase I and Alexa Fluor™ 488-labeled phalloidin, respectively. Both were from ThermoFisher Scientific (Warsaw, Poland) and diluted at 1:100. At the end, the coverslips were mounted on glass slides with Dako Mounting Medium (Agilent Technologies Inc., Santa Clara, CA, USA). Photos were taken and analyzed with Leica TCS SP8 Confocal Laser Scanning Microscope and Leica Application Suite X (LAS X). Some of the microphotographs are presented as negatives of grayscale mode to better present the details of the stainings. </p> <p>Plot profiles. Distribution of proteins across the cells was presented as a mean of fluorescence intensity expressed in arbitrary units (AU) along the straight line profiles (µm). Lines were created manually with quantify tools of Leica Application Suite X (LAS X). Fluorescence intensity was then automatically generated for each channel separately. Photos were taken at the same settings. Nine cells per group were analyzed. </p> <p>Area under curve graph was created by using Graphpad Prism 8 based on the leading edge dynamics’ data. Results were presented as a fold change normalized to the CR-CTRL group. </p> </div> <div class="4.7.liveimagingandkymographanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.7. Live Imaging and Kymograph Analysis</div> <p>The cells were seeded onto Nunc™ Lab-Tek™ Chambered Coverglass (ThermoFisher Scientific, Warsaw, Poland) and on the following day transfected with pCMV-LifeAct-TagRFP (ibidi GmbH, Gräfelfing, Germany) with the help of Lipofectamine™ 3000 reagent (ThermoFisher Scientific, Warsaw, Poland). On the next day, the medium was changed to medium without FBS. Then, 24 h later, the cells were stimulated with 100 nM PMA and observed with the help of Leica TCS SP8 Confocal Laser Scanning Microscope. Pictures were taken every 10 s for 5 min prior to stimulation and 10 min after addition of PMA. Obtained movies were then analyzed with ImageJ software (version 1.52p, F. Cordelieres, Institute Curie, Paris, France). Kymographs were generated along 5-pixel-wide line regions oriented in the direction of individual protrusions. Quantitative data about the leading edge position were then exported to Excel and finally area under curve (AUC) was calculated and presented as bars for every condition (<i>n</i> = 6).</p> </div> <div class="4.8.westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.8. Western Blot Analysis</div> <p>The cell lysates were prepared by scraping the cells on ice in cytoskeletal-bound protein extraction buffer (10 mM Tris-HCl pH 7.4, 100 mM NaCl, 1 mM ethylenediaminetetraacetic acid (EDTA), 1 mM ethylene glycol-bis(2-aminoethylether)-N,N,N′,N′-tetraacetic acid (EGTA), 1 mM NaF, 20 mM Na<sub>4</sub>P<sub>2</sub>O<sub>7</sub>, 2 mM Na<sub>3</sub>VO<sub>4</sub>, 1% Triton X-100, 10% glycerol, 0.1% SDS, 0.5% sodium deoxycholate) or 2-D electrophoresis cell lysis buffer (7 M urea, 2 M thiourea, 4% 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate hydrate (CHAPS), 50 mM Dithiothreitol, 0.2% BioLyte<sup>®</sup> 3–10, 1:100 serine phosphatase inhibitor, 1:100 tyrosine phosphatase inhibitor), both with addition of 1:100 protease inhibitor cocktail. The protein concentration was established by using the Bradford protein assay. The samples containing forty μg of protein were run on either 12.5% or 15% polyacrylamide gel by SDS-PAGE and transferred to nitrocellulose by wet transfer. The efficiency of the transfer and equal protein loading were determined by using the Ponceau S staining of the membrane. The membranes were blocked as next in 5% skimmed milk or 5% bovine serum albumin in Tris-buffered saline with 0.1% Tween 20 (TBS-T), depending on antibody manufacturer’s recommendations. The rest of the procedure is described elsewhere [<a href="#B62-ijms-21-02746">62</a>]. The densitometric analysis was performed using the Image Lab software (Bio-Rad). The bands were standardized to whole protein content in analyzed lane (Ponceau S) and then normalized against mean of protein expression in the control group.</p> <p>Stain-free gel (Mini-PROTEAN<sup>®</sup> TGX Stain-Free™ Precast Gel) (<a href="#app1-ijms-21-02746">Figure S2</a>) was bought at Bio-Rad (Hercules, CA, USA). The proteins upon SDS-PAGE were visualized in the "stain-free modus" with the help of the ChemiDoc MP System and Image Lab software (version 5.2.1, Bio-Rad, Hercules, CA, USA).</p> </div> <div class="4.9.quantitativepolymerasechainreaction(qpcr)" title="sec"> <div class="title" tagx="title" title="title">
4.9. Quantitative Polymerase Chain Reaction (qPCR) </div> <p>RNA was isolated with GenElute<sup>TM</sup> Mammalian Total RNA Miniprep Kit (Merck, Darmstadt, Germany). Next, RNA was treated with DNase I (Merck, Darmstadt, Germany) and reversely transcribed using High Capacity cDNA Reverse Transcription Kit (ThermoFisher Scientific, Warsaw, Poland), following the manufacturer’s instructions. Real-time PCR reactions were carried out using the PowerUp™ SYBR™ Green Master Mix (ThermoFisher Scientific, Warsaw, Poland), in accordance to the seller’s recommendations on StepOnePlus Real-Time PCR Systems device (ThermoFisher Scientific, Warsaw, Poland). The primers are listed in <a href="#app1-ijms-21-02746">Table S2</a>. Results were normalized against expression of <i>HPRT1</i>.</p> </div> <div class="4.10.2-dpage" title="sec"> <div class="title" tagx="title" title="title">
4.10. 2-D PAGE</div> <p>Proteins were extracted from A375 cells in 2-D electrophoresis cell lysis buffer (7 M urea, 2 M thiourea, 4% CHAPS, 50 mM Dithiothreitol, 0.2% BioLyte<sup>®</sup> 3–10, 1:100 Phosphatase Inhibitor Cocktail 2, 1:100 Phosphatase Inhibitor Cocktail 3). A sample containing twenty μg of protein was run for 2-D PAGE using ReadyStrip™ IPG Strips pH 4–7 (Bio-Rad, Hercules, CA, USA) for the first dimension and following for second dimension on 12.5% SDS-PAGE. Finally, the gel was stained with Deep Silver Kit (MoBiTec GmbH, Göttingen, Germany), according to the manufacturer’s instructions. The photo was taken with the ChemiDoc MP System (Bio-Rad, Hercules, CA, USA).</p> </div> <div class="4.11.rho-gtpasesassays" title="sec"> <div class="title" tagx="title" title="title">
4.11. Rho-GTPases Assays</div> <p>G-LISA Activation Assays (Cytoskeleton, Inc., Denver, CO, USA) were used to compare the levels of activated Rac1 (BK128), RhoA (BK124), and Cdc42 (BK127) GTPases in different groups of cells after LPA and PMA stimulations. Lysates were prepared according to the protocol, except for lysates for G-LISA Cdc42 Activation Assay, in which lysis buffer was a mixture of GL35:GL36 buffers in ratio 1:1. This was done upon an advice of technical support of Cytoskeleton, Inc. The Precision Red Protein Assay reagent was used for quantification of protein concentration in lysates, which was around 0.5 mg/mL. Active proteins were detected by incubation with primary antibody detecting given GTPase and appropriate secondary antibody conjugated with horseradish peroxidase (HRP). Colorimetric signal was measured at 490 nm using a microplate spectrophotometer. The data were presented as ratio between stimulated and nonstimulated (starved for 24 h) cells with normalization to the control cells.</p> </div> <div class="4.12.f:gactinratioassay" title="sec"> <div class="title" tagx="title" title="title">
4.12. F:G Actin Ratio Assay</div> <p>Cells were harvested and lysed in actin stabilizing buffer (50 mM 1,4-piperazinediethanesulfonic acid (PIPES) pH 6.9; 50 mM NaCl; 5 mM MgCl<sub>2</sub>; 5 mM EGTA; 5% glycerol; 0.1% Nonidet P40; 0.1% Triton X-100; 0.1% Tween 20; 0.1% 2-mercaptoethanol; 0.001% AntifoamC; cocktail of protease inhibitors 1:100; 1 mM ATP). Lysates were incubated for 10 min at 37 °C and centrifuged 100–300× <i>g</i> for 5 min (elimination of nonlysed cells, plasma membranes, and cellular organelles). Supernatants were centrifuged for 60 min at 100,000× <i>g</i> in Optima MAX-XP with TLA-55 rotor to separate F-actin from G-actin. G-actin was present in supernatant and F-actin in pellet. Supernatants were collected, while pellets were resuspended in actin depolymerization solution (10 μM cytochalasin D in water), and incubated for 1 h with occasional vortexing. Both fractions were analyzed using SDS-PAGE and Western blot. One-fifth of G-actin and F-actin fractions were loaded on lanes. Densitometric analysis was done as it is described in the Western blot analysis section. </p> </div> <div class="4.13.focaladhesionandcellareaanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.13. Focal Adhesion and Cell Area Analysis</div> <p>The number of focal adhesions for untreated, PMA-, or LPA-stimulated cells was counted manually using annotation tools of LasX application, according to immunostaining for α-parvin or VASP. The data are presented as a mean FAs’ number per cell. FAs’ and cell area were marked manually and measured automatically by using "Analyze &amp;gt; measure" command in ImageJ software (version 1.52p, F. Cordelieres, Institute Curie, Paris, France) (with previous scale settings). In the case of FAs’ number and cell area, 30 cells per group were analyzed. Then, area of FAs presents in nine cells chosen from earlier analyzed pool of cells were measured, which gave us the opportunity to observe the potential correlation between the number and size of FAs.</p> </div> <div class="4.14.lamellipodiumwidthandthicknessanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.14. Lamellipodium Width and Thickness Analysis</div> <p>All analyses were done using the ZEISS LSM 510 confocal microscope (Zeiss, Oberkochen, Germany) and dedicated to this microscope software. The cells were seeded onto glass coverslips, treated with 100 nM PMA for 5 min, and stained with phalloidin-Alexa Fluor™ 488 and DNase I-Alexa Fluor™ 568. The images of immunofluorescent staining were analyzed within a clone by choosing approx. 10 cells per clone in different planes of sight. The width of lamellipodium was measured by guiding a histogram through the lamellipodium using a ruler in the software. The thickness of the lamellipodium was calculated by guiding a histogram through the lamellipodium and reading the whole intensity of fluorescence through the lamellipodium.</p> </div> <div class="4.15.stressfibers’formationanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.15. Stress Fibers’ Formation Analysis</div> <p>The analysis was done using the ZEISS LSM 510 confocal microscope (Zeiss, Oberkochen, Germany) and dedicated to this microscope software. The cells were seeded onto glass coverslips and, after 24 h, treated with 1 μM LPA for 10 min. Staining and choosing of cells for analysis was done as described in the paragraph above. The number of thickly bundled stress fibers per cell was assessed by guiding a histogram through the width of the cell containing thick stress fibers and identifying the number of fluorescent values that were higher or equal to 75% of the highest fluorescence peak.</p> </div> <div class="4.16.gelatindigestionassay" title="sec"> <div class="title" tagx="title" title="title">
4.16. Gelatin Digestion Assay </div> <p>The assay was conducted in order to assess the number of active invadopodia in the cells. Poly-L-lysine coated coverslips (Corning, Corning, NY, USA) placed in a 24-well plate were rinsed with 1xPBS. Then the coverslips were fixed by incubation with 0.5% glutaraldehyde for 15 min at room temperature (RT), followed by another washing with PBS. The coverslips were then inverted onto a drop of fluorescein-conjugated gelatin solution (ThermoFisher Scientific, Warsaw, Poland) and incubated in the dark for 10 min at RT. After that, the coverslips were again inverted with the gelatin-coated side up and the residual reactive groups were quenched by adding 5 mg/mL sodium borohydride and 1 min incubation at RT, followed by washing with PBS. Before seeding the cells, the coverslips were rinsed with warm cell culture medium. The cells were seeded at the density of 30,000 per coverslip in a 24-well plate in full cell culture medium and kept at 37 °C, 5% CO<sub>2</sub> for 12 h before fixing with 4% FA. The cells were stained using phalloidin-Alexa Fluor™ 568 and Hoechst 33342 (ThermoFisher Scientific, Warsaw, Poland) to detect F-actin and cell nuclei, respectively. The images were obtained by using confocal microscopy. The number of invadopodia and the area of gelatin digestion was estimated by using the ImageJ software (version 1.52p, F. Cordelieres, Institute Curie, Paris, France). A total number of 10 cells per clone were analyzed.</p> </div> <div class="4.17.statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
4.17. Statistical Analysis</div> <p>For each analysis, groups consisting of 3 clones (three biological replicates) were analyzed. Statistical analysis and graphs were performed with the use of GraphPad Prism 7 and 8 (GraphPad Software Inc., San Diego, CA, USA). Normality distribution was analyzed with Shapiro–Wilks and D’Agostino–Pearson normality tests. Statistical significances of analyzed data were determined using the parametric or nonparametric versions of unpaired student’s t-test or ANOVA (one-way or two-way) tests, properly to obtained data set. The significance levels were set at <i>p</i> &amp;lt; 0.05 (*), <i>p</i> &amp;lt; 0.01 (**), <i>p</i> &amp;lt; 0.001 (***), and <i>p</i> &amp;lt; 0.0001 (****). All data were expressed as means ± standard deviations (SD) or ± standard of the mean (SEM) or median.</p> </div> </div> </div> <div class="back" title="back"> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>Supplementary materials can be found at <span class="uri" title="uri"><a href="https://www.mdpi.com/1422-0067/21/8/2746/s1">https://www.mdpi.com/1422-0067/21/8/2746/s1</a></span>.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="ijms-21-02746-s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, A.J.M.; methodology, N.M., E.M. (Ewa Mrówczyńska), A.M., E.M. (Ewa Mazurkiewicz), I.P., and A.J.M.; validation, N.M., E.M. (Ewa Mrówczyńska), A.M., E.M. (Ewa Mazurkiewicz), I.P., and A.J.M.; formal analysis, N.M., E.M. (Ewa Mrówczyńska), A.M., E.M. (Ewa Mazurkiewicz), I.P., and A.J.M.; investigation, N.M., E.M. (Ewa Mrówczyńska), A.M., E.M. (Ewa Mazurkiewicz), I.P., and A.J.M.; resources, A.J.M.; data curation, N.M. and A.J.M.; writing—original draft preparation, A.J.M.; writing—review and editing, A.J.M.; visualization, N.M., E.M. (Ewa Mrówczyńska), A.M., E.M. (Ewa Mazurkiewicz), I.P., and A.J.M.; supervision, A.J.M.; project administration, A.J.M.; funding acquisition, A.J.M. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This research was funded by NATIONAL SCIENCE CENTRE, POLAND, grant number 2015/17/B/NZ3/03604 (Opus 9, granted to A.J.M.). N.M. was a recipient of START scholarship (no. 050.2017) funded by Foundation for Polish Science and scholarship for young researchers (no. 444/STYP/13/2018) funded by Ministry of Science and Higher Education.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results.</p> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div><span class="array" tagx="array" title="array">   ABPs Actin-binding proteins   <i>ACTB</i> gene coding for β actin   <i>ACTG1</i> gene coding for γ actin   CR-CTRL control cells   CR-<i>ACTB</i> cells with knocked out <i>ACTB</i>   CR-<i>ACTG1</i> cells with knocked out <i>ACTG1</i>   F-actin filamentous, polymerized actin   G-actin globular, monomeric actin   FA focal adhesion   LPA lysophosphatidic acid   PMA phorbol-12-myristate-13-acetate   nFA nascent focal adhesion  </span></div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-ijms-21-02746" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perrin</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ervasti</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The actin gene family: Function follows isoform</span><span class="source" tagx="source" title="source">Cytoskeleton (Hoboken)</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">630</span><span class="lpage" tagx="lpage" title="lpage">634</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cm.20475">10.1002/cm.20475</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20737541">20737541</a></span></span></li> <li tag="ref"><a name="B2-ijms-21-02746" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurst</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimada</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasser</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nuclear Actin and Actin-Binding Proteins in DNA Repair</span><span class="source" tagx="source" title="source">Trends Cell Biol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">462</span><span class="lpage" tagx="lpage" title="lpage">476</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tcb.2019.02.010">10.1016/j.tcb.2019.02.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30954333">30954333</a></span></span></li> <li tag="ref"><a name="B3-ijms-21-02746" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanchoin</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boujemaa-Paterski</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sykes</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plastino</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Actin dynamics, architecture, and mechanics in cell motility</span><span class="source" tagx="source" title="source">Physiol. Rev.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">235</span><span class="lpage" tagx="lpage" title="lpage">263</span><span class="pub-id"><a href="https://dx.doi.org/10.1152/physrev.00018.2013">10.1152/physrev.00018.2013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24382887">24382887</a></span></span></li> <li tag="ref"><a name="B4-ijms-21-02746" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winder</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayscough</span><span class="given-names" tagx="given-names" title="given-names">K.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Actin-binding proteins</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">118</span><span class="fpage" tagx="fpage" title="fpage">651</span><span class="lpage" tagx="lpage" title="lpage">654</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.01670">10.1242/jcs.01670</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15701920">15701920</a></span></span></li> <li tag="ref"><a name="B5-ijms-21-02746" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">R.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Guardians of the actin monomer</span><span class="source" tagx="source" title="source">Eur. J. Cell Biol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">316</span><span class="lpage" tagx="lpage" title="lpage">332</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejcb.2013.10.012">10.1016/j.ejcb.2013.10.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24268205">24268205</a></span></span></li> <li tag="ref"><a name="B6-ijms-21-02746" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sibony-Benyamini</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gil-Henn</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Invadopodia: The leading force</span><span class="source" tagx="source" title="source">Eur. J. Cell Biol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">896</span><span class="lpage" tagx="lpage" title="lpage">901</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejcb.2012.04.001">10.1016/j.ejcb.2012.04.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22633185">22633185</a></span></span></li> <li tag="ref"><a name="B7-ijms-21-02746" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simiczyjew</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pietraszek-Gremplewicz</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Are non-muscle actin isoforms functionally equivalent?</span><span class="source" tagx="source" title="source">Histol. Histopathol.</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">11896</span></span></li> <li tag="ref"><a name="B8-ijms-21-02746" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ampe</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Troys</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mammalian Actins: Isoform-Specific Functions and Diseases</span><span class="source" tagx="source" title="source">Handb. Exp. Pharmacol.</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">37</span></span></li> <li tag="ref"><a name="B9-ijms-21-02746" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandekerckhove</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">At least six different actins are expressed in a higher mammal: An analysis based on the amino acid sequence of the amino-terminal tryptic peptide</span><span class="source" tagx="source" title="source">J. Mol. Biol.</span><span class="year" tagx="year" title="year">1978</span><span class="volume" tagx="volume" title="volume">126</span><span class="fpage" tagx="fpage" title="fpage">783</span><span class="lpage" tagx="lpage" title="lpage">802</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0022-2836(78)90020-7">10.1016/0022-2836(78)90020-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/745245">745245</a></span></span></li> <li tag="ref"><a name="B10-ijms-21-02746" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kashina</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Regulation of actin isoforms in cellular and developmental processes</span><span class="source" tagx="source" title="source">Semin. Cell Dev. Biol.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.semcdb.2019.12.003">10.1016/j.semcdb.2019.12.003</a></span></span></li> <li tag="ref"><a name="B11-ijms-21-02746" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergeron</span><span class="given-names" tagx="given-names" title="given-names">S.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiem</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friderici</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubenstein</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ion-dependent Polymerization Differences between Mammalian β- and γ-Nonmuscle Actin Isoforms</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">285</span><span class="fpage" tagx="fpage" title="fpage">16087</span><span class="lpage" tagx="lpage" title="lpage">16095</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M110.110130">10.1074/jbc.M110.110130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20308063">20308063</a></span></span></li> <li tag="ref"><a name="B12-ijms-21-02746" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pollard</span><span class="given-names" tagx="given-names" title="given-names">T.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Actin and Actin-Binding Proteins</span><span class="source" tagx="source" title="source">Cold Spring Harb. Perspect. Biol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">a018226</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/cshperspect.a018226">10.1101/cshperspect.a018226</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26988969">26988969</a></span></span></li> <li tag="ref"><a name="B13-ijms-21-02746" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saha</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shabalina</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kashina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Differential arginylation of actin isoforms is regulated by coding sequence-dependent degradation</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">329</span><span class="fpage" tagx="fpage" title="fpage">1534</span><span class="lpage" tagx="lpage" title="lpage">1537</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1191701">10.1126/science.1191701</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20847274">20847274</a></span></span></li> <li tag="ref"><a name="B14-ijms-21-02746" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vedula</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kashina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The makings of the ‘actin code’: Regulation of actin’s biological function at the amino acid and nucleotide level</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">131</span><span class="fpage" tagx="fpage" title="fpage">jcs215509</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.215509">10.1242/jcs.215509</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29739859">29739859</a></span></span></li> <li tag="ref"><a name="B15-ijms-21-02746" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunning</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Beta and gamma actin mRNAs are differentially located within myoblasts</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">825</span><span class="lpage" tagx="lpage" title="lpage">832</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.122.4.825">10.1083/jcb.122.4.825</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8349732">8349732</a></span></span></li> <li tag="ref"><a name="B16-ijms-21-02746" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vedula</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurosaka</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leu</span><span class="given-names" tagx="given-names" title="given-names">N.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolf</span><span class="given-names" tagx="given-names" title="given-names">Y.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shabalina</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sterling</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">D.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kashina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diverse functions of homologous actin isoforms are defined by their nucleotide, rather than their amino acid sequence</span><span class="source" tagx="source" title="source">Elife</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e31661</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.31661">10.7554/eLife.31661</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29244021">29244021</a></span></span></li> <li tag="ref"><a name="B17-ijms-21-02746" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verloes</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Donato</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masliah-Planchon</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jongmans</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdul-Raman</span><span class="given-names" tagx="given-names" title="given-names">O.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albrecht</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allanson</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunner</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertola</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chassaing</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Baraitser–Winter cerebrofrontofacial syndrome: Delineation of the spectrum in 42 cases</span><span class="source" tagx="source" title="source">Eur. J. Hum. Genet.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">292</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ejhg.2014.95">10.1038/ejhg.2014.95</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25052316">25052316</a></span></span></li> <li tag="ref"><a name="B18-ijms-21-02746" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Donato</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rump</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koenig</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Der Kaloustian</span><span class="given-names" tagx="given-names" title="given-names">V.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halal</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonntag</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hackmann</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schrock</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Severe forms of Baraitser-Winter syndrome are caused by ACTB mutations rather than ACTG1 mutations</span><span class="source" tagx="source" title="source">Eur. J. Hum. Genet.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">179</span><span class="lpage" tagx="lpage" title="lpage">183</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ejhg.2013.130">10.1038/ejhg.2013.130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23756437">23756437</a></span></span></li> <li tag="ref"><a name="B19-ijms-21-02746" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kemper</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Goeje</span><span class="given-names" tagx="given-names" title="given-names">P.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peeper</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Amerongen</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phenotype switching: Tumor cell plasticity as a resistance mechanism and target for therapy</span><span class="source" tagx="source" title="source">Cancer Res.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">74</span><span class="fpage" tagx="fpage" title="fpage">5937</span><span class="lpage" tagx="lpage" title="lpage">5941</span><span class="pub-id"><a href="https://dx.doi.org/10.1158/0008-5472.CAN-14-1174">10.1158/0008-5472.CAN-14-1174</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25320006">25320006</a></span></span></li> <li tag="ref"><a name="B20-ijms-21-02746" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quintana</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shackleton</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabel</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fullen</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">T.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Efficient tumour formation by single human melanoma cells</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">456</span><span class="fpage" tagx="fpage" title="fpage">593</span><span class="lpage" tagx="lpage" title="lpage">598</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature07567">10.1038/nature07567</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19052619">19052619</a></span></span></li> <li tag="ref"><a name="B21-ijms-21-02746" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oser</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condeelis</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The cofilin activity cycle in lamellipodia and invadopodia</span><span class="source" tagx="source" title="source">J. Cell. Biochem.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">108</span><span class="fpage" tagx="fpage" title="fpage">1252</span><span class="lpage" tagx="lpage" title="lpage">1262</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcb.22372">10.1002/jcb.22372</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19862699">19862699</a></span></span></li> <li tag="ref"><a name="B22-ijms-21-02746" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoumacher</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Louvard</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vignjevic</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cytoskeleton networks in basement membrane transmigration</span><span class="source" tagx="source" title="source">Eur. J. Cell Biol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">93</span><span class="lpage" tagx="lpage" title="lpage">99</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejcb.2010.05.010">10.1016/j.ejcb.2010.05.010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20609495">20609495</a></span></span></li> <li tag="ref"><a name="B23-ijms-21-02746" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ridley</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The small GTP-binding protein rho regulates the assembly of focal adhesions and actin stress fibers in response to growth factors</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">389</span><span class="lpage" tagx="lpage" title="lpage">399</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(92)90163-7">10.1016/0092-8674(92)90163-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1643657">1643657</a></span></span></li> <li tag="ref"><a name="B24-ijms-21-02746" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schönichen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mannherz</span><span class="given-names" tagx="given-names" title="given-names">H.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behrmann</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kühn</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silván</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoenenberger</span><span class="given-names" tagx="given-names" title="given-names">C.-A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fackler</span><span class="given-names" tagx="given-names" title="given-names">O.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raunser</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehmelt</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">FHOD1 is a combined actin filament capping and bundling factor that selectively associates with actin arcs and stress fibers</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">126</span><span class="fpage" tagx="fpage" title="fpage">1891</span><span class="lpage" tagx="lpage" title="lpage">1901</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.126706">10.1242/jcs.126706</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23444374">23444374</a></span></span></li> <li tag="ref"><a name="B25-ijms-21-02746" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicente-Manzanares</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adelstein</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horwitz</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Non-muscle myosin II takes centre stage in cell adhesion and migration</span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">778</span><span class="lpage" tagx="lpage" title="lpage">790</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm2786">10.1038/nrm2786</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19851336">19851336</a></span></span></li> <li tag="ref"><a name="B26-ijms-21-02746" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ravid</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spudich</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Control of Nonmuscle Myosins by Phosphorylation</span><span class="source" tagx="source" title="source">Annu. Rev. Biochem.</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">721</span><span class="lpage" tagx="lpage" title="lpage">759</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.bi.61.070192.003445">10.1146/annurev.bi.61.070192.003445</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1497323">1497323</a></span></span></li> <li tag="ref"><a name="B27-ijms-21-02746" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ridley</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paterson</span><span class="given-names" tagx="given-names" title="given-names">H.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnston</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diekmann</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The small GTP-binding protein rac regulates growth factor-induced membrane ruffling</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">70</span><span class="fpage" tagx="fpage" title="fpage">401</span><span class="lpage" tagx="lpage" title="lpage">410</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(92)90164-8">10.1016/0092-8674(92)90164-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1643658">1643658</a></span></span></li> <li tag="ref"><a name="B28-ijms-21-02746" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlier</span><span class="given-names" tagx="given-names" title="given-names">M.-F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shekhar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Global treadmilling coordinates actin turnover and controls the size of actin networks</span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="year" tagx="year" title="year">2017</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm.2016.172">10.1038/nrm.2016.172</a></span></span></li> <li tag="ref"><a name="B29-ijms-21-02746" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harbeck</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hüttelmaier</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlüter</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jockusch</span><span class="given-names" tagx="given-names" title="given-names">B.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Illenberger</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phosphorylation of the vasodilator-stimulated phosphoprotein regulates its interaction witn actin</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">275</span><span class="fpage" tagx="fpage" title="fpage">30817</span><span class="lpage" tagx="lpage" title="lpage">30825</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M005066200">10.1074/jbc.M005066200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10882740">10882740</a></span></span></li> <li tag="ref"><a name="B30-ijms-21-02746" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roskoski</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Src protein-tyrosine kinase structure, mechanism, and small molecule inhibitors This paper is dedicated to the memory of Prof. Donald F. Steiner (1930–2014)—Advisor, mentor, and discoverer of proinsulin</span><span class="source" tagx="source" title="source">Pharmacol. Res.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phrs.2015.01.003">10.1016/j.phrs.2015.01.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25662515">25662515</a></span></span></li> <li tag="ref"><a name="B31-ijms-21-02746" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">C.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Focal Adhesion Kinase (FAK)</span><span class="source" tagx="source" title="source">Encyclopedia of Signaling Molecules</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer International Publishing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">1800</span><span class="lpage" tagx="lpage" title="lpage">1812</span></span></li> <li tag="ref"><a name="B32-ijms-21-02746" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goidin</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamessier</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Staquet</span><span class="given-names" tagx="given-names" title="given-names">M.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmitt</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berthier-Vergnes</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ribosomal 18S RNA Prevails over Glyceraldehyde-3-Phosphate Dehydrogenase and β-Actin Genes as Internal Standard for Quantitative Comparison of mRNA Levels in Invasive and Noninvasive Human Melanoma Cell Subpopulations</span><span class="source" tagx="source" title="source">Anal. Biochem.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">295</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/abio.2001.5171">10.1006/abio.2001.5171</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11476540">11476540</a></span></span></li> <li tag="ref"><a name="B33-ijms-21-02746" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dugina</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zwaenepoel</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabbiani</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clément</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaponnier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Beta and gamma-cytoplasmic actins display distinct distribution and functional diversity</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">2980</span><span class="lpage" tagx="lpage" title="lpage">2988</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.041970">10.1242/jcs.041970</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19638415">19638415</a></span></span></li> <li tag="ref"><a name="B34-ijms-21-02746" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dugina</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khromova</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rybko</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blizniukov</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shagieva</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaponnier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopnin</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopnin</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tumor promotion by γ and suppression by β non-muscle actin isoforms</span><span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">14556</span><span class="lpage" tagx="lpage" title="lpage">14571</span><span class="pub-id"><a href="https://dx.doi.org/10.18632/oncotarget.3989">10.18632/oncotarget.3989</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26008973">26008973</a></span></span></li> <li tag="ref"><a name="B35-ijms-21-02746" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shum</span><span class="given-names" tagx="given-names" title="given-names">M.S.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasquier</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Po’uha</span><span class="given-names" tagx="given-names" title="given-names">S.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Neill</span><span class="given-names" tagx="given-names" title="given-names">G.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaponnier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunning</span><span class="given-names" tagx="given-names" title="given-names">P.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kavallaris</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">γ-Actin regulates cell migration and modulates the ROCK signaling pathway</span><span class="source" tagx="source" title="source">FASEB J.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">4423</span><span class="lpage" tagx="lpage" title="lpage">4433</span><span class="pub-id"><a href="https://dx.doi.org/10.1096/fj.11-185447">10.1096/fj.11-185447</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21908715">21908715</a></span></span></li> <li tag="ref"><a name="B36-ijms-21-02746" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoock</span><span class="given-names" tagx="given-names" title="given-names">T.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newcomb</span><span class="given-names" tagx="given-names" title="given-names">P.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herman</span><span class="given-names" tagx="given-names" title="given-names">I.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Beta actin and its mRNA are localized at the plasma membrane and the regions of moving cytoplasm during the cellular response to injury</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="year" tagx="year" title="year">1991</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">653</span><span class="lpage" tagx="lpage" title="lpage">664</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.112.4.653">10.1083/jcb.112.4.653</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1993736">1993736</a></span></span></li> <li tag="ref"><a name="B37-ijms-21-02746" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasquier</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuset</span><span class="given-names" tagx="given-names" title="given-names">M.-P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinnappan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carnell</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macmillan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kavallaris</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">γ-Actin plays a key role in endothelial cell motility and neovessel maintenance</span><span class="source" tagx="source" title="source">Vasc. Cell</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">2</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13221-014-0027-2">10.1186/s13221-014-0027-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25705373">25705373</a></span></span></li> <li tag="ref"><a name="B38-ijms-21-02746" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khaitlina</span><span class="given-names" tagx="given-names" title="given-names">S.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional specificity of actin isoforms</span><span class="source" tagx="source" title="source">Int. Rev. Cytol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">202</span><span class="fpage" tagx="fpage" title="fpage">35</span><span class="lpage" tagx="lpage" title="lpage">98</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11061563">11061563</a></span></span></li> <li tag="ref"><a name="B39-ijms-21-02746" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latham</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehmke</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reinke</span><span class="given-names" tagx="given-names" title="given-names">P.Y.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taft</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eicke</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reindl</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stenzel</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyons</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friez</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Variants in exons 5 and 6 of ACTB cause syndromic thrombocytopenia</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">4250</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-06713-0">10.1038/s41467-018-06713-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30315159">30315159</a></span></span></li> <li tag="ref"><a name="B40-ijms-21-02746" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moradi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivadasan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saal</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lüningschrör</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dombert</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathod</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dieterich</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blum</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sendtner</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Differential roles of α-, β-, and γ-actin in axon growth and collateral branch formation in motoneurons</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">216</span><span class="fpage" tagx="fpage" title="fpage">793</span><span class="lpage" tagx="lpage" title="lpage">814</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201604117">10.1083/jcb.201604117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28246119">28246119</a></span></span></li> <li tag="ref"><a name="B41-ijms-21-02746" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bunnell</span><span class="given-names" tagx="given-names" title="given-names">T.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burbach</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimizu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ervasti</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">β-Actin specifically controls cell growth, migration, and the G-actin pool</span><span class="source" tagx="source" title="source">Mol. Biol. Cell</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">4047</span><span class="lpage" tagx="lpage" title="lpage">4058</span><span class="pub-id"><a href="https://dx.doi.org/10.1091/mbc.e11-06-0582">10.1091/mbc.e11-06-0582</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21900491">21900491</a></span></span></li> <li tag="ref"><a name="B42-ijms-21-02746" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">E.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nordheim</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Linking actin dynamics and gene transcription to drive cellular motile functions</span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">353</span><span class="lpage" tagx="lpage" title="lpage">365</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm2890">10.1038/nrm2890</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20414257">20414257</a></span></span></li> <li tag="ref"><a name="B43-ijms-21-02746" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peckham</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wells</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zicha</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunn</span><span class="given-names" tagx="given-names" title="given-names">G.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Specific changes to the mechanism of cell locomotion induced by overexpression of beta-actin</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">1367</span><span class="lpage" tagx="lpage" title="lpage">1377</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11257002">11257002</a></span></span></li> <li tag="ref"><a name="B44-ijms-21-02746" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suarez</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovar</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Internetwork competition for monomers governs actin cytoskeleton organization</span><span class="source" tagx="source" title="source">Nat. Rev. Mol. Cell Biol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">799</span><span class="lpage" tagx="lpage" title="lpage">810</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrm.2016.106">10.1038/nrm.2016.106</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27625321">27625321</a></span></span></li> <li tag="ref"><a name="B45-ijms-21-02746" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">T.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christensen</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barone</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suarez</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sirotkin</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovar</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Homeostatic actin cytoskeleton networks are regulated by assembly factor competition for monomers</span><span class="source" tagx="source" title="source">Curr. Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">579</span><span class="lpage" tagx="lpage" title="lpage">585</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cub.2014.01.072">10.1016/j.cub.2014.01.072</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24560576">24560576</a></span></span></li> <li tag="ref"><a name="B46-ijms-21-02746" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murugesan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beach</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meinhardt</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madison</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betzig</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Formin-generated actomyosin arcs propel t cell receptor microcluster movement at the immune synapse</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">215</span><span class="fpage" tagx="fpage" title="fpage">383</span><span class="lpage" tagx="lpage" title="lpage">399</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.201603080">10.1083/jcb.201603080</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27799367">27799367</a></span></span></li> <li tag="ref"><a name="B47-ijms-21-02746" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antkowiak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillotin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boiero Sanders</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colombo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincentelli</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michelot</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sizes of actin networks sharing a common environment are determined by the relative rates of assembly</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">17</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e3000317</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.3000317">10.1371/journal.pbio.3000317</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31181075">31181075</a></span></span></li> <li tag="ref"><a name="B48-ijms-21-02746" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiedemann</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuurman</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vale</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular requirements for actin-based lamella formation in Drosophila S2 cells</span><span class="source" tagx="source" title="source">J. Cell Biol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">162</span><span class="fpage" tagx="fpage" title="fpage">1079</span><span class="lpage" tagx="lpage" title="lpage">1088</span><span class="pub-id"><a href="https://dx.doi.org/10.1083/jcb.200303023">10.1083/jcb.200303023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12975351">12975351</a></span></span></li> <li tag="ref"><a name="B49-ijms-21-02746" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholson-Dykstra</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgs</span><span class="given-names" tagx="given-names" title="given-names">H.N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Arp2 depletion inhibits sheet-like protrusions but not linear protrusions of fibroblasts and lymphocytes</span><span class="source" tagx="source" title="source">Cell Motil. Cytoskeleton</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">904</span><span class="lpage" tagx="lpage" title="lpage">922</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/cm.20312">10.1002/cm.20312</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18720401">18720401</a></span></span></li> <li tag="ref"><a name="B50-ijms-21-02746" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asokan</span><span class="given-names" tagx="given-names" title="given-names">S.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berginski</span><span class="given-names" tagx="given-names" title="given-names">M.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haynes</span><span class="given-names" tagx="given-names" title="given-names">E.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharpless</span><span class="given-names" tagx="given-names" title="given-names">N.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffith</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bear</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Arp2/3 is critical for lamellipodia and response to extracellular matrix cues but is dispensable for chemotaxis</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">148</span><span class="fpage" tagx="fpage" title="fpage">973</span><span class="lpage" tagx="lpage" title="lpage">987</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2011.12.034">10.1016/j.cell.2011.12.034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22385962">22385962</a></span></span></li> <li tag="ref"><a name="B51-ijms-21-02746" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diensthuber</span><span class="given-names" tagx="given-names" title="given-names">R.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chizhov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claus</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heissler</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Preller</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taft</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manstein</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Distinct Functional Interactions between Actin Isoforms and Nonsarcomeric Myosins</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e70636</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0070636">10.1371/journal.pone.0070636</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23923011">23923011</a></span></span></li> <li tag="ref"><a name="B52-ijms-21-02746" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ridley</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rho GTPase signalling in cell migration</span><span class="source" tagx="source" title="source">Curr. Opin. Cell Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">112</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ceb.2015.08.005">10.1016/j.ceb.2015.08.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26363959">26363959</a></span></span></li> <li tag="ref"><a name="B53-ijms-21-02746" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dang</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gautreau</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evidence for a cell cycle checkpoint that senses branched actin in the lamellipodium</span><span class="source" tagx="source" title="source">BioEssays</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">1021</span><span class="lpage" tagx="lpage" title="lpage">1024</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/bies.201200119">10.1002/bies.201200119</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23080513">23080513</a></span></span></li> <li tag="ref"><a name="B54-ijms-21-02746" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wozniak</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modzelewska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keely</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Focal adhesion regulation of cell behavior</span><span class="source" tagx="source" title="source">Biochim. Biophys. Acta Mol. Cell Res.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">1692</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">119</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbamcr.2004.04.007">10.1016/j.bbamcr.2004.04.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15246682">15246682</a></span></span></li> <li tag="ref"><a name="B55-ijms-21-02746" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schiller</span><span class="given-names" tagx="given-names" title="given-names">H.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fässler</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanosensitivity and compositional dynamics of cell–matrix adhesions</span><span class="source" tagx="source" title="source">EMBO Rep.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">509</span><span class="lpage" tagx="lpage" title="lpage">519</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/embor.2013.49">10.1038/embor.2013.49</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23681438">23681438</a></span></span></li> <li tag="ref"><a name="B56-ijms-21-02746" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horton</span><span class="given-names" tagx="given-names" title="given-names">E.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Humphries</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Askari</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Humphries</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The integrin adhesome network at a glance</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">4159</span><span class="lpage" tagx="lpage" title="lpage">4163</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.192054">10.1242/jcs.192054</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27799358">27799358</a></span></span></li> <li tag="ref"><a name="B57-ijms-21-02746" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burridge</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guilluy</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Focal adhesions, stress fibers and mechanical tension</span><span class="source" tagx="source" title="source">Exp. Cell Res.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">343</span><span class="fpage" tagx="fpage" title="fpage">14</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.yexcr.2015.10.029">10.1016/j.yexcr.2015.10.029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26519907">26519907</a></span></span></li> <li tag="ref"><a name="B58-ijms-21-02746" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchsbaum</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rho activation at a glance</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">120</span><span class="fpage" tagx="fpage" title="fpage">1149</span><span class="lpage" tagx="lpage" title="lpage">1152</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.03428">10.1242/jcs.03428</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17376960">17376960</a></span></span></li> <li tag="ref"><a name="B59-ijms-21-02746" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mannherz</span><span class="given-names" tagx="given-names" title="given-names">H.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Molecular Basis of Directional Cell Migration</span><span class="source" tagx="source" title="source">Cell Signaling and Growth Factors in Development</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Wiley-VCH Verlag GmbH</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Weinheim, Germany</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">321</span><span class="lpage" tagx="lpage" title="lpage">345</span></span></li> <li tag="ref"><a name="B60-ijms-21-02746" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tondeleir</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambrechts</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonckheere</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doll</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandamme</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakkali</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterschoot</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemaistre</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debeir</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cells Lacking -Actin are Genetically Reprogrammed and Maintain Conditional Migratory Capacity</span><span class="source" tagx="source" title="source">Mol. Cell. Proteomics</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">255</span><span class="lpage" tagx="lpage" title="lpage">271</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/mcp.M111.015099">10.1074/mcp.M111.015099</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22448045">22448045</a></span></span></li> <li tag="ref"><a name="B61-ijms-21-02746" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dormoy-Raclet</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ménard</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clair</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurban</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazroui</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Marco</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Roretz</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pause</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallouzi</span><span class="given-names" tagx="given-names" title="given-names">I.-E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The RNA-binding protein HuR promotes cell migration and cell invasion by stabilizing the beta-actin mRNA in a U-rich-element-dependent manner</span><span class="source" tagx="source" title="source">Mol. Cell. Biol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">5365</span><span class="lpage" tagx="lpage" title="lpage">5380</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MCB.00113-07">10.1128/MCB.00113-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17548472">17548472</a></span></span></li> <li tag="ref"><a name="B62-ijms-21-02746" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makowiecka</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malek</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazurkiewicz</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mrówczyńska</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Thymosin β4 Regulates Focal Adhesion Formation in Human Melanoma Cells and Affects Their Migration and Invasion</span><span class="source" tagx="source" title="source">Front. Cell Dev. Biol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">304</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fcell.2019.00304">10.3389/fcell.2019.00304</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31921836">31921836</a></span></span></li> <li tag="ref"><a name="B63-ijms-21-02746" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rofstad</span><span class="given-names" tagx="given-names" title="given-names">E.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Integrins as therapeutic targets in the organ-specific metastasis of human malignant melanoma</span><span class="source" tagx="source" title="source">J. Exp. Clin. Cancer Res.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">92</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13046-018-0763-x">10.1186/s13046-018-0763-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29703238">29703238</a></span></span></li> <li tag="ref"><a name="B64-ijms-21-02746" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dratkiewicz</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pietraszek-Gremplewicz</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simiczyjew</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Gefitinib or lapatinib with foretinib synergistically induce a cytotoxic effect in melanoma cell lines</span><span class="source" tagx="source" title="source">Oncotarget</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">18254</span><span class="lpage" tagx="lpage" title="lpage">18268</span><span class="pub-id"><a href="https://dx.doi.org/10.18632/oncotarget.24810">10.18632/oncotarget.24810</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29719603">29719603</a></span></span></li> <li tag="ref"><a name="B65-ijms-21-02746" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simiczyjew</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Popow-Woźniak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malicka-Błaszkiewicz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of overexpression of β- and γ-actin isoforms on actin cytoskeleton organization and migration of human colon cancer cells</span><span class="source" tagx="source" title="source">Histochem. Cell Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">142</span><span class="fpage" tagx="fpage" title="fpage">307</span><span class="lpage" tagx="lpage" title="lpage">322</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00418-014-1199-9">10.1007/s00418-014-1199-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24682235">24682235</a></span></span></li> <li tag="ref"><a name="B66-ijms-21-02746" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simiczyjew</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dratkiewicz</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Involvement of β- and γ-actin isoforms in actin cytoskeleton organization and migration abilities of bleb-forming human colon cancer cells</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0173709</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0173709">10.1371/journal.pone.0173709</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28333953">28333953</a></span></span></li> <li tag="ref"><a name="B67-ijms-21-02746" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simiczyjew</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ampe</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malicka-Błaszkiewicz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Troys</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Active invadopodia of mesenchymally migrating cancer cells contain both β and γ cytoplasmic actin isoforms</span><span class="source" tagx="source" title="source">Exp. Cell Res.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">339</span><span class="fpage" tagx="fpage" title="fpage">206</span><span class="lpage" tagx="lpage" title="lpage">219</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.yexcr.2015.11.003">10.1016/j.yexcr.2015.11.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26548725">26548725</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Differing distribution of non-muscle isoactins in A375 cells. (<b>A</b>) The cells were stained with antibodies detecting β and γ actin isoforms. Pink arrows point at invadopodia, red ones at stress fibers and blue ones at lamellipodia. (<b>B</b>) A375 cells growing on coverslips were fixed and stained to detect β or γ actin and globular actin by using appropriate antibodies and DNase I coupled to Alexa Fluor™ 594. Red arrows highlight stress fibers and blue ones lamellipodia. (<b>C</b>) Total protein extract from A375 cells (twenty μg of protein) was subjected to 2-D PAGE. The gel was stained with silver. (<b>D</b>) Immunocytochemically stained cells to detect β and γ actin. Three lines were drawn on a merged photo of the cell and the fluorescence histograms representing signal intensities for every fluorochrome were prepared on this basis.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Successful inactivation of <i>ACTB</i> and <i>ACTG1</i> genes and altered actin cytoskeleton architecture in CR-<i>ACTB</i> [clones with inactivated <i>ACTB</i> with the help of CRISPR/Cas9(D10A) technique] and CR-<i>ACTG1</i> [clones with inactivated <i>ACTG1</i> with the help of CRISPR/Cas9(D10A) technique] clones. (<b>A</b>) Western blot analysis of control cells and clones without either β or γ actin. To identify isoactins and total actin, appropriate antibodies were used. Corresponding Ponceau S stainings of membranes are shown in <a href="#app1-ijms-21-02746">Figure S6</a>. Forty μg of protein was loaded on every lane. Green rectangles represent statistically higher level of protein in comparison to control cells. (<b>B</b>) Densitometric analysis of β, γ, and total actin level in tested clones done on membranes shown in (<b>A</b>) (<i>n</i> = 3). (<b>C</b>) The qRT-PCR analysis of <i>ACTB</i> and <i>ACTG1</i> expression level in CR-<i>ACTB</i> and CR-<i>ACTG1</i> clones (<i>n</i> = 9). (<b>D</b>) Clones devoid of β or γ actin were immunostained with antibodies recognizing non-muscle actins. All pictures were taken at the same settings used during microscopic observations. Pictures of two other clones for every condition are shown in <a href="#app1-ijms-21-02746">Figure S4</a>. (<b>E</b>) Representative plots presenting fluorescence distribution of stained isoactins in tested clones shown in (<b>D</b>). Quantitative analysis of area under curve (AUC) from plots (<i>n</i> = 6). (<b>F</b>) Clones were immunostained with anti-β and anti-γ actin antibodies to show cellular distribution of both isoactins. Red arrows point at stress fibers and green ones at actin mesh. (<b>G</b>) Western blot analysis of filamentous and globular actin levels in clones without β or γ actin growing on coverslips for 72 h. Nitrocellulose membranes were incubated with mouse anti-total actin antibodies. Quantitative estimation of filamentous (<b>F</b>) to globular (<b>G</b>) ratio (<i>n</i> = 8–9). (<b>H</b>) Clones were stained with fluorescently labelled phalloidin and DNase I to detect F- and G-actin. White arrows highlight F-actin accumulation. Results are expressed as the mean ±SD (<b>B</b>, <b>E</b>, <b>G</b>) or ±SEM (<b>C</b>), <i>p</i> ≤ 0.05 (*), <i>p</i> ≤ 0.01 (**), <i>p</i> ≤ 0.001 (***).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Cells devoid of β or γ actin had impaired invasion abilities, but only CR-<i>ACTG1</i> cells’ 2-D migration was affected. (<b>A</b>–<b>D</b>) Spontaneous 2-D migration of tested cells. The cells of studied clones growing on IncuCyte<sup>®</sup> ImageLock plates were recorded over 72 h with IncuCyte<sup>®</sup> Live Cell Analysis Imaging System (<i>n</i> = 30). (<b>A</b>) Trajectories of single cells’ migration. (<b>B</b>) Calculated covered distances of the cells. (<b>C</b>) Estimation of velocity and (<b>D</b>) directionality of cells’ migration. (<b>E</b>) Wound healing assay was performed on control (CR-CTRL), CR-<i>ACTB,</i> and CR-<i>ACTG1</i> cells to analyze collective migration over 72 h (<i>n</i> = 3). (<b>F</b>) Analysis of 3-D migration/invasion of control cells and cells without β or γ actin (<i>n</i> = 16). The cells were seeded onto Matrigel™ gel placed in the upper compartment of a Transwell™ and to the lower compartment a medium enriched in serum was added. (<b>G</b>) Gelatin digestion assay. The cells seeded on fluorescently labeled gelatin were 12 h later fixed and stained with Alexa Fluor™ 488 phalloidin. Bright spots represent digested gelatin by cells. Pink arrows point at invadopodia. (<b>H</b>) Quantitative analysis of invadopodia number and (<b>I</b>) dimensions of digested area (<i>n</i> = 36). (<b>J</b>) Cells after fixation were stained with antibodies recognizing cofilin and cortactin. Across the area rich in invadopodia a line was drawn and intensities of fluorescence for detected cortactin and cofilin were plotted on corresponding graphs. Pink arrows point at invadopodia. (<b>K</b>) Western blot analysis of cell lysates. Membranes were probed with antibodies detecting cofilin and cofilin phosphorylated at Ser3 (p-cofilin<sup>3</sup>). Corresponding densitometric analysis and Ponceau S stainings of membranes are shown in <a href="#app1-ijms-21-02746">Figure S8</a>. Forty μg of protein was loaded on every lane. Green rectangles represent statistically higher level of protein in comparison to control cells. (<b>L</b>) Calculated p-cofilin<sup>3</sup>:cofilin ratio (<i>n</i> = 3). Results are expressed as the mean ± SD (<b>B</b>–<b>F</b>,<b>H</b>,<b>K</b>) or median (<b>I</b>), <i>p</i> ≤ 0.05 (*), <i>p</i> ≤ 0.01 (**), <i>p</i> ≤ 0.0001 (****).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Stress fibers’ formation is disturbed in clones devoid of β or γ actin. (<b>A</b>) CR-CTRL, CR-<i>ACTB,</i> and CR-<i>ACTG1</i> cells upon stimulation with LPA were fixed and stained with Alexa Fluor™ 488 phalloidin. Red arrows point at stress fibers. (<b>B</b>) The number of thick stress fibers was calculated for every condition and presented as a bar chart (<i>n</i> = 61–81). (<b>C</b>) Activity of RhoA, Rac1, and Cdc42 was estimated after lysophosphatidic acid (LPA) stimulation (<i>n</i> = 3). (<b>D</b>) F:G actin ratio was evaluated in the cells on the base of Western blot analysis of F- and G-actin levels in studied clones (<i>n</i> = 9). Membranes were incubated with anti-total actin antibodies. (<b>E</b>) The cells were immunostained with antibodies detecting the human FH1/FH2 domain-containing protein 1 (FHOD1) and Myosin IIa. Yellow arrows highlight Myosin IIa accumulation. (<b>F</b>) Plots of averaged (<i>n</i> = 9) and individual fluorescence intensity distribution for FHOD1 and Myosin IIa staining across a cell. (<b>G</b>) Western blot analysis of cell lysates. Membranes were probed for FHOD1, Myosin IIa, and myosin light chain phosphorylated at Thr18 and Ser19 (pMLC<sup>18/19</sup>). Corresponding densitometric analysis and Ponceau S stainings of membranes are shown in <a href="#app1-ijms-21-02746">Figure S9</a>. Forty μg of protein was loaded on every lane. (<b>H</b>) The cells were stained with anti-pMLC<sup>19</sup> antibodies and fluorescently labeled phalloidin. Results are expressed as the mean ± SD (<b>C</b>,<b>D</b>) or ± SEM (<b>B</b>), <i>p</i> ≤ 0.05 (*), <i>p</i> ≤ 0.01 (**), <i>p</i> ≤ 0.0001 (****).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Lamellipodial dynamics are altered in cells devoid of β or γ actin. (<b>A</b>) Control clones and clones devoid β or γ actin after 5 min of incubation with phorbol-12-myristate-13-acetate (PMA) were fixed and F-actin was stained with the help of Alexa Fluor™ 488 phalloidin. (<b>B</b>) Quantification of leading edge width upon incubation with PMA (<i>n</i> = 32–34). (<b>C</b>) Estimation of leading edge thickness of cells exposed to PMA (<i>n</i> = 23–32). (<b>D</b>) Representative movie frame with marked lines from which kymographs were created. (<b>E</b>) Plots showing positon of the leading edge represented by F-actin detection under control and PMA exposition conditions (<i>n</i> = 6). (<b>F</b>) Calculated AUC from plots presented in (<b>E</b>). (<b>G</b>) Estimation of activity of RhoA, Rac1, and Cdc42 was upon PMA stimulation (<i>n</i> = 3). (<b>H</b>) F:G actin ratio estimation (<i>n</i> = 9). The membranes were probed for total actin. Blue arrows point at lamellipodia. Results are expressed as the mean ± SD (<b>B</b>,<b>C</b>,<b>F</b>–<b>H</b>) or ± SEM (<b>E</b>), <i>p</i> ≤ 0.05 (*), <i>p</i> ≤ 0.01 (**), <i>p</i> ≤ 0.001 (***), <i>p</i> ≤ 0.0001 (****).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Changed distribution of actin binding proteins (ABPs) within leading edge upon lamellipodium formation challenge. (<b>A</b>) The cells were stained with vasodilator-stimulated phosphoprotein (VASP), actin-related protein 3 (Arp3), and neural Wiskott-Aldrich syndrome protein (N-WASP), recognizing antibodies. Yellow arrows point at circular dorsal ruffles. (<b>B</b>) Plots of averaged fluorescence intensity distribution of stained VASP, Arp3, and N-WASP. Lines were drawn from pericentral region of a cell (position 0 μm) towards the border of a cell (10 μm) (<i>n</i> = 9). Representative white lines are shown on merged microphotographs in (<b>A</b>). Black arrow points at the range for which there is a statistically significant difference for CR-<i>ACTG1</i> cells in comparison to control cells. (<b>C</b>) The cells were stained using specific antibodies to detect Wiskott-Aldrich syndrome protein family member 2 (WAVE-2) and Arp3. Yellow arrows point at circular dorsal ruffles. (<b>D</b>) Plots of averaged fluorescence intensity distribution for WAVE-2 from the center of a cell (position 0 μm) towards the border of a cell (position 15 μm). Representative white lines are shown on merged microphotographs in (<b>C</b>). (<b>E</b>) Cell lysates were analyzed by Western blotting. Membranes were incubated with antibodies directed against N-WASP, WAVE-2, and Arp3. Corresponding densitometric analysis and Ponceau S stainings of membranes are shown in <a href="#app1-ijms-21-02746">Figure S11</a>. Forty μg of protein was loaded on every lane. Green and red rectangles represent statistically higher and lower levels of protein, respectively, in comparison to control cells. Results are expressed as the mean only, <i>p</i> ≤ 0.05 (*), <i>p</i> ≤ 0.01 (**).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Focal adhesion formation is altered in clones devoid of β or γ actin. (<b>A</b>) Immunostaining detecting α-parvin, VASP, and total actin. The genetically nonmanipulated A375 cells were grown under full medium conditions. White arrows point at focal adhesions, while yellows ones at nascent focal adhesions (FAs)/accumulation of VASP. (<b>B</b>,<b>E</b>,<b>H</b>) The clones growing either in full medium or incubated with LPA or PMA were fixed and stained with antibodies recognizing α-parvin or VASP. (<b>C</b>,<b>F</b>,<b>I</b>) The number of α-parvin-rich focal adhesions as well as their surface area were calculated for every tested condition (<i>n</i> = 30). (<b>D</b>,<b>G</b>) The number of VASP-rich focal adhesions was calculated (<i>n</i> = 30). Results are expressed as the mean ± SD, <i>p</i> ≤ 0.05 (*), <i>p</i> ≤ 0.01 (**), <i>p</i> ≤ 0.001 (***), <i>p</i> ≤ 0.0001 (****).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>Trends in FAs’ formation by the studied clones and their surface area. (<b>A</b>–<b>C</b>) Slope graphs presenting trends upon LPA or PMA stimulation in number and surface area of focal adhesion detected either by staining with α-parvin or VASP-detecting antibodies (<i>n</i> = 30). (<b>D</b>) Clones’ cells surface area upon LPA and PMA stimulation was calculated and presented as a bar chart (<i>n</i> = 30). Results are expressed as the mean ± SD (<b>A</b>–<b>C</b>) or ± SEM (<b>D</b>), <i>p</i> ≤ 0.0001 (****).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9</span></a></div>   <p>Changed FAs’ signaling in CR-CTRL, CR-<i>ACTB,</i> and CR-<i>ACTG1</i> clones. Western blot analysis of control clones and clones without given isoactin. Corresponding Ponceau S stainings of membranes are shown in <a href="#app1-ijms-21-02746">Figure S14</a>. Forty μg of protein was loaded on every lane. Membranes were probed for proteins found within focal adhesions and playing a significant role in FAs’ signaling: VASP, phosphorylated at Ser157 VASP (pVASP<sup>157</sup>), phosphorylated at Ser239 VASP (pVASP<sup>239</sup>), α-parvin, phosphorylated at Tyr530 proto-oncogene tyrosine-protein kinase Src (pSrc<sup>530</sup>), and phosphorylated at Tyr397 focal adhesion kinase 1 (pFAK<sup>397</sup>). Densitometric analysis of studied proteins levels in tested clones was done and presented as bar charts (<i>n</i> = 3–6). Green and red rectangles represent statistically higher and lower level of protein, respectively, in comparison to control cells. Results are expressed as the mean ± SD, <i>p</i> ≤ 0.05 (*), <i>p</i> ≤ 0.01 (**), <i>p</i> ≤ 0.001 (***).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 10</span></a></div>   <p>Proposed mechanism of influence of disturbed actin organization in CR-<i>ACTB</i> and CR-<i>ACTG1</i> cells on melanoma cells’ motility and adhesion. Cells devoid of β actin that grew in full medium exhibited more nascent (VASP-rich) focal adhesions (nFAs) and mature (VASP- and α-parvin-rich) focal adhesions (FAs) than did CR-CTRL cells. Although the CR-<i>ACTG1</i> and CR-<i>ACTB</i> cells possess more mature FAs under the same conditions than control cells, FAs of CR-<i>ACTG1</i> cells are smaller. Moreover, CR-<i>ACTG1</i> cells have many more nascent FAs than both control and CR-<i>ACTB</i> cells. Alterations in FA dynamics, stress fiber formation, and the distribution of some ABPs at the leading edge results in changes in Rac1 activation, improved 2-D migration, and impaired invasion in CR-<i>ACTB</i> cells. In the case of CR-<i>ACTG1</i> cells, the impaired FA turnover, increased number of thick stress fibers, and decreased level of Arp3 at the leading edge led to lowered activation status of RhoA and finally to impeded 2-D and 3-D migration. For details, please see the discussion. In summary, the lack of γ actin has more severe effects on melanoma cells than the lack of β actin. We believe that changes in the formation/turnover of FAs might be caused by the different properties of non-muscle actins’ interactions with ABPs.</p>   </div> </div>  </body></html>